
<!doctype html>
<html lang="en">
<head>
<meta charset="utf-8" />
<title>Xylaria_sp._Z50 - 86 region(s) - antiSMASH results</title>
<style>img {width: 20px; height: 20px;}</style>
<link rel="stylesheet" type="text/css" href="css/fungi.css">
<meta property="og:title" content="Xylaria_sp._Z50 - 59 record(s) - 86 region(s)" />
<meta property="og:description" content="">
<meta property="og:image" content="https://antismash.secondarymetabolites.org/images/fungi_logo.png" />
<meta property="og:image:alt" content="antiSMASH logo" />
<meta property="og:image:width" content="600" />
<meta property="og:image:height" content="600" />
<meta property="og:url" content="https://fungismash.secondarymetabolites.org/upload/fungi-3e96dfc3-c1f2-40d2-b797-2e3f61ce8aea/index.html" />
<meta property="og:type" content="website" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:site" content="@antismash_dev" />
</head>

<body>
<div id="header">
<nav>
<div class="main-link">
<div>
<a href="https://fungismash.secondarymetabolites.org/">
<img src="images/fungi_antismash_logo.svg" alt="antiSMASH logo" style="width:40px;height:unset;">
</a>
</div>
<div>
<a href="https://fungismash.secondarymetabolites.org/">
antiSMASH version 7.1.0
</a>
</div>
</div>
<div class="custom-description"></div>
<div class="ancillary-links">
<div class="ancillary-link dropdown-menu" id="download-dropdown"><a href="#" id="download-dropdown-link"><img src="images/download.svg" alt="download"> &nbsp; Download</a>
<ul class="dropdown-options">

<li><a href="Xylaria_sp._Z50.zip">Download all results</a></li>

<li><a href="Xylaria_sp._Z50.gbk">Download GenBank summary file</a></li>
<li><a href="Xylaria_sp._Z50.json">Download JSON results file</a></li>

<li><a href="fungi-3e96dfc3-c1f2-40d2-b797-2e3f61ce8aea.log">Download log file</a></li>

</ul>
</div>
<div class="ancillary-link"><a href="https://fungismash.secondarymetabolites.org/#!/about"><img src="images/about.svg" alt="about"> &nbsp; About</a></div>
<div class="ancillary-link"><a href="https://docs.antismash.secondarymetabolites.org/"><img src="images/help.svg" alt="help"> &nbsp; Help</a></div>
<div class="ancillary-link"><a href="https://fungismash.secondarymetabolites.org/#!/contact"><img src="images/contact.svg" alt="contact"> &nbsp; Contact</a></div>
</div>
</nav>
<div id="buttons">
<span id="region-type">Select genomic region:</span>
<div id="regionbuttons">
<div class="arrow-left" id="prev-region"></div>
<div class="regbutton" style="min-width: inherit;"><a href="#">Overview</a></div>
<div style="display: flex; flex-wrap: wrap">


<div class="regbutton PKS T1PKS r1c1">
<a href="#r1c1">1.1</a>
</div>

<div class="regbutton terpene terpene r1c2">
<a href="#r1c2">1.2</a>
</div>

<div class="regbutton terpene terpene r1c3">
<a href="#r1c3">1.3</a>
</div>



<div class="regbutton terpene terpene r2c1">
<a href="#r2c1">2.1</a>
</div>

<div class="regbutton NRPS NRPS-like r2c2">
<a href="#r2c2">2.2</a>
</div>



<div class="regbutton RiPP fungal-RiPP-like r3c1">
<a href="#r3c1">3.1</a>
</div>

<div class="regbutton PKS T1PKS r3c2">
<a href="#r3c2">3.2</a>
</div>

<div class="regbutton RiPP fungal-RiPP-like r3c3">
<a href="#r3c3">3.3</a>
</div>



<div class="regbutton PKS T1PKS r4c1">
<a href="#r4c1">4.1</a>
</div>

<div class="regbutton NRPS NRPS-like r4c2">
<a href="#r4c2">4.2</a>
</div>



<div class="regbutton RiPP fungal-RiPP-like r5c1">
<a href="#r5c1">5.1</a>
</div>

<div class="regbutton NRPS NRPS-like r5c2">
<a href="#r5c2">5.2</a>
</div>

<div class="regbutton NRPS NRPS-like r5c3">
<a href="#r5c3">5.3</a>
</div>



<div class="regbutton NRPS NRPS-like r6c1">
<a href="#r6c1">6.1</a>
</div>

<div class="regbutton PKS T1PKS r6c2">
<a href="#r6c2">6.2</a>
</div>



<div class="regbutton terpene terpene r8c1">
<a href="#r8c1">8.1</a>
</div>

<div class="regbutton PKS T1PKS r8c2">
<a href="#r8c2">8.2</a>
</div>

<div class="regbutton hybrid r8c3">
<a href="#r8c3">8.3</a>
</div>



<div class="regbutton PKS T1PKS r9c1">
<a href="#r9c1">9.1</a>
</div>



<div class="regbutton PKS T1PKS r10c1">
<a href="#r10c1">10.1</a>
</div>

<div class="regbutton hybrid r10c2">
<a href="#r10c2">10.2</a>
</div>



<div class="regbutton NRPS NRPS r12c1">
<a href="#r12c1">12.1</a>
</div>

<div class="regbutton RiPP fungal-RiPP-like r12c2">
<a href="#r12c2">12.2</a>
</div>



<div class="regbutton RiPP fungal-RiPP-like r13c1">
<a href="#r13c1">13.1</a>
</div>

<div class="regbutton terpene terpene r13c2">
<a href="#r13c2">13.2</a>
</div>

<div class="regbutton RiPP fungal-RiPP r13c3">
<a href="#r13c3">13.3</a>
</div>



<div class="regbutton PKS T1PKS r16c1">
<a href="#r16c1">16.1</a>
</div>



<div class="regbutton PKS T3PKS r17c1">
<a href="#r17c1">17.1</a>
</div>



<div class="regbutton PKS T1PKS r18c1">
<a href="#r18c1">18.1</a>
</div>



<div class="regbutton terpene terpene r19c1">
<a href="#r19c1">19.1</a>
</div>



<div class="regbutton PKS T1PKS r20c1">
<a href="#r20c1">20.1</a>
</div>

<div class="regbutton RiPP fungal-RiPP-like r20c2">
<a href="#r20c2">20.2</a>
</div>



<div class="regbutton NRPS NRPS-like r21c1">
<a href="#r21c1">21.1</a>
</div>



<div class="regbutton RiPP fungal-RiPP-like r22c1">
<a href="#r22c1">22.1</a>
</div>



<div class="regbutton NRPS r24c1">
<a href="#r24c1">24.1</a>
</div>

<div class="regbutton PKS T1PKS r24c2">
<a href="#r24c2">24.2</a>
</div>



<div class="regbutton PKS T1PKS r26c1">
<a href="#r26c1">26.1</a>
</div>



<div class="regbutton PKS T1PKS r28c1">
<a href="#r28c1">28.1</a>
</div>



<div class="regbutton terpene terpene r29c1">
<a href="#r29c1">29.1</a>
</div>

<div class="regbutton NRPS NRPS-like r29c2">
<a href="#r29c2">29.2</a>
</div>



<div class="regbutton PKS T1PKS r30c1">
<a href="#r30c1">30.1</a>
</div>

<div class="regbutton NRPS NRPS-like r30c2">
<a href="#r30c2">30.2</a>
</div>



<div class="regbutton terpene terpene r31c1">
<a href="#r31c1">31.1</a>
</div>



<div class="regbutton NRPS NRPS-like r32c1">
<a href="#r32c1">32.1</a>
</div>



<div class="regbutton hybrid r34c1">
<a href="#r34c1">34.1</a>
</div>



<div class="regbutton PKS T1PKS r35c1">
<a href="#r35c1">35.1</a>
</div>

<div class="regbutton terpene terpene r35c2">
<a href="#r35c2">35.2</a>
</div>



<div class="regbutton RiPP fungal-RiPP-like r36c1">
<a href="#r36c1">36.1</a>
</div>



<div class="regbutton hybrid r38c1">
<a href="#r38c1">38.1</a>
</div>

<div class="regbutton PKS T1PKS r38c2">
<a href="#r38c2">38.2</a>
</div>

<div class="regbutton PKS T1PKS r38c3">
<a href="#r38c3">38.3</a>
</div>



<div class="regbutton PKS T1PKS r39c1">
<a href="#r39c1">39.1</a>
</div>

<div class="regbutton RiPP fungal-RiPP-like r39c2">
<a href="#r39c2">39.2</a>
</div>



<div class="regbutton PKS T1PKS r42c1">
<a href="#r42c1">42.1</a>
</div>



<div class="regbutton terpene terpene r43c1">
<a href="#r43c1">43.1</a>
</div>



<div class="regbutton NRPS NRPS-like r44c1">
<a href="#r44c1">44.1</a>
</div>



<div class="regbutton PKS T1PKS r46c1">
<a href="#r46c1">46.1</a>
</div>



<div class="regbutton RiPP fungal-RiPP-like r47c1">
<a href="#r47c1">47.1</a>
</div>



<div class="regbutton hybrid r49c1">
<a href="#r49c1">49.1</a>
</div>



<div class="regbutton NRPS NRPS r50c1">
<a href="#r50c1">50.1</a>
</div>



<div class="regbutton PKS T3PKS r51c1">
<a href="#r51c1">51.1</a>
</div>



<div class="regbutton RiPP fungal-RiPP-like r53c1">
<a href="#r53c1">53.1</a>
</div>



<div class="regbutton hybrid r56c1">
<a href="#r56c1">56.1</a>
</div>



<div class="regbutton NRPS NRPS r57c1">
<a href="#r57c1">57.1</a>
</div>

<div class="regbutton PKS T1PKS r57c2">
<a href="#r57c2">57.2</a>
</div>



<div class="regbutton PKS T1PKS r59c1">
<a href="#r59c1">59.1</a>
</div>

<div class="regbutton terpene terpene r59c2">
<a href="#r59c2">59.2</a>
</div>



<div class="regbutton RiPP fungal-RiPP-like r63c1">
<a href="#r63c1">63.1</a>
</div>



<div class="regbutton PKS T1PKS r64c1">
<a href="#r64c1">64.1</a>
</div>



<div class="regbutton NRPS NRPS-like r69c1">
<a href="#r69c1">69.1</a>
</div>

<div class="regbutton PKS T1PKS r69c2">
<a href="#r69c2">69.2</a>
</div>



<div class="regbutton terpene terpene r70c1">
<a href="#r70c1">70.1</a>
</div>



<div class="regbutton NRPS NRPS-like r76c1">
<a href="#r76c1">76.1</a>
</div>

<div class="regbutton terpene terpene r76c2">
<a href="#r76c2">76.2</a>
</div>



<div class="regbutton RiPP fungal-RiPP-like r78c1">
<a href="#r78c1">78.1</a>
</div>



<div class="regbutton PKS T1PKS r85c1">
<a href="#r85c1">85.1</a>
</div>



<div class="regbutton other indole r89c1">
<a href="#r89c1">89.1</a>
</div>



<div class="regbutton terpene terpene r90c1">
<a href="#r90c1">90.1</a>
</div>



<div class="regbutton RiPP fungal-RiPP-like r95c1">
<a href="#r95c1">95.1</a>
</div>



<div class="regbutton hybrid r96c1">
<a href="#r96c1">96.1</a>
</div>



<div class="regbutton PKS T1PKS r97c1">
<a href="#r97c1">97.1</a>
</div>



<div class="regbutton other indole r98c1">
<a href="#r98c1">98.1</a>
</div>



<div class="regbutton hybrid r103c1">
<a href="#r103c1">103.1</a>
</div>



<div class="regbutton NRPS NRPS-like r107c1">
<a href="#r107c1">107.1</a>
</div>



<div class="regbutton NRPS NRPS r110c1">
<a href="#r110c1">110.1</a>
</div>



<div class="regbutton PKS T1PKS r112c1">
<a href="#r112c1">112.1</a>
</div>


</div>
<div id="last-regbutton"><div class="arrow-right" id="next-region"></div></div>
</div>
</div>
</div>

<!-- overview page -->

<div class="page" id="overview">


<h3>Identified secondary metabolite regions using strictness &#39;relaxed&#39;<span id="truncated"></span></h3>

<div class="overview-layout">
<div id="record-tables">
<div id="single-record-tables">
<div class="record-overview-header">
<strong>scaffold_1</strong> (Xylaria sp)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-1" data-filename="scaffold_1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-1" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r1c1">
<td class="regbutton PKS T1PKS r1c1">
<a href="#r1c1">Region&nbsp1.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">117,940</td>
<td class="digits table-split-left">165,651</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r1c2">
<td class="regbutton terpene terpene r1c2">
<a href="#r1c2">Region&nbsp1.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">408,074</td>
<td class="digits table-split-left">429,212</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r1c3">
<td class="regbutton terpene terpene r1c3">
<a href="#r1c3">Region&nbsp1.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">892,788</td>
<td class="digits table-split-left">914,401</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001839/1" target="_blank">squalestatin S1</a></td>
<td>Terpene</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 40%, #ffffff00 40%)">40%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_2</strong> (Xylaria sp)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-2" data-filename="scaffold_2_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-2" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r2c1">
<td class="regbutton terpene terpene r2c1">
<a href="#r2c1">Region&nbsp2.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">190,620</td>
<td class="digits table-split-left">211,845</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000688/1" target="_blank">copalyl diphosphate</a></td>
<td>Terpene</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 42%, #ffffff00 42%)">42%</td>

</tr>
<tr class="linked-row even" data-anchor="#r2c2">
<td class="regbutton NRPS NRPS-like r2c2">
<a href="#r2c2">Region&nbsp2.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">1,207,872</td>
<td class="digits table-split-left">1,250,363</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_3</strong> (Xylaria sp)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-3" data-filename="scaffold_3_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-3" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r3c1">
<td class="regbutton RiPP fungal-RiPP-like r3c1">
<a href="#r3c1">Region&nbsp3.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">245,315</td>
<td class="digits table-split-left">308,903</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r3c2">
<td class="regbutton PKS T1PKS r3c2">
<a href="#r3c2">Region&nbsp3.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">731,949</td>
<td class="digits table-split-left">779,370</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r3c3">
<td class="regbutton RiPP fungal-RiPP-like r3c3">
<a href="#r3c3">Region&nbsp3.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">1,003,045</td>
<td class="digits table-split-left">1,064,010</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_4</strong> (Xylaria sp)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-4" data-filename="scaffold_4_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-4" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r4c1">
<td class="regbutton PKS T1PKS r4c1">
<a href="#r4c1">Region&nbsp4.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">222,420</td>
<td class="digits table-split-left">271,222</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r4c2">
<td class="regbutton NRPS NRPS-like r4c2">
<a href="#r4c2">Region&nbsp4.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">703,038</td>
<td class="digits table-split-left">746,468</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_5</strong> (Xylaria sp)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-5" data-filename="scaffold_5_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-5" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r5c1">
<td class="regbutton RiPP fungal-RiPP-like r5c1">
<a href="#r5c1">Region&nbsp5.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">249,984</td>
<td class="digits table-split-left">312,664</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r5c2">
<td class="regbutton NRPS NRPS-like r5c2">
<a href="#r5c2">Region&nbsp5.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">765,547</td>
<td class="digits table-split-left">808,937</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r5c3">
<td class="regbutton NRPS NRPS-like r5c3">
<a href="#r5c3">Region&nbsp5.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">820,835</td>
<td class="digits table-split-left">863,942</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_6</strong> (Xylaria sp)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-6" data-filename="scaffold_6_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-6" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r6c1">
<td class="regbutton NRPS NRPS-like r6c1">
<a href="#r6c1">Region&nbsp6.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">161,974</td>
<td class="digits table-split-left">205,181</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r6c2">
<td class="regbutton PKS T1PKS r6c2">
<a href="#r6c2">Region&nbsp6.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">444,815</td>
<td class="digits table-split-left">493,842</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_8</strong> (Xylaria sp)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-8" data-filename="scaffold_8_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-8" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r8c1">
<td class="regbutton terpene terpene r8c1">
<a href="#r8c1">Region&nbsp8.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">357,322</td>
<td class="digits table-split-left">378,518</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r8c2">
<td class="regbutton PKS T1PKS r8c2">
<a href="#r8c2">Region&nbsp8.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">498,237</td>
<td class="digits table-split-left">545,506</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r8c3">
<td class="regbutton hybrid r8c3">
<a href="#r8c3">Region&nbsp8.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">659,102</td>
<td class="digits table-split-left">756,482</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_9</strong> (Xylaria sp)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-9" data-filename="scaffold_9_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-9" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r9c1">
<td class="regbutton PKS T1PKS r9c1">
<a href="#r9c1">Region&nbsp9.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">6,961</td>
<td class="digits table-split-left">53,915</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002734/1" target="_blank">2,4-dihydroxy-5,6-dimethyl benzaldehyde</a></td>
<td>Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 28%, #ffffff00 28%)">28%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_10</strong> (Xylaria sp)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-10" data-filename="scaffold_10_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-10" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r10c1">
<td class="regbutton PKS T1PKS r10c1">
<a href="#r10c1">Region&nbsp10.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">180,315</td>
<td class="digits table-split-left">228,285</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r10c2">
<td class="regbutton hybrid r10c2">
<a href="#r10c2">Region&nbsp10.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">827,589</td>
<td class="digits table-split-left">879,905</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_12</strong> (Xylaria sp)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-12" data-filename="scaffold_12_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-12" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r12c1">
<td class="regbutton NRPS NRPS r12c1">
<a href="#r12c1">Region&nbsp12.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">1,635</td>
<td class="digits table-split-left">46,668</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001123/1" target="_blank">aspirochlorine</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 13%, #ffffff00 13%)">13%</td>

</tr>
<tr class="linked-row even" data-anchor="#r12c2">
<td class="regbutton RiPP fungal-RiPP-like r12c2">
<a href="#r12c2">Region&nbsp12.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">583,247</td>
<td class="digits table-split-left">644,383</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001277/1" target="_blank">nivalenol/vomitoxin/3-acetyldeoxynivalenol/15-acetyldeoxynivalenol/neosolaniol/calonectrin/apotrichodiol/isotrichotriol/15-deacetylcalonectrin/T-2 toxin/3-acetyl T-2 toxin/trichodiene</a></td>
<td>Terpene</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 9%, #ffffff00 9%)">9%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_13</strong> (Xylaria sp)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-13" data-filename="scaffold_13_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-13" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r13c1">
<td class="regbutton RiPP fungal-RiPP-like r13c1">
<a href="#r13c1">Region&nbsp13.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">14,892</td>
<td class="digits table-split-left">78,003</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r13c2">
<td class="regbutton terpene terpene r13c2">
<a href="#r13c2">Region&nbsp13.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">210,304</td>
<td class="digits table-split-left">231,876</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001995/1" target="_blank">heptelidic acid</a></td>
<td>Terpene</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 27%, #ffffff00 27%)">27%</td>

</tr>
<tr class="linked-row odd" data-anchor="#r13c3">
<td class="regbutton RiPP fungal-RiPP r13c3">
<a href="#r13c3">Region&nbsp13.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp" target="_blank">fungal-RiPP</a>
</td>
<td class="digits">771,214</td>
<td class="digits table-split-left">808,011</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_16</strong> (Xylaria sp)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-16" data-filename="scaffold_16_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-16" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r16c1">
<td class="regbutton PKS T1PKS r16c1">
<a href="#r16c1">Region&nbsp16.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">325,288</td>
<td class="digits table-split-left">371,470</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_17</strong> (Xylaria sp)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-17" data-filename="scaffold_17_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-17" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r17c1">
<td class="regbutton PKS T3PKS r17c1">
<a href="#r17c1">Region&nbsp17.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t3pks" target="_blank">T3PKS</a>
</td>
<td class="digits">212,546</td>
<td class="digits table-split-left">253,996</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_18</strong> (Xylaria sp)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-18" data-filename="scaffold_18_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-18" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r18c1">
<td class="regbutton PKS T1PKS r18c1">
<a href="#r18c1">Region&nbsp18.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">101,596</td>
<td class="digits table-split-left">147,598</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_19</strong> (Xylaria sp)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-19" data-filename="scaffold_19_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-19" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r19c1">
<td class="regbutton terpene terpene r19c1">
<a href="#r19c1">Region&nbsp19.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">180,152</td>
<td class="digits table-split-left">203,374</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_20</strong> (Xylaria sp)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-20" data-filename="scaffold_20_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-20" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r20c1">
<td class="regbutton PKS T1PKS r20c1">
<a href="#r20c1">Region&nbsp20.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">233,962</td>
<td class="digits table-split-left">282,045</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r20c2">
<td class="regbutton RiPP fungal-RiPP-like r20c2">
<a href="#r20c2">Region&nbsp20.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">299,772</td>
<td class="digits table-split-left">366,449</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_21</strong> (Xylaria sp)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-21" data-filename="scaffold_21_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-21" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r21c1">
<td class="regbutton NRPS NRPS-like r21c1">
<a href="#r21c1">Region&nbsp21.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">37,598</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_22</strong> (Xylaria sp)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-22" data-filename="scaffold_22_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-22" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r22c1">
<td class="regbutton RiPP fungal-RiPP-like r22c1">
<a href="#r22c1">Region&nbsp22.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">299,445</td>
<td class="digits table-split-left">360,472</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_24</strong> (Xylaria sp)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-24" data-filename="scaffold_24_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-24" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r24c1">
<td class="regbutton NRPS r24c1">
<a href="#r24c1">Region&nbsp24.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrp-metallophore" target="_blank">NRP-metallophore</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">23,601</td>
<td class="digits table-split-left">84,623</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r24c2">
<td class="regbutton PKS T1PKS r24c2">
<a href="#r24c2">Region&nbsp24.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">454,601</td>
<td class="digits table-split-left">501,516</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002233/1" target="_blank">trichoxide</a></td>
<td>Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 66%, #ffffff00 66%)">66%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_26</strong> (Xylaria sp)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-26" data-filename="scaffold_26_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-26" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r26c1">
<td class="regbutton PKS T1PKS r26c1">
<a href="#r26c1">Region&nbsp26.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">215,940</td>
<td class="digits table-split-left">262,542</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001906/1" target="_blank">naphthalene</a></td>
<td>Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 33%, #ffffff00 33%)">33%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_28</strong> (Xylaria sp)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-28" data-filename="scaffold_28_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-28" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r28c1">
<td class="regbutton PKS T1PKS r28c1">
<a href="#r28c1">Region&nbsp28.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">340,992</td>
<td class="digits table-split-left">387,995</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002211/1" target="_blank">HEx-pks1 polyketide</a></td>
<td>Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 60%, #ffffff00 60%)">60%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_29</strong> (Xylaria sp)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-29" data-filename="scaffold_29_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-29" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r29c1">
<td class="regbutton terpene terpene r29c1">
<a href="#r29c1">Region&nbsp29.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">6,283</td>
<td class="digits table-split-left">27,900</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r29c2">
<td class="regbutton NRPS NRPS-like r29c2">
<a href="#r29c2">Region&nbsp29.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">69,701</td>
<td class="digits table-split-left">112,796</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_30</strong> (Xylaria sp)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-30" data-filename="scaffold_30_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-30" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r30c1">
<td class="regbutton PKS T1PKS r30c1">
<a href="#r30c1">Region&nbsp30.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">46,992</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r30c2">
<td class="regbutton NRPS NRPS-like r30c2">
<a href="#r30c2">Region&nbsp30.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">137,164</td>
<td class="digits table-split-left">179,980</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_31</strong> (Xylaria sp)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-31" data-filename="scaffold_31_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-31" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r31c1">
<td class="regbutton terpene terpene r31c1">
<a href="#r31c1">Region&nbsp31.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">257,262</td>
<td class="digits table-split-left">279,619</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001248/1" target="_blank">clavaric acid</a></td>
<td>Terpene</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_32</strong> (Xylaria sp)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-32" data-filename="scaffold_32_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-32" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r32c1">
<td class="regbutton NRPS NRPS-like r32c1">
<a href="#r32c1">Region&nbsp32.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">124,347</td>
<td class="digits table-split-left">167,941</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_34</strong> (Xylaria sp)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-34" data-filename="scaffold_34_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-34" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r34c1">
<td class="regbutton hybrid r34c1">
<a href="#r34c1">Region&nbsp34.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">52,342</td>
<td class="digits table-split-left">126,418</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_35</strong> (Xylaria sp)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-35" data-filename="scaffold_35_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-35" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r35c1">
<td class="regbutton PKS T1PKS r35c1">
<a href="#r35c1">Region&nbsp35.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">13,697</td>
<td class="digits table-split-left">57,162</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r35c2">
<td class="regbutton terpene terpene r35c2">
<a href="#r35c2">Region&nbsp35.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">280,865</td>
<td class="digits table-split-left">302,065</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_36</strong> (Xylaria sp)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-36" data-filename="scaffold_36_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-36" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r36c1">
<td class="regbutton RiPP fungal-RiPP-like r36c1">
<a href="#r36c1">Region&nbsp36.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">100,669</td>
<td class="digits table-split-left">162,622</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_38</strong> (Xylaria sp)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-38" data-filename="scaffold_38_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-38" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r38c1">
<td class="regbutton hybrid r38c1">
<a href="#r38c1">Region&nbsp38.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">144,862</td>
<td class="digits table-split-left">199,842</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001254/1" target="_blank">ACT-Toxin II</a></td>
<td>Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>
<tr class="linked-row even" data-anchor="#r38c2">
<td class="regbutton PKS T1PKS r38c2">
<a href="#r38c2">Region&nbsp38.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">202,623</td>
<td class="digits table-split-left">248,768</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000120/1" target="_blank">patulin</a></td>
<td>Polyketide:Iterative type I polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 20%, #ffffff00 20%)">20%</td>

</tr>
<tr class="linked-row odd" data-anchor="#r38c3">
<td class="regbutton PKS T1PKS r38c3">
<a href="#r38c3">Region&nbsp38.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">297,407</td>
<td class="digits table-split-left">343,178</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002615/1" target="_blank">4-oxomacrophorin A/macrophorin A/5&#39;-epimacrophorin B</a></td>
<td>Terpene</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 54%, #ffffff00 54%)">54%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_39</strong> (Xylaria sp)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-39" data-filename="scaffold_39_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-39" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r39c1">
<td class="regbutton PKS T1PKS r39c1">
<a href="#r39c1">Region&nbsp39.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">103,098</td>
<td class="digits table-split-left">151,905</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r39c2">
<td class="regbutton RiPP fungal-RiPP-like r39c2">
<a href="#r39c2">Region&nbsp39.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">184,852</td>
<td class="digits table-split-left">245,900</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_42</strong> (Xylaria sp)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-42" data-filename="scaffold_42_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-42" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r42c1">
<td class="regbutton PKS T1PKS r42c1">
<a href="#r42c1">Region&nbsp42.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">96,505</td>
<td class="digits table-split-left">142,187</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001244/1" target="_blank">(-)-Mellein</a></td>
<td>Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_43</strong> (Xylaria sp)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-43" data-filename="scaffold_43_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-43" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r43c1">
<td class="regbutton terpene terpene r43c1">
<a href="#r43c1">Region&nbsp43.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">46,690</td>
<td class="digits table-split-left">67,862</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_44</strong> (Xylaria sp)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-44" data-filename="scaffold_44_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-44" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r44c1">
<td class="regbutton NRPS NRPS-like r44c1">
<a href="#r44c1">Region&nbsp44.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">41,663</td>
<td class="digits table-split-left">84,928</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_46</strong> (Xylaria sp)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-46" data-filename="scaffold_46_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-46" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r46c1">
<td class="regbutton PKS T1PKS r46c1">
<a href="#r46c1">Region&nbsp46.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">222,062</td>
<td class="digits table-split-left">269,635</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_47</strong> (Xylaria sp)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-47" data-filename="scaffold_47_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-47" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r47c1">
<td class="regbutton RiPP fungal-RiPP-like r47c1">
<a href="#r47c1">Region&nbsp47.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">144,004</td>
<td class="digits table-split-left">205,042</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_49</strong> (Xylaria sp)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-49" data-filename="scaffold_49_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-49" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r49c1">
<td class="regbutton hybrid r49c1">
<a href="#r49c1">Region&nbsp49.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">61,418</td>
<td class="digits table-split-left">113,940</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_50</strong> (Xylaria sp)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-50" data-filename="scaffold_50_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-50" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r50c1">
<td class="regbutton NRPS NRPS r50c1">
<a href="#r50c1">Region&nbsp50.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">5,795</td>
<td class="digits table-split-left">54,855</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000342/1" target="_blank">enniatin</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_51</strong> (Xylaria sp)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-51" data-filename="scaffold_51_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-51" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r51c1">
<td class="regbutton PKS T3PKS r51c1">
<a href="#r51c1">Region&nbsp51.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t3pks" target="_blank">T3PKS</a>
</td>
<td class="digits">174,050</td>
<td class="digits table-split-left">215,263</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_53</strong> (Xylaria sp)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-53" data-filename="scaffold_53_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-53" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r53c1">
<td class="regbutton RiPP fungal-RiPP-like r53c1">
<a href="#r53c1">Region&nbsp53.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">113,468</td>
<td class="digits table-split-left">173,947</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_56</strong> (Xylaria sp)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-56" data-filename="scaffold_56_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-56" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r56c1">
<td class="regbutton hybrid r56c1">
<a href="#r56c1">Region&nbsp56.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#indole" target="_blank">indole</a>
</td>
<td class="digits">4,427</td>
<td class="digits table-split-left">51,315</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002209/1" target="_blank">ustethylin A</a></td>
<td>Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 20%, #ffffff00 20%)">20%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_57</strong> (Xylaria sp)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-57" data-filename="scaffold_57_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-57" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r57c1">
<td class="regbutton NRPS NRPS r57c1">
<a href="#r57c1">Region&nbsp57.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">84,081</td>
<td class="digits table-split-left">132,472</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r57c2">
<td class="regbutton PKS T1PKS r57c2">
<a href="#r57c2">Region&nbsp57.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">175,718</td>
<td class="digits table-split-left">223,463</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_59</strong> (Xylaria sp)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-59" data-filename="scaffold_59_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-59" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r59c1">
<td class="regbutton PKS T1PKS r59c1">
<a href="#r59c1">Region&nbsp59.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">45,907</td>
<td class="digits table-split-left">92,927</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r59c2">
<td class="regbutton terpene terpene r59c2">
<a href="#r59c2">Region&nbsp59.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">179,384</td>
<td class="digits table-split-left">202,107</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_63</strong> (Xylaria sp)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-63" data-filename="scaffold_63_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-63" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r63c1">
<td class="regbutton RiPP fungal-RiPP-like r63c1">
<a href="#r63c1">Region&nbsp63.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">49,420</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_64</strong> (Xylaria sp)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-64" data-filename="scaffold_64_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-64" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r64c1">
<td class="regbutton PKS T1PKS r64c1">
<a href="#r64c1">Region&nbsp64.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">150,337</td>
<td class="digits table-split-left">197,634</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002211/1" target="_blank">HEx-pks1 polyketide</a></td>
<td>Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 40%, #ffffff00 40%)">40%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_69</strong> (Xylaria sp)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-69" data-filename="scaffold_69_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-69" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r69c1">
<td class="regbutton NRPS NRPS-like r69c1">
<a href="#r69c1">Region&nbsp69.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">27,797</td>
<td class="digits table-split-left">71,095</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r69c2">
<td class="regbutton PKS T1PKS r69c2">
<a href="#r69c2">Region&nbsp69.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">90,072</td>
<td class="digits table-split-left">138,331</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001976/1" target="_blank">eupenifeldin</a></td>
<td>Terpene</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 18%, #ffffff00 18%)">18%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_70</strong> (Xylaria sp)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-70" data-filename="scaffold_70_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-70" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r70c1">
<td class="regbutton terpene terpene r70c1">
<a href="#r70c1">Region&nbsp70.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">148,313</td>
<td class="digits table-split-left">169,677</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_76</strong> (Xylaria sp)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-76" data-filename="scaffold_76_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-76" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r76c1">
<td class="regbutton NRPS NRPS-like r76c1">
<a href="#r76c1">Region&nbsp76.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">5,896</td>
<td class="digits table-split-left">49,759</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002276/1" target="_blank">choline</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>
<tr class="linked-row even" data-anchor="#r76c2">
<td class="regbutton terpene terpene r76c2">
<a href="#r76c2">Region&nbsp76.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">54,975</td>
<td class="digits table-split-left">75,625</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002260/1" target="_blank">trichobrasilenol/xylarenic acid B/brasilane A/brasilane F/brasilane E/brasilane D</a></td>
<td>Terpene</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 60%, #ffffff00 60%)">60%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_78</strong> (Xylaria sp)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-78" data-filename="scaffold_78_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-78" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r78c1">
<td class="regbutton RiPP fungal-RiPP-like r78c1">
<a href="#r78c1">Region&nbsp78.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">112,872</td>
<td class="digits table-split-left">161,769</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_85</strong> (Xylaria sp)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-85" data-filename="scaffold_85_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-85" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r85c1">
<td class="regbutton PKS T1PKS r85c1">
<a href="#r85c1">Region&nbsp85.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">64,072</td>
<td class="digits table-split-left">111,174</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_89</strong> (Xylaria sp)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-89" data-filename="scaffold_89_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-89" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r89c1">
<td class="regbutton other indole r89c1">
<a href="#r89c1">Region&nbsp89.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#indole" target="_blank">indole</a>
</td>
<td class="digits">5,389</td>
<td class="digits table-split-left">26,597</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_90</strong> (Xylaria sp)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-90" data-filename="scaffold_90_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-90" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r90c1">
<td class="regbutton terpene terpene r90c1">
<a href="#r90c1">Region&nbsp90.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">70,780</td>
<td class="digits table-split-left">93,107</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_95</strong> (Xylaria sp)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-95" data-filename="scaffold_95_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-95" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r95c1">
<td class="regbutton RiPP fungal-RiPP-like r95c1">
<a href="#r95c1">Region&nbsp95.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">37,880</td>
<td class="digits table-split-left">117,375</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_96</strong> (Xylaria sp)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-96" data-filename="scaffold_96_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-96" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r96c1">
<td class="regbutton hybrid r96c1">
<a href="#r96c1">Region&nbsp96.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">4,746</td>
<td class="digits table-split-left">61,945</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000037/1" target="_blank">cichorine</a></td>
<td>Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_97</strong> (Xylaria sp)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-97" data-filename="scaffold_97_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-97" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r97c1">
<td class="regbutton PKS T1PKS r97c1">
<a href="#r97c1">Region&nbsp97.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">48,698</td>
<td class="digits table-split-left">96,542</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_98</strong> (Xylaria sp)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-98" data-filename="scaffold_98_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-98" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r98c1">
<td class="regbutton other indole r98c1">
<a href="#r98c1">Region&nbsp98.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#indole" target="_blank">indole</a>
</td>
<td class="digits">13,622</td>
<td class="digits table-split-left">36,644</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_103</strong> (Xylaria sp)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-103" data-filename="scaffold_103_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-103" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r103c1">
<td class="regbutton hybrid r103c1">
<a href="#r103c1">Region&nbsp103.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">35,586</td>
<td class="digits table-split-left">94,714</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002257/1" target="_blank">rufoschweinitzin</a></td>
<td>Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 85%, #ffffff00 85%)">85%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_107</strong> (Xylaria sp)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-107" data-filename="scaffold_107_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-107" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r107c1">
<td class="regbutton NRPS NRPS-like r107c1">
<a href="#r107c1">Region&nbsp107.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">40,584</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_110</strong> (Xylaria sp)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-110" data-filename="scaffold_110_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-110" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r110c1">
<td class="regbutton NRPS NRPS r110c1">
<a href="#r110c1">Region&nbsp110.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">24,204</td>
<td class="digits table-split-left">64,804</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001249/1" target="_blank">dimethylcoprogen</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>scaffold_112</strong> (Xylaria sp)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-112" data-filename="scaffold_112_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-112" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r112c1">
<td class="regbutton PKS T1PKS r112c1">
<a href="#r112c1">Region&nbsp112.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">29,794</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div>
</div>

<div id="compact-record-table" style="display: none;">
<div class="record-table">
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>


<tr class="linked-row odd" data-anchor="#r1c1">
<td class="regbutton PKS T1PKS r1c1">
<a href="#r1c1">Region&nbsp1.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">117,940</td>
<td class="digits table-split-left">165,651</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r1c2">
<td class="regbutton terpene terpene r1c2">
<a href="#r1c2">Region&nbsp1.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">408,074</td>
<td class="digits table-split-left">429,212</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r1c3">
<td class="regbutton terpene terpene r1c3">
<a href="#r1c3">Region&nbsp1.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">892,788</td>
<td class="digits table-split-left">914,401</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001839/1" target="_blank">squalestatin S1</a></td>
<td>Terpene</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 40%, #ffffff00 40%)">40%</td>

</tr>


<tr class="linked-row even" data-anchor="#r2c1">
<td class="regbutton terpene terpene r2c1">
<a href="#r2c1">Region&nbsp2.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">190,620</td>
<td class="digits table-split-left">211,845</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000688/1" target="_blank">copalyl diphosphate</a></td>
<td>Terpene</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 42%, #ffffff00 42%)">42%</td>

</tr>
<tr class="linked-row odd" data-anchor="#r2c2">
<td class="regbutton NRPS NRPS-like r2c2">
<a href="#r2c2">Region&nbsp2.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">1,207,872</td>
<td class="digits table-split-left">1,250,363</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row even" data-anchor="#r3c1">
<td class="regbutton RiPP fungal-RiPP-like r3c1">
<a href="#r3c1">Region&nbsp3.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">245,315</td>
<td class="digits table-split-left">308,903</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r3c2">
<td class="regbutton PKS T1PKS r3c2">
<a href="#r3c2">Region&nbsp3.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">731,949</td>
<td class="digits table-split-left">779,370</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r3c3">
<td class="regbutton RiPP fungal-RiPP-like r3c3">
<a href="#r3c3">Region&nbsp3.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">1,003,045</td>
<td class="digits table-split-left">1,064,010</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row odd" data-anchor="#r4c1">
<td class="regbutton PKS T1PKS r4c1">
<a href="#r4c1">Region&nbsp4.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">222,420</td>
<td class="digits table-split-left">271,222</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r4c2">
<td class="regbutton NRPS NRPS-like r4c2">
<a href="#r4c2">Region&nbsp4.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">703,038</td>
<td class="digits table-split-left">746,468</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row odd" data-anchor="#r5c1">
<td class="regbutton RiPP fungal-RiPP-like r5c1">
<a href="#r5c1">Region&nbsp5.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">249,984</td>
<td class="digits table-split-left">312,664</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r5c2">
<td class="regbutton NRPS NRPS-like r5c2">
<a href="#r5c2">Region&nbsp5.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">765,547</td>
<td class="digits table-split-left">808,937</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r5c3">
<td class="regbutton NRPS NRPS-like r5c3">
<a href="#r5c3">Region&nbsp5.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">820,835</td>
<td class="digits table-split-left">863,942</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row even" data-anchor="#r6c1">
<td class="regbutton NRPS NRPS-like r6c1">
<a href="#r6c1">Region&nbsp6.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">161,974</td>
<td class="digits table-split-left">205,181</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r6c2">
<td class="regbutton PKS T1PKS r6c2">
<a href="#r6c2">Region&nbsp6.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">444,815</td>
<td class="digits table-split-left">493,842</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row even" data-anchor="#r8c1">
<td class="regbutton terpene terpene r8c1">
<a href="#r8c1">Region&nbsp8.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">357,322</td>
<td class="digits table-split-left">378,518</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r8c2">
<td class="regbutton PKS T1PKS r8c2">
<a href="#r8c2">Region&nbsp8.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">498,237</td>
<td class="digits table-split-left">545,506</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r8c3">
<td class="regbutton hybrid r8c3">
<a href="#r8c3">Region&nbsp8.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">659,102</td>
<td class="digits table-split-left">756,482</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row odd" data-anchor="#r9c1">
<td class="regbutton PKS T1PKS r9c1">
<a href="#r9c1">Region&nbsp9.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">6,961</td>
<td class="digits table-split-left">53,915</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002734/1" target="_blank">2,4-dihydroxy-5,6-dimethyl benzaldehyde</a></td>
<td>Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 28%, #ffffff00 28%)">28%</td>

</tr>


<tr class="linked-row even" data-anchor="#r10c1">
<td class="regbutton PKS T1PKS r10c1">
<a href="#r10c1">Region&nbsp10.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">180,315</td>
<td class="digits table-split-left">228,285</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r10c2">
<td class="regbutton hybrid r10c2">
<a href="#r10c2">Region&nbsp10.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">827,589</td>
<td class="digits table-split-left">879,905</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row even" data-anchor="#r12c1">
<td class="regbutton NRPS NRPS r12c1">
<a href="#r12c1">Region&nbsp12.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">1,635</td>
<td class="digits table-split-left">46,668</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001123/1" target="_blank">aspirochlorine</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 13%, #ffffff00 13%)">13%</td>

</tr>
<tr class="linked-row odd" data-anchor="#r12c2">
<td class="regbutton RiPP fungal-RiPP-like r12c2">
<a href="#r12c2">Region&nbsp12.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">583,247</td>
<td class="digits table-split-left">644,383</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001277/1" target="_blank">nivalenol/vomitoxin/3-acetyldeoxynivalenol/15-acetyldeoxynivalenol/neosolaniol/calonectrin/apotrichodiol/isotrichotriol/15-deacetylcalonectrin/T-2 toxin/3-acetyl T-2 toxin/trichodiene</a></td>
<td>Terpene</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 9%, #ffffff00 9%)">9%</td>

</tr>


<tr class="linked-row even" data-anchor="#r13c1">
<td class="regbutton RiPP fungal-RiPP-like r13c1">
<a href="#r13c1">Region&nbsp13.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">14,892</td>
<td class="digits table-split-left">78,003</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r13c2">
<td class="regbutton terpene terpene r13c2">
<a href="#r13c2">Region&nbsp13.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">210,304</td>
<td class="digits table-split-left">231,876</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001995/1" target="_blank">heptelidic acid</a></td>
<td>Terpene</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 27%, #ffffff00 27%)">27%</td>

</tr>
<tr class="linked-row even" data-anchor="#r13c3">
<td class="regbutton RiPP fungal-RiPP r13c3">
<a href="#r13c3">Region&nbsp13.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp" target="_blank">fungal-RiPP</a>
</td>
<td class="digits">771,214</td>
<td class="digits table-split-left">808,011</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row odd" data-anchor="#r16c1">
<td class="regbutton PKS T1PKS r16c1">
<a href="#r16c1">Region&nbsp16.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">325,288</td>
<td class="digits table-split-left">371,470</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row even" data-anchor="#r17c1">
<td class="regbutton PKS T3PKS r17c1">
<a href="#r17c1">Region&nbsp17.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t3pks" target="_blank">T3PKS</a>
</td>
<td class="digits">212,546</td>
<td class="digits table-split-left">253,996</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row odd" data-anchor="#r18c1">
<td class="regbutton PKS T1PKS r18c1">
<a href="#r18c1">Region&nbsp18.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">101,596</td>
<td class="digits table-split-left">147,598</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row even" data-anchor="#r19c1">
<td class="regbutton terpene terpene r19c1">
<a href="#r19c1">Region&nbsp19.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">180,152</td>
<td class="digits table-split-left">203,374</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row odd" data-anchor="#r20c1">
<td class="regbutton PKS T1PKS r20c1">
<a href="#r20c1">Region&nbsp20.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">233,962</td>
<td class="digits table-split-left">282,045</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r20c2">
<td class="regbutton RiPP fungal-RiPP-like r20c2">
<a href="#r20c2">Region&nbsp20.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">299,772</td>
<td class="digits table-split-left">366,449</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row odd" data-anchor="#r21c1">
<td class="regbutton NRPS NRPS-like r21c1">
<a href="#r21c1">Region&nbsp21.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">37,598</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row even" data-anchor="#r22c1">
<td class="regbutton RiPP fungal-RiPP-like r22c1">
<a href="#r22c1">Region&nbsp22.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">299,445</td>
<td class="digits table-split-left">360,472</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row odd" data-anchor="#r24c1">
<td class="regbutton NRPS r24c1">
<a href="#r24c1">Region&nbsp24.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrp-metallophore" target="_blank">NRP-metallophore</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">23,601</td>
<td class="digits table-split-left">84,623</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r24c2">
<td class="regbutton PKS T1PKS r24c2">
<a href="#r24c2">Region&nbsp24.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">454,601</td>
<td class="digits table-split-left">501,516</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002233/1" target="_blank">trichoxide</a></td>
<td>Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 66%, #ffffff00 66%)">66%</td>

</tr>


<tr class="linked-row odd" data-anchor="#r26c1">
<td class="regbutton PKS T1PKS r26c1">
<a href="#r26c1">Region&nbsp26.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">215,940</td>
<td class="digits table-split-left">262,542</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001906/1" target="_blank">naphthalene</a></td>
<td>Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 33%, #ffffff00 33%)">33%</td>

</tr>


<tr class="linked-row even" data-anchor="#r28c1">
<td class="regbutton PKS T1PKS r28c1">
<a href="#r28c1">Region&nbsp28.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">340,992</td>
<td class="digits table-split-left">387,995</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002211/1" target="_blank">HEx-pks1 polyketide</a></td>
<td>Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 60%, #ffffff00 60%)">60%</td>

</tr>


<tr class="linked-row odd" data-anchor="#r29c1">
<td class="regbutton terpene terpene r29c1">
<a href="#r29c1">Region&nbsp29.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">6,283</td>
<td class="digits table-split-left">27,900</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r29c2">
<td class="regbutton NRPS NRPS-like r29c2">
<a href="#r29c2">Region&nbsp29.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">69,701</td>
<td class="digits table-split-left">112,796</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row odd" data-anchor="#r30c1">
<td class="regbutton PKS T1PKS r30c1">
<a href="#r30c1">Region&nbsp30.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">46,992</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r30c2">
<td class="regbutton NRPS NRPS-like r30c2">
<a href="#r30c2">Region&nbsp30.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">137,164</td>
<td class="digits table-split-left">179,980</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row odd" data-anchor="#r31c1">
<td class="regbutton terpene terpene r31c1">
<a href="#r31c1">Region&nbsp31.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">257,262</td>
<td class="digits table-split-left">279,619</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001248/1" target="_blank">clavaric acid</a></td>
<td>Terpene</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>


<tr class="linked-row even" data-anchor="#r32c1">
<td class="regbutton NRPS NRPS-like r32c1">
<a href="#r32c1">Region&nbsp32.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">124,347</td>
<td class="digits table-split-left">167,941</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row odd" data-anchor="#r34c1">
<td class="regbutton hybrid r34c1">
<a href="#r34c1">Region&nbsp34.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">52,342</td>
<td class="digits table-split-left">126,418</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row even" data-anchor="#r35c1">
<td class="regbutton PKS T1PKS r35c1">
<a href="#r35c1">Region&nbsp35.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">13,697</td>
<td class="digits table-split-left">57,162</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r35c2">
<td class="regbutton terpene terpene r35c2">
<a href="#r35c2">Region&nbsp35.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">280,865</td>
<td class="digits table-split-left">302,065</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row even" data-anchor="#r36c1">
<td class="regbutton RiPP fungal-RiPP-like r36c1">
<a href="#r36c1">Region&nbsp36.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">100,669</td>
<td class="digits table-split-left">162,622</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row odd" data-anchor="#r38c1">
<td class="regbutton hybrid r38c1">
<a href="#r38c1">Region&nbsp38.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">144,862</td>
<td class="digits table-split-left">199,842</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001254/1" target="_blank">ACT-Toxin II</a></td>
<td>Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>
<tr class="linked-row even" data-anchor="#r38c2">
<td class="regbutton PKS T1PKS r38c2">
<a href="#r38c2">Region&nbsp38.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">202,623</td>
<td class="digits table-split-left">248,768</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000120/1" target="_blank">patulin</a></td>
<td>Polyketide:Iterative type I polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 20%, #ffffff00 20%)">20%</td>

</tr>
<tr class="linked-row odd" data-anchor="#r38c3">
<td class="regbutton PKS T1PKS r38c3">
<a href="#r38c3">Region&nbsp38.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">297,407</td>
<td class="digits table-split-left">343,178</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002615/1" target="_blank">4-oxomacrophorin A/macrophorin A/5&#39;-epimacrophorin B</a></td>
<td>Terpene</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 54%, #ffffff00 54%)">54%</td>

</tr>


<tr class="linked-row even" data-anchor="#r39c1">
<td class="regbutton PKS T1PKS r39c1">
<a href="#r39c1">Region&nbsp39.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">103,098</td>
<td class="digits table-split-left">151,905</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r39c2">
<td class="regbutton RiPP fungal-RiPP-like r39c2">
<a href="#r39c2">Region&nbsp39.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">184,852</td>
<td class="digits table-split-left">245,900</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row even" data-anchor="#r42c1">
<td class="regbutton PKS T1PKS r42c1">
<a href="#r42c1">Region&nbsp42.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">96,505</td>
<td class="digits table-split-left">142,187</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001244/1" target="_blank">(-)-Mellein</a></td>
<td>Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>


<tr class="linked-row odd" data-anchor="#r43c1">
<td class="regbutton terpene terpene r43c1">
<a href="#r43c1">Region&nbsp43.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">46,690</td>
<td class="digits table-split-left">67,862</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row even" data-anchor="#r44c1">
<td class="regbutton NRPS NRPS-like r44c1">
<a href="#r44c1">Region&nbsp44.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">41,663</td>
<td class="digits table-split-left">84,928</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row odd" data-anchor="#r46c1">
<td class="regbutton PKS T1PKS r46c1">
<a href="#r46c1">Region&nbsp46.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">222,062</td>
<td class="digits table-split-left">269,635</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row even" data-anchor="#r47c1">
<td class="regbutton RiPP fungal-RiPP-like r47c1">
<a href="#r47c1">Region&nbsp47.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">144,004</td>
<td class="digits table-split-left">205,042</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row odd" data-anchor="#r49c1">
<td class="regbutton hybrid r49c1">
<a href="#r49c1">Region&nbsp49.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">61,418</td>
<td class="digits table-split-left">113,940</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row even" data-anchor="#r50c1">
<td class="regbutton NRPS NRPS r50c1">
<a href="#r50c1">Region&nbsp50.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">5,795</td>
<td class="digits table-split-left">54,855</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000342/1" target="_blank">enniatin</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>


<tr class="linked-row odd" data-anchor="#r51c1">
<td class="regbutton PKS T3PKS r51c1">
<a href="#r51c1">Region&nbsp51.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t3pks" target="_blank">T3PKS</a>
</td>
<td class="digits">174,050</td>
<td class="digits table-split-left">215,263</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row even" data-anchor="#r53c1">
<td class="regbutton RiPP fungal-RiPP-like r53c1">
<a href="#r53c1">Region&nbsp53.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">113,468</td>
<td class="digits table-split-left">173,947</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row odd" data-anchor="#r56c1">
<td class="regbutton hybrid r56c1">
<a href="#r56c1">Region&nbsp56.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#indole" target="_blank">indole</a>
</td>
<td class="digits">4,427</td>
<td class="digits table-split-left">51,315</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002209/1" target="_blank">ustethylin A</a></td>
<td>Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 20%, #ffffff00 20%)">20%</td>

</tr>


<tr class="linked-row even" data-anchor="#r57c1">
<td class="regbutton NRPS NRPS r57c1">
<a href="#r57c1">Region&nbsp57.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">84,081</td>
<td class="digits table-split-left">132,472</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r57c2">
<td class="regbutton PKS T1PKS r57c2">
<a href="#r57c2">Region&nbsp57.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">175,718</td>
<td class="digits table-split-left">223,463</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row even" data-anchor="#r59c1">
<td class="regbutton PKS T1PKS r59c1">
<a href="#r59c1">Region&nbsp59.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">45,907</td>
<td class="digits table-split-left">92,927</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r59c2">
<td class="regbutton terpene terpene r59c2">
<a href="#r59c2">Region&nbsp59.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">179,384</td>
<td class="digits table-split-left">202,107</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row even" data-anchor="#r63c1">
<td class="regbutton RiPP fungal-RiPP-like r63c1">
<a href="#r63c1">Region&nbsp63.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">49,420</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row odd" data-anchor="#r64c1">
<td class="regbutton PKS T1PKS r64c1">
<a href="#r64c1">Region&nbsp64.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">150,337</td>
<td class="digits table-split-left">197,634</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002211/1" target="_blank">HEx-pks1 polyketide</a></td>
<td>Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 40%, #ffffff00 40%)">40%</td>

</tr>


<tr class="linked-row even" data-anchor="#r69c1">
<td class="regbutton NRPS NRPS-like r69c1">
<a href="#r69c1">Region&nbsp69.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">27,797</td>
<td class="digits table-split-left">71,095</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r69c2">
<td class="regbutton PKS T1PKS r69c2">
<a href="#r69c2">Region&nbsp69.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">90,072</td>
<td class="digits table-split-left">138,331</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001976/1" target="_blank">eupenifeldin</a></td>
<td>Terpene</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 18%, #ffffff00 18%)">18%</td>

</tr>


<tr class="linked-row even" data-anchor="#r70c1">
<td class="regbutton terpene terpene r70c1">
<a href="#r70c1">Region&nbsp70.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">148,313</td>
<td class="digits table-split-left">169,677</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row odd" data-anchor="#r76c1">
<td class="regbutton NRPS NRPS-like r76c1">
<a href="#r76c1">Region&nbsp76.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">5,896</td>
<td class="digits table-split-left">49,759</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002276/1" target="_blank">choline</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>
<tr class="linked-row even" data-anchor="#r76c2">
<td class="regbutton terpene terpene r76c2">
<a href="#r76c2">Region&nbsp76.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">54,975</td>
<td class="digits table-split-left">75,625</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002260/1" target="_blank">trichobrasilenol/xylarenic acid B/brasilane A/brasilane F/brasilane E/brasilane D</a></td>
<td>Terpene</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 60%, #ffffff00 60%)">60%</td>

</tr>


<tr class="linked-row odd" data-anchor="#r78c1">
<td class="regbutton RiPP fungal-RiPP-like r78c1">
<a href="#r78c1">Region&nbsp78.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">112,872</td>
<td class="digits table-split-left">161,769</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row even" data-anchor="#r85c1">
<td class="regbutton PKS T1PKS r85c1">
<a href="#r85c1">Region&nbsp85.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">64,072</td>
<td class="digits table-split-left">111,174</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row odd" data-anchor="#r89c1">
<td class="regbutton other indole r89c1">
<a href="#r89c1">Region&nbsp89.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#indole" target="_blank">indole</a>
</td>
<td class="digits">5,389</td>
<td class="digits table-split-left">26,597</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row even" data-anchor="#r90c1">
<td class="regbutton terpene terpene r90c1">
<a href="#r90c1">Region&nbsp90.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">70,780</td>
<td class="digits table-split-left">93,107</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row odd" data-anchor="#r95c1">
<td class="regbutton RiPP fungal-RiPP-like r95c1">
<a href="#r95c1">Region&nbsp95.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#fungal-ripp-like" target="_blank">fungal-RiPP-like</a>
</td>
<td class="digits">37,880</td>
<td class="digits table-split-left">117,375</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row even" data-anchor="#r96c1">
<td class="regbutton hybrid r96c1">
<a href="#r96c1">Region&nbsp96.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">4,746</td>
<td class="digits table-split-left">61,945</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000037/1" target="_blank">cichorine</a></td>
<td>Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>


<tr class="linked-row odd" data-anchor="#r97c1">
<td class="regbutton PKS T1PKS r97c1">
<a href="#r97c1">Region&nbsp97.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">48,698</td>
<td class="digits table-split-left">96,542</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row even" data-anchor="#r98c1">
<td class="regbutton other indole r98c1">
<a href="#r98c1">Region&nbsp98.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#indole" target="_blank">indole</a>
</td>
<td class="digits">13,622</td>
<td class="digits table-split-left">36,644</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row odd" data-anchor="#r103c1">
<td class="regbutton hybrid r103c1">
<a href="#r103c1">Region&nbsp103.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">35,586</td>
<td class="digits table-split-left">94,714</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002257/1" target="_blank">rufoschweinitzin</a></td>
<td>Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 85%, #ffffff00 85%)">85%</td>

</tr>


<tr class="linked-row even" data-anchor="#r107c1">
<td class="regbutton NRPS NRPS-like r107c1">
<a href="#r107c1">Region&nbsp107.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">40,584</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row odd" data-anchor="#r110c1">
<td class="regbutton NRPS NRPS r110c1">
<a href="#r110c1">Region&nbsp110.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">24,204</td>
<td class="digits table-split-left">64,804</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001249/1" target="_blank">dimethylcoprogen</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>


<tr class="linked-row even" data-anchor="#r112c1">
<td class="regbutton PKS T1PKS r112c1">
<a href="#r112c1">Region&nbsp112.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">29,794</td>

<td colspan="3"></td>

</tr>


</tbody>
</table>
</div>
</div>


<dl id="regionless-records">
<dt class="record-overview-header">No secondary metabolite regions were found in these records:</dt>

<dd><strong>scaffold_7</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_11</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_14</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_15</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_23</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_25</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_27</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_33</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_37</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_40</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_41</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_45</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_48</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_52</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_54</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_55</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_58</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_60</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_61</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_62</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_65</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_66</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_67</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_68</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_71</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_72</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_73</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_74</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_75</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_77</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_79</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_80</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_81</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_82</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_83</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_84</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_86</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_87</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_88</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_91</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_92</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_93</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_94</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_99</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_100</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_101</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_102</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_104</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_105</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_106</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_108</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_109</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_111</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_113</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_114</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_115</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_116</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_117</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_118</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_119</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_120</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_121</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_122</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_123</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_124</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_125</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_126</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_127</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_128</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_129</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_130</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_131</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_132</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_133</strong> (Xylaria sp)</dd>

<dd><strong>scaffold_134</strong> (Xylaria sp)</dd>

</dl>

</div>

<div class="overview-switches">
<div class="overview-switch-compact switch-container"><span class="switch-desc">Compact view</span> <label class="switch">  <input class="overview-switch-compact" type="checkbox">  <span class="slider"></span> </label></div>
</div>

</div>
</div>








<div class="page" id='r1c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_1 - Region 1 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-715"></div> <span class="help-tooltip" id="region_svg-help-715">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r1c1-svg" data-filename="scaffold_1_r1c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 117,940 - 165,651 nt. (total: 47,712 nt)

<a class="cluster-rules-header" id="r1c1-rules-header" href="#r1c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r1c1-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r1c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-716"></div> <span class="help-tooltip" id="focus-panel-help-716">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r1c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r1c1-gene-table-details" data-name="r1c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r1c1-nrps_pks" data-name="r1c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r1c1-MIBiG-cluster-compare" data-name="r1c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r1c1-clusterblast" data-name="r1c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r1c1-knownclusterblast" data-name="r1c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r1c1-subclusterblast" data-name="r1c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r1c1-nrps-pks-bubbles" data-name="r1c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r1c1-pfam-details" data-name="r1c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r1c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-9"></div> <span class="help-tooltip" id="gt-help-tooltip-help-9">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r1c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r1c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r1c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r1c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r1c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-1"></div> <span class="help-tooltip" id="nrps-domain-help-1">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c1-details-svg" data-filename="scaffold_1_r1c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r1c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r1c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-2"></div> <span class="help-tooltip" id="MIBiG-help-2">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r1c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r1c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r1c1"></div>
<div id="comparison-MIBiG-r1c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002228: 282-17978">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002228/index.html#r1c1" target="_new">BGC0002228</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.29</td>
<td>NRP</td>

<td>CIML B, CIML A, CIML D, CIML C</td>


<td>Colletotrichum incanum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000046: 1267-21753">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000046/index.html#r1c1" target="_new">BGC0000046</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.27</td>
<td>Polyketide</td>

<td>depudecin</td>


<td>Alternaria brassicicola</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002227: 270-18409">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002227/index.html#r1c1" target="_new">BGC0002227</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.84)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.27</td>
<td>NRP</td>

<td>AKML B, AKML D, AKML A, AKML C</td>


<td>Aspergillus luchuensis IFO 4308</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001124: 3228-30082">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001124/index.html#r1c1" target="_new">BGC0001124</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.26</td>
<td>Polyketide</td>

<td>pyranonigrin E</td>


<td>Aspergillus niger ATCC 1015</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002266: 1041-19123">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002266/index.html#r1c1" target="_new">BGC0002266</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.26</td>
<td>Terpene, Polyketide</td>

<td>calidoustene A, calidoustene B, calidoustene C</td>


<td>Aspergillus calidoustus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001913: 7717-40102">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001913/index.html#r1c1" target="_new">BGC0001913</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.25</td>
<td>Polyketide</td>

<td>phomoidride</td>


<td>fungal sp. ATCC 74256</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002211: 74-16330">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002211/index.html#r1c1" target="_new">BGC0002211</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>Polyketide</td>

<td>HEx-pks1 polyketide</td>


<td>Paraphaeosphaeria sporulosa</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000056: 2-6689">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000056/index.html#r1c1" target="_new">BGC0000056</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.41, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Polyketide</td>

<td>esperamicin</td>


<td>Actinomadura verrucosospora</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002240: 96-11225">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002240/index.html#r1c1" target="_new">BGC0002240</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Polyketide</td>

<td>BAB, BAA</td>


<td>Metarhizium anisopliae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001141: 262-37660">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001141/index.html#r1c1" target="_new">BGC0001141</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.21</td>
<td>Polyketide</td>

<td>4-epi-15-epi-brefeldin A</td>


<td>Penicillium brefeldianum</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r1c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002228: 282-17978">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002228/index.html#r1c1" target="_new">BGC0002228</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>NRP</td>
<td>CIML B, CIML A, CIML D, CIML C</td>
<td>Colletotrichum incanum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000046: 1267-21753">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000046/index.html#r1c1" target="_new">BGC0000046</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>Polyketide</td>
<td>depudecin</td>
<td>Alternaria brassicicola</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002227: 270-18409">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002227/index.html#r1c1" target="_new">BGC0002227</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.84)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>NRP</td>
<td>AKML B, AKML D, AKML A, AKML C</td>
<td>Aspergillus luchuensis IFO 4308</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001124: 3228-30082">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001124/index.html#r1c1" target="_new">BGC0001124</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>Polyketide</td>
<td>pyranonigrin E</td>
<td>Aspergillus niger ATCC 1015</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002266: 1041-19123">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002266/index.html#r1c1" target="_new">BGC0002266</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Terpene, Polyketide</td>
<td>calidoustene A, calidoustene B, calidoustene C</td>
<td>Aspergillus calidoustus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001913: 7717-40102">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001913/index.html#r1c1" target="_new">BGC0001913</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Polyketide</td>
<td>phomoidride</td>
<td>fungal sp. ATCC 74256</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002211: 74-16330">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002211/index.html#r1c1" target="_new">BGC0002211</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Polyketide</td>
<td>HEx-pks1 polyketide</td>
<td>Paraphaeosphaeria sporulosa</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000056: 2-6689">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000056/index.html#r1c1" target="_new">BGC0000056</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.41, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Polyketide</td>
<td>esperamicin</td>
<td>Actinomadura verrucosospora</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002240: 96-11225">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002240/index.html#r1c1" target="_new">BGC0002240</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Polyketide</td>
<td>BAB, BAA</td>
<td>Metarhizium anisopliae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001141: 262-37660">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001141/index.html#r1c1" target="_new">BGC0001141</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Polyketide</td>
<td>4-epi-15-epi-brefeldin A</td>
<td>Penicillium brefeldianum</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r1c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-3"></div> <span class="help-tooltip" id="cb-general-help-3">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r1c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r1c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r1c1_1.svg">
NW_024467524 (2917559-2985739): Aspergillus versicolor CBS 583.65 unplaced ge... (7% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r1c1_2.svg">
NT_165977 (3031183-3103199): Chaetomium globosum CBS 148.51 scaffold 2 genomi... (11% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r1c1_3.svg">
NW_019170464 (740714-852986): Aspergillus glaucus CBS 516.65 unplaced genomic... (6% of genes show similarity), T1PKS,indole
</option>

<option value = "svg/clusterblast_r1c1_4.svg">
NZ_CP029189 (1019213-1210762): Streptomyces venezuelae strain ATCC 21018 chro... (16% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r1c1_5.svg">
NZ_CP023689 (953739-1189865): Streptomyces chartreusis strain ATCC 14922 chro... (11% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r1c1_6.svg">
NW_024467308 (148929-260597): Aspergillus ruber CBS 135680 unplaced genomic s... (8% of genes show similarity), T1PKS,indole
</option>

<option value = "svg/clusterblast_r1c1_7.svg">
NC_014815 (6360861-6579562): Micromonospora sp. L5, complete sequence (11% of genes show similarity), 2dos,NRPS,NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r1c1_8.svg">
NZ_CP116353 (8558814-8795265): Streptomyces sp. HUAS 31 chromosome, complete ... (9% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r1c1_9.svg">
NZ_CP095481 (5894087-6205936): Amycolatopsis sp. WQ 127309 chromosome, comple... (4% of genes show similarity), NRPS,NRPS-like,T1PKS,nucleoside
</option>

<option value = "svg/clusterblast_r1c1_10.svg">
NZ_CP029711 (7772223-7941678): Streptosporangium sp. &#39;caverna&#39; chromosome (11% of genes show similarity), T1PKS
</option>

</select>
<button id= "clusterblast-r1c1-download">Download graphic</button>
</div>
<div id="clusterblast-r1c1-svg"></div>

</div>
</div>

<div class="body-details-section r1c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-4"></div> <span class="help-tooltip" id="cb-known-help-4">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r1c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-5"></div> <span class="help-tooltip" id="cb-sub-help-5">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r1c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-8"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-8">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c1-domain-bubble-svg" data-filename="scaffold_1_r1c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r1c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (117939 - 165651): single T1PKS
</option>


</select>

<div class="details-svg" id="r1c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r1c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r1c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-10"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-10">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c1-pfam-details-svg" data-filename="scaffold_1_r1c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r1c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r1c1-nrps_pks_products" data-name="r1c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r1c1-nrps_pks_monomers" data-name="r1c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r1c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-6"></div> <span class="help-tooltip" id="nrps-structures-help-6">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 117939 - 165651:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r1c1-1" class="smiles-canvas" data-smiles="C([*])C(-O)"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(pk)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r1c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r1c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-7"></div> <span class="help-tooltip" id="nrps-monomers-help-7">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">FUN_000044</span></strong>: pk
<div class="collapser collapser-target-FUN_000044 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (538..854)</strong>: pk
<div class="collapser collapser-target-FUN_000044 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-FUN_000044-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 70.8%</dd>
<dd>Methylmalonyl-CoA: 58.3%</dd>
<dd>Ethylmalonyl-CoA: 54.2%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-FUN_000044-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 71.4</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 61.6</dt>

<dd></dd><dt>Malonyl-CoA: 57.1</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 46.9</dt>

<dd></dd><dt>Propionyl-CoA: 41.9</dt>

<dd></dd><dt>Isobutyryl-CoA: 18.5</dt>

<dd></dd><dt>CHC-CoA: 18.1</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 17.8</dt>

<dd></dd><dt>trans-1,2-CPDA: 17.4</dt>

<dd></dd><dt>fatty_acid: 15.2</dt>

<dd></dd><dt>inactive: 14.6</dt>

<dd></dd><dt>Benzoyl-CoA: 14.0</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 4.9</dt>

<dd></dd><dt>Acetyl-CoA: 3.2</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>










</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r1c2' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_1 - Region 2 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-717"></div> <span class="help-tooltip" id="region_svg-help-717">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_1.region002.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r1c2-svg" data-filename="scaffold_1_r1c2.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 408,074 - 429,212 nt. (total: 21,139 nt)

<a class="cluster-rules-header" id="r1c2-rules-header" href="#r1c2">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r1c2-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>

<div class="region-svg-container">
<div id='r1c2-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-718"></div> <span class="help-tooltip" id="focus-panel-help-718">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r1c2">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r1c2-gene-table-details" data-name="r1c2-gene-table-details">
Gene overview
</div>



<div class="body-details-header r1c2-MIBiG-cluster-compare" data-name="r1c2-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r1c2-clusterblast" data-name="r1c2-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r1c2-knownclusterblast" data-name="r1c2-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r1c2-subclusterblast" data-name="r1c2-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r1c2-pfam-details" data-name="r1c2-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r1c2-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-15"></div> <span class="help-tooltip" id="gt-help-tooltip-help-15">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r1c2">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r1c2-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r1c2-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r1c2">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r1c2-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-11"></div> <span class="help-tooltip" id="MIBiG-help-11">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r1c2-selector">Analysis type:</label>
<select id="comparison-MIBiG-r1c2-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r1c2"></div>
<div id="comparison-MIBiG-r1c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">terpene</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002176: 0-1199">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002176/index.html#r1c1" target="_new">BGC0002176</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.34</td>
<td>Terpene</td>

<td>culmorin, (+)-juniperol(longiborneol), 15-acetyldeoxynivalenol</td>


<td>Fusarium graminearum PH-1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001278: 588-54488">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001278/index.html#r1c1" target="_new">BGC0001278</a></td>





<td class="cc-heat-cell" title="0.23 (id:0.42, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>


<td>0.05</td>
<td>Terpene</td>

<td>nivalenol, deoxynivalenol, 3-acetyldeoxynivalenol, 15-acetyldeoxynivalenol, neosolaniol, calonectrin, apotrichodiol, isotrichotriol, 15-decalonectrin, T-2 toxin, 3-acetyl T-2 toxin, trichodiene</td>


<td>Fusarium sporotrichioides</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r1c2-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002176: 0-1199">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002176/index.html#r1c1" target="_new">BGC0002176</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Terpene</td>
<td>culmorin, (+)-juniperol(longiborneol), 15-acetyldeoxynivalenol</td>
<td>Fusarium graminearum PH-1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001278: 588-54488">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001278/index.html#r1c1" target="_new">BGC0001278</a></td>


<td class="cc-heat-cell" title="0.23 (id:0.42, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>
<td>0.23</td>
<td>Terpene</td>
<td>nivalenol, deoxynivalenol, 3-acetyldeoxynivalenol, 15-acetyldeoxynivalenol, neosolaniol, calonectrin, apotrichodiol, isotrichotriol, 15-decalonectrin, T-2 toxin, 3-acetyl T-2 toxin, trichodiene</td>
<td>Fusarium sporotrichioides</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r1c2-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-12"></div> <span class="help-tooltip" id="cb-general-help-12">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r1c2-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r1c2_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r1c2_1.svg">
NW_021167081 (1611604-1643292): Sodiomyces alkalinus F11 unplaced genomic sca... (18% of genes show similarity), terpene
</option>

</select>
<button id= "clusterblast-r1c2-download">Download graphic</button>
</div>
<div id="clusterblast-r1c2-svg"></div>

</div>
</div>

<div class="body-details-section r1c2-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-13"></div> <span class="help-tooltip" id="cb-known-help-13">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r1c2-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-14"></div> <span class="help-tooltip" id="cb-sub-help-14">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r1c2-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-16"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-16">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c2-pfam-details-svg" data-filename="scaffold_1_r1c2_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r1c2-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">









</div>









</div>

</div>

</div>

<div class="page" id='r1c3' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_1 - Region 3 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-719"></div> <span class="help-tooltip" id="region_svg-help-719">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_1.region003.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r1c3-svg" data-filename="scaffold_1_r1c3.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 892,788 - 914,401 nt. (total: 21,614 nt)

<a class="cluster-rules-header" id="r1c3-rules-header" href="#r1c3">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r1c3-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>

<div class="region-svg-container">
<div id='r1c3-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-720"></div> <span class="help-tooltip" id="focus-panel-help-720">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r1c3">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r1c3-gene-table-details" data-name="r1c3-gene-table-details">
Gene overview
</div>



<div class="body-details-header r1c3-MIBiG-cluster-compare" data-name="r1c3-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r1c3-clusterblast" data-name="r1c3-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r1c3-knownclusterblast" data-name="r1c3-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r1c3-subclusterblast" data-name="r1c3-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r1c3-pfam-details" data-name="r1c3-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r1c3-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-21"></div> <span class="help-tooltip" id="gt-help-tooltip-help-21">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r1c3">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r1c3-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r1c3-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r1c3">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r1c3-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-17"></div> <span class="help-tooltip" id="MIBiG-help-17">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r1c3-selector">Analysis type:</label>
<select id="comparison-MIBiG-r1c3-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r1c3"></div>
<div id="comparison-MIBiG-r1c3-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">terpene</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000622: 395-4938">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000622/index.html#r1c1" target="_new">BGC0000622</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.32</td>
<td>RiPP</td>

<td>megacin</td>


<td>Bacillus megaterium</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001839: 5117-20679">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001839/index.html#r1c1" target="_new">BGC0001839</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.54, order:0.43, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.28</td>
<td>Terpene</td>

<td>squalestatin S1</td>


<td>Aspergillus sp. Z5</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000595: 0-21771">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000595/index.html#r1c1" target="_new">BGC0000595</a></td>





<td class="cc-heat-cell" title="0.42 (id:0.44, order:N/A, components:0.39)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>


<td>0.13</td>
<td>RiPP</td>

<td>SCO-2138</td>


<td>Streptomyces coelicolor A3(2)</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000596: 0-21820">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000596/index.html#r1c1" target="_new">BGC0000596</a></td>





<td class="cc-heat-cell" title="0.42 (id:0.44, order:N/A, components:0.39)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>


<td>0.13</td>
<td>RiPP</td>

<td>SLI-2138</td>


<td>Streptomyces lividans TK24</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000597: 0-10846">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000597/index.html#r1c1" target="_new">BGC0000597</a></td>





<td class="cc-heat-cell" title="0.36 (id:0.45, order:N/A, components:0.29)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>


<td>0.10</td>
<td>RiPP</td>

<td>SWA-2138</td>


<td>Streptomyces sp. e14</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000842: 990-43743">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000842/index.html#r1c1" target="_new">BGC0000842</a></td>





<td class="cc-heat-cell" title="0.29 (id:0.46, order:N/A, components:0.18)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>


<td>0.07</td>
<td>Other (Non-NRP beta-lactam)</td>

<td>carbapenem MM4550</td>


<td>Streptomyces argenteolus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000473: 867-13221">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000473/index.html#r1c1" target="_new">BGC0000473</a></td>





<td class="cc-heat-cell" title="0.27 (id:0.43, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>


<td>0.07</td>
<td>RiPP</td>

<td>microcyclamide</td>


<td>Microcystis aeruginosa NIES-298</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001339: 45-69577">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001339/index.html#r1c1" target="_new">BGC0001339</a></td>





<td class="cc-heat-cell" title="0.23 (id:0.53, order:N/A, components:0.10)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>


<td>0.05</td>
<td>Polyketide</td>

<td>squalestatin S1</td>


<td>Phoma sp. MF5453</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002122: 2491-54939">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002122/index.html#r1c1" target="_new">BGC0002122</a></td>





<td class="cc-heat-cell" title="0.22 (id:0.45, order:N/A, components:0.11)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.22), rgb(0, 0, 0, 0.22) 22%, white 22%)"></td>


<td>0.05</td>
<td>NRP</td>

<td>gramicidin S</td>


<td>Aneurinibacillus migulanus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002338: 56-82305">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002338/index.html#r1c1" target="_new">BGC0002338</a></td>





<td class="cc-heat-cell" title="0.18 (id:0.46, order:N/A, components:0.07)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.18), rgb(0, 0, 0, 0.18) 18%, white 18%)"></td>


<td>0.04</td>
<td>Polyketide</td>

<td>sorangipyranone</td>


<td>Myxococcales bacterium</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r1c3-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000622: 395-4938">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000622/index.html#r1c1" target="_new">BGC0000622</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>RiPP</td>
<td>megacin</td>
<td>Bacillus megaterium</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001839: 5117-20679">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001839/index.html#r1c1" target="_new">BGC0001839</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.54, order:0.43, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>Terpene</td>
<td>squalestatin S1</td>
<td>Aspergillus sp. Z5</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000595: 0-21771">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000595/index.html#r1c1" target="_new">BGC0000595</a></td>


<td class="cc-heat-cell" title="0.42 (id:0.44, order:N/A, components:0.39)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
<td>0.42</td>
<td>RiPP</td>
<td>SCO-2138</td>
<td>Streptomyces coelicolor A3(2)</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000596: 0-21820">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000596/index.html#r1c1" target="_new">BGC0000596</a></td>


<td class="cc-heat-cell" title="0.42 (id:0.44, order:N/A, components:0.39)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
<td>0.42</td>
<td>RiPP</td>
<td>SLI-2138</td>
<td>Streptomyces lividans TK24</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000597: 0-10846">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000597/index.html#r1c1" target="_new">BGC0000597</a></td>


<td class="cc-heat-cell" title="0.36 (id:0.45, order:N/A, components:0.29)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>
<td>0.36</td>
<td>RiPP</td>
<td>SWA-2138</td>
<td>Streptomyces sp. e14</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000842: 990-43743">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000842/index.html#r1c1" target="_new">BGC0000842</a></td>


<td class="cc-heat-cell" title="0.29 (id:0.46, order:N/A, components:0.18)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>
<td>0.29</td>
<td>Other (Non-NRP beta-lactam)</td>
<td>carbapenem MM4550</td>
<td>Streptomyces argenteolus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000473: 867-13221">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000473/index.html#r1c1" target="_new">BGC0000473</a></td>


<td class="cc-heat-cell" title="0.27 (id:0.43, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>
<td>0.27</td>
<td>RiPP</td>
<td>microcyclamide</td>
<td>Microcystis aeruginosa NIES-298</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001339: 45-69577">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001339/index.html#r1c1" target="_new">BGC0001339</a></td>


<td class="cc-heat-cell" title="0.23 (id:0.53, order:N/A, components:0.10)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>
<td>0.23</td>
<td>Polyketide</td>
<td>squalestatin S1</td>
<td>Phoma sp. MF5453</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002122: 2491-54939">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002122/index.html#r1c1" target="_new">BGC0002122</a></td>


<td class="cc-heat-cell" title="0.22 (id:0.45, order:N/A, components:0.11)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.22), rgb(0, 0, 0, 0.22) 22%, white 22%)"></td>
<td>0.22</td>
<td>NRP</td>
<td>gramicidin S</td>
<td>Aneurinibacillus migulanus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002338: 56-82305">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002338/index.html#r1c1" target="_new">BGC0002338</a></td>


<td class="cc-heat-cell" title="0.18 (id:0.46, order:N/A, components:0.07)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.18), rgb(0, 0, 0, 0.18) 18%, white 18%)"></td>
<td>0.18</td>
<td>Polyketide</td>
<td>sorangipyranone</td>
<td>Myxococcales bacterium</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r1c3-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-18"></div> <span class="help-tooltip" id="cb-general-help-18">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r1c3-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r1c3_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r1c3_1.svg">
NW_022984651 (919341-950958): Daldinia childiae strain JS-1345 chromosome Unk... (40% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r1c3_2.svg">
NC_030957 (2977491-3080937): Colletotrichum higginsianum IMI 349063 chromosom... (15% of genes show similarity), fungal-RiPP-like,terpene
</option>

<option value = "svg/clusterblast_r1c3_3.svg">
NW_006917094 (2557064-2588592): Pestalotiopsis fici W106-1 unplaced genomic s... (40% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r1c3_4.svg">
NW_009799103 (1803097-1834751): Neurospora tetrasperma FGSC 2508 unplaced gen... (33% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r1c3_5.svg">
NC_016459 (1649781-1681459): Thermothielavioides terrestris NRRL 8126 chromos... (33% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r1c3_6.svg">
NC_016476 (3077694-3109368): Thermothelomyces thermophilus ATCC 42464 chromos... (40% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r1c3_7.svg">
NT_165983 (454288-485903): Chaetomium globosum CBS 148.51 scaffold 8 genomic ... (33% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r1c3_8.svg">
NW_009276944 (136519-168106): Verticillium dahliae VdLs.17 supercont1.13 geno... (50% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r1c3_9.svg">
NW_025408672 (521045-552605): Hirsutella rhossiliensis strain HR02 chromosome... (44% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r1c3_10.svg">
NC_072843 (3340945-3372495): Fusarium oxysporum Fo47 chromosome IV, complete ... (40% of genes show similarity), terpene
</option>

</select>
<button id= "clusterblast-r1c3-download">Download graphic</button>
</div>
<div id="clusterblast-r1c3-svg"></div>

</div>
</div>

<div class="body-details-section r1c3-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-19"></div> <span class="help-tooltip" id="cb-known-help-19">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r1c3-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r1c3_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r1c3_1.svg">
squalestatin S1
</option>

</select>
<button id="knownclusterblast-r1c3-download">Download graphic</button>
</div>
<div id="knownclusterblast-r1c3-svg"></div>

</div>
</div>

<div class="body-details-section r1c3-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-20"></div> <span class="help-tooltip" id="cb-sub-help-20">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r1c3-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-22"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-22">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c3-pfam-details-svg" data-filename="scaffold_1_r1c3_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r1c3-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">









</div>









</div>

</div>

</div>





<div class="page" id='r2c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_2 - Region 1 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-721"></div> <span class="help-tooltip" id="region_svg-help-721">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_2.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r2c1-svg" data-filename="scaffold_2_r2c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 190,620 - 211,845 nt. (total: 21,226 nt)

<a class="cluster-rules-header" id="r2c1-rules-header" href="#r2c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r2c1-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>

<div class="region-svg-container">
<div id='r2c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-722"></div> <span class="help-tooltip" id="focus-panel-help-722">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r2c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r2c1-gene-table-details" data-name="r2c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r2c1-MIBiG-cluster-compare" data-name="r2c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r2c1-clusterblast" data-name="r2c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r2c1-knownclusterblast" data-name="r2c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r2c1-subclusterblast" data-name="r2c1-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r2c1-pfam-details" data-name="r2c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r2c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-27"></div> <span class="help-tooltip" id="gt-help-tooltip-help-27">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r2c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r2c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r2c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r2c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r2c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-23"></div> <span class="help-tooltip" id="MIBiG-help-23">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r2c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r2c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r2c1"></div>
<div id="comparison-MIBiG-r2c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">terpene</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002736: 3-5057">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002736/index.html#r1c1" target="_new">BGC0002736</a></td>





<td class="cc-heat-cell" title="0.70 (id:0.45, order:1.00, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>


<td>0.37</td>
<td>Terpene</td>

<td>preaspterpenacid I</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002320: 3-7131">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002320/index.html#r1c1" target="_new">BGC0002320</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.48, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.26</td>
<td>Terpene</td>

<td>conidiogenone</td>


<td>Penicillium rubens Wisconsin 54-1255</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002610: 0-6468">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002610/index.html#r1c1" target="_new">BGC0002610</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.25</td>
<td>Terpene</td>

<td>stellatic acid</td>


<td>Aspergillus stellatus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002162: 0-5465">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002162/index.html#r1c1" target="_new">BGC0002162</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.25</td>
<td>Terpene</td>

<td>sesterfisherol, sesterfisheric acid, compound 6, compound 7a, compound 5, compound 8</td>


<td>Aspergillus fischeri NRRL 181</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000688: 696-13661">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000688/index.html#r1c1" target="_new">BGC0000688</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.56, order:0.45, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>Terpene</td>

<td>copalyl diphosphate</td>


<td>Diaporthe amygdali</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002260: 942-16838">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002260/index.html#r1c1" target="_new">BGC0002260</a></td>





<td class="cc-heat-cell" title="0.51 (id:0.45, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>


<td>0.19</td>
<td>Terpene</td>

<td>trichobrasilenol, xylarenic acid B, brasilane A, brasilane F, brasilane E, brasilane D</td>


<td>Annulohypoxylon truncatum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000673: 0-9097">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000673/index.html#r1c1" target="_new">BGC0000673</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.50, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>Terpene</td>

<td>pimara-8(14),15-diene</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001969: 0-7810">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001969/index.html#r1c1" target="_new">BGC0001969</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.47, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.17</td>
<td>Terpene</td>

<td>asperterpenoid A</td>


<td>Talaromyces wortmannii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002400: 0-52506">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002400/index.html#r1c1" target="_new">BGC0002400</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.15</td>
<td>Terpene</td>

<td>boleracene</td>


<td>Brassica oleracea var. oleracea</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002397: 152-16704">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002397/index.html#r1c1" target="_new">BGC0002397</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.15</td>
<td>Terpene</td>

<td>astallatene</td>


<td>Arabidopsis thaliana</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r2c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002736: 3-5057">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002736/index.html#r1c1" target="_new">BGC0002736</a></td>


<td class="cc-heat-cell" title="0.70 (id:0.45, order:1.00, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
<td>0.70</td>
<td>Terpene</td>
<td>preaspterpenacid I</td>
<td>Aspergillus terreus NIH2624</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002320: 3-7131">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002320/index.html#r1c1" target="_new">BGC0002320</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.48, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>Terpene</td>
<td>conidiogenone</td>
<td>Penicillium rubens Wisconsin 54-1255</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002610: 0-6468">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002610/index.html#r1c1" target="_new">BGC0002610</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Terpene</td>
<td>stellatic acid</td>
<td>Aspergillus stellatus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002162: 0-5465">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002162/index.html#r1c1" target="_new">BGC0002162</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Terpene</td>
<td>sesterfisherol, sesterfisheric acid, compound 6, compound 7a, compound 5, compound 8</td>
<td>Aspergillus fischeri NRRL 181</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000688: 696-13661">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000688/index.html#r1c1" target="_new">BGC0000688</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.56, order:0.45, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Terpene</td>
<td>copalyl diphosphate</td>
<td>Diaporthe amygdali</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002260: 942-16838">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002260/index.html#r1c1" target="_new">BGC0002260</a></td>


<td class="cc-heat-cell" title="0.51 (id:0.45, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
<td>0.51</td>
<td>Terpene</td>
<td>trichobrasilenol, xylarenic acid B, brasilane A, brasilane F, brasilane E, brasilane D</td>
<td>Annulohypoxylon truncatum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000673: 0-9097">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000673/index.html#r1c1" target="_new">BGC0000673</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.50, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Terpene</td>
<td>pimara-8(14),15-diene</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001969: 0-7810">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001969/index.html#r1c1" target="_new">BGC0001969</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.47, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>Terpene</td>
<td>asperterpenoid A</td>
<td>Talaromyces wortmannii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002400: 0-52506">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002400/index.html#r1c1" target="_new">BGC0002400</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>Terpene</td>
<td>boleracene</td>
<td>Brassica oleracea var. oleracea</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002397: 152-16704">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002397/index.html#r1c1" target="_new">BGC0002397</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>Terpene</td>
<td>astallatene</td>
<td>Arabidopsis thaliana</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r2c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-24"></div> <span class="help-tooltip" id="cb-general-help-24">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r2c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r2c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r2c1_1.svg">
NW_024467532 (1067639-1098773): Aspergillus versicolor CBS 583.65 unplaced ge... (38% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r2c1_2.svg">
NW_020292110 (122545-258460): Aspergillus ibericus CBS 121593 unplaced genomi... (6% of genes show similarity), NRPS,T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r2c1_3.svg">
NW_022985018 (1161025-1297869): Dothidotthia symphoricarpi CBS 119687 unplace... (6% of genes show similarity), T1PKS,fungal-RiPP-like,terpene
</option>

<option value = "svg/clusterblast_r2c1_4.svg">
NC_036626 (4276159-4309215): Fusarium fujikuroi IMI 58289 draft genome, chrom... (25% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r2c1_5.svg">
NC_058390 (226406-259263): Fusarium musae strain F31 chromosome 4, whole geno... (33% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r2c1_6.svg">
NW_022156716 (6659694-6736206): Pyricularia grisea strain NI907 chromosome Un... (13% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r2c1_7.svg">
NW_025408675 (1627067-1681362): Hirsutella rhossiliensis strain HR02 chromoso... (36% of genes show similarity), indole,terpene
</option>

<option value = "svg/clusterblast_r2c1_8.svg">
NC_036437 (172443-252235): Aspergillus oryzae RIB40 DNA, chromosome 3 (6% of genes show similarity), NRPS-like,terpene
</option>

<option value = "svg/clusterblast_r2c1_9.svg">
NC_054693 (237013-316681): Aspergillus flavus NRRL3357 chromosome 3, complete... (6% of genes show similarity), NRPS-like,terpene
</option>

<option value = "svg/clusterblast_r2c1_10.svg">
NW_011942193 (236578-275727): Metarhizium robertsii ARSEF 23 MAA Scf 5, whole... (18% of genes show similarity), indole,terpene
</option>

</select>
<button id= "clusterblast-r2c1-download">Download graphic</button>
</div>
<div id="clusterblast-r2c1-svg"></div>

</div>
</div>

<div class="body-details-section r2c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-25"></div> <span class="help-tooltip" id="cb-known-help-25">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r2c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r2c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r2c1_1.svg">
copalyl diphosphate
</option>

<option value = "svg/knownclusterblast_r2c1_2.svg">
aphidicolin/aphidicolan-16-ol/3-deoxyaphidicolin/17-deoxyaphidicolin
</option>

</select>
<button id="knownclusterblast-r2c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r2c1-svg"></div>

</div>
</div>

<div class="body-details-section r2c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-26"></div> <span class="help-tooltip" id="cb-sub-help-26">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r2c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-28"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-28">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c1-pfam-details-svg" data-filename="scaffold_2_r2c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r2c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">









</div>









</div>

</div>

</div>

<div class="page" id='r2c2' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_2 - Region 2 - NRPS-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-723"></div> <span class="help-tooltip" id="region_svg-help-723">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_2.region002.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r2c2-svg" data-filename="scaffold_2_r2c2.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1,207,872 - 1,250,363 nt. (total: 42,492 nt)

<a class="cluster-rules-header" id="r2c2-rules-header" href="#r2c2">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r2c2-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r2c2-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-724"></div> <span class="help-tooltip" id="focus-panel-help-724">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r2c2">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r2c2-gene-table-details" data-name="r2c2-gene-table-details">
Gene overview
</div>



<div class="body-details-header r2c2-nrps_pks" data-name="r2c2-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r2c2-MIBiG-cluster-compare" data-name="r2c2-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r2c2-clusterblast" data-name="r2c2-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r2c2-knownclusterblast" data-name="r2c2-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r2c2-subclusterblast" data-name="r2c2-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r2c2-pfam-details" data-name="r2c2-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r2c2-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-34"></div> <span class="help-tooltip" id="gt-help-tooltip-help-34">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r2c2">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r2c2-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r2c2-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r2c2">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r2c2-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-29"></div> <span class="help-tooltip" id="nrps-domain-help-29">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c2-details-svg" data-filename="scaffold_2_r2c2_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r2c2-details-svg"></div>
</div>
</div>



<div class="body-details-section r2c2-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-30"></div> <span class="help-tooltip" id="MIBiG-help-30">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r2c2-selector">Analysis type:</label>
<select id="comparison-MIBiG-r2c2-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r2c2"></div>
<div id="comparison-MIBiG-r2c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000099: 0-8104">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000099/index.html#r1c1" target="_new">BGC0000099</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.15</td>
<td>Polyketide</td>

<td>monascorubrin</td>


<td>Talaromyces marneffei</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001268: 894-12763">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001268/index.html#r1c1" target="_new">BGC0001268</a></td>





<td class="cc-heat-cell" title="0.44 (id:0.41, order:N/A, components:0.48)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>


<td>0.14</td>
<td>NRP, Polyketide</td>

<td>fusarin C</td>


<td>Fusarium fujikuroi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001599: 0-9373">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001599/index.html#r1c1" target="_new">BGC0001599</a></td>





<td class="cc-heat-cell" title="0.42 (id:0.41, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>


<td>0.13</td>
<td>NRP</td>

<td>fragin</td>


<td>Burkholderia cenocepacia H111</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001390: 100-19645">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001390/index.html#r1c1" target="_new">BGC0001390</a></td>





<td class="cc-heat-cell" title="0.40 (id:0.42, order:N/A, components:0.39)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>


<td>0.12</td>
<td>NRP, Polyketide</td>

<td>LL-Z1272beta</td>


<td>Stachybotrys bisbyi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001600: 0-23599">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001600/index.html#r1c1" target="_new">BGC0001600</a></td>





<td class="cc-heat-cell" title="0.35 (id:0.42, order:N/A, components:0.29)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>


<td>0.10</td>
<td>Polyketide</td>

<td>fusarielin H</td>


<td>Fusarium graminearum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001624: 23-43456">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001624/index.html#r1c1" target="_new">BGC0001624</a></td>





<td class="cc-heat-cell" title="0.35 (id:0.49, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>


<td>0.10</td>
<td>Other</td>

<td>iso-A82775C</td>


<td>Pestalotiopsis fici</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001722: 0-18441">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001722/index.html#r1c1" target="_new">BGC0001722</a></td>





<td class="cc-heat-cell" title="0.34 (id:0.42, order:N/A, components:0.28)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>


<td>0.09</td>
<td>Polyketide</td>

<td>aspernidine A</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001182: 0-33994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001182/index.html#r1c1" target="_new">BGC0001182</a></td>





<td class="cc-heat-cell" title="0.34 (id:0.41, order:N/A, components:0.28)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>


<td>0.09</td>
<td>NRP, Polyketide</td>

<td>chaetoglobosins</td>


<td>Chaetomium globosum CBS 148.51</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002062: 1607-32928">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002062/index.html#r1c1" target="_new">BGC0002062</a></td>





<td class="cc-heat-cell" title="0.33 (id:0.50, order:N/A, components:0.22)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>


<td>0.09</td>
<td>Polyketide</td>

<td>agnestin A, agnestin B</td>


<td>Paecilomyces divaricatus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001338: 273-42813">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001338/index.html#r1c1" target="_new">BGC0001338</a></td>





<td class="cc-heat-cell" title="0.33 (id:0.41, order:N/A, components:0.26)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>


<td>0.09</td>
<td>Polyketide</td>

<td>citrinin</td>


<td>Monascus ruber</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r2c2-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000099: 0-8104">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000099/index.html#r1c1" target="_new">BGC0000099</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>Polyketide</td>
<td>monascorubrin</td>
<td>Talaromyces marneffei</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001268: 894-12763">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001268/index.html#r1c1" target="_new">BGC0001268</a></td>


<td class="cc-heat-cell" title="0.44 (id:0.41, order:N/A, components:0.48)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
<td>0.44</td>
<td>NRP, Polyketide</td>
<td>fusarin C</td>
<td>Fusarium fujikuroi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001599: 0-9373">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001599/index.html#r1c1" target="_new">BGC0001599</a></td>


<td class="cc-heat-cell" title="0.42 (id:0.41, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
<td>0.42</td>
<td>NRP</td>
<td>fragin</td>
<td>Burkholderia cenocepacia H111</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001390: 100-19645">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001390/index.html#r1c1" target="_new">BGC0001390</a></td>


<td class="cc-heat-cell" title="0.40 (id:0.42, order:N/A, components:0.39)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>
<td>0.40</td>
<td>NRP, Polyketide</td>
<td>LL-Z1272beta</td>
<td>Stachybotrys bisbyi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001600: 0-23599">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001600/index.html#r1c1" target="_new">BGC0001600</a></td>


<td class="cc-heat-cell" title="0.35 (id:0.42, order:N/A, components:0.29)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>
<td>0.35</td>
<td>Polyketide</td>
<td>fusarielin H</td>
<td>Fusarium graminearum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001624: 23-43456">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001624/index.html#r1c1" target="_new">BGC0001624</a></td>


<td class="cc-heat-cell" title="0.35 (id:0.49, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>
<td>0.35</td>
<td>Other</td>
<td>iso-A82775C</td>
<td>Pestalotiopsis fici</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001722: 0-18441">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001722/index.html#r1c1" target="_new">BGC0001722</a></td>


<td class="cc-heat-cell" title="0.34 (id:0.42, order:N/A, components:0.28)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>
<td>0.34</td>
<td>Polyketide</td>
<td>aspernidine A</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001182: 0-33994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001182/index.html#r1c1" target="_new">BGC0001182</a></td>


<td class="cc-heat-cell" title="0.34 (id:0.41, order:N/A, components:0.28)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>
<td>0.34</td>
<td>NRP, Polyketide</td>
<td>chaetoglobosins</td>
<td>Chaetomium globosum CBS 148.51</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002062: 1607-32928">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002062/index.html#r1c1" target="_new">BGC0002062</a></td>


<td class="cc-heat-cell" title="0.33 (id:0.50, order:N/A, components:0.22)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>
<td>0.33</td>
<td>Polyketide</td>
<td>agnestin A, agnestin B</td>
<td>Paecilomyces divaricatus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001338: 273-42813">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001338/index.html#r1c1" target="_new">BGC0001338</a></td>


<td class="cc-heat-cell" title="0.33 (id:0.41, order:N/A, components:0.26)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>
<td>0.33</td>
<td>Polyketide</td>
<td>citrinin</td>
<td>Monascus ruber</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r2c2-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-31"></div> <span class="help-tooltip" id="cb-general-help-31">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r2c2-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r2c2_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r2c2_1.svg">
NC_054697 (763907-942184): Aspergillus flavus NRRL3357 chromosome 7, complete... (3% of genes show similarity), NRPS-like,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r2c2_2.svg">
NW_024469015 (107323-190724): Aspergillus fijiensis CBS 313.89 unplaced genom... (11% of genes show similarity), NRPS-like,terpene
</option>

<option value = "svg/clusterblast_r2c2_3.svg">
NW_024467533 (148342-258756): Aspergillus versicolor CBS 583.65 unplaced geno... (4% of genes show similarity), NRPS,NRPS-like,betalactone
</option>

<option value = "svg/clusterblast_r2c2_4.svg">
NW_008481799 (388930-472807): Cyphellophora europaea CBS 101466 unplaced geno... (6% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r2c2_5.svg">
NC_054860 (4191964-4296027): Aspergillus puulaauensis MK2 DNA, chromosome 4, ... (4% of genes show similarity), NRPS,NRPS-like
</option>

<option value = "svg/clusterblast_r2c2_6.svg">
NW_026250022 (562601-594124): Hypoxylon trugodes strain CBS 135444 unplaced g... (20% of genes show similarity), indole
</option>

<option value = "svg/clusterblast_r2c2_7.svg">
NW_026054567 (410981-502039): Daldinia decipiens strain CBS 113046 unplaced g... (11% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r2c2_8.svg">
NW_026250020 (3474063-3565041): Hypoxylon trugodes strain CBS 135444 unplaced... (11% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r2c2_9.svg">
NW_025953038 (42837-133391): Daldinia caldariorum strain CBS 122874 unplaced ... (13% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r2c2_10.svg">
NC_036624 (4849209-4913016): Fusarium fujikuroi IMI 58289 draft genome, chrom... (8% of genes show similarity), NRPS-like
</option>

</select>
<button id= "clusterblast-r2c2-download">Download graphic</button>
</div>
<div id="clusterblast-r2c2-svg"></div>

</div>
</div>

<div class="body-details-section r2c2-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-32"></div> <span class="help-tooltip" id="cb-known-help-32">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r2c2-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-33"></div> <span class="help-tooltip" id="cb-sub-help-33">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r2c2-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-35"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-35">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c2-pfam-details-svg" data-filename="scaffold_2_r2c2_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r2c2-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">













</div>













</div>

</div>

</div>





<div class="page" id='r3c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_3 - Region 1 - fungal-RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-725"></div> <span class="help-tooltip" id="region_svg-help-725">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_3.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r3c1-svg" data-filename="scaffold_3_r3c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 245,315 - 308,903 nt. (total: 63,589 nt)

<a class="cluster-rules-header" id="r3c1-rules-header" href="#r3c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r3c1-rules">fungal-RiPP-like: DUF3328</div>

<div class="region-svg-container">
<div id='r3c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-726"></div> <span class="help-tooltip" id="focus-panel-help-726">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r3c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r3c1-gene-table-details" data-name="r3c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r3c1-MIBiG-cluster-compare" data-name="r3c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r3c1-clusterblast" data-name="r3c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r3c1-knownclusterblast" data-name="r3c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r3c1-subclusterblast" data-name="r3c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r3c1-pfam-details" data-name="r3c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r3c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-40"></div> <span class="help-tooltip" id="gt-help-tooltip-help-40">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r3c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r3c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r3c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r3c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r3c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-36"></div> <span class="help-tooltip" id="MIBiG-help-36">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r3c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r3c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r3c1"></div>
<div id="comparison-MIBiG-r3c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">fungal-RiPP-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001775: 2711-20673">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001775/index.html#r1c1" target="_new">BGC0001775</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.15</td>
<td>Terpene</td>

<td>sespendole</td>


<td>Cordana terrestris</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001260: 0-15275">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001260/index.html#r1c1" target="_new">BGC0001260</a></td>





<td class="cc-heat-cell" title="0.41 (id:0.41, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>


<td>0.12</td>
<td>Terpene</td>

<td>terpendole E</td>


<td>Tolypocladium album</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001624: 23-43456">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001624/index.html#r1c1" target="_new">BGC0001624</a></td>





<td class="cc-heat-cell" title="0.40 (id:0.44, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>


<td>0.12</td>
<td>Other</td>

<td>iso-A82775C</td>


<td>Pestalotiopsis fici</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002603: 0-17042">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002603/index.html#r1c1" target="_new">BGC0002603</a></td>





<td class="cc-heat-cell" title="0.38 (id:0.44, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>


<td>0.11</td>
<td>Polyketide</td>

<td>chevalone E</td>


<td>Aspergillus versicolor</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002604: 0-17897">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002604/index.html#r1c1" target="_new">BGC0002604</a></td>





<td class="cc-heat-cell" title="0.38 (id:0.43, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>


<td>0.11</td>
<td>Polyketide</td>

<td>sartorypyrone A</td>


<td>Aspergillus felis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001858: 715-19483">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001858/index.html#r1c1" target="_new">BGC0001858</a></td>





<td class="cc-heat-cell" title="0.37 (id:0.42, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>


<td>0.11</td>
<td>Polyketide</td>

<td>alternapyrone B, alternapyrone C, alternapyrone D, alternapyrone E, alternapyrone F</td>


<td>Parastagonospora nodorum SN15</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002429: 0-14296">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002429/index.html#r1c1" target="_new">BGC0002429</a></td>





<td class="cc-heat-cell" title="0.37 (id:0.42, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>


<td>0.11</td>
<td>Terpene, Polyketide</td>

<td>higginsianin B</td>


<td>Colletotrichum higginsianum IMI 349063</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002240: 96-11225">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002240/index.html#r1c1" target="_new">BGC0002240</a></td>





<td class="cc-heat-cell" title="0.37 (id:0.42, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>


<td>0.11</td>
<td>Polyketide</td>

<td>BAB, BAA</td>


<td>Metarhizium anisopliae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001398: 4999-64171">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001398/index.html#r1c1" target="_new">BGC0001398</a></td>





<td class="cc-heat-cell" title="0.36 (id:0.43, order:0.29, components:0.37)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>


<td>0.10</td>
<td>RiPP</td>

<td>phomopsin A, phomopsin B, phomopsin E</td>


<td>Phomopsis leptostromiformis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001122: 0-43663">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001122/index.html#r1c1" target="_new">BGC0001122</a></td>





<td class="cc-heat-cell" title="0.35 (id:0.41, order:N/A, components:0.30)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>


<td>0.10</td>
<td>NRP, Polyketide</td>

<td>dihydroisoflavipucine, isoflavipucine</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r3c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001775: 2711-20673">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001775/index.html#r1c1" target="_new">BGC0001775</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>Terpene</td>
<td>sespendole</td>
<td>Cordana terrestris</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001260: 0-15275">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001260/index.html#r1c1" target="_new">BGC0001260</a></td>


<td class="cc-heat-cell" title="0.41 (id:0.41, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
<td>0.41</td>
<td>Terpene</td>
<td>terpendole E</td>
<td>Tolypocladium album</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001624: 23-43456">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001624/index.html#r1c1" target="_new">BGC0001624</a></td>


<td class="cc-heat-cell" title="0.40 (id:0.44, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>
<td>0.40</td>
<td>Other</td>
<td>iso-A82775C</td>
<td>Pestalotiopsis fici</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002603: 0-17042">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002603/index.html#r1c1" target="_new">BGC0002603</a></td>


<td class="cc-heat-cell" title="0.38 (id:0.44, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
<td>0.38</td>
<td>Polyketide</td>
<td>chevalone E</td>
<td>Aspergillus versicolor</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002604: 0-17897">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002604/index.html#r1c1" target="_new">BGC0002604</a></td>


<td class="cc-heat-cell" title="0.38 (id:0.43, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
<td>0.38</td>
<td>Polyketide</td>
<td>sartorypyrone A</td>
<td>Aspergillus felis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001858: 715-19483">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001858/index.html#r1c1" target="_new">BGC0001858</a></td>


<td class="cc-heat-cell" title="0.37 (id:0.42, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>
<td>0.37</td>
<td>Polyketide</td>
<td>alternapyrone B, alternapyrone C, alternapyrone D, alternapyrone E, alternapyrone F</td>
<td>Parastagonospora nodorum SN15</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002429: 0-14296">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002429/index.html#r1c1" target="_new">BGC0002429</a></td>


<td class="cc-heat-cell" title="0.37 (id:0.42, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>
<td>0.37</td>
<td>Terpene, Polyketide</td>
<td>higginsianin B</td>
<td>Colletotrichum higginsianum IMI 349063</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002240: 96-11225">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002240/index.html#r1c1" target="_new">BGC0002240</a></td>


<td class="cc-heat-cell" title="0.37 (id:0.42, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>
<td>0.37</td>
<td>Polyketide</td>
<td>BAB, BAA</td>
<td>Metarhizium anisopliae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001398: 4999-64171">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001398/index.html#r1c1" target="_new">BGC0001398</a></td>


<td class="cc-heat-cell" title="0.36 (id:0.43, order:0.29, components:0.37)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>
<td>0.36</td>
<td>RiPP</td>
<td>phomopsin A, phomopsin B, phomopsin E</td>
<td>Phomopsis leptostromiformis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001122: 0-43663">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001122/index.html#r1c1" target="_new">BGC0001122</a></td>


<td class="cc-heat-cell" title="0.35 (id:0.41, order:N/A, components:0.30)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>
<td>0.35</td>
<td>NRP, Polyketide</td>
<td>dihydroisoflavipucine, isoflavipucine</td>
<td>Aspergillus terreus NIH2624</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r3c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-37"></div> <span class="help-tooltip" id="cb-general-help-37">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r3c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r3c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r3c1_1.svg">
NW_026250024 (709770-802898): Hypoxylon trugodes strain CBS 135444 unplaced g... (8% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c1_2.svg">
NW_020292110 (122545-258460): Aspergillus ibericus CBS 121593 unplaced genomi... (6% of genes show similarity), NRPS,T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c1_3.svg">
NW_024467870 (644043-736030): Aspergillus sydowii CBS 593.65 unplaced genomic... (4% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c1_4.svg">
NW_024468911 (2241962-2337395): Cryphonectria parasitica EP155 unplaced genom... (5% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c1_5.svg">
NW_024467535 (329778-421672): Aspergillus versicolor CBS 583.65 unplaced geno... (5% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c1_6.svg">
NW_009276927 (275764-367388): Verticillium dahliae VdLs.17 supercont1.8 genom... (13% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c1_7.svg">
NW_020291814 (6202-188758): Aspergillus costaricaensis CBS 115574 unplaced ge... (2% of genes show similarity), NRP-metallophore,NRPS,NRPS-like,T1PKS,fungal-RiPP-like,terpene
</option>

<option value = "svg/clusterblast_r3c1_8.svg">
NW_007360992 (708533-804064): Glarea lozoyensis ATCC 20868 chromosome Unknown... (5% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c1_9.svg">
NW_026249859 (494664-585701): Alternaria incomplexa strain BMP 0042 chromosom... (10% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c1_10.svg">
NW_020291494 (131618-251935): Aspergillus aculeatinus CBS 121060 unplaced gen... (4% of genes show similarity), fungal-RiPP-like
</option>

</select>
<button id= "clusterblast-r3c1-download">Download graphic</button>
</div>
<div id="clusterblast-r3c1-svg"></div>

</div>
</div>

<div class="body-details-section r3c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-38"></div> <span class="help-tooltip" id="cb-known-help-38">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r3c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-39"></div> <span class="help-tooltip" id="cb-sub-help-39">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r3c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-41"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-41">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c1-pfam-details-svg" data-filename="scaffold_3_r3c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r3c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">











</div>











</div>

</div>

</div>

<div class="page" id='r3c2' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_3 - Region 2 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-727"></div> <span class="help-tooltip" id="region_svg-help-727">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_3.region002.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r3c2-svg" data-filename="scaffold_3_r3c2.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 731,949 - 779,370 nt. (total: 47,422 nt)

<a class="cluster-rules-header" id="r3c2-rules-header" href="#r3c2">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r3c2-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r3c2-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-728"></div> <span class="help-tooltip" id="focus-panel-help-728">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r3c2">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r3c2-gene-table-details" data-name="r3c2-gene-table-details">
Gene overview
</div>



<div class="body-details-header r3c2-nrps_pks" data-name="r3c2-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r3c2-MIBiG-cluster-compare" data-name="r3c2-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r3c2-clusterblast" data-name="r3c2-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r3c2-knownclusterblast" data-name="r3c2-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r3c2-subclusterblast" data-name="r3c2-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r3c2-pfam-details" data-name="r3c2-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r3c2-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-48"></div> <span class="help-tooltip" id="gt-help-tooltip-help-48">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r3c2">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r3c2-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r3c2-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r3c2">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r3c2-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-42"></div> <span class="help-tooltip" id="nrps-domain-help-42">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c2-details-svg" data-filename="scaffold_3_r3c2_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r3c2-details-svg"></div>
</div>
</div>



<div class="body-details-section r3c2-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-43"></div> <span class="help-tooltip" id="MIBiG-help-43">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r3c2-selector">Analysis type:</label>
<select id="comparison-MIBiG-r3c2-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r3c2"></div>
<div id="comparison-MIBiG-r3c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001276: 1101-6575">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001276/index.html#r1c1" target="_new">BGC0001276</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.19</td>
<td>Polyketide</td>

<td>6-methylsalicyclic acid</td>


<td>Aspergillus terreus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001400: 30-15436">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001400/index.html#r1c1" target="_new">BGC0001400</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.19</td>
<td>Polyketide</td>

<td>citreoviridin</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001998: 0-14131">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001998/index.html#r1c1" target="_new">BGC0001998</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>Polyketide</td>

<td>aspernidgulene A1, aspernidgulene A2, aspernidgulene B1</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001244: 0-5358">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001244/index.html#r1c1" target="_new">BGC0001244</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>Polyketide</td>

<td>(-)-Mellein</td>


<td>Parastagonospora nodorum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001275: 1063-6614">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001275/index.html#r1c1" target="_new">BGC0001275</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>Polyketide</td>

<td>6-methylsalicyclic acid</td>


<td>Glarea lozoyensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001273: 0-5301">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001273/index.html#r1c1" target="_new">BGC0001273</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>Polyketide</td>

<td>asperlactone</td>


<td>Aspergillus ochraceus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002155: 78-16511">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002155/index.html#r1c1" target="_new">BGC0002155</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.44, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>Polyketide</td>

<td>nectriapyrone C, nectriapyrone D, nectriapyrone</td>


<td>Pyricularia oryzae 70-15</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001164: 0-8328">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001164/index.html#r1c1" target="_new">BGC0001164</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>Polyketide</td>

<td>1-heptadecene</td>


<td>Cyanothece sp. PCC 7822</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001254: 82-7456">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001254/index.html#r1c1" target="_new">BGC0001254</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>Polyketide</td>

<td>ACT-Toxin II</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002238: 216-10949">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002238/index.html#r1c1" target="_new">BGC0002238</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>Polyketide</td>

<td>2,4&#39;-dihydroxy-3&#39;-methoxypropiophenone</td>


<td>Aspergillus oryzae RIB40</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r3c2-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001276: 1101-6575">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001276/index.html#r1c1" target="_new">BGC0001276</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Polyketide</td>
<td>6-methylsalicyclic acid</td>
<td>Aspergillus terreus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001400: 30-15436">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001400/index.html#r1c1" target="_new">BGC0001400</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Polyketide</td>
<td>citreoviridin</td>
<td>Aspergillus terreus NIH2624</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001998: 0-14131">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001998/index.html#r1c1" target="_new">BGC0001998</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Polyketide</td>
<td>aspernidgulene A1, aspernidgulene A2, aspernidgulene B1</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001244: 0-5358">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001244/index.html#r1c1" target="_new">BGC0001244</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Polyketide</td>
<td>(-)-Mellein</td>
<td>Parastagonospora nodorum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001275: 1063-6614">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001275/index.html#r1c1" target="_new">BGC0001275</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Polyketide</td>
<td>6-methylsalicyclic acid</td>
<td>Glarea lozoyensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001273: 0-5301">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001273/index.html#r1c1" target="_new">BGC0001273</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Polyketide</td>
<td>asperlactone</td>
<td>Aspergillus ochraceus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002155: 78-16511">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002155/index.html#r1c1" target="_new">BGC0002155</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.44, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>Polyketide</td>
<td>nectriapyrone C, nectriapyrone D, nectriapyrone</td>
<td>Pyricularia oryzae 70-15</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001164: 0-8328">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001164/index.html#r1c1" target="_new">BGC0001164</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>Polyketide</td>
<td>1-heptadecene</td>
<td>Cyanothece sp. PCC 7822</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001254: 82-7456">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001254/index.html#r1c1" target="_new">BGC0001254</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>Polyketide</td>
<td>ACT-Toxin II</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002238: 216-10949">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002238/index.html#r1c1" target="_new">BGC0002238</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>Polyketide</td>
<td>2,4&#39;-dihydroxy-3&#39;-methoxypropiophenone</td>
<td>Aspergillus oryzae RIB40</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r3c2-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-44"></div> <span class="help-tooltip" id="cb-general-help-44">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r3c2-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r3c2_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r3c2_1.svg">
NT_165978 (203661-313028): Chaetomium globosum CBS 148.51 scaffold 3 genomic ... (12% of genes show similarity), NI-siderophore,NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r3c2_2.svg">
NW_024469024 (26525-267638): Aspergillus fijiensis CBS 313.89 unplaced genomi... (3% of genes show similarity), NRPS,T1PKS,fungal-RiPP-like,other
</option>

<option value = "svg/clusterblast_r3c2_3.svg">
NC_026474 (11149537-11254849): Fusarium graminearum PH-1 chromosome 1, whole ... (6% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r3c2_4.svg">
NC_064680 (3715185-3853805): Colletotrichum lupini chromosome 7, complete seq... (4% of genes show similarity), T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c2_5.svg">
NW_025764716 (588555-656757): Emericellopsis atlantica unplaced genomic scaff... (6% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r3c2_6.svg">
NW_020292110 (41804-109723): Aspergillus ibericus CBS 121593 unplaced genomic... (9% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r3c2_7.svg">
NW_025953267 (201738-434522): Annulohypoxylon maeteangense strain CBS 123835 ... (2% of genes show similarity), T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c2_8.svg">
NW_020291498 (165271-359753): Aspergillus aculeatinus CBS 121060 unplaced gen... (2% of genes show similarity), NRPS,T1PKS,fungal-RiPP-like,other
</option>

<option value = "svg/clusterblast_r3c2_9.svg">
NW_022985451 (998305-1066533): Mytilinidion resinicola strain CBS 304.34 unpl... (8% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r3c2_10.svg">
NW_017263599 (247828-370093): Mollisia scopiformis unplaced genomic scaffold ... (4% of genes show similarity), T1PKS,fungal-RiPP-like
</option>

</select>
<button id= "clusterblast-r3c2-download">Download graphic</button>
</div>
<div id="clusterblast-r3c2-svg"></div>

</div>
</div>

<div class="body-details-section r3c2-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-45"></div> <span class="help-tooltip" id="cb-known-help-45">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r3c2-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-46"></div> <span class="help-tooltip" id="cb-sub-help-46">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r3c2-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-49"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-49">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c2-pfam-details-svg" data-filename="scaffold_3_r3c2_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r3c2-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r3c2-nrps_pks_monomers" data-name="r3c2-nrps_pks_monomers">
NRPS/PKS substrates
</div>




</div>










<div class="sidepanel-details-section r3c2-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-47"></div> <span class="help-tooltip" id="nrps-monomers-help-47">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">FUN_000976</span></strong>: pk
<div class="collapser collapser-target-FUN_000976 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (532..849)</strong>: pk
<div class="collapser collapser-target-FUN_000976 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA, Methylmalonyl-CoA
<div class="collapser collapser-target-FUN_000976-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 66.7%</dd>
<dd>Methylmalonyl-CoA: 66.7%</dd>
<dd>Isobutyryl-CoA: 58.3%</dd>
</dl>

</div><br>

minowa:  Methoxymalonyl-CoA
<div class="collapser collapser-target-FUN_000976-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methoxymalonyl-CoA: 67.0</dt>

<dd></dd><dt>Methylmalonyl-CoA: 64.2</dt>

<dd></dd><dt>Malonyl-CoA: 45.4</dt>

<dd></dd><dt>Isobutyryl-CoA: 41.8</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 17.7</dt>

<dd></dd><dt>Propionyl-CoA: 16.4</dt>

<dd></dd><dt>inactive: 15.5</dt>

<dd></dd><dt>trans-1,2-CPDA: 14.2</dt>

<dd></dd><dt>CHC-CoA: 12.3</dt>

<dd></dd><dt>fatty_acid: 10.8</dt>

<dd></dd><dt>Benzoyl-CoA: 0.0</dt>

<dd></dd><dt>Acetyl-CoA: 0.0</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

<dd></dd><dt>Ethylmalonyl-CoA: -4.1</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>








</div>
<br>

</dl>
</div>
</div>




</div>

</div>

</div>

<div class="page" id='r3c3' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_3 - Region 3 - fungal-RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-729"></div> <span class="help-tooltip" id="region_svg-help-729">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_3.region003.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r3c3-svg" data-filename="scaffold_3_r3c3.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1,003,045 - 1,064,010 nt. (total: 60,966 nt)

<a class="cluster-rules-header" id="r3c3-rules-header" href="#r3c3">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r3c3-rules">fungal-RiPP-like: DUF3328</div>

<div class="region-svg-container">
<div id='r3c3-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-730"></div> <span class="help-tooltip" id="focus-panel-help-730">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r3c3">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r3c3-gene-table-details" data-name="r3c3-gene-table-details">
Gene overview
</div>



<div class="body-details-header r3c3-MIBiG-cluster-compare" data-name="r3c3-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r3c3-clusterblast" data-name="r3c3-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r3c3-knownclusterblast" data-name="r3c3-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r3c3-subclusterblast" data-name="r3c3-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r3c3-pfam-details" data-name="r3c3-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r3c3-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-54"></div> <span class="help-tooltip" id="gt-help-tooltip-help-54">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r3c3">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r3c3-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r3c3-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r3c3">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r3c3-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-50"></div> <span class="help-tooltip" id="MIBiG-help-50">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r3c3-selector">Analysis type:</label>
<select id="comparison-MIBiG-r3c3-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r3c3"></div>
<div id="comparison-MIBiG-r3c3-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">fungal-RiPP-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000551: 0-6728">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000551/index.html#r1c1" target="_new">BGC0000551</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.15</td>
<td>RiPP</td>

<td>SapB</td>


<td>Streptomyces coelicolor A3(2)</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000521: 0-12132">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000521/index.html#r1c1" target="_new">BGC0000521</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.15</td>
<td>RiPP</td>

<td>lacticin 481</td>


<td>Lactococcus lactis subsp. lactis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002447: 0-8196">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002447/index.html#r1c1" target="_new">BGC0002447</a></td>





<td class="cc-heat-cell" title="0.45 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>


<td>0.15</td>
<td>RiPP</td>

<td>cerecyclin</td>


<td>Bacillus cereus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000539: 155-11740">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000539/index.html#r1c1" target="_new">BGC0000539</a></td>





<td class="cc-heat-cell" title="0.45 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>


<td>0.15</td>
<td>RiPP</td>

<td>nukacin ISK-1</td>


<td>Staphylococcus warneri</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000519: 0-6400">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000519/index.html#r1c1" target="_new">BGC0000519</a></td>





<td class="cc-heat-cell" title="0.45 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>


<td>0.15</td>
<td>RiPP</td>

<td>labyrinthopeptin A2, labyrinthopeptin A1, labyrinthopeptin A3</td>


<td>Actinomadura namibiensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002323: 0-9758">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002323/index.html#r1c1" target="_new">BGC0002323</a></td>





<td class="cc-heat-cell" title="0.42 (id:0.44, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>


<td>0.13</td>
<td>RiPP</td>

<td>nisin J</td>


<td>Staphylococcus capitis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002663: 0-16274">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002663/index.html#r1c1" target="_new">BGC0002663</a></td>





<td class="cc-heat-cell" title="0.42 (id:0.41, order:N/A, components:0.43)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>


<td>0.13</td>
<td>RiPP</td>

<td>amyloliquecidin GF610</td>


<td>Bacillus amyloliquefaciens UMAF6639</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000552: 1388-11054">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000552/index.html#r1c1" target="_new">BGC0000552</a></td>





<td class="cc-heat-cell" title="0.42 (id:0.44, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>


<td>0.13</td>
<td>RiPP</td>

<td>SmbA, SmbB</td>


<td>Streptococcus mutans</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002490: 0-5965">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002490/index.html#r1c1" target="_new">BGC0002490</a></td>





<td class="cc-heat-cell" title="0.41 (id:0.41, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>


<td>0.13</td>
<td>Other</td>

<td>yersinopine</td>


<td>Yersinia pestis CO92</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000498: 1-14995">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000498/index.html#r1c1" target="_new">BGC0000498</a></td>





<td class="cc-heat-cell" title="0.41 (id:0.42, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>


<td>0.13</td>
<td>RiPP</td>

<td>bovicin HJ50</td>


<td>Streptococcus equinus</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r3c3-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000551: 0-6728">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000551/index.html#r1c1" target="_new">BGC0000551</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>RiPP</td>
<td>SapB</td>
<td>Streptomyces coelicolor A3(2)</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000521: 0-12132">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000521/index.html#r1c1" target="_new">BGC0000521</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>RiPP</td>
<td>lacticin 481</td>
<td>Lactococcus lactis subsp. lactis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002447: 0-8196">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002447/index.html#r1c1" target="_new">BGC0002447</a></td>


<td class="cc-heat-cell" title="0.45 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
<td>0.45</td>
<td>RiPP</td>
<td>cerecyclin</td>
<td>Bacillus cereus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000539: 155-11740">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000539/index.html#r1c1" target="_new">BGC0000539</a></td>


<td class="cc-heat-cell" title="0.45 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
<td>0.45</td>
<td>RiPP</td>
<td>nukacin ISK-1</td>
<td>Staphylococcus warneri</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000519: 0-6400">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000519/index.html#r1c1" target="_new">BGC0000519</a></td>


<td class="cc-heat-cell" title="0.45 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
<td>0.45</td>
<td>RiPP</td>
<td>labyrinthopeptin A2, labyrinthopeptin A1, labyrinthopeptin A3</td>
<td>Actinomadura namibiensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002323: 0-9758">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002323/index.html#r1c1" target="_new">BGC0002323</a></td>


<td class="cc-heat-cell" title="0.42 (id:0.44, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
<td>0.42</td>
<td>RiPP</td>
<td>nisin J</td>
<td>Staphylococcus capitis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002663: 0-16274">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002663/index.html#r1c1" target="_new">BGC0002663</a></td>


<td class="cc-heat-cell" title="0.42 (id:0.41, order:N/A, components:0.43)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
<td>0.42</td>
<td>RiPP</td>
<td>amyloliquecidin GF610</td>
<td>Bacillus amyloliquefaciens UMAF6639</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000552: 1388-11054">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000552/index.html#r1c1" target="_new">BGC0000552</a></td>


<td class="cc-heat-cell" title="0.42 (id:0.44, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
<td>0.42</td>
<td>RiPP</td>
<td>SmbA, SmbB</td>
<td>Streptococcus mutans</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002490: 0-5965">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002490/index.html#r1c1" target="_new">BGC0002490</a></td>


<td class="cc-heat-cell" title="0.41 (id:0.41, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
<td>0.41</td>
<td>Other</td>
<td>yersinopine</td>
<td>Yersinia pestis CO92</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000498: 1-14995">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000498/index.html#r1c1" target="_new">BGC0000498</a></td>


<td class="cc-heat-cell" title="0.41 (id:0.42, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
<td>0.41</td>
<td>RiPP</td>
<td>bovicin HJ50</td>
<td>Streptococcus equinus</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r3c3-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-51"></div> <span class="help-tooltip" id="cb-general-help-51">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r3c3-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r3c3_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r3c3_1.svg">
NW_025953249 (3016690-3115240): Annulohypoxylon maeteangense strain CBS 12383... (11% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c3_2.svg">
NW_020292160 (69830-164319): Aspergillus ibericus CBS 121593 unplaced genomic... (5% of genes show similarity), NRPS-like,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c3_3.svg">
NW_007360991 (631990-722959): Glarea lozoyensis ATCC 20868 chromosome Unknown... (8% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c3_4.svg">
NW_020291606 (579604-818783): Aspergillus piperis CBS 112811 unplaced genomic... (2% of genes show similarity), T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c3_5.svg">
NW_020291510 (214662-351091): Aspergillus aculeatinus CBS 121060 unplaced gen... (5% of genes show similarity), NRPS-like,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c3_6.svg">
NC_072354 (1379896-1470826): Talaromyces marneffei chromosome 7, complete seq... (8% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c3_7.svg">
NW_024468980 (670954-805564): Aspergillus fijiensis CBS 313.89 unplaced genom... (5% of genes show similarity), NRPS-like,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c3_8.svg">
NW_020167520 (2107982-2198732): Pseudogymnoascus destructans isolate 20631-21... (8% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r3c3_9.svg">
NW_006917091 (3402044-3489915): Pestalotiopsis fici W106-1 unplaced genomic s... (11% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r3c3_10.svg">
NC_038013 (719067-800490): Fusarium venenatum strain A3/5 genome assembly, ch... (6% of genes show similarity), isocyanide-nrp,terpene
</option>

</select>
<button id= "clusterblast-r3c3-download">Download graphic</button>
</div>
<div id="clusterblast-r3c3-svg"></div>

</div>
</div>

<div class="body-details-section r3c3-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-52"></div> <span class="help-tooltip" id="cb-known-help-52">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r3c3-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-53"></div> <span class="help-tooltip" id="cb-sub-help-53">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r3c3-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-55"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-55">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c3-pfam-details-svg" data-filename="scaffold_3_r3c3_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r3c3-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">











</div>











</div>

</div>

</div>





<div class="page" id='r4c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_4 - Region 1 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-731"></div> <span class="help-tooltip" id="region_svg-help-731">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_4.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r4c1-svg" data-filename="scaffold_4_r4c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 222,420 - 271,222 nt. (total: 48,803 nt)

<a class="cluster-rules-header" id="r4c1-rules-header" href="#r4c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r4c1-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r4c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-732"></div> <span class="help-tooltip" id="focus-panel-help-732">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r4c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r4c1-gene-table-details" data-name="r4c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r4c1-nrps_pks" data-name="r4c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r4c1-MIBiG-cluster-compare" data-name="r4c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r4c1-clusterblast" data-name="r4c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r4c1-knownclusterblast" data-name="r4c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r4c1-subclusterblast" data-name="r4c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r4c1-nrps-pks-bubbles" data-name="r4c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r4c1-pfam-details" data-name="r4c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r4c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-64"></div> <span class="help-tooltip" id="gt-help-tooltip-help-64">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r4c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r4c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r4c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r4c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r4c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-56"></div> <span class="help-tooltip" id="nrps-domain-help-56">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c1-details-svg" data-filename="scaffold_4_r4c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r4c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r4c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-57"></div> <span class="help-tooltip" id="MIBiG-help-57">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r4c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r4c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r4c1"></div>
<div id="comparison-MIBiG-r4c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002525: 1000-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002525/index.html#r1c1" target="_new">BGC0002525</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.32</td>
<td>Polyketide</td>

<td>fusarubin, 1233A, 1233B, NG-391, lucilactaene</td>


<td>Fusarium sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002191: 260-8785">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002191/index.html#r1c1" target="_new">BGC0002191</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.31</td>
<td>Polyketide</td>

<td>prolipyrone B, gibepyrone D</td>


<td>Fusarium graminearum PH-1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002155: 78-16511">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002155/index.html#r1c1" target="_new">BGC0002155</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.43, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.30</td>
<td>Polyketide</td>

<td>nectriapyrone C, nectriapyrone D, nectriapyrone</td>


<td>Pyricularia oryzae 70-15</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001252: 0-7958">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001252/index.html#r1c1" target="_new">BGC0001252</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.42, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.29</td>
<td>Polyketide</td>

<td>UNII-YC2Q1O94PT</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002194: 147-15389">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002194/index.html#r1c1" target="_new">BGC0002194</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.28</td>
<td>Polyketide</td>

<td>epipyrone A</td>


<td>Epicoccum nigrum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000146: 754-20637">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000146/index.html#r1c1" target="_new">BGC0000146</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.42, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.27</td>
<td>Polyketide</td>

<td>solanapyrone D</td>


<td>Alternaria solani</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002515: 0-30087">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002515/index.html#r1c1" target="_new">BGC0002515</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.42, order:N/A, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.27</td>
<td>Polyketide</td>

<td>solanapyrone A</td>


<td>Ascochyta rabiei</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001899: 0-54810">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001899/index.html#r1c1" target="_new">BGC0001899</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.26</td>
<td>Polyketide</td>

<td>abscisic acid</td>


<td>Leptosphaeria maculans JN3</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002224: 685-37748">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002224/index.html#r1c1" target="_new">BGC0002224</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.26</td>
<td>Polyketide</td>

<td>zopfiellin</td>


<td>Diffractella curvata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002222: 888-32276">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002222/index.html#r1c1" target="_new">BGC0002222</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.25</td>
<td>Polyketide</td>

<td>zopfiellin</td>


<td>Diffractella curvata</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r4c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002525: 1000-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002525/index.html#r1c1" target="_new">BGC0002525</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Polyketide</td>
<td>fusarubin, 1233A, 1233B, NG-391, lucilactaene</td>
<td>Fusarium sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002191: 260-8785">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002191/index.html#r1c1" target="_new">BGC0002191</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Polyketide</td>
<td>prolipyrone B, gibepyrone D</td>
<td>Fusarium graminearum PH-1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002155: 78-16511">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002155/index.html#r1c1" target="_new">BGC0002155</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.43, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>Polyketide</td>
<td>nectriapyrone C, nectriapyrone D, nectriapyrone</td>
<td>Pyricularia oryzae 70-15</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001252: 0-7958">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001252/index.html#r1c1" target="_new">BGC0001252</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.42, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>Polyketide</td>
<td>UNII-YC2Q1O94PT</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002194: 147-15389">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002194/index.html#r1c1" target="_new">BGC0002194</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>Polyketide</td>
<td>epipyrone A</td>
<td>Epicoccum nigrum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000146: 754-20637">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000146/index.html#r1c1" target="_new">BGC0000146</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.42, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>Polyketide</td>
<td>solanapyrone D</td>
<td>Alternaria solani</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002515: 0-30087">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002515/index.html#r1c1" target="_new">BGC0002515</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.42, order:N/A, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>Polyketide</td>
<td>solanapyrone A</td>
<td>Ascochyta rabiei</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001899: 0-54810">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001899/index.html#r1c1" target="_new">BGC0001899</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>Polyketide</td>
<td>abscisic acid</td>
<td>Leptosphaeria maculans JN3</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002224: 685-37748">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002224/index.html#r1c1" target="_new">BGC0002224</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Polyketide</td>
<td>zopfiellin</td>
<td>Diffractella curvata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002222: 888-32276">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002222/index.html#r1c1" target="_new">BGC0002222</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Polyketide</td>
<td>zopfiellin</td>
<td>Diffractella curvata</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r4c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-58"></div> <span class="help-tooltip" id="cb-general-help-58">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r4c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r4c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r4c1_1.svg">
NW_019154044 (23579-139973): Pochonia chlamydosporia 170 chromosome Unknown P... (7% of genes show similarity), NRPS,T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r4c1_2.svg">
NW_006917099 (1833549-1902213): Pestalotiopsis fici W106-1 unplaced genomic s... (12% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r4c1_3.svg">
NW_025953267 (201738-434522): Annulohypoxylon maeteangense strain CBS 123835 ... (3% of genes show similarity), T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r4c1_4.svg">
NW_025953058 (130642-226995): Daldinia caldariorum strain CBS 122874 unplaced... (26% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r4c1_5.svg">
NW_022987215 (251333-319784): Trematosphaeria pertusa strain CBS 122368 unpla... (12% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r4c1_6.svg">
NW_006917097 (1595225-1663737): Pestalotiopsis fici W106-1 unplaced genomic s... (16% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r4c1_7.svg">
NW_025953816 (281521-436136): Annulohypoxylon truncatum strain CBS 140777 unp... (3% of genes show similarity), T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r4c1_8.svg">
NW_025408668 (1200492-1268825): Hirsutella rhossiliensis strain HR02 chromoso... (25% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r4c1_9.svg">
NW_026249867 (161946-230009): Alternaria incomplexa strain BMP 0042 chromosom... (8% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r4c1_10.svg">
NW_026055700 (31065-99128): Alternaria ventricosa strain BMP 2768 chromosome ... (8% of genes show similarity), T1PKS
</option>

</select>
<button id= "clusterblast-r4c1-download">Download graphic</button>
</div>
<div id="clusterblast-r4c1-svg"></div>

</div>
</div>

<div class="body-details-section r4c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-59"></div> <span class="help-tooltip" id="cb-known-help-59">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r4c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-60"></div> <span class="help-tooltip" id="cb-sub-help-60">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r4c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-63"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-63">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c1-domain-bubble-svg" data-filename="scaffold_4_r4c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r4c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (222419 - 271222): single T1PKS
</option>


</select>

<div class="details-svg" id="r4c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r4c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r4c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-65"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-65">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c1-pfam-details-svg" data-filename="scaffold_4_r4c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r4c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r4c1-nrps_pks_products" data-name="r4c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r4c1-nrps_pks_monomers" data-name="r4c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r4c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-61"></div> <span class="help-tooltip" id="nrps-structures-help-61">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 222419 - 271222:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r4c1-1" class="smiles-canvas" data-smiles="C([*])C(-O)(C)"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Me-pk)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-pk" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-pk*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r4c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)(C)"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r4c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-62"></div> <span class="help-tooltip" id="nrps-monomers-help-62">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">FUN_001157</span></strong>: pk
<div class="collapser collapser-target-FUN_001157 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (435..754)</strong>: pk
<div class="collapser collapser-target-FUN_001157 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-FUN_001157-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 75.0%</dd>
<dd>Ethylmalonyl-CoA: 58.3%</dd>
<dd>Methylmalonyl-CoA: 58.3%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-FUN_001157-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 87.5</dt>

<dd></dd><dt>Malonyl-CoA: 84.0</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 81.8</dt>

<dd></dd><dt>Isobutyryl-CoA: 49.9</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 49.0</dt>

<dd></dd><dt>Propionyl-CoA: 40.0</dt>

<dd></dd><dt>trans-1,2-CPDA: 39.6</dt>

<dd></dd><dt>CHC-CoA: 29.3</dt>

<dd></dd><dt>fatty_acid: 28.8</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 26.5</dt>

<dd></dd><dt>inactive: 18.7</dt>

<dd></dd><dt>Benzoyl-CoA: 15.7</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 10.4</dt>

<dd></dd><dt>Acetyl-CoA: 7.1</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>












</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r4c2' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_4 - Region 2 - NRPS-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-733"></div> <span class="help-tooltip" id="region_svg-help-733">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_4.region002.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r4c2-svg" data-filename="scaffold_4_r4c2.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 703,038 - 746,468 nt. (total: 43,431 nt)

<a class="cluster-rules-header" id="r4c2-rules-header" href="#r4c2">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r4c2-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r4c2-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-734"></div> <span class="help-tooltip" id="focus-panel-help-734">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r4c2">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r4c2-gene-table-details" data-name="r4c2-gene-table-details">
Gene overview
</div>



<div class="body-details-header r4c2-nrps_pks" data-name="r4c2-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r4c2-MIBiG-cluster-compare" data-name="r4c2-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r4c2-clusterblast" data-name="r4c2-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r4c2-knownclusterblast" data-name="r4c2-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r4c2-subclusterblast" data-name="r4c2-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r4c2-nrps-pks-bubbles" data-name="r4c2-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r4c2-pfam-details" data-name="r4c2-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r4c2-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-74"></div> <span class="help-tooltip" id="gt-help-tooltip-help-74">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r4c2">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r4c2-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r4c2-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r4c2">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r4c2-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-66"></div> <span class="help-tooltip" id="nrps-domain-help-66">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c2-details-svg" data-filename="scaffold_4_r4c2_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r4c2-details-svg"></div>
</div>
</div>



<div class="body-details-section r4c2-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-67"></div> <span class="help-tooltip" id="MIBiG-help-67">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r4c2-selector">Analysis type:</label>
<select id="comparison-MIBiG-r4c2-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r4c2"></div>
<div id="comparison-MIBiG-r4c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002212: 0-24132">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002212/index.html#r1c1" target="_new">BGC0002212</a></td>





<td class="cc-heat-cell" title="0.39 (id:0.42, order:N/A, components:0.36)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>


<td>0.11</td>
<td>Polyketide</td>

<td>HEx-pks15 polyketide</td>


<td>Moniliophthora roreri MCA 2997</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002582: 0-7207">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002582/index.html#r1c1" target="_new">BGC0002582</a></td>





<td class="cc-heat-cell" title="0.32 (id:0.41, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.32), rgb(0, 0, 0, 0.32) 32%, white 32%)"></td>


<td>0.08</td>
<td>NRP</td>

<td>guanipiperazine A, guanipiperazine B</td>


<td>Streptomyces chrestomyceticus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001600: 0-23599">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001600/index.html#r1c1" target="_new">BGC0001600</a></td>





<td class="cc-heat-cell" title="0.29 (id:0.41, order:N/A, components:0.21)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>


<td>0.07</td>
<td>Polyketide</td>

<td>fusarielin H</td>


<td>Fusarium graminearum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001722: 0-18441">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001722/index.html#r1c1" target="_new">BGC0001722</a></td>





<td class="cc-heat-cell" title="0.29 (id:0.43, order:N/A, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>


<td>0.07</td>
<td>Polyketide</td>

<td>aspernidine A</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001976: 11125-46899">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001976/index.html#r1c1" target="_new">BGC0001976</a></td>





<td class="cc-heat-cell" title="0.27 (id:0.43, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>


<td>0.06</td>
<td>Terpene</td>

<td>eupenifeldin</td>


<td>Phoma sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001923: 495-32413">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001923/index.html#r1c1" target="_new">BGC0001923</a></td>





<td class="cc-heat-cell" title="0.26 (id:0.41, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>


<td>0.06</td>
<td>Terpene, Polyketide</td>

<td>ascochlorin</td>


<td>Acremonium egyptiacum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002209: 560-36987">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002209/index.html#r1c1" target="_new">BGC0002209</a></td>





<td class="cc-heat-cell" title="0.26 (id:0.41, order:N/A, components:0.16)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>


<td>0.06</td>
<td>Polyketide</td>

<td>ustethylin A</td>


<td>Aspergillus ustus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001827: 0-43539">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001827/index.html#r1c1" target="_new">BGC0001827</a></td>





<td class="cc-heat-cell" title="0.25 (id:0.42, order:N/A, components:0.15)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>


<td>0.06</td>
<td>Polyketide</td>

<td>xenovulene A</td>


<td>Sarocladium sp. &#39;schorii&#39;</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000154: 266-31301">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000154/index.html#r1c1" target="_new">BGC0000154</a></td>





<td class="cc-heat-cell" title="0.25 (id:0.42, order:N/A, components:0.14)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>


<td>0.06</td>
<td>Polyketide</td>

<td>stipitatic acid</td>


<td>Talaromyces stipitatus ATCC 10500</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001190: 82-52910">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001190/index.html#r1c1" target="_new">BGC0001190</a></td>





<td class="cc-heat-cell" title="0.24 (id:0.42, order:0.33, components:0.11)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.24), rgb(0, 0, 0, 0.24) 24%, white 24%)"></td>


<td>0.06</td>
<td>Polyketide</td>

<td>fusaric acid</td>


<td>Fusarium verticillioides 7600</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r4c2-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002212: 0-24132">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002212/index.html#r1c1" target="_new">BGC0002212</a></td>


<td class="cc-heat-cell" title="0.39 (id:0.42, order:N/A, components:0.36)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
<td>0.39</td>
<td>Polyketide</td>
<td>HEx-pks15 polyketide</td>
<td>Moniliophthora roreri MCA 2997</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002582: 0-7207">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002582/index.html#r1c1" target="_new">BGC0002582</a></td>


<td class="cc-heat-cell" title="0.32 (id:0.41, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.32), rgb(0, 0, 0, 0.32) 32%, white 32%)"></td>
<td>0.32</td>
<td>NRP</td>
<td>guanipiperazine A, guanipiperazine B</td>
<td>Streptomyces chrestomyceticus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001600: 0-23599">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001600/index.html#r1c1" target="_new">BGC0001600</a></td>


<td class="cc-heat-cell" title="0.29 (id:0.41, order:N/A, components:0.21)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>
<td>0.29</td>
<td>Polyketide</td>
<td>fusarielin H</td>
<td>Fusarium graminearum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001722: 0-18441">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001722/index.html#r1c1" target="_new">BGC0001722</a></td>


<td class="cc-heat-cell" title="0.29 (id:0.43, order:N/A, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>
<td>0.29</td>
<td>Polyketide</td>
<td>aspernidine A</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001976: 11125-46899">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001976/index.html#r1c1" target="_new">BGC0001976</a></td>


<td class="cc-heat-cell" title="0.27 (id:0.43, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>
<td>0.27</td>
<td>Terpene</td>
<td>eupenifeldin</td>
<td>Phoma sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001923: 495-32413">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001923/index.html#r1c1" target="_new">BGC0001923</a></td>


<td class="cc-heat-cell" title="0.26 (id:0.41, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>
<td>0.26</td>
<td>Terpene, Polyketide</td>
<td>ascochlorin</td>
<td>Acremonium egyptiacum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002209: 560-36987">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002209/index.html#r1c1" target="_new">BGC0002209</a></td>


<td class="cc-heat-cell" title="0.26 (id:0.41, order:N/A, components:0.16)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>
<td>0.26</td>
<td>Polyketide</td>
<td>ustethylin A</td>
<td>Aspergillus ustus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001827: 0-43539">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001827/index.html#r1c1" target="_new">BGC0001827</a></td>


<td class="cc-heat-cell" title="0.25 (id:0.42, order:N/A, components:0.15)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>
<td>0.25</td>
<td>Polyketide</td>
<td>xenovulene A</td>
<td>Sarocladium sp. &#39;schorii&#39;</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000154: 266-31301">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000154/index.html#r1c1" target="_new">BGC0000154</a></td>


<td class="cc-heat-cell" title="0.25 (id:0.42, order:N/A, components:0.14)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>
<td>0.25</td>
<td>Polyketide</td>
<td>stipitatic acid</td>
<td>Talaromyces stipitatus ATCC 10500</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001190: 82-52910">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001190/index.html#r1c1" target="_new">BGC0001190</a></td>


<td class="cc-heat-cell" title="0.24 (id:0.42, order:0.33, components:0.11)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.24), rgb(0, 0, 0, 0.24) 24%, white 24%)"></td>
<td>0.24</td>
<td>Polyketide</td>
<td>fusaric acid</td>
<td>Fusarium verticillioides 7600</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r4c2-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-68"></div> <span class="help-tooltip" id="cb-general-help-68">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r4c2-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r4c2_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r4c2_1.svg">
NW_006917091 (5354931-5428146): Pestalotiopsis fici W106-1 unplaced genomic s... (11% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r4c2_2.svg">
NW_022987212 (1353304-1416569): Trematosphaeria pertusa strain CBS 122368 unp... (5% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r4c2_3.svg">
NW_017263592 (1034310-1097481): Mollisia scopiformis unplaced genomic scaffol... (7% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r4c2_4.svg">
NW_022983483 (985826-1049150): Aaosphaeria arxii CBS 175.79 unplaced genomic ... (10% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r4c2_5.svg">
NW_026250019 (2151060-2242030): Hypoxylon trugodes strain CBS 135444 unplaced... (5% of genes show similarity), NRPS-like,terpene
</option>

<option value = "svg/clusterblast_r4c2_6.svg">
NT_165982 (1135729-1226820): Chaetomium globosum CBS 148.51 scaffold 7 genomi... (6% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r4c2_7.svg">
NW_020291594 (1747021-1820836): Aspergillus piperis CBS 112811 unplaced genom... (8% of genes show similarity), T1PKS,terpene
</option>

<option value = "svg/clusterblast_r4c2_8.svg">
NC_054849 (2647776-2721226): Aspergillus luchuensis IFO 4308 DNA, chromosome ... (7% of genes show similarity), T1PKS,terpene
</option>

<option value = "svg/clusterblast_r4c2_9.svg">
NW_024467836 (90960-158318): Aspergillus wentii DTO 134E9 unplaced genomic sc... (6% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r4c2_10.svg">
NW_020291813 (699136-760367): Aspergillus costaricaensis CBS 115574 unplaced ... (9% of genes show similarity), T3PKS
</option>

</select>
<button id= "clusterblast-r4c2-download">Download graphic</button>
</div>
<div id="clusterblast-r4c2-svg"></div>

</div>
</div>

<div class="body-details-section r4c2-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-69"></div> <span class="help-tooltip" id="cb-known-help-69">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r4c2-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-70"></div> <span class="help-tooltip" id="cb-sub-help-70">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r4c2-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-73"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-73">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c2-domain-bubble-svg" data-filename="scaffold_4_r4c2_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r4c2-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC2">
Candidate 2 (703037 - 746468): single NRPS-like
</option>


</select>

<div class="details-svg" id="r4c2-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r4c2-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r4c2-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-75"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-75">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c2-pfam-details-svg" data-filename="scaffold_4_r4c2_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r4c2-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r4c2-nrps_pks_products" data-name="r4c2-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r4c2-nrps_pks_monomers" data-name="r4c2-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r4c2-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-71"></div> <span class="help-tooltip" id="nrps-structures-help-71">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 2, location 703037 - 746468:
<div class="collapser collapser-target-CC2 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r4c2-2" class="smiles-canvas" data-smiles="NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r4c2-2-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r4c2-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-72"></div> <span class="help-tooltip" id="nrps-monomers-help-72">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">FUN_001289</span></strong>: X
<div class="collapser collapser-target-FUN_001289 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (143..407)</strong>: X
<div class="collapser collapser-target-FUN_001289 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  (unknown)
<div class="collapser collapser-target-FUN_001289-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>  <strong>NOTE: uncertain match</strong><br>
<dl>   <dt>Predicted physicochemical class:</dt>
<dd>N/A</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">--QFVSP--K</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>

<dt>Stachelhaus code match:</dt>
<dd>0% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>





<div class="page" id='r5c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_5 - Region 1 - fungal-RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-735"></div> <span class="help-tooltip" id="region_svg-help-735">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_5.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r5c1-svg" data-filename="scaffold_5_r5c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 249,984 - 312,664 nt. (total: 62,681 nt)

<a class="cluster-rules-header" id="r5c1-rules-header" href="#r5c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r5c1-rules">fungal-RiPP-like: DUF3328</div>

<div class="region-svg-container">
<div id='r5c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-736"></div> <span class="help-tooltip" id="focus-panel-help-736">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r5c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r5c1-gene-table-details" data-name="r5c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r5c1-MIBiG-cluster-compare" data-name="r5c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r5c1-clusterblast" data-name="r5c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r5c1-knownclusterblast" data-name="r5c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r5c1-subclusterblast" data-name="r5c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r5c1-pfam-details" data-name="r5c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r5c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-80"></div> <span class="help-tooltip" id="gt-help-tooltip-help-80">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r5c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r5c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r5c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r5c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r5c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-76"></div> <span class="help-tooltip" id="MIBiG-help-76">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r5c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r5c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r5c1"></div>
<div id="comparison-MIBiG-r5c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">fungal-RiPP-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001306: 0-6409">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001306/index.html#r1c1" target="_new">BGC0001306</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.33</td>
<td>RiPP</td>

<td>asperipin 2a</td>


<td>Aspergillus flavus NRRL3357</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000231: 189-5010">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000231/index.html#r1c1" target="_new">BGC0000231</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.43, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>Polyketide</td>

<td>griseusin A, griseusin B</td>


<td>Streptomyces griseus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000627: 0-28295">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000627/index.html#r1c1" target="_new">BGC0000627</a></td>





<td class="cc-heat-cell" title="0.45 (id:0.48, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>


<td>0.15</td>
<td>RiPP</td>

<td>ustiloxin B</td>


<td>Aspergillus flavus NRRL3357</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002240: 96-11225">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002240/index.html#r1c1" target="_new">BGC0002240</a></td>





<td class="cc-heat-cell" title="0.43 (id:0.44, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>


<td>0.14</td>
<td>Polyketide</td>

<td>BAB, BAA</td>


<td>Metarhizium anisopliae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002238: 216-10949">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002238/index.html#r1c1" target="_new">BGC0002238</a></td>





<td class="cc-heat-cell" title="0.43 (id:0.44, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>


<td>0.14</td>
<td>Polyketide</td>

<td>2,4&#39;-dihydroxy-3&#39;-methoxypropiophenone</td>


<td>Aspergillus oryzae RIB40</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002525: 1000-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002525/index.html#r1c1" target="_new">BGC0002525</a></td>





<td class="cc-heat-cell" title="0.42 (id:0.45, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>


<td>0.13</td>
<td>Polyketide</td>

<td>fusarubin, 1233A, 1233B, NG-391, lucilactaene</td>


<td>Fusarium sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001398: 4999-64171">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001398/index.html#r1c1" target="_new">BGC0001398</a></td>





<td class="cc-heat-cell" title="0.42 (id:0.48, order:N/A, components:0.37)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>


<td>0.13</td>
<td>RiPP</td>

<td>phomopsin A, phomopsin B, phomopsin E</td>


<td>Phomopsis leptostromiformis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000234: 152-6978">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000234/index.html#r1c1" target="_new">BGC0000234</a></td>





<td class="cc-heat-cell" title="0.41 (id:0.44, order:N/A, components:0.39)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>


<td>0.13</td>
<td>Polyketide</td>

<td>jadomycin</td>


<td>Streptomyces venezuelae ATCC 10712</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002607: 0-24204">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002607/index.html#r1c1" target="_new">BGC0002607</a></td>





<td class="cc-heat-cell" title="0.41 (id:0.44, order:N/A, components:0.37)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>


<td>0.12</td>
<td>NRP, Polyketide</td>

<td>ochratoxin A</td>


<td>Aspergillus westerdijkiae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002598: 1765-25932">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002598/index.html#r1c1" target="_new">BGC0002598</a></td>





<td class="cc-heat-cell" title="0.41 (id:0.44, order:N/A, components:0.37)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>


<td>0.12</td>
<td>Polyketide</td>

<td>ochratoxin A</td>


<td>Aspergillus westerdijkiae</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r5c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001306: 0-6409">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001306/index.html#r1c1" target="_new">BGC0001306</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>RiPP</td>
<td>asperipin 2a</td>
<td>Aspergillus flavus NRRL3357</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000231: 189-5010">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000231/index.html#r1c1" target="_new">BGC0000231</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.43, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>Polyketide</td>
<td>griseusin A, griseusin B</td>
<td>Streptomyces griseus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000627: 0-28295">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000627/index.html#r1c1" target="_new">BGC0000627</a></td>


<td class="cc-heat-cell" title="0.45 (id:0.48, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
<td>0.45</td>
<td>RiPP</td>
<td>ustiloxin B</td>
<td>Aspergillus flavus NRRL3357</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002240: 96-11225">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002240/index.html#r1c1" target="_new">BGC0002240</a></td>


<td class="cc-heat-cell" title="0.43 (id:0.44, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
<td>0.43</td>
<td>Polyketide</td>
<td>BAB, BAA</td>
<td>Metarhizium anisopliae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002238: 216-10949">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002238/index.html#r1c1" target="_new">BGC0002238</a></td>


<td class="cc-heat-cell" title="0.43 (id:0.44, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
<td>0.43</td>
<td>Polyketide</td>
<td>2,4&#39;-dihydroxy-3&#39;-methoxypropiophenone</td>
<td>Aspergillus oryzae RIB40</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002525: 1000-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002525/index.html#r1c1" target="_new">BGC0002525</a></td>


<td class="cc-heat-cell" title="0.42 (id:0.45, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
<td>0.42</td>
<td>Polyketide</td>
<td>fusarubin, 1233A, 1233B, NG-391, lucilactaene</td>
<td>Fusarium sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001398: 4999-64171">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001398/index.html#r1c1" target="_new">BGC0001398</a></td>


<td class="cc-heat-cell" title="0.42 (id:0.48, order:N/A, components:0.37)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
<td>0.42</td>
<td>RiPP</td>
<td>phomopsin A, phomopsin B, phomopsin E</td>
<td>Phomopsis leptostromiformis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000234: 152-6978">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000234/index.html#r1c1" target="_new">BGC0000234</a></td>


<td class="cc-heat-cell" title="0.41 (id:0.44, order:N/A, components:0.39)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
<td>0.41</td>
<td>Polyketide</td>
<td>jadomycin</td>
<td>Streptomyces venezuelae ATCC 10712</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002607: 0-24204">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002607/index.html#r1c1" target="_new">BGC0002607</a></td>


<td class="cc-heat-cell" title="0.41 (id:0.44, order:N/A, components:0.37)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
<td>0.41</td>
<td>NRP, Polyketide</td>
<td>ochratoxin A</td>
<td>Aspergillus westerdijkiae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002598: 1765-25932">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002598/index.html#r1c1" target="_new">BGC0002598</a></td>


<td class="cc-heat-cell" title="0.41 (id:0.44, order:N/A, components:0.37)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
<td>0.41</td>
<td>Polyketide</td>
<td>ochratoxin A</td>
<td>Aspergillus westerdijkiae</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r5c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-77"></div> <span class="help-tooltip" id="cb-general-help-77">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r5c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r5c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r5c1_1.svg">
NW_020194479 (1449493-1553698): Amorphotheca resinae ATCC 22711 unplaced geno... (7% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r5c1_2.svg">
NW_020292127 (15270-136339): Aspergillus ibericus CBS 121593 unplaced genomic... (7% of genes show similarity), NRPS-like,T1PKS,fungal-RiPP-like,terpene
</option>

<option value = "svg/clusterblast_r5c1_3.svg">
NW_017263599 (247828-370093): Mollisia scopiformis unplaced genomic scaffold ... (8% of genes show similarity), T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r5c1_4.svg">
NC_016460 (398818-506276): Thermothielavioides terrestris NRRL 8126 chromosom... (12% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r5c1_5.svg">
NW_006921543 (1393727-1488400): Pseudocercospora fijiensis CIRAD86 unplaced g... (25% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r5c1_6.svg">
NW_025091264 (5067321-5160178): Aspergillus udagawae IFM 46973 DNA, scaffold ... (8% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r5c1_7.svg">
NW_022986860 (22048-112928): Westerdykella ornata strain CBS 379.55 unplaced ... (4% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r5c1_8.svg">
NW_025953261 (590289-740142): Annulohypoxylon maeteangense strain CBS 123835 ... (4% of genes show similarity), T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r5c1_9.svg">
NW_017306195 (991961-1083838): Alternaria alternata strain SRC1lrK2f unplaced... (7% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r5c1_10.svg">
NW_025953252 (1577169-1673001): Annulohypoxylon maeteangense strain CBS 12383... (8% of genes show similarity), fungal-RiPP-like
</option>

</select>
<button id= "clusterblast-r5c1-download">Download graphic</button>
</div>
<div id="clusterblast-r5c1-svg"></div>

</div>
</div>

<div class="body-details-section r5c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-78"></div> <span class="help-tooltip" id="cb-known-help-78">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r5c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-79"></div> <span class="help-tooltip" id="cb-sub-help-79">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r5c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-81"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-81">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c1-pfam-details-svg" data-filename="scaffold_5_r5c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r5c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">











</div>











</div>

</div>

</div>

<div class="page" id='r5c2' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_5 - Region 2 - NRPS-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-737"></div> <span class="help-tooltip" id="region_svg-help-737">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_5.region002.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r5c2-svg" data-filename="scaffold_5_r5c2.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 765,547 - 808,937 nt. (total: 43,391 nt)

<a class="cluster-rules-header" id="r5c2-rules-header" href="#r5c2">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r5c2-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r5c2-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-738"></div> <span class="help-tooltip" id="focus-panel-help-738">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r5c2">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r5c2-gene-table-details" data-name="r5c2-gene-table-details">
Gene overview
</div>



<div class="body-details-header r5c2-nrps_pks" data-name="r5c2-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r5c2-MIBiG-cluster-compare" data-name="r5c2-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r5c2-clusterblast" data-name="r5c2-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r5c2-knownclusterblast" data-name="r5c2-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r5c2-subclusterblast" data-name="r5c2-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r5c2-nrps-pks-bubbles" data-name="r5c2-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r5c2-pfam-details" data-name="r5c2-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r5c2-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-90"></div> <span class="help-tooltip" id="gt-help-tooltip-help-90">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r5c2">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r5c2-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r5c2-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r5c2">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r5c2-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-82"></div> <span class="help-tooltip" id="nrps-domain-help-82">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c2-details-svg" data-filename="scaffold_5_r5c2_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r5c2-details-svg"></div>
</div>
</div>



<div class="body-details-section r5c2-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-83"></div> <span class="help-tooltip" id="MIBiG-help-83">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r5c2-selector">Analysis type:</label>
<select id="comparison-MIBiG-r5c2-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r5c2"></div>
<div id="comparison-MIBiG-r5c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001599: 0-9373">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001599/index.html#r1c1" target="_new">BGC0001599</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.41, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>NRP</td>

<td>fragin</td>


<td>Burkholderia cenocepacia H111</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001626: 0-40540">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001626/index.html#r1c1" target="_new">BGC0001626</a></td>





<td class="cc-heat-cell" title="0.44 (id:0.42, order:N/A, components:0.46)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>


<td>0.14</td>
<td>Polyketide</td>

<td>isoindolinone</td>


<td>Stachybotrys chlorohalonata IBT 40285</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002428: 0-1537">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002428/index.html#r1c1" target="_new">BGC0002428</a></td>





<td class="cc-heat-cell" title="0.41 (id:0.41, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>


<td>0.13</td>
<td>Terpene, Polyketide</td>

<td>DDP-13, DDP-14</td>


<td>Fusarium graminearum PH-1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001390: 100-19645">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001390/index.html#r1c1" target="_new">BGC0001390</a></td>





<td class="cc-heat-cell" title="0.40 (id:0.41, order:N/A, components:0.39)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>


<td>0.12</td>
<td>NRP, Polyketide</td>

<td>LL-Z1272beta</td>


<td>Stachybotrys bisbyi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001544: 0-27727">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001544/index.html#r1c1" target="_new">BGC0001544</a></td>





<td class="cc-heat-cell" title="0.40 (id:0.45, order:N/A, components:0.36)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>


<td>0.12</td>
<td>NRP, Polyketide</td>

<td>chlorflavonin</td>


<td>Aspergillus campestris IBT 28561</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002206: 173-38907">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002206/index.html#r1c1" target="_new">BGC0002206</a></td>





<td class="cc-heat-cell" title="0.38 (id:0.42, order:N/A, components:0.34)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>


<td>0.11</td>
<td>Polyketide</td>

<td>harziphilone, t22azaphilone, isoharziphilone-1, isoharziphilone-2, compound 4, compound 1</td>


<td>Trichoderma guizhouense</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001445: 0-31620">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001445/index.html#r1c1" target="_new">BGC0001445</a></td>





<td class="cc-heat-cell" title="0.38 (id:0.43, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>


<td>0.11</td>
<td>NRP, Polyketide</td>

<td>leporin B</td>


<td>Aspergillus flavus NRRL3357</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001976: 11125-46899">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001976/index.html#r1c1" target="_new">BGC0001976</a></td>





<td class="cc-heat-cell" title="0.38 (id:0.43, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>


<td>0.11</td>
<td>Terpene</td>

<td>eupenifeldin</td>


<td>Phoma sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002212: 0-24132">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002212/index.html#r1c1" target="_new">BGC0002212</a></td>





<td class="cc-heat-cell" title="0.37 (id:0.41, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>


<td>0.11</td>
<td>Polyketide</td>

<td>HEx-pks15 polyketide</td>


<td>Moniliophthora roreri MCA 2997</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001827: 0-43539">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001827/index.html#r1c1" target="_new">BGC0001827</a></td>





<td class="cc-heat-cell" title="0.37 (id:0.43, order:N/A, components:0.32)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>


<td>0.11</td>
<td>Polyketide</td>

<td>xenovulene A</td>


<td>Sarocladium sp. &#39;schorii&#39;</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r5c2-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001599: 0-9373">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001599/index.html#r1c1" target="_new">BGC0001599</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.41, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>NRP</td>
<td>fragin</td>
<td>Burkholderia cenocepacia H111</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001626: 0-40540">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001626/index.html#r1c1" target="_new">BGC0001626</a></td>


<td class="cc-heat-cell" title="0.44 (id:0.42, order:N/A, components:0.46)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
<td>0.44</td>
<td>Polyketide</td>
<td>isoindolinone</td>
<td>Stachybotrys chlorohalonata IBT 40285</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002428: 0-1537">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002428/index.html#r1c1" target="_new">BGC0002428</a></td>


<td class="cc-heat-cell" title="0.41 (id:0.41, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
<td>0.41</td>
<td>Terpene, Polyketide</td>
<td>DDP-13, DDP-14</td>
<td>Fusarium graminearum PH-1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001390: 100-19645">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001390/index.html#r1c1" target="_new">BGC0001390</a></td>


<td class="cc-heat-cell" title="0.40 (id:0.41, order:N/A, components:0.39)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>
<td>0.40</td>
<td>NRP, Polyketide</td>
<td>LL-Z1272beta</td>
<td>Stachybotrys bisbyi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001544: 0-27727">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001544/index.html#r1c1" target="_new">BGC0001544</a></td>


<td class="cc-heat-cell" title="0.40 (id:0.45, order:N/A, components:0.36)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>
<td>0.40</td>
<td>NRP, Polyketide</td>
<td>chlorflavonin</td>
<td>Aspergillus campestris IBT 28561</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002206: 173-38907">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002206/index.html#r1c1" target="_new">BGC0002206</a></td>


<td class="cc-heat-cell" title="0.38 (id:0.42, order:N/A, components:0.34)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
<td>0.38</td>
<td>Polyketide</td>
<td>harziphilone, t22azaphilone, isoharziphilone-1, isoharziphilone-2, compound 4, compound 1</td>
<td>Trichoderma guizhouense</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001445: 0-31620">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001445/index.html#r1c1" target="_new">BGC0001445</a></td>


<td class="cc-heat-cell" title="0.38 (id:0.43, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
<td>0.38</td>
<td>NRP, Polyketide</td>
<td>leporin B</td>
<td>Aspergillus flavus NRRL3357</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001976: 11125-46899">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001976/index.html#r1c1" target="_new">BGC0001976</a></td>


<td class="cc-heat-cell" title="0.38 (id:0.43, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
<td>0.38</td>
<td>Terpene</td>
<td>eupenifeldin</td>
<td>Phoma sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002212: 0-24132">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002212/index.html#r1c1" target="_new">BGC0002212</a></td>


<td class="cc-heat-cell" title="0.37 (id:0.41, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>
<td>0.37</td>
<td>Polyketide</td>
<td>HEx-pks15 polyketide</td>
<td>Moniliophthora roreri MCA 2997</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001827: 0-43539">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001827/index.html#r1c1" target="_new">BGC0001827</a></td>


<td class="cc-heat-cell" title="0.37 (id:0.43, order:N/A, components:0.32)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>
<td>0.37</td>
<td>Polyketide</td>
<td>xenovulene A</td>
<td>Sarocladium sp. &#39;schorii&#39;</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r5c2-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-84"></div> <span class="help-tooltip" id="cb-general-help-84">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r5c2-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r5c2_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r5c2_1.svg">
NW_022987214 (2579931-2686648): Trematosphaeria pertusa strain CBS 122368 unp... (6% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r5c2_2.svg">
NC_070545 (184528-247900): Fusarium falciforme chromosome 2, complete sequence (11% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r5c2_3.svg">
NC_070530 (90953-154325): Fusarium keratoplasticum isolate Fu6.1 chromosome 2... (18% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r5c2_4.svg">
NW_022985375 (598323-661610): Macroventuria anomochaeta strain CBS 525.71 unp... (7% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r5c2_5.svg">
NW_022985015 (779580-842937): Dothidotthia symphoricarpi CBS 119687 unplaced ... (10% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r5c2_6.svg">
NW_024468939 (560863-624073): Cucurbitaria berberidis CBS 394.84 unplaced gen... (7% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r5c2_7.svg">
NW_017253363 (283557-346817): Paraphaeosphaeria sporulosa strain AP3s5-JAC2a ... (7% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r5c2_8.svg">
NW_017306210 (608787-672172): Alternaria alternata strain SRC1lrK2f unplaced ... (8% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r5c2_9.svg">
NW_024467861 (2402641-2465996): Aspergillus sydowii CBS 593.65 unplaced genom... (7% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r5c2_10.svg">
NC_054859 (2644636-2707968): Aspergillus puulaauensis MK2 DNA, chromosome 3, ... (7% of genes show similarity), NRPS-like
</option>

</select>
<button id= "clusterblast-r5c2-download">Download graphic</button>
</div>
<div id="clusterblast-r5c2-svg"></div>

</div>
</div>

<div class="body-details-section r5c2-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-85"></div> <span class="help-tooltip" id="cb-known-help-85">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r5c2-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-86"></div> <span class="help-tooltip" id="cb-sub-help-86">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r5c2-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-89"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-89">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c2-domain-bubble-svg" data-filename="scaffold_5_r5c2_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r5c2-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC2">
Candidate 2 (765546 - 808937): single NRPS-like
</option>


</select>

<div class="details-svg" id="r5c2-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r5c2-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r5c2-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-91"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-91">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c2-pfam-details-svg" data-filename="scaffold_5_r5c2_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r5c2-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r5c2-nrps_pks_products" data-name="r5c2-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r5c2-nrps_pks_monomers" data-name="r5c2-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r5c2-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-87"></div> <span class="help-tooltip" id="nrps-structures-help-87">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 2, location 765546 - 808937:
<div class="collapser collapser-target-CC2 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r5c2-2" class="smiles-canvas" data-smiles="NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r5c2-2-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r5c2-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-88"></div> <span class="help-tooltip" id="nrps-monomers-help-88">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">FUN_001551</span></strong>: X
<div class="collapser collapser-target-FUN_001551 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (57..359)</strong>: X
<div class="collapser collapser-target-FUN_001551 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad
<div class="collapser collapser-target-FUN_001551-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophilic (Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">SHYHLANRLK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>

<dt>Stachelhaus code match:</dt>
<dd>0% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r5c3' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_5 - Region 3 - NRPS-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-739"></div> <span class="help-tooltip" id="region_svg-help-739">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_5.region003.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r5c3-svg" data-filename="scaffold_5_r5c3.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 820,835 - 863,942 nt. (total: 43,108 nt)

<a class="cluster-rules-header" id="r5c3-rules-header" href="#r5c3">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r5c3-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r5c3-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-740"></div> <span class="help-tooltip" id="focus-panel-help-740">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r5c3">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r5c3-gene-table-details" data-name="r5c3-gene-table-details">
Gene overview
</div>



<div class="body-details-header r5c3-nrps_pks" data-name="r5c3-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r5c3-MIBiG-cluster-compare" data-name="r5c3-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r5c3-clusterblast" data-name="r5c3-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r5c3-knownclusterblast" data-name="r5c3-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r5c3-subclusterblast" data-name="r5c3-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r5c3-nrps-pks-bubbles" data-name="r5c3-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r5c3-pfam-details" data-name="r5c3-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r5c3-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-100"></div> <span class="help-tooltip" id="gt-help-tooltip-help-100">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r5c3">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r5c3-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r5c3-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r5c3">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r5c3-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-92"></div> <span class="help-tooltip" id="nrps-domain-help-92">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c3-details-svg" data-filename="scaffold_5_r5c3_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r5c3-details-svg"></div>
</div>
</div>



<div class="body-details-section r5c3-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-93"></div> <span class="help-tooltip" id="MIBiG-help-93">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r5c3-selector">Analysis type:</label>
<select id="comparison-MIBiG-r5c3-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r5c3"></div>
<div id="comparison-MIBiG-r5c3-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001168: 0-4582">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001168/index.html#r1c1" target="_new">BGC0001168</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>NRP</td>

<td>livipeptin</td>


<td>Streptomyces lividans 1326</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001268: 894-12763">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001268/index.html#r1c1" target="_new">BGC0001268</a></td>





<td class="cc-heat-cell" title="0.44 (id:0.41, order:N/A, components:0.48)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>


<td>0.14</td>
<td>NRP, Polyketide</td>

<td>fusarin C</td>


<td>Fusarium fujikuroi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002212: 0-24132">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002212/index.html#r1c1" target="_new">BGC0002212</a></td>





<td class="cc-heat-cell" title="0.41 (id:0.41, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>


<td>0.13</td>
<td>Polyketide</td>

<td>HEx-pks15 polyketide</td>


<td>Moniliophthora roreri MCA 2997</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001627: 0-10806">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001627/index.html#r1c1" target="_new">BGC0001627</a></td>





<td class="cc-heat-cell" title="0.37 (id:0.41, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>


<td>0.11</td>
<td>NRP</td>

<td>isonitrile lipopeptides</td>


<td>Mycobacterium tuberculosis H37Rv</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001600: 0-23599">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001600/index.html#r1c1" target="_new">BGC0001600</a></td>





<td class="cc-heat-cell" title="0.35 (id:0.41, order:N/A, components:0.29)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>


<td>0.10</td>
<td>Polyketide</td>

<td>fusarielin H</td>


<td>Fusarium graminearum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001026: 1-19101">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001026/index.html#r1c1" target="_new">BGC0001026</a></td>





<td class="cc-heat-cell" title="0.32 (id:0.41, order:N/A, components:0.26)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.32), rgb(0, 0, 0, 0.32) 32%, white 32%)"></td>


<td>0.09</td>
<td>NRP, Polyketide</td>

<td>NG-391</td>


<td>Metarhizium anisopliae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001626: 0-40540">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001626/index.html#r1c1" target="_new">BGC0001626</a></td>





<td class="cc-heat-cell" title="0.31 (id:0.41, order:N/A, components:0.24)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>


<td>0.08</td>
<td>Polyketide</td>

<td>isoindolinone</td>


<td>Stachybotrys chlorohalonata IBT 40285</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001923: 495-32413">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001923/index.html#r1c1" target="_new">BGC0001923</a></td>





<td class="cc-heat-cell" title="0.30 (id:0.41, order:N/A, components:0.22)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>


<td>0.08</td>
<td>Terpene, Polyketide</td>

<td>ascochlorin</td>


<td>Acremonium egyptiacum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002734: 207-20832">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002734/index.html#r1c1" target="_new">BGC0002734</a></td>





<td class="cc-heat-cell" title="0.29 (id:0.41, order:N/A, components:0.21)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>


<td>0.07</td>
<td>Polyketide</td>

<td>2,4-dihydroxy-5,6-dimethyl benzaldehyde</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001436: 25-56534">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001436/index.html#r1c1" target="_new">BGC0001436</a></td>





<td class="cc-heat-cell" title="0.29 (id:0.44, order:N/A, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>


<td>0.07</td>
<td>Polyketide</td>

<td>cornexistin</td>


<td>Paecilomyces divaricatus</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r5c3-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001168: 0-4582">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001168/index.html#r1c1" target="_new">BGC0001168</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>NRP</td>
<td>livipeptin</td>
<td>Streptomyces lividans 1326</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001268: 894-12763">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001268/index.html#r1c1" target="_new">BGC0001268</a></td>


<td class="cc-heat-cell" title="0.44 (id:0.41, order:N/A, components:0.48)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
<td>0.44</td>
<td>NRP, Polyketide</td>
<td>fusarin C</td>
<td>Fusarium fujikuroi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002212: 0-24132">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002212/index.html#r1c1" target="_new">BGC0002212</a></td>


<td class="cc-heat-cell" title="0.41 (id:0.41, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
<td>0.41</td>
<td>Polyketide</td>
<td>HEx-pks15 polyketide</td>
<td>Moniliophthora roreri MCA 2997</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001627: 0-10806">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001627/index.html#r1c1" target="_new">BGC0001627</a></td>


<td class="cc-heat-cell" title="0.37 (id:0.41, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>
<td>0.37</td>
<td>NRP</td>
<td>isonitrile lipopeptides</td>
<td>Mycobacterium tuberculosis H37Rv</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001600: 0-23599">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001600/index.html#r1c1" target="_new">BGC0001600</a></td>


<td class="cc-heat-cell" title="0.35 (id:0.41, order:N/A, components:0.29)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>
<td>0.35</td>
<td>Polyketide</td>
<td>fusarielin H</td>
<td>Fusarium graminearum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001026: 1-19101">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001026/index.html#r1c1" target="_new">BGC0001026</a></td>


<td class="cc-heat-cell" title="0.32 (id:0.41, order:N/A, components:0.26)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.32), rgb(0, 0, 0, 0.32) 32%, white 32%)"></td>
<td>0.32</td>
<td>NRP, Polyketide</td>
<td>NG-391</td>
<td>Metarhizium anisopliae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001626: 0-40540">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001626/index.html#r1c1" target="_new">BGC0001626</a></td>


<td class="cc-heat-cell" title="0.31 (id:0.41, order:N/A, components:0.24)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>
<td>0.31</td>
<td>Polyketide</td>
<td>isoindolinone</td>
<td>Stachybotrys chlorohalonata IBT 40285</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001923: 495-32413">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001923/index.html#r1c1" target="_new">BGC0001923</a></td>


<td class="cc-heat-cell" title="0.30 (id:0.41, order:N/A, components:0.22)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>
<td>0.30</td>
<td>Terpene, Polyketide</td>
<td>ascochlorin</td>
<td>Acremonium egyptiacum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002734: 207-20832">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002734/index.html#r1c1" target="_new">BGC0002734</a></td>


<td class="cc-heat-cell" title="0.29 (id:0.41, order:N/A, components:0.21)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>
<td>0.29</td>
<td>Polyketide</td>
<td>2,4-dihydroxy-5,6-dimethyl benzaldehyde</td>
<td>Aspergillus terreus NIH2624</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001436: 25-56534">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001436/index.html#r1c1" target="_new">BGC0001436</a></td>


<td class="cc-heat-cell" title="0.29 (id:0.44, order:N/A, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>
<td>0.29</td>
<td>Polyketide</td>
<td>cornexistin</td>
<td>Paecilomyces divaricatus</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r5c3-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-94"></div> <span class="help-tooltip" id="cb-general-help-94">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<p>No significant ClusterBlast hits found.</p>
</div>

</div>
</div>

<div class="body-details-section r5c3-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-95"></div> <span class="help-tooltip" id="cb-known-help-95">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r5c3-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-96"></div> <span class="help-tooltip" id="cb-sub-help-96">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r5c3-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-99"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-99">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c3-domain-bubble-svg" data-filename="scaffold_5_r5c3_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r5c3-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC3">
Candidate 3 (820834 - 863942): single NRPS-like
</option>


</select>

<div class="details-svg" id="r5c3-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r5c3-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r5c3-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-101"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-101">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c3-pfam-details-svg" data-filename="scaffold_5_r5c3_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r5c3-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r5c3-nrps_pks_products" data-name="r5c3-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r5c3-nrps_pks_monomers" data-name="r5c3-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r5c3-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-97"></div> <span class="help-tooltip" id="nrps-structures-help-97">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 3, location 820834 - 863942:
<div class="collapser collapser-target-CC3 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r5c3-3" class="smiles-canvas" data-smiles="NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r5c3-3-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r5c3-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-98"></div> <span class="help-tooltip" id="nrps-monomers-help-98">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">FUN_001555</span></strong>: X
<div class="collapser collapser-target-FUN_001555 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (42..340)</strong>: X
<div class="collapser collapser-target-FUN_001555 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr
<div class="collapser collapser-target-FUN_001555-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">PGFVVGCVFK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Val <span class="serif">DGFFVGGVFK</span> (35% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>70% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>





<div class="page" id='r6c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_6 - Region 1 - NRPS-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-741"></div> <span class="help-tooltip" id="region_svg-help-741">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_6.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r6c1-svg" data-filename="scaffold_6_r6c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 161,974 - 205,181 nt. (total: 43,208 nt)

<a class="cluster-rules-header" id="r6c1-rules-header" href="#r6c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r6c1-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r6c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-742"></div> <span class="help-tooltip" id="focus-panel-help-742">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r6c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r6c1-gene-table-details" data-name="r6c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r6c1-nrps_pks" data-name="r6c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r6c1-MIBiG-cluster-compare" data-name="r6c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r6c1-clusterblast" data-name="r6c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r6c1-knownclusterblast" data-name="r6c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r6c1-subclusterblast" data-name="r6c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r6c1-nrps-pks-bubbles" data-name="r6c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r6c1-pfam-details" data-name="r6c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r6c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-110"></div> <span class="help-tooltip" id="gt-help-tooltip-help-110">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r6c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r6c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r6c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r6c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r6c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-102"></div> <span class="help-tooltip" id="nrps-domain-help-102">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c1-details-svg" data-filename="scaffold_6_r6c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r6c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r6c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-103"></div> <span class="help-tooltip" id="MIBiG-help-103">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r6c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r6c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r6c1"></div>
<div id="comparison-MIBiG-r6c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001168: 0-4582">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001168/index.html#r1c1" target="_new">BGC0001168</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.21</td>
<td>NRP</td>

<td>livipeptin</td>


<td>Streptomyces lividans 1326</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001268: 894-12763">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001268/index.html#r1c1" target="_new">BGC0001268</a></td>





<td class="cc-heat-cell" title="0.44 (id:0.41, order:N/A, components:0.48)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>


<td>0.15</td>
<td>NRP, Polyketide</td>

<td>fusarin C</td>


<td>Fusarium fujikuroi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002240: 96-11225">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002240/index.html#r1c1" target="_new">BGC0002240</a></td>





<td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>


<td>0.13</td>
<td>Polyketide</td>

<td>BAB, BAA</td>


<td>Metarhizium anisopliae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001600: 0-23599">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001600/index.html#r1c1" target="_new">BGC0001600</a></td>





<td class="cc-heat-cell" title="0.40 (id:0.43, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>


<td>0.12</td>
<td>Polyketide</td>

<td>fusarielin H</td>


<td>Fusarium graminearum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002209: 560-36987">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002209/index.html#r1c1" target="_new">BGC0002209</a></td>





<td class="cc-heat-cell" title="0.37 (id:0.42, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>


<td>0.11</td>
<td>Polyketide</td>

<td>ustethylin A</td>


<td>Aspergillus ustus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001390: 100-19645">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001390/index.html#r1c1" target="_new">BGC0001390</a></td>





<td class="cc-heat-cell" title="0.34 (id:0.42, order:N/A, components:0.28)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>


<td>0.09</td>
<td>NRP, Polyketide</td>

<td>LL-Z1272beta</td>


<td>Stachybotrys bisbyi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002278: 0-28520">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002278/index.html#r1c1" target="_new">BGC0002278</a></td>





<td class="cc-heat-cell" title="0.32 (id:0.41, order:N/A, components:0.26)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.32), rgb(0, 0, 0, 0.32) 32%, white 32%)"></td>


<td>0.09</td>
<td>Alkaloid, NRP</td>

<td>brasiliamide I, brasiliamide A, brasiliamide B, 2,2-dibenzylpiperazine, brasiliamide D</td>


<td>Penicillium brasilianum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002167: 0-15099">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002167/index.html#r1c1" target="_new">BGC0002167</a></td>





<td class="cc-heat-cell" title="0.31 (id:0.42, order:N/A, components:0.23)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>


<td>0.08</td>
<td>NRP</td>

<td>actinopolymorphol C, morpholine containing hemiacetal piperazine compound, piperazine compound 2, piperazine compound 1, 3-(p-hydroxyphenyl)-1,2-propanediol, N,N-dioxide containing derivate, O-sulfonated actinopolymorphol C, C-3 sulfonylated derivative</td>


<td>Aspergillus flavus NRRL3357</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002324: 0-17497">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002324/index.html#r1c1" target="_new">BGC0002324</a></td>





<td class="cc-heat-cell" title="0.29 (id:0.42, order:N/A, components:0.20)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>


<td>0.07</td>
<td>NRP, Polyketide</td>

<td>myxochelin B, myxochelin N, myxochelin O, myxochelin P, myxochelin Q, myxochelin A</td>


<td>Archangium violaceum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001218: 1349-26398">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001218/index.html#r1c1" target="_new">BGC0001218</a></td>





<td class="cc-heat-cell" title="0.28 (id:0.41, order:N/A, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>


<td>0.07</td>
<td>NRP, Polyketide</td>

<td>fumosorinone</td>


<td>Cordyceps fumosorosea</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r6c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001168: 0-4582">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001168/index.html#r1c1" target="_new">BGC0001168</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>NRP</td>
<td>livipeptin</td>
<td>Streptomyces lividans 1326</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001268: 894-12763">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001268/index.html#r1c1" target="_new">BGC0001268</a></td>


<td class="cc-heat-cell" title="0.44 (id:0.41, order:N/A, components:0.48)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
<td>0.44</td>
<td>NRP, Polyketide</td>
<td>fusarin C</td>
<td>Fusarium fujikuroi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002240: 96-11225">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002240/index.html#r1c1" target="_new">BGC0002240</a></td>


<td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
<td>0.42</td>
<td>Polyketide</td>
<td>BAB, BAA</td>
<td>Metarhizium anisopliae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001600: 0-23599">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001600/index.html#r1c1" target="_new">BGC0001600</a></td>


<td class="cc-heat-cell" title="0.40 (id:0.43, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>
<td>0.40</td>
<td>Polyketide</td>
<td>fusarielin H</td>
<td>Fusarium graminearum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002209: 560-36987">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002209/index.html#r1c1" target="_new">BGC0002209</a></td>


<td class="cc-heat-cell" title="0.37 (id:0.42, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>
<td>0.37</td>
<td>Polyketide</td>
<td>ustethylin A</td>
<td>Aspergillus ustus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001390: 100-19645">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001390/index.html#r1c1" target="_new">BGC0001390</a></td>


<td class="cc-heat-cell" title="0.34 (id:0.42, order:N/A, components:0.28)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>
<td>0.34</td>
<td>NRP, Polyketide</td>
<td>LL-Z1272beta</td>
<td>Stachybotrys bisbyi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002278: 0-28520">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002278/index.html#r1c1" target="_new">BGC0002278</a></td>


<td class="cc-heat-cell" title="0.32 (id:0.41, order:N/A, components:0.26)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.32), rgb(0, 0, 0, 0.32) 32%, white 32%)"></td>
<td>0.32</td>
<td>Alkaloid, NRP</td>
<td>brasiliamide I, brasiliamide A, brasiliamide B, 2,2-dibenzylpiperazine, brasiliamide D</td>
<td>Penicillium brasilianum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002167: 0-15099">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002167/index.html#r1c1" target="_new">BGC0002167</a></td>


<td class="cc-heat-cell" title="0.31 (id:0.42, order:N/A, components:0.23)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>
<td>0.31</td>
<td>NRP</td>
<td>actinopolymorphol C, morpholine containing hemiacetal piperazine compound, piperazine compound 2, piperazine compound 1, 3-(p-hydroxyphenyl)-1,2-propanediol, N,N-dioxide containing derivate, O-sulfonated actinopolymorphol C, C-3 sulfonylated derivative</td>
<td>Aspergillus flavus NRRL3357</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002324: 0-17497">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002324/index.html#r1c1" target="_new">BGC0002324</a></td>


<td class="cc-heat-cell" title="0.29 (id:0.42, order:N/A, components:0.20)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>
<td>0.29</td>
<td>NRP, Polyketide</td>
<td>myxochelin B, myxochelin N, myxochelin O, myxochelin P, myxochelin Q, myxochelin A</td>
<td>Archangium violaceum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001218: 1349-26398">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001218/index.html#r1c1" target="_new">BGC0001218</a></td>


<td class="cc-heat-cell" title="0.28 (id:0.41, order:N/A, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>
<td>0.28</td>
<td>NRP, Polyketide</td>
<td>fumosorinone</td>
<td>Cordyceps fumosorosea</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r6c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-104"></div> <span class="help-tooltip" id="cb-general-help-104">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r6c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r6c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r6c1_1.svg">
NW_026250019 (404700-467985): Hypoxylon trugodes strain CBS 135444 unplaced g... (15% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r6c1_2.svg">
NW_020291811 (552900-616141): Aspergillus costaricaensis CBS 115574 unplaced ... (10% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r6c1_3.svg">
NW_025953810 (264034-384248): Annulohypoxylon truncatum strain CBS 140777 unp... (4% of genes show similarity), NRPS-like,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r6c1_4.svg">
NW_006917099 (1200604-1277164): Pestalotiopsis fici W106-1 unplaced genomic s... (20% of genes show similarity), NRPS
</option>

</select>
<button id= "clusterblast-r6c1-download">Download graphic</button>
</div>
<div id="clusterblast-r6c1-svg"></div>

</div>
</div>

<div class="body-details-section r6c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-105"></div> <span class="help-tooltip" id="cb-known-help-105">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r6c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-106"></div> <span class="help-tooltip" id="cb-sub-help-106">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r6c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-109"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-109">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c1-domain-bubble-svg" data-filename="scaffold_6_r6c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r6c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (161973 - 205181): single NRPS-like
</option>


</select>

<div class="details-svg" id="r6c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r6c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r6c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-111"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-111">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c1-pfam-details-svg" data-filename="scaffold_6_r6c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r6c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r6c1-nrps_pks_products" data-name="r6c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r6c1-nrps_pks_monomers" data-name="r6c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r6c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-107"></div> <span class="help-tooltip" id="nrps-structures-help-107">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 161973 - 205181:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r6c1-1" class="smiles-canvas" data-smiles="NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r6c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r6c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-108"></div> <span class="help-tooltip" id="nrps-monomers-help-108">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">FUN_001660</span></strong>: X
<div class="collapser collapser-target-FUN_001660 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (49..337)</strong>: X
<div class="collapser collapser-target-FUN_001660 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr
<div class="collapser collapser-target-FUN_001660-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">--FHTAI--K</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>

<dt>Stachelhaus code match:</dt>
<dd>0% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r6c2' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_6 - Region 2 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-743"></div> <span class="help-tooltip" id="region_svg-help-743">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_6.region002.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r6c2-svg" data-filename="scaffold_6_r6c2.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 444,815 - 493,842 nt. (total: 49,028 nt)

<a class="cluster-rules-header" id="r6c2-rules-header" href="#r6c2">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r6c2-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r6c2-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-744"></div> <span class="help-tooltip" id="focus-panel-help-744">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r6c2">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r6c2-gene-table-details" data-name="r6c2-gene-table-details">
Gene overview
</div>



<div class="body-details-header r6c2-nrps_pks" data-name="r6c2-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r6c2-MIBiG-cluster-compare" data-name="r6c2-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r6c2-clusterblast" data-name="r6c2-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r6c2-knownclusterblast" data-name="r6c2-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r6c2-subclusterblast" data-name="r6c2-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r6c2-nrps-pks-bubbles" data-name="r6c2-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r6c2-pfam-details" data-name="r6c2-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r6c2-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-120"></div> <span class="help-tooltip" id="gt-help-tooltip-help-120">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r6c2">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r6c2-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r6c2-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r6c2">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r6c2-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-112"></div> <span class="help-tooltip" id="nrps-domain-help-112">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c2-details-svg" data-filename="scaffold_6_r6c2_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r6c2-details-svg"></div>
</div>
</div>



<div class="body-details-section r6c2-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-113"></div> <span class="help-tooltip" id="MIBiG-help-113">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r6c2-selector">Analysis type:</label>
<select id="comparison-MIBiG-r6c2-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r6c2"></div>
<div id="comparison-MIBiG-r6c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002191: 260-8785">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002191/index.html#r1c1" target="_new">BGC0002191</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.29</td>
<td>Polyketide</td>

<td>prolipyrone B, gibepyrone D</td>


<td>Fusarium graminearum PH-1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001252: 0-7958">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001252/index.html#r1c1" target="_new">BGC0001252</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.28</td>
<td>Polyketide</td>

<td>UNII-YC2Q1O94PT</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002171: 8-7317">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002171/index.html#r1c1" target="_new">BGC0002171</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.45, order:1.00, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.27</td>
<td>NRP, Polyketide</td>

<td>pyrophen, campyrone B</td>


<td>Aspergillus niger ATCC 1015</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002275: 0-10320">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002275/index.html#r1c1" target="_new">BGC0002275</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.45, order:1.00, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.27</td>
<td>NRP, Polyketide</td>

<td>pyrophen, campyrone B</td>


<td>Aspergillus niger CBS 101883</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001606: 0-22544">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001606/index.html#r1c1" target="_new">BGC0001606</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.42, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.27</td>
<td>Polyketide</td>

<td>gibepyrone-A</td>


<td>Fusarium fujikuroi IMI 58289</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002222: 888-32276">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002222/index.html#r1c1" target="_new">BGC0002222</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.26</td>
<td>Polyketide</td>

<td>zopfiellin</td>


<td>Diffractella curvata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002155: 78-16511">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002155/index.html#r1c1" target="_new">BGC0002155</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.49, order:0.46, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.26</td>
<td>Polyketide</td>

<td>nectriapyrone C, nectriapyrone D, nectriapyrone</td>


<td>Pyricularia oryzae 70-15</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002194: 147-15389">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002194/index.html#r1c1" target="_new">BGC0002194</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.45, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.25</td>
<td>Polyketide</td>

<td>epipyrone A</td>


<td>Epicoccum nigrum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002224: 685-37748">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002224/index.html#r1c1" target="_new">BGC0002224</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.24</td>
<td>Polyketide</td>

<td>zopfiellin</td>


<td>Diffractella curvata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002525: 1000-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002525/index.html#r1c1" target="_new">BGC0002525</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.24</td>
<td>Polyketide</td>

<td>fusarubin, 1233A, 1233B, NG-391, lucilactaene</td>


<td>Fusarium sp.</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r6c2-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002191: 260-8785">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002191/index.html#r1c1" target="_new">BGC0002191</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>Polyketide</td>
<td>prolipyrone B, gibepyrone D</td>
<td>Fusarium graminearum PH-1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001252: 0-7958">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001252/index.html#r1c1" target="_new">BGC0001252</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>Polyketide</td>
<td>UNII-YC2Q1O94PT</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002171: 8-7317">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002171/index.html#r1c1" target="_new">BGC0002171</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.45, order:1.00, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>NRP, Polyketide</td>
<td>pyrophen, campyrone B</td>
<td>Aspergillus niger ATCC 1015</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002275: 0-10320">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002275/index.html#r1c1" target="_new">BGC0002275</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.45, order:1.00, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>NRP, Polyketide</td>
<td>pyrophen, campyrone B</td>
<td>Aspergillus niger CBS 101883</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001606: 0-22544">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001606/index.html#r1c1" target="_new">BGC0001606</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.42, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>Polyketide</td>
<td>gibepyrone-A</td>
<td>Fusarium fujikuroi IMI 58289</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002222: 888-32276">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002222/index.html#r1c1" target="_new">BGC0002222</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>Polyketide</td>
<td>zopfiellin</td>
<td>Diffractella curvata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002155: 78-16511">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002155/index.html#r1c1" target="_new">BGC0002155</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.49, order:0.46, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Polyketide</td>
<td>nectriapyrone C, nectriapyrone D, nectriapyrone</td>
<td>Pyricularia oryzae 70-15</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002194: 147-15389">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002194/index.html#r1c1" target="_new">BGC0002194</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.45, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Polyketide</td>
<td>epipyrone A</td>
<td>Epicoccum nigrum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002224: 685-37748">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002224/index.html#r1c1" target="_new">BGC0002224</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Polyketide</td>
<td>zopfiellin</td>
<td>Diffractella curvata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002525: 1000-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002525/index.html#r1c1" target="_new">BGC0002525</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Polyketide</td>
<td>fusarubin, 1233A, 1233B, NG-391, lucilactaene</td>
<td>Fusarium sp.</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r6c2-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-114"></div> <span class="help-tooltip" id="cb-general-help-114">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r6c2-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r6c2_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r6c2_1.svg">
NW_025953044 (1245032-1332694): Daldinia caldariorum strain CBS 122874 unplac... (16% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r6c2_2.svg">
NW_026054544 (86282-154649): Daldinia decipiens strain CBS 113046 unplaced ge... (9% of genes show similarity), T1PKS,terpene
</option>

<option value = "svg/clusterblast_r6c2_3.svg">
NC_070542 (87719-155711): Fusarium keratoplasticum isolate Fu6.1 chromosome 1... (11% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r6c2_4.svg">
NC_070550 (1020024-1087996): Fusarium falciforme chromosome 7, complete sequence (12% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r6c2_5.svg">
NZ_CP030771 (7915496-7990387): Streptomyces sp. YIM 121038 chromosome, comple... (4% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r6c2_6.svg">
NC_064678 (89657-156738): Colletotrichum lupini chromosome 5, complete sequence (21% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r6c2_7.svg">
NC_064676 (6809055-6903508): Colletotrichum lupini chromosome 3, complete seq... (6% of genes show similarity), T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r6c2_8.svg">
NC_030954 (5210919-5335660): Colletotrichum higginsianum IMI 349063 chromosom... (5% of genes show similarity), T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r6c2_9.svg">
NC_043743 (1243597-1335373): Pyricularia pennisetigena strain Br36 chromosome... (8% of genes show similarity), T1PKS,terpene
</option>

<option value = "svg/clusterblast_r6c2_10.svg">
NW_022156719 (1737811-1806959): Pyricularia grisea strain NI907 chromosome V ... (11% of genes show similarity), T1PKS
</option>

</select>
<button id= "clusterblast-r6c2-download">Download graphic</button>
</div>
<div id="clusterblast-r6c2-svg"></div>

</div>
</div>

<div class="body-details-section r6c2-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-115"></div> <span class="help-tooltip" id="cb-known-help-115">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r6c2-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-116"></div> <span class="help-tooltip" id="cb-sub-help-116">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r6c2-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-119"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-119">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c2-domain-bubble-svg" data-filename="scaffold_6_r6c2_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r6c2-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC2">
Candidate 2 (444814 - 493842): single T1PKS
</option>


</select>

<div class="details-svg" id="r6c2-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r6c2-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r6c2-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-121"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-121">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c2-pfam-details-svg" data-filename="scaffold_6_r6c2_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r6c2-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r6c2-nrps_pks_products" data-name="r6c2-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r6c2-nrps_pks_monomers" data-name="r6c2-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r6c2-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-117"></div> <span class="help-tooltip" id="nrps-structures-help-117">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 2, location 444814 - 493842:
<div class="collapser collapser-target-CC2 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r6c2-2" class="smiles-canvas" data-smiles="C([*])C(-O)(C)"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Me-pk)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-pk" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-pk*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r6c2-2-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)(C)"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r6c2-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-118"></div> <span class="help-tooltip" id="nrps-monomers-help-118">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">FUN_001742</span></strong>: pk
<div class="collapser collapser-target-FUN_001742 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (534..822)</strong>: pk
<div class="collapser collapser-target-FUN_001742 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-FUN_001742-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 66.7%</dd>
<dd>2-Rhyd-Malonyl-CoA: 58.3%</dd>
<dd>2-Methylbutyryl-CoA: 58.3%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-FUN_001742-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 62.9</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 57.1</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 31.5</dt>

<dd></dd><dt>Malonyl-CoA: 28.4</dt>

<dd></dd><dt>Propionyl-CoA: 26.5</dt>

<dd></dd><dt>trans-1,2-CPDA: 22.0</dt>

<dd></dd><dt>fatty_acid: 19.3</dt>

<dd></dd><dt>Isobutyryl-CoA: 17.8</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 16.0</dt>

<dd></dd><dt>CHC-CoA: 13.7</dt>

<dd></dd><dt>inactive: 9.2</dt>

<dd></dd><dt>Benzoyl-CoA: 0.0</dt>

<dd></dd><dt>Acetyl-CoA: 0.0</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>












</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>





<div class="page" id='r8c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_8 - Region 1 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-745"></div> <span class="help-tooltip" id="region_svg-help-745">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_8.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r8c1-svg" data-filename="scaffold_8_r8c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 357,322 - 378,518 nt. (total: 21,197 nt)

<a class="cluster-rules-header" id="r8c1-rules-header" href="#r8c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r8c1-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>

<div class="region-svg-container">
<div id='r8c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-746"></div> <span class="help-tooltip" id="focus-panel-help-746">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r8c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r8c1-gene-table-details" data-name="r8c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r8c1-MIBiG-cluster-compare" data-name="r8c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r8c1-clusterblast" data-name="r8c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r8c1-knownclusterblast" data-name="r8c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r8c1-subclusterblast" data-name="r8c1-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r8c1-pfam-details" data-name="r8c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r8c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-126"></div> <span class="help-tooltip" id="gt-help-tooltip-help-126">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r8c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r8c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r8c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r8c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r8c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-122"></div> <span class="help-tooltip" id="MIBiG-help-122">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r8c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r8c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r8c1"></div>
<div id="comparison-MIBiG-r8c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">terpene</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000673: 0-9097">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000673/index.html#r1c1" target="_new">BGC0000673</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.49, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>Terpene</td>

<td>pimara-8(14),15-diene</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000688: 696-13661">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000688/index.html#r1c1" target="_new">BGC0000688</a></td>





<td class="cc-heat-cell" title="0.39 (id:0.52, order:N/A, components:0.29)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>


<td>0.12</td>
<td>Terpene</td>

<td>copalyl diphosphate</td>


<td>Diaporthe amygdali</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000676: 621-14838">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000676/index.html#r1c1" target="_new">BGC0000676</a></td>





<td class="cc-heat-cell" title="0.39 (id:0.52, order:N/A, components:0.29)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>


<td>0.12</td>
<td>Terpene</td>

<td>aphidicolin, aphidicolan-16-ol, 3-deoxyaphidicolin, 17-deoxyaphidicolin</td>


<td>Phoma betae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002604: 0-17897">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002604/index.html#r1c1" target="_new">BGC0002604</a></td>





<td class="cc-heat-cell" title="0.35 (id:0.53, order:N/A, components:0.23)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>


<td>0.10</td>
<td>Polyketide</td>

<td>sartorypyrone A</td>


<td>Aspergillus felis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002320: 3-7131">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002320/index.html#r1c1" target="_new">BGC0002320</a></td>





<td class="cc-heat-cell" title="0.35 (id:0.48, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>


<td>0.10</td>
<td>Terpene</td>

<td>conidiogenone</td>


<td>Penicillium rubens Wisconsin 54-1255</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002427: 0-21541">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002427/index.html#r1c1" target="_new">BGC0002427</a></td>





<td class="cc-heat-cell" title="0.32 (id:0.51, order:N/A, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.32), rgb(0, 0, 0, 0.32) 32%, white 32%)"></td>


<td>0.08</td>
<td>Terpene, Polyketide</td>

<td>subglutinol A, subglutinol B</td>


<td>Metarhizium robertsii ARSEF 23</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001082: 0-18809">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001082/index.html#r1c1" target="_new">BGC0001082</a></td>





<td class="cc-heat-cell" title="0.31 (id:0.51, order:N/A, components:0.18)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>


<td>0.08</td>
<td>Terpene</td>

<td>paxilline, paspaline, 13-dehydroxypaxilline, paspaline B</td>


<td>Penicillium paxilli</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001969: 0-7810">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001969/index.html#r1c1" target="_new">BGC0001969</a></td>





<td class="cc-heat-cell" title="0.28 (id:0.48, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>


<td>0.07</td>
<td>Terpene</td>

<td>asperterpenoid A</td>


<td>Talaromyces wortmannii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002603: 0-17042">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002603/index.html#r1c1" target="_new">BGC0002603</a></td>





<td class="cc-heat-cell" title="0.26 (id:0.52, order:N/A, components:0.13)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>


<td>0.06</td>
<td>Polyketide</td>

<td>chevalone E</td>


<td>Aspergillus versicolor</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001776: 15-68182">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001776/index.html#r1c1" target="_new">BGC0001776</a></td>





<td class="cc-heat-cell" title="0.25 (id:0.52, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>


<td>0.06</td>
<td>Terpene</td>

<td>shearinine D</td>


<td>Penicillium janthinellum</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r8c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000673: 0-9097">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000673/index.html#r1c1" target="_new">BGC0000673</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.49, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>Terpene</td>
<td>pimara-8(14),15-diene</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000688: 696-13661">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000688/index.html#r1c1" target="_new">BGC0000688</a></td>


<td class="cc-heat-cell" title="0.39 (id:0.52, order:N/A, components:0.29)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
<td>0.39</td>
<td>Terpene</td>
<td>copalyl diphosphate</td>
<td>Diaporthe amygdali</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000676: 621-14838">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000676/index.html#r1c1" target="_new">BGC0000676</a></td>


<td class="cc-heat-cell" title="0.39 (id:0.52, order:N/A, components:0.29)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
<td>0.39</td>
<td>Terpene</td>
<td>aphidicolin, aphidicolan-16-ol, 3-deoxyaphidicolin, 17-deoxyaphidicolin</td>
<td>Phoma betae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002604: 0-17897">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002604/index.html#r1c1" target="_new">BGC0002604</a></td>


<td class="cc-heat-cell" title="0.35 (id:0.53, order:N/A, components:0.23)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>
<td>0.35</td>
<td>Polyketide</td>
<td>sartorypyrone A</td>
<td>Aspergillus felis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002320: 3-7131">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002320/index.html#r1c1" target="_new">BGC0002320</a></td>


<td class="cc-heat-cell" title="0.35 (id:0.48, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>
<td>0.35</td>
<td>Terpene</td>
<td>conidiogenone</td>
<td>Penicillium rubens Wisconsin 54-1255</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002427: 0-21541">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002427/index.html#r1c1" target="_new">BGC0002427</a></td>


<td class="cc-heat-cell" title="0.32 (id:0.51, order:N/A, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.32), rgb(0, 0, 0, 0.32) 32%, white 32%)"></td>
<td>0.32</td>
<td>Terpene, Polyketide</td>
<td>subglutinol A, subglutinol B</td>
<td>Metarhizium robertsii ARSEF 23</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001082: 0-18809">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001082/index.html#r1c1" target="_new">BGC0001082</a></td>


<td class="cc-heat-cell" title="0.31 (id:0.51, order:N/A, components:0.18)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>
<td>0.31</td>
<td>Terpene</td>
<td>paxilline, paspaline, 13-dehydroxypaxilline, paspaline B</td>
<td>Penicillium paxilli</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001969: 0-7810">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001969/index.html#r1c1" target="_new">BGC0001969</a></td>


<td class="cc-heat-cell" title="0.28 (id:0.48, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>
<td>0.28</td>
<td>Terpene</td>
<td>asperterpenoid A</td>
<td>Talaromyces wortmannii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002603: 0-17042">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002603/index.html#r1c1" target="_new">BGC0002603</a></td>


<td class="cc-heat-cell" title="0.26 (id:0.52, order:N/A, components:0.13)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>
<td>0.26</td>
<td>Polyketide</td>
<td>chevalone E</td>
<td>Aspergillus versicolor</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001776: 15-68182">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001776/index.html#r1c1" target="_new">BGC0001776</a></td>


<td class="cc-heat-cell" title="0.25 (id:0.52, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>
<td>0.25</td>
<td>Terpene</td>
<td>shearinine D</td>
<td>Penicillium janthinellum</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r8c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-123"></div> <span class="help-tooltip" id="cb-general-help-123">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r8c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r8c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r8c1_1.svg">
NW_025953795 (839976-871122): Annulohypoxylon truncatum strain CBS 140777 unp... (26% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r8c1_2.svg">
NW_026054544 (86282-154649): Daldinia decipiens strain CBS 113046 unplaced ge... (18% of genes show similarity), T1PKS,terpene
</option>

<option value = "svg/clusterblast_r8c1_3.svg">
NW_026250019 (2151060-2242030): Hypoxylon trugodes strain CBS 135444 unplaced... (11% of genes show similarity), NRPS-like,terpene
</option>

<option value = "svg/clusterblast_r8c1_4.svg">
NW_025953254 (339129-370278): Annulohypoxylon maeteangense strain CBS 123835 ... (26% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r8c1_5.svg">
NW_009276924 (220128-251337): Verticillium dahliae VdLs.17 supercont1.21 geno... (33% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r8c1_6.svg">
NC_016457 (667703-698978): Thermothielavioides terrestris NRRL 8126 chromosom... (37% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r8c1_7.svg">
NW_025953044 (772845-804021): Daldinia caldariorum strain CBS 122874 unplaced... (21% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r8c1_8.svg">
NW_006917094 (1335307-1366507): Pestalotiopsis fici W106-1 unplaced genomic s... (18% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r8c1_9.svg">
NC_064678 (2945413-2977179): Colletotrichum lupini chromosome 5, complete seq... (30% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r8c1_10.svg">
NW_020194475 (1665213-1696396): Amorphotheca resinae ATCC 22711 unplaced geno... (16% of genes show similarity), terpene
</option>

</select>
<button id= "clusterblast-r8c1-download">Download graphic</button>
</div>
<div id="clusterblast-r8c1-svg"></div>

</div>
</div>

<div class="body-details-section r8c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-124"></div> <span class="help-tooltip" id="cb-known-help-124">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r8c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-125"></div> <span class="help-tooltip" id="cb-sub-help-125">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r8c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-127"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-127">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r8c1-pfam-details-svg" data-filename="scaffold_8_r8c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r8c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">









</div>









</div>

</div>

</div>

<div class="page" id='r8c2' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_8 - Region 2 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-747"></div> <span class="help-tooltip" id="region_svg-help-747">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_8.region002.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r8c2-svg" data-filename="scaffold_8_r8c2.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 498,237 - 545,506 nt. (total: 47,270 nt)

<a class="cluster-rules-header" id="r8c2-rules-header" href="#r8c2">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r8c2-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r8c2-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-748"></div> <span class="help-tooltip" id="focus-panel-help-748">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r8c2">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r8c2-gene-table-details" data-name="r8c2-gene-table-details">
Gene overview
</div>



<div class="body-details-header r8c2-nrps_pks" data-name="r8c2-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r8c2-MIBiG-cluster-compare" data-name="r8c2-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r8c2-clusterblast" data-name="r8c2-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r8c2-knownclusterblast" data-name="r8c2-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r8c2-subclusterblast" data-name="r8c2-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r8c2-nrps-pks-bubbles" data-name="r8c2-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r8c2-pfam-details" data-name="r8c2-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r8c2-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-136"></div> <span class="help-tooltip" id="gt-help-tooltip-help-136">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r8c2">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r8c2-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r8c2-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r8c2">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r8c2-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-128"></div> <span class="help-tooltip" id="nrps-domain-help-128">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r8c2-details-svg" data-filename="scaffold_8_r8c2_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r8c2-details-svg"></div>
</div>
</div>



<div class="body-details-section r8c2-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-129"></div> <span class="help-tooltip" id="MIBiG-help-129">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r8c2-selector">Analysis type:</label>
<select id="comparison-MIBiG-r8c2-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r8c2"></div>
<div id="comparison-MIBiG-r8c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002228: 282-17978">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002228/index.html#r1c1" target="_new">BGC0002228</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.28</td>
<td>NRP</td>

<td>CIML B, CIML A, CIML D, CIML C</td>


<td>Colletotrichum incanum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001124: 3228-30082">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001124/index.html#r1c1" target="_new">BGC0001124</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.26</td>
<td>Polyketide</td>

<td>pyranonigrin E</td>


<td>Aspergillus niger ATCC 1015</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002227: 270-18409">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002227/index.html#r1c1" target="_new">BGC0002227</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.43, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.25</td>
<td>NRP</td>

<td>AKML B, AKML D, AKML A, AKML C</td>


<td>Aspergillus luchuensis IFO 4308</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001913: 7717-40102">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001913/index.html#r1c1" target="_new">BGC0001913</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>Polyketide</td>

<td>phomoidride</td>


<td>fungal sp. ATCC 74256</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002211: 74-16330">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002211/index.html#r1c1" target="_new">BGC0002211</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>Polyketide</td>

<td>HEx-pks1 polyketide</td>


<td>Paraphaeosphaeria sporulosa</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001340: 1840-51432">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001340/index.html#r1c1" target="_new">BGC0001340</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.45, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Polyketide</td>

<td>byssochlamic acid</td>


<td>Byssochlamys fulva</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001749: 322-54057">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001749/index.html#r1c1" target="_new">BGC0001749</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Polyketide</td>

<td>epipyriculol</td>


<td>Pyricularia oryzae 70-15</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000046: 1267-21753">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000046/index.html#r1c1" target="_new">BGC0000046</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.42, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Polyketide</td>

<td>depudecin</td>


<td>Alternaria brassicicola</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002233: 556-31132">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002233/index.html#r1c1" target="_new">BGC0002233</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Polyketide</td>

<td>trichoxide</td>


<td>Trichoderma virens Gv29-8</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001998: 0-14131">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001998/index.html#r1c1" target="_new">BGC0001998</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Polyketide</td>

<td>aspernidgulene A1, aspernidgulene A2, aspernidgulene B1</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r8c2-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002228: 282-17978">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002228/index.html#r1c1" target="_new">BGC0002228</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>NRP</td>
<td>CIML B, CIML A, CIML D, CIML C</td>
<td>Colletotrichum incanum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001124: 3228-30082">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001124/index.html#r1c1" target="_new">BGC0001124</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>Polyketide</td>
<td>pyranonigrin E</td>
<td>Aspergillus niger ATCC 1015</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002227: 270-18409">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002227/index.html#r1c1" target="_new">BGC0002227</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.43, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>NRP</td>
<td>AKML B, AKML D, AKML A, AKML C</td>
<td>Aspergillus luchuensis IFO 4308</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001913: 7717-40102">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001913/index.html#r1c1" target="_new">BGC0001913</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Polyketide</td>
<td>phomoidride</td>
<td>fungal sp. ATCC 74256</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002211: 74-16330">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002211/index.html#r1c1" target="_new">BGC0002211</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Polyketide</td>
<td>HEx-pks1 polyketide</td>
<td>Paraphaeosphaeria sporulosa</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001340: 1840-51432">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001340/index.html#r1c1" target="_new">BGC0001340</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.45, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Polyketide</td>
<td>byssochlamic acid</td>
<td>Byssochlamys fulva</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001749: 322-54057">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001749/index.html#r1c1" target="_new">BGC0001749</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Polyketide</td>
<td>epipyriculol</td>
<td>Pyricularia oryzae 70-15</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000046: 1267-21753">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000046/index.html#r1c1" target="_new">BGC0000046</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.42, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Polyketide</td>
<td>depudecin</td>
<td>Alternaria brassicicola</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002233: 556-31132">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002233/index.html#r1c1" target="_new">BGC0002233</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Polyketide</td>
<td>trichoxide</td>
<td>Trichoderma virens Gv29-8</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001998: 0-14131">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001998/index.html#r1c1" target="_new">BGC0001998</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Polyketide</td>
<td>aspernidgulene A1, aspernidgulene A2, aspernidgulene B1</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r8c2-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-130"></div> <span class="help-tooltip" id="cb-general-help-130">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r8c2-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r8c2_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r8c2_1.svg">
NW_025953815 (268762-331510): Annulohypoxylon truncatum strain CBS 140777 unp... (21% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r8c2_2.svg">
NC_054854 (855340-923059): Aspergillus luchuensis IFO 4308 DNA, chromosome 6,... (7% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r8c2_3.svg">
NW_024467308 (148929-260597): Aspergillus ruber CBS 135680 unplaced genomic s... (4% of genes show similarity), T1PKS,indole
</option>

<option value = "svg/clusterblast_r8c2_4.svg">
NC_064678 (6729000-6796682): Colletotrichum lupini chromosome 5, complete seq... (11% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r8c2_5.svg">
NW_020291494 (375619-484726): Aspergillus aculeatinus CBS 121060 unplaced gen... (7% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r8c2_6.svg">
NW_020291597 (473335-541498): Aspergillus piperis CBS 112811 unplaced genomic... (7% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r8c2_7.svg">
NW_020292057 (672652-740811): Aspergillus vadensis CBS 113365 unplaced genomi... (7% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r8c2_8.svg">
NW_020291801 (1121120-1189303): Aspergillus costaricaensis CBS 115574 unplace... (7% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r8c2_9.svg">
NW_006711148 (452235-529083): Trichoderma reesei QM6a unplaced genomic scaffo... (9% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r8c2_10.svg">
NW_020292041 (545254-624399): Aspergillus vadensis CBS 113365 unplaced genomi... (6% of genes show similarity), NRPS,T1PKS
</option>

</select>
<button id= "clusterblast-r8c2-download">Download graphic</button>
</div>
<div id="clusterblast-r8c2-svg"></div>

</div>
</div>

<div class="body-details-section r8c2-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-131"></div> <span class="help-tooltip" id="cb-known-help-131">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r8c2-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-132"></div> <span class="help-tooltip" id="cb-sub-help-132">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r8c2-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-135"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-135">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r8c2-domain-bubble-svg" data-filename="scaffold_8_r8c2_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r8c2-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC2">
Candidate 2 (498236 - 545506): single T1PKS
</option>


</select>

<div class="details-svg" id="r8c2-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r8c2-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r8c2-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-137"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-137">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r8c2-pfam-details-svg" data-filename="scaffold_8_r8c2_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r8c2-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r8c2-nrps_pks_products" data-name="r8c2-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r8c2-nrps_pks_monomers" data-name="r8c2-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r8c2-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-133"></div> <span class="help-tooltip" id="nrps-structures-help-133">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 2, location 498236 - 545506:
<div class="collapser collapser-target-CC2 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r8c2-2" class="smiles-canvas" data-smiles="C([*])C(-O)"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS linker matching; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(pk)</dd>

<dt><b>Non-colinear ordering:</b></dt><dd>

<span class="serif">FUN_002270</span>

+ <span class="serif">FUN_002271</span>

</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r8c2-2-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r8c2-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-134"></div> <span class="help-tooltip" id="nrps-monomers-help-134">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">FUN_002270</span></strong>: pk
<div class="collapser collapser-target-FUN_002270 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (491..815)</strong>: pk
<div class="collapser collapser-target-FUN_002270 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-FUN_002270-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 79.2%</dd>
<dd>Methylmalonyl-CoA: 62.5%</dd>
<dd>Isobutyryl-CoA: 58.3%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-FUN_002270-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 81.1</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 73.2</dt>

<dd></dd><dt>Malonyl-CoA: 65.7</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 43.9</dt>

<dd></dd><dt>Propionyl-CoA: 41.8</dt>

<dd></dd><dt>Isobutyryl-CoA: 38.8</dt>

<dd></dd><dt>trans-1,2-CPDA: 28.2</dt>

<dd></dd><dt>fatty_acid: 25.5</dt>

<dd></dd><dt>inactive: 21.9</dt>

<dd></dd><dt>CHC-CoA: 20.5</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 18.2</dt>

<dd></dd><dt>Benzoyl-CoA: 11.6</dt>

<dd></dd><dt>Acetyl-CoA: 0.0</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>








</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r8c3' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_8 - Region 3 - T1PKS,fungal-RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-749"></div> <span class="help-tooltip" id="region_svg-help-749">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_8.region003.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r8c3-svg" data-filename="scaffold_8_r8c3.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 659,102 - 756,482 nt. (total: 97,381 nt)

<a class="cluster-rules-header" id="r8c3-rules-header" href="#r8c3">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r8c3-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))<br>fungal-RiPP-like: DUF3328</div>

<div class="region-svg-container">
<div id='r8c3-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-750"></div> <span class="help-tooltip" id="focus-panel-help-750">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r8c3">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r8c3-gene-table-details" data-name="r8c3-gene-table-details">
Gene overview
</div>



<div class="body-details-header r8c3-nrps_pks" data-name="r8c3-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r8c3-MIBiG-cluster-compare" data-name="r8c3-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r8c3-clusterblast" data-name="r8c3-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r8c3-knownclusterblast" data-name="r8c3-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r8c3-subclusterblast" data-name="r8c3-subclusterblast">
SubClusterBlast
</div>







<div class="body-details-header r8c3-nrps-pks-bubbles" data-name="r8c3-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r8c3-pfam-details" data-name="r8c3-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r8c3-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-146"></div> <span class="help-tooltip" id="gt-help-tooltip-help-146">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r8c3">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r8c3-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r8c3-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r8c3">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r8c3-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-138"></div> <span class="help-tooltip" id="nrps-domain-help-138">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r8c3-details-svg" data-filename="scaffold_8_r8c3_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r8c3-details-svg"></div>
</div>
</div>



<div class="body-details-section r8c3-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-139"></div> <span class="help-tooltip" id="MIBiG-help-139">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r8c3-selector">Analysis type:</label>
<select id="comparison-MIBiG-r8c3-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r8c3"></div>
<div id="comparison-MIBiG-r8c3-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>

<th><span class="cc-heat-product">fungal-RiPP-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001273: 0-5301">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001273/index.html#r1c1" target="_new">BGC0001273</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.41, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>


<td>0.28</td>
<td>Polyketide</td>

<td>asperlactone</td>


<td>Aspergillus ochraceus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001068: 0-35628">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001068/index.html#r1c1" target="_new">BGC0001068</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>


<td>0.26</td>
<td>Terpene, Polyketide</td>

<td>pyripyropene A</td>


<td>unidentified unclassified sequences.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000043: 0-13097">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000043/index.html#r1c1" target="_new">BGC0000043</a></td>





<td class="cc-heat-cell" title="0.36 (id:0.43, order:N/A, components:0.30)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>






<td class="cc-heat-cell" title="0.38 (id:0.42, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>


<td>0.21</td>
<td>Polyketide</td>

<td>curacin A</td>


<td>Moorea producens 19L</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001954: 0-19896">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001954/index.html#r1c1" target="_new">BGC0001954</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.43, order:N/A, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>


<td>0.20</td>
<td>NRP, Polyketide</td>

<td>wortmanamide A, wortmanamide B</td>


<td>Talaromyces wortmannii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002155: 78-16511">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002155/index.html#r1c1" target="_new">BGC0002155</a></td>





<td class="cc-heat-cell" title="0.51 (id:0.43, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>


<td>0.19</td>
<td>Polyketide</td>

<td>nectriapyrone C, nectriapyrone D, nectriapyrone</td>


<td>Pyricularia oryzae 70-15</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001254: 82-7456">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001254/index.html#r1c1" target="_new">BGC0001254</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>


<td>0.18</td>
<td>Polyketide</td>

<td>ACT-Toxin II</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002191: 260-8785">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002191/index.html#r1c1" target="_new">BGC0002191</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>


<td>0.18</td>
<td>Polyketide</td>

<td>prolipyrone B, gibepyrone D</td>


<td>Fusarium graminearum PH-1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000996: 0-16575">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000996/index.html#r1c1" target="_new">BGC0000996</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.59)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>


<td>0.18</td>
<td>NRP, Polyketide</td>

<td>frontalamide B, frontalamide A</td>


<td>Streptomyces sp. SPB78</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002240: 96-11225">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002240/index.html#r1c1" target="_new">BGC0002240</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>


<td>0.18</td>
<td>Polyketide</td>

<td>BAB, BAA</td>


<td>Metarhizium anisopliae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001998: 0-14131">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001998/index.html#r1c1" target="_new">BGC0001998</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>


<td>0.18</td>
<td>Polyketide</td>

<td>aspernidgulene A1, aspernidgulene A2, aspernidgulene B1</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r8c3-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001273: 0-5301">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001273/index.html#r1c1" target="_new">BGC0001273</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.41, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>Polyketide</td>
<td>asperlactone</td>
<td>Aspergillus ochraceus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001068: 0-35628">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001068/index.html#r1c1" target="_new">BGC0001068</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>Terpene, Polyketide</td>
<td>pyripyropene A</td>
<td>unidentified unclassified sequences.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001954: 0-19896">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001954/index.html#r1c1" target="_new">BGC0001954</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.43, order:N/A, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>NRP, Polyketide</td>
<td>wortmanamide A, wortmanamide B</td>
<td>Talaromyces wortmannii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002155: 78-16511">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002155/index.html#r1c1" target="_new">BGC0002155</a></td>


<td class="cc-heat-cell" title="0.51 (id:0.43, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
<td>0.51</td>
<td>Polyketide</td>
<td>nectriapyrone C, nectriapyrone D, nectriapyrone</td>
<td>Pyricularia oryzae 70-15</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001254: 82-7456">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001254/index.html#r1c1" target="_new">BGC0001254</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Polyketide</td>
<td>ACT-Toxin II</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002191: 260-8785">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002191/index.html#r1c1" target="_new">BGC0002191</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Polyketide</td>
<td>prolipyrone B, gibepyrone D</td>
<td>Fusarium graminearum PH-1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000996: 0-16575">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000996/index.html#r1c1" target="_new">BGC0000996</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.59)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>NRP, Polyketide</td>
<td>frontalamide B, frontalamide A</td>
<td>Streptomyces sp. SPB78</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002240: 96-11225">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002240/index.html#r1c1" target="_new">BGC0002240</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Polyketide</td>
<td>BAB, BAA</td>
<td>Metarhizium anisopliae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001998: 0-14131">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001998/index.html#r1c1" target="_new">BGC0001998</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Polyketide</td>
<td>aspernidgulene A1, aspernidgulene A2, aspernidgulene B1</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001400: 30-15436">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001400/index.html#r1c1" target="_new">BGC0001400</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Polyketide</td>
<td>citreoviridin</td>
<td>Aspergillus terreus NIH2624</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r8c3-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-140"></div> <span class="help-tooltip" id="cb-general-help-140">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r8c3-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r8c3_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r8c3_1.svg">
NC_072355 (9859-77137): Talaromyces marneffei chromosome 8, complete sequence (17% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r8c3_2.svg">
NW_017971443 (180850-345227): Talaromyces atroroseus strain IBT 11181 chromos... (4% of genes show similarity), NRPS-like,T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r8c3_3.svg">
NW_019154045 (358401-688746): Pochonia chlamydosporia 170 chromosome Unknown ... (2% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r8c3_4.svg">
NW_022985443 (1533-77825): Mytilinidion resinicola strain CBS 304.34 unplaced... (9% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r8c3_5.svg">
NW_021167080 (4131403-4218796): Sodiomyces alkalinus F11 unplaced genomic sca... (14% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r8c3_6.svg">
NW_025091264 (5340181-5432359): Aspergillus udagawae IFM 46973 DNA, scaffold ... (10% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r8c3_7.svg">
NW_025091217 (15487-82066): Aspergillus viridinutans IFM 47045 DNA, scaffold ... (25% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r8c3_8.svg">
NW_026054642 (296356-363048): Ophidiomyces ophidiicola strain CBS 122913 chro... (10% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r8c3_9.svg">
NW_003052500 (3318567-3442588): Uncinocarpus reesii 1704 scaffold 1 genomic s... (7% of genes show similarity), NRP-metallophore,NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r8c3_10.svg">
NW_025091286 (2424447-2527542): Aspergillus pseudoviridinutans IFM 55266 DNA,... (7% of genes show similarity), NRPS,T1PKS,other
</option>

</select>
<button id= "clusterblast-r8c3-download">Download graphic</button>
</div>
<div id="clusterblast-r8c3-svg"></div>

</div>
</div>

<div class="body-details-section r8c3-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-141"></div> <span class="help-tooltip" id="cb-known-help-141">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r8c3-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-142"></div> <span class="help-tooltip" id="cb-sub-help-142">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>







<div class="body-details-section r8c3-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-145"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-145">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r8c3-domain-bubble-svg" data-filename="scaffold_8_r8c3_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r8c3-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC3">
Candidate 3 (659101 - 756482): neighbouring T1PKS-fungal-RiPP-like
</option>



<option value="CC4">
Candidate 4 (659101 - 705725): single T1PKS
</option>


</select>

<div class="details-svg" id="r8c3-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r8c3-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r8c3-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-147"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-147">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r8c3-pfam-details-svg" data-filename="scaffold_8_r8c3_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r8c3-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r8c3-nrps_pks_products" data-name="r8c3-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r8c3-nrps_pks_monomers" data-name="r8c3-nrps_pks_monomers">
NRPS/PKS substrates
</div>








</div>










<div class="sidepanel-details-section r8c3-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-143"></div> <span class="help-tooltip" id="nrps-structures-help-143">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 3, location 659101 - 756482:
<div class="collapser collapser-target-CC3 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r8c3-3" class="smiles-canvas" data-smiles="C([*])C(-O)"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(pk)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r8c3-3-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 4, location 659101 - 705725:
<div class="collapser collapser-target-CC4 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r8c3-4" class="smiles-canvas" data-smiles="C([*])C(-O)"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(pk)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r8c3-4-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r8c3-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-144"></div> <span class="help-tooltip" id="nrps-monomers-help-144">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">FUN_002321</span></strong>: pk
<div class="collapser collapser-target-FUN_002321 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (461..773)</strong>: pk
<div class="collapser collapser-target-FUN_002321 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-FUN_002321-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 79.2%</dd>
<dd>Isobutyryl-CoA: 66.7%</dd>
<dd>Methylmalonyl-CoA: 66.7%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-FUN_002321-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 68.3</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 49.6</dt>

<dd></dd><dt>Malonyl-CoA: 42.1</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 32.4</dt>

<dd></dd><dt>Isobutyryl-CoA: 22.5</dt>

<dd></dd><dt>Propionyl-CoA: 21.5</dt>

<dd></dd><dt>Benzoyl-CoA: 19.7</dt>

<dd></dd><dt>fatty_acid: 14.6</dt>

<dd></dd><dt>CHC-CoA: 11.0</dt>

<dd></dd><dt>inactive: 10.2</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 8.7</dt>

<dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

<dd></dd><dt>Acetyl-CoA: 0.0</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>








</div>
<br>

</dl>
</div>
</div>








</div>

</div>

</div>





<div class="page" id='r9c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_9 - Region 1 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-751"></div> <span class="help-tooltip" id="region_svg-help-751">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_9.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r9c1-svg" data-filename="scaffold_9_r9c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 6,961 - 53,915 nt. (total: 46,955 nt)

<a class="cluster-rules-header" id="r9c1-rules-header" href="#r9c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r9c1-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r9c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-752"></div> <span class="help-tooltip" id="focus-panel-help-752">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r9c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r9c1-gene-table-details" data-name="r9c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r9c1-nrps_pks" data-name="r9c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r9c1-MIBiG-cluster-compare" data-name="r9c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r9c1-clusterblast" data-name="r9c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r9c1-knownclusterblast" data-name="r9c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r9c1-subclusterblast" data-name="r9c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r9c1-nrps-pks-bubbles" data-name="r9c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r9c1-pfam-details" data-name="r9c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r9c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-156"></div> <span class="help-tooltip" id="gt-help-tooltip-help-156">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r9c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r9c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r9c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r9c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r9c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-148"></div> <span class="help-tooltip" id="nrps-domain-help-148">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r9c1-details-svg" data-filename="scaffold_9_r9c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r9c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r9c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-149"></div> <span class="help-tooltip" id="MIBiG-help-149">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r9c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r9c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r9c1"></div>
<div id="comparison-MIBiG-r9c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001990: 130-13646">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001990/index.html#r1c1" target="_new">BGC0001990</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.34</td>
<td>Other</td>

<td>BU-4704, xanthocillin-X monomethylether, melanocin E, melanocin F, fumiformamide</td>


<td>Aspergillus fumigatus Af293</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000037: 0-7821">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000037/index.html#r1c1" target="_new">BGC0000037</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.33</td>
<td>Polyketide</td>

<td>cichorine</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264: 8442-40380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.29</td>
<td>Polyketide</td>

<td>betaenone A, betaenone B, betaenone C</td>


<td>Phoma betae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000812: 502-25834">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000812/index.html#r1c1" target="_new">BGC0000812</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.24</td>
<td>Alkaloid</td>

<td>huperzine A</td>


<td>Shiraia sp. slf14</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001720: 0-45622">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001720/index.html#r1c1" target="_new">BGC0001720</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Polyketide</td>

<td>oosporein</td>


<td>Beauveria bassiana ARSEF 2860</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001893: 155-19434">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001893/index.html#r1c1" target="_new">BGC0001893</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Other</td>

<td>abscisic acid</td>


<td>Botrytis cinerea B05.10</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001390: 100-19645">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001390/index.html#r1c1" target="_new">BGC0001390</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.45, order:0.52, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>NRP, Polyketide</td>

<td>LL-Z1272beta</td>


<td>Stachybotrys bisbyi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002219: 0-26397">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002219/index.html#r1c1" target="_new">BGC0002219</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Terpene</td>

<td>(-)-(R)-nephthenol</td>


<td>Stereum hirsutum FP-91666 SS1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001262: 1986-31560">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001262/index.html#r1c1" target="_new">BGC0001262</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.41, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Polyketide</td>

<td>AK-toxin I</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000056: 2-6689">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000056/index.html#r1c1" target="_new">BGC0000056</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.41, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Polyketide</td>

<td>esperamicin</td>


<td>Actinomadura verrucosospora</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r9c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001990: 130-13646">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001990/index.html#r1c1" target="_new">BGC0001990</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Other</td>
<td>BU-4704, xanthocillin-X monomethylether, melanocin E, melanocin F, fumiformamide</td>
<td>Aspergillus fumigatus Af293</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000037: 0-7821">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000037/index.html#r1c1" target="_new">BGC0000037</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Polyketide</td>
<td>cichorine</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264: 8442-40380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>Polyketide</td>
<td>betaenone A, betaenone B, betaenone C</td>
<td>Phoma betae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000812: 502-25834">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000812/index.html#r1c1" target="_new">BGC0000812</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Alkaloid</td>
<td>huperzine A</td>
<td>Shiraia sp. slf14</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001720: 0-45622">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001720/index.html#r1c1" target="_new">BGC0001720</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Polyketide</td>
<td>oosporein</td>
<td>Beauveria bassiana ARSEF 2860</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001893: 155-19434">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001893/index.html#r1c1" target="_new">BGC0001893</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Other</td>
<td>abscisic acid</td>
<td>Botrytis cinerea B05.10</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001390: 100-19645">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001390/index.html#r1c1" target="_new">BGC0001390</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.45, order:0.52, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>NRP, Polyketide</td>
<td>LL-Z1272beta</td>
<td>Stachybotrys bisbyi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002219: 0-26397">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002219/index.html#r1c1" target="_new">BGC0002219</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Terpene</td>
<td>(-)-(R)-nephthenol</td>
<td>Stereum hirsutum FP-91666 SS1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001262: 1986-31560">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001262/index.html#r1c1" target="_new">BGC0001262</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.41, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Polyketide</td>
<td>AK-toxin I</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000056: 2-6689">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000056/index.html#r1c1" target="_new">BGC0000056</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.41, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Polyketide</td>
<td>esperamicin</td>
<td>Actinomadura verrucosospora</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r9c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-150"></div> <span class="help-tooltip" id="cb-general-help-150">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r9c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r9c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r9c1_1.svg">
NT_165982 (1674335-1763165): Chaetomium globosum CBS 148.51 scaffold 7 genomi... (17% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r9c1_2.svg">
NC_063023 (2204101-2274338): Fulvia fulva chromosome 12, complete sequence (16% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r9c1_3.svg">
NW_009276941 (347077-418070): Verticillium dahliae VdLs.17 supercont1.26 geno... (17% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r9c1_4.svg">
NW_025091283 (4319746-4671991): Aspergillus pseudoviridinutans IFM 55266 DNA,... (6% of genes show similarity), NRPS,NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r9c1_5.svg">
NC_072349 (290456-369836): Talaromyces marneffei chromosome 2, complete sequence (20% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r9c1_6.svg">
NC_066257 (109084-244310): Aspergillus nidulans FGSC A4 chromosome I (11% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r9c1_7.svg">
NC_064677 (5429112-5558494): Colletotrichum lupini chromosome 4, complete seq... (28% of genes show similarity), NRPS-like,T1PKS,indole,terpene
</option>

<option value = "svg/clusterblast_r9c1_8.svg">
NC_049561 (857603-965575): Talaromyces rugulosus chromosome I, complete sequence (8% of genes show similarity), NRPS-like,T1PKS,terpene
</option>

<option value = "svg/clusterblast_r9c1_9.svg">
NW_006917091 (1121274-1194772): Pestalotiopsis fici W106-1 unplaced genomic s... (14% of genes show similarity), NRPS-like,T1PKS,indole
</option>

<option value = "svg/clusterblast_r9c1_10.svg">
NW_006711153 (4745-99477): Trichoderma reesei QM6a unplaced genomic scaffold ... (7% of genes show similarity), NRPS,NRPS-like,T1PKS
</option>

</select>
<button id= "clusterblast-r9c1-download">Download graphic</button>
</div>
<div id="clusterblast-r9c1-svg"></div>

</div>
</div>

<div class="body-details-section r9c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-151"></div> <span class="help-tooltip" id="cb-known-help-151">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r9c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r9c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r9c1_1.svg">
2,4-dihydroxy-5,6-dimethyl benzaldehyde
</option>

</select>
<button id="knownclusterblast-r9c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r9c1-svg"></div>

</div>
</div>

<div class="body-details-section r9c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-152"></div> <span class="help-tooltip" id="cb-sub-help-152">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r9c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-155"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-155">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r9c1-domain-bubble-svg" data-filename="scaffold_9_r9c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r9c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (6960 - 53915): single T1PKS
</option>


</select>

<div class="details-svg" id="r9c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r9c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r9c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-157"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-157">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r9c1-pfam-details-svg" data-filename="scaffold_9_r9c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r9c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r9c1-nrps_pks_products" data-name="r9c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r9c1-nrps_pks_monomers" data-name="r9c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r9c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-153"></div> <span class="help-tooltip" id="nrps-structures-help-153">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 6960 - 53915:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r9c1-1" class="smiles-canvas" data-smiles="C([*])C(-O)"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(pk)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r9c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r9c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-154"></div> <span class="help-tooltip" id="nrps-monomers-help-154">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">FUN_002387</span></strong>: pk
<div class="collapser collapser-target-FUN_002387 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (950..1238)</strong>: pk
<div class="collapser collapser-target-FUN_002387 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-FUN_002387-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 58.3%</dd>
</dl>

</div><br>

minowa:  Methoxymalonyl-CoA
<div class="collapser collapser-target-FUN_002387-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methoxymalonyl-CoA: 48.0</dt>

<dd></dd><dt>Malonyl-CoA: 36.6</dt>

<dd></dd><dt>Methylmalonyl-CoA: 30.7</dt>

<dd></dd><dt>inactive: 20.7</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 20.0</dt>

<dd></dd><dt>Propionyl-CoA: 18.6</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 17.9</dt>

<dd></dd><dt>CHC-CoA: 16.7</dt>

<dd></dd><dt>trans-1,2-CPDA: 12.2</dt>

<dd></dd><dt>fatty_acid: 9.2</dt>

<dd></dd><dt>Benzoyl-CoA: 9.2</dt>

<dd></dd><dt>Isobutyryl-CoA: 0.0</dt>

<dd></dd><dt>Acetyl-CoA: 0.0</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>








</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>





<div class="page" id='r10c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_10 - Region 1 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-753"></div> <span class="help-tooltip" id="region_svg-help-753">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_10.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r10c1-svg" data-filename="scaffold_10_r10c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 180,315 - 228,285 nt. (total: 47,971 nt)

<a class="cluster-rules-header" id="r10c1-rules-header" href="#r10c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r10c1-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r10c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-754"></div> <span class="help-tooltip" id="focus-panel-help-754">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r10c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r10c1-gene-table-details" data-name="r10c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r10c1-nrps_pks" data-name="r10c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r10c1-MIBiG-cluster-compare" data-name="r10c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r10c1-clusterblast" data-name="r10c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r10c1-knownclusterblast" data-name="r10c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r10c1-subclusterblast" data-name="r10c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r10c1-nrps-pks-bubbles" data-name="r10c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r10c1-pfam-details" data-name="r10c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r10c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-166"></div> <span class="help-tooltip" id="gt-help-tooltip-help-166">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r10c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r10c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r10c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r10c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r10c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-158"></div> <span class="help-tooltip" id="nrps-domain-help-158">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r10c1-details-svg" data-filename="scaffold_10_r10c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r10c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r10c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-159"></div> <span class="help-tooltip" id="MIBiG-help-159">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r10c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r10c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r10c1"></div>
<div id="comparison-MIBiG-r10c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002155: 78-16511">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002155/index.html#r1c1" target="_new">BGC0002155</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.33</td>
<td>Polyketide</td>

<td>nectriapyrone C, nectriapyrone D, nectriapyrone</td>


<td>Pyricularia oryzae 70-15</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002191: 260-8785">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002191/index.html#r1c1" target="_new">BGC0002191</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.32</td>
<td>Polyketide</td>

<td>prolipyrone B, gibepyrone D</td>


<td>Fusarium graminearum PH-1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001252: 0-7958">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001252/index.html#r1c1" target="_new">BGC0001252</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.42, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.30</td>
<td>Polyketide</td>

<td>UNII-YC2Q1O94PT</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002525: 1000-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002525/index.html#r1c1" target="_new">BGC0002525</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.30</td>
<td>Polyketide</td>

<td>fusarubin, 1233A, 1233B, NG-391, lucilactaene</td>


<td>Fusarium sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002194: 147-15389">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002194/index.html#r1c1" target="_new">BGC0002194</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.82)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.27</td>
<td>Polyketide</td>

<td>epipyrone A</td>


<td>Epicoccum nigrum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002515: 0-30087">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002515/index.html#r1c1" target="_new">BGC0002515</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.42, order:N/A, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.27</td>
<td>Polyketide</td>

<td>solanapyrone A</td>


<td>Ascochyta rabiei</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002224: 685-37748">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002224/index.html#r1c1" target="_new">BGC0002224</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.45, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.27</td>
<td>Polyketide</td>

<td>zopfiellin</td>


<td>Diffractella curvata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002223: 2980-46085">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002223/index.html#r1c1" target="_new">BGC0002223</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.25</td>
<td>Polyketide</td>

<td>scytalidin</td>


<td>Scytalidium album</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002222: 888-32276">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002222/index.html#r1c1" target="_new">BGC0002222</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.45, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.25</td>
<td>Polyketide</td>

<td>zopfiellin</td>


<td>Diffractella curvata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001899: 0-54810">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001899/index.html#r1c1" target="_new">BGC0001899</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.25</td>
<td>Polyketide</td>

<td>abscisic acid</td>


<td>Leptosphaeria maculans JN3</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r10c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002155: 78-16511">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002155/index.html#r1c1" target="_new">BGC0002155</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Polyketide</td>
<td>nectriapyrone C, nectriapyrone D, nectriapyrone</td>
<td>Pyricularia oryzae 70-15</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002191: 260-8785">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002191/index.html#r1c1" target="_new">BGC0002191</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Polyketide</td>
<td>prolipyrone B, gibepyrone D</td>
<td>Fusarium graminearum PH-1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001252: 0-7958">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001252/index.html#r1c1" target="_new">BGC0001252</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.42, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>Polyketide</td>
<td>UNII-YC2Q1O94PT</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002525: 1000-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002525/index.html#r1c1" target="_new">BGC0002525</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>Polyketide</td>
<td>fusarubin, 1233A, 1233B, NG-391, lucilactaene</td>
<td>Fusarium sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002194: 147-15389">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002194/index.html#r1c1" target="_new">BGC0002194</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.82)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>Polyketide</td>
<td>epipyrone A</td>
<td>Epicoccum nigrum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002515: 0-30087">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002515/index.html#r1c1" target="_new">BGC0002515</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.42, order:N/A, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>Polyketide</td>
<td>solanapyrone A</td>
<td>Ascochyta rabiei</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002224: 685-37748">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002224/index.html#r1c1" target="_new">BGC0002224</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.45, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>Polyketide</td>
<td>zopfiellin</td>
<td>Diffractella curvata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002223: 2980-46085">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002223/index.html#r1c1" target="_new">BGC0002223</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Polyketide</td>
<td>scytalidin</td>
<td>Scytalidium album</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002222: 888-32276">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002222/index.html#r1c1" target="_new">BGC0002222</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.45, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Polyketide</td>
<td>zopfiellin</td>
<td>Diffractella curvata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001899: 0-54810">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001899/index.html#r1c1" target="_new">BGC0001899</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Polyketide</td>
<td>abscisic acid</td>
<td>Leptosphaeria maculans JN3</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r10c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-160"></div> <span class="help-tooltip" id="cb-general-help-160">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r10c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r10c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r10c1_1.svg">
NW_024468935 (6766227-6834435): Cucurbitaria berberidis CBS 394.84 unplaced g... (14% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r10c1_2.svg">
NW_017306203 (257647-325908): Alternaria alternata strain SRC1lrK2f unplaced ... (15% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r10c1_3.svg">
NW_026250029 (279519-347509): Hypoxylon trugodes strain CBS 135444 unplaced g... (20% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r10c1_4.svg">
NW_026249867 (161946-230009): Alternaria incomplexa strain BMP 0042 chromosom... (12% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r10c1_5.svg">
NW_026055700 (31065-99128): Alternaria ventricosa strain BMP 2768 chromosome ... (12% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r10c1_6.svg">
NW_025408668 (1200492-1268825): Hirsutella rhossiliensis strain HR02 chromoso... (16% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r10c1_7.svg">
NW_023336280 (65196-169882): Aspergillus tubingensis WU-2223L DNA, scaffold 6... (5% of genes show similarity), NRPS,T1PKS,terpene
</option>

<option value = "svg/clusterblast_r10c1_8.svg">
NW_020291608 (94205-186802): Aspergillus piperis CBS 112811 unplaced genomic ... (6% of genes show similarity), T1PKS,terpene
</option>

<option value = "svg/clusterblast_r10c1_9.svg">
NW_020290894 (433814-525041): Aspergillus niger CBS 101883 unplaced genomic s... (7% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r10c1_10.svg">
NW_020291836 (40952-145610): Aspergillus costaricaensis CBS 115574 unplaced g... (5% of genes show similarity), NRPS,T1PKS,terpene
</option>

</select>
<button id= "clusterblast-r10c1-download">Download graphic</button>
</div>
<div id="clusterblast-r10c1-svg"></div>

</div>
</div>

<div class="body-details-section r10c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-161"></div> <span class="help-tooltip" id="cb-known-help-161">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r10c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-162"></div> <span class="help-tooltip" id="cb-sub-help-162">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r10c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-165"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-165">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r10c1-domain-bubble-svg" data-filename="scaffold_10_r10c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r10c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (180314 - 228285): single T1PKS
</option>


</select>

<div class="details-svg" id="r10c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r10c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r10c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-167"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-167">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r10c1-pfam-details-svg" data-filename="scaffold_10_r10c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r10c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r10c1-nrps_pks_products" data-name="r10c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r10c1-nrps_pks_monomers" data-name="r10c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r10c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-163"></div> <span class="help-tooltip" id="nrps-structures-help-163">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 180314 - 228285:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r10c1-1" class="smiles-canvas" data-smiles="C(C)CC(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Me-redmal)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-redmal" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-redmal*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r10c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C(C)CC(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r10c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-164"></div> <span class="help-tooltip" id="nrps-monomers-help-164">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">FUN_002727</span></strong>: mal
<div class="collapser collapser-target-FUN_002727 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (448..749)</strong>: mal
<div class="collapser collapser-target-FUN_002727 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-FUN_002727-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 66.7%</dd>
<dd>Methylmalonyl-CoA: 54.2%</dd>
</dl>

</div><br>

minowa:  Malonyl-CoA
<div class="collapser collapser-target-FUN_002727-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Malonyl-CoA: 98.4</dt>

<dd></dd><dt>Methylmalonyl-CoA: 71.4</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 71.1</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 43.0</dt>

<dd></dd><dt>Isobutyryl-CoA: 35.2</dt>

<dd></dd><dt>inactive: 32.6</dt>

<dd></dd><dt>trans-1,2-CPDA: 25.3</dt>

<dd></dd><dt>Propionyl-CoA: 22.3</dt>

<dd></dd><dt>fatty_acid: 17.9</dt>

<dd></dd><dt>CHC-CoA: 16.7</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 13.1</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 12.9</dt>

<dd></dd><dt>Benzoyl-CoA: 12.3</dt>

<dd></dd><dt>Acetyl-CoA: 6.8</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>












</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r10c2' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_10 - Region 2 - NRPS,T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-755"></div> <span class="help-tooltip" id="region_svg-help-755">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_10.region002.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r10c2-svg" data-filename="scaffold_10_r10c2.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 827,589 - 879,905 nt. (total: 52,317 nt)

<a class="cluster-rules-header" id="r10c2-rules-header" href="#r10c2">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r10c2-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))<br>T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r10c2-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-756"></div> <span class="help-tooltip" id="focus-panel-help-756">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r10c2">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r10c2-gene-table-details" data-name="r10c2-gene-table-details">
Gene overview
</div>



<div class="body-details-header r10c2-nrps_pks" data-name="r10c2-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r10c2-MIBiG-cluster-compare" data-name="r10c2-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r10c2-clusterblast" data-name="r10c2-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r10c2-knownclusterblast" data-name="r10c2-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r10c2-subclusterblast" data-name="r10c2-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r10c2-nrps-pks-bubbles" data-name="r10c2-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r10c2-pfam-details" data-name="r10c2-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r10c2-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-176"></div> <span class="help-tooltip" id="gt-help-tooltip-help-176">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r10c2">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r10c2-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r10c2-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r10c2">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r10c2-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-168"></div> <span class="help-tooltip" id="nrps-domain-help-168">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r10c2-details-svg" data-filename="scaffold_10_r10c2_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r10c2-details-svg"></div>
</div>
</div>



<div class="body-details-section r10c2-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-169"></div> <span class="help-tooltip" id="MIBiG-help-169">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r10c2-selector">Analysis type:</label>
<select id="comparison-MIBiG-r10c2-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r10c2"></div>
<div id="comparison-MIBiG-r10c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>

<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002622: 258-56047">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002622/index.html#r1c1" target="_new">BGC0002622</a></td>





<td class="cc-heat-cell" title="0.69 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>






<td class="cc-heat-cell" title="0.69 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>


<td>0.72</td>
<td>Other</td>

<td>hydroxycinnamoylputrescine</td>


<td>Oryza sativa Japonica Group</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002610: 0-6468">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002610/index.html#r1c1" target="_new">BGC0002610</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>






<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.64</td>
<td>Terpene</td>

<td>stellatic acid</td>


<td>Aspergillus stellatus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001893: 155-19434">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001893/index.html#r1c1" target="_new">BGC0001893</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>






<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.63</td>
<td>Other</td>

<td>abscisic acid</td>


<td>Botrytis cinerea B05.10</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002181: 1003-18265">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002181/index.html#r1c1" target="_new">BGC0002181</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>






<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.62</td>
<td>Terpene</td>

<td>pleuromutilin</td>


<td>Rhodocybe pseudopiperita</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001475: 5-9618">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001475/index.html#r1c1" target="_new">BGC0001475</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>






<td class="cc-heat-cell" title="0.62 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.56</td>
<td>Terpene</td>

<td>aspterric acid</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001445: 0-31620">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001445/index.html#r1c1" target="_new">BGC0001445</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>






<td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.55</td>
<td>NRP, Polyketide</td>

<td>leporin B</td>


<td>Aspergillus flavus NRRL3357</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001604: 5569-24234">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001604/index.html#r1c1" target="_new">BGC0001604</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.41, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>






<td class="cc-heat-cell" title="0.61 (id:0.41, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.55</td>
<td>Terpene</td>

<td>gibberellin</td>


<td>Fusarium fujikuroi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001268: 894-12763">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001268/index.html#r1c1" target="_new">BGC0001268</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>






<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.54</td>
<td>NRP, Polyketide</td>

<td>fusarin C</td>


<td>Fusarium fujikuroi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002651: 0-16858">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002651/index.html#r1c1" target="_new">BGC0002651</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.41, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>






<td class="cc-heat-cell" title="0.61 (id:0.41, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.54</td>
<td>Terpene</td>

<td>sordaricin B</td>


<td>Rosellinia necatrix</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000064: 0-11857">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000064/index.html#r1c1" target="_new">BGC0000064</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>






<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.54</td>
<td>NRP, Polyketide</td>

<td>fusarin</td>


<td>Fusarium verticillioides</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r10c2-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002622: 258-56047">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002622/index.html#r1c1" target="_new">BGC0002622</a></td>


<td class="cc-heat-cell" title="0.69 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
<td>0.69</td>
<td>Other</td>
<td>hydroxycinnamoylputrescine</td>
<td>Oryza sativa Japonica Group</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002610: 0-6468">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002610/index.html#r1c1" target="_new">BGC0002610</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Terpene</td>
<td>stellatic acid</td>
<td>Aspergillus stellatus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001893: 155-19434">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001893/index.html#r1c1" target="_new">BGC0001893</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Other</td>
<td>abscisic acid</td>
<td>Botrytis cinerea B05.10</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002181: 1003-18265">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002181/index.html#r1c1" target="_new">BGC0002181</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Terpene</td>
<td>pleuromutilin</td>
<td>Rhodocybe pseudopiperita</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001475: 5-9618">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001475/index.html#r1c1" target="_new">BGC0001475</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>Terpene</td>
<td>aspterric acid</td>
<td>Aspergillus terreus NIH2624</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001445: 0-31620">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001445/index.html#r1c1" target="_new">BGC0001445</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>NRP, Polyketide</td>
<td>leporin B</td>
<td>Aspergillus flavus NRRL3357</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001604: 5569-24234">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001604/index.html#r1c1" target="_new">BGC0001604</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.41, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>Terpene</td>
<td>gibberellin</td>
<td>Fusarium fujikuroi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001268: 894-12763">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001268/index.html#r1c1" target="_new">BGC0001268</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>NRP, Polyketide</td>
<td>fusarin C</td>
<td>Fusarium fujikuroi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002651: 0-16858">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002651/index.html#r1c1" target="_new">BGC0002651</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.41, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>Terpene</td>
<td>sordaricin B</td>
<td>Rosellinia necatrix</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000064: 0-11857">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000064/index.html#r1c1" target="_new">BGC0000064</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>NRP, Polyketide</td>
<td>fusarin</td>
<td>Fusarium verticillioides</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r10c2-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-170"></div> <span class="help-tooltip" id="cb-general-help-170">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r10c2-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r10c2_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r10c2_1.svg">
NT_165978 (33819-106089): Chaetomium globosum CBS 148.51 scaffold 3 genomic s... (23% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r10c2_2.svg">
NW_021940922 (928677-1070633): Pyricularia pennisetigena strain Br36 chromoso... (33% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r10c2_3.svg">
NW_025091226 (54054-126393): Aspergillus viridinutans IFM 47045 DNA, scaffold... (25% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r10c2_4.svg">
NW_025953061 (124491-197395): Daldinia caldariorum strain CBS 122874 unplaced... (24% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r10c2_5.svg">
NC_043741 (7379068-7451945): Pyricularia pennisetigena strain Br36 chromosome... (21% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r10c2_6.svg">
NC_030955 (5495539-5568376): Colletotrichum higginsianum IMI 349063 chromosom... (27% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r10c2_7.svg">
NW_022474205 (1490965-1563612): Venustampulla echinocandica strain BP 5553 ch... (27% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r10c2_8.svg">
NW_024067566 (1440324-1543594): Penicillium roqueforti strain LCP96 04111 chr... (17% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r10c2_9.svg">
NW_014013668 (697938-770017): Trichoderma virens Gv29-8 chromosome Unknown TR... (17% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r10c2_10.svg">
NC_064655 (2499760-2581055): Penicillium oxalicum strain HP7-1 chromosome VI,... (13% of genes show similarity), NRPS,T1PKS
</option>

</select>
<button id= "clusterblast-r10c2-download">Download graphic</button>
</div>
<div id="clusterblast-r10c2-svg"></div>

</div>
</div>

<div class="body-details-section r10c2-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-171"></div> <span class="help-tooltip" id="cb-known-help-171">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r10c2-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-172"></div> <span class="help-tooltip" id="cb-sub-help-172">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r10c2-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-175"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-175">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r10c2-domain-bubble-svg" data-filename="scaffold_10_r10c2_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r10c2-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC2">
Candidate 2 (827588 - 879905): chemical hybrid NRPS-T1PKS
</option>


</select>

<div class="details-svg" id="r10c2-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r10c2-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r10c2-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-177"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-177">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r10c2-pfam-details-svg" data-filename="scaffold_10_r10c2_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r10c2-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r10c2-nrps_pks_products" data-name="r10c2-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r10c2-nrps_pks_monomers" data-name="r10c2-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r10c2-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-173"></div> <span class="help-tooltip" id="nrps-structures-help-173">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 2, location 827588 - 879905:
<div class="collapser collapser-target-CC2 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r10c2-2" class="smiles-canvas" data-smiles="NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r10c2-2-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r10c2-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-174"></div> <span class="help-tooltip" id="nrps-monomers-help-174">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">FUN_002899</span></strong>: pk - X
<div class="collapser collapser-target-FUN_002899 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (557..873)</strong>: pk
<div class="collapser collapser-target-FUN_002899 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-FUN_002899-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 79.2%</dd>
<dd>Methylmalonyl-CoA: 66.7%</dd>
<dd>2-Methylbutyryl-CoA: 58.3%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-FUN_002899-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 82.2</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 68.2</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 42.8</dt>

<dd></dd><dt>Isobutyryl-CoA: 41.7</dt>

<dd></dd><dt>Malonyl-CoA: 41.3</dt>

<dd></dd><dt>Benzoyl-CoA: 38.2</dt>

<dd></dd><dt>Propionyl-CoA: 34.5</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 28.1</dt>

<dd></dd><dt>fatty_acid: 25.7</dt>

<dd></dd><dt>trans-1,2-CPDA: 23.8</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 17.4</dt>

<dd></dd><dt>CHC-CoA: 13.0</dt>

<dd></dd><dt>inactive: 10.9</dt>

<dd></dd><dt>Acetyl-CoA: 4.5</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>











<dd><strong>AMP-binding (3033..3436)</strong>: X
<div class="collapser collapser-target-FUN_002899 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr
<div class="collapser collapser-target-FUN_002899-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DLGIIAVIGK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>3Me-Glu <span class="serif">DLGKTGVINK</span> (56% 8 match)</dd>
<dd>Asp <span class="serif">DLTKIGHIGK</span> (44% 8 match)</dd>
<dd>cohOhi <span class="serif">DVHIIAYIVK</span> (41% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>60% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>





<div class="page" id='r12c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_12 - Region 1 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-757"></div> <span class="help-tooltip" id="region_svg-help-757">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_12.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r12c1-svg" data-filename="scaffold_12_r12c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1,635 - 46,668 nt. (total: 45,034 nt)

<a class="cluster-rules-header" id="r12c1-rules-header" href="#r12c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r12c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r12c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-758"></div> <span class="help-tooltip" id="focus-panel-help-758">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r12c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r12c1-gene-table-details" data-name="r12c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r12c1-nrps_pks" data-name="r12c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r12c1-MIBiG-cluster-compare" data-name="r12c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r12c1-clusterblast" data-name="r12c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r12c1-knownclusterblast" data-name="r12c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r12c1-subclusterblast" data-name="r12c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r12c1-nrps-pks-bubbles" data-name="r12c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r12c1-pfam-details" data-name="r12c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r12c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-186"></div> <span class="help-tooltip" id="gt-help-tooltip-help-186">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r12c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r12c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r12c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r12c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r12c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-178"></div> <span class="help-tooltip" id="nrps-domain-help-178">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r12c1-details-svg" data-filename="scaffold_12_r12c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r12c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r12c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-179"></div> <span class="help-tooltip" id="MIBiG-help-179">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r12c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r12c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r12c1"></div>
<div id="comparison-MIBiG-r12c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001989: 0-7977">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001989/index.html#r1c1" target="_new">BGC0001989</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.31</td>
<td>Other</td>

<td>pulcherrimin</td>


<td>Kluyveromyces lactis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002321: 0-6666">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002321/index.html#r1c1" target="_new">BGC0002321</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.31</td>
<td>Other</td>

<td>pulcherriminic acid, pulcherrimin, cyclodileucine, compound 3</td>


<td>Metschnikowia aff. pulcherrima</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000457: 0-11274">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000457/index.html#r1c1" target="_new">BGC0000457</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.41, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.28</td>
<td>NRP</td>

<td>vicibactin</td>


<td>Rhizobium etli CFN 42</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002226: 138-12428">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002226/index.html#r1c1" target="_new">BGC0002226</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.25</td>
<td>NRP</td>

<td>acu-dioxomorpholine B, acu-dioxomorpholine A</td>


<td>Aspergillus aculeatus ATCC 16872</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002135: 0-11508">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002135/index.html#r1c1" target="_new">BGC0002135</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.41, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.23</td>
<td>NRP</td>

<td>bovienimide A</td>


<td>Xenorhabdus bovienii SS-2004</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002437: 0-10596">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002437/index.html#r1c1" target="_new">BGC0002437</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.41, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>NRP</td>

<td>thermoactinoamide A</td>


<td>Thermoactinomyces sp. AS95</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833: 14964-30015">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>NRP</td>

<td>icosalide A, icosalide B</td>


<td>Burkholderia gladioli</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132: 0-12417">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.41, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>NRP</td>

<td>xenotetrapeptide</td>


<td>Xenorhabdus nematophila ATCC 19061</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002286: 0-16374">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002286/index.html#r1c1" target="_new">BGC0002286</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.41, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>NRP</td>

<td>ririwpeptide A, ririwpeptide B, ririwpeptide C</td>


<td>Photorhabdus laumondii subsp. laumondii TTO1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128: 35-15686">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.41, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>NRP</td>

<td>gamexpeptide C</td>


<td>Photorhabdus laumondii subsp. laumondii TTO1</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r12c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001989: 0-7977">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001989/index.html#r1c1" target="_new">BGC0001989</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Other</td>
<td>pulcherrimin</td>
<td>Kluyveromyces lactis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002321: 0-6666">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002321/index.html#r1c1" target="_new">BGC0002321</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Other</td>
<td>pulcherriminic acid, pulcherrimin, cyclodileucine, compound 3</td>
<td>Metschnikowia aff. pulcherrima</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000457: 0-11274">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000457/index.html#r1c1" target="_new">BGC0000457</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.41, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>NRP</td>
<td>vicibactin</td>
<td>Rhizobium etli CFN 42</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002226: 138-12428">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002226/index.html#r1c1" target="_new">BGC0002226</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>NRP</td>
<td>acu-dioxomorpholine B, acu-dioxomorpholine A</td>
<td>Aspergillus aculeatus ATCC 16872</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002135: 0-11508">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002135/index.html#r1c1" target="_new">BGC0002135</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.41, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>NRP</td>
<td>bovienimide A</td>
<td>Xenorhabdus bovienii SS-2004</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002437: 0-10596">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002437/index.html#r1c1" target="_new">BGC0002437</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.41, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>NRP</td>
<td>thermoactinoamide A</td>
<td>Thermoactinomyces sp. AS95</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833: 14964-30015">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>NRP</td>
<td>icosalide A, icosalide B</td>
<td>Burkholderia gladioli</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132: 0-12417">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.41, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>NRP</td>
<td>xenotetrapeptide</td>
<td>Xenorhabdus nematophila ATCC 19061</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002286: 0-16374">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002286/index.html#r1c1" target="_new">BGC0002286</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.41, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>NRP</td>
<td>ririwpeptide A, ririwpeptide B, ririwpeptide C</td>
<td>Photorhabdus laumondii subsp. laumondii TTO1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128: 35-15686">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.41, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>NRP</td>
<td>gamexpeptide C</td>
<td>Photorhabdus laumondii subsp. laumondii TTO1</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r12c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-180"></div> <span class="help-tooltip" id="cb-general-help-180">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r12c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r12c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r12c1_1.svg">
NW_025953252 (64303-129031): Annulohypoxylon maeteangense strain CBS 123835 u... (21% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r12c1_2.svg">
NC_054696 (3781861-3846644): Aspergillus flavus NRRL3357 chromosome 6, comple... (20% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r12c1_3.svg">
NW_025091283 (4319746-4671991): Aspergillus pseudoviridinutans IFM 55266 DNA,... (4% of genes show similarity), NRPS,NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r12c1_4.svg">
NW_026054665 (96370-161167): Ophidiomyces ophidiicola strain CBS 122913 chrom... (16% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r12c1_5.svg">
NC_036627 (33856-98555): Fusarium fujikuroi IMI 58289 draft genome, chromosom... (10% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r12c1_6.svg">
NW_025091285 (13966-78743): Aspergillus pseudoviridinutans IFM 55266 DNA, sca... (22% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r12c1_7.svg">
NC_036436 (62484-127411): Aspergillus oryzae RIB40 DNA, chromosome 2 (19% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r12c1_8.svg">
NC_063077 (36917-101739): Purpureocillium takamizusanense chromosome 10, comp... (20% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r12c1_9.svg">
NW_025091230 (206680-271452): Aspergillus viridinutans IFM 47045 DNA, scaffol... (14% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r12c1_10.svg">
NC_054862 (3508654-3573352): Aspergillus puulaauensis MK2 DNA, chromosome 6, ... (11% of genes show similarity), NRPS
</option>

</select>
<button id= "clusterblast-r12c1-download">Download graphic</button>
</div>
<div id="clusterblast-r12c1-svg"></div>

</div>
</div>

<div class="body-details-section r12c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-181"></div> <span class="help-tooltip" id="cb-known-help-181">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r12c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r12c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r12c1_1.svg">
aspirochlorine
</option>

<option value = "svg/knownclusterblast_r12c1_2.svg">
penigainamide A/penigainamide B/penigainamide C/adametizine A/FA2097/outovirin A/outovirin C/pretrichodermamide C
</option>

<option value = "svg/knownclusterblast_r12c1_3.svg">
acetylaranotin
</option>

</select>
<button id="knownclusterblast-r12c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r12c1-svg"></div>

</div>
</div>

<div class="body-details-section r12c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-182"></div> <span class="help-tooltip" id="cb-sub-help-182">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r12c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-185"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-185">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r12c1-domain-bubble-svg" data-filename="scaffold_12_r12c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r12c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (1634 - 46668): single NRPS
</option>


</select>

<div class="details-svg" id="r12c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r12c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r12c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-187"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-187">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r12c1-pfam-details-svg" data-filename="scaffold_12_r12c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r12c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r12c1-nrps_pks_products" data-name="r12c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r12c1-nrps_pks_monomers" data-name="r12c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r12c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-183"></div> <span class="help-tooltip" id="nrps-structures-help-183">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 1634 - 46668:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r12c1-1" class="smiles-canvas" data-smiles="NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r12c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r12c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-184"></div> <span class="help-tooltip" id="nrps-monomers-help-184">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">FUN_003160</span></strong>: X
<div class="collapser collapser-target-FUN_003160 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>AMP-binding (605..928)</strong>: X
<div class="collapser collapser-target-FUN_003160 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  (unknown)
<div class="collapser collapser-target-FUN_003160-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>N/A</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DGHIGV-AAK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Phe <span class="serif">DGHIYVMCGK</span> (62% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>60% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r12c2' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_12 - Region 2 - fungal-RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-759"></div> <span class="help-tooltip" id="region_svg-help-759">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_12.region002.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r12c2-svg" data-filename="scaffold_12_r12c2.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 583,247 - 644,383 nt. (total: 61,137 nt)

<a class="cluster-rules-header" id="r12c2-rules-header" href="#r12c2">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r12c2-rules">fungal-RiPP-like: DUF3328</div>

<div class="region-svg-container">
<div id='r12c2-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-760"></div> <span class="help-tooltip" id="focus-panel-help-760">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r12c2">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r12c2-gene-table-details" data-name="r12c2-gene-table-details">
Gene overview
</div>



<div class="body-details-header r12c2-MIBiG-cluster-compare" data-name="r12c2-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r12c2-clusterblast" data-name="r12c2-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r12c2-knownclusterblast" data-name="r12c2-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r12c2-subclusterblast" data-name="r12c2-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r12c2-pfam-details" data-name="r12c2-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r12c2-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-192"></div> <span class="help-tooltip" id="gt-help-tooltip-help-192">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r12c2">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r12c2-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r12c2-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r12c2">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r12c2-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-188"></div> <span class="help-tooltip" id="MIBiG-help-188">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r12c2-selector">Analysis type:</label>
<select id="comparison-MIBiG-r12c2-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r12c2"></div>
<div id="comparison-MIBiG-r12c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">fungal-RiPP-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001306: 0-6409">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001306/index.html#r1c1" target="_new">BGC0001306</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.34</td>
<td>RiPP</td>

<td>asperipin 2a</td>


<td>Aspergillus flavus NRRL3357</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002670: 0-27523">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002670/index.html#r1c1" target="_new">BGC0002670</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.41, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Other</td>

<td>fosfonochlorin</td>


<td>Fusarium oxysporum f. sp. lycopersici 4287</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002403: 0-129425">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002403/index.html#r1c1" target="_new">BGC0002403</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>Polyketide</td>

<td>-Diketones</td>


<td>Hordeum vulgare subsp. vulgare</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001775: 2711-20673">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001775/index.html#r1c1" target="_new">BGC0001775</a></td>





<td class="cc-heat-cell" title="0.51 (id:0.42, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>


<td>0.19</td>
<td>Terpene</td>

<td>sespendole</td>


<td>Cordana terrestris</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002216: 305-18235">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002216/index.html#r1c1" target="_new">BGC0002216</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.42, order:N/A, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>Terpene</td>

<td>HEx-tc1 terpene</td>


<td>Trichoderma virens Gv29-8</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264: 8442-40380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.16</td>
<td>Polyketide</td>

<td>betaenone A, betaenone B, betaenone C</td>


<td>Phoma betae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000894: 395-43446">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000894/index.html#r1c1" target="_new">BGC0000894</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.15</td>
<td>Other</td>

<td>citrinin</td>


<td>Monascus aurantiacus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001882: 0-43357">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001882/index.html#r1c1" target="_new">BGC0001882</a></td>





<td class="cc-heat-cell" title="0.43 (id:0.56, order:N/A, components:0.34)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>


<td>0.14</td>
<td>Polyketide</td>

<td>chrysoxanthone A, chrysoxanthone B, chrysoxanthone C</td>


<td>Penicillium rubens Wisconsin 54-1255</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001277: 621-57840">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001277/index.html#r1c1" target="_new">BGC0001277</a></td>





<td class="cc-heat-cell" title="0.43 (id:0.55, order:0.36, components:0.41)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>


<td>0.14</td>
<td>Terpene</td>

<td>nivalenol, vomitoxin, 3-acetyldeoxynivalenol, 15-acetyldeoxynivalenol, neosolaniol, calonectrin, apotrichodiol, isotrichotriol, 15-deacetylcalonectrin, T-2 toxin, 3-acetyl T-2 toxin, trichodiene</td>


<td>Fusarium graminearum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002213: 0-33851">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002213/index.html#r1c1" target="_new">BGC0002213</a></td>





<td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>


<td>0.14</td>
<td>Polyketide</td>

<td>HEx-pks16 polyketide</td>


<td>Punctularia strigosozonata HHB-11173 SS5</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r12c2-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001306: 0-6409">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001306/index.html#r1c1" target="_new">BGC0001306</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>RiPP</td>
<td>asperipin 2a</td>
<td>Aspergillus flavus NRRL3357</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002670: 0-27523">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002670/index.html#r1c1" target="_new">BGC0002670</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.41, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Other</td>
<td>fosfonochlorin</td>
<td>Fusarium oxysporum f. sp. lycopersici 4287</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002403: 0-129425">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002403/index.html#r1c1" target="_new">BGC0002403</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Polyketide</td>
<td>-Diketones</td>
<td>Hordeum vulgare subsp. vulgare</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001775: 2711-20673">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001775/index.html#r1c1" target="_new">BGC0001775</a></td>


<td class="cc-heat-cell" title="0.51 (id:0.42, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
<td>0.51</td>
<td>Terpene</td>
<td>sespendole</td>
<td>Cordana terrestris</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002216: 305-18235">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002216/index.html#r1c1" target="_new">BGC0002216</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.42, order:N/A, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>Terpene</td>
<td>HEx-tc1 terpene</td>
<td>Trichoderma virens Gv29-8</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264: 8442-40380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>Polyketide</td>
<td>betaenone A, betaenone B, betaenone C</td>
<td>Phoma betae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000894: 395-43446">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000894/index.html#r1c1" target="_new">BGC0000894</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>Other</td>
<td>citrinin</td>
<td>Monascus aurantiacus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001882: 0-43357">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001882/index.html#r1c1" target="_new">BGC0001882</a></td>


<td class="cc-heat-cell" title="0.43 (id:0.56, order:N/A, components:0.34)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
<td>0.43</td>
<td>Polyketide</td>
<td>chrysoxanthone A, chrysoxanthone B, chrysoxanthone C</td>
<td>Penicillium rubens Wisconsin 54-1255</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001277: 621-57840">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001277/index.html#r1c1" target="_new">BGC0001277</a></td>


<td class="cc-heat-cell" title="0.43 (id:0.55, order:0.36, components:0.41)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
<td>0.43</td>
<td>Terpene</td>
<td>nivalenol, vomitoxin, 3-acetyldeoxynivalenol, 15-acetyldeoxynivalenol, neosolaniol, calonectrin, apotrichodiol, isotrichotriol, 15-deacetylcalonectrin, T-2 toxin, 3-acetyl T-2 toxin, trichodiene</td>
<td>Fusarium graminearum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002213: 0-33851">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002213/index.html#r1c1" target="_new">BGC0002213</a></td>


<td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
<td>0.43</td>
<td>Polyketide</td>
<td>HEx-pks16 polyketide</td>
<td>Punctularia strigosozonata HHB-11173 SS5</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r12c2-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-189"></div> <span class="help-tooltip" id="cb-general-help-189">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r12c2-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r12c2_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r12c2_1.svg">
NW_015622513 (4594674-4688646): Exophiala spinifera strain CBS 89968 unplaced... (7% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r12c2_2.svg">
NW_006917092 (5012573-5103587): Pestalotiopsis fici W106-1 unplaced genomic s... (5% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r12c2_3.svg">
NC_049566 (57470-149701): Talaromyces rugulosus chromosome VI, complete sequence (6% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r12c2_4.svg">
NC_072348 (1750133-1909935): Talaromyces marneffei chromosome 1, complete seq... (3% of genes show similarity), NRPS-like,fungal-RiPP-like,other
</option>

<option value = "svg/clusterblast_r12c2_5.svg">
NW_017971443 (484511-631944): Talaromyces atroroseus strain IBT 11181 chromos... (3% of genes show similarity), NRPS-like,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r12c2_6.svg">
NC_054851 (3991582-4054711): Aspergillus luchuensis IFO 4308 DNA, chromosome ... (18% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r12c2_7.svg">
NC_049563 (2312417-2482120): Talaromyces rugulosus chromosome III, complete s... (5% of genes show similarity), NRPS,NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r12c2_8.svg">
NW_006271969 (4860304-4927848): Cordyceps militaris CM01 unplaced genomic sca... (16% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r12c2_9.svg">
NC_064656 (2030492-2120598): Penicillium oxalicum strain HP7-1 chromosome VII... (11% of genes show similarity), NRPS,terpene
</option>

<option value = "svg/clusterblast_r12c2_10.svg">
NW_025091283 (3059889-3123711): Aspergillus pseudoviridinutans IFM 55266 DNA,... (6% of genes show similarity), NRPS-like
</option>

</select>
<button id= "clusterblast-r12c2-download">Download graphic</button>
</div>
<div id="clusterblast-r12c2-svg"></div>

</div>
</div>

<div class="body-details-section r12c2-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-190"></div> <span class="help-tooltip" id="cb-known-help-190">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r12c2-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r12c2_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r12c2_1.svg">
nivalenol/vomitoxin/3-acetyldeoxynivalenol/15-acetyldeoxynivalenol/neosolaniol/calonectrin/apotrichodiol/isotrichotriol/15-deacetylcalonectrin/T-2 toxin/3-acetyl T-2 toxin/trichodiene
</option>

<option value = "svg/knownclusterblast_r12c2_2.svg">
nivalenol/deoxynivalenol/3-acetyldeoxynivalenol/15-acetyldeoxynivalenol/neosolaniol/calonectrin/apotrichodiol/isotrichotriol/15-decalonectrin/T-2 toxin/3-acetyl T-2 toxin/trichodiene
</option>

</select>
<button id="knownclusterblast-r12c2-download">Download graphic</button>
</div>
<div id="knownclusterblast-r12c2-svg"></div>

</div>
</div>

<div class="body-details-section r12c2-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-191"></div> <span class="help-tooltip" id="cb-sub-help-191">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r12c2-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-193"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-193">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r12c2-pfam-details-svg" data-filename="scaffold_12_r12c2_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r12c2-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">











</div>











</div>

</div>

</div>





<div class="page" id='r13c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_13 - Region 1 - fungal-RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-761"></div> <span class="help-tooltip" id="region_svg-help-761">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_13.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r13c1-svg" data-filename="scaffold_13_r13c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 14,892 - 78,003 nt. (total: 63,112 nt)

<a class="cluster-rules-header" id="r13c1-rules-header" href="#r13c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r13c1-rules">fungal-RiPP-like: DUF3328</div>

<div class="region-svg-container">
<div id='r13c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-762"></div> <span class="help-tooltip" id="focus-panel-help-762">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r13c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r13c1-gene-table-details" data-name="r13c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r13c1-MIBiG-cluster-compare" data-name="r13c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r13c1-clusterblast" data-name="r13c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r13c1-knownclusterblast" data-name="r13c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r13c1-subclusterblast" data-name="r13c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r13c1-pfam-details" data-name="r13c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r13c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-198"></div> <span class="help-tooltip" id="gt-help-tooltip-help-198">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r13c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r13c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r13c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r13c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r13c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-194"></div> <span class="help-tooltip" id="MIBiG-help-194">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r13c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r13c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r13c1"></div>
<div id="comparison-MIBiG-r13c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">fungal-RiPP-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001475: 5-9618">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001475/index.html#r1c1" target="_new">BGC0001475</a></td>





<td class="cc-heat-cell" title="0.42 (id:0.52, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>


<td>0.13</td>
<td>Terpene</td>

<td>aspterric acid</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002707: 3-21301">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002707/index.html#r1c1" target="_new">BGC0002707</a></td>





<td class="cc-heat-cell" title="0.31 (id:0.47, order:N/A, components:0.20)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>


<td>0.08</td>
<td>Terpene</td>

<td>(+)--cadinol</td>


<td>Coniophora puteana RWD-64-598 SS2</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001928: 108-8402">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001928/index.html#r1c1" target="_new">BGC0001928</a></td>





<td class="cc-heat-cell" title="0.24 (id:0.47, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.24), rgb(0, 0, 0, 0.24) 24%, white 24%)"></td>


<td>0.06</td>
<td>Other</td>

<td>(2R,3S,4S)-5-fluoro-2,3,4-trihydroxypentanoic acid</td>


<td>Streptomyces sp. MA37</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001289: 16-10643">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001289/index.html#r1c1" target="_new">BGC0001289</a></td>





<td class="cc-heat-cell" title="0.24 (id:0.47, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.24), rgb(0, 0, 0, 0.24) 24%, white 24%)"></td>


<td>0.06</td>
<td>RiPP</td>

<td>penisin</td>


<td>Paenibacillus ehimensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000145: 1-32637">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000145/index.html#r1c1" target="_new">BGC0000145</a></td>





<td class="cc-heat-cell" title="0.18 (id:0.48, order:N/A, components:0.07)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.18), rgb(0, 0, 0, 0.18) 18%, white 18%)"></td>


<td>0.04</td>
<td>NRP, Polyketide</td>

<td>salinosporamide A</td>


<td>Salinispora tropica</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001047: 1-22119">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001047/index.html#r1c1" target="_new">BGC0001047</a></td>





<td class="cc-heat-cell" title="0.17 (id:0.45, order:N/A, components:0.07)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.17), rgb(0, 0, 0, 0.17) 17%, white 17%)"></td>


<td>0.04</td>
<td>NRP, Polyketide</td>

<td>syringolin A</td>


<td>Pseudomonas syringae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001005: 0-46783">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001005/index.html#r1c1" target="_new">BGC0001005</a></td>





<td class="cc-heat-cell" title="0.17 (id:0.45, order:N/A, components:0.06)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.17), rgb(0, 0, 0, 0.17) 17%, white 17%)"></td>


<td>0.03</td>
<td>NRP, Polyketide</td>

<td>locillomycin, locillomycin B, locillomycin C</td>


<td>Bacillus subtilis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002351: 0-37128">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002351/index.html#r1c1" target="_new">BGC0002351</a></td>





<td class="cc-heat-cell" title="0.16 (id:0.44, order:N/A, components:0.06)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.16), rgb(0, 0, 0, 0.16) 16%, white 16%)"></td>


<td>0.03</td>
<td>NRP</td>

<td>thanafactin A</td>


<td>Pseudomonas sp. SHC52</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001041: 0-42010">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001041/index.html#r1c1" target="_new">BGC0001041</a></td>





<td class="cc-heat-cell" title="0.16 (id:0.48, order:N/A, components:0.05)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.16), rgb(0, 0, 0, 0.16) 16%, white 16%)"></td>


<td>0.03</td>
<td>NRP, Polyketide</td>

<td>salinosporamide A</td>


<td>Salinispora tropica CNB-440</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001205: 0-47244">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001205/index.html#r1c1" target="_new">BGC0001205</a></td>





<td class="cc-heat-cell" title="0.16 (id:0.43, order:N/A, components:0.06)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.16), rgb(0, 0, 0, 0.16) 16%, white 16%)"></td>


<td>0.03</td>
<td>Polyketide</td>

<td>communesin A, communesin B, communesin C, communesin D, communesin E, communesin G, communesin H</td>


<td>Penicillium expansum</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r13c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001475: 5-9618">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001475/index.html#r1c1" target="_new">BGC0001475</a></td>


<td class="cc-heat-cell" title="0.42 (id:0.52, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
<td>0.42</td>
<td>Terpene</td>
<td>aspterric acid</td>
<td>Aspergillus terreus NIH2624</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002707: 3-21301">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002707/index.html#r1c1" target="_new">BGC0002707</a></td>


<td class="cc-heat-cell" title="0.31 (id:0.47, order:N/A, components:0.20)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>
<td>0.31</td>
<td>Terpene</td>
<td>(+)--cadinol</td>
<td>Coniophora puteana RWD-64-598 SS2</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001928: 108-8402">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001928/index.html#r1c1" target="_new">BGC0001928</a></td>


<td class="cc-heat-cell" title="0.24 (id:0.47, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.24), rgb(0, 0, 0, 0.24) 24%, white 24%)"></td>
<td>0.24</td>
<td>Other</td>
<td>(2R,3S,4S)-5-fluoro-2,3,4-trihydroxypentanoic acid</td>
<td>Streptomyces sp. MA37</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001289: 16-10643">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001289/index.html#r1c1" target="_new">BGC0001289</a></td>


<td class="cc-heat-cell" title="0.24 (id:0.47, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.24), rgb(0, 0, 0, 0.24) 24%, white 24%)"></td>
<td>0.24</td>
<td>RiPP</td>
<td>penisin</td>
<td>Paenibacillus ehimensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000145: 1-32637">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000145/index.html#r1c1" target="_new">BGC0000145</a></td>


<td class="cc-heat-cell" title="0.18 (id:0.48, order:N/A, components:0.07)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.18), rgb(0, 0, 0, 0.18) 18%, white 18%)"></td>
<td>0.18</td>
<td>NRP, Polyketide</td>
<td>salinosporamide A</td>
<td>Salinispora tropica</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001047: 1-22119">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001047/index.html#r1c1" target="_new">BGC0001047</a></td>


<td class="cc-heat-cell" title="0.17 (id:0.45, order:N/A, components:0.07)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.17), rgb(0, 0, 0, 0.17) 17%, white 17%)"></td>
<td>0.17</td>
<td>NRP, Polyketide</td>
<td>syringolin A</td>
<td>Pseudomonas syringae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001005: 0-46783">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001005/index.html#r1c1" target="_new">BGC0001005</a></td>


<td class="cc-heat-cell" title="0.17 (id:0.45, order:N/A, components:0.06)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.17), rgb(0, 0, 0, 0.17) 17%, white 17%)"></td>
<td>0.17</td>
<td>NRP, Polyketide</td>
<td>locillomycin, locillomycin B, locillomycin C</td>
<td>Bacillus subtilis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002351: 0-37128">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002351/index.html#r1c1" target="_new">BGC0002351</a></td>


<td class="cc-heat-cell" title="0.16 (id:0.44, order:N/A, components:0.06)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.16), rgb(0, 0, 0, 0.16) 16%, white 16%)"></td>
<td>0.16</td>
<td>NRP</td>
<td>thanafactin A</td>
<td>Pseudomonas sp. SHC52</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001041: 0-42010">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001041/index.html#r1c1" target="_new">BGC0001041</a></td>


<td class="cc-heat-cell" title="0.16 (id:0.48, order:N/A, components:0.05)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.16), rgb(0, 0, 0, 0.16) 16%, white 16%)"></td>
<td>0.16</td>
<td>NRP, Polyketide</td>
<td>salinosporamide A</td>
<td>Salinispora tropica CNB-440</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001205: 0-47244">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001205/index.html#r1c1" target="_new">BGC0001205</a></td>


<td class="cc-heat-cell" title="0.16 (id:0.43, order:N/A, components:0.06)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.16), rgb(0, 0, 0, 0.16) 16%, white 16%)"></td>
<td>0.16</td>
<td>Polyketide</td>
<td>communesin A, communesin B, communesin C, communesin D, communesin E, communesin G, communesin H</td>
<td>Penicillium expansum</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r13c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-195"></div> <span class="help-tooltip" id="cb-general-help-195">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r13c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r13c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r13c1_1.svg">
NW_006912905 (2600367-2631558): Capronia epimyces CBS 606.96 aczIQ-supercont1... (14% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r13c1_2.svg">
NW_022986860 (22048-112928): Westerdykella ornata strain CBS 379.55 unplaced ... (4% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r13c1_3.svg">
NW_020194479 (1449493-1553698): Amorphotheca resinae ATCC 22711 unplaced geno... (5% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r13c1_4.svg">
NC_011894 (5419835-5456146): Methylobacterium nodulans ORS 2060, complete seq... (8% of genes show similarity), oligosaccharide
</option>

<option value = "svg/clusterblast_r13c1_5.svg">
NW_024468986 (554990-661675): Aspergillus fijiensis CBS 313.89 unplaced genom... (7% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r13c1_6.svg">
NW_009276940 (1583392-1646635): Verticillium dahliae VdLs.17 supercont1.3 gen... (9% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r13c1_7.svg">
NW_015971674 (93018-183947): Fonsecaea multimorphosa CBS 102226 unplaced geno... (5% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r13c1_8.svg">
NW_014013673 (128037-191883): Trichoderma virens Gv29-8 chromosome Unknown TR... (7% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r13c1_9.svg">
NC_070549 (252971-316778): Fusarium falciforme chromosome 6, complete sequence (12% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r13c1_10.svg">
NC_070534 (194101-306469): Fusarium keratoplasticum isolate Fu6.1 chromosome ... (4% of genes show similarity), NRPS-like,T1PKS
</option>

</select>
<button id= "clusterblast-r13c1-download">Download graphic</button>
</div>
<div id="clusterblast-r13c1-svg"></div>

</div>
</div>

<div class="body-details-section r13c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-196"></div> <span class="help-tooltip" id="cb-known-help-196">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r13c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-197"></div> <span class="help-tooltip" id="cb-sub-help-197">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r13c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-199"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-199">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r13c1-pfam-details-svg" data-filename="scaffold_13_r13c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r13c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">











</div>











</div>

</div>

</div>

<div class="page" id='r13c2' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_13 - Region 2 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-763"></div> <span class="help-tooltip" id="region_svg-help-763">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_13.region002.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r13c2-svg" data-filename="scaffold_13_r13c2.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 210,304 - 231,876 nt. (total: 21,573 nt)

<a class="cluster-rules-header" id="r13c2-rules-header" href="#r13c2">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r13c2-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>

<div class="region-svg-container">
<div id='r13c2-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-764"></div> <span class="help-tooltip" id="focus-panel-help-764">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r13c2">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r13c2-gene-table-details" data-name="r13c2-gene-table-details">
Gene overview
</div>



<div class="body-details-header r13c2-MIBiG-cluster-compare" data-name="r13c2-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r13c2-clusterblast" data-name="r13c2-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r13c2-knownclusterblast" data-name="r13c2-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r13c2-subclusterblast" data-name="r13c2-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r13c2-pfam-details" data-name="r13c2-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r13c2-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-204"></div> <span class="help-tooltip" id="gt-help-tooltip-help-204">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r13c2">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r13c2-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r13c2-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r13c2">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r13c2-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-200"></div> <span class="help-tooltip" id="MIBiG-help-200">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r13c2-selector">Analysis type:</label>
<select id="comparison-MIBiG-r13c2-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r13c2"></div>
<div id="comparison-MIBiG-r13c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">terpene</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001990: 130-13646">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001990/index.html#r1c1" target="_new">BGC0001990</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.33</td>
<td>Other</td>

<td>BU-4704, xanthocillin-X monomethylether, melanocin E, melanocin F, fumiformamide</td>


<td>Aspergillus fumigatus Af293</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001969: 0-7810">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001969/index.html#r1c1" target="_new">BGC0001969</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.33</td>
<td>Terpene</td>

<td>asperterpenoid A</td>


<td>Talaromyces wortmannii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000669: 0-34330">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000669/index.html#r1c1" target="_new">BGC0000669</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.26</td>
<td>Terpene</td>

<td>marneral</td>


<td>Arabidopsis thaliana</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000677: 120-4940">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000677/index.html#r1c1" target="_new">BGC0000677</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.26</td>
<td>Terpene</td>

<td>cyclooctatin</td>


<td>Streptomyces melanosporofaciens</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002173: 92-7542">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002173/index.html#r1c1" target="_new">BGC0002173</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>Terpene, Saccharide</td>

<td>fumihopaside A, compound 3, 21--H-hopane-3beta, 22-diol</td>


<td>Aspergillus fumigatus A1163</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001219: 0-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001219/index.html#r1c1" target="_new">BGC0001219</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Polyketide</td>

<td>shanorellin</td>


<td>Chaetomium globosum CBS 148.51</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000676: 621-14838">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000676/index.html#r1c1" target="_new">BGC0000676</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.42, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.22</td>
<td>Terpene</td>

<td>aphidicolin, aphidicolan-16-ol, 3-deoxyaphidicolin, 17-deoxyaphidicolin</td>


<td>Phoma betae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000631: 1094-28776">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000631/index.html#r1c1" target="_new">BGC0000631</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.68)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.21</td>
<td>Terpene</td>

<td>botcinin H, botcinin I, botcinin J, cinbotolide D, botcinin K</td>


<td>Botrytis cinerea B05.10</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002644: 0-7903">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002644/index.html#r1c1" target="_new">BGC0002644</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>Other, Terpene</td>

<td>domoic acid, isodomoic acid A, isodomoic acid B, isodomoic acid C, 7&#39;-carboxy-L-NGG, prekainic acid, kainic acid</td>


<td>Chondria armata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002651: 0-16858">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002651/index.html#r1c1" target="_new">BGC0002651</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.42, order:N/A, components:0.55)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>Terpene</td>

<td>sordaricin B</td>


<td>Rosellinia necatrix</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r13c2-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001990: 130-13646">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001990/index.html#r1c1" target="_new">BGC0001990</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Other</td>
<td>BU-4704, xanthocillin-X monomethylether, melanocin E, melanocin F, fumiformamide</td>
<td>Aspergillus fumigatus Af293</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001969: 0-7810">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001969/index.html#r1c1" target="_new">BGC0001969</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Terpene</td>
<td>asperterpenoid A</td>
<td>Talaromyces wortmannii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000669: 0-34330">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000669/index.html#r1c1" target="_new">BGC0000669</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>Terpene</td>
<td>marneral</td>
<td>Arabidopsis thaliana</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000677: 120-4940">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000677/index.html#r1c1" target="_new">BGC0000677</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Terpene</td>
<td>cyclooctatin</td>
<td>Streptomyces melanosporofaciens</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002173: 92-7542">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002173/index.html#r1c1" target="_new">BGC0002173</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Terpene, Saccharide</td>
<td>fumihopaside A, compound 3, 21--H-hopane-3beta, 22-diol</td>
<td>Aspergillus fumigatus A1163</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001219: 0-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001219/index.html#r1c1" target="_new">BGC0001219</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Polyketide</td>
<td>shanorellin</td>
<td>Chaetomium globosum CBS 148.51</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000676: 621-14838">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000676/index.html#r1c1" target="_new">BGC0000676</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.42, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Terpene</td>
<td>aphidicolin, aphidicolan-16-ol, 3-deoxyaphidicolin, 17-deoxyaphidicolin</td>
<td>Phoma betae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000631: 1094-28776">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000631/index.html#r1c1" target="_new">BGC0000631</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.68)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Terpene</td>
<td>botcinin H, botcinin I, botcinin J, cinbotolide D, botcinin K</td>
<td>Botrytis cinerea B05.10</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002644: 0-7903">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002644/index.html#r1c1" target="_new">BGC0002644</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Other, Terpene</td>
<td>domoic acid, isodomoic acid A, isodomoic acid B, isodomoic acid C, 7&#39;-carboxy-L-NGG, prekainic acid, kainic acid</td>
<td>Chondria armata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002651: 0-16858">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002651/index.html#r1c1" target="_new">BGC0002651</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.42, order:N/A, components:0.55)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>Terpene</td>
<td>sordaricin B</td>
<td>Rosellinia necatrix</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r13c2-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-201"></div> <span class="help-tooltip" id="cb-general-help-201">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r13c2-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r13c2_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r13c2_1.svg">
NW_014013657 (1821768-1853192): Trichoderma virens Gv29-8 chromosome Unknown ... (23% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r13c2_2.svg">
NC_049561 (111243-176756): Talaromyces rugulosus chromosome I, complete sequence (7% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r13c2_3.svg">
NC_036441 (1029545-1060095): Aspergillus oryzae RIB40 DNA, chromosome 7 (25% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r13c2_4.svg">
NW_022985067 (1748825-1780241): Eremomyces bilateralis CBS 781.70 unplaced ge... (15% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r13c2_5.svg">
NW_022983490 (330572-424081): Aaosphaeria arxii CBS 175.79 unplaced genomic s... (6% of genes show similarity), T1PKS,fungal-RiPP-like,terpene
</option>

</select>
<button id= "clusterblast-r13c2-download">Download graphic</button>
</div>
<div id="clusterblast-r13c2-svg"></div>

</div>
</div>

<div class="body-details-section r13c2-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-202"></div> <span class="help-tooltip" id="cb-known-help-202">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r13c2-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r13c2_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r13c2_1.svg">
heptelidic acid
</option>

</select>
<button id="knownclusterblast-r13c2-download">Download graphic</button>
</div>
<div id="knownclusterblast-r13c2-svg"></div>

</div>
</div>

<div class="body-details-section r13c2-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-203"></div> <span class="help-tooltip" id="cb-sub-help-203">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r13c2-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-205"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-205">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r13c2-pfam-details-svg" data-filename="scaffold_13_r13c2_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r13c2-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">









</div>









</div>

</div>

</div>

<div class="page" id='r13c3' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_13 - Region 3 - fungal-RiPP <div class="help-container"> <div class="help-icon" data-id="region_svg-help-765"></div> <span class="help-tooltip" id="region_svg-help-765">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_13.region003.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r13c3-svg" data-filename="scaffold_13_r13c3.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 771,214 - 808,011 nt. (total: 36,798 nt)

<a class="cluster-rules-header" id="r13c3-rules-header" href="#r13c3">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r13c3-rules">fungal-RiPP: ((pop or ustH) and (p450 or TP_methylase) or DUF3328 and (Peptidase_S41 or Peptidase_S28 or pop or ustH))</div>

<div class="region-svg-container">
<div id='r13c3-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-766"></div> <span class="help-tooltip" id="focus-panel-help-766">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r13c3">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r13c3-gene-table-details" data-name="r13c3-gene-table-details">
Gene overview
</div>



<div class="body-details-header r13c3-MIBiG-cluster-compare" data-name="r13c3-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r13c3-clusterblast" data-name="r13c3-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r13c3-knownclusterblast" data-name="r13c3-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r13c3-subclusterblast" data-name="r13c3-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r13c3-pfam-details" data-name="r13c3-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r13c3-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-210"></div> <span class="help-tooltip" id="gt-help-tooltip-help-210">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r13c3">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r13c3-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r13c3-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r13c3">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r13c3-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-206"></div> <span class="help-tooltip" id="MIBiG-help-206">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r13c3-selector">Analysis type:</label>
<select id="comparison-MIBiG-r13c3-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r13c3"></div>
<div id="comparison-MIBiG-r13c3-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">fungal-RiPP</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000627: 0-28295">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000627/index.html#r1c1" target="_new">BGC0000627</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.51, order:N/A, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.24</td>
<td>RiPP</td>

<td>ustiloxin B</td>


<td>Aspergillus flavus NRRL3357</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001893: 155-19434">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001893/index.html#r1c1" target="_new">BGC0001893</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>Other</td>

<td>abscisic acid</td>


<td>Botrytis cinerea B05.10</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002403: 0-129425">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002403/index.html#r1c1" target="_new">BGC0002403</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>Polyketide</td>

<td>-Diketones</td>


<td>Hordeum vulgare subsp. vulgare</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001668: 56-7544">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001668/index.html#r1c1" target="_new">BGC0001668</a></td>





<td class="cc-heat-cell" title="0.44 (id:0.43, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>


<td>0.14</td>
<td>NRP</td>

<td>microperfuranone</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002213: 0-33851">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002213/index.html#r1c1" target="_new">BGC0002213</a></td>





<td class="cc-heat-cell" title="0.38 (id:0.43, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>


<td>0.11</td>
<td>Polyketide</td>

<td>HEx-pks16 polyketide</td>


<td>Punctularia strigosozonata HHB-11173 SS5</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002363: 0-22640">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002363/index.html#r1c1" target="_new">BGC0002363</a></td>





<td class="cc-heat-cell" title="0.36 (id:0.48, order:N/A, components:0.28)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>


<td>0.10</td>
<td>RiPP</td>

<td>berninamycin K, berninamycin J, berninamycin A, berninamycin B</td>


<td>Streptomyces sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001425: 0-44135">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001425/index.html#r1c1" target="_new">BGC0001425</a></td>





<td class="cc-heat-cell" title="0.35 (id:0.47, order:N/A, components:0.27)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>


<td>0.10</td>
<td>NRP, Polyketide</td>

<td>myxochromide A</td>


<td>Myxococcus xanthus DK 1622</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001421: 0-47099">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001421/index.html#r1c1" target="_new">BGC0001421</a></td>





<td class="cc-heat-cell" title="0.35 (id:0.46, order:N/A, components:0.27)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>


<td>0.10</td>
<td>NRP, Polyketide</td>

<td>myxochromide B</td>


<td>Myxococcus sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001428: 0-44682">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001428/index.html#r1c1" target="_new">BGC0001428</a></td>





<td class="cc-heat-cell" title="0.35 (id:0.46, order:N/A, components:0.27)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>


<td>0.10</td>
<td>NRP, Polyketide</td>

<td>myxochromide A</td>


<td>Myxococcus xanthus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001424: 0-41589">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001424/index.html#r1c1" target="_new">BGC0001424</a></td>





<td class="cc-heat-cell" title="0.35 (id:0.46, order:N/A, components:0.27)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>


<td>0.10</td>
<td>NRP, Polyketide</td>

<td>myxochromide C</td>


<td>Myxococcus virescens</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r13c3-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000627: 0-28295">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000627/index.html#r1c1" target="_new">BGC0000627</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.51, order:N/A, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>RiPP</td>
<td>ustiloxin B</td>
<td>Aspergillus flavus NRRL3357</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001893: 155-19434">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001893/index.html#r1c1" target="_new">BGC0001893</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Other</td>
<td>abscisic acid</td>
<td>Botrytis cinerea B05.10</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002403: 0-129425">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002403/index.html#r1c1" target="_new">BGC0002403</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>Polyketide</td>
<td>-Diketones</td>
<td>Hordeum vulgare subsp. vulgare</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001668: 56-7544">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001668/index.html#r1c1" target="_new">BGC0001668</a></td>


<td class="cc-heat-cell" title="0.44 (id:0.43, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
<td>0.44</td>
<td>NRP</td>
<td>microperfuranone</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002213: 0-33851">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002213/index.html#r1c1" target="_new">BGC0002213</a></td>


<td class="cc-heat-cell" title="0.38 (id:0.43, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
<td>0.38</td>
<td>Polyketide</td>
<td>HEx-pks16 polyketide</td>
<td>Punctularia strigosozonata HHB-11173 SS5</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002363: 0-22640">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002363/index.html#r1c1" target="_new">BGC0002363</a></td>


<td class="cc-heat-cell" title="0.36 (id:0.48, order:N/A, components:0.28)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>
<td>0.36</td>
<td>RiPP</td>
<td>berninamycin K, berninamycin J, berninamycin A, berninamycin B</td>
<td>Streptomyces sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001425: 0-44135">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001425/index.html#r1c1" target="_new">BGC0001425</a></td>


<td class="cc-heat-cell" title="0.35 (id:0.47, order:N/A, components:0.27)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>
<td>0.35</td>
<td>NRP, Polyketide</td>
<td>myxochromide A</td>
<td>Myxococcus xanthus DK 1622</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001421: 0-47099">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001421/index.html#r1c1" target="_new">BGC0001421</a></td>


<td class="cc-heat-cell" title="0.35 (id:0.46, order:N/A, components:0.27)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>
<td>0.35</td>
<td>NRP, Polyketide</td>
<td>myxochromide B</td>
<td>Myxococcus sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001428: 0-44682">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001428/index.html#r1c1" target="_new">BGC0001428</a></td>


<td class="cc-heat-cell" title="0.35 (id:0.46, order:N/A, components:0.27)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>
<td>0.35</td>
<td>NRP, Polyketide</td>
<td>myxochromide A</td>
<td>Myxococcus xanthus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001424: 0-41589">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001424/index.html#r1c1" target="_new">BGC0001424</a></td>


<td class="cc-heat-cell" title="0.35 (id:0.46, order:N/A, components:0.27)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>
<td>0.35</td>
<td>NRP, Polyketide</td>
<td>myxochromide C</td>
<td>Myxococcus virescens</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r13c3-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-207"></div> <span class="help-tooltip" id="cb-general-help-207">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r13c3-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r13c3_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r13c3_1.svg">
NC_058393 (3117776-3200618): Fusarium musae strain F31 chromosome 7, whole ge... (7% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r13c3_2.svg">
NW_022983480 (1409834-1473053): Aaosphaeria arxii CBS 175.79 unplaced genomic... (7% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r13c3_3.svg">
NC_044974 (348825-508778): Pyricularia grisea strain NI907 chromosome VI, who... (3% of genes show similarity), NRPS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r13c3_4.svg">
NZ_CP101468 (36289-78869): Rhodococcus gordoniae strain DND3 plasmid pDND3A, ... (7% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r13c3_5.svg">
NZ_CP029607 (3775436-3873150): Actinosynnema pretiosum subsp. pretiosum strai... (3% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r13c3_6.svg">
NW_025953795 (523593-591758): Annulohypoxylon truncatum strain CBS 140777 unp... (11% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r13c3_7.svg">
NZ_CP017705 (4711315-4850025): Brevibacillus laterosporus DSM 25 chromosome, ... (2% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r13c3_8.svg">
NZ_CP032410 (1539053-1675703): Brevibacillus laterosporus strain E7593-50 chr... (3% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r13c3_9.svg">
NZ_CP048799 (1094745-1238114): Brevibacillus sp. 7WMA2 chromosome, complete g... (2% of genes show similarity), NRPS,T1PKS,terpene
</option>

<option value = "svg/clusterblast_r13c3_10.svg">
NZ_CP032848 (3955715-4089490): Brevibacillus laterosporus strain Bl-zj chromo... (3% of genes show similarity), NRPS,T1PKS,terpene
</option>

</select>
<button id= "clusterblast-r13c3-download">Download graphic</button>
</div>
<div id="clusterblast-r13c3-svg"></div>

</div>
</div>

<div class="body-details-section r13c3-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-208"></div> <span class="help-tooltip" id="cb-known-help-208">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r13c3-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-209"></div> <span class="help-tooltip" id="cb-sub-help-209">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r13c3-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-211"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-211">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r13c3-pfam-details-svg" data-filename="scaffold_13_r13c3_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r13c3-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">











</div>











</div>

</div>

</div>





<div class="page" id='r16c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_16 - Region 1 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-767"></div> <span class="help-tooltip" id="region_svg-help-767">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_16.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r16c1-svg" data-filename="scaffold_16_r16c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 325,288 - 371,470 nt. (total: 46,183 nt)

<a class="cluster-rules-header" id="r16c1-rules-header" href="#r16c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r16c1-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r16c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-768"></div> <span class="help-tooltip" id="focus-panel-help-768">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r16c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r16c1-gene-table-details" data-name="r16c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r16c1-nrps_pks" data-name="r16c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r16c1-MIBiG-cluster-compare" data-name="r16c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r16c1-clusterblast" data-name="r16c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r16c1-knownclusterblast" data-name="r16c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r16c1-subclusterblast" data-name="r16c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r16c1-nrps-pks-bubbles" data-name="r16c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r16c1-pfam-details" data-name="r16c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r16c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-220"></div> <span class="help-tooltip" id="gt-help-tooltip-help-220">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r16c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r16c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r16c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r16c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r16c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-212"></div> <span class="help-tooltip" id="nrps-domain-help-212">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r16c1-details-svg" data-filename="scaffold_16_r16c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r16c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r16c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-213"></div> <span class="help-tooltip" id="MIBiG-help-213">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r16c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r16c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r16c1"></div>
<div id="comparison-MIBiG-r16c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002191: 260-8785">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002191/index.html#r1c1" target="_new">BGC0002191</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.31</td>
<td>Polyketide</td>

<td>prolipyrone B, gibepyrone D</td>


<td>Fusarium graminearum PH-1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002155: 78-16511">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002155/index.html#r1c1" target="_new">BGC0002155</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.44, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.31</td>
<td>Polyketide</td>

<td>nectriapyrone C, nectriapyrone D, nectriapyrone</td>


<td>Pyricularia oryzae 70-15</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001252: 0-7958">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001252/index.html#r1c1" target="_new">BGC0001252</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.42, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.30</td>
<td>Polyketide</td>

<td>UNII-YC2Q1O94PT</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002525: 1000-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002525/index.html#r1c1" target="_new">BGC0002525</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.43, order:N/A, components:0.82)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.26</td>
<td>Polyketide</td>

<td>fusarubin, 1233A, 1233B, NG-391, lucilactaene</td>


<td>Fusarium sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002222: 888-32276">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002222/index.html#r1c1" target="_new">BGC0002222</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.25</td>
<td>Polyketide</td>

<td>zopfiellin</td>


<td>Diffractella curvata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002515: 0-30087">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002515/index.html#r1c1" target="_new">BGC0002515</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.42, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>Polyketide</td>

<td>solanapyrone A</td>


<td>Ascochyta rabiei</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002224: 685-37748">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002224/index.html#r1c1" target="_new">BGC0002224</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>Polyketide</td>

<td>zopfiellin</td>


<td>Diffractella curvata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002194: 147-15389">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002194/index.html#r1c1" target="_new">BGC0002194</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>Polyketide</td>

<td>epipyrone A</td>


<td>Epicoccum nigrum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001606: 0-22544">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001606/index.html#r1c1" target="_new">BGC0001606</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.42, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>Polyketide</td>

<td>gibepyrone-A</td>


<td>Fusarium fujikuroi IMI 58289</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000146: 754-20637">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000146/index.html#r1c1" target="_new">BGC0000146</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Polyketide</td>

<td>solanapyrone D</td>


<td>Alternaria solani</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r16c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002191: 260-8785">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002191/index.html#r1c1" target="_new">BGC0002191</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Polyketide</td>
<td>prolipyrone B, gibepyrone D</td>
<td>Fusarium graminearum PH-1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002155: 78-16511">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002155/index.html#r1c1" target="_new">BGC0002155</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.44, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>Polyketide</td>
<td>nectriapyrone C, nectriapyrone D, nectriapyrone</td>
<td>Pyricularia oryzae 70-15</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001252: 0-7958">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001252/index.html#r1c1" target="_new">BGC0001252</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.42, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>Polyketide</td>
<td>UNII-YC2Q1O94PT</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002525: 1000-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002525/index.html#r1c1" target="_new">BGC0002525</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.43, order:N/A, components:0.82)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Polyketide</td>
<td>fusarubin, 1233A, 1233B, NG-391, lucilactaene</td>
<td>Fusarium sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002222: 888-32276">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002222/index.html#r1c1" target="_new">BGC0002222</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Polyketide</td>
<td>zopfiellin</td>
<td>Diffractella curvata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002515: 0-30087">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002515/index.html#r1c1" target="_new">BGC0002515</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.42, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Polyketide</td>
<td>solanapyrone A</td>
<td>Ascochyta rabiei</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002224: 685-37748">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002224/index.html#r1c1" target="_new">BGC0002224</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Polyketide</td>
<td>zopfiellin</td>
<td>Diffractella curvata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002194: 147-15389">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002194/index.html#r1c1" target="_new">BGC0002194</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Polyketide</td>
<td>epipyrone A</td>
<td>Epicoccum nigrum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001606: 0-22544">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001606/index.html#r1c1" target="_new">BGC0001606</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.42, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Polyketide</td>
<td>gibepyrone-A</td>
<td>Fusarium fujikuroi IMI 58289</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000146: 754-20637">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000146/index.html#r1c1" target="_new">BGC0000146</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Polyketide</td>
<td>solanapyrone D</td>
<td>Alternaria solani</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r16c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-214"></div> <span class="help-tooltip" id="cb-general-help-214">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r16c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r16c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r16c1_1.svg">
NW_024467308 (148929-260597): Aspergillus ruber CBS 135680 unplaced genomic s... (4% of genes show similarity), T1PKS,indole
</option>

<option value = "svg/clusterblast_r16c1_2.svg">
NW_003052498 (3549167-3614153): Uncinocarpus reesii 1704 scaffold 3 genomic s... (7% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r16c1_3.svg">
NZ_CP104794 (7464894-7683158): Streptomyces lusitanus strain FZ201 chromosome... (5% of genes show similarity), PKS-like,T1PKS,butyrolactone
</option>

<option value = "svg/clusterblast_r16c1_4.svg">
NC_049561 (1082430-1146591): Talaromyces rugulosus chromosome I, complete seq... (8% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r16c1_5.svg">
NW_022474206 (2604837-2669288): Venustampulla echinocandica strain BP 5553 ch... (9% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r16c1_6.svg">
NC_054853 (2681752-2888163): Aspergillus luchuensis IFO 4308 DNA, chromosome ... (2% of genes show similarity), NRPS-like,T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r16c1_7.svg">
NC_014623 (4751260-4894265): Stigmatella aurantiaca DW4/3-1, complete sequence (5% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r16c1_8.svg">
NW_021940919 (209289-272614): Pyricularia pennisetigena strain Br36 chromosom... (12% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r16c1_9.svg">
NZ_CP061007 (6402895-6499084): Saccharopolyspora spinosa strain CCTCC M206084... (8% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r16c1_10.svg">
NZ_CP017170 (5536714-5629392): Myxococcus xanthus strain GH5.1.9c20 chromosom... (8% of genes show similarity), NRPS,T1PKS
</option>

</select>
<button id= "clusterblast-r16c1-download">Download graphic</button>
</div>
<div id="clusterblast-r16c1-svg"></div>

</div>
</div>

<div class="body-details-section r16c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-215"></div> <span class="help-tooltip" id="cb-known-help-215">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r16c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-216"></div> <span class="help-tooltip" id="cb-sub-help-216">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r16c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-219"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-219">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r16c1-domain-bubble-svg" data-filename="scaffold_16_r16c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r16c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (325287 - 371470): single T1PKS
</option>


</select>

<div class="details-svg" id="r16c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r16c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r16c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-221"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-221">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r16c1-pfam-details-svg" data-filename="scaffold_16_r16c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r16c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r16c1-nrps_pks_products" data-name="r16c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r16c1-nrps_pks_monomers" data-name="r16c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r16c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-217"></div> <span class="help-tooltip" id="nrps-structures-help-217">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 325287 - 371470:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r16c1-1" class="smiles-canvas" data-smiles="C([*])C(-O)(C)"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS linker matching; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Me-pk)</dd>

<dt><b>Non-colinear ordering:</b></dt><dd>

<span class="serif">FUN_004031</span>

+ <span class="serif">FUN_004030</span>

</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-pk" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-pk*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r16c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)(C)"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r16c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-218"></div> <span class="help-tooltip" id="nrps-monomers-help-218">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">FUN_004031</span></strong>: pk
<div class="collapser collapser-target-FUN_004031 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (566..841)</strong>: pk
<div class="collapser collapser-target-FUN_004031 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-FUN_004031-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 58.3%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-FUN_004031-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 79.1</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 74.9</dt>

<dd></dd><dt>Malonyl-CoA: 70.6</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 66.1</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 44.1</dt>

<dd></dd><dt>Propionyl-CoA: 43.1</dt>

<dd></dd><dt>Isobutyryl-CoA: 30.3</dt>

<dd></dd><dt>inactive: 25.3</dt>

<dd></dd><dt>CHC-CoA: 17.9</dt>

<dd></dd><dt>Benzoyl-CoA: 16.9</dt>

<dd></dd><dt>trans-1,2-CPDA: 16.2</dt>

<dd></dd><dt>Acetyl-CoA: 16.0</dt>

<dd></dd><dt>fatty_acid: 11.3</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>





<div class="page" id='r17c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_17 - Region 1 - T3PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-769"></div> <span class="help-tooltip" id="region_svg-help-769">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_17.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r17c1-svg" data-filename="scaffold_17_r17c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 212,546 - 253,996 nt. (total: 41,451 nt)

<a class="cluster-rules-header" id="r17c1-rules-header" href="#r17c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r17c1-rules">T3PKS: (Chal_sti_synt_C or Chal_sti_synt_N)</div>

<div class="region-svg-container">
<div id='r17c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-770"></div> <span class="help-tooltip" id="focus-panel-help-770">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r17c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r17c1-gene-table-details" data-name="r17c1-gene-table-details">
Gene overview
</div>





<div class="body-details-header r17c1-MIBiG-cluster-compare" data-name="r17c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r17c1-clusterblast" data-name="r17c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r17c1-knownclusterblast" data-name="r17c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r17c1-subclusterblast" data-name="r17c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r17c1-pfam-details" data-name="r17c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r17c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-226"></div> <span class="help-tooltip" id="gt-help-tooltip-help-226">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r17c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r17c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r17c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r17c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>





<div class="body-details-section r17c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-222"></div> <span class="help-tooltip" id="MIBiG-help-222">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r17c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r17c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r17c1"></div>
<div id="comparison-MIBiG-r17c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T3PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002348: 0-18134">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002348/index.html#r1c1" target="_new">BGC0002348</a></td>





<td class="cc-heat-cell" title="0.41 (id:0.50, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>


<td>0.13</td>
<td>NRP</td>

<td>aspulvinone E</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001487: 0-9112">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001487/index.html#r1c1" target="_new">BGC0001487</a></td>





<td class="cc-heat-cell" title="0.37 (id:0.41, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>


<td>0.11</td>
<td>Other</td>

<td>3,7-dihydroxytropolone</td>


<td>Streptomyces cyaneofuscatus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001076: 0-29709">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001076/index.html#r1c1" target="_new">BGC0001076</a></td>





<td class="cc-heat-cell" title="0.30 (id:0.41, order:N/A, components:0.22)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>


<td>0.08</td>
<td>Other (Nucleoside)</td>

<td>liposidomycin B, liposidomycin A, liposidomycin C, liposidomycin Z, liposidomycin H, liposidomycin G, liposidomycin L, liposidomycin M, liposidomycin K, liposidomycin N</td>


<td>Streptomyces sp. SN-1061M</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000872: 0-35385">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000872/index.html#r1c1" target="_new">BGC0000872</a></td>





<td class="cc-heat-cell" title="0.30 (id:0.41, order:N/A, components:0.22)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>


<td>0.08</td>
<td>Other</td>

<td>A-90289 A, A-90289 B</td>


<td>Streptomyces sp. SANK 60405</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000420: 0-20659">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000420/index.html#r1c1" target="_new">BGC0000420</a></td>





<td class="cc-heat-cell" title="0.29 (id:0.41, order:N/A, components:0.20)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>


<td>0.07</td>
<td>NRP</td>

<td>histidyltryptophanyldiketopiperazine, dehydrohistidyltryptophanyldiketopiperazine, roquefortine D, roquefortine C, glandicoline A, glandicoline B, meleagrine</td>


<td>Penicillium rubens Wisconsin 54-1255</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000679: 0-47916">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000679/index.html#r1c1" target="_new">BGC0000679</a></td>





<td class="cc-heat-cell" title="0.26 (id:0.43, order:N/A, components:0.16)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>


<td>0.06</td>
<td>Terpene</td>

<td>diazepinomicin</td>


<td>Micromonospora sp. M42</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000843: 0-23666">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000843/index.html#r1c1" target="_new">BGC0000843</a></td>





<td class="cc-heat-cell" title="0.25 (id:0.43, order:N/A, components:0.15)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>


<td>0.06</td>
<td>Other (Non-NRP beta-lactam)</td>

<td>alanylclavam, 2-hydroxymethylclavam, 2-formyloxymethylclavam, clavam-2-carboxylate</td>


<td>Streptomyces clavuligerus ATCC 27064</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001295: 0-18154">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001295/index.html#r1c1" target="_new">BGC0001295</a></td>





<td class="cc-heat-cell" title="0.24 (id:0.41, order:N/A, components:0.14)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.24), rgb(0, 0, 0, 0.24) 24%, white 24%)"></td>


<td>0.06</td>
<td>Other</td>

<td>cremeomycin</td>


<td>Streptomyces cremeus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001848: 0-58198">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001848/index.html#r1c1" target="_new">BGC0001848</a></td>





<td class="cc-heat-cell" title="0.22 (id:0.43, order:N/A, components:0.11)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.22), rgb(0, 0, 0, 0.22) 22%, white 22%)"></td>


<td>0.05</td>
<td>Other</td>

<td>diazaquinomycin H, diazaquinomycin J</td>


<td>Micromonospora sp. B006</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001377: 819-44027">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001377/index.html#r1c1" target="_new">BGC0001377</a></td>





<td class="cc-heat-cell" title="0.21 (id:0.42, order:N/A, components:0.11)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.21), rgb(0, 0, 0, 0.21) 21%, white 21%)"></td>


<td>0.05</td>
<td>Polyketide</td>

<td>uncialamycin</td>


<td>Streptomyces uncialis</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r17c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002348: 0-18134">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002348/index.html#r1c1" target="_new">BGC0002348</a></td>


<td class="cc-heat-cell" title="0.41 (id:0.50, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
<td>0.41</td>
<td>NRP</td>
<td>aspulvinone E</td>
<td>Aspergillus terreus NIH2624</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001487: 0-9112">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001487/index.html#r1c1" target="_new">BGC0001487</a></td>


<td class="cc-heat-cell" title="0.37 (id:0.41, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>
<td>0.37</td>
<td>Other</td>
<td>3,7-dihydroxytropolone</td>
<td>Streptomyces cyaneofuscatus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001076: 0-29709">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001076/index.html#r1c1" target="_new">BGC0001076</a></td>


<td class="cc-heat-cell" title="0.30 (id:0.41, order:N/A, components:0.22)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>
<td>0.30</td>
<td>Other (Nucleoside)</td>
<td>liposidomycin B, liposidomycin A, liposidomycin C, liposidomycin Z, liposidomycin H, liposidomycin G, liposidomycin L, liposidomycin M, liposidomycin K, liposidomycin N</td>
<td>Streptomyces sp. SN-1061M</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000872: 0-35385">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000872/index.html#r1c1" target="_new">BGC0000872</a></td>


<td class="cc-heat-cell" title="0.30 (id:0.41, order:N/A, components:0.22)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>
<td>0.30</td>
<td>Other</td>
<td>A-90289 A, A-90289 B</td>
<td>Streptomyces sp. SANK 60405</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000420: 0-20659">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000420/index.html#r1c1" target="_new">BGC0000420</a></td>


<td class="cc-heat-cell" title="0.29 (id:0.41, order:N/A, components:0.20)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>
<td>0.29</td>
<td>NRP</td>
<td>histidyltryptophanyldiketopiperazine, dehydrohistidyltryptophanyldiketopiperazine, roquefortine D, roquefortine C, glandicoline A, glandicoline B, meleagrine</td>
<td>Penicillium rubens Wisconsin 54-1255</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000679: 0-47916">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000679/index.html#r1c1" target="_new">BGC0000679</a></td>


<td class="cc-heat-cell" title="0.26 (id:0.43, order:N/A, components:0.16)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>
<td>0.26</td>
<td>Terpene</td>
<td>diazepinomicin</td>
<td>Micromonospora sp. M42</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000843: 0-23666">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000843/index.html#r1c1" target="_new">BGC0000843</a></td>


<td class="cc-heat-cell" title="0.25 (id:0.43, order:N/A, components:0.15)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>
<td>0.25</td>
<td>Other (Non-NRP beta-lactam)</td>
<td>alanylclavam, 2-hydroxymethylclavam, 2-formyloxymethylclavam, clavam-2-carboxylate</td>
<td>Streptomyces clavuligerus ATCC 27064</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001295: 0-18154">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001295/index.html#r1c1" target="_new">BGC0001295</a></td>


<td class="cc-heat-cell" title="0.24 (id:0.41, order:N/A, components:0.14)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.24), rgb(0, 0, 0, 0.24) 24%, white 24%)"></td>
<td>0.24</td>
<td>Other</td>
<td>cremeomycin</td>
<td>Streptomyces cremeus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001848: 0-58198">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001848/index.html#r1c1" target="_new">BGC0001848</a></td>


<td class="cc-heat-cell" title="0.22 (id:0.43, order:N/A, components:0.11)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.22), rgb(0, 0, 0, 0.22) 22%, white 22%)"></td>
<td>0.22</td>
<td>Other</td>
<td>diazaquinomycin H, diazaquinomycin J</td>
<td>Micromonospora sp. B006</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001377: 819-44027">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001377/index.html#r1c1" target="_new">BGC0001377</a></td>


<td class="cc-heat-cell" title="0.21 (id:0.42, order:N/A, components:0.11)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.21), rgb(0, 0, 0, 0.21) 21%, white 21%)"></td>
<td>0.21</td>
<td>Polyketide</td>
<td>uncialamycin</td>
<td>Streptomyces uncialis</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r17c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-223"></div> <span class="help-tooltip" id="cb-general-help-223">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r17c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r17c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r17c1_1.svg">
NW_026054545 (141343-202765): Daldinia decipiens strain CBS 113046 unplaced g... (21% of genes show similarity), T3PKS
</option>

<option value = "svg/clusterblast_r17c1_2.svg">
NW_025953037 (2094131-2155555): Daldinia caldariorum strain CBS 122874 unplac... (33% of genes show similarity), T3PKS
</option>

<option value = "svg/clusterblast_r17c1_3.svg">
NW_026250021 (502671-612769): Hypoxylon trugodes strain CBS 135444 unplaced g... (17% of genes show similarity), T1PKS,T3PKS
</option>

<option value = "svg/clusterblast_r17c1_4.svg">
NW_025953250 (386309-447332): Annulohypoxylon maeteangense strain CBS 123835 ... (33% of genes show similarity), T3PKS
</option>

<option value = "svg/clusterblast_r17c1_5.svg">
NW_025953834 (73723-135142): Annulohypoxylon truncatum strain CBS 140777 unpl... (23% of genes show similarity), T3PKS
</option>

<option value = "svg/clusterblast_r17c1_6.svg">
NW_006917097 (1298417-1359854): Pestalotiopsis fici W106-1 unplaced genomic s... (11% of genes show similarity), T3PKS
</option>

<option value = "svg/clusterblast_r17c1_7.svg">
NW_013581189 (8134-132316): Aureobasidium namibiae CBS 147.97 unplaced genomi... (6% of genes show similarity), NRPS-like,T3PKS
</option>

<option value = "svg/clusterblast_r17c1_8.svg">
NC_054696 (2233834-2294976): Aspergillus flavus NRRL3357 chromosome 6, comple... (10% of genes show similarity), T3PKS
</option>

<option value = "svg/clusterblast_r17c1_9.svg">
NW_025408669 (4012538-4073910): Hirsutella rhossiliensis strain HR02 chromoso... (100% of genes show similarity), T3PKS
</option>

<option value = "svg/clusterblast_r17c1_10.svg">
NW_020292103 (9361-181402): Aspergillus ibericus CBS 121593 unplaced genomic ... (2% of genes show similarity), NRPS-like,T1PKS,fungal-RiPP-like,terpene
</option>

</select>
<button id= "clusterblast-r17c1-download">Download graphic</button>
</div>
<div id="clusterblast-r17c1-svg"></div>

</div>
</div>

<div class="body-details-section r17c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-224"></div> <span class="help-tooltip" id="cb-known-help-224">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r17c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-225"></div> <span class="help-tooltip" id="cb-sub-help-225">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r17c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-227"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-227">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r17c1-pfam-details-svg" data-filename="scaffold_17_r17c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r17c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">













</div>













</div>

</div>

</div>





<div class="page" id='r18c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_18 - Region 1 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-771"></div> <span class="help-tooltip" id="region_svg-help-771">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_18.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r18c1-svg" data-filename="scaffold_18_r18c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 101,596 - 147,598 nt. (total: 46,003 nt)

<a class="cluster-rules-header" id="r18c1-rules-header" href="#r18c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r18c1-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r18c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-772"></div> <span class="help-tooltip" id="focus-panel-help-772">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r18c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r18c1-gene-table-details" data-name="r18c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r18c1-nrps_pks" data-name="r18c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r18c1-MIBiG-cluster-compare" data-name="r18c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r18c1-clusterblast" data-name="r18c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r18c1-knownclusterblast" data-name="r18c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r18c1-subclusterblast" data-name="r18c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r18c1-nrps-pks-bubbles" data-name="r18c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r18c1-pfam-details" data-name="r18c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r18c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-236"></div> <span class="help-tooltip" id="gt-help-tooltip-help-236">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r18c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r18c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r18c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r18c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r18c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-228"></div> <span class="help-tooltip" id="nrps-domain-help-228">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r18c1-details-svg" data-filename="scaffold_18_r18c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r18c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r18c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-229"></div> <span class="help-tooltip" id="MIBiG-help-229">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r18c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r18c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r18c1"></div>
<div id="comparison-MIBiG-r18c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000030: 0-18435">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000030/index.html#r1c1" target="_new">BGC0000030</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.33</td>
<td>Polyketide</td>

<td>bikaverin</td>


<td>Fusarium fujikuroi IMI 58289</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002301: 2654-19498">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002301/index.html#r1c1" target="_new">BGC0002301</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.33</td>
<td>Polyketide</td>

<td>(P)-ustilaginoidin A</td>


<td>Aschersonia paraphysata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001446: 0-15813">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001446/index.html#r1c1" target="_new">BGC0001446</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.44, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.29</td>
<td>Polyketide</td>

<td>asparasone A</td>


<td>Aspergillus flavus NRRL3357</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002177: 386-51733">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002177/index.html#r1c1" target="_new">BGC0002177</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.27</td>
<td>Polyketide</td>

<td>ustilaginoidin N, ustilaginoidin O, ustilaginoidin M, ustilaginoidin A, ustilaginoidin F, ustilaginoidin E, ustilaginoidin D, ustilaginoidin G</td>


<td>Ustilaginoidea virens</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002154: 337-7126">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002154/index.html#r1c1" target="_new">BGC0002154</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Polyketide</td>

<td>1,3,5,8-tetrahydroxynapthalene, 1-(alpha-l-(2-O-methyl)-6-deoxymannopyranosyloxy)-3,6,8-trimethoxynaphthalene</td>


<td>Pyricularia oryzae 70-15</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001583: 3200-15837">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001583/index.html#r1c1" target="_new">BGC0001583</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Polyketide</td>

<td>emodin</td>


<td>Escovopsis weberi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001258: 43-7265">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001258/index.html#r1c1" target="_new">BGC0001258</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Polyketide</td>

<td>1,3,6,8-tetrahydroxynaphthalene</td>


<td>Glarea lozoyensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001284: 0-5292">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001284/index.html#r1c1" target="_new">BGC0001284</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Polyketide</td>

<td>alternariol</td>


<td>Parastagonospora nodorum SN15</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002175: 214-6865">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002175/index.html#r1c1" target="_new">BGC0002175</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Polyketide</td>

<td>YWA1</td>


<td>Aspergillus oryzae RIB40</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001906: 35-64276">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001906/index.html#r1c1" target="_new">BGC0001906</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Polyketide</td>

<td>naphthalene</td>


<td>Daldinia eschscholzii IFB-TL01</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r18c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000030: 0-18435">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000030/index.html#r1c1" target="_new">BGC0000030</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Polyketide</td>
<td>bikaverin</td>
<td>Fusarium fujikuroi IMI 58289</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002301: 2654-19498">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002301/index.html#r1c1" target="_new">BGC0002301</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Polyketide</td>
<td>(P)-ustilaginoidin A</td>
<td>Aschersonia paraphysata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001446: 0-15813">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001446/index.html#r1c1" target="_new">BGC0001446</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.44, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>Polyketide</td>
<td>asparasone A</td>
<td>Aspergillus flavus NRRL3357</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002177: 386-51733">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002177/index.html#r1c1" target="_new">BGC0002177</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>Polyketide</td>
<td>ustilaginoidin N, ustilaginoidin O, ustilaginoidin M, ustilaginoidin A, ustilaginoidin F, ustilaginoidin E, ustilaginoidin D, ustilaginoidin G</td>
<td>Ustilaginoidea virens</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002154: 337-7126">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002154/index.html#r1c1" target="_new">BGC0002154</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Polyketide</td>
<td>1,3,5,8-tetrahydroxynapthalene, 1-(alpha-l-(2-O-methyl)-6-deoxymannopyranosyloxy)-3,6,8-trimethoxynaphthalene</td>
<td>Pyricularia oryzae 70-15</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001583: 3200-15837">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001583/index.html#r1c1" target="_new">BGC0001583</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Polyketide</td>
<td>emodin</td>
<td>Escovopsis weberi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001258: 43-7265">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001258/index.html#r1c1" target="_new">BGC0001258</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Polyketide</td>
<td>1,3,6,8-tetrahydroxynaphthalene</td>
<td>Glarea lozoyensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001284: 0-5292">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001284/index.html#r1c1" target="_new">BGC0001284</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Polyketide</td>
<td>alternariol</td>
<td>Parastagonospora nodorum SN15</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002175: 214-6865">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002175/index.html#r1c1" target="_new">BGC0002175</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Polyketide</td>
<td>YWA1</td>
<td>Aspergillus oryzae RIB40</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001906: 35-64276">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001906/index.html#r1c1" target="_new">BGC0001906</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Polyketide</td>
<td>naphthalene</td>
<td>Daldinia eschscholzii IFB-TL01</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r18c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-230"></div> <span class="help-tooltip" id="cb-general-help-230">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r18c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r18c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r18c1_1.svg">
NW_024467834 (3924829-3991573): Aspergillus wentii DTO 134E9 unplaced genomic... (10% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r18c1_2.svg">
NZ_CP115450 (1385197-1515743): Kitasatospora sp. HUAS 3-15 chromosome, comple... (5% of genes show similarity), NRPS-like,T1PKS,T2PKS
</option>

<option value = "svg/clusterblast_r18c1_3.svg">
NW_017253369 (141732-208056): Paraphaeosphaeria sporulosa strain AP3s5-JAC2a ... (7% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r18c1_4.svg">
NW_020291592 (339371-529932): Aspergillus piperis CBS 112811 unplaced genomic... (2% of genes show similarity), NRPS-like,T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r18c1_5.svg">
NW_020292051 (280608-471430): Aspergillus vadensis CBS 113365 unplaced genomi... (2% of genes show similarity), NRPS-like,T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r18c1_6.svg">
NW_020291822 (391784-582973): Aspergillus costaricaensis CBS 115574 unplaced ... (2% of genes show similarity), NRPS-like,T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r18c1_7.svg">
NC_054853 (2681752-2888163): Aspergillus luchuensis IFO 4308 DNA, chromosome ... (2% of genes show similarity), NRPS-like,T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r18c1_8.svg">
NW_023336287 (400215-591232): Aspergillus tubingensis WU-2223L DNA, scaffold ... (2% of genes show similarity), NRPS-like,T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r18c1_9.svg">
NW_020290890 (450315-638141): Aspergillus niger CBS 101883 unplaced genomic s... (2% of genes show similarity), NRPS-like,T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r18c1_10.svg">
NC_054693 (1148196-1213642): Aspergillus flavus NRRL3357 chromosome 3, comple... (9% of genes show similarity), T1PKS
</option>

</select>
<button id= "clusterblast-r18c1-download">Download graphic</button>
</div>
<div id="clusterblast-r18c1-svg"></div>

</div>
</div>

<div class="body-details-section r18c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-231"></div> <span class="help-tooltip" id="cb-known-help-231">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r18c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-232"></div> <span class="help-tooltip" id="cb-sub-help-232">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r18c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-235"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-235">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r18c1-domain-bubble-svg" data-filename="scaffold_18_r18c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r18c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (101595 - 147598): single T1PKS
</option>


</select>

<div class="details-svg" id="r18c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r18c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r18c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-237"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-237">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r18c1-pfam-details-svg" data-filename="scaffold_18_r18c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r18c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r18c1-nrps_pks_products" data-name="r18c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r18c1-nrps_pks_monomers" data-name="r18c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r18c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-233"></div> <span class="help-tooltip" id="nrps-structures-help-233">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 101595 - 147598:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r18c1-1" class="smiles-canvas" data-smiles="CC(=O)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(mal)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r18c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="CC(=O)C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r18c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-234"></div> <span class="help-tooltip" id="nrps-monomers-help-234">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">FUN_004366</span></strong>: mal
<div class="collapser collapser-target-FUN_004366 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>







<dd><strong>PKS_AT (793..1115)</strong>: mal
<div class="collapser collapser-target-FUN_004366 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-FUN_004366-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 75.0%</dd>
<dd>inactive: 58.3%</dd>
<dd>Methylmalonyl-CoA: 58.3%</dd>
</dl>

</div><br>

minowa:  Malonyl-CoA
<div class="collapser collapser-target-FUN_004366-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Malonyl-CoA: 89.8</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 53.2</dt>

<dd></dd><dt>Methylmalonyl-CoA: 46.5</dt>

<dd></dd><dt>inactive: 39.6</dt>

<dd></dd><dt>Benzoyl-CoA: 24.8</dt>

<dd></dd><dt>Isobutyryl-CoA: 22.9</dt>

<dd></dd><dt>Propionyl-CoA: 22.4</dt>

<dd></dd><dt>Acetyl-CoA: 22.4</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 22.2</dt>

<dd></dd><dt>CHC-CoA: 18.8</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 18.4</dt>

<dd></dd><dt>fatty_acid: 13.2</dt>

<dd></dd><dt>trans-1,2-CPDA: 12.5</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 11.2</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>








</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>





<div class="page" id='r19c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_19 - Region 1 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-773"></div> <span class="help-tooltip" id="region_svg-help-773">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_19.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r19c1-svg" data-filename="scaffold_19_r19c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 180,152 - 203,374 nt. (total: 23,223 nt)

<a class="cluster-rules-header" id="r19c1-rules-header" href="#r19c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r19c1-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>

<div class="region-svg-container">
<div id='r19c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-774"></div> <span class="help-tooltip" id="focus-panel-help-774">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r19c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r19c1-gene-table-details" data-name="r19c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r19c1-MIBiG-cluster-compare" data-name="r19c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r19c1-clusterblast" data-name="r19c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r19c1-knownclusterblast" data-name="r19c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r19c1-subclusterblast" data-name="r19c1-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r19c1-pfam-details" data-name="r19c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r19c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-242"></div> <span class="help-tooltip" id="gt-help-tooltip-help-242">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r19c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r19c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r19c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r19c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r19c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-238"></div> <span class="help-tooltip" id="MIBiG-help-238">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r19c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r19c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r19c1"></div>
<div id="comparison-MIBiG-r19c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">terpene</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000648: 1057-11443">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000648/index.html#r1c1" target="_new">BGC0000648</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.46, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.21</td>
<td>Terpene</td>

<td>carotenoid</td>


<td>Myxococcus xanthus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000650: 28-7720">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000650/index.html#r1c1" target="_new">BGC0000650</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>Terpene</td>

<td>carotenoid</td>


<td>Algoriphagus sp. KK10202C</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000664: 0-8391">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000664/index.html#r1c1" target="_new">BGC0000664</a></td>





<td class="cc-heat-cell" title="0.39 (id:0.43, order:N/A, components:0.35)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>


<td>0.11</td>
<td>Terpene</td>

<td>isorenieratene</td>


<td>Streptomyces griseus subsp. griseus NBRC 13350</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000633: 0-14284">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000633/index.html#r1c1" target="_new">BGC0000633</a></td>





<td class="cc-heat-cell" title="0.38 (id:0.42, order:N/A, components:0.35)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>


<td>0.11</td>
<td>Terpene</td>

<td>carotenoid</td>


<td>Streptomyces avermitilis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000637: 540-9054">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000637/index.html#r1c1" target="_new">BGC0000637</a></td>





<td class="cc-heat-cell" title="0.38 (id:0.42, order:N/A, components:0.35)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>


<td>0.11</td>
<td>Terpene</td>

<td>carotenoid</td>


<td>Corynebacterium glutamicum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000645: 0-7475">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000645/index.html#r1c1" target="_new">BGC0000645</a></td>





<td class="cc-heat-cell" title="0.38 (id:0.43, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>


<td>0.11</td>
<td>Terpene</td>

<td>carotenoid</td>


<td>Halobacillus halophilus DSM 2266</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001227: 0-12144">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001227/index.html#r1c1" target="_new">BGC0001227</a></td>





<td class="cc-heat-cell" title="0.38 (id:0.43, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>


<td>0.11</td>
<td>Terpene</td>

<td>isorenieratene</td>


<td>Streptomyces collinus Tu 365</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001456: 0-9777">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001456/index.html#r1c1" target="_new">BGC0001456</a></td>





<td class="cc-heat-cell" title="0.37 (id:0.43, order:N/A, components:0.32)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>


<td>0.11</td>
<td>Terpene</td>

<td>isorenieratene</td>


<td>Streptomyces argillaceus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000644: 97-6276">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000644/index.html#r1c1" target="_new">BGC0000644</a></td>





<td class="cc-heat-cell" title="0.34 (id:0.43, order:N/A, components:0.27)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>


<td>0.09</td>
<td>Terpene</td>

<td>carotenoid</td>


<td>Dietzia sp. CQ4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000649: 148-10747">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000649/index.html#r1c1" target="_new">BGC0000649</a></td>





<td class="cc-heat-cell" title="0.33 (id:0.43, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>


<td>0.09</td>
<td>Terpene</td>

<td>carotenoid</td>


<td>Streptomyces griseus subsp. griseus NBRC 13350</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r19c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000648: 1057-11443">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000648/index.html#r1c1" target="_new">BGC0000648</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.46, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Terpene</td>
<td>carotenoid</td>
<td>Myxococcus xanthus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000650: 28-7720">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000650/index.html#r1c1" target="_new">BGC0000650</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Terpene</td>
<td>carotenoid</td>
<td>Algoriphagus sp. KK10202C</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000664: 0-8391">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000664/index.html#r1c1" target="_new">BGC0000664</a></td>


<td class="cc-heat-cell" title="0.39 (id:0.43, order:N/A, components:0.35)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
<td>0.39</td>
<td>Terpene</td>
<td>isorenieratene</td>
<td>Streptomyces griseus subsp. griseus NBRC 13350</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000633: 0-14284">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000633/index.html#r1c1" target="_new">BGC0000633</a></td>


<td class="cc-heat-cell" title="0.38 (id:0.42, order:N/A, components:0.35)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
<td>0.38</td>
<td>Terpene</td>
<td>carotenoid</td>
<td>Streptomyces avermitilis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000637: 540-9054">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000637/index.html#r1c1" target="_new">BGC0000637</a></td>


<td class="cc-heat-cell" title="0.38 (id:0.42, order:N/A, components:0.35)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
<td>0.38</td>
<td>Terpene</td>
<td>carotenoid</td>
<td>Corynebacterium glutamicum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000645: 0-7475">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000645/index.html#r1c1" target="_new">BGC0000645</a></td>


<td class="cc-heat-cell" title="0.38 (id:0.43, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
<td>0.38</td>
<td>Terpene</td>
<td>carotenoid</td>
<td>Halobacillus halophilus DSM 2266</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001227: 0-12144">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001227/index.html#r1c1" target="_new">BGC0001227</a></td>


<td class="cc-heat-cell" title="0.38 (id:0.43, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
<td>0.38</td>
<td>Terpene</td>
<td>isorenieratene</td>
<td>Streptomyces collinus Tu 365</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001456: 0-9777">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001456/index.html#r1c1" target="_new">BGC0001456</a></td>


<td class="cc-heat-cell" title="0.37 (id:0.43, order:N/A, components:0.32)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>
<td>0.37</td>
<td>Terpene</td>
<td>isorenieratene</td>
<td>Streptomyces argillaceus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000644: 97-6276">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000644/index.html#r1c1" target="_new">BGC0000644</a></td>


<td class="cc-heat-cell" title="0.34 (id:0.43, order:N/A, components:0.27)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>
<td>0.34</td>
<td>Terpene</td>
<td>carotenoid</td>
<td>Dietzia sp. CQ4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000649: 148-10747">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000649/index.html#r1c1" target="_new">BGC0000649</a></td>


<td class="cc-heat-cell" title="0.33 (id:0.43, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>
<td>0.33</td>
<td>Terpene</td>
<td>carotenoid</td>
<td>Streptomyces griseus subsp. griseus NBRC 13350</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r19c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-239"></div> <span class="help-tooltip" id="cb-general-help-239">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r19c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r19c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r19c1_1.svg">
NW_026250017 (1810074-1842000): Hypoxylon trugodes strain CBS 135444 unplaced... (66% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r19c1_2.svg">
NW_026054578 (88717-120632): Daldinia decipiens strain CBS 113046 unplaced ge... (60% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r19c1_3.svg">
NW_025953253 (251499-283423): Annulohypoxylon maeteangense strain CBS 123835 ... (66% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r19c1_4.svg">
NW_025953792 (736581-768497): Annulohypoxylon truncatum strain CBS 140777 unp... (75% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r19c1_5.svg">
NW_006917097 (701431-733356): Pestalotiopsis fici W106-1 unplaced genomic sca... (62% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r19c1_6.svg">
NC_064677 (5578622-5613132): Colletotrichum lupini chromosome 4, complete seq... (44% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r19c1_7.svg">
NW_026055682 (2068786-2100680): Alternaria ventricosa strain BMP 2768 chromos... (27% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r19c1_8.svg">
NW_022985012 (349248-382118): Dothidotthia symphoricarpi CBS 119687 unplaced ... (21% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r19c1_9.svg">
NW_022983483 (83805-115745): Aaosphaeria arxii CBS 175.79 unplaced genomic sc... (25% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r19c1_10.svg">
NW_026249864 (756472-788366): Alternaria incomplexa strain BMP 0042 chromosom... (27% of genes show similarity), terpene
</option>

</select>
<button id= "clusterblast-r19c1-download">Download graphic</button>
</div>
<div id="clusterblast-r19c1-svg"></div>

</div>
</div>

<div class="body-details-section r19c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-240"></div> <span class="help-tooltip" id="cb-known-help-240">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r19c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-241"></div> <span class="help-tooltip" id="cb-sub-help-241">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r19c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-243"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-243">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r19c1-pfam-details-svg" data-filename="scaffold_19_r19c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r19c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">









</div>









</div>

</div>

</div>





<div class="page" id='r20c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_20 - Region 1 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-775"></div> <span class="help-tooltip" id="region_svg-help-775">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_20.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r20c1-svg" data-filename="scaffold_20_r20c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 233,962 - 282,045 nt. (total: 48,084 nt)

<a class="cluster-rules-header" id="r20c1-rules-header" href="#r20c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r20c1-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r20c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-776"></div> <span class="help-tooltip" id="focus-panel-help-776">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r20c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r20c1-gene-table-details" data-name="r20c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r20c1-nrps_pks" data-name="r20c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r20c1-MIBiG-cluster-compare" data-name="r20c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r20c1-clusterblast" data-name="r20c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r20c1-knownclusterblast" data-name="r20c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r20c1-subclusterblast" data-name="r20c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r20c1-nrps-pks-bubbles" data-name="r20c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r20c1-pfam-details" data-name="r20c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r20c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-252"></div> <span class="help-tooltip" id="gt-help-tooltip-help-252">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r20c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r20c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r20c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r20c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r20c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-244"></div> <span class="help-tooltip" id="nrps-domain-help-244">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r20c1-details-svg" data-filename="scaffold_20_r20c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r20c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r20c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-245"></div> <span class="help-tooltip" id="MIBiG-help-245">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r20c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r20c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r20c1"></div>
<div id="comparison-MIBiG-r20c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001893: 155-19434">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001893/index.html#r1c1" target="_new">BGC0001893</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.34</td>
<td>Other</td>

<td>abscisic acid</td>


<td>Botrytis cinerea B05.10</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002525: 1000-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002525/index.html#r1c1" target="_new">BGC0002525</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.33</td>
<td>Polyketide</td>

<td>fusarubin, 1233A, 1233B, NG-391, lucilactaene</td>


<td>Fusarium sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002162: 0-5465">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002162/index.html#r1c1" target="_new">BGC0002162</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.32</td>
<td>Terpene</td>

<td>sesterfisherol, sesterfisheric acid, compound 6, compound 7a, compound 5, compound 8</td>


<td>Aspergillus fischeri NRRL 181</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002610: 0-6468">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002610/index.html#r1c1" target="_new">BGC0002610</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.32</td>
<td>Terpene</td>

<td>stellatic acid</td>


<td>Aspergillus stellatus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002191: 260-8785">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002191/index.html#r1c1" target="_new">BGC0002191</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.32</td>
<td>Polyketide</td>

<td>prolipyrone B, gibepyrone D</td>


<td>Fusarium graminearum PH-1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002155: 78-16511">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002155/index.html#r1c1" target="_new">BGC0002155</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.44, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.30</td>
<td>Polyketide</td>

<td>nectriapyrone C, nectriapyrone D, nectriapyrone</td>


<td>Pyricularia oryzae 70-15</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002515: 0-30087">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002515/index.html#r1c1" target="_new">BGC0002515</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.43, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.30</td>
<td>Polyketide</td>

<td>solanapyrone A</td>


<td>Ascochyta rabiei</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001252: 0-7958">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001252/index.html#r1c1" target="_new">BGC0001252</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.42, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.30</td>
<td>Polyketide</td>

<td>UNII-YC2Q1O94PT</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001606: 0-22544">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001606/index.html#r1c1" target="_new">BGC0001606</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.30</td>
<td>Polyketide</td>

<td>gibepyrone-A</td>


<td>Fusarium fujikuroi IMI 58289</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002224: 685-37748">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002224/index.html#r1c1" target="_new">BGC0002224</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.29</td>
<td>Polyketide</td>

<td>zopfiellin</td>


<td>Diffractella curvata</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r20c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001893: 155-19434">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001893/index.html#r1c1" target="_new">BGC0001893</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Other</td>
<td>abscisic acid</td>
<td>Botrytis cinerea B05.10</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002525: 1000-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002525/index.html#r1c1" target="_new">BGC0002525</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Polyketide</td>
<td>fusarubin, 1233A, 1233B, NG-391, lucilactaene</td>
<td>Fusarium sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002162: 0-5465">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002162/index.html#r1c1" target="_new">BGC0002162</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Terpene</td>
<td>sesterfisherol, sesterfisheric acid, compound 6, compound 7a, compound 5, compound 8</td>
<td>Aspergillus fischeri NRRL 181</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002610: 0-6468">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002610/index.html#r1c1" target="_new">BGC0002610</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Terpene</td>
<td>stellatic acid</td>
<td>Aspergillus stellatus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002191: 260-8785">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002191/index.html#r1c1" target="_new">BGC0002191</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Polyketide</td>
<td>prolipyrone B, gibepyrone D</td>
<td>Fusarium graminearum PH-1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002155: 78-16511">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002155/index.html#r1c1" target="_new">BGC0002155</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.44, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>Polyketide</td>
<td>nectriapyrone C, nectriapyrone D, nectriapyrone</td>
<td>Pyricularia oryzae 70-15</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002515: 0-30087">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002515/index.html#r1c1" target="_new">BGC0002515</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.43, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>Polyketide</td>
<td>solanapyrone A</td>
<td>Ascochyta rabiei</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001252: 0-7958">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001252/index.html#r1c1" target="_new">BGC0001252</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.42, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>Polyketide</td>
<td>UNII-YC2Q1O94PT</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001606: 0-22544">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001606/index.html#r1c1" target="_new">BGC0001606</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>Polyketide</td>
<td>gibepyrone-A</td>
<td>Fusarium fujikuroi IMI 58289</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002224: 685-37748">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002224/index.html#r1c1" target="_new">BGC0002224</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>Polyketide</td>
<td>zopfiellin</td>
<td>Diffractella curvata</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r20c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-246"></div> <span class="help-tooltip" id="cb-general-help-246">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r20c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r20c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r20c1_1.svg">
NW_025953261 (590289-740142): Annulohypoxylon maeteangense strain CBS 123835 ... (6% of genes show similarity), T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r20c1_2.svg">
NW_024469024 (26525-267638): Aspergillus fijiensis CBS 313.89 unplaced genomi... (3% of genes show similarity), NRPS,T1PKS,fungal-RiPP-like,other
</option>

<option value = "svg/clusterblast_r20c1_3.svg">
NW_020291498 (165271-359753): Aspergillus aculeatinus CBS 121060 unplaced gen... (4% of genes show similarity), NRPS,T1PKS,fungal-RiPP-like,other
</option>

<option value = "svg/clusterblast_r20c1_4.svg">
NW_006917097 (1595225-1663737): Pestalotiopsis fici W106-1 unplaced genomic s... (11% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r20c1_5.svg">
NW_020291592 (339371-529932): Aspergillus piperis CBS 112811 unplaced genomic... (2% of genes show similarity), NRPS-like,T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r20c1_6.svg">
NW_020290659 (1722-70409): Aspergillus saccharolyticus JOP 1030-1 unplaced ge... (13% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r20c1_7.svg">
NW_020292051 (280608-471430): Aspergillus vadensis CBS 113365 unplaced genomi... (2% of genes show similarity), NRPS-like,T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r20c1_8.svg">
NC_036436 (1264208-1374657): Aspergillus oryzae RIB40 DNA, chromosome 2 (6% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r20c1_9.svg">
NW_020291822 (391784-582973): Aspergillus costaricaensis CBS 115574 unplaced ... (2% of genes show similarity), NRPS-like,T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r20c1_10.svg">
NW_022474206 (2604837-2669288): Venustampulla echinocandica strain BP 5553 ch... (13% of genes show similarity), T1PKS
</option>

</select>
<button id= "clusterblast-r20c1-download">Download graphic</button>
</div>
<div id="clusterblast-r20c1-svg"></div>

</div>
</div>

<div class="body-details-section r20c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-247"></div> <span class="help-tooltip" id="cb-known-help-247">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r20c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-248"></div> <span class="help-tooltip" id="cb-sub-help-248">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r20c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-251"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-251">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r20c1-domain-bubble-svg" data-filename="scaffold_20_r20c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r20c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (233961 - 282045): single T1PKS
</option>


</select>

<div class="details-svg" id="r20c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r20c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r20c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-253"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-253">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r20c1-pfam-details-svg" data-filename="scaffold_20_r20c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r20c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r20c1-nrps_pks_products" data-name="r20c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r20c1-nrps_pks_monomers" data-name="r20c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r20c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-249"></div> <span class="help-tooltip" id="nrps-structures-help-249">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 233961 - 282045:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r20c1-1" class="smiles-canvas" data-smiles="C([*])C(-O)(C)"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Me-pk)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-pk" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-pk*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r20c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)(C)"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r20c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-250"></div> <span class="help-tooltip" id="nrps-monomers-help-250">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">FUN_004756</span></strong>: pk
<div class="collapser collapser-target-FUN_004756 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (594..895)</strong>: pk
<div class="collapser collapser-target-FUN_004756 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-FUN_004756-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 66.7%</dd>
<dd>Methylmalonyl-CoA: 58.3%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-FUN_004756-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 90.8</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 82.3</dt>

<dd></dd><dt>Malonyl-CoA: 76.3</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 66.1</dt>

<dd></dd><dt>Propionyl-CoA: 49.8</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 47.4</dt>

<dd></dd><dt>inactive: 44.8</dt>

<dd></dd><dt>Isobutyryl-CoA: 36.6</dt>

<dd></dd><dt>CHC-CoA: 33.1</dt>

<dd></dd><dt>trans-1,2-CPDA: 32.6</dt>

<dd></dd><dt>Benzoyl-CoA: 29.9</dt>

<dd></dd><dt>fatty_acid: 29.2</dt>

<dd></dd><dt>Acetyl-CoA: 16.3</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 15.6</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>












</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r20c2' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_20 - Region 2 - fungal-RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-777"></div> <span class="help-tooltip" id="region_svg-help-777">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_20.region002.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r20c2-svg" data-filename="scaffold_20_r20c2.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 299,772 - 366,449 nt. (total: 66,678 nt)

<a class="cluster-rules-header" id="r20c2-rules-header" href="#r20c2">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r20c2-rules">fungal-RiPP-like: DUF3328</div>

<div class="region-svg-container">
<div id='r20c2-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-778"></div> <span class="help-tooltip" id="focus-panel-help-778">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r20c2">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r20c2-gene-table-details" data-name="r20c2-gene-table-details">
Gene overview
</div>



<div class="body-details-header r20c2-MIBiG-cluster-compare" data-name="r20c2-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r20c2-clusterblast" data-name="r20c2-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r20c2-knownclusterblast" data-name="r20c2-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r20c2-subclusterblast" data-name="r20c2-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r20c2-pfam-details" data-name="r20c2-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r20c2-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-258"></div> <span class="help-tooltip" id="gt-help-tooltip-help-258">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r20c2">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r20c2-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r20c2-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r20c2">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r20c2-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-254"></div> <span class="help-tooltip" id="MIBiG-help-254">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r20c2-selector">Analysis type:</label>
<select id="comparison-MIBiG-r20c2-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r20c2"></div>
<div id="comparison-MIBiG-r20c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">fungal-RiPP-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001398: 4999-64171">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001398/index.html#r1c1" target="_new">BGC0001398</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.43, order:N/A, components:0.63)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.20</td>
<td>RiPP</td>

<td>phomopsin A, phomopsin B, phomopsin E</td>


<td>Phomopsis leptostromiformis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002067: 0-37001">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002067/index.html#r1c1" target="_new">BGC0002067</a></td>





<td class="cc-heat-cell" title="0.33 (id:0.41, order:N/A, components:0.27)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>


<td>0.09</td>
<td>NRP, Polyketide</td>

<td>atpenin B</td>


<td>Penicillium oxalicum 114-2</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001033: 0-58753">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001033/index.html#r1c1" target="_new">BGC0001033</a></td>





<td class="cc-heat-cell" title="0.28 (id:0.41, order:N/A, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>


<td>0.07</td>
<td>NRP, Polyketide</td>

<td>paenilamicin, paenilamicin A2, paenilamicin B1, paenilamicin B2</td>


<td>Paenibacillus larvae subsp. larvae DSM 25430</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002361: 0-35222">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002361/index.html#r1c1" target="_new">BGC0002361</a></td>





<td class="cc-heat-cell" title="0.25 (id:0.45, order:N/A, components:0.13)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>


<td>0.06</td>
<td>Polyketide, Saccharide</td>

<td>baikalomycin B, baikalomycin A, baikalomycin C</td>


<td>Streptomyces sp. IB201691-2A2</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000972: 468-54001">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000972/index.html#r1c1" target="_new">BGC0000972</a></td>





<td class="cc-heat-cell" title="0.23 (id:0.41, order:N/A, components:0.13)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>


<td>0.05</td>
<td>NRP, Polyketide</td>

<td>N-myristoyl-D-asparagine, cis-7-tetradecenoyl-D-asparagine, (R)-N1-((S)-5-oxohexan-2-yl)-2-tetradecanamidosuccinamide</td>


<td>Escherichia coli</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002198: 0-114880">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002198/index.html#r1c1" target="_new">BGC0002198</a></td>





<td class="cc-heat-cell" title="0.23 (id:0.48, order:0.36, components:0.07)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>


<td>0.05</td>
<td>NRP</td>

<td>nanangelenin B, terreazepine</td>


<td>Aspergillus terreus var. terreus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000268: 497-77076">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000268/index.html#r1c1" target="_new">BGC0000268</a></td>





<td class="cc-heat-cell" title="0.21 (id:0.45, order:N/A, components:0.10)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.21), rgb(0, 0, 0, 0.21) 21%, white 21%)"></td>


<td>0.05</td>
<td>Polyketide</td>

<td>Sch-47554, Sch-47555</td>


<td>Streptomyces sp. SCC 2136</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r20c2-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001398: 4999-64171">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001398/index.html#r1c1" target="_new">BGC0001398</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.43, order:N/A, components:0.63)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>RiPP</td>
<td>phomopsin A, phomopsin B, phomopsin E</td>
<td>Phomopsis leptostromiformis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002067: 0-37001">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002067/index.html#r1c1" target="_new">BGC0002067</a></td>


<td class="cc-heat-cell" title="0.33 (id:0.41, order:N/A, components:0.27)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>
<td>0.33</td>
<td>NRP, Polyketide</td>
<td>atpenin B</td>
<td>Penicillium oxalicum 114-2</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001033: 0-58753">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001033/index.html#r1c1" target="_new">BGC0001033</a></td>


<td class="cc-heat-cell" title="0.28 (id:0.41, order:N/A, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>
<td>0.28</td>
<td>NRP, Polyketide</td>
<td>paenilamicin, paenilamicin A2, paenilamicin B1, paenilamicin B2</td>
<td>Paenibacillus larvae subsp. larvae DSM 25430</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002361: 0-35222">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002361/index.html#r1c1" target="_new">BGC0002361</a></td>


<td class="cc-heat-cell" title="0.25 (id:0.45, order:N/A, components:0.13)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>
<td>0.25</td>
<td>Polyketide, Saccharide</td>
<td>baikalomycin B, baikalomycin A, baikalomycin C</td>
<td>Streptomyces sp. IB201691-2A2</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000972: 468-54001">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000972/index.html#r1c1" target="_new">BGC0000972</a></td>


<td class="cc-heat-cell" title="0.23 (id:0.41, order:N/A, components:0.13)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>
<td>0.23</td>
<td>NRP, Polyketide</td>
<td>N-myristoyl-D-asparagine, cis-7-tetradecenoyl-D-asparagine, (R)-N1-((S)-5-oxohexan-2-yl)-2-tetradecanamidosuccinamide</td>
<td>Escherichia coli</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002198: 0-114880">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002198/index.html#r1c1" target="_new">BGC0002198</a></td>


<td class="cc-heat-cell" title="0.23 (id:0.48, order:0.36, components:0.07)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>
<td>0.23</td>
<td>NRP</td>
<td>nanangelenin B, terreazepine</td>
<td>Aspergillus terreus var. terreus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000268: 497-77076">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000268/index.html#r1c1" target="_new">BGC0000268</a></td>


<td class="cc-heat-cell" title="0.21 (id:0.45, order:N/A, components:0.10)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.21), rgb(0, 0, 0, 0.21) 21%, white 21%)"></td>
<td>0.21</td>
<td>Polyketide</td>
<td>Sch-47554, Sch-47555</td>
<td>Streptomyces sp. SCC 2136</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r20c2-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-255"></div> <span class="help-tooltip" id="cb-general-help-255">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r20c2-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r20c2_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r20c2_1.svg">
NC_072354 (1379896-1470826): Talaromyces marneffei chromosome 7, complete seq... (8% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r20c2_2.svg">
NW_022983484 (349700-470118): Aaosphaeria arxii CBS 175.79 unplaced genomic s... (5% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r20c2_3.svg">
NW_017306202 (274026-364977): Alternaria alternata strain SRC1lrK2f unplaced ... (6% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r20c2_4.svg">
NW_017253367 (604257-694877): Paraphaeosphaeria sporulosa strain AP3s5-JAC2a ... (5% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r20c2_5.svg">
NW_017263599 (247828-370093): Mollisia scopiformis unplaced genomic scaffold ... (4% of genes show similarity), T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r20c2_6.svg">
NC_030958 (2200426-2268092): Colletotrichum higginsianum IMI 349063 chromosom... (23% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r20c2_7.svg">
NC_054390 (6640166-6737458): Drechmeria coniospora strain ARSEF 6962 chromoso... (14% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r20c2_8.svg">
NW_024468910 (2525048-2615941): Cryphonectria parasitica EP155 unplaced genom... (6% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r20c2_9.svg">
NC_066262 (1311775-1378556): Aspergillus nidulans FGSC A4 chromosome VI (8% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r20c2_10.svg">
NW_014574707 (595356-657106): Metarhizium brunneum ARSEF 3297 chromosome Unkn... (9% of genes show similarity), NRPS-like
</option>

</select>
<button id= "clusterblast-r20c2-download">Download graphic</button>
</div>
<div id="clusterblast-r20c2-svg"></div>

</div>
</div>

<div class="body-details-section r20c2-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-256"></div> <span class="help-tooltip" id="cb-known-help-256">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r20c2-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-257"></div> <span class="help-tooltip" id="cb-sub-help-257">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r20c2-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-259"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-259">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r20c2-pfam-details-svg" data-filename="scaffold_20_r20c2_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r20c2-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">











</div>











</div>

</div>

</div>





<div class="page" id='r21c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_21 - Region 1 - NRPS-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-779"></div> <span class="help-tooltip" id="region_svg-help-779">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_21.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r21c1-svg" data-filename="scaffold_21_r21c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1 - 37,598 nt. (total: 37,598 nt)

<a class="cluster-rules-header" id="r21c1-rules-header" href="#r21c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r21c1-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r21c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-780"></div> <span class="help-tooltip" id="focus-panel-help-780">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r21c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r21c1-gene-table-details" data-name="r21c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r21c1-nrps_pks" data-name="r21c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r21c1-MIBiG-cluster-compare" data-name="r21c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r21c1-clusterblast" data-name="r21c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r21c1-knownclusterblast" data-name="r21c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r21c1-subclusterblast" data-name="r21c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r21c1-pfam-details" data-name="r21c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r21c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-265"></div> <span class="help-tooltip" id="gt-help-tooltip-help-265">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r21c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r21c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r21c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r21c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r21c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-260"></div> <span class="help-tooltip" id="nrps-domain-help-260">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r21c1-details-svg" data-filename="scaffold_21_r21c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r21c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r21c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-261"></div> <span class="help-tooltip" id="MIBiG-help-261">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r21c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r21c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r21c1"></div>
<div id="comparison-MIBiG-r21c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000099: 0-8104">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000099/index.html#r1c1" target="_new">BGC0000099</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.47, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>Polyketide</td>

<td>monascorubrin</td>


<td>Talaromyces marneffei</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000156: 500-13275">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000156/index.html#r1c1" target="_new">BGC0000156</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.47, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.15</td>
<td>Polyketide</td>

<td>TAN-1612, 1-(2,3,5,10-tetrahydroxy-7-methoxy-4-oxo-1,2,3,4-tetrahydroanthracen-2-yl)pentane-2,4-dione, desmethyl TAN-1612</td>


<td>Aspergillus niger</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001390: 100-19645">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001390/index.html#r1c1" target="_new">BGC0001390</a></td>





<td class="cc-heat-cell" title="0.42 (id:0.45, order:N/A, components:0.39)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>


<td>0.13</td>
<td>NRP, Polyketide</td>

<td>LL-Z1272beta</td>


<td>Stachybotrys bisbyi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002507: 371-17726">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002507/index.html#r1c1" target="_new">BGC0002507</a></td>





<td class="cc-heat-cell" title="0.38 (id:0.44, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>


<td>0.11</td>
<td>Polyketide</td>

<td>sporandol</td>


<td>Chrysosporium merdarium</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001144: 0-15201">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001144/index.html#r1c1" target="_new">BGC0001144</a></td>





<td class="cc-heat-cell" title="0.37 (id:0.45, order:N/A, components:0.31)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>


<td>0.11</td>
<td>Polyketide</td>

<td>neosartoricin B</td>


<td>Trichophyton tonsurans CBS 112818</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000684: 369-21746">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000684/index.html#r1c1" target="_new">BGC0000684</a></td>





<td class="cc-heat-cell" title="0.37 (id:0.46, order:N/A, components:0.29)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>


<td>0.10</td>
<td>Polyketide</td>

<td>asperthecin</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001909: 2476-67696">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001909/index.html#r1c1" target="_new">BGC0001909</a></td>





<td class="cc-heat-cell" title="0.36 (id:0.44, order:N/A, components:0.30)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>


<td>0.10</td>
<td>Polyketide</td>

<td>strobilurin A</td>


<td>Strobilurus tenacellus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001722: 0-18441">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001722/index.html#r1c1" target="_new">BGC0001722</a></td>





<td class="cc-heat-cell" title="0.36 (id:0.45, order:N/A, components:0.28)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>


<td>0.10</td>
<td>Polyketide</td>

<td>aspernidine A</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001338: 273-42813">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001338/index.html#r1c1" target="_new">BGC0001338</a></td>





<td class="cc-heat-cell" title="0.35 (id:0.47, order:N/A, components:0.26)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>


<td>0.10</td>
<td>Polyketide</td>

<td>citrinin</td>


<td>Monascus ruber</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002255: 803-34333">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002255/index.html#r1c1" target="_new">BGC0002255</a></td>





<td class="cc-heat-cell" title="0.33 (id:0.46, order:N/A, components:0.24)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>


<td>0.09</td>
<td>Polyketide</td>

<td>lijiquinone</td>


<td>Ascomycota sp. F53</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r21c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000099: 0-8104">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000099/index.html#r1c1" target="_new">BGC0000099</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.47, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>Polyketide</td>
<td>monascorubrin</td>
<td>Talaromyces marneffei</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000156: 500-13275">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000156/index.html#r1c1" target="_new">BGC0000156</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.47, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>Polyketide</td>
<td>TAN-1612, 1-(2,3,5,10-tetrahydroxy-7-methoxy-4-oxo-1,2,3,4-tetrahydroanthracen-2-yl)pentane-2,4-dione, desmethyl TAN-1612</td>
<td>Aspergillus niger</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001390: 100-19645">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001390/index.html#r1c1" target="_new">BGC0001390</a></td>


<td class="cc-heat-cell" title="0.42 (id:0.45, order:N/A, components:0.39)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
<td>0.42</td>
<td>NRP, Polyketide</td>
<td>LL-Z1272beta</td>
<td>Stachybotrys bisbyi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002507: 371-17726">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002507/index.html#r1c1" target="_new">BGC0002507</a></td>


<td class="cc-heat-cell" title="0.38 (id:0.44, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
<td>0.38</td>
<td>Polyketide</td>
<td>sporandol</td>
<td>Chrysosporium merdarium</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001144: 0-15201">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001144/index.html#r1c1" target="_new">BGC0001144</a></td>


<td class="cc-heat-cell" title="0.37 (id:0.45, order:N/A, components:0.31)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>
<td>0.37</td>
<td>Polyketide</td>
<td>neosartoricin B</td>
<td>Trichophyton tonsurans CBS 112818</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000684: 369-21746">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000684/index.html#r1c1" target="_new">BGC0000684</a></td>


<td class="cc-heat-cell" title="0.37 (id:0.46, order:N/A, components:0.29)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>
<td>0.37</td>
<td>Polyketide</td>
<td>asperthecin</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001909: 2476-67696">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001909/index.html#r1c1" target="_new">BGC0001909</a></td>


<td class="cc-heat-cell" title="0.36 (id:0.44, order:N/A, components:0.30)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>
<td>0.36</td>
<td>Polyketide</td>
<td>strobilurin A</td>
<td>Strobilurus tenacellus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001722: 0-18441">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001722/index.html#r1c1" target="_new">BGC0001722</a></td>


<td class="cc-heat-cell" title="0.36 (id:0.45, order:N/A, components:0.28)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>
<td>0.36</td>
<td>Polyketide</td>
<td>aspernidine A</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001338: 273-42813">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001338/index.html#r1c1" target="_new">BGC0001338</a></td>


<td class="cc-heat-cell" title="0.35 (id:0.47, order:N/A, components:0.26)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>
<td>0.35</td>
<td>Polyketide</td>
<td>citrinin</td>
<td>Monascus ruber</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002255: 803-34333">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002255/index.html#r1c1" target="_new">BGC0002255</a></td>


<td class="cc-heat-cell" title="0.33 (id:0.46, order:N/A, components:0.24)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>
<td>0.33</td>
<td>Polyketide</td>
<td>lijiquinone</td>
<td>Ascomycota sp. F53</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r21c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-262"></div> <span class="help-tooltip" id="cb-general-help-262">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r21c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r21c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r21c1_1.svg">
NC_064677 (5429112-5558494): Colletotrichum lupini chromosome 4, complete seq... (14% of genes show similarity), NRPS-like,T1PKS,indole,terpene
</option>

<option value = "svg/clusterblast_r21c1_2.svg">
NW_006917094 (1817974-1885108): Pestalotiopsis fici W106-1 unplaced genomic s... (9% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r21c1_3.svg">
NW_026054574 (134434-201520): Daldinia decipiens strain CBS 113046 unplaced g... (8% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r21c1_4.svg">
NW_025953045 (1222353-1289479): Daldinia caldariorum strain CBS 122874 unplac... (12% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r21c1_5.svg">
NW_017263603 (640381-705945): Mollisia scopiformis unplaced genomic scaffold ... (8% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r21c1_6.svg">
NW_011942151 (20096-254382): Metarhizium robertsii ARSEF 23 MAA Scf 11, whole... (2% of genes show similarity), NRPS,NRPS-like,T1PKS,fungal-RiPP-like,terpene
</option>

</select>
<button id= "clusterblast-r21c1-download">Download graphic</button>
</div>
<div id="clusterblast-r21c1-svg"></div>

</div>
</div>

<div class="body-details-section r21c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-263"></div> <span class="help-tooltip" id="cb-known-help-263">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r21c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-264"></div> <span class="help-tooltip" id="cb-sub-help-264">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r21c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-266"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-266">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r21c1-pfam-details-svg" data-filename="scaffold_21_r21c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r21c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">













</div>













</div>

</div>

</div>





<div class="page" id='r22c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_22 - Region 1 - fungal-RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-781"></div> <span class="help-tooltip" id="region_svg-help-781">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_22.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r22c1-svg" data-filename="scaffold_22_r22c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 299,445 - 360,472 nt. (total: 61,028 nt)

<a class="cluster-rules-header" id="r22c1-rules-header" href="#r22c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r22c1-rules">fungal-RiPP-like: DUF3328</div>

<div class="region-svg-container">
<div id='r22c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-782"></div> <span class="help-tooltip" id="focus-panel-help-782">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r22c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r22c1-gene-table-details" data-name="r22c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r22c1-MIBiG-cluster-compare" data-name="r22c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r22c1-clusterblast" data-name="r22c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r22c1-knownclusterblast" data-name="r22c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r22c1-subclusterblast" data-name="r22c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r22c1-pfam-details" data-name="r22c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r22c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-271"></div> <span class="help-tooltip" id="gt-help-tooltip-help-271">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r22c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r22c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r22c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r22c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r22c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-267"></div> <span class="help-tooltip" id="MIBiG-help-267">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r22c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r22c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r22c1"></div>
<div id="comparison-MIBiG-r22c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">fungal-RiPP-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002449: 47-14377">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002449/index.html#r1c1" target="_new">BGC0002449</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>Other</td>

<td>sarasinoside</td>


<td>uncultured bacterium</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000673: 0-9097">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000673/index.html#r1c1" target="_new">BGC0000673</a></td>





<td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>


<td>0.15</td>
<td>Terpene</td>

<td>pimara-8(14),15-diene</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001879: 0-39328">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001879/index.html#r1c1" target="_new">BGC0001879</a></td>





<td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>


<td>0.13</td>
<td>Other (Unknown)</td>

<td>type R and F tailocins</td>


<td>Pseudomonas fluorescens</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000627: 0-28295">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000627/index.html#r1c1" target="_new">BGC0000627</a></td>





<td class="cc-heat-cell" title="0.39 (id:0.46, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>


<td>0.12</td>
<td>RiPP</td>

<td>ustiloxin B</td>


<td>Aspergillus flavus NRRL3357</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001398: 4999-64171">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001398/index.html#r1c1" target="_new">BGC0001398</a></td>





<td class="cc-heat-cell" title="0.32 (id:0.45, order:N/A, components:0.23)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.32), rgb(0, 0, 0, 0.32) 32%, white 32%)"></td>


<td>0.09</td>
<td>RiPP</td>

<td>phomopsin A, phomopsin B, phomopsin E</td>


<td>Phomopsis leptostromiformis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000784: 2-17453">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000784/index.html#r1c1" target="_new">BGC0000784</a></td>





<td class="cc-heat-cell" title="0.27 (id:0.41, order:N/A, components:0.18)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>


<td>0.07</td>
<td>Saccharide</td>

<td>O-antigen</td>


<td>Pseudomonas aeruginosa</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000761: 427-27626">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000761/index.html#r1c1" target="_new">BGC0000761</a></td>





<td class="cc-heat-cell" title="0.26 (id:0.42, order:N/A, components:0.16)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>


<td>0.06</td>
<td>Saccharide</td>

<td>exopolysaccharide</td>


<td>Methylobacillus sp. 12S</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001281: 8011-344234">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001281/index.html#r1c1" target="_new">BGC0001281</a></td>





<td class="cc-heat-cell" title="0.23 (id:0.44, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>


<td>0.05</td>
<td>Polyketide</td>

<td>ustilagic acid</td>


<td>Ustilago maydis 521</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000085: 0-68094">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000085/index.html#r1c1" target="_new">BGC0000085</a></td>





<td class="cc-heat-cell" title="0.18 (id:0.41, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.18), rgb(0, 0, 0, 0.18) 18%, white 18%)"></td>


<td>0.04</td>
<td>Polyketide</td>

<td>lankamycin</td>


<td>Streptomyces rochei</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001944: 0-165922">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001944/index.html#r1c1" target="_new">BGC0001944</a></td>





<td class="cc-heat-cell" title="0.15 (id:0.42, order:N/A, components:0.05)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.15), rgb(0, 0, 0, 0.15) 15%, white 15%)"></td>


<td>0.03</td>
<td>Polyketide</td>

<td>funisamine</td>


<td>Streptosporangium sp.</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r22c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002449: 47-14377">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002449/index.html#r1c1" target="_new">BGC0002449</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Other</td>
<td>sarasinoside</td>
<td>uncultured bacterium</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000673: 0-9097">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000673/index.html#r1c1" target="_new">BGC0000673</a></td>


<td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
<td>0.45</td>
<td>Terpene</td>
<td>pimara-8(14),15-diene</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001879: 0-39328">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001879/index.html#r1c1" target="_new">BGC0001879</a></td>


<td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
<td>0.42</td>
<td>Other (Unknown)</td>
<td>type R and F tailocins</td>
<td>Pseudomonas fluorescens</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000627: 0-28295">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000627/index.html#r1c1" target="_new">BGC0000627</a></td>


<td class="cc-heat-cell" title="0.39 (id:0.46, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
<td>0.39</td>
<td>RiPP</td>
<td>ustiloxin B</td>
<td>Aspergillus flavus NRRL3357</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001398: 4999-64171">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001398/index.html#r1c1" target="_new">BGC0001398</a></td>


<td class="cc-heat-cell" title="0.32 (id:0.45, order:N/A, components:0.23)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.32), rgb(0, 0, 0, 0.32) 32%, white 32%)"></td>
<td>0.32</td>
<td>RiPP</td>
<td>phomopsin A, phomopsin B, phomopsin E</td>
<td>Phomopsis leptostromiformis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000784: 2-17453">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000784/index.html#r1c1" target="_new">BGC0000784</a></td>


<td class="cc-heat-cell" title="0.27 (id:0.41, order:N/A, components:0.18)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>
<td>0.27</td>
<td>Saccharide</td>
<td>O-antigen</td>
<td>Pseudomonas aeruginosa</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000761: 427-27626">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000761/index.html#r1c1" target="_new">BGC0000761</a></td>


<td class="cc-heat-cell" title="0.26 (id:0.42, order:N/A, components:0.16)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>
<td>0.26</td>
<td>Saccharide</td>
<td>exopolysaccharide</td>
<td>Methylobacillus sp. 12S</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001281: 8011-344234">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001281/index.html#r1c1" target="_new">BGC0001281</a></td>


<td class="cc-heat-cell" title="0.23 (id:0.44, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>
<td>0.23</td>
<td>Polyketide</td>
<td>ustilagic acid</td>
<td>Ustilago maydis 521</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000085: 0-68094">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000085/index.html#r1c1" target="_new">BGC0000085</a></td>


<td class="cc-heat-cell" title="0.18 (id:0.41, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.18), rgb(0, 0, 0, 0.18) 18%, white 18%)"></td>
<td>0.18</td>
<td>Polyketide</td>
<td>lankamycin</td>
<td>Streptomyces rochei</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001944: 0-165922">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001944/index.html#r1c1" target="_new">BGC0001944</a></td>


<td class="cc-heat-cell" title="0.15 (id:0.42, order:N/A, components:0.05)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.15), rgb(0, 0, 0, 0.15) 15%, white 15%)"></td>
<td>0.15</td>
<td>Polyketide</td>
<td>funisamine</td>
<td>Streptosporangium sp.</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r22c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-268"></div> <span class="help-tooltip" id="cb-general-help-268">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r22c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r22c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r22c1_1.svg">
NW_025953810 (264034-384248): Annulohypoxylon truncatum strain CBS 140777 unp... (4% of genes show similarity), NRPS-like,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r22c1_2.svg">
NW_001820816 (98387-215394): Sclerotinia sclerotiorum 1980 UF-70 scaffold 20 ... (4% of genes show similarity), T1PKS,fungal-RiPP-like
</option>

</select>
<button id= "clusterblast-r22c1-download">Download graphic</button>
</div>
<div id="clusterblast-r22c1-svg"></div>

</div>
</div>

<div class="body-details-section r22c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-269"></div> <span class="help-tooltip" id="cb-known-help-269">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r22c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-270"></div> <span class="help-tooltip" id="cb-sub-help-270">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r22c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-272"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-272">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r22c1-pfam-details-svg" data-filename="scaffold_22_r22c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r22c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">











</div>











</div>

</div>

</div>





<div class="page" id='r24c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_24 - Region 1 - NRP-metallophore,NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-783"></div> <span class="help-tooltip" id="region_svg-help-783">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_24.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r24c1-svg" data-filename="scaffold_24_r24c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 23,601 - 84,623 nt. (total: 61,023 nt)

<a class="cluster-rules-header" id="r24c1-rules-header" href="#r24c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r24c1-rules">NRP-metallophore: (VibH_like or Cy_tandem or (cds(Condensation and AMP-binding) and ((IBH_Asp and not SBH_Asp) or IBH_His or TBH_Asp or CyanoBH_Asp1 or CyanoBH_Asp2 or IPL or SalSyn or (EntA and EntC) or (GrbD and GrbE) or (FbnL and FbnM) or PvdO or PvdP or (Orn_monoox and not (KtzT or MetRS-like)) or Lys_monoox or VbsL)))<br>NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r24c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-784"></div> <span class="help-tooltip" id="focus-panel-help-784">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r24c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r24c1-gene-table-details" data-name="r24c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r24c1-nrps_pks" data-name="r24c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r24c1-MIBiG-cluster-compare" data-name="r24c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r24c1-clusterblast" data-name="r24c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r24c1-knownclusterblast" data-name="r24c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r24c1-subclusterblast" data-name="r24c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r24c1-nrps-pks-bubbles" data-name="r24c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r24c1-pfam-details" data-name="r24c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r24c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-281"></div> <span class="help-tooltip" id="gt-help-tooltip-help-281">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r24c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r24c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r24c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r24c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r24c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-273"></div> <span class="help-tooltip" id="nrps-domain-help-273">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r24c1-details-svg" data-filename="scaffold_24_r24c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r24c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r24c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-274"></div> <span class="help-tooltip" id="MIBiG-help-274">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r24c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r24c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r24c1"></div>
<div id="comparison-MIBiG-r24c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRP-metallophore</span></th>

<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001527: 0-8170">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001527/index.html#r1c1" target="_new">BGC0001527</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>






<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.63</td>
<td>Other</td>

<td>basidioferrin</td>


<td>Gelatoporia subvermispora</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001251: 1268-15341">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001251/index.html#r1c1" target="_new">BGC0001251</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>






<td class="cc-heat-cell" title="0.58 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.50</td>
<td>NRP</td>

<td>epichloenin A</td>


<td>Epichloe festucae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132: 0-12417">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>






<td class="cc-heat-cell" title="0.53 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.40</td>
<td>NRP</td>

<td>xenotetrapeptide</td>


<td>Xenorhabdus nematophila ATCC 19061</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002518: 0-10561">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002518/index.html#r1c1" target="_new">BGC0002518</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>






<td class="cc-heat-cell" title="0.53 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.40</td>
<td>NRP</td>

<td>syringafactin A, syringafactin C</td>


<td>Pseudomonas sp. SZ57</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135: 0-16848">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>






<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.40</td>
<td>NRP</td>

<td>bicornutin A1, bicornutin A2</td>


<td>Xenorhabdus budapestensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>






<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.40</td>
<td>NRP</td>

<td>AM-toxin</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844: 234-20663">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>






<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.40</td>
<td>NRP</td>

<td>holrhizin</td>


<td>Paraburkholderia rhizoxinica HKI 454</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128: 35-15686">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>






<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.40</td>
<td>NRP</td>

<td>gamexpeptide C</td>


<td>Photorhabdus laumondii subsp. laumondii TTO1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758: 0-22977">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>






<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.40</td>
<td>NRP</td>

<td>rhizomide A, rhizomide B, rhizomide C</td>


<td>Paraburkholderia rhizoxinica HKI 454</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001641: 0-49104">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001641/index.html#r1c1" target="_new">BGC0001641</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>






<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.40</td>
<td>NRP</td>

<td>kolossin</td>


<td>Photorhabdus laumondii subsp. laumondii TTO1</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r24c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001527: 0-8170">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001527/index.html#r1c1" target="_new">BGC0001527</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Other</td>
<td>basidioferrin</td>
<td>Gelatoporia subvermispora</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001251: 1268-15341">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001251/index.html#r1c1" target="_new">BGC0001251</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>NRP</td>
<td>epichloenin A</td>
<td>Epichloe festucae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132: 0-12417">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>NRP</td>
<td>xenotetrapeptide</td>
<td>Xenorhabdus nematophila ATCC 19061</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002518: 0-10561">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002518/index.html#r1c1" target="_new">BGC0002518</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>NRP</td>
<td>syringafactin A, syringafactin C</td>
<td>Pseudomonas sp. SZ57</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135: 0-16848">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>NRP</td>
<td>bicornutin A1, bicornutin A2</td>
<td>Xenorhabdus budapestensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>NRP</td>
<td>AM-toxin</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001844: 234-20663">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001844/index.html#r1c1" target="_new">BGC0001844</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>NRP</td>
<td>holrhizin</td>
<td>Paraburkholderia rhizoxinica HKI 454</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128: 35-15686">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>NRP</td>
<td>gamexpeptide C</td>
<td>Photorhabdus laumondii subsp. laumondii TTO1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001758: 0-22977">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001758/index.html#r1c1" target="_new">BGC0001758</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>NRP</td>
<td>rhizomide A, rhizomide B, rhizomide C</td>
<td>Paraburkholderia rhizoxinica HKI 454</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001641: 0-49104">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001641/index.html#r1c1" target="_new">BGC0001641</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>NRP</td>
<td>kolossin</td>
<td>Photorhabdus laumondii subsp. laumondii TTO1</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r24c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-275"></div> <span class="help-tooltip" id="cb-general-help-275">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r24c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r24c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r24c1_1.svg">
NW_026054569 (345332-425705): Daldinia decipiens strain CBS 113046 unplaced g... (35% of genes show similarity), NRP-metallophore,NRPS
</option>

<option value = "svg/clusterblast_r24c1_2.svg">
NW_025953796 (8058-88896): Annulohypoxylon truncatum strain CBS 140777 unplac... (33% of genes show similarity), NRP-metallophore,NRPS
</option>

<option value = "svg/clusterblast_r24c1_3.svg">
NW_024468916 (858969-933881): Cryphonectria parasitica EP155 unplaced genomic... (37% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r24c1_4.svg">
NC_030959 (128983-262697): Colletotrichum higginsianum IMI 349063 chromosome ... (10% of genes show similarity), NRPS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r24c1_5.svg">
NC_064678 (6600642-6675608): Colletotrichum lupini chromosome 5, complete seq... (25% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r24c1_6.svg">
NC_070530 (3742001-3826089): Fusarium keratoplasticum isolate Fu6.1 chromosom... (17% of genes show similarity), NRP-metallophore,NRPS
</option>

<option value = "svg/clusterblast_r24c1_7.svg">
NC_070545 (3816730-3900832): Fusarium falciforme chromosome 2, complete sequence (17% of genes show similarity), NRP-metallophore,NRPS
</option>

<option value = "svg/clusterblast_r24c1_8.svg">
NC_036623 (2195750-2279651): Fusarium fujikuroi IMI 58289 draft genome, chrom... (16% of genes show similarity), NRP-metallophore,NRPS
</option>

<option value = "svg/clusterblast_r24c1_9.svg">
NW_006711178 (9878-84649): Trichoderma reesei QM6a unplaced genomic scaffold ... (13% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r24c1_10.svg">
NC_058389 (2698749-2782755): Fusarium musae strain F31 chromosome 3, whole ge... (16% of genes show similarity), NRP-metallophore,NRPS
</option>

</select>
<button id= "clusterblast-r24c1-download">Download graphic</button>
</div>
<div id="clusterblast-r24c1-svg"></div>

</div>
</div>

<div class="body-details-section r24c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-276"></div> <span class="help-tooltip" id="cb-known-help-276">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r24c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-277"></div> <span class="help-tooltip" id="cb-sub-help-277">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r24c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-280"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-280">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r24c1-domain-bubble-svg" data-filename="scaffold_24_r24c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r24c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (23600 - 84623): chemical hybrid NRP-metallophore-NRPS
</option>


</select>

<div class="details-svg" id="r24c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r24c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r24c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-282"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-282">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r24c1-pfam-details-svg" data-filename="scaffold_24_r24c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r24c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r24c1-nrps_pks_products" data-name="r24c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r24c1-nrps_pks_monomers" data-name="r24c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r24c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-278"></div> <span class="help-tooltip" id="nrps-structures-help-278">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 23600 - 84623:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r24c1-1" class="smiles-canvas" data-smiles="NC([*])C(=O)NC([*])C(=O)NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X - X - X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x,x,x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r24c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)NC([*])C(=O)NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r24c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-279"></div> <span class="help-tooltip" id="nrps-monomers-help-279">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">FUN_005408</span></strong>: X - X - X
<div class="collapser collapser-target-FUN_005408 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (93..450)</strong>: X
<div class="collapser collapser-target-FUN_005408 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  (unknown)
<div class="collapser collapser-target-FUN_005408-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>N/A</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DVFELIMIHK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Orn <span class="serif">DVSELGTINK</span> (59% 8 match)</dd>
<dd>Ser <span class="serif">DLFNLGLIHK</span> (44% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>60% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>







<dd><strong>AMP-binding (1124..1479)</strong>: X
<div class="collapser collapser-target-FUN_005408 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  (unknown)
<div class="collapser collapser-target-FUN_005408-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>  <strong>NOTE: uncertain match</strong><br>
<dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DVFSVSV--K</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Ala <span class="serif">DVFSVAVVYK</span> (38% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>70% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>











<dd><strong>AMP-binding (2703..3088)</strong>: X
<div class="collapser collapser-target-FUN_005408 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  (unknown)
<div class="collapser collapser-target-FUN_005408-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>N/A</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DVLDIGAIGK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Val <span class="serif">DVLSFGAIIK</span> (65% 8 match)</dd>
<dd>Phe or Trp <span class="serif">DVLSIGAVVK</span> (62% 8 match)</dd>
<dd>Arg <span class="serif">DVESIGAIAK</span> (53% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>70% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>














</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r24c2' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_24 - Region 2 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-785"></div> <span class="help-tooltip" id="region_svg-help-785">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_24.region002.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r24c2-svg" data-filename="scaffold_24_r24c2.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 454,601 - 501,516 nt. (total: 46,916 nt)

<a class="cluster-rules-header" id="r24c2-rules-header" href="#r24c2">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r24c2-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r24c2-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-786"></div> <span class="help-tooltip" id="focus-panel-help-786">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r24c2">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r24c2-gene-table-details" data-name="r24c2-gene-table-details">
Gene overview
</div>



<div class="body-details-header r24c2-nrps_pks" data-name="r24c2-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r24c2-MIBiG-cluster-compare" data-name="r24c2-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r24c2-clusterblast" data-name="r24c2-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r24c2-knownclusterblast" data-name="r24c2-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r24c2-subclusterblast" data-name="r24c2-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r24c2-nrps-pks-bubbles" data-name="r24c2-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r24c2-pfam-details" data-name="r24c2-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r24c2-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-291"></div> <span class="help-tooltip" id="gt-help-tooltip-help-291">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r24c2">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r24c2-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r24c2-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r24c2">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r24c2-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-283"></div> <span class="help-tooltip" id="nrps-domain-help-283">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r24c2-details-svg" data-filename="scaffold_24_r24c2_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r24c2-details-svg"></div>
</div>
</div>



<div class="body-details-section r24c2-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-284"></div> <span class="help-tooltip" id="MIBiG-help-284">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r24c2-selector">Analysis type:</label>
<select id="comparison-MIBiG-r24c2-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r24c2"></div>
<div id="comparison-MIBiG-r24c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002233: 556-31132">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002233/index.html#r1c1" target="_new">BGC0002233</a></td>





<td class="cc-heat-cell" title="0.68 (id:0.55, order:0.61, components:0.96)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>


<td>0.35</td>
<td>Polyketide</td>

<td>trichoxide</td>


<td>Trichoderma virens Gv29-8</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002234: 0-20938">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002234/index.html#r1c1" target="_new">BGC0002234</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.52, order:0.64, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.32</td>
<td>Polyketide</td>

<td>flavoglaucin</td>


<td>Aspergillus ruber CBS 135680</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264: 8442-40380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.45, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.30</td>
<td>Polyketide</td>

<td>betaenone A, betaenone B, betaenone C</td>


<td>Phoma betae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001124: 3228-30082">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001124/index.html#r1c1" target="_new">BGC0001124</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.29</td>
<td>Polyketide</td>

<td>pyranonigrin E</td>


<td>Aspergillus niger ATCC 1015</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000046: 1267-21753">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000046/index.html#r1c1" target="_new">BGC0000046</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.28</td>
<td>Polyketide</td>

<td>depudecin</td>


<td>Alternaria brassicicola</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002228: 282-17978">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002228/index.html#r1c1" target="_new">BGC0002228</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.43, order:N/A, components:0.85)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.27</td>
<td>NRP</td>

<td>CIML B, CIML A, CIML D, CIML C</td>


<td>Colletotrichum incanum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002211: 74-16330">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002211/index.html#r1c1" target="_new">BGC0002211</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.85)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.27</td>
<td>Polyketide</td>

<td>HEx-pks1 polyketide</td>


<td>Paraphaeosphaeria sporulosa</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001141: 262-37660">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001141/index.html#r1c1" target="_new">BGC0001141</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.26</td>
<td>Polyketide</td>

<td>4-epi-15-epi-brefeldin A</td>


<td>Penicillium brefeldianum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001067: 0-35154">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001067/index.html#r1c1" target="_new">BGC0001067</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.43, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.25</td>
<td>Polyketide, Terpene</td>

<td>fumagillin, -trans-bergamotene, fumagillol</td>


<td>Aspergillus fumigatus Af293</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001504: 0-13734">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001504/index.html#r1c1" target="_new">BGC0001504</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.43, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.25</td>
<td>Polyketide</td>

<td>amycomicin</td>


<td>Amycolatopsis sp. AA4</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r24c2-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002233: 556-31132">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002233/index.html#r1c1" target="_new">BGC0002233</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.55, order:0.61, components:0.96)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>Polyketide</td>
<td>trichoxide</td>
<td>Trichoderma virens Gv29-8</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002234: 0-20938">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002234/index.html#r1c1" target="_new">BGC0002234</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.52, order:0.64, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Polyketide</td>
<td>flavoglaucin</td>
<td>Aspergillus ruber CBS 135680</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264: 8442-40380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.45, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>Polyketide</td>
<td>betaenone A, betaenone B, betaenone C</td>
<td>Phoma betae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001124: 3228-30082">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001124/index.html#r1c1" target="_new">BGC0001124</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>Polyketide</td>
<td>pyranonigrin E</td>
<td>Aspergillus niger ATCC 1015</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000046: 1267-21753">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000046/index.html#r1c1" target="_new">BGC0000046</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>Polyketide</td>
<td>depudecin</td>
<td>Alternaria brassicicola</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002228: 282-17978">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002228/index.html#r1c1" target="_new">BGC0002228</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.43, order:N/A, components:0.85)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>NRP</td>
<td>CIML B, CIML A, CIML D, CIML C</td>
<td>Colletotrichum incanum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002211: 74-16330">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002211/index.html#r1c1" target="_new">BGC0002211</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.85)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>Polyketide</td>
<td>HEx-pks1 polyketide</td>
<td>Paraphaeosphaeria sporulosa</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001141: 262-37660">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001141/index.html#r1c1" target="_new">BGC0001141</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>Polyketide</td>
<td>4-epi-15-epi-brefeldin A</td>
<td>Penicillium brefeldianum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001067: 0-35154">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001067/index.html#r1c1" target="_new">BGC0001067</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.43, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Polyketide, Terpene</td>
<td>fumagillin, -trans-bergamotene, fumagillol</td>
<td>Aspergillus fumigatus Af293</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001504: 0-13734">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001504/index.html#r1c1" target="_new">BGC0001504</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.43, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Polyketide</td>
<td>amycomicin</td>
<td>Amycolatopsis sp. AA4</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r24c2-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-285"></div> <span class="help-tooltip" id="cb-general-help-285">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r24c2-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r24c2_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r24c2_1.svg">
NW_014013663 (509438-653982): Trichoderma virens Gv29-8 chromosome Unknown TR... (25% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r24c2_2.svg">
NW_020290619 (11115-78674): Aspergillus saccharolyticus JOP 1030-1 unplaced g... (33% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r24c2_3.svg">
NW_006711162 (541322-608930): Trichoderma reesei QM6a unplaced genomic scaffo... (47% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r24c2_4.svg">
NC_057364 (496585-564367): Aspergillus chevalieri M1 DNA, chromosome 3, nearl... (33% of genes show similarity), T1PKS,indole
</option>

<option value = "svg/clusterblast_r24c2_5.svg">
NW_024467308 (148929-260597): Aspergillus ruber CBS 135680 unplaced genomic s... (17% of genes show similarity), T1PKS,indole
</option>

<option value = "svg/clusterblast_r24c2_6.svg">
NW_019170464 (740714-852986): Aspergillus glaucus CBS 516.65 unplaced genomic... (15% of genes show similarity), T1PKS,indole
</option>

<option value = "svg/clusterblast_r24c2_7.svg">
NC_036631 (2086864-2154579): Fusarium fujikuroi IMI 58289 draft genome, chrom... (37% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r24c2_8.svg">
NW_026054548 (624109-703016): Daldinia decipiens strain CBS 113046 unplaced g... (34% of genes show similarity), T1PKS,terpene
</option>

<option value = "svg/clusterblast_r24c2_9.svg">
NW_025953039 (501888-574789): Daldinia caldariorum strain CBS 122874 unplaced... (34% of genes show similarity), T1PKS,terpene
</option>

<option value = "svg/clusterblast_r24c2_10.svg">
NC_030956 (1262040-1337854): Colletotrichum higginsianum IMI 349063 chromosom... (34% of genes show similarity), T1PKS,indole
</option>

</select>
<button id= "clusterblast-r24c2-download">Download graphic</button>
</div>
<div id="clusterblast-r24c2-svg"></div>

</div>
</div>

<div class="body-details-section r24c2-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-286"></div> <span class="help-tooltip" id="cb-known-help-286">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r24c2-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r24c2_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r24c2_1.svg">
trichoxide
</option>

<option value = "svg/knownclusterblast_r24c2_2.svg">
flavoglaucin
</option>

<option value = "svg/knownclusterblast_r24c2_3.svg">
sordarial
</option>

<option value = "svg/knownclusterblast_r24c2_4.svg">
epipyriculol
</option>

</select>
<button id="knownclusterblast-r24c2-download">Download graphic</button>
</div>
<div id="knownclusterblast-r24c2-svg"></div>

</div>
</div>

<div class="body-details-section r24c2-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-287"></div> <span class="help-tooltip" id="cb-sub-help-287">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r24c2-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-290"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-290">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r24c2-domain-bubble-svg" data-filename="scaffold_24_r24c2_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r24c2-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC2">
Candidate 2 (454600 - 501516): single T1PKS
</option>


</select>

<div class="details-svg" id="r24c2-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r24c2-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r24c2-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-292"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-292">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r24c2-pfam-details-svg" data-filename="scaffold_24_r24c2_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r24c2-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r24c2-nrps_pks_products" data-name="r24c2-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r24c2-nrps_pks_monomers" data-name="r24c2-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r24c2-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-288"></div> <span class="help-tooltip" id="nrps-structures-help-288">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 2, location 454600 - 501516:
<div class="collapser collapser-target-CC2 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r24c2-2" class="smiles-canvas" data-smiles="C([*])C(-O)"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS linker matching; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(pk)</dd>

<dt><b>Non-colinear ordering:</b></dt><dd>

<span class="serif">FUN_005528</span>

+ <span class="serif">FUN_005529</span>

</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r24c2-2-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r24c2-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-289"></div> <span class="help-tooltip" id="nrps-monomers-help-289">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">FUN_005528</span></strong>: pk
<div class="collapser collapser-target-FUN_005528 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (545..869)</strong>: pk
<div class="collapser collapser-target-FUN_005528 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-FUN_005528-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 62.5%</dd>
<dd>Ethylmalonyl-CoA: 58.3%</dd>
<dd>Methylmalonyl-CoA: 58.3%</dd>
</dl>

</div><br>

minowa:  Methoxymalonyl-CoA
<div class="collapser collapser-target-FUN_005528-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methoxymalonyl-CoA: 62.6</dt>

<dd></dd><dt>Methylmalonyl-CoA: 60.5</dt>

<dd></dd><dt>Malonyl-CoA: 35.8</dt>

<dd></dd><dt>Isobutyryl-CoA: 33.9</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 24.6</dt>

<dd></dd><dt>fatty_acid: 18.0</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 14.8</dt>

<dd></dd><dt>inactive: 14.3</dt>

<dd></dd><dt>Propionyl-CoA: 13.7</dt>

<dd></dd><dt>Benzoyl-CoA: 11.9</dt>

<dd></dd><dt>CHC-CoA: 10.4</dt>

<dd></dd><dt>Acetyl-CoA: 10.4</dt>

<dd></dd><dt>trans-1,2-CPDA: 9.4</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>





<div class="page" id='r26c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_26 - Region 1 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-787"></div> <span class="help-tooltip" id="region_svg-help-787">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_26.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r26c1-svg" data-filename="scaffold_26_r26c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 215,940 - 262,542 nt. (total: 46,603 nt)

<a class="cluster-rules-header" id="r26c1-rules-header" href="#r26c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r26c1-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r26c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-788"></div> <span class="help-tooltip" id="focus-panel-help-788">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r26c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r26c1-gene-table-details" data-name="r26c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r26c1-nrps_pks" data-name="r26c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r26c1-MIBiG-cluster-compare" data-name="r26c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r26c1-clusterblast" data-name="r26c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r26c1-knownclusterblast" data-name="r26c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r26c1-subclusterblast" data-name="r26c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r26c1-nrps-pks-bubbles" data-name="r26c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r26c1-pfam-details" data-name="r26c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r26c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-301"></div> <span class="help-tooltip" id="gt-help-tooltip-help-301">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r26c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r26c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r26c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r26c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r26c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-293"></div> <span class="help-tooltip" id="nrps-domain-help-293">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r26c1-details-svg" data-filename="scaffold_26_r26c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r26c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r26c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-294"></div> <span class="help-tooltip" id="MIBiG-help-294">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r26c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r26c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r26c1"></div>
<div id="comparison-MIBiG-r26c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001257: 399-7066">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001257/index.html#r1c1" target="_new">BGC0001257</a></td>





<td class="cc-heat-cell" title="0.79 (id:0.62, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.79), rgb(0, 0, 0, 0.79) 79%, white 79%)"></td>


<td>0.51</td>
<td>Polyketide</td>

<td>1,3,6,8-tetrahydroxynaphthalene</td>


<td>Nodulisporium sp. ATCC74245</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002154: 337-7126">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002154/index.html#r1c1" target="_new">BGC0002154</a></td>





<td class="cc-heat-cell" title="0.75 (id:0.57, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.75), rgb(0, 0, 0, 0.75) 75%, white 75%)"></td>


<td>0.45</td>
<td>Polyketide</td>

<td>1,3,5,8-tetrahydroxynapthalene, 1-(alpha-l-(2-O-methyl)-6-deoxymannopyranosyloxy)-3,6,8-trimethoxynaphthalene</td>


<td>Pyricularia oryzae 70-15</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001258: 43-7265">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001258/index.html#r1c1" target="_new">BGC0001258</a></td>





<td class="cc-heat-cell" title="0.73 (id:0.54, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>


<td>0.42</td>
<td>Polyketide</td>

<td>1,3,6,8-tetrahydroxynaphthalene</td>


<td>Glarea lozoyensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002175: 214-6865">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002175/index.html#r1c1" target="_new">BGC0002175</a></td>





<td class="cc-heat-cell" title="0.69 (id:0.48, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>


<td>0.37</td>
<td>Polyketide</td>

<td>YWA1</td>


<td>Aspergillus oryzae RIB40</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000107: 144-6819">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000107/index.html#r1c1" target="_new">BGC0000107</a></td>





<td class="cc-heat-cell" title="0.69 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>


<td>0.36</td>
<td>Polyketide</td>

<td>naphthopyrone</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001284: 0-5292">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001284/index.html#r1c1" target="_new">BGC0001284</a></td>





<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>


<td>0.35</td>
<td>Polyketide</td>

<td>alternariol</td>


<td>Parastagonospora nodorum SN15</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000156: 500-13275">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000156/index.html#r1c1" target="_new">BGC0000156</a></td>





<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>


<td>0.35</td>
<td>Polyketide</td>

<td>TAN-1612, 1-(2,3,5,10-tetrahydroxy-7-methoxy-4-oxo-1,2,3,4-tetrahydroanthracen-2-yl)pentane-2,4-dione, desmethyl TAN-1612</td>


<td>Aspergillus niger</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001266: 10707-26494">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001266/index.html#r1c1" target="_new">BGC0001266</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.46, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.32</td>
<td>Polyketide</td>

<td>grayanic acid</td>


<td>Cladonia grayi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002161: 1109-36992">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002161/index.html#r1c1" target="_new">BGC0002161</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.57, order:0.51, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.31</td>
<td>Polyketide</td>

<td>scytalone, T3HN</td>


<td>Pestalotiopsis fici W106-1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000684: 369-21746">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000684/index.html#r1c1" target="_new">BGC0000684</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.46, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.30</td>
<td>Polyketide</td>

<td>asperthecin</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r26c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001257: 399-7066">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001257/index.html#r1c1" target="_new">BGC0001257</a></td>


<td class="cc-heat-cell" title="0.79 (id:0.62, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.79), rgb(0, 0, 0, 0.79) 79%, white 79%)"></td>
<td>0.79</td>
<td>Polyketide</td>
<td>1,3,6,8-tetrahydroxynaphthalene</td>
<td>Nodulisporium sp. ATCC74245</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002154: 337-7126">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002154/index.html#r1c1" target="_new">BGC0002154</a></td>


<td class="cc-heat-cell" title="0.75 (id:0.57, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.75), rgb(0, 0, 0, 0.75) 75%, white 75%)"></td>
<td>0.75</td>
<td>Polyketide</td>
<td>1,3,5,8-tetrahydroxynapthalene, 1-(alpha-l-(2-O-methyl)-6-deoxymannopyranosyloxy)-3,6,8-trimethoxynaphthalene</td>
<td>Pyricularia oryzae 70-15</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001258: 43-7265">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001258/index.html#r1c1" target="_new">BGC0001258</a></td>


<td class="cc-heat-cell" title="0.73 (id:0.54, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>
<td>0.73</td>
<td>Polyketide</td>
<td>1,3,6,8-tetrahydroxynaphthalene</td>
<td>Glarea lozoyensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002175: 214-6865">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002175/index.html#r1c1" target="_new">BGC0002175</a></td>


<td class="cc-heat-cell" title="0.69 (id:0.48, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
<td>0.69</td>
<td>Polyketide</td>
<td>YWA1</td>
<td>Aspergillus oryzae RIB40</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000107: 144-6819">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000107/index.html#r1c1" target="_new">BGC0000107</a></td>


<td class="cc-heat-cell" title="0.69 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
<td>0.69</td>
<td>Polyketide</td>
<td>naphthopyrone</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001284: 0-5292">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001284/index.html#r1c1" target="_new">BGC0001284</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>Polyketide</td>
<td>alternariol</td>
<td>Parastagonospora nodorum SN15</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000156: 500-13275">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000156/index.html#r1c1" target="_new">BGC0000156</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>Polyketide</td>
<td>TAN-1612, 1-(2,3,5,10-tetrahydroxy-7-methoxy-4-oxo-1,2,3,4-tetrahydroanthracen-2-yl)pentane-2,4-dione, desmethyl TAN-1612</td>
<td>Aspergillus niger</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001266: 10707-26494">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001266/index.html#r1c1" target="_new">BGC0001266</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.46, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Polyketide</td>
<td>grayanic acid</td>
<td>Cladonia grayi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002161: 1109-36992">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002161/index.html#r1c1" target="_new">BGC0002161</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.57, order:0.51, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Polyketide</td>
<td>scytalone, T3HN</td>
<td>Pestalotiopsis fici W106-1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000684: 369-21746">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000684/index.html#r1c1" target="_new">BGC0000684</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.46, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>Polyketide</td>
<td>asperthecin</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r26c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-295"></div> <span class="help-tooltip" id="cb-general-help-295">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r26c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r26c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r26c1_1.svg">
NW_026250019 (722171-788775): Hypoxylon trugodes strain CBS 135444 unplaced g... (28% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r26c1_2.svg">
NW_025953265 (611975-678563): Annulohypoxylon maeteangense strain CBS 123835 ... (36% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r26c1_3.svg">
NW_026054544 (710053-825119): Daldinia decipiens strain CBS 113046 unplaced g... (13% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r26c1_4.svg">
NW_024468917 (334387-401101): Cryphonectria parasitica EP155 unplaced genomic... (28% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r26c1_5.svg">
NW_025953044 (465004-531610): Daldinia caldariorum strain CBS 122874 unplaced... (33% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r26c1_6.svg">
NW_009276921 (649963-716705): Verticillium dahliae VdLs.17 supercont1.1 genom... (26% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r26c1_7.svg">
NW_024065901 (83116-149857): Colletotrichum karsti strain CkLH20 chromosome U... (75% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r26c1_8.svg">
NC_030959 (1613218-1680027): Colletotrichum higginsianum IMI 349063 chromosom... (44% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r26c1_9.svg">
NC_043741 (6601956-6668728): Pyricularia pennisetigena strain Br36 chromosome... (26% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r26c1_10.svg">
NC_064678 (4708621-4845863): Colletotrichum lupini chromosome 5, complete seq... (12% of genes show similarity), NRPS,T1PKS
</option>

</select>
<button id= "clusterblast-r26c1-download">Download graphic</button>
</div>
<div id="clusterblast-r26c1-svg"></div>

</div>
</div>

<div class="body-details-section r26c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-296"></div> <span class="help-tooltip" id="cb-known-help-296">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r26c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r26c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r26c1_1.svg">
naphthalene
</option>

<option value = "svg/knownclusterblast_r26c1_2.svg">
scytalone/T3HN
</option>

<option value = "svg/knownclusterblast_r26c1_3.svg">
1,3,6,8-tetrahydroxynaphthalene
</option>

<option value = "svg/knownclusterblast_r26c1_4.svg">
1,3,5,8-tetrahydroxynapthalene/1-(alpha-l-(2-O-methyl)-6-deoxymannopyranosyloxy)-3,6,8-trimethoxynaphthalene
</option>

<option value = "svg/knownclusterblast_r26c1_5.svg">
1,3,6,8-tetrahydroxynaphthalene
</option>

</select>
<button id="knownclusterblast-r26c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r26c1-svg"></div>

</div>
</div>

<div class="body-details-section r26c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-297"></div> <span class="help-tooltip" id="cb-sub-help-297">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r26c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-300"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-300">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r26c1-domain-bubble-svg" data-filename="scaffold_26_r26c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r26c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (215939 - 262542): single T1PKS
</option>


</select>

<div class="details-svg" id="r26c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r26c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r26c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-302"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-302">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r26c1-pfam-details-svg" data-filename="scaffold_26_r26c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r26c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r26c1-nrps_pks_products" data-name="r26c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r26c1-nrps_pks_monomers" data-name="r26c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r26c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-298"></div> <span class="help-tooltip" id="nrps-structures-help-298">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 215939 - 262542:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r26c1-1" class="smiles-canvas" data-smiles="CC(=O)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(mal)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r26c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="CC(=O)C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r26c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-299"></div> <span class="help-tooltip" id="nrps-monomers-help-299">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">FUN_005747</span></strong>: mal
<div class="collapser collapser-target-FUN_005747 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>







<dd><strong>PKS_AT (911..1211)</strong>: mal
<div class="collapser collapser-target-FUN_005747 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-FUN_005747-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 95.8%</dd>
<dd>inactive: 62.5%</dd>
<dd>Methylmalonyl-CoA: 54.2%</dd>
</dl>

</div><br>

minowa:  Malonyl-CoA
<div class="collapser collapser-target-FUN_005747-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Malonyl-CoA: 125.6</dt>

<dd></dd><dt>inactive: 93.0</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 57.4</dt>

<dd></dd><dt>Methylmalonyl-CoA: 54.9</dt>

<dd></dd><dt>Propionyl-CoA: 32.3</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 23.8</dt>

<dd></dd><dt>Isobutyryl-CoA: 22.5</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 22.1</dt>

<dd></dd><dt>Benzoyl-CoA: 20.1</dt>

<dd></dd><dt>CHC-CoA: 17.4</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 16.0</dt>

<dd></dd><dt>Acetyl-CoA: 14.4</dt>

<dd></dd><dt>fatty_acid: 13.2</dt>

<dd></dd><dt>trans-1,2-CPDA: 9.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>










</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>





<div class="page" id='r28c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_28 - Region 1 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-789"></div> <span class="help-tooltip" id="region_svg-help-789">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_28.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r28c1-svg" data-filename="scaffold_28_r28c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 340,992 - 387,995 nt. (total: 47,004 nt)

<a class="cluster-rules-header" id="r28c1-rules-header" href="#r28c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r28c1-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r28c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-790"></div> <span class="help-tooltip" id="focus-panel-help-790">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r28c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r28c1-gene-table-details" data-name="r28c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r28c1-nrps_pks" data-name="r28c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r28c1-MIBiG-cluster-compare" data-name="r28c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r28c1-clusterblast" data-name="r28c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r28c1-knownclusterblast" data-name="r28c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r28c1-subclusterblast" data-name="r28c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r28c1-nrps-pks-bubbles" data-name="r28c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r28c1-pfam-details" data-name="r28c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r28c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-311"></div> <span class="help-tooltip" id="gt-help-tooltip-help-311">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r28c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r28c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r28c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r28c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r28c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-303"></div> <span class="help-tooltip" id="nrps-domain-help-303">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r28c1-details-svg" data-filename="scaffold_28_r28c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r28c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r28c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-304"></div> <span class="help-tooltip" id="MIBiG-help-304">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r28c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r28c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r28c1"></div>
<div id="comparison-MIBiG-r28c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002228: 282-17978">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002228/index.html#r1c1" target="_new">BGC0002228</a></td>





<td class="cc-heat-cell" title="0.70 (id:0.50, order:0.69, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>


<td>0.38</td>
<td>NRP</td>

<td>CIML B, CIML A, CIML D, CIML C</td>


<td>Colletotrichum incanum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001219: 0-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001219/index.html#r1c1" target="_new">BGC0001219</a></td>





<td class="cc-heat-cell" title="0.69 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>


<td>0.36</td>
<td>Polyketide</td>

<td>shanorellin</td>


<td>Chaetomium globosum CBS 148.51</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001124: 3228-30082">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001124/index.html#r1c1" target="_new">BGC0001124</a></td>





<td class="cc-heat-cell" title="0.68 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>


<td>0.35</td>
<td>Polyketide</td>

<td>pyranonigrin E</td>


<td>Aspergillus niger ATCC 1015</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002245: 219-11090">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002245/index.html#r1c1" target="_new">BGC0002245</a></td>





<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>


<td>0.35</td>
<td>Terpene</td>

<td>terpestacin, preterpestacin 3, preterpestacin 2, preterpestacin 1</td>


<td>Bipolaris maydis C5</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001659: 0-16557">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001659/index.html#r1c1" target="_new">BGC0001659</a></td>





<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>


<td>0.35</td>
<td>Terpene</td>

<td>mangicol A</td>


<td>Fusarium equiseti</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000677: 120-4940">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000677/index.html#r1c1" target="_new">BGC0000677</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.34</td>
<td>Terpene</td>

<td>cyclooctatin</td>


<td>Streptomyces melanosporofaciens</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002610: 0-6468">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002610/index.html#r1c1" target="_new">BGC0002610</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.33</td>
<td>Terpene</td>

<td>stellatic acid</td>


<td>Aspergillus stellatus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002233: 556-31132">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002233/index.html#r1c1" target="_new">BGC0002233</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.47, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.33</td>
<td>Polyketide</td>

<td>trichoxide</td>


<td>Trichoderma virens Gv29-8</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001990: 130-13646">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001990/index.html#r1c1" target="_new">BGC0001990</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.33</td>
<td>Other</td>

<td>BU-4704, xanthocillin-X monomethylether, melanocin E, melanocin F, fumiformamide</td>


<td>Aspergillus fumigatus Af293</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002211: 74-16330">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002211/index.html#r1c1" target="_new">BGC0002211</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.50, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.32</td>
<td>Polyketide</td>

<td>HEx-pks1 polyketide</td>


<td>Paraphaeosphaeria sporulosa</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r28c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002228: 282-17978">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002228/index.html#r1c1" target="_new">BGC0002228</a></td>


<td class="cc-heat-cell" title="0.70 (id:0.50, order:0.69, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
<td>0.70</td>
<td>NRP</td>
<td>CIML B, CIML A, CIML D, CIML C</td>
<td>Colletotrichum incanum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001219: 0-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001219/index.html#r1c1" target="_new">BGC0001219</a></td>


<td class="cc-heat-cell" title="0.69 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
<td>0.69</td>
<td>Polyketide</td>
<td>shanorellin</td>
<td>Chaetomium globosum CBS 148.51</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001124: 3228-30082">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001124/index.html#r1c1" target="_new">BGC0001124</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>Polyketide</td>
<td>pyranonigrin E</td>
<td>Aspergillus niger ATCC 1015</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002245: 219-11090">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002245/index.html#r1c1" target="_new">BGC0002245</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>Terpene</td>
<td>terpestacin, preterpestacin 3, preterpestacin 2, preterpestacin 1</td>
<td>Bipolaris maydis C5</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001659: 0-16557">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001659/index.html#r1c1" target="_new">BGC0001659</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>Terpene</td>
<td>mangicol A</td>
<td>Fusarium equiseti</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000677: 120-4940">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000677/index.html#r1c1" target="_new">BGC0000677</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Terpene</td>
<td>cyclooctatin</td>
<td>Streptomyces melanosporofaciens</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002610: 0-6468">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002610/index.html#r1c1" target="_new">BGC0002610</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Terpene</td>
<td>stellatic acid</td>
<td>Aspergillus stellatus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002233: 556-31132">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002233/index.html#r1c1" target="_new">BGC0002233</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.47, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Polyketide</td>
<td>trichoxide</td>
<td>Trichoderma virens Gv29-8</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001990: 130-13646">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001990/index.html#r1c1" target="_new">BGC0001990</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Other</td>
<td>BU-4704, xanthocillin-X monomethylether, melanocin E, melanocin F, fumiformamide</td>
<td>Aspergillus fumigatus Af293</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002211: 74-16330">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002211/index.html#r1c1" target="_new">BGC0002211</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.50, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Polyketide</td>
<td>HEx-pks1 polyketide</td>
<td>Paraphaeosphaeria sporulosa</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r28c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-305"></div> <span class="help-tooltip" id="cb-general-help-305">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r28c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r28c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r28c1_1.svg">
NC_049565 (2694595-2813931): Talaromyces rugulosus chromosome V, complete seq... (30% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r28c1_2.svg">
NW_026249941 (3336813-3404118): Alternaria arbusti strain BMP 1465 chromosome... (33% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r28c1_3.svg">
NW_020194484 (7340-83776): Amorphotheca resinae ATCC 22711 unplaced genomic s... (35% of genes show similarity), T1PKS,terpene
</option>

<option value = "svg/clusterblast_r28c1_4.svg">
NW_022474205 (7019281-7087036): Venustampulla echinocandica strain BP 5553 ch... (28% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r28c1_5.svg">
NW_021167084 (1837567-1907553): Sodiomyces alkalinus F11 unplaced genomic sca... (20% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r28c1_6.svg">
NW_026055699 (512819-580124): Alternaria ventricosa strain BMP 2768 chromosom... (36% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r28c1_7.svg">
NW_026249875 (10804-78109): Alternaria incomplexa strain BMP 0042 chromosome ... (30% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r28c1_8.svg">
NC_054862 (2647617-2714965): Aspergillus puulaauensis MK2 DNA, chromosome 6, ... (18% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r28c1_9.svg">
NW_006271974 (2321315-2405136): Cordyceps militaris CM01 unplaced genomic sca... (35% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r28c1_10.svg">
NW_025544748 (424962-492810): Colletotrichum gloeosporioides strain Lc1 chrom... (25% of genes show similarity), T1PKS
</option>

</select>
<button id= "clusterblast-r28c1-download">Download graphic</button>
</div>
<div id="clusterblast-r28c1-svg"></div>

</div>
</div>

<div class="body-details-section r28c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-306"></div> <span class="help-tooltip" id="cb-known-help-306">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r28c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r28c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r28c1_1.svg">
HEx-pks1 polyketide
</option>

<option value = "svg/knownclusterblast_r28c1_2.svg">
fumagillin/-trans-bergamotene/fumagillol
</option>

<option value = "svg/knownclusterblast_r28c1_3.svg">
4-epi-15-epi-brefeldin A
</option>

</select>
<button id="knownclusterblast-r28c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r28c1-svg"></div>

</div>
</div>

<div class="body-details-section r28c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-307"></div> <span class="help-tooltip" id="cb-sub-help-307">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r28c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-310"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-310">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r28c1-domain-bubble-svg" data-filename="scaffold_28_r28c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r28c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (340991 - 387995): single T1PKS
</option>


</select>

<div class="details-svg" id="r28c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r28c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r28c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-312"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-312">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r28c1-pfam-details-svg" data-filename="scaffold_28_r28c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r28c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r28c1-nrps_pks_products" data-name="r28c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r28c1-nrps_pks_monomers" data-name="r28c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r28c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-308"></div> <span class="help-tooltip" id="nrps-structures-help-308">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 340991 - 387995:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r28c1-1" class="smiles-canvas" data-smiles="C([*])C(-O)"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(pk)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r28c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r28c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-309"></div> <span class="help-tooltip" id="nrps-monomers-help-309">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">FUN_006051</span></strong>: pk
<div class="collapser collapser-target-FUN_006051 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (382..699)</strong>: pk
<div class="collapser collapser-target-FUN_006051 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-FUN_006051-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 75.0%</dd>
<dd>Methylmalonyl-CoA: 66.7%</dd>
<dd>Methoxymalonyl-CoA: 62.5%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-FUN_006051-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 79.9</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 67.9</dt>

<dd></dd><dt>Malonyl-CoA: 44.4</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 37.0</dt>

<dd></dd><dt>Propionyl-CoA: 36.8</dt>

<dd></dd><dt>Isobutyryl-CoA: 35.4</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 32.1</dt>

<dd></dd><dt>trans-1,2-CPDA: 26.7</dt>

<dd></dd><dt>fatty_acid: 19.5</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 19.3</dt>

<dd></dd><dt>CHC-CoA: 15.6</dt>

<dd></dd><dt>Acetyl-CoA: 14.7</dt>

<dd></dd><dt>inactive: 12.8</dt>

<dd></dd><dt>Benzoyl-CoA: 11.3</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>










</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>





<div class="page" id='r29c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_29 - Region 1 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-791"></div> <span class="help-tooltip" id="region_svg-help-791">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_29.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r29c1-svg" data-filename="scaffold_29_r29c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 6,283 - 27,900 nt. (total: 21,618 nt)

<a class="cluster-rules-header" id="r29c1-rules-header" href="#r29c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r29c1-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>

<div class="region-svg-container">
<div id='r29c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-792"></div> <span class="help-tooltip" id="focus-panel-help-792">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r29c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r29c1-gene-table-details" data-name="r29c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r29c1-MIBiG-cluster-compare" data-name="r29c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r29c1-clusterblast" data-name="r29c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r29c1-knownclusterblast" data-name="r29c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r29c1-subclusterblast" data-name="r29c1-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r29c1-pfam-details" data-name="r29c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r29c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-317"></div> <span class="help-tooltip" id="gt-help-tooltip-help-317">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r29c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r29c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r29c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r29c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r29c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-313"></div> <span class="help-tooltip" id="MIBiG-help-313">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r29c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r29c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r29c1"></div>
<div id="comparison-MIBiG-r29c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">terpene</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002610: 0-6468">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002610/index.html#r1c1" target="_new">BGC0002610</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>Terpene</td>

<td>stellatic acid</td>


<td>Aspergillus stellatus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001969: 0-7810">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001969/index.html#r1c1" target="_new">BGC0001969</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.21</td>
<td>Terpene</td>

<td>asperterpenoid A</td>


<td>Talaromyces wortmannii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001286: 99-14958">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001286/index.html#r1c1" target="_new">BGC0001286</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.21</td>
<td>Other</td>

<td>itaconic acid</td>


<td>Ustilago maydis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002670: 0-27523">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002670/index.html#r1c1" target="_new">BGC0002670</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>Other</td>

<td>fosfonochlorin</td>


<td>Fusarium oxysporum f. sp. lycopersici 4287</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001475: 5-9618">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001475/index.html#r1c1" target="_new">BGC0001475</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>Terpene</td>

<td>aspterric acid</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001775: 2711-20673">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001775/index.html#r1c1" target="_new">BGC0001775</a></td>





<td class="cc-heat-cell" title="0.44 (id:0.43, order:N/A, components:0.46)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>


<td>0.15</td>
<td>Terpene</td>

<td>sespendole</td>


<td>Cordana terrestris</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001679: 12-12481">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001679/index.html#r1c1" target="_new">BGC0001679</a></td>





<td class="cc-heat-cell" title="0.43 (id:0.45, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>


<td>0.14</td>
<td>NRP</td>

<td>N-Acetyltryptophan</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000631: 1094-28776">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000631/index.html#r1c1" target="_new">BGC0000631</a></td>





<td class="cc-heat-cell" title="0.43 (id:0.41, order:N/A, components:0.45)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>


<td>0.14</td>
<td>Terpene</td>

<td>botcinin H, botcinin I, botcinin J, cinbotolide D, botcinin K</td>


<td>Botrytis cinerea B05.10</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002651: 0-16858">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002651/index.html#r1c1" target="_new">BGC0002651</a></td>





<td class="cc-heat-cell" title="0.41 (id:0.45, order:N/A, components:0.37)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>


<td>0.13</td>
<td>Terpene</td>

<td>sordaricin B</td>


<td>Rosellinia necatrix</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002445: 12358-34562">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002445/index.html#r1c1" target="_new">BGC0002445</a></td>





<td class="cc-heat-cell" title="0.41 (id:0.41, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>


<td>0.13</td>
<td>Terpene</td>

<td>armillyl orsellinate, 8-hydroxy-6-protoilludene</td>


<td>Armillaria gallica</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r29c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002610: 0-6468">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002610/index.html#r1c1" target="_new">BGC0002610</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Terpene</td>
<td>stellatic acid</td>
<td>Aspergillus stellatus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001969: 0-7810">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001969/index.html#r1c1" target="_new">BGC0001969</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Terpene</td>
<td>asperterpenoid A</td>
<td>Talaromyces wortmannii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001286: 99-14958">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001286/index.html#r1c1" target="_new">BGC0001286</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Other</td>
<td>itaconic acid</td>
<td>Ustilago maydis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002670: 0-27523">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002670/index.html#r1c1" target="_new">BGC0002670</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Other</td>
<td>fosfonochlorin</td>
<td>Fusarium oxysporum f. sp. lycopersici 4287</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001475: 5-9618">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001475/index.html#r1c1" target="_new">BGC0001475</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>Terpene</td>
<td>aspterric acid</td>
<td>Aspergillus terreus NIH2624</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001775: 2711-20673">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001775/index.html#r1c1" target="_new">BGC0001775</a></td>


<td class="cc-heat-cell" title="0.44 (id:0.43, order:N/A, components:0.46)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
<td>0.44</td>
<td>Terpene</td>
<td>sespendole</td>
<td>Cordana terrestris</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001679: 12-12481">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001679/index.html#r1c1" target="_new">BGC0001679</a></td>


<td class="cc-heat-cell" title="0.43 (id:0.45, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
<td>0.43</td>
<td>NRP</td>
<td>N-Acetyltryptophan</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000631: 1094-28776">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000631/index.html#r1c1" target="_new">BGC0000631</a></td>


<td class="cc-heat-cell" title="0.43 (id:0.41, order:N/A, components:0.45)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
<td>0.43</td>
<td>Terpene</td>
<td>botcinin H, botcinin I, botcinin J, cinbotolide D, botcinin K</td>
<td>Botrytis cinerea B05.10</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002651: 0-16858">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002651/index.html#r1c1" target="_new">BGC0002651</a></td>


<td class="cc-heat-cell" title="0.41 (id:0.45, order:N/A, components:0.37)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
<td>0.41</td>
<td>Terpene</td>
<td>sordaricin B</td>
<td>Rosellinia necatrix</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002445: 12358-34562">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002445/index.html#r1c1" target="_new">BGC0002445</a></td>


<td class="cc-heat-cell" title="0.41 (id:0.41, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
<td>0.41</td>
<td>Terpene</td>
<td>armillyl orsellinate, 8-hydroxy-6-protoilludene</td>
<td>Armillaria gallica</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r29c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-314"></div> <span class="help-tooltip" id="cb-general-help-314">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r29c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r29c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r29c1_1.svg">
NC_049561 (111243-176756): Talaromyces rugulosus chromosome I, complete sequence (7% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r29c1_2.svg">
NC_064675 (322453-353960): Colletotrichum lupini chromosome 2, complete sequence (18% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r29c1_3.svg">
NW_022983490 (330572-424081): Aaosphaeria arxii CBS 175.79 unplaced genomic s... (9% of genes show similarity), T1PKS,fungal-RiPP-like,terpene
</option>

<option value = "svg/clusterblast_r29c1_4.svg">
NW_025764725 (172366-203821): Emericellopsis atlantica unplaced genomic scaff... (28% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r29c1_5.svg">
NW_022985367 (885970-917204): Macroventuria anomochaeta strain CBS 525.71 unp... (20% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r29c1_6.svg">
NW_020292124 (32434-64082): Aspergillus ibericus CBS 121593 unplaced genomic ... (13% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r29c1_7.svg">
NC_066260 (1045893-1155772): Aspergillus nidulans FGSC A4 chromosome IV (5% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r29c1_8.svg">
NW_022985439 (2314126-2345084): Mytilinidion resinicola strain CBS 304.34 unp... (15% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r29c1_9.svg">
NW_025091266 (907356-1099460): Aspergillus udagawae IFM 46973 DNA, scaffold 3... (4% of genes show similarity), NRPS-like,T1PKS,terpene
</option>

<option value = "svg/clusterblast_r29c1_10.svg">
NW_023336275 (1318803-1477610): Aspergillus tubingensis WU-2223L DNA, scaffol... (3% of genes show similarity), NRPS,T1PKS,fungal-RiPP-like
</option>

</select>
<button id= "clusterblast-r29c1-download">Download graphic</button>
</div>
<div id="clusterblast-r29c1-svg"></div>

</div>
</div>

<div class="body-details-section r29c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-315"></div> <span class="help-tooltip" id="cb-known-help-315">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r29c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-316"></div> <span class="help-tooltip" id="cb-sub-help-316">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r29c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-318"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-318">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r29c1-pfam-details-svg" data-filename="scaffold_29_r29c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r29c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">









</div>









</div>

</div>

</div>

<div class="page" id='r29c2' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_29 - Region 2 - NRPS-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-793"></div> <span class="help-tooltip" id="region_svg-help-793">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_29.region002.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r29c2-svg" data-filename="scaffold_29_r29c2.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 69,701 - 112,796 nt. (total: 43,096 nt)

<a class="cluster-rules-header" id="r29c2-rules-header" href="#r29c2">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r29c2-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r29c2-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-794"></div> <span class="help-tooltip" id="focus-panel-help-794">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r29c2">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r29c2-gene-table-details" data-name="r29c2-gene-table-details">
Gene overview
</div>



<div class="body-details-header r29c2-nrps_pks" data-name="r29c2-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r29c2-MIBiG-cluster-compare" data-name="r29c2-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r29c2-clusterblast" data-name="r29c2-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r29c2-knownclusterblast" data-name="r29c2-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r29c2-subclusterblast" data-name="r29c2-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r29c2-nrps-pks-bubbles" data-name="r29c2-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r29c2-pfam-details" data-name="r29c2-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r29c2-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-327"></div> <span class="help-tooltip" id="gt-help-tooltip-help-327">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r29c2">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r29c2-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r29c2-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r29c2">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r29c2-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-319"></div> <span class="help-tooltip" id="nrps-domain-help-319">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r29c2-details-svg" data-filename="scaffold_29_r29c2_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r29c2-details-svg"></div>
</div>
</div>



<div class="body-details-section r29c2-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-320"></div> <span class="help-tooltip" id="MIBiG-help-320">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r29c2-selector">Analysis type:</label>
<select id="comparison-MIBiG-r29c2-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r29c2"></div>
<div id="comparison-MIBiG-r29c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000099: 0-8104">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000099/index.html#r1c1" target="_new">BGC0000099</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.16</td>
<td>Polyketide</td>

<td>monascorubrin</td>


<td>Talaromyces marneffei</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002212: 0-24132">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002212/index.html#r1c1" target="_new">BGC0002212</a></td>





<td class="cc-heat-cell" title="0.38 (id:0.41, order:N/A, components:0.36)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>


<td>0.11</td>
<td>Polyketide</td>

<td>HEx-pks15 polyketide</td>


<td>Moniliophthora roreri MCA 2997</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001390: 100-19645">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001390/index.html#r1c1" target="_new">BGC0001390</a></td>





<td class="cc-heat-cell" title="0.34 (id:0.41, order:N/A, components:0.28)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>


<td>0.09</td>
<td>NRP, Polyketide</td>

<td>LL-Z1272beta</td>


<td>Stachybotrys bisbyi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002509: 94-21985">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002509/index.html#r1c1" target="_new">BGC0002509</a></td>





<td class="cc-heat-cell" title="0.31 (id:0.41, order:N/A, components:0.23)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>


<td>0.08</td>
<td>NRP</td>

<td>montamide A, capsimycin, clifednamide A, frontalamide A, combamide A</td>


<td>Streptomyces sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001600: 0-23599">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001600/index.html#r1c1" target="_new">BGC0001600</a></td>





<td class="cc-heat-cell" title="0.29 (id:0.41, order:N/A, components:0.21)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>


<td>0.07</td>
<td>Polyketide</td>

<td>fusarielin H</td>


<td>Fusarium graminearum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001722: 0-18441">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001722/index.html#r1c1" target="_new">BGC0001722</a></td>





<td class="cc-heat-cell" title="0.29 (id:0.42, order:N/A, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>


<td>0.07</td>
<td>Polyketide</td>

<td>aspernidine A</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001242: 200-22183">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001242/index.html#r1c1" target="_new">BGC0001242</a></td>





<td class="cc-heat-cell" title="0.27 (id:0.42, order:N/A, components:0.18)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>


<td>0.07</td>
<td>Polyketide</td>

<td>oxyjavanicin</td>


<td>Fusarium fujikuroi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001976: 11125-46899">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001976/index.html#r1c1" target="_new">BGC0001976</a></td>





<td class="cc-heat-cell" title="0.26 (id:0.41, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>


<td>0.06</td>
<td>Terpene</td>

<td>eupenifeldin</td>


<td>Phoma sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001923: 495-32413">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001923/index.html#r1c1" target="_new">BGC0001923</a></td>





<td class="cc-heat-cell" title="0.26 (id:0.41, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>


<td>0.06</td>
<td>Terpene, Polyketide</td>

<td>ascochlorin</td>


<td>Acremonium egyptiacum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001255: 1908-38951">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001255/index.html#r1c1" target="_new">BGC0001255</a></td>





<td class="cc-heat-cell" title="0.26 (id:0.41, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>


<td>0.06</td>
<td>NRP, Polyketide</td>

<td>equisetin</td>


<td>Fusarium heterosporum</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r29c2-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000099: 0-8104">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000099/index.html#r1c1" target="_new">BGC0000099</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>Polyketide</td>
<td>monascorubrin</td>
<td>Talaromyces marneffei</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002212: 0-24132">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002212/index.html#r1c1" target="_new">BGC0002212</a></td>


<td class="cc-heat-cell" title="0.38 (id:0.41, order:N/A, components:0.36)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
<td>0.38</td>
<td>Polyketide</td>
<td>HEx-pks15 polyketide</td>
<td>Moniliophthora roreri MCA 2997</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001390: 100-19645">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001390/index.html#r1c1" target="_new">BGC0001390</a></td>


<td class="cc-heat-cell" title="0.34 (id:0.41, order:N/A, components:0.28)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>
<td>0.34</td>
<td>NRP, Polyketide</td>
<td>LL-Z1272beta</td>
<td>Stachybotrys bisbyi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002509: 94-21985">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002509/index.html#r1c1" target="_new">BGC0002509</a></td>


<td class="cc-heat-cell" title="0.31 (id:0.41, order:N/A, components:0.23)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>
<td>0.31</td>
<td>NRP</td>
<td>montamide A, capsimycin, clifednamide A, frontalamide A, combamide A</td>
<td>Streptomyces sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001600: 0-23599">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001600/index.html#r1c1" target="_new">BGC0001600</a></td>


<td class="cc-heat-cell" title="0.29 (id:0.41, order:N/A, components:0.21)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>
<td>0.29</td>
<td>Polyketide</td>
<td>fusarielin H</td>
<td>Fusarium graminearum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001722: 0-18441">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001722/index.html#r1c1" target="_new">BGC0001722</a></td>


<td class="cc-heat-cell" title="0.29 (id:0.42, order:N/A, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>
<td>0.29</td>
<td>Polyketide</td>
<td>aspernidine A</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001242: 200-22183">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001242/index.html#r1c1" target="_new">BGC0001242</a></td>


<td class="cc-heat-cell" title="0.27 (id:0.42, order:N/A, components:0.18)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>
<td>0.27</td>
<td>Polyketide</td>
<td>oxyjavanicin</td>
<td>Fusarium fujikuroi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001976: 11125-46899">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001976/index.html#r1c1" target="_new">BGC0001976</a></td>


<td class="cc-heat-cell" title="0.26 (id:0.41, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>
<td>0.26</td>
<td>Terpene</td>
<td>eupenifeldin</td>
<td>Phoma sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001923: 495-32413">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001923/index.html#r1c1" target="_new">BGC0001923</a></td>


<td class="cc-heat-cell" title="0.26 (id:0.41, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>
<td>0.26</td>
<td>Terpene, Polyketide</td>
<td>ascochlorin</td>
<td>Acremonium egyptiacum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001255: 1908-38951">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001255/index.html#r1c1" target="_new">BGC0001255</a></td>


<td class="cc-heat-cell" title="0.26 (id:0.41, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>
<td>0.26</td>
<td>NRP, Polyketide</td>
<td>equisetin</td>
<td>Fusarium heterosporum</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r29c2-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-321"></div> <span class="help-tooltip" id="cb-general-help-321">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r29c2-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r29c2_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r29c2_1.svg">
NW_025953803 (440909-571501): Annulohypoxylon truncatum strain CBS 140777 unp... (10% of genes show similarity), NRPS,T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r29c2_2.svg">
NW_025953249 (3016690-3115240): Annulohypoxylon maeteangense strain CBS 12383... (11% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r29c2_3.svg">
NW_026250022 (2631049-2722100): Hypoxylon trugodes strain CBS 135444 unplaced... (8% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r29c2_4.svg">
NW_014013660 (824720-892315): Trichoderma virens Gv29-8 chromosome Unknown TR... (9% of genes show similarity), T1PKS
</option>

</select>
<button id= "clusterblast-r29c2-download">Download graphic</button>
</div>
<div id="clusterblast-r29c2-svg"></div>

</div>
</div>

<div class="body-details-section r29c2-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-322"></div> <span class="help-tooltip" id="cb-known-help-322">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r29c2-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-323"></div> <span class="help-tooltip" id="cb-sub-help-323">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r29c2-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-326"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-326">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r29c2-domain-bubble-svg" data-filename="scaffold_29_r29c2_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r29c2-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC2">
Candidate 2 (69700 - 112796): single NRPS-like
</option>


</select>

<div class="details-svg" id="r29c2-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r29c2-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r29c2-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-328"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-328">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r29c2-pfam-details-svg" data-filename="scaffold_29_r29c2_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r29c2-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r29c2-nrps_pks_products" data-name="r29c2-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r29c2-nrps_pks_monomers" data-name="r29c2-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r29c2-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-324"></div> <span class="help-tooltip" id="nrps-structures-help-324">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 2, location 69700 - 112796:
<div class="collapser collapser-target-CC2 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r29c2-2" class="smiles-canvas" data-smiles="NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r29c2-2-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r29c2-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-325"></div> <span class="help-tooltip" id="nrps-monomers-help-325">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">FUN_006100</span></strong>: X
<div class="collapser collapser-target-FUN_006100 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (36..337)</strong>: X
<div class="collapser collapser-target-FUN_006100 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr
<div class="collapser collapser-target-FUN_006100-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">LAHAMGCILK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>

<dt>Stachelhaus code match:</dt>
<dd>0% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>





<div class="page" id='r30c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_30 - Region 1 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-795"></div> <span class="help-tooltip" id="region_svg-help-795">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_30.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r30c1-svg" data-filename="scaffold_30_r30c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1 - 46,992 nt. (total: 46,992 nt)

<a class="cluster-rules-header" id="r30c1-rules-header" href="#r30c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r30c1-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r30c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-796"></div> <span class="help-tooltip" id="focus-panel-help-796">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r30c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r30c1-gene-table-details" data-name="r30c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r30c1-nrps_pks" data-name="r30c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r30c1-MIBiG-cluster-compare" data-name="r30c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r30c1-clusterblast" data-name="r30c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r30c1-knownclusterblast" data-name="r30c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r30c1-subclusterblast" data-name="r30c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r30c1-nrps-pks-bubbles" data-name="r30c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r30c1-pfam-details" data-name="r30c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r30c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-337"></div> <span class="help-tooltip" id="gt-help-tooltip-help-337">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r30c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r30c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r30c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r30c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r30c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-329"></div> <span class="help-tooltip" id="nrps-domain-help-329">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r30c1-details-svg" data-filename="scaffold_30_r30c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r30c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r30c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-330"></div> <span class="help-tooltip" id="MIBiG-help-330">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r30c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r30c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r30c1"></div>
<div id="comparison-MIBiG-r30c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001252: 0-7958">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001252/index.html#r1c1" target="_new">BGC0001252</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.43, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.30</td>
<td>Polyketide</td>

<td>UNII-YC2Q1O94PT</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002191: 260-8785">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002191/index.html#r1c1" target="_new">BGC0002191</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.29</td>
<td>Polyketide</td>

<td>prolipyrone B, gibepyrone D</td>


<td>Fusarium graminearum PH-1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002155: 78-16511">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002155/index.html#r1c1" target="_new">BGC0002155</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.28</td>
<td>Polyketide</td>

<td>nectriapyrone C, nectriapyrone D, nectriapyrone</td>


<td>Pyricularia oryzae 70-15</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002222: 888-32276">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002222/index.html#r1c1" target="_new">BGC0002222</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.45, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.26</td>
<td>Polyketide</td>

<td>zopfiellin</td>


<td>Diffractella curvata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002194: 147-15389">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002194/index.html#r1c1" target="_new">BGC0002194</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.47, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.25</td>
<td>Polyketide</td>

<td>epipyrone A</td>


<td>Epicoccum nigrum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002224: 685-37748">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002224/index.html#r1c1" target="_new">BGC0002224</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.25</td>
<td>Polyketide</td>

<td>zopfiellin</td>


<td>Diffractella curvata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002515: 0-30087">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002515/index.html#r1c1" target="_new">BGC0002515</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.43, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.25</td>
<td>Polyketide</td>

<td>solanapyrone A</td>


<td>Ascochyta rabiei</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002525: 1000-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002525/index.html#r1c1" target="_new">BGC0002525</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>Polyketide</td>

<td>fusarubin, 1233A, 1233B, NG-391, lucilactaene</td>


<td>Fusarium sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001606: 0-22544">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001606/index.html#r1c1" target="_new">BGC0001606</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>Polyketide</td>

<td>gibepyrone-A</td>


<td>Fusarium fujikuroi IMI 58289</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002223: 2980-46085">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002223/index.html#r1c1" target="_new">BGC0002223</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.68)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Polyketide</td>

<td>scytalidin</td>


<td>Scytalidium album</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r30c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001252: 0-7958">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001252/index.html#r1c1" target="_new">BGC0001252</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.43, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>Polyketide</td>
<td>UNII-YC2Q1O94PT</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002191: 260-8785">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002191/index.html#r1c1" target="_new">BGC0002191</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>Polyketide</td>
<td>prolipyrone B, gibepyrone D</td>
<td>Fusarium graminearum PH-1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002155: 78-16511">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002155/index.html#r1c1" target="_new">BGC0002155</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>Polyketide</td>
<td>nectriapyrone C, nectriapyrone D, nectriapyrone</td>
<td>Pyricularia oryzae 70-15</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002222: 888-32276">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002222/index.html#r1c1" target="_new">BGC0002222</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.45, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Polyketide</td>
<td>zopfiellin</td>
<td>Diffractella curvata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002194: 147-15389">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002194/index.html#r1c1" target="_new">BGC0002194</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.47, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Polyketide</td>
<td>epipyrone A</td>
<td>Epicoccum nigrum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002224: 685-37748">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002224/index.html#r1c1" target="_new">BGC0002224</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Polyketide</td>
<td>zopfiellin</td>
<td>Diffractella curvata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002515: 0-30087">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002515/index.html#r1c1" target="_new">BGC0002515</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.43, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Polyketide</td>
<td>solanapyrone A</td>
<td>Ascochyta rabiei</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002525: 1000-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002525/index.html#r1c1" target="_new">BGC0002525</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Polyketide</td>
<td>fusarubin, 1233A, 1233B, NG-391, lucilactaene</td>
<td>Fusarium sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001606: 0-22544">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001606/index.html#r1c1" target="_new">BGC0001606</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Polyketide</td>
<td>gibepyrone-A</td>
<td>Fusarium fujikuroi IMI 58289</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002223: 2980-46085">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002223/index.html#r1c1" target="_new">BGC0002223</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.68)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Polyketide</td>
<td>scytalidin</td>
<td>Scytalidium album</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r30c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-331"></div> <span class="help-tooltip" id="cb-general-help-331">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r30c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r30c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r30c1_1.svg">
NC_064655 (797649-865606): Penicillium oxalicum strain HP7-1 chromosome VI, w... (9% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r30c1_2.svg">
NW_022985443 (760417-893141): Mytilinidion resinicola strain CBS 304.34 unpla... (3% of genes show similarity), NRPS-like,T1PKS,phosphonate-like
</option>

<option value = "svg/clusterblast_r30c1_3.svg">
NZ_CP022540 (3492163-3581798): Antarctobacter heliothermus strain SMS3 chromo... (8% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r30c1_4.svg">
NZ_JAFEJA010000002 (1277917-1387100): Streptomyces zhihengii strain YIM T102 ... (6% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r30c1_5.svg">
NW_025953253 (740469-807954): Annulohypoxylon maeteangense strain CBS 123835 ... (13% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r30c1_6.svg">
NW_025953257 (328883-396554): Annulohypoxylon maeteangense strain CBS 123835 ... (11% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r30c1_7.svg">
NW_025953787 (2032241-2099463): Annulohypoxylon truncatum strain CBS 140777 u... (11% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r30c1_8.svg">
NW_025953055 (276846-344269): Daldinia caldariorum strain CBS 122874 unplaced... (11% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r30c1_9.svg">
NW_026250018 (661526-728676): Hypoxylon trugodes strain CBS 135444 unplaced g... (12% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r30c1_10.svg">
NW_026054554 (165113-232305): Daldinia decipiens strain CBS 113046 unplaced g... (11% of genes show similarity), T1PKS
</option>

</select>
<button id= "clusterblast-r30c1-download">Download graphic</button>
</div>
<div id="clusterblast-r30c1-svg"></div>

</div>
</div>

<div class="body-details-section r30c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-332"></div> <span class="help-tooltip" id="cb-known-help-332">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r30c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-333"></div> <span class="help-tooltip" id="cb-sub-help-333">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r30c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-336"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-336">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r30c1-domain-bubble-svg" data-filename="scaffold_30_r30c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r30c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (0 - 46992): single T1PKS
</option>


</select>

<div class="details-svg" id="r30c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r30c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r30c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-338"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-338">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r30c1-pfam-details-svg" data-filename="scaffold_30_r30c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r30c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r30c1-nrps_pks_products" data-name="r30c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r30c1-nrps_pks_monomers" data-name="r30c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r30c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-334"></div> <span class="help-tooltip" id="nrps-structures-help-334">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 0 - 46992:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r30c1-1" class="smiles-canvas" data-smiles="C([*])C(-O)(C)"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Me-pk)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-pk" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-pk*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r30c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)(C)"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r30c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-335"></div> <span class="help-tooltip" id="nrps-monomers-help-335">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">FUN_006193</span></strong>: pk
<div class="collapser collapser-target-FUN_006193 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (606..912)</strong>: pk
<div class="collapser collapser-target-FUN_006193 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-FUN_006193-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 70.8%</dd>
<dd>Acetyl-CoA: 58.3%</dd>
<dd>CE-Malonyl-CoA: 54.2%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-FUN_006193-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 77.9</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 76.6</dt>

<dd></dd><dt>Malonyl-CoA: 76.6</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 56.2</dt>

<dd></dd><dt>Isobutyryl-CoA: 52.8</dt>

<dd></dd><dt>Propionyl-CoA: 41.9</dt>

<dd></dd><dt>trans-1,2-CPDA: 35.9</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 26.5</dt>

<dd></dd><dt>Acetyl-CoA: 26.0</dt>

<dd></dd><dt>CHC-CoA: 22.4</dt>

<dd></dd><dt>Benzoyl-CoA: 21.0</dt>

<dd></dd><dt>inactive: 20.5</dt>

<dd></dd><dt>fatty_acid: 17.6</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 9.8</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>












</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r30c2' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_30 - Region 2 - NRPS-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-797"></div> <span class="help-tooltip" id="region_svg-help-797">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_30.region002.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r30c2-svg" data-filename="scaffold_30_r30c2.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 137,164 - 179,980 nt. (total: 42,817 nt)

<a class="cluster-rules-header" id="r30c2-rules-header" href="#r30c2">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r30c2-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r30c2-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-798"></div> <span class="help-tooltip" id="focus-panel-help-798">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r30c2">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r30c2-gene-table-details" data-name="r30c2-gene-table-details">
Gene overview
</div>



<div class="body-details-header r30c2-nrps_pks" data-name="r30c2-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r30c2-MIBiG-cluster-compare" data-name="r30c2-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r30c2-clusterblast" data-name="r30c2-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r30c2-knownclusterblast" data-name="r30c2-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r30c2-subclusterblast" data-name="r30c2-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r30c2-nrps-pks-bubbles" data-name="r30c2-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r30c2-pfam-details" data-name="r30c2-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r30c2-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-347"></div> <span class="help-tooltip" id="gt-help-tooltip-help-347">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r30c2">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r30c2-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r30c2-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r30c2">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r30c2-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-339"></div> <span class="help-tooltip" id="nrps-domain-help-339">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r30c2-details-svg" data-filename="scaffold_30_r30c2_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r30c2-details-svg"></div>
</div>
</div>



<div class="body-details-section r30c2-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-340"></div> <span class="help-tooltip" id="MIBiG-help-340">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r30c2-selector">Analysis type:</label>
<select id="comparison-MIBiG-r30c2-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r30c2"></div>
<div id="comparison-MIBiG-r30c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002274: 0-2779">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002274/index.html#r1c1" target="_new">BGC0002274</a></td>





<td class="cc-heat-cell" title="0.69 (id:0.48, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>


<td>0.36</td>
<td>NRP</td>

<td>aspulvinone H, aspulvinone B1</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002245: 219-11090">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002245/index.html#r1c1" target="_new">BGC0002245</a></td>





<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>


<td>0.35</td>
<td>Terpene</td>

<td>terpestacin, preterpestacin 3, preterpestacin 2, preterpestacin 1</td>


<td>Bipolaris maydis C5</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002193: 447-3288">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002193/index.html#r1c1" target="_new">BGC0002193</a></td>





<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>


<td>0.35</td>
<td>Other</td>

<td>phenguignardic acid</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001668: 56-7544">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001668/index.html#r1c1" target="_new">BGC0001668</a></td>





<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>


<td>0.35</td>
<td>NRP</td>

<td>microperfuranone</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002348: 0-18134">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002348/index.html#r1c1" target="_new">BGC0002348</a></td>





<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>


<td>0.35</td>
<td>NRP</td>

<td>aspulvinone E</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002277: 0-3145">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002277/index.html#r1c1" target="_new">BGC0002277</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.34</td>
<td>NRP</td>

<td>atromentin</td>


<td>Suillus grevillei</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002610: 0-6468">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002610/index.html#r1c1" target="_new">BGC0002610</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.33</td>
<td>Terpene</td>

<td>stellatic acid</td>


<td>Aspergillus stellatus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001528: 2981-16881">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001528/index.html#r1c1" target="_new">BGC0001528</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.32</td>
<td>Terpene</td>

<td>betaestacin</td>


<td>Phoma betae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000677: 120-4940">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000677/index.html#r1c1" target="_new">BGC0000677</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.31</td>
<td>Terpene</td>

<td>cyclooctatin</td>


<td>Streptomyces melanosporofaciens</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002181: 1003-18265">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002181/index.html#r1c1" target="_new">BGC0002181</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.27</td>
<td>Terpene</td>

<td>pleuromutilin</td>


<td>Rhodocybe pseudopiperita</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r30c2-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002274: 0-2779">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002274/index.html#r1c1" target="_new">BGC0002274</a></td>


<td class="cc-heat-cell" title="0.69 (id:0.48, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
<td>0.69</td>
<td>NRP</td>
<td>aspulvinone H, aspulvinone B1</td>
<td>Aspergillus terreus NIH2624</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002245: 219-11090">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002245/index.html#r1c1" target="_new">BGC0002245</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>Terpene</td>
<td>terpestacin, preterpestacin 3, preterpestacin 2, preterpestacin 1</td>
<td>Bipolaris maydis C5</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002193: 447-3288">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002193/index.html#r1c1" target="_new">BGC0002193</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>Other</td>
<td>phenguignardic acid</td>
<td>Aspergillus terreus NIH2624</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001668: 56-7544">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001668/index.html#r1c1" target="_new">BGC0001668</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>NRP</td>
<td>microperfuranone</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002348: 0-18134">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002348/index.html#r1c1" target="_new">BGC0002348</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>NRP</td>
<td>aspulvinone E</td>
<td>Aspergillus terreus NIH2624</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002277: 0-3145">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002277/index.html#r1c1" target="_new">BGC0002277</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>NRP</td>
<td>atromentin</td>
<td>Suillus grevillei</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002610: 0-6468">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002610/index.html#r1c1" target="_new">BGC0002610</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Terpene</td>
<td>stellatic acid</td>
<td>Aspergillus stellatus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001528: 2981-16881">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001528/index.html#r1c1" target="_new">BGC0001528</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Terpene</td>
<td>betaestacin</td>
<td>Phoma betae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000677: 120-4940">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000677/index.html#r1c1" target="_new">BGC0000677</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>Terpene</td>
<td>cyclooctatin</td>
<td>Streptomyces melanosporofaciens</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002181: 1003-18265">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002181/index.html#r1c1" target="_new">BGC0002181</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>Terpene</td>
<td>pleuromutilin</td>
<td>Rhodocybe pseudopiperita</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r30c2-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-341"></div> <span class="help-tooltip" id="cb-general-help-341">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r30c2-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r30c2_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r30c2_1.svg">
NW_011942151 (20096-254382): Metarhizium robertsii ARSEF 23 MAA Scf 11, whole... (8% of genes show similarity), NRPS,NRPS-like,T1PKS,fungal-RiPP-like,terpene
</option>

<option value = "svg/clusterblast_r30c2_2.svg">
NW_024468914 (848006-976607): Cryphonectria parasitica EP155 unplaced genomic... (13% of genes show similarity), NRPS-like,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r30c2_3.svg">
NT_165977 (528235-591199): Chaetomium globosum CBS 148.51 scaffold 2 genomic ... (28% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r30c2_4.svg">
NW_014574694 (19387-240297): Metarhizium brunneum ARSEF 3297 chromosome Unkno... (7% of genes show similarity), NRPS,NRPS-like,T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r30c2_5.svg">
NW_022986852 (964825-1027603): Westerdykella ornata strain CBS 379.55 unplace... (17% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r30c2_6.svg">
NW_022985557 (585495-648402): Pseudovirgaria hyperparasitica strain CBS 12173... (24% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r30c2_7.svg">
NW_017253368 (4829-67775): Paraphaeosphaeria sporulosa strain AP3s5-JAC2a unp... (36% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r30c2_8.svg">
NW_025408669 (1835836-1898581): Hirsutella rhossiliensis strain HR02 chromoso... (31% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r30c2_9.svg">
NW_017263589 (58196-121106): Mollisia scopiformis unplaced genomic scaffold L... (26% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r30c2_10.svg">
NW_021167081 (357009-420018): Sodiomyces alkalinus F11 unplaced genomic scaff... (16% of genes show similarity), NRPS-like
</option>

</select>
<button id= "clusterblast-r30c2-download">Download graphic</button>
</div>
<div id="clusterblast-r30c2-svg"></div>

</div>
</div>

<div class="body-details-section r30c2-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-342"></div> <span class="help-tooltip" id="cb-known-help-342">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r30c2-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-343"></div> <span class="help-tooltip" id="cb-sub-help-343">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r30c2-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-346"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-346">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r30c2-domain-bubble-svg" data-filename="scaffold_30_r30c2_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r30c2-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC2">
Candidate 2 (137163 - 179980): single NRPS-like
</option>


</select>

<div class="details-svg" id="r30c2-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r30c2-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r30c2-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-348"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-348">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r30c2-pfam-details-svg" data-filename="scaffold_30_r30c2_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r30c2-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r30c2-nrps_pks_products" data-name="r30c2-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r30c2-nrps_pks_monomers" data-name="r30c2-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r30c2-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-344"></div> <span class="help-tooltip" id="nrps-structures-help-344">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 2, location 137163 - 179980:
<div class="collapser collapser-target-CC2 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r30c2-2" class="smiles-canvas" data-smiles="NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r30c2-2-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r30c2-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-345"></div> <span class="help-tooltip" id="nrps-monomers-help-345">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">FUN_006236</span></strong>: X
<div class="collapser collapser-target-FUN_006236 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (15..447)</strong>: X
<div class="collapser collapser-target-FUN_006236 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  (unknown)
<div class="collapser collapser-target-FUN_006236-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>N/A</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>glu,gln (Glu, Gln)</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">VAVFVGGAGK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>4-ohPpa <span class="serif">VAVFAGGAGK</span> (74% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>90% (moderate)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>





<div class="page" id='r31c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_31 - Region 1 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-799"></div> <span class="help-tooltip" id="region_svg-help-799">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_31.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r31c1-svg" data-filename="scaffold_31_r31c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 257,262 - 279,619 nt. (total: 22,358 nt)

<a class="cluster-rules-header" id="r31c1-rules-header" href="#r31c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r31c1-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>

<div class="region-svg-container">
<div id='r31c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-800"></div> <span class="help-tooltip" id="focus-panel-help-800">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r31c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r31c1-gene-table-details" data-name="r31c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r31c1-MIBiG-cluster-compare" data-name="r31c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r31c1-clusterblast" data-name="r31c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r31c1-knownclusterblast" data-name="r31c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r31c1-subclusterblast" data-name="r31c1-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r31c1-pfam-details" data-name="r31c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r31c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-353"></div> <span class="help-tooltip" id="gt-help-tooltip-help-353">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r31c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r31c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r31c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r31c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r31c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-349"></div> <span class="help-tooltip" id="MIBiG-help-349">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r31c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r31c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r31c1"></div>
<div id="comparison-MIBiG-r31c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">terpene</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002173: 92-7542">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002173/index.html#r1c1" target="_new">BGC0002173</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.31</td>
<td>Terpene, Saccharide</td>

<td>fumihopaside A, compound 3, 21--H-hopane-3beta, 22-diol</td>


<td>Aspergillus fumigatus A1163</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001990: 130-13646">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001990/index.html#r1c1" target="_new">BGC0001990</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Other</td>

<td>BU-4704, xanthocillin-X monomethylether, melanocin E, melanocin F, fumiformamide</td>


<td>Aspergillus fumigatus Af293</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000669: 0-34330">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000669/index.html#r1c1" target="_new">BGC0000669</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.21</td>
<td>Terpene</td>

<td>marneral</td>


<td>Arabidopsis thaliana</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001248: 1352-4192">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001248/index.html#r1c1" target="_new">BGC0001248</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.50, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>Terpene</td>

<td>clavaric acid</td>


<td>Hypholoma sublateritium</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001317: 0-277585">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001317/index.html#r1c1" target="_new">BGC0001317</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.46, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>Terpene</td>

<td>lupeol</td>


<td>Lotus japonicus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002449: 47-14377">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002449/index.html#r1c1" target="_new">BGC0002449</a></td>





<td class="cc-heat-cell" title="0.44 (id:0.43, order:N/A, components:0.45)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>


<td>0.14</td>
<td>Other</td>

<td>sarasinoside</td>


<td>uncultured bacterium</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002402: 203-37725">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002402/index.html#r1c1" target="_new">BGC0002402</a></td>





<td class="cc-heat-cell" title="0.44 (id:0.45, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>


<td>0.14</td>
<td>Terpene</td>

<td>yossosides</td>


<td>Spinacia oleracea</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001219: 0-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001219/index.html#r1c1" target="_new">BGC0001219</a></td>





<td class="cc-heat-cell" title="0.43 (id:0.44, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>


<td>0.14</td>
<td>Polyketide</td>

<td>shanorellin</td>


<td>Chaetomium globosum CBS 148.51</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001893: 155-19434">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001893/index.html#r1c1" target="_new">BGC0001893</a></td>





<td class="cc-heat-cell" title="0.42 (id:0.43, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>


<td>0.13</td>
<td>Other</td>

<td>abscisic acid</td>


<td>Botrytis cinerea B05.10</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000676: 621-14838">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000676/index.html#r1c1" target="_new">BGC0000676</a></td>





<td class="cc-heat-cell" title="0.42 (id:0.41, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>


<td>0.13</td>
<td>Terpene</td>

<td>aphidicolin, aphidicolan-16-ol, 3-deoxyaphidicolin, 17-deoxyaphidicolin</td>


<td>Phoma betae</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r31c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002173: 92-7542">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002173/index.html#r1c1" target="_new">BGC0002173</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>Terpene, Saccharide</td>
<td>fumihopaside A, compound 3, 21--H-hopane-3beta, 22-diol</td>
<td>Aspergillus fumigatus A1163</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001990: 130-13646">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001990/index.html#r1c1" target="_new">BGC0001990</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Other</td>
<td>BU-4704, xanthocillin-X monomethylether, melanocin E, melanocin F, fumiformamide</td>
<td>Aspergillus fumigatus Af293</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000669: 0-34330">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000669/index.html#r1c1" target="_new">BGC0000669</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Terpene</td>
<td>marneral</td>
<td>Arabidopsis thaliana</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001248: 1352-4192">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001248/index.html#r1c1" target="_new">BGC0001248</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.50, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Terpene</td>
<td>clavaric acid</td>
<td>Hypholoma sublateritium</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001317: 0-277585">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001317/index.html#r1c1" target="_new">BGC0001317</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.46, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>Terpene</td>
<td>lupeol</td>
<td>Lotus japonicus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002449: 47-14377">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002449/index.html#r1c1" target="_new">BGC0002449</a></td>


<td class="cc-heat-cell" title="0.44 (id:0.43, order:N/A, components:0.45)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
<td>0.44</td>
<td>Other</td>
<td>sarasinoside</td>
<td>uncultured bacterium</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002402: 203-37725">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002402/index.html#r1c1" target="_new">BGC0002402</a></td>


<td class="cc-heat-cell" title="0.44 (id:0.45, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
<td>0.44</td>
<td>Terpene</td>
<td>yossosides</td>
<td>Spinacia oleracea</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001219: 0-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001219/index.html#r1c1" target="_new">BGC0001219</a></td>


<td class="cc-heat-cell" title="0.43 (id:0.44, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
<td>0.43</td>
<td>Polyketide</td>
<td>shanorellin</td>
<td>Chaetomium globosum CBS 148.51</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001893: 155-19434">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001893/index.html#r1c1" target="_new">BGC0001893</a></td>


<td class="cc-heat-cell" title="0.42 (id:0.43, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
<td>0.42</td>
<td>Other</td>
<td>abscisic acid</td>
<td>Botrytis cinerea B05.10</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000676: 621-14838">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000676/index.html#r1c1" target="_new">BGC0000676</a></td>


<td class="cc-heat-cell" title="0.42 (id:0.41, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
<td>0.42</td>
<td>Terpene</td>
<td>aphidicolin, aphidicolan-16-ol, 3-deoxyaphidicolin, 17-deoxyaphidicolin</td>
<td>Phoma betae</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r31c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-350"></div> <span class="help-tooltip" id="cb-general-help-350">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r31c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r31c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r31c1_1.svg">
NW_014013668 (153970-222143): Trichoderma virens Gv29-8 chromosome Unknown TR... (18% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r31c1_2.svg">
NW_014574706 (185039-217400): Metarhizium brunneum ARSEF 3297 chromosome Unkn... (30% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r31c1_3.svg">
NW_025764724 (207194-239564): Emericellopsis atlantica unplaced genomic scaff... (23% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r31c1_4.svg">
NC_043740 (4056696-4089123): Pyricularia pennisetigena strain Br36 chromosome... (44% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r31c1_5.svg">
NW_006271977 (650506-682810): Cordyceps militaris CM01 unplaced genomic scaff... (28% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r31c1_6.svg">
NC_035794 (3381868-3414218): Pochonia chlamydosporia 170 chromosome 5, whole ... (33% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r31c1_7.svg">
NC_016473 (2758341-2790421): Thermothelomyces thermophilus ATCC 42464 chromos... (30% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r31c1_8.svg">
NW_025408667 (5243071-5275419): Hirsutella rhossiliensis strain HR02 chromoso... (33% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r31c1_9.svg">
NC_054389 (10607680-10640044): Drechmeria coniospora strain ARSEF 6962 chromo... (25% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r31c1_10.svg">
NC_070537 (2323736-2356720): Fusarium keratoplasticum isolate Fu6.1 chromosom... (25% of genes show similarity), terpene
</option>

</select>
<button id= "clusterblast-r31c1-download">Download graphic</button>
</div>
<div id="clusterblast-r31c1-svg"></div>

</div>
</div>

<div class="body-details-section r31c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-351"></div> <span class="help-tooltip" id="cb-known-help-351">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r31c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r31c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r31c1_1.svg">
clavaric acid
</option>

</select>
<button id="knownclusterblast-r31c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r31c1-svg"></div>

</div>
</div>

<div class="body-details-section r31c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-352"></div> <span class="help-tooltip" id="cb-sub-help-352">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r31c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-354"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-354">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r31c1-pfam-details-svg" data-filename="scaffold_31_r31c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r31c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">









</div>









</div>

</div>

</div>





<div class="page" id='r32c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_32 - Region 1 - NRPS-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-801"></div> <span class="help-tooltip" id="region_svg-help-801">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_32.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r32c1-svg" data-filename="scaffold_32_r32c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 124,347 - 167,941 nt. (total: 43,595 nt)

<a class="cluster-rules-header" id="r32c1-rules-header" href="#r32c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r32c1-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r32c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-802"></div> <span class="help-tooltip" id="focus-panel-help-802">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r32c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r32c1-gene-table-details" data-name="r32c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r32c1-nrps_pks" data-name="r32c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r32c1-MIBiG-cluster-compare" data-name="r32c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r32c1-clusterblast" data-name="r32c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r32c1-knownclusterblast" data-name="r32c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r32c1-subclusterblast" data-name="r32c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r32c1-nrps-pks-bubbles" data-name="r32c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r32c1-pfam-details" data-name="r32c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r32c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-363"></div> <span class="help-tooltip" id="gt-help-tooltip-help-363">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r32c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r32c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r32c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r32c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r32c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-355"></div> <span class="help-tooltip" id="nrps-domain-help-355">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r32c1-details-svg" data-filename="scaffold_32_r32c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r32c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r32c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-356"></div> <span class="help-tooltip" id="MIBiG-help-356">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r32c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r32c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r32c1"></div>
<div id="comparison-MIBiG-r32c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001168: 0-4582">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001168/index.html#r1c1" target="_new">BGC0001168</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.31</td>
<td>NRP</td>

<td>livipeptin</td>


<td>Streptomyces lividans 1326</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002602: 406-14432">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002602/index.html#r1c1" target="_new">BGC0002602</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.41, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.21</td>
<td>Alkaloid</td>

<td>aspergillic acid</td>


<td>Aspergillus flavus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001516: 0-14978">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001516/index.html#r1c1" target="_new">BGC0001516</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.41, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.21</td>
<td>NRP</td>

<td>aspergillic acid</td>


<td>Aspergillus flavus NRRL3357</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001599: 0-9373">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001599/index.html#r1c1" target="_new">BGC0001599</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.42, order:N/A, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.20</td>
<td>NRP</td>

<td>fragin</td>


<td>Burkholderia cenocepacia H111</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002164: 0-8322">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002164/index.html#r1c1" target="_new">BGC0002164</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>peramine</td>


<td>Epichloe festucae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002582: 0-7207">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002582/index.html#r1c1" target="_new">BGC0002582</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>guanipiperazine A, guanipiperazine B</td>


<td>Streptomyces chrestomyceticus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002278: 0-28520">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002278/index.html#r1c1" target="_new">BGC0002278</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.41, order:N/A, components:0.59)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>Alkaloid, NRP</td>

<td>brasiliamide I, brasiliamide A, brasiliamide B, 2,2-dibenzylpiperazine, brasiliamide D</td>


<td>Penicillium brasilianum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001517: 8989-24405">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001517/index.html#r1c1" target="_new">BGC0001517</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.41, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>asperphenamate</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002166: 0-8397">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002166/index.html#r1c1" target="_new">BGC0002166</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.41, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.17</td>
<td>NRP</td>

<td>peramine, intermediate 1, intermediate 2</td>


<td>Metarhizium rileyi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002135: 0-11508">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002135/index.html#r1c1" target="_new">BGC0002135</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.17</td>
<td>NRP</td>

<td>bovienimide A</td>


<td>Xenorhabdus bovienii SS-2004</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r32c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001168: 0-4582">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001168/index.html#r1c1" target="_new">BGC0001168</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>NRP</td>
<td>livipeptin</td>
<td>Streptomyces lividans 1326</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002602: 406-14432">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002602/index.html#r1c1" target="_new">BGC0002602</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.41, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Alkaloid</td>
<td>aspergillic acid</td>
<td>Aspergillus flavus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001516: 0-14978">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001516/index.html#r1c1" target="_new">BGC0001516</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.41, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>NRP</td>
<td>aspergillic acid</td>
<td>Aspergillus flavus NRRL3357</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001599: 0-9373">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001599/index.html#r1c1" target="_new">BGC0001599</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.42, order:N/A, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>NRP</td>
<td>fragin</td>
<td>Burkholderia cenocepacia H111</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002164: 0-8322">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002164/index.html#r1c1" target="_new">BGC0002164</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>NRP</td>
<td>peramine</td>
<td>Epichloe festucae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002582: 0-7207">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002582/index.html#r1c1" target="_new">BGC0002582</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>NRP</td>
<td>guanipiperazine A, guanipiperazine B</td>
<td>Streptomyces chrestomyceticus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002278: 0-28520">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002278/index.html#r1c1" target="_new">BGC0002278</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.41, order:N/A, components:0.59)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>Alkaloid, NRP</td>
<td>brasiliamide I, brasiliamide A, brasiliamide B, 2,2-dibenzylpiperazine, brasiliamide D</td>
<td>Penicillium brasilianum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001517: 8989-24405">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001517/index.html#r1c1" target="_new">BGC0001517</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.41, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>NRP</td>
<td>asperphenamate</td>
<td>Aspergillus terreus NIH2624</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002166: 0-8397">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002166/index.html#r1c1" target="_new">BGC0002166</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.41, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>NRP</td>
<td>peramine, intermediate 1, intermediate 2</td>
<td>Metarhizium rileyi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002135: 0-11508">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002135/index.html#r1c1" target="_new">BGC0002135</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>NRP</td>
<td>bovienimide A</td>
<td>Xenorhabdus bovienii SS-2004</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r32c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-357"></div> <span class="help-tooltip" id="cb-general-help-357">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r32c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r32c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r32c1_1.svg">
NW_022985370 (102721-166307): Macroventuria anomochaeta strain CBS 525.71 unp... (8% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r32c1_2.svg">
NW_017264921 (1614502-1679090): Kwoniella dejecticola CBS 10117 unplaced geno... (11% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r32c1_3.svg">
NW_017566967 (1486580-1551139): Kwoniella pini CBS 10737 unplaced genomic sca... (8% of genes show similarity), NRPS-like
</option>

</select>
<button id= "clusterblast-r32c1-download">Download graphic</button>
</div>
<div id="clusterblast-r32c1-svg"></div>

</div>
</div>

<div class="body-details-section r32c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-358"></div> <span class="help-tooltip" id="cb-known-help-358">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r32c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-359"></div> <span class="help-tooltip" id="cb-sub-help-359">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r32c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-362"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-362">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r32c1-domain-bubble-svg" data-filename="scaffold_32_r32c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r32c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (124346 - 167941): single NRPS-like
</option>


</select>

<div class="details-svg" id="r32c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r32c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r32c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-364"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-364">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r32c1-pfam-details-svg" data-filename="scaffold_32_r32c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r32c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r32c1-nrps_pks_products" data-name="r32c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r32c1-nrps_pks_monomers" data-name="r32c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r32c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-360"></div> <span class="help-tooltip" id="nrps-structures-help-360">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 124346 - 167941:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r32c1-1" class="smiles-canvas" data-smiles="NC(CCCC(=O)O)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Aad)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=aad" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=aad*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r32c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC(CCCC(=O)O)C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r32c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-361"></div> <span class="help-tooltip" id="nrps-monomers-help-361">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">FUN_006438</span></strong>: Aad
<div class="collapser collapser-target-FUN_006438 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=aad" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=aad*" target="_new">relaxed</a>
</dd>
<br>



<dd><strong>AMP-binding (26..493)</strong>: Aad
<div class="collapser collapser-target-FUN_006438 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Aad
<div class="collapser collapser-target-FUN_006438-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Aliphatic chain with H-bond donor (Asp, Asn, Glu, Gln, Aad)</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DPRHFVLRAK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Aad <span class="serif">DPRHFVMRAK</span> (82% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>90% (moderate)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>





<div class="page" id='r34c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_34 - Region 1 - NRPS-like,T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-803"></div> <span class="help-tooltip" id="region_svg-help-803">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_34.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r34c1-svg" data-filename="scaffold_34_r34c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 52,342 - 126,418 nt. (total: 74,077 nt)

<a class="cluster-rules-header" id="r34c1-rules-header" href="#r34c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r34c1-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))<br>T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r34c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-804"></div> <span class="help-tooltip" id="focus-panel-help-804">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r34c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r34c1-gene-table-details" data-name="r34c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r34c1-nrps_pks" data-name="r34c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r34c1-MIBiG-cluster-compare" data-name="r34c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r34c1-clusterblast" data-name="r34c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r34c1-knownclusterblast" data-name="r34c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r34c1-subclusterblast" data-name="r34c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r34c1-nrps-pks-bubbles" data-name="r34c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r34c1-pfam-details" data-name="r34c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r34c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-373"></div> <span class="help-tooltip" id="gt-help-tooltip-help-373">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r34c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r34c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r34c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r34c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r34c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-365"></div> <span class="help-tooltip" id="nrps-domain-help-365">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r34c1-details-svg" data-filename="scaffold_34_r34c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r34c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r34c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-366"></div> <span class="help-tooltip" id="MIBiG-help-366">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r34c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r34c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r34c1"></div>
<div id="comparison-MIBiG-r34c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS-like</span></th>

<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001616: 488-11614">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001616/index.html#r1c1" target="_new">BGC0001616</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>






<td class="cc-heat-cell" title="0.62 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.56</td>
<td>NRP, Polyketide</td>

<td>himeic acid A</td>


<td>Aspergillus japonicus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001893: 155-19434">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001893/index.html#r1c1" target="_new">BGC0001893</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>






<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.47</td>
<td>Other</td>

<td>abscisic acid</td>


<td>Botrytis cinerea B05.10</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001317: 0-277585">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001317/index.html#r1c1" target="_new">BGC0001317</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>






<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.47</td>
<td>Terpene</td>

<td>lupeol</td>


<td>Lotus japonicus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001969: 0-7810">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001969/index.html#r1c1" target="_new">BGC0001969</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>






<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.43</td>
<td>Terpene</td>

<td>asperterpenoid A</td>


<td>Talaromyces wortmannii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264: 8442-40380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264</a></td>





<td class="cc-heat-cell" title="0.44 (id:0.43, order:0.32, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>






<td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.42</td>
<td>Polyketide</td>

<td>betaenone A, betaenone B, betaenone C</td>


<td>Phoma betae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000057: 0-26552">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000057/index.html#r1c1" target="_new">BGC0000057</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.45, order:N/A, components:0.47)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>






<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.37</td>
<td>Polyketide</td>

<td>F9775A, F9775B, orsellinic acid</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002213: 0-33851">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002213/index.html#r1c1" target="_new">BGC0002213</a></td>





<td class="cc-heat-cell" title="0.42 (id:0.43, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>






<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.37</td>
<td>Polyketide</td>

<td>HEx-pks16 polyketide</td>


<td>Punctularia strigosozonata HHB-11173 SS5</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001811: 32-25672">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001811/index.html#r1c1" target="_new">BGC0001811</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.46, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>






<td class="cc-heat-cell" title="0.51 (id:0.42, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>


<td>0.37</td>
<td>Terpene</td>

<td>trichodiene-11-one</td>


<td>Fusarium asiaticum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000930: 2573-28230">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000930/index.html#r1c1" target="_new">BGC0000930</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.46, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>






<td class="cc-heat-cell" title="0.51 (id:0.42, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>


<td>0.37</td>
<td>Terpene</td>

<td>deoxynivalenol</td>


<td>Fusarium graminearum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000012: 1892-19092">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000012/index.html#r1c1" target="_new">BGC0000012</a></td>





<td class="cc-heat-cell" title="0.41 (id:0.44, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>






<td class="cc-heat-cell" title="0.56 (id:0.44, order:0.42, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.35</td>
<td>Polyketide</td>

<td>alternapyrone</td>


<td>Alternaria solani</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r34c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002515: 0-30087">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002515/index.html#r1c1" target="_new">BGC0002515</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>Polyketide</td>
<td>solanapyrone A</td>
<td>Ascochyta rabiei</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000146: 754-20637">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000146/index.html#r1c1" target="_new">BGC0000146</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>Polyketide</td>
<td>solanapyrone D</td>
<td>Alternaria solani</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002525: 1000-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002525/index.html#r1c1" target="_new">BGC0002525</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Polyketide</td>
<td>fusarubin, 1233A, 1233B, NG-391, lucilactaene</td>
<td>Fusarium sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001317: 0-277585">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001317/index.html#r1c1" target="_new">BGC0001317</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Terpene</td>
<td>lupeol</td>
<td>Lotus japonicus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002194: 147-15389">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002194/index.html#r1c1" target="_new">BGC0002194</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Polyketide</td>
<td>epipyrone A</td>
<td>Epicoccum nigrum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001893: 155-19434">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001893/index.html#r1c1" target="_new">BGC0001893</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Other</td>
<td>abscisic acid</td>
<td>Botrytis cinerea B05.10</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000894: 395-43446">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000894/index.html#r1c1" target="_new">BGC0000894</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Other</td>
<td>citrinin</td>
<td>Monascus aurantiacus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001606: 0-22544">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001606/index.html#r1c1" target="_new">BGC0001606</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Polyketide</td>
<td>gibepyrone-A</td>
<td>Fusarium fujikuroi IMI 58289</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002162: 0-5465">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002162/index.html#r1c1" target="_new">BGC0002162</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Terpene</td>
<td>sesterfisherol, sesterfisheric acid, compound 6, compound 7a, compound 5, compound 8</td>
<td>Aspergillus fischeri NRRL 181</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002191: 260-8785">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002191/index.html#r1c1" target="_new">BGC0002191</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Polyketide</td>
<td>prolipyrone B, gibepyrone D</td>
<td>Fusarium graminearum PH-1</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r34c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-367"></div> <span class="help-tooltip" id="cb-general-help-367">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r34c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r34c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r34c1_1.svg">
NW_025953251 (1891693-2006513): Annulohypoxylon maeteangense strain CBS 12383... (16% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r34c1_2.svg">
NW_017264195 (1737648-1805945): Xylona heveae TC161 unplaced genomic scaffold... (15% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r34c1_3.svg">
NW_026055692 (251875-319758): Alternaria ventricosa strain BMP 2768 chromosom... (13% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r34c1_4.svg">
NW_026249861 (757279-825156): Alternaria incomplexa strain BMP 0042 chromosom... (14% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r34c1_5.svg">
NW_017306213 (429756-499159): Alternaria alternata strain SRC1lrK2f unplaced ... (13% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r34c1_6.svg">
NW_016017384 (2619275-2687148): Sphaerulina musiva SO2202 unplaced genomic sc... (11% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r34c1_7.svg">
NW_001820835 (1857217-1928718): Sclerotinia sclerotiorum 1980 UF-70 scaffold ... (10% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r34c1_8.svg">
NW_024468910 (5414182-5490619): Cryphonectria parasitica EP155 unplaced genom... (23% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r34c1_9.svg">
NW_024469024 (26525-267638): Aspergillus fijiensis CBS 313.89 unplaced genomi... (6% of genes show similarity), NRPS,T1PKS,fungal-RiPP-like,other
</option>

<option value = "svg/clusterblast_r34c1_10.svg">
NC_016474 (4540934-4674970): Thermothelomyces thermophilus ATCC 42464 chromos... (6% of genes show similarity), T1PKS
</option>

</select>
<button id= "clusterblast-r34c1-download">Download graphic</button>
</div>
<div id="clusterblast-r34c1-svg"></div>

</div>
</div>

<div class="body-details-section r34c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-368"></div> <span class="help-tooltip" id="cb-known-help-368">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r34c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-369"></div> <span class="help-tooltip" id="cb-sub-help-369">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r34c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-372"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-372">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r34c1-domain-bubble-svg" data-filename="scaffold_34_r34c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r34c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (52341 - 126418): neighbouring NRPS-like-T1PKS
</option>





<option value="CC3">
Candidate 3 (78231 - 126418): single T1PKS
</option>


</select>

<div class="details-svg" id="r34c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r34c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r34c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-374"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-374">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r34c1-pfam-details-svg" data-filename="scaffold_34_r34c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r34c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r34c1-nrps_pks_products" data-name="r34c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r34c1-nrps_pks_monomers" data-name="r34c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r34c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-370"></div> <span class="help-tooltip" id="nrps-structures-help-370">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 52341 - 126418:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r34c1-1" class="smiles-canvas" data-smiles="C([*])C(-O)(C)"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Me-pk)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-pk" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-pk*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r34c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)(C)"></canvas>
</div>
</div>


<hr>






<div>
<div class="prediction-text">
For candidate cluster 3, location 78231 - 126418:
<div class="collapser collapser-target-CC3 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r34c1-3" class="smiles-canvas" data-smiles="C([*])C(-O)(C)"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Me-pk)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-pk" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-pk*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r34c1-3-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)(C)"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r34c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-371"></div> <span class="help-tooltip" id="nrps-monomers-help-371">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">FUN_006600</span></strong>: X
<div class="collapser collapser-target-FUN_006600 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (20..306)</strong>: X
<div class="collapser collapser-target-FUN_006600 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  (unknown)
<div class="collapser collapser-target-FUN_006600-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>  <strong>NOTE: uncertain match</strong><br>
<dl>   <dt>Predicted physicochemical class:</dt>
<dd>N/A</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">YAWVAG---K</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>

<dt>Stachelhaus code match:</dt>
<dd>0% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

<strong><span class="serif">FUN_006605</span></strong>: pk
<div class="collapser collapser-target-FUN_006605 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (606..922)</strong>: pk
<div class="collapser collapser-target-FUN_006605 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-FUN_006605-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 70.8%</dd>
<dd>Methylmalonyl-CoA: 62.5%</dd>
<dd>Isobutyryl-CoA: 58.3%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-FUN_006605-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 62.7</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 57.3</dt>

<dd></dd><dt>Malonyl-CoA: 43.0</dt>

<dd></dd><dt>Isobutyryl-CoA: 23.5</dt>

<dd></dd><dt>Propionyl-CoA: 18.5</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 17.6</dt>

<dd></dd><dt>inactive: 17.0</dt>

<dd></dd><dt>trans-1,2-CPDA: 16.8</dt>

<dd></dd><dt>CHC-CoA: 15.1</dt>

<dd></dd><dt>Benzoyl-CoA: 12.6</dt>

<dd></dd><dt>fatty_acid: 12.0</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 8.8</dt>

<dd></dd><dt>Acetyl-CoA: 0.0</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>












</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>





<div class="page" id='r35c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_35 - Region 1 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-805"></div> <span class="help-tooltip" id="region_svg-help-805">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_35.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r35c1-svg" data-filename="scaffold_35_r35c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 13,697 - 57,162 nt. (total: 43,466 nt)

<a class="cluster-rules-header" id="r35c1-rules-header" href="#r35c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r35c1-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r35c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-806"></div> <span class="help-tooltip" id="focus-panel-help-806">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r35c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r35c1-gene-table-details" data-name="r35c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r35c1-nrps_pks" data-name="r35c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r35c1-MIBiG-cluster-compare" data-name="r35c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r35c1-clusterblast" data-name="r35c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r35c1-knownclusterblast" data-name="r35c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r35c1-subclusterblast" data-name="r35c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r35c1-nrps-pks-bubbles" data-name="r35c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r35c1-pfam-details" data-name="r35c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r35c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-383"></div> <span class="help-tooltip" id="gt-help-tooltip-help-383">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r35c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r35c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r35c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r35c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r35c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-375"></div> <span class="help-tooltip" id="nrps-domain-help-375">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r35c1-details-svg" data-filename="scaffold_35_r35c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r35c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r35c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-376"></div> <span class="help-tooltip" id="MIBiG-help-376">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r35c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r35c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r35c1"></div>
<div id="comparison-MIBiG-r35c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002191: 260-8785">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002191/index.html#r1c1" target="_new">BGC0002191</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.33</td>
<td>Polyketide</td>

<td>prolipyrone B, gibepyrone D</td>


<td>Fusarium graminearum PH-1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001252: 0-7958">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001252/index.html#r1c1" target="_new">BGC0001252</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.33</td>
<td>Polyketide</td>

<td>UNII-YC2Q1O94PT</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002155: 78-16511">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002155/index.html#r1c1" target="_new">BGC0002155</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.44, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.30</td>
<td>Polyketide</td>

<td>nectriapyrone C, nectriapyrone D, nectriapyrone</td>


<td>Pyricularia oryzae 70-15</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002525: 1000-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002525/index.html#r1c1" target="_new">BGC0002525</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.49, order:N/A, components:0.82)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.30</td>
<td>Polyketide</td>

<td>fusarubin, 1233A, 1233B, NG-391, lucilactaene</td>


<td>Fusarium sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001606: 0-22544">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001606/index.html#r1c1" target="_new">BGC0001606</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.28</td>
<td>Polyketide</td>

<td>gibepyrone-A</td>


<td>Fusarium fujikuroi IMI 58289</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002222: 888-32276">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002222/index.html#r1c1" target="_new">BGC0002222</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.28</td>
<td>Polyketide</td>

<td>zopfiellin</td>


<td>Diffractella curvata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002515: 0-30087">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002515/index.html#r1c1" target="_new">BGC0002515</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.27</td>
<td>Polyketide</td>

<td>solanapyrone A</td>


<td>Ascochyta rabiei</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002224: 685-37748">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002224/index.html#r1c1" target="_new">BGC0002224</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.45, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.26</td>
<td>Polyketide</td>

<td>zopfiellin</td>


<td>Diffractella curvata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000146: 754-20637">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000146/index.html#r1c1" target="_new">BGC0000146</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>Polyketide</td>

<td>solanapyrone D</td>


<td>Alternaria solani</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002223: 2980-46085">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002223/index.html#r1c1" target="_new">BGC0002223</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>Polyketide</td>

<td>scytalidin</td>


<td>Scytalidium album</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r35c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002191: 260-8785">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002191/index.html#r1c1" target="_new">BGC0002191</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Polyketide</td>
<td>prolipyrone B, gibepyrone D</td>
<td>Fusarium graminearum PH-1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001252: 0-7958">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001252/index.html#r1c1" target="_new">BGC0001252</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Polyketide</td>
<td>UNII-YC2Q1O94PT</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002155: 78-16511">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002155/index.html#r1c1" target="_new">BGC0002155</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.44, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>Polyketide</td>
<td>nectriapyrone C, nectriapyrone D, nectriapyrone</td>
<td>Pyricularia oryzae 70-15</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002525: 1000-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002525/index.html#r1c1" target="_new">BGC0002525</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.49, order:N/A, components:0.82)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>Polyketide</td>
<td>fusarubin, 1233A, 1233B, NG-391, lucilactaene</td>
<td>Fusarium sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001606: 0-22544">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001606/index.html#r1c1" target="_new">BGC0001606</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>Polyketide</td>
<td>gibepyrone-A</td>
<td>Fusarium fujikuroi IMI 58289</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002222: 888-32276">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002222/index.html#r1c1" target="_new">BGC0002222</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>Polyketide</td>
<td>zopfiellin</td>
<td>Diffractella curvata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002515: 0-30087">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002515/index.html#r1c1" target="_new">BGC0002515</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>Polyketide</td>
<td>solanapyrone A</td>
<td>Ascochyta rabiei</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002224: 685-37748">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002224/index.html#r1c1" target="_new">BGC0002224</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.45, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>Polyketide</td>
<td>zopfiellin</td>
<td>Diffractella curvata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000146: 754-20637">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000146/index.html#r1c1" target="_new">BGC0000146</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Polyketide</td>
<td>solanapyrone D</td>
<td>Alternaria solani</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002223: 2980-46085">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002223/index.html#r1c1" target="_new">BGC0002223</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Polyketide</td>
<td>scytalidin</td>
<td>Scytalidium album</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r35c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-377"></div> <span class="help-tooltip" id="cb-general-help-377">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r35c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r35c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r35c1_1.svg">
NZ_CP054916 (8040289-8243324): Streptomyces sp. NA02950 chromosome, complete ... (6% of genes show similarity), T1PKS,butyrolactone,hglE-KS,thioamide-NRP
</option>

<option value = "svg/clusterblast_r35c1_2.svg">
NZ_CP023690 (7957332-8031858): Streptomyces spectabilis strain ATCC 27465 chr... (9% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r35c1_3.svg">
NZ_QQAU01000003 (36564-138179): Lentzea flaviverrucosa strain DSM 44664 Ga024... (10% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r35c1_4.svg">
NZ_CP094828 (4703006-4778542): Acuticoccus sp. I52.16.1 chromosome, complete ... (8% of genes show similarity), NRPS,NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r35c1_5.svg">
NC_022785 (11126401-11244528): Streptomyces rapamycinicus NRRL 5491, complete... (8% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r35c1_6.svg">
NZ_CP030771 (7915496-7990387): Streptomyces sp. YIM 121038 chromosome, comple... (7% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r35c1_7.svg">
NZ_CP023689 (953739-1189865): Streptomyces chartreusis strain ATCC 14922 chro... (20% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r35c1_8.svg">
NC_049566 (158150-226119): Talaromyces rugulosus chromosome VI, complete sequ... (16% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r35c1_9.svg">
NZ_CP073721 (2935945-3274075): Dactylosporangium roseum strain NRRL B-16295 c... (3% of genes show similarity), NRPS,NRPS-like,PKS-like,T1PKS,T2PKS
</option>

<option value = "svg/clusterblast_r35c1_10.svg">
NZ_CP056041 (8564408-8747614): Streptomyces chartreusis strain NA02069 chromo... (13% of genes show similarity), T1PKS
</option>

</select>
<button id= "clusterblast-r35c1-download">Download graphic</button>
</div>
<div id="clusterblast-r35c1-svg"></div>

</div>
</div>

<div class="body-details-section r35c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-378"></div> <span class="help-tooltip" id="cb-known-help-378">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r35c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-379"></div> <span class="help-tooltip" id="cb-sub-help-379">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r35c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-382"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-382">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r35c1-domain-bubble-svg" data-filename="scaffold_35_r35c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r35c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (13696 - 57162): single T1PKS
</option>


</select>

<div class="details-svg" id="r35c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r35c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r35c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-384"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-384">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r35c1-pfam-details-svg" data-filename="scaffold_35_r35c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r35c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r35c1-nrps_pks_products" data-name="r35c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r35c1-nrps_pks_monomers" data-name="r35c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r35c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-380"></div> <span class="help-tooltip" id="nrps-structures-help-380">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 13696 - 57162:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r35c1-1" class="smiles-canvas" data-smiles="C([*])C(-O)(C)"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS linker matching; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Me-pk)</dd>

<dt><b>Non-colinear ordering:</b></dt><dd>

<span class="serif">FUN_006685</span>

+ <span class="serif">FUN_006684</span>

</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-pk" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-pk*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r35c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)(C)"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r35c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-381"></div> <span class="help-tooltip" id="nrps-monomers-help-381">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">FUN_006685</span></strong>: pk
<div class="collapser collapser-target-FUN_006685 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (567..873)</strong>: pk
<div class="collapser collapser-target-FUN_006685 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-FUN_006685-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 75.0%</dd>
<dd>Methylmalonyl-CoA: 66.7%</dd>
<dd>Methoxymalonyl-CoA: 62.5%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-FUN_006685-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 75.4</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 66.2</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 47.0</dt>

<dd></dd><dt>Propionyl-CoA: 35.0</dt>

<dd></dd><dt>trans-1,2-CPDA: 34.0</dt>

<dd></dd><dt>Isobutyryl-CoA: 31.9</dt>

<dd></dd><dt>Malonyl-CoA: 27.9</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 24.4</dt>

<dd></dd><dt>Benzoyl-CoA: 20.2</dt>

<dd></dd><dt>fatty_acid: 18.7</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 15.6</dt>

<dd></dd><dt>inactive: 14.5</dt>

<dd></dd><dt>CHC-CoA: 14.1</dt>

<dd></dd><dt>Acetyl-CoA: 8.6</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r35c2' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_35 - Region 2 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-807"></div> <span class="help-tooltip" id="region_svg-help-807">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_35.region002.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r35c2-svg" data-filename="scaffold_35_r35c2.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 280,865 - 302,065 nt. (total: 21,201 nt)

<a class="cluster-rules-header" id="r35c2-rules-header" href="#r35c2">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r35c2-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>

<div class="region-svg-container">
<div id='r35c2-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-808"></div> <span class="help-tooltip" id="focus-panel-help-808">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r35c2">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r35c2-gene-table-details" data-name="r35c2-gene-table-details">
Gene overview
</div>



<div class="body-details-header r35c2-MIBiG-cluster-compare" data-name="r35c2-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r35c2-clusterblast" data-name="r35c2-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r35c2-knownclusterblast" data-name="r35c2-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r35c2-subclusterblast" data-name="r35c2-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r35c2-pfam-details" data-name="r35c2-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r35c2-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-389"></div> <span class="help-tooltip" id="gt-help-tooltip-help-389">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r35c2">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r35c2-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r35c2-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r35c2">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r35c2-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-385"></div> <span class="help-tooltip" id="MIBiG-help-385">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r35c2-selector">Analysis type:</label>
<select id="comparison-MIBiG-r35c2-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r35c2"></div>
<div id="comparison-MIBiG-r35c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">terpene</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002176: 0-1199">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002176/index.html#r1c1" target="_new">BGC0002176</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.34</td>
<td>Terpene</td>

<td>culmorin, (+)-juniperol(longiborneol), 15-acetyldeoxynivalenol</td>


<td>Fusarium graminearum PH-1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001278: 588-54488">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001278/index.html#r1c1" target="_new">BGC0001278</a></td>





<td class="cc-heat-cell" title="0.16 (id:0.41, order:N/A, components:0.06)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.16), rgb(0, 0, 0, 0.16) 16%, white 16%)"></td>


<td>0.03</td>
<td>Terpene</td>

<td>nivalenol, deoxynivalenol, 3-acetyldeoxynivalenol, 15-acetyldeoxynivalenol, neosolaniol, calonectrin, apotrichodiol, isotrichotriol, 15-decalonectrin, T-2 toxin, 3-acetyl T-2 toxin, trichodiene</td>


<td>Fusarium sporotrichioides</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r35c2-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002176: 0-1199">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002176/index.html#r1c1" target="_new">BGC0002176</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Terpene</td>
<td>culmorin, (+)-juniperol(longiborneol), 15-acetyldeoxynivalenol</td>
<td>Fusarium graminearum PH-1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001278: 588-54488">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001278/index.html#r1c1" target="_new">BGC0001278</a></td>


<td class="cc-heat-cell" title="0.16 (id:0.41, order:N/A, components:0.06)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.16), rgb(0, 0, 0, 0.16) 16%, white 16%)"></td>
<td>0.16</td>
<td>Terpene</td>
<td>nivalenol, deoxynivalenol, 3-acetyldeoxynivalenol, 15-acetyldeoxynivalenol, neosolaniol, calonectrin, apotrichodiol, isotrichotriol, 15-decalonectrin, T-2 toxin, 3-acetyl T-2 toxin, trichodiene</td>
<td>Fusarium sporotrichioides</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r35c2-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-386"></div> <span class="help-tooltip" id="cb-general-help-386">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<p>No significant ClusterBlast hits found.</p>
</div>

</div>
</div>

<div class="body-details-section r35c2-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-387"></div> <span class="help-tooltip" id="cb-known-help-387">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r35c2-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-388"></div> <span class="help-tooltip" id="cb-sub-help-388">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r35c2-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-390"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-390">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r35c2-pfam-details-svg" data-filename="scaffold_35_r35c2_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r35c2-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">









</div>









</div>

</div>

</div>





<div class="page" id='r36c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_36 - Region 1 - fungal-RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-809"></div> <span class="help-tooltip" id="region_svg-help-809">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_36.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r36c1-svg" data-filename="scaffold_36_r36c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 100,669 - 162,622 nt. (total: 61,954 nt)

<a class="cluster-rules-header" id="r36c1-rules-header" href="#r36c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r36c1-rules">fungal-RiPP-like: DUF3328</div>

<div class="region-svg-container">
<div id='r36c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-810"></div> <span class="help-tooltip" id="focus-panel-help-810">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r36c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r36c1-gene-table-details" data-name="r36c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r36c1-MIBiG-cluster-compare" data-name="r36c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r36c1-clusterblast" data-name="r36c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r36c1-knownclusterblast" data-name="r36c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r36c1-subclusterblast" data-name="r36c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r36c1-pfam-details" data-name="r36c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r36c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-395"></div> <span class="help-tooltip" id="gt-help-tooltip-help-395">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r36c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r36c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r36c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r36c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r36c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-391"></div> <span class="help-tooltip" id="MIBiG-help-391">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r36c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r36c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r36c1"></div>
<div id="comparison-MIBiG-r36c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">fungal-RiPP-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001990: 130-13646">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001990/index.html#r1c1" target="_new">BGC0001990</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>Other</td>

<td>BU-4704, xanthocillin-X monomethylether, melanocin E, melanocin F, fumiformamide</td>


<td>Aspergillus fumigatus Af293</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000667: 565-7939">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000667/index.html#r1c1" target="_new">BGC0000667</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.15</td>
<td>Terpene</td>

<td>PR-toxin</td>


<td>Penicillium roqueforti FM164</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002262: 0-50372">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002262/index.html#r1c1" target="_new">BGC0002262</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.15</td>
<td>Polyketide</td>

<td>trienylfuranol A, sporothriolide, dihydrosporothriolide, deoxysporothric acid, dihydroisosporothric acid, sporothric acid, isosporothric acid, sporochartine B</td>


<td>Hypomontagnella monticulosa</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001398: 4999-64171">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001398/index.html#r1c1" target="_new">BGC0001398</a></td>





<td class="cc-heat-cell" title="0.44 (id:0.42, order:N/A, components:0.47)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>


<td>0.15</td>
<td>RiPP</td>

<td>phomopsin A, phomopsin B, phomopsin E</td>


<td>Phomopsis leptostromiformis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264: 8442-40380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264</a></td>





<td class="cc-heat-cell" title="0.43 (id:0.46, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>


<td>0.14</td>
<td>Polyketide</td>

<td>betaenone A, betaenone B, betaenone C</td>


<td>Phoma betae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002180: 263-24876">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002180/index.html#r1c1" target="_new">BGC0002180</a></td>





<td class="cc-heat-cell" title="0.40 (id:0.48, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>


<td>0.12</td>
<td>Polyketide</td>

<td>(+)-asperlin, (2Z,4Z,6E)-octa-2,4,6-trienoic acid</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001266: 10707-26494">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001266/index.html#r1c1" target="_new">BGC0001266</a></td>





<td class="cc-heat-cell" title="0.40 (id:0.47, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>


<td>0.12</td>
<td>Polyketide</td>

<td>grayanic acid</td>


<td>Cladonia grayi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002210: 100-28102">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002210/index.html#r1c1" target="_new">BGC0002210</a></td>





<td class="cc-heat-cell" title="0.39 (id:0.46, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>


<td>0.12</td>
<td>Polyketide, NRP</td>

<td>pyranoviolin A</td>


<td>Aspergillus violaceofuscus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002216: 305-18235">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002216/index.html#r1c1" target="_new">BGC0002216</a></td>





<td class="cc-heat-cell" title="0.39 (id:0.41, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>


<td>0.12</td>
<td>Terpene</td>

<td>HEx-tc1 terpene</td>


<td>Trichoderma virens Gv29-8</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000046: 1267-21753">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000046/index.html#r1c1" target="_new">BGC0000046</a></td>





<td class="cc-heat-cell" title="0.39 (id:0.46, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>


<td>0.12</td>
<td>Polyketide</td>

<td>depudecin</td>


<td>Alternaria brassicicola</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r36c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001990: 130-13646">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001990/index.html#r1c1" target="_new">BGC0001990</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>Other</td>
<td>BU-4704, xanthocillin-X monomethylether, melanocin E, melanocin F, fumiformamide</td>
<td>Aspergillus fumigatus Af293</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000667: 565-7939">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000667/index.html#r1c1" target="_new">BGC0000667</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>Terpene</td>
<td>PR-toxin</td>
<td>Penicillium roqueforti FM164</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002262: 0-50372">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002262/index.html#r1c1" target="_new">BGC0002262</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>Polyketide</td>
<td>trienylfuranol A, sporothriolide, dihydrosporothriolide, deoxysporothric acid, dihydroisosporothric acid, sporothric acid, isosporothric acid, sporochartine B</td>
<td>Hypomontagnella monticulosa</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001398: 4999-64171">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001398/index.html#r1c1" target="_new">BGC0001398</a></td>


<td class="cc-heat-cell" title="0.44 (id:0.42, order:N/A, components:0.47)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
<td>0.44</td>
<td>RiPP</td>
<td>phomopsin A, phomopsin B, phomopsin E</td>
<td>Phomopsis leptostromiformis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264: 8442-40380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264</a></td>


<td class="cc-heat-cell" title="0.43 (id:0.46, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
<td>0.43</td>
<td>Polyketide</td>
<td>betaenone A, betaenone B, betaenone C</td>
<td>Phoma betae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002180: 263-24876">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002180/index.html#r1c1" target="_new">BGC0002180</a></td>


<td class="cc-heat-cell" title="0.40 (id:0.48, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>
<td>0.40</td>
<td>Polyketide</td>
<td>(+)-asperlin, (2Z,4Z,6E)-octa-2,4,6-trienoic acid</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001266: 10707-26494">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001266/index.html#r1c1" target="_new">BGC0001266</a></td>


<td class="cc-heat-cell" title="0.40 (id:0.47, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>
<td>0.40</td>
<td>Polyketide</td>
<td>grayanic acid</td>
<td>Cladonia grayi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002210: 100-28102">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002210/index.html#r1c1" target="_new">BGC0002210</a></td>


<td class="cc-heat-cell" title="0.39 (id:0.46, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
<td>0.39</td>
<td>Polyketide, NRP</td>
<td>pyranoviolin A</td>
<td>Aspergillus violaceofuscus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002216: 305-18235">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002216/index.html#r1c1" target="_new">BGC0002216</a></td>


<td class="cc-heat-cell" title="0.39 (id:0.41, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
<td>0.39</td>
<td>Terpene</td>
<td>HEx-tc1 terpene</td>
<td>Trichoderma virens Gv29-8</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000046: 1267-21753">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000046/index.html#r1c1" target="_new">BGC0000046</a></td>


<td class="cc-heat-cell" title="0.39 (id:0.46, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
<td>0.39</td>
<td>Polyketide</td>
<td>depudecin</td>
<td>Alternaria brassicicola</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r36c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-392"></div> <span class="help-tooltip" id="cb-general-help-392">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r36c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r36c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r36c1_1.svg">
NW_025953810 (264034-384248): Annulohypoxylon truncatum strain CBS 140777 unp... (14% of genes show similarity), NRPS-like,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r36c1_2.svg">
NW_025953253 (1303940-1416522): Annulohypoxylon maeteangense strain CBS 12383... (10% of genes show similarity), T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r36c1_3.svg">
NW_022987217 (1001823-1150250): Trematosphaeria pertusa strain CBS 122368 unp... (4% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r36c1_4.svg">
NW_022985567 (502836-594756): Pseudovirgaria hyperparasitica strain CBS 12173... (7% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r36c1_5.svg">
NC_054692 (1627715-1719410): Aspergillus flavus NRRL3357 chromosome 2, comple... (9% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r36c1_6.svg">
NC_054854 (2943347-3175583): Aspergillus luchuensis IFO 4308 DNA, chromosome ... (3% of genes show similarity), NRP-metallophore,NRPS,NRPS-like,T1PKS,fungal-RiPP-like,terpene
</option>

<option value = "svg/clusterblast_r36c1_7.svg">
NW_019154045 (358401-688746): Pochonia chlamydosporia 170 chromosome Unknown ... (2% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r36c1_8.svg">
NW_022985438 (2416691-2508564): Mytilinidion resinicola strain CBS 304.34 unp... (5% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r36c1_9.svg">
NC_036436 (1578644-1670400): Aspergillus oryzae RIB40 DNA, chromosome 2 (7% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r36c1_10.svg">
NC_072350 (525276-617173): Talaromyces marneffei chromosome 3, complete sequence (10% of genes show similarity), fungal-RiPP-like
</option>

</select>
<button id= "clusterblast-r36c1-download">Download graphic</button>
</div>
<div id="clusterblast-r36c1-svg"></div>

</div>
</div>

<div class="body-details-section r36c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-393"></div> <span class="help-tooltip" id="cb-known-help-393">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r36c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-394"></div> <span class="help-tooltip" id="cb-sub-help-394">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r36c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-396"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-396">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r36c1-pfam-details-svg" data-filename="scaffold_36_r36c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r36c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">











</div>











</div>

</div>

</div>





<div class="page" id='r38c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_38 - Region 1 - NRPS,T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-811"></div> <span class="help-tooltip" id="region_svg-help-811">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_38.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r38c1-svg" data-filename="scaffold_38_r38c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 144,862 - 199,842 nt. (total: 54,981 nt)

<a class="cluster-rules-header" id="r38c1-rules-header" href="#r38c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r38c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))<br>T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r38c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-812"></div> <span class="help-tooltip" id="focus-panel-help-812">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r38c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r38c1-gene-table-details" data-name="r38c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r38c1-nrps_pks" data-name="r38c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r38c1-MIBiG-cluster-compare" data-name="r38c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r38c1-clusterblast" data-name="r38c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r38c1-knownclusterblast" data-name="r38c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r38c1-subclusterblast" data-name="r38c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r38c1-nrps-pks-bubbles" data-name="r38c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r38c1-pfam-details" data-name="r38c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r38c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-405"></div> <span class="help-tooltip" id="gt-help-tooltip-help-405">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r38c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r38c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r38c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r38c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r38c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-397"></div> <span class="help-tooltip" id="nrps-domain-help-397">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r38c1-details-svg" data-filename="scaffold_38_r38c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r38c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r38c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-398"></div> <span class="help-tooltip" id="MIBiG-help-398">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r38c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r38c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r38c1"></div>
<div id="comparison-MIBiG-r38c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>

<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900: 2044-8911">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>






<td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.67</td>
<td>Other</td>

<td>ferrichrome</td>


<td>Aspergillus oryzae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249: 663-5994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>






<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.65</td>
<td>NRP</td>

<td>dimethylcoprogen</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001219: 0-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001219/index.html#r1c1" target="_new">BGC0001219</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>






<td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.64</td>
<td>Polyketide</td>

<td>shanorellin</td>


<td>Chaetomium globosum CBS 148.51</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264: 8442-40380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.47, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>






<td class="cc-heat-cell" title="0.65 (id:0.47, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.63</td>
<td>Polyketide</td>

<td>betaenone A, betaenone B, betaenone C</td>


<td>Phoma betae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000348: 0-13540">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000348/index.html#r1c1" target="_new">BGC0000348</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.44, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>






<td class="cc-heat-cell" title="0.63 (id:0.44, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.60</td>
<td>NRP</td>

<td>ergovaline</td>


<td>Epichloe festucae var. lolii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001718: 0-15216">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001718/index.html#r1c1" target="_new">BGC0001718</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>






<td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.57</td>
<td>NRP</td>

<td>okaramine D</td>


<td>Aspergillus aculeatus ATCC 16872</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375: 0-9457">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>






<td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.57</td>
<td>NRP</td>

<td>indigoidine</td>


<td>Streptomyces chromofuscus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002248: 5-21557">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002248/index.html#r1c1" target="_new">BGC0002248</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>






<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.55</td>
<td>Terpene, NRP</td>

<td>flavunoidine</td>


<td>Aspergillus flavus NRRL3357</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001717: 0-19662">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001717/index.html#r1c1" target="_new">BGC0001717</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>






<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.55</td>
<td>NRP</td>

<td>okaramine B</td>


<td>Penicillium simplicissimum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000343: 1-6208">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000343/index.html#r1c1" target="_new">BGC0000343</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>






<td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.55</td>
<td>NRP</td>

<td>enterobactin</td>


<td>Pseudomonas sp. J465</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r38c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900: 2044-8911">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Other</td>
<td>ferrichrome</td>
<td>Aspergillus oryzae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249: 663-5994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>NRP</td>
<td>dimethylcoprogen</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001219: 0-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001219/index.html#r1c1" target="_new">BGC0001219</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Polyketide</td>
<td>shanorellin</td>
<td>Chaetomium globosum CBS 148.51</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264: 8442-40380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.47, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Polyketide</td>
<td>betaenone A, betaenone B, betaenone C</td>
<td>Phoma betae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000348: 0-13540">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000348/index.html#r1c1" target="_new">BGC0000348</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.44, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>NRP</td>
<td>ergovaline</td>
<td>Epichloe festucae var. lolii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001718: 0-15216">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001718/index.html#r1c1" target="_new">BGC0001718</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>NRP</td>
<td>okaramine D</td>
<td>Aspergillus aculeatus ATCC 16872</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375: 0-9457">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>NRP</td>
<td>indigoidine</td>
<td>Streptomyces chromofuscus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002248: 5-21557">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002248/index.html#r1c1" target="_new">BGC0002248</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>Terpene, NRP</td>
<td>flavunoidine</td>
<td>Aspergillus flavus NRRL3357</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001717: 0-19662">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001717/index.html#r1c1" target="_new">BGC0001717</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>NRP</td>
<td>okaramine B</td>
<td>Penicillium simplicissimum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000343: 1-6208">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000343/index.html#r1c1" target="_new">BGC0000343</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>NRP</td>
<td>enterobactin</td>
<td>Pseudomonas sp. J465</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r38c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-399"></div> <span class="help-tooltip" id="cb-general-help-399">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r38c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r38c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r38c1_1.svg">
NW_022474217 (335692-414245): Venustampulla echinocandica strain BP 5553 chro... (10% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r38c1_2.svg">
NC_054698 (709776-786331): Aspergillus flavus NRRL3357 chromosome 8, complete... (9% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r38c1_3.svg">
NC_036442 (659368-735911): Aspergillus oryzae RIB40 DNA, chromosome 8 (9% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r38c1_4.svg">
NW_011942150 (718356-830095): Metarhizium robertsii ARSEF 23 MAA Scf 10, whol... (16% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r38c1_5.svg">
NC_010628 (4166428-4350413): Nostoc punctiforme PCC 73102, complete sequence (3% of genes show similarity), NRP-metallophore,NRPS,RRE-containing,T1PKS,lanthipeptide-class-v,spliceotide,transAT-PKS-like
</option>

<option value = "svg/clusterblast_r38c1_6.svg">
NZ_CP012673 (4679675-4765076): Sorangium cellulosum strain So ce26 chromosome... (8% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r38c1_7.svg">
NW_024467832 (4333687-4396473): Aspergillus wentii DTO 134E9 unplaced genomic... (10% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r38c1_8.svg">
NW_024467861 (3117513-3226103): Aspergillus sydowii CBS 593.65 unplaced genom... (6% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r38c1_9.svg">
NZ_LT629701 (594552-694367): Allokutzneria albata strain DSM 44149 chromosome I (14% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r38c1_10.svg">
NZ_CP095376 (670089-825635): Amycolatopsis pretoriensis strain FBCC-B4732 chr... (15% of genes show similarity), NRPS,T1PKS
</option>

</select>
<button id= "clusterblast-r38c1-download">Download graphic</button>
</div>
<div id="clusterblast-r38c1-svg"></div>

</div>
</div>

<div class="body-details-section r38c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-400"></div> <span class="help-tooltip" id="cb-known-help-400">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r38c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r38c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r38c1_1.svg">
ACT-Toxin II
</option>

</select>
<button id="knownclusterblast-r38c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r38c1-svg"></div>

</div>
</div>

<div class="body-details-section r38c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-401"></div> <span class="help-tooltip" id="cb-sub-help-401">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r38c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-404"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-404">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r38c1-domain-bubble-svg" data-filename="scaffold_38_r38c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r38c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (144861 - 199842): neighbouring NRPS-T1PKS
</option>



<option value="CC2">
Candidate 2 (144861 - 188761): single NRPS
</option>



<option value="CC3">
Candidate 3 (150950 - 199842): single T1PKS
</option>


</select>

<div class="details-svg" id="r38c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r38c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r38c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-406"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-406">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r38c1-pfam-details-svg" data-filename="scaffold_38_r38c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r38c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r38c1-nrps_pks_products" data-name="r38c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r38c1-nrps_pks_monomers" data-name="r38c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r38c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-402"></div> <span class="help-tooltip" id="nrps-structures-help-402">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 144861 - 199842:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r38c1-1" class="smiles-canvas" data-smiles="NC([*])C(=O)C([*])C(-O)"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X) + (pk)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r38c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)C([*])C(-O)"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 2, location 144861 - 188761:
<div class="collapser collapser-target-CC2 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r38c1-2" class="smiles-canvas" data-smiles="NC([*])C(=O)C([*])C(-O)"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X) + (pk)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r38c1-2-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)C([*])C(-O)"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 3, location 150950 - 199842:
<div class="collapser collapser-target-CC3 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r38c1-3" class="smiles-canvas" data-smiles="NC([*])C(=O)C([*])C(-O)"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X) + (pk)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r38c1-3-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)C([*])C(-O)"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r38c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-403"></div> <span class="help-tooltip" id="nrps-monomers-help-403">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">FUN_006991</span></strong>: X
<div class="collapser collapser-target-FUN_006991 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (243..642)</strong>: X
<div class="collapser collapser-target-FUN_006991 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr
<div class="collapser collapser-target-FUN_006991-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">AGMFCGTAVK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Ile or Val <span class="serif">DGLFVGIAVK</span> (47% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>60% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

<strong><span class="serif">FUN_006993</span></strong>: pk
<div class="collapser collapser-target-FUN_006993 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>PKS_AT (360..672)</strong>: pk
<div class="collapser collapser-target-FUN_006993 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-FUN_006993-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 79.2%</dd>
<dd>Methylmalonyl-CoA: 75.0%</dd>
<dd>Ethylmalonyl-CoA: 66.7%</dd>
</dl>

</div><br>

minowa:  Methoxymalonyl-CoA
<div class="collapser collapser-target-FUN_006993-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methoxymalonyl-CoA: 88.9</dt>

<dd></dd><dt>Methylmalonyl-CoA: 88.6</dt>

<dd></dd><dt>Malonyl-CoA: 83.3</dt>

<dd></dd><dt>Isobutyryl-CoA: 61.1</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 60.4</dt>

<dd></dd><dt>Propionyl-CoA: 49.6</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 34.8</dt>

<dd></dd><dt>fatty_acid: 27.4</dt>

<dd></dd><dt>trans-1,2-CPDA: 27.3</dt>

<dd></dd><dt>Benzoyl-CoA: 24.0</dt>

<dd></dd><dt>CHC-CoA: 20.4</dt>

<dd></dd><dt>inactive: 11.9</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 8.9</dt>

<dd></dd><dt>Acetyl-CoA: 8.5</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>












</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r38c2' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_38 - Region 2 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-813"></div> <span class="help-tooltip" id="region_svg-help-813">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_38.region002.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r38c2-svg" data-filename="scaffold_38_r38c2.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 202,623 - 248,768 nt. (total: 46,146 nt)

<a class="cluster-rules-header" id="r38c2-rules-header" href="#r38c2">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r38c2-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r38c2-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-814"></div> <span class="help-tooltip" id="focus-panel-help-814">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r38c2">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r38c2-gene-table-details" data-name="r38c2-gene-table-details">
Gene overview
</div>



<div class="body-details-header r38c2-nrps_pks" data-name="r38c2-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r38c2-MIBiG-cluster-compare" data-name="r38c2-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r38c2-clusterblast" data-name="r38c2-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r38c2-knownclusterblast" data-name="r38c2-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r38c2-subclusterblast" data-name="r38c2-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r38c2-nrps-pks-bubbles" data-name="r38c2-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r38c2-pfam-details" data-name="r38c2-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r38c2-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-415"></div> <span class="help-tooltip" id="gt-help-tooltip-help-415">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r38c2">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r38c2-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r38c2-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r38c2">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r38c2-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-407"></div> <span class="help-tooltip" id="nrps-domain-help-407">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r38c2-details-svg" data-filename="scaffold_38_r38c2_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r38c2-details-svg"></div>
</div>
</div>



<div class="body-details-section r38c2-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-408"></div> <span class="help-tooltip" id="MIBiG-help-408">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r38c2-selector">Analysis type:</label>
<select id="comparison-MIBiG-r38c2-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r38c2"></div>
<div id="comparison-MIBiG-r38c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002670: 0-27523">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002670/index.html#r1c1" target="_new">BGC0002670</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.47, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.32</td>
<td>Other</td>

<td>fosfonochlorin</td>


<td>Fusarium oxysporum f. sp. lycopersici 4287</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001219: 0-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001219/index.html#r1c1" target="_new">BGC0001219</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.28</td>
<td>Polyketide</td>

<td>shanorellin</td>


<td>Chaetomium globosum CBS 148.51</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000044: 0-21257">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000044/index.html#r1c1" target="_new">BGC0000044</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.45, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.26</td>
<td>Polyketide</td>

<td>dawenol</td>


<td>Stigmatella aurantiaca DW4/3-1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001911: 591-10299">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001911/index.html#r1c1" target="_new">BGC0001911</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.47, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.26</td>
<td>Terpene</td>

<td>raimonol</td>


<td>Streptomyces anulatus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001910: 237-11923">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001910/index.html#r1c1" target="_new">BGC0001910</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.25</td>
<td>Terpene</td>

<td>cyslabdan</td>


<td>Streptomyces cyslabdanicus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001267: 3-35442">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001267/index.html#r1c1" target="_new">BGC0001267</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.25</td>
<td>Terpene</td>

<td>lysergic acid, elymoclavine</td>


<td>Claviceps fusiformis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000677: 120-4940">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000677/index.html#r1c1" target="_new">BGC0000677</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.24</td>
<td>Terpene</td>

<td>cyclooctatin</td>


<td>Streptomyces melanosporofaciens</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000894: 395-43446">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000894/index.html#r1c1" target="_new">BGC0000894</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>Other</td>

<td>citrinin</td>


<td>Monascus aurantiacus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002391: 62090-71258">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002391/index.html#r1c1" target="_new">BGC0002391</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Terpene</td>

<td>zealexin B2, zealexin C2</td>


<td>Zea mays</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002515: 0-30087">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002515/index.html#r1c1" target="_new">BGC0002515</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.49, order:N/A, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Polyketide</td>

<td>solanapyrone A</td>


<td>Ascochyta rabiei</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r38c2-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002670: 0-27523">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002670/index.html#r1c1" target="_new">BGC0002670</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.47, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Other</td>
<td>fosfonochlorin</td>
<td>Fusarium oxysporum f. sp. lycopersici 4287</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001219: 0-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001219/index.html#r1c1" target="_new">BGC0001219</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>Polyketide</td>
<td>shanorellin</td>
<td>Chaetomium globosum CBS 148.51</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000044: 0-21257">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000044/index.html#r1c1" target="_new">BGC0000044</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.45, order:N/A, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Polyketide</td>
<td>dawenol</td>
<td>Stigmatella aurantiaca DW4/3-1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001911: 591-10299">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001911/index.html#r1c1" target="_new">BGC0001911</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.47, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Terpene</td>
<td>raimonol</td>
<td>Streptomyces anulatus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001910: 237-11923">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001910/index.html#r1c1" target="_new">BGC0001910</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Terpene</td>
<td>cyslabdan</td>
<td>Streptomyces cyslabdanicus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001267: 3-35442">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001267/index.html#r1c1" target="_new">BGC0001267</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Terpene</td>
<td>lysergic acid, elymoclavine</td>
<td>Claviceps fusiformis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000677: 120-4940">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000677/index.html#r1c1" target="_new">BGC0000677</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Terpene</td>
<td>cyclooctatin</td>
<td>Streptomyces melanosporofaciens</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000894: 395-43446">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000894/index.html#r1c1" target="_new">BGC0000894</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Other</td>
<td>citrinin</td>
<td>Monascus aurantiacus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002391: 62090-71258">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002391/index.html#r1c1" target="_new">BGC0002391</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Terpene</td>
<td>zealexin B2, zealexin C2</td>
<td>Zea mays</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002515: 0-30087">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002515/index.html#r1c1" target="_new">BGC0002515</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.49, order:N/A, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Polyketide</td>
<td>solanapyrone A</td>
<td>Ascochyta rabiei</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r38c2-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-409"></div> <span class="help-tooltip" id="cb-general-help-409">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r38c2-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r38c2_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r38c2_1.svg">
NW_025091264 (5340181-5432359): Aspergillus udagawae IFM 46973 DNA, scaffold ... (21% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r38c2_2.svg">
NW_025953252 (370558-489779): Annulohypoxylon maeteangense strain CBS 123835 ... (10% of genes show similarity), T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r38c2_3.svg">
NW_003052500 (3318567-3442588): Uncinocarpus reesii 1704 scaffold 1 genomic s... (10% of genes show similarity), NRP-metallophore,NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r38c2_4.svg">
NC_030957 (3250063-3360781): Colletotrichum higginsianum IMI 349063 chromosom... (8% of genes show similarity), T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r38c2_5.svg">
NW_024468940 (1557228-1624623): Cucurbitaria berberidis CBS 394.84 unplaced g... (13% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r38c2_6.svg">
NC_072355 (9859-77137): Talaromyces marneffei chromosome 8, complete sequence (23% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r38c2_7.svg">
NW_020290668 (56150-133269): Aspergillus saccharolyticus JOP 1030-1 unplaced ... (19% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r38c2_8.svg">
NC_054852 (412974-524906): Aspergillus luchuensis IFO 4308 DNA, chromosome 4,... (7% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r38c2_9.svg">
NZ_BMVD01000001 (447983-529599): Streptomyces galbus strain JCM 4639 sequence... (5% of genes show similarity), T1PKS,T3PKS
</option>

<option value = "svg/clusterblast_r38c2_10.svg">
NZ_KQ948550 (597710-678881): Streptomyces longwoodensis strain DSM 41677 PRJN... (5% of genes show similarity), T1PKS,T3PKS
</option>

</select>
<button id= "clusterblast-r38c2-download">Download graphic</button>
</div>
<div id="clusterblast-r38c2-svg"></div>

</div>
</div>

<div class="body-details-section r38c2-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-410"></div> <span class="help-tooltip" id="cb-known-help-410">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r38c2-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r38c2_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r38c2_1.svg">
patulin
</option>

</select>
<button id="knownclusterblast-r38c2-download">Download graphic</button>
</div>
<div id="knownclusterblast-r38c2-svg"></div>

</div>
</div>

<div class="body-details-section r38c2-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-411"></div> <span class="help-tooltip" id="cb-sub-help-411">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r38c2-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-414"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-414">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r38c2-domain-bubble-svg" data-filename="scaffold_38_r38c2_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r38c2-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC4">
Candidate 4 (202622 - 248768): single T1PKS
</option>


</select>

<div class="details-svg" id="r38c2-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r38c2-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r38c2-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-416"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-416">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r38c2-pfam-details-svg" data-filename="scaffold_38_r38c2_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r38c2-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r38c2-nrps_pks_products" data-name="r38c2-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r38c2-nrps_pks_monomers" data-name="r38c2-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r38c2-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-412"></div> <span class="help-tooltip" id="nrps-structures-help-412">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 4, location 202622 - 248768:
<div class="collapser collapser-target-CC4 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r38c2-4" class="smiles-canvas" data-smiles="C([*])C(-O)"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(pk)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r38c2-4-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r38c2-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-413"></div> <span class="help-tooltip" id="nrps-monomers-help-413">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">FUN_007010</span></strong>: pk
<div class="collapser collapser-target-FUN_007010 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>PKS_AT (218..531)</strong>: pk
<div class="collapser collapser-target-FUN_007010 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-FUN_007010-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 79.2%</dd>
<dd>Methylmalonyl-CoA: 62.5%</dd>
<dd>Isobutyryl-CoA: 58.3%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-FUN_007010-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 63.2</dt>

<dd></dd><dt>Malonyl-CoA: 59.3</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 56.7</dt>

<dd></dd><dt>Benzoyl-CoA: 35.9</dt>

<dd></dd><dt>Isobutyryl-CoA: 30.3</dt>

<dd></dd><dt>inactive: 28.2</dt>

<dd></dd><dt>Propionyl-CoA: 26.1</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 21.2</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 19.6</dt>

<dd></dd><dt>Acetyl-CoA: 9.4</dt>

<dd></dd><dt>fatty_acid: 8.9</dt>

<dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

<dd></dd><dt>CHC-CoA: 0.0</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>










</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r38c3' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_38 - Region 3 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-815"></div> <span class="help-tooltip" id="region_svg-help-815">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_38.region003.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r38c3-svg" data-filename="scaffold_38_r38c3.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 297,407 - 343,178 nt. (total: 45,772 nt)

<a class="cluster-rules-header" id="r38c3-rules-header" href="#r38c3">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r38c3-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r38c3-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-816"></div> <span class="help-tooltip" id="focus-panel-help-816">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r38c3">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r38c3-gene-table-details" data-name="r38c3-gene-table-details">
Gene overview
</div>



<div class="body-details-header r38c3-nrps_pks" data-name="r38c3-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r38c3-MIBiG-cluster-compare" data-name="r38c3-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r38c3-clusterblast" data-name="r38c3-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r38c3-knownclusterblast" data-name="r38c3-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r38c3-subclusterblast" data-name="r38c3-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r38c3-nrps-pks-bubbles" data-name="r38c3-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r38c3-pfam-details" data-name="r38c3-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r38c3-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-425"></div> <span class="help-tooltip" id="gt-help-tooltip-help-425">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r38c3">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r38c3-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r38c3-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r38c3">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r38c3-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-417"></div> <span class="help-tooltip" id="nrps-domain-help-417">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r38c3-details-svg" data-filename="scaffold_38_r38c3_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r38c3-details-svg"></div>
</div>
</div>



<div class="body-details-section r38c3-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-418"></div> <span class="help-tooltip" id="MIBiG-help-418">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r38c3-selector">Analysis type:</label>
<select id="comparison-MIBiG-r38c3-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r38c3"></div>
<div id="comparison-MIBiG-r38c3-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001276: 1101-6575">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001276/index.html#r1c1" target="_new">BGC0001276</a></td>





<td class="cc-heat-cell" title="0.70 (id:0.49, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>


<td>0.37</td>
<td>Polyketide</td>

<td>6-methylsalicyclic acid</td>


<td>Aspergillus terreus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002610: 0-6468">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002610/index.html#r1c1" target="_new">BGC0002610</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.32</td>
<td>Terpene</td>

<td>stellatic acid</td>


<td>Aspergillus stellatus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002721: 182-542486">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002721/index.html#r1c1" target="_new">BGC0002721</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.32</td>
<td>Saccharide</td>

<td>-hydroxynitrile glucoside, -hydroxynitrile glucoside, -hydroxynitrile glucoside</td>


<td>Hordeum vulgare subsp. vulgare</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002181: 1003-18265">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002181/index.html#r1c1" target="_new">BGC0002181</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.31</td>
<td>Terpene</td>

<td>pleuromutilin</td>


<td>Rhodocybe pseudopiperita</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000677: 120-4940">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000677/index.html#r1c1" target="_new">BGC0000677</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.31</td>
<td>Terpene</td>

<td>cyclooctatin</td>


<td>Streptomyces melanosporofaciens</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000631: 1094-28776">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000631/index.html#r1c1" target="_new">BGC0000631</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.28</td>
<td>Terpene</td>

<td>botcinin H, botcinin I, botcinin J, cinbotolide D, botcinin K</td>


<td>Botrytis cinerea B05.10</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002216: 305-18235">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002216/index.html#r1c1" target="_new">BGC0002216</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.27</td>
<td>Terpene</td>

<td>HEx-tc1 terpene</td>


<td>Trichoderma virens Gv29-8</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001659: 0-16557">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001659/index.html#r1c1" target="_new">BGC0001659</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.27</td>
<td>Terpene</td>

<td>mangicol A</td>


<td>Fusarium equiseti</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001314: 1711-56297">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001314/index.html#r1c1" target="_new">BGC0001314</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.41, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.27</td>
<td>Terpene</td>

<td>tirucalla</td>


<td>Arabidopsis thaliana</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002221: 391-40725">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002221/index.html#r1c1" target="_new">BGC0002221</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.41, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.26</td>
<td>Terpene</td>

<td>ent-kauren-16alpha-ol</td>


<td>Serpula lacrymans var. lacrymans S7.9</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r38c3-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001276: 1101-6575">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001276/index.html#r1c1" target="_new">BGC0001276</a></td>


<td class="cc-heat-cell" title="0.70 (id:0.49, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
<td>0.70</td>
<td>Polyketide</td>
<td>6-methylsalicyclic acid</td>
<td>Aspergillus terreus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002610: 0-6468">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002610/index.html#r1c1" target="_new">BGC0002610</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Terpene</td>
<td>stellatic acid</td>
<td>Aspergillus stellatus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002721: 182-542486">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002721/index.html#r1c1" target="_new">BGC0002721</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Saccharide</td>
<td>-hydroxynitrile glucoside, -hydroxynitrile glucoside, -hydroxynitrile glucoside</td>
<td>Hordeum vulgare subsp. vulgare</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002181: 1003-18265">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002181/index.html#r1c1" target="_new">BGC0002181</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Terpene</td>
<td>pleuromutilin</td>
<td>Rhodocybe pseudopiperita</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000677: 120-4940">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000677/index.html#r1c1" target="_new">BGC0000677</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>Terpene</td>
<td>cyclooctatin</td>
<td>Streptomyces melanosporofaciens</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000631: 1094-28776">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000631/index.html#r1c1" target="_new">BGC0000631</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>Terpene</td>
<td>botcinin H, botcinin I, botcinin J, cinbotolide D, botcinin K</td>
<td>Botrytis cinerea B05.10</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002216: 305-18235">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002216/index.html#r1c1" target="_new">BGC0002216</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>Terpene</td>
<td>HEx-tc1 terpene</td>
<td>Trichoderma virens Gv29-8</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001659: 0-16557">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001659/index.html#r1c1" target="_new">BGC0001659</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>Terpene</td>
<td>mangicol A</td>
<td>Fusarium equiseti</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001314: 1711-56297">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001314/index.html#r1c1" target="_new">BGC0001314</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.41, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>Terpene</td>
<td>tirucalla</td>
<td>Arabidopsis thaliana</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002221: 391-40725">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002221/index.html#r1c1" target="_new">BGC0002221</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.41, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>Terpene</td>
<td>ent-kauren-16alpha-ol</td>
<td>Serpula lacrymans var. lacrymans S7.9</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r38c3-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-419"></div> <span class="help-tooltip" id="cb-general-help-419">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r38c3-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r38c3_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r38c3_1.svg">
NW_017253373 (134337-244017): Paraphaeosphaeria sporulosa strain AP3s5-JAC2a ... (22% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r38c3_2.svg">
NW_024467833 (1832924-1898297): Aspergillus wentii DTO 134E9 unplaced genomic... (20% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r38c3_3.svg">
NW_024468979 (223239-288504): Aspergillus fijiensis CBS 313.89 unplaced genom... (21% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r38c3_4.svg">
NW_024067594 (118838-184232): Penicillium roqueforti strain LCP96 04111 chrom... (22% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r38c3_5.svg">
NW_022985439 (53353-118812): Mytilinidion resinicola strain CBS 304.34 unplac... (14% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r38c3_6.svg">
NW_024467837 (2291366-2393881): Aspergillus wentii DTO 134E9 unplaced genomic... (7% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r38c3_7.svg">
NZ_CP012159 (7479051-7589890): Chondromyces crocatus strain Cm c5 chromosome,... (25% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r38c3_8.svg">
NZ_CP012752 (5672926-5843995): Kibdelosporangium phytohabitans strain KLBMP11... (4% of genes show similarity), NRPS-like,T1PKS,T3PKS
</option>

<option value = "svg/clusterblast_r38c3_9.svg">
NZ_CP036534 (10490-225803): Streptomyces sp. VN1 chromosome, complete genome (5% of genes show similarity), NRPS,NRPS-like,T1PKS,T3PKS,oligosaccharide,other
</option>

<option value = "svg/clusterblast_r38c3_10.svg">
NZ_CP007155 (1540569-1652929): Kutzneria albida DSM 43870 chromosome, complet... (9% of genes show similarity), NRPS,T1PKS
</option>

</select>
<button id= "clusterblast-r38c3-download">Download graphic</button>
</div>
<div id="clusterblast-r38c3-svg"></div>

</div>
</div>

<div class="body-details-section r38c3-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-420"></div> <span class="help-tooltip" id="cb-known-help-420">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r38c3-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r38c3_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r38c3_1.svg">
4-oxomacrophorin A/macrophorin A/5&#39;-epimacrophorin B
</option>

<option value = "svg/knownclusterblast_r38c3_2.svg">
yanuthone D
</option>

<option value = "svg/knownclusterblast_r38c3_3.svg">
patulin
</option>

<option value = "svg/knownclusterblast_r38c3_4.svg">
6-MSA/aculin A
</option>

<option value = "svg/knownclusterblast_r38c3_5.svg">
terreic acid
</option>

<option value = "svg/knownclusterblast_r38c3_6.svg">
6-methylsalicyclic acid
</option>

</select>
<button id="knownclusterblast-r38c3-download">Download graphic</button>
</div>
<div id="knownclusterblast-r38c3-svg"></div>

</div>
</div>

<div class="body-details-section r38c3-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-421"></div> <span class="help-tooltip" id="cb-sub-help-421">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r38c3-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-424"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-424">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r38c3-domain-bubble-svg" data-filename="scaffold_38_r38c3_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r38c3-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC5">
Candidate 5 (297406 - 343178): single T1PKS
</option>


</select>

<div class="details-svg" id="r38c3-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r38c3-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r38c3-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-426"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-426">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r38c3-pfam-details-svg" data-filename="scaffold_38_r38c3_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r38c3-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r38c3-nrps_pks_products" data-name="r38c3-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r38c3-nrps_pks_monomers" data-name="r38c3-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r38c3-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-422"></div> <span class="help-tooltip" id="nrps-structures-help-422">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 5, location 297406 - 343178:
<div class="collapser collapser-target-CC5 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r38c3-5" class="smiles-canvas" data-smiles="C([*])C(-O)"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(pk)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r38c3-5-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r38c3-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-423"></div> <span class="help-tooltip" id="nrps-monomers-help-423">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">FUN_007047</span></strong>: pk
<div class="collapser collapser-target-FUN_007047 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (602..898)</strong>: pk
<div class="collapser collapser-target-FUN_007047 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-FUN_007047-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 79.2%</dd>
<dd>Methylmalonyl-CoA: 70.8%</dd>
<dd>Ethylmalonyl-CoA: 66.7%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-FUN_007047-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 99.7</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 75.4</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 69.6</dt>

<dd></dd><dt>Malonyl-CoA: 60.8</dt>

<dd></dd><dt>Isobutyryl-CoA: 53.0</dt>

<dd></dd><dt>trans-1,2-CPDA: 44.5</dt>

<dd></dd><dt>Propionyl-CoA: 37.5</dt>

<dd></dd><dt>Acetyl-CoA: 35.1</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 31.1</dt>

<dd></dd><dt>fatty_acid: 26.8</dt>

<dd></dd><dt>Benzoyl-CoA: 21.6</dt>

<dd></dd><dt>CHC-CoA: 18.8</dt>

<dd></dd><dt>inactive: 10.4</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>








</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>





<div class="page" id='r39c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_39 - Region 1 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-817"></div> <span class="help-tooltip" id="region_svg-help-817">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_39.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r39c1-svg" data-filename="scaffold_39_r39c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 103,098 - 151,905 nt. (total: 48,808 nt)

<a class="cluster-rules-header" id="r39c1-rules-header" href="#r39c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r39c1-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r39c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-818"></div> <span class="help-tooltip" id="focus-panel-help-818">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r39c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r39c1-gene-table-details" data-name="r39c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r39c1-nrps_pks" data-name="r39c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r39c1-MIBiG-cluster-compare" data-name="r39c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r39c1-clusterblast" data-name="r39c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r39c1-knownclusterblast" data-name="r39c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r39c1-subclusterblast" data-name="r39c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r39c1-nrps-pks-bubbles" data-name="r39c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r39c1-pfam-details" data-name="r39c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r39c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-435"></div> <span class="help-tooltip" id="gt-help-tooltip-help-435">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r39c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r39c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r39c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r39c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r39c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-427"></div> <span class="help-tooltip" id="nrps-domain-help-427">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r39c1-details-svg" data-filename="scaffold_39_r39c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r39c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r39c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-428"></div> <span class="help-tooltip" id="MIBiG-help-428">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r39c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r39c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r39c1"></div>
<div id="comparison-MIBiG-r39c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002191: 260-8785">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002191/index.html#r1c1" target="_new">BGC0002191</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.32</td>
<td>Polyketide</td>

<td>prolipyrone B, gibepyrone D</td>


<td>Fusarium graminearum PH-1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002155: 78-16511">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002155/index.html#r1c1" target="_new">BGC0002155</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.43, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.30</td>
<td>Polyketide</td>

<td>nectriapyrone C, nectriapyrone D, nectriapyrone</td>


<td>Pyricularia oryzae 70-15</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001252: 0-7958">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001252/index.html#r1c1" target="_new">BGC0001252</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.42, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.30</td>
<td>Polyketide</td>

<td>UNII-YC2Q1O94PT</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002525: 1000-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002525/index.html#r1c1" target="_new">BGC0002525</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.43, order:N/A, components:0.82)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.26</td>
<td>Polyketide</td>

<td>fusarubin, 1233A, 1233B, NG-391, lucilactaene</td>


<td>Fusarium sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002222: 888-32276">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002222/index.html#r1c1" target="_new">BGC0002222</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.25</td>
<td>Polyketide</td>

<td>zopfiellin</td>


<td>Diffractella curvata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002515: 0-30087">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002515/index.html#r1c1" target="_new">BGC0002515</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.42, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>Polyketide</td>

<td>solanapyrone A</td>


<td>Ascochyta rabiei</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002224: 685-37748">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002224/index.html#r1c1" target="_new">BGC0002224</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>Polyketide</td>

<td>zopfiellin</td>


<td>Diffractella curvata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001606: 0-22544">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001606/index.html#r1c1" target="_new">BGC0001606</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.42, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>Polyketide</td>

<td>gibepyrone-A</td>


<td>Fusarium fujikuroi IMI 58289</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002194: 147-15389">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002194/index.html#r1c1" target="_new">BGC0002194</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>Polyketide</td>

<td>epipyrone A</td>


<td>Epicoccum nigrum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000146: 754-20637">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000146/index.html#r1c1" target="_new">BGC0000146</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Polyketide</td>

<td>solanapyrone D</td>


<td>Alternaria solani</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r39c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002191: 260-8785">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002191/index.html#r1c1" target="_new">BGC0002191</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Polyketide</td>
<td>prolipyrone B, gibepyrone D</td>
<td>Fusarium graminearum PH-1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002155: 78-16511">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002155/index.html#r1c1" target="_new">BGC0002155</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.43, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>Polyketide</td>
<td>nectriapyrone C, nectriapyrone D, nectriapyrone</td>
<td>Pyricularia oryzae 70-15</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001252: 0-7958">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001252/index.html#r1c1" target="_new">BGC0001252</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.42, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>Polyketide</td>
<td>UNII-YC2Q1O94PT</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002525: 1000-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002525/index.html#r1c1" target="_new">BGC0002525</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.43, order:N/A, components:0.82)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Polyketide</td>
<td>fusarubin, 1233A, 1233B, NG-391, lucilactaene</td>
<td>Fusarium sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002222: 888-32276">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002222/index.html#r1c1" target="_new">BGC0002222</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Polyketide</td>
<td>zopfiellin</td>
<td>Diffractella curvata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002515: 0-30087">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002515/index.html#r1c1" target="_new">BGC0002515</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.42, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Polyketide</td>
<td>solanapyrone A</td>
<td>Ascochyta rabiei</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002224: 685-37748">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002224/index.html#r1c1" target="_new">BGC0002224</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Polyketide</td>
<td>zopfiellin</td>
<td>Diffractella curvata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001606: 0-22544">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001606/index.html#r1c1" target="_new">BGC0001606</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.42, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Polyketide</td>
<td>gibepyrone-A</td>
<td>Fusarium fujikuroi IMI 58289</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002194: 147-15389">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002194/index.html#r1c1" target="_new">BGC0002194</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Polyketide</td>
<td>epipyrone A</td>
<td>Epicoccum nigrum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000146: 754-20637">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000146/index.html#r1c1" target="_new">BGC0000146</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Polyketide</td>
<td>solanapyrone D</td>
<td>Alternaria solani</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r39c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-429"></div> <span class="help-tooltip" id="cb-general-help-429">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r39c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r39c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r39c1_1.svg">
NW_026250022 (2861670-2932341): Hypoxylon trugodes strain CBS 135444 unplaced... (10% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r39c1_2.svg">
NW_022983481 (741577-810039): Aaosphaeria arxii CBS 175.79 unplaced genomic s... (9% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r39c1_3.svg">
NW_025953042 (491783-560136): Daldinia caldariorum strain CBS 122874 unplaced... (10% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r39c1_4.svg">
NW_025408670 (875820-944244): Hirsutella rhossiliensis strain HR02 chromosome... (11% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r39c1_5.svg">
NC_054849 (1247525-1383127): Aspergillus luchuensis IFO 4308 DNA, chromosome ... (3% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r39c1_6.svg">
NW_001820818 (162340-230599): Sclerotinia sclerotiorum 1980 UF-70 scaffold 18... (8% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r39c1_7.svg">
NW_020291592 (339371-529932): Aspergillus piperis CBS 112811 unplaced genomic... (2% of genes show similarity), NRPS-like,T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r39c1_8.svg">
NW_023336287 (400215-591232): Aspergillus tubingensis WU-2223L DNA, scaffold ... (2% of genes show similarity), NRPS-like,T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r39c1_9.svg">
NW_020292051 (280608-471430): Aspergillus vadensis CBS 113365 unplaced genomi... (2% of genes show similarity), NRPS-like,T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r39c1_10.svg">
NW_020290890 (450315-638141): Aspergillus niger CBS 101883 unplaced genomic s... (2% of genes show similarity), NRPS-like,T1PKS,fungal-RiPP-like
</option>

</select>
<button id= "clusterblast-r39c1-download">Download graphic</button>
</div>
<div id="clusterblast-r39c1-svg"></div>

</div>
</div>

<div class="body-details-section r39c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-430"></div> <span class="help-tooltip" id="cb-known-help-430">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r39c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-431"></div> <span class="help-tooltip" id="cb-sub-help-431">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r39c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-434"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-434">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r39c1-domain-bubble-svg" data-filename="scaffold_39_r39c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r39c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (103097 - 151905): single T1PKS
</option>


</select>

<div class="details-svg" id="r39c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r39c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r39c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-436"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-436">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r39c1-pfam-details-svg" data-filename="scaffold_39_r39c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r39c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r39c1-nrps_pks_products" data-name="r39c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r39c1-nrps_pks_monomers" data-name="r39c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r39c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-432"></div> <span class="help-tooltip" id="nrps-structures-help-432">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 103097 - 151905:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r39c1-1" class="smiles-canvas" data-smiles="C([*])C(-O)(C)"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Me-pk)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-pk" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-pk*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r39c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)(C)"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r39c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-433"></div> <span class="help-tooltip" id="nrps-monomers-help-433">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">FUN_007075</span></strong>: pk
<div class="collapser collapser-target-FUN_007075 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (458..782)</strong>: pk
<div class="collapser collapser-target-FUN_007075 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-FUN_007075-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 75.0%</dd>
<dd>Methylmalonyl-CoA: 62.5%</dd>
<dd>Ethylmalonyl-CoA: 58.3%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-FUN_007075-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 69.9</dt>

<dd></dd><dt>Malonyl-CoA: 62.5</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 61.2</dt>

<dd></dd><dt>Isobutyryl-CoA: 53.7</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 32.3</dt>

<dd></dd><dt>Propionyl-CoA: 27.6</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 27.6</dt>

<dd></dd><dt>fatty_acid: 23.6</dt>

<dd></dd><dt>trans-1,2-CPDA: 20.5</dt>

<dd></dd><dt>CHC-CoA: 18.3</dt>

<dd></dd><dt>Benzoyl-CoA: 16.5</dt>

<dd></dd><dt>inactive: 10.2</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 7.1</dt>

<dd></dd><dt>Acetyl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>












</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r39c2' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_39 - Region 2 - fungal-RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-819"></div> <span class="help-tooltip" id="region_svg-help-819">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_39.region002.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r39c2-svg" data-filename="scaffold_39_r39c2.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 184,852 - 245,900 nt. (total: 61,049 nt)

<a class="cluster-rules-header" id="r39c2-rules-header" href="#r39c2">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r39c2-rules">fungal-RiPP-like: DUF3328</div>

<div class="region-svg-container">
<div id='r39c2-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-820"></div> <span class="help-tooltip" id="focus-panel-help-820">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r39c2">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r39c2-gene-table-details" data-name="r39c2-gene-table-details">
Gene overview
</div>



<div class="body-details-header r39c2-MIBiG-cluster-compare" data-name="r39c2-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r39c2-clusterblast" data-name="r39c2-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r39c2-knownclusterblast" data-name="r39c2-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r39c2-subclusterblast" data-name="r39c2-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r39c2-pfam-details" data-name="r39c2-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r39c2-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-441"></div> <span class="help-tooltip" id="gt-help-tooltip-help-441">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r39c2">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r39c2-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r39c2-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r39c2">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r39c2-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-437"></div> <span class="help-tooltip" id="MIBiG-help-437">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r39c2-selector">Analysis type:</label>
<select id="comparison-MIBiG-r39c2-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r39c2"></div>
<div id="comparison-MIBiG-r39c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">fungal-RiPP-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002185: 100-28222">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002185/index.html#r1c1" target="_new">BGC0002185</a></td>





<td class="cc-heat-cell" title="0.34 (id:0.43, order:N/A, components:0.27)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>


<td>0.09</td>
<td>Polyketide</td>

<td>gregatin A</td>


<td>Penicillium sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002255: 803-34333">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002255/index.html#r1c1" target="_new">BGC0002255</a></td>





<td class="cc-heat-cell" title="0.34 (id:0.45, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>


<td>0.09</td>
<td>Polyketide</td>

<td>lijiquinone</td>


<td>Ascomycota sp. F53</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001398: 4999-64171">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001398/index.html#r1c1" target="_new">BGC0001398</a></td>





<td class="cc-heat-cell" title="0.32 (id:0.45, order:N/A, components:0.23)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.32), rgb(0, 0, 0, 0.32) 32%, white 32%)"></td>


<td>0.09</td>
<td>RiPP</td>

<td>phomopsin A, phomopsin B, phomopsin E</td>


<td>Phomopsis leptostromiformis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002267: 0-50173">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002267/index.html#r1c1" target="_new">BGC0002267</a></td>





<td class="cc-heat-cell" title="0.32 (id:0.46, order:N/A, components:0.22)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.32), rgb(0, 0, 0, 0.32) 32%, white 32%)"></td>


<td>0.08</td>
<td>Polyketide</td>

<td>azasperpyranone A, azasperpyranone B, azasperpyranone C, azasperpyranone D, azasperpyranone E, azasperpyranone F, azasperpyranone G, azasperpyranone H</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002729: 0-27318">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002729/index.html#r1c1" target="_new">BGC0002729</a></td>





<td class="cc-heat-cell" title="0.32 (id:0.45, order:N/A, components:0.22)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.32), rgb(0, 0, 0, 0.32) 32%, white 32%)"></td>


<td>0.08</td>
<td>Polyketide</td>

<td>sordarial</td>


<td>Neurospora crassa OR74A</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002234: 0-20938">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002234/index.html#r1c1" target="_new">BGC0002234</a></td>





<td class="cc-heat-cell" title="0.29 (id:0.45, order:N/A, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>


<td>0.07</td>
<td>Polyketide</td>

<td>flavoglaucin</td>


<td>Aspergillus ruber CBS 135680</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002000: 0-12502">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002000/index.html#r1c1" target="_new">BGC0002000</a></td>





<td class="cc-heat-cell" title="0.27 (id:0.43, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>


<td>0.07</td>
<td>Polyketide</td>

<td>setomimycin</td>


<td>Streptomyces aurantiacus JA 4570</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001712: 230-41063">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001712/index.html#r1c1" target="_new">BGC0001712</a></td>





<td class="cc-heat-cell" title="0.27 (id:0.43, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>


<td>0.06</td>
<td>Other</td>

<td>ochrindole A</td>


<td>Aspergillus steynii IBT 23096</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002233: 556-31132">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002233/index.html#r1c1" target="_new">BGC0002233</a></td>





<td class="cc-heat-cell" title="0.26 (id:0.45, order:N/A, components:0.15)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>


<td>0.06</td>
<td>Polyketide</td>

<td>trichoxide</td>


<td>Trichoderma virens Gv29-8</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001377: 819-44027">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001377/index.html#r1c1" target="_new">BGC0001377</a></td>





<td class="cc-heat-cell" title="0.25 (id:0.45, order:N/A, components:0.14)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>


<td>0.06</td>
<td>Polyketide</td>

<td>uncialamycin</td>


<td>Streptomyces uncialis</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r39c2-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002185: 100-28222">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002185/index.html#r1c1" target="_new">BGC0002185</a></td>


<td class="cc-heat-cell" title="0.34 (id:0.43, order:N/A, components:0.27)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>
<td>0.34</td>
<td>Polyketide</td>
<td>gregatin A</td>
<td>Penicillium sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002255: 803-34333">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002255/index.html#r1c1" target="_new">BGC0002255</a></td>


<td class="cc-heat-cell" title="0.34 (id:0.45, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>
<td>0.34</td>
<td>Polyketide</td>
<td>lijiquinone</td>
<td>Ascomycota sp. F53</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001398: 4999-64171">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001398/index.html#r1c1" target="_new">BGC0001398</a></td>


<td class="cc-heat-cell" title="0.32 (id:0.45, order:N/A, components:0.23)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.32), rgb(0, 0, 0, 0.32) 32%, white 32%)"></td>
<td>0.32</td>
<td>RiPP</td>
<td>phomopsin A, phomopsin B, phomopsin E</td>
<td>Phomopsis leptostromiformis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002267: 0-50173">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002267/index.html#r1c1" target="_new">BGC0002267</a></td>


<td class="cc-heat-cell" title="0.32 (id:0.46, order:N/A, components:0.22)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.32), rgb(0, 0, 0, 0.32) 32%, white 32%)"></td>
<td>0.32</td>
<td>Polyketide</td>
<td>azasperpyranone A, azasperpyranone B, azasperpyranone C, azasperpyranone D, azasperpyranone E, azasperpyranone F, azasperpyranone G, azasperpyranone H</td>
<td>Aspergillus terreus NIH2624</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002729: 0-27318">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002729/index.html#r1c1" target="_new">BGC0002729</a></td>


<td class="cc-heat-cell" title="0.32 (id:0.45, order:N/A, components:0.22)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.32), rgb(0, 0, 0, 0.32) 32%, white 32%)"></td>
<td>0.32</td>
<td>Polyketide</td>
<td>sordarial</td>
<td>Neurospora crassa OR74A</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002234: 0-20938">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002234/index.html#r1c1" target="_new">BGC0002234</a></td>


<td class="cc-heat-cell" title="0.29 (id:0.45, order:N/A, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>
<td>0.29</td>
<td>Polyketide</td>
<td>flavoglaucin</td>
<td>Aspergillus ruber CBS 135680</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002000: 0-12502">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002000/index.html#r1c1" target="_new">BGC0002000</a></td>


<td class="cc-heat-cell" title="0.27 (id:0.43, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>
<td>0.27</td>
<td>Polyketide</td>
<td>setomimycin</td>
<td>Streptomyces aurantiacus JA 4570</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001712: 230-41063">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001712/index.html#r1c1" target="_new">BGC0001712</a></td>


<td class="cc-heat-cell" title="0.27 (id:0.43, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>
<td>0.27</td>
<td>Other</td>
<td>ochrindole A</td>
<td>Aspergillus steynii IBT 23096</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002233: 556-31132">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002233/index.html#r1c1" target="_new">BGC0002233</a></td>


<td class="cc-heat-cell" title="0.26 (id:0.45, order:N/A, components:0.15)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>
<td>0.26</td>
<td>Polyketide</td>
<td>trichoxide</td>
<td>Trichoderma virens Gv29-8</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001377: 819-44027">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001377/index.html#r1c1" target="_new">BGC0001377</a></td>


<td class="cc-heat-cell" title="0.25 (id:0.45, order:N/A, components:0.14)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>
<td>0.25</td>
<td>Polyketide</td>
<td>uncialamycin</td>
<td>Streptomyces uncialis</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r39c2-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-438"></div> <span class="help-tooltip" id="cb-general-help-438">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r39c2-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r39c2_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r39c2_1.svg">
NW_006271977 (184210-275183): Cordyceps militaris CM01 unplaced genomic scaff... (10% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r39c2_2.svg">
NW_020292074 (140873-244304): Aspergillus vadensis CBS 113365 unplaced genomi... (4% of genes show similarity), NRPS-like,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r39c2_3.svg">
NC_054854 (3761874-3865306): Aspergillus luchuensis IFO 4308 DNA, chromosome ... (4% of genes show similarity), NRPS-like,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r39c2_4.svg">
NW_020291596 (1375571-1479003): Aspergillus piperis CBS 112811 unplaced genom... (4% of genes show similarity), NRPS-like,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r39c2_5.svg">
NW_020291840 (131998-235388): Aspergillus costaricaensis CBS 115574 unplaced ... (4% of genes show similarity), NRPS-like,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r39c2_6.svg">
NW_020290910 (174371-277754): Aspergillus niger CBS 101883 unplaced genomic s... (4% of genes show similarity), NRPS-like,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r39c2_7.svg">
NW_023336277 (142329-245745): Aspergillus tubingensis WU-2223L DNA, scaffold ... (4% of genes show similarity), NRPS-like,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r39c2_8.svg">
NW_022985556 (1051519-1142556): Pseudovirgaria hyperparasitica strain CBS 121... (5% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r39c2_9.svg">
NW_025953810 (264034-384248): Annulohypoxylon truncatum strain CBS 140777 unp... (4% of genes show similarity), NRPS-like,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r39c2_10.svg">
NC_054857 (688182-779289): Aspergillus puulaauensis MK2 DNA, chromosome 1, ne... (4% of genes show similarity), fungal-RiPP-like
</option>

</select>
<button id= "clusterblast-r39c2-download">Download graphic</button>
</div>
<div id="clusterblast-r39c2-svg"></div>

</div>
</div>

<div class="body-details-section r39c2-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-439"></div> <span class="help-tooltip" id="cb-known-help-439">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r39c2-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-440"></div> <span class="help-tooltip" id="cb-sub-help-440">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r39c2-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-442"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-442">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r39c2-pfam-details-svg" data-filename="scaffold_39_r39c2_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r39c2-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">











</div>











</div>

</div>

</div>





<div class="page" id='r42c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_42 - Region 1 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-821"></div> <span class="help-tooltip" id="region_svg-help-821">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_42.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r42c1-svg" data-filename="scaffold_42_r42c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 96,505 - 142,187 nt. (total: 45,683 nt)

<a class="cluster-rules-header" id="r42c1-rules-header" href="#r42c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r42c1-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r42c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-822"></div> <span class="help-tooltip" id="focus-panel-help-822">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r42c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r42c1-gene-table-details" data-name="r42c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r42c1-nrps_pks" data-name="r42c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r42c1-MIBiG-cluster-compare" data-name="r42c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r42c1-clusterblast" data-name="r42c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r42c1-knownclusterblast" data-name="r42c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r42c1-subclusterblast" data-name="r42c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r42c1-nrps-pks-bubbles" data-name="r42c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r42c1-pfam-details" data-name="r42c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r42c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-451"></div> <span class="help-tooltip" id="gt-help-tooltip-help-451">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r42c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r42c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r42c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r42c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r42c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-443"></div> <span class="help-tooltip" id="nrps-domain-help-443">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r42c1-details-svg" data-filename="scaffold_42_r42c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r42c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r42c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-444"></div> <span class="help-tooltip" id="MIBiG-help-444">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r42c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r42c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r42c1"></div>
<div id="comparison-MIBiG-r42c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001244: 0-5358">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001244/index.html#r1c1" target="_new">BGC0001244</a></td>





<td class="cc-heat-cell" title="0.72 (id:0.51, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>


<td>0.40</td>
<td>Polyketide</td>

<td>(-)-Mellein</td>


<td>Parastagonospora nodorum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001275: 1063-6614">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001275/index.html#r1c1" target="_new">BGC0001275</a></td>





<td class="cc-heat-cell" title="0.70 (id:0.49, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>


<td>0.37</td>
<td>Polyketide</td>

<td>6-methylsalicyclic acid</td>


<td>Glarea lozoyensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001273: 0-5301">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001273/index.html#r1c1" target="_new">BGC0001273</a></td>





<td class="cc-heat-cell" title="0.69 (id:0.48, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>


<td>0.37</td>
<td>Polyketide</td>

<td>asperlactone</td>


<td>Aspergillus ochraceus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002229: 510-22108">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002229/index.html#r1c1" target="_new">BGC0002229</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.50, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.31</td>
<td>Polyketide</td>

<td>6-MSA, aculin A</td>


<td>Aspergillus aculeatus ATCC 16872</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000170: 0-23998">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000170/index.html#r1c1" target="_new">BGC0000170</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.48, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.25</td>
<td>Polyketide</td>

<td>yanuthone D</td>


<td>Aspergillus niger ATCC 1015</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001276: 1101-6575">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001276/index.html#r1c1" target="_new">BGC0001276</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.50, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.25</td>
<td>Polyketide</td>

<td>6-methylsalicyclic acid</td>


<td>Aspergillus terreus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001068: 0-35628">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001068/index.html#r1c1" target="_new">BGC0001068</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>Terpene, Polyketide</td>

<td>pyripyropene A</td>


<td>unidentified unclassified sequences.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002615: 973-23784">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002615/index.html#r1c1" target="_new">BGC0002615</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.47, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>Terpene</td>

<td>4-oxomacrophorin A, macrophorin A, 5&#39;-epimacrophorin B</td>


<td>Penicillium terrestre</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001954: 0-19896">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001954/index.html#r1c1" target="_new">BGC0001954</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.43, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>NRP, Polyketide</td>

<td>wortmanamide A, wortmanamide B</td>


<td>Talaromyces wortmannii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000996: 0-16575">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000996/index.html#r1c1" target="_new">BGC0000996</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>NRP, Polyketide</td>

<td>frontalamide B, frontalamide A</td>


<td>Streptomyces sp. SPB78</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r42c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001244: 0-5358">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001244/index.html#r1c1" target="_new">BGC0001244</a></td>


<td class="cc-heat-cell" title="0.72 (id:0.51, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
<td>0.72</td>
<td>Polyketide</td>
<td>(-)-Mellein</td>
<td>Parastagonospora nodorum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001275: 1063-6614">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001275/index.html#r1c1" target="_new">BGC0001275</a></td>


<td class="cc-heat-cell" title="0.70 (id:0.49, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
<td>0.70</td>
<td>Polyketide</td>
<td>6-methylsalicyclic acid</td>
<td>Glarea lozoyensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001273: 0-5301">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001273/index.html#r1c1" target="_new">BGC0001273</a></td>


<td class="cc-heat-cell" title="0.69 (id:0.48, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
<td>0.69</td>
<td>Polyketide</td>
<td>asperlactone</td>
<td>Aspergillus ochraceus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002229: 510-22108">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002229/index.html#r1c1" target="_new">BGC0002229</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.50, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Polyketide</td>
<td>6-MSA, aculin A</td>
<td>Aspergillus aculeatus ATCC 16872</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000170: 0-23998">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000170/index.html#r1c1" target="_new">BGC0000170</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.48, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Polyketide</td>
<td>yanuthone D</td>
<td>Aspergillus niger ATCC 1015</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001276: 1101-6575">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001276/index.html#r1c1" target="_new">BGC0001276</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.50, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Polyketide</td>
<td>6-methylsalicyclic acid</td>
<td>Aspergillus terreus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001068: 0-35628">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001068/index.html#r1c1" target="_new">BGC0001068</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Terpene, Polyketide</td>
<td>pyripyropene A</td>
<td>unidentified unclassified sequences.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002615: 973-23784">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002615/index.html#r1c1" target="_new">BGC0002615</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.47, order:N/A, components:0.69)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Terpene</td>
<td>4-oxomacrophorin A, macrophorin A, 5&#39;-epimacrophorin B</td>
<td>Penicillium terrestre</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001954: 0-19896">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001954/index.html#r1c1" target="_new">BGC0001954</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.43, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>NRP, Polyketide</td>
<td>wortmanamide A, wortmanamide B</td>
<td>Talaromyces wortmannii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000996: 0-16575">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000996/index.html#r1c1" target="_new">BGC0000996</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>NRP, Polyketide</td>
<td>frontalamide B, frontalamide A</td>
<td>Streptomyces sp. SPB78</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r42c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-445"></div> <span class="help-tooltip" id="cb-general-help-445">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r42c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r42c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r42c1_1.svg">
NC_010572 (7079071-7191428): Streptomyces griseus subsp. griseus NBRC 13350, ... (11% of genes show similarity), T1PKS,terpene
</option>

<option value = "svg/clusterblast_r42c1_2.svg">
NZ_GL877172 (7313304-7425731): Streptomyces sp. ACT-1 strain XylebKG-1 scaffo... (8% of genes show similarity), T1PKS,terpene
</option>

<option value = "svg/clusterblast_r42c1_3.svg">
NZ_CP080332 (3952770-4019717): Mycobacterium diernhoferi strain ATCC 19340 ch... (4% of genes show similarity), NRP-metallophore,NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r42c1_4.svg">
NC_021658 (3622043-3722843): Sorangium cellulosum So0157-2, complete sequence (8% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r42c1_5.svg">
NZ_CP060404 (7794487-7878091): Streptomyces buecherae strain AC541 chromosome... (8% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r42c1_6.svg">
NZ_CP054929 (770264-853845): Streptomyces buecherae strain NA00687 chromosome... (8% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r42c1_7.svg">
NZ_AP023396 (3284353-3510319): Nocardia wallacei strain FMUON74 chromosome, c... (2% of genes show similarity), NAPAA,NRPS,T1PKS,redox-cofactor
</option>

<option value = "svg/clusterblast_r42c1_8.svg">
NZ_CP114200 (7489569-7686997): Streptomyces sp. 71268 chromosome (2% of genes show similarity), PKS-like,T1PKS,terpene,transAT-PKS
</option>

<option value = "svg/clusterblast_r42c1_9.svg">
NZ_JABEPP010000006 (292534-361228): Enterovirga sp. DB1703 contig6, whole gen... (3% of genes show similarity), NAPAA,T1PKS
</option>

<option value = "svg/clusterblast_r42c1_10.svg">
NC_022785 (2500403-2707075): Streptomyces rapamycinicus NRRL 5491, complete s... (2% of genes show similarity), NRPS,NRPS-like,T1PKS
</option>

</select>
<button id= "clusterblast-r42c1-download">Download graphic</button>
</div>
<div id="clusterblast-r42c1-svg"></div>

</div>
</div>

<div class="body-details-section r42c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-446"></div> <span class="help-tooltip" id="cb-known-help-446">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r42c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r42c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r42c1_1.svg">
(-)-Mellein
</option>

<option value = "svg/knownclusterblast_r42c1_2.svg">
6-methylsalicyclic acid
</option>

<option value = "svg/knownclusterblast_r42c1_3.svg">
6-methylsalicyclic acid
</option>

</select>
<button id="knownclusterblast-r42c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r42c1-svg"></div>

</div>
</div>

<div class="body-details-section r42c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-447"></div> <span class="help-tooltip" id="cb-sub-help-447">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r42c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-450"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-450">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r42c1-domain-bubble-svg" data-filename="scaffold_42_r42c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r42c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (96504 - 142187): single T1PKS
</option>


</select>

<div class="details-svg" id="r42c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r42c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r42c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-452"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-452">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r42c1-pfam-details-svg" data-filename="scaffold_42_r42c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r42c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r42c1-nrps_pks_products" data-name="r42c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r42c1-nrps_pks_monomers" data-name="r42c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r42c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-448"></div> <span class="help-tooltip" id="nrps-structures-help-448">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 96504 - 142187:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r42c1-1" class="smiles-canvas" data-smiles="C([*])C(-O)"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(pk)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r42c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r42c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-449"></div> <span class="help-tooltip" id="nrps-monomers-help-449">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">FUN_007330</span></strong>: pk
<div class="collapser collapser-target-FUN_007330 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (588..880)</strong>: pk
<div class="collapser collapser-target-FUN_007330 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-FUN_007330-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 75.0%</dd>
<dd>Methylmalonyl-CoA: 62.5%</dd>
<dd>CE-Malonyl-CoA: 58.3%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-FUN_007330-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 79.5</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 55.4</dt>

<dd></dd><dt>Malonyl-CoA: 55.2</dt>

<dd></dd><dt>Isobutyryl-CoA: 54.2</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 54.2</dt>

<dd></dd><dt>Acetyl-CoA: 22.7</dt>

<dd></dd><dt>fatty_acid: 21.0</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 20.7</dt>

<dd></dd><dt>Benzoyl-CoA: 19.8</dt>

<dd></dd><dt>Propionyl-CoA: 13.3</dt>

<dd></dd><dt>inactive: 11.6</dt>

<dd></dd><dt>trans-1,2-CPDA: 10.7</dt>

<dd></dd><dt>CHC-CoA: 9.1</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 5.9</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>








</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>





<div class="page" id='r43c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_43 - Region 1 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-823"></div> <span class="help-tooltip" id="region_svg-help-823">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_43.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r43c1-svg" data-filename="scaffold_43_r43c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 46,690 - 67,862 nt. (total: 21,173 nt)

<a class="cluster-rules-header" id="r43c1-rules-header" href="#r43c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r43c1-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>

<div class="region-svg-container">
<div id='r43c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-824"></div> <span class="help-tooltip" id="focus-panel-help-824">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r43c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r43c1-gene-table-details" data-name="r43c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r43c1-MIBiG-cluster-compare" data-name="r43c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r43c1-clusterblast" data-name="r43c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r43c1-knownclusterblast" data-name="r43c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r43c1-subclusterblast" data-name="r43c1-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r43c1-pfam-details" data-name="r43c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r43c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-457"></div> <span class="help-tooltip" id="gt-help-tooltip-help-457">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r43c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r43c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r43c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r43c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r43c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-453"></div> <span class="help-tooltip" id="MIBiG-help-453">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r43c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r43c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r43c1"></div>
<div id="comparison-MIBiG-r43c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">terpene</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002176: 0-1199">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002176/index.html#r1c1" target="_new">BGC0002176</a></td>





<td class="cc-heat-cell" title="0.68 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>


<td>0.35</td>
<td>Terpene</td>

<td>culmorin, (+)-juniperol(longiborneol), 15-acetyldeoxynivalenol</td>


<td>Fusarium graminearum PH-1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002391: 62090-71258">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002391/index.html#r1c1" target="_new">BGC0002391</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.24</td>
<td>Terpene</td>

<td>zealexin B2, zealexin C2</td>


<td>Zea mays</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000687: 3134-24057">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000687/index.html#r1c1" target="_new">BGC0000687</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.41, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Terpene</td>

<td>phyllocladan-16-ol</td>


<td>Diaporthe amygdali</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001969: 0-7810">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001969/index.html#r1c1" target="_new">BGC0001969</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Terpene</td>

<td>asperterpenoid A</td>


<td>Talaromyces wortmannii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002670: 0-27523">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002670/index.html#r1c1" target="_new">BGC0002670</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Other</td>

<td>fosfonochlorin</td>


<td>Fusarium oxysporum f. sp. lycopersici 4287</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001893: 155-19434">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001893/index.html#r1c1" target="_new">BGC0001893</a></td>





<td class="cc-heat-cell" title="0.51 (id:0.45, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>


<td>0.19</td>
<td>Other</td>

<td>abscisic acid</td>


<td>Botrytis cinerea B05.10</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000677: 120-4940">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000677/index.html#r1c1" target="_new">BGC0000677</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>Terpene</td>

<td>cyclooctatin</td>


<td>Streptomyces melanosporofaciens</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000676: 621-14838">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000676/index.html#r1c1" target="_new">BGC0000676</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.44, order:N/A, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.17</td>
<td>Terpene</td>

<td>aphidicolin, aphidicolan-16-ol, 3-deoxyaphidicolin, 17-deoxyaphidicolin</td>


<td>Phoma betae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002245: 219-11090">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002245/index.html#r1c1" target="_new">BGC0002245</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.16</td>
<td>Terpene</td>

<td>terpestacin, preterpestacin 3, preterpestacin 2, preterpestacin 1</td>


<td>Bipolaris maydis C5</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001262: 1986-31560">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001262/index.html#r1c1" target="_new">BGC0001262</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.16</td>
<td>Polyketide</td>

<td>AK-toxin I</td>


<td>Alternaria alternata</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r43c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002176: 0-1199">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002176/index.html#r1c1" target="_new">BGC0002176</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>Terpene</td>
<td>culmorin, (+)-juniperol(longiborneol), 15-acetyldeoxynivalenol</td>
<td>Fusarium graminearum PH-1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002391: 62090-71258">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002391/index.html#r1c1" target="_new">BGC0002391</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Terpene</td>
<td>zealexin B2, zealexin C2</td>
<td>Zea mays</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000687: 3134-24057">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000687/index.html#r1c1" target="_new">BGC0000687</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.41, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Terpene</td>
<td>phyllocladan-16-ol</td>
<td>Diaporthe amygdali</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001969: 0-7810">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001969/index.html#r1c1" target="_new">BGC0001969</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.46, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Terpene</td>
<td>asperterpenoid A</td>
<td>Talaromyces wortmannii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002670: 0-27523">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002670/index.html#r1c1" target="_new">BGC0002670</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Other</td>
<td>fosfonochlorin</td>
<td>Fusarium oxysporum f. sp. lycopersici 4287</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001893: 155-19434">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001893/index.html#r1c1" target="_new">BGC0001893</a></td>


<td class="cc-heat-cell" title="0.51 (id:0.45, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
<td>0.51</td>
<td>Other</td>
<td>abscisic acid</td>
<td>Botrytis cinerea B05.10</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000677: 120-4940">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000677/index.html#r1c1" target="_new">BGC0000677</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Terpene</td>
<td>cyclooctatin</td>
<td>Streptomyces melanosporofaciens</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000676: 621-14838">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000676/index.html#r1c1" target="_new">BGC0000676</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.44, order:N/A, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>Terpene</td>
<td>aphidicolin, aphidicolan-16-ol, 3-deoxyaphidicolin, 17-deoxyaphidicolin</td>
<td>Phoma betae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002245: 219-11090">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002245/index.html#r1c1" target="_new">BGC0002245</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>Terpene</td>
<td>terpestacin, preterpestacin 3, preterpestacin 2, preterpestacin 1</td>
<td>Bipolaris maydis C5</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001262: 1986-31560">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001262/index.html#r1c1" target="_new">BGC0001262</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>Polyketide</td>
<td>AK-toxin I</td>
<td>Alternaria alternata</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r43c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-454"></div> <span class="help-tooltip" id="cb-general-help-454">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r43c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r43c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r43c1_1.svg">
NW_022985017 (1154412-1187947): Dothidotthia symphoricarpi CBS 119687 unplace... (12% of genes show similarity), terpene
</option>

</select>
<button id= "clusterblast-r43c1-download">Download graphic</button>
</div>
<div id="clusterblast-r43c1-svg"></div>

</div>
</div>

<div class="body-details-section r43c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-455"></div> <span class="help-tooltip" id="cb-known-help-455">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r43c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-456"></div> <span class="help-tooltip" id="cb-sub-help-456">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r43c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-458"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-458">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r43c1-pfam-details-svg" data-filename="scaffold_43_r43c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r43c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">









</div>









</div>

</div>

</div>





<div class="page" id='r44c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_44 - Region 1 - NRPS-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-825"></div> <span class="help-tooltip" id="region_svg-help-825">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_44.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r44c1-svg" data-filename="scaffold_44_r44c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 41,663 - 84,928 nt. (total: 43,266 nt)

<a class="cluster-rules-header" id="r44c1-rules-header" href="#r44c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r44c1-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r44c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-826"></div> <span class="help-tooltip" id="focus-panel-help-826">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r44c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r44c1-gene-table-details" data-name="r44c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r44c1-nrps_pks" data-name="r44c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r44c1-MIBiG-cluster-compare" data-name="r44c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r44c1-clusterblast" data-name="r44c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r44c1-knownclusterblast" data-name="r44c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r44c1-subclusterblast" data-name="r44c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r44c1-nrps-pks-bubbles" data-name="r44c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r44c1-pfam-details" data-name="r44c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r44c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-467"></div> <span class="help-tooltip" id="gt-help-tooltip-help-467">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r44c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r44c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r44c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r44c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r44c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-459"></div> <span class="help-tooltip" id="nrps-domain-help-459">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r44c1-details-svg" data-filename="scaffold_44_r44c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r44c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r44c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-460"></div> <span class="help-tooltip" id="MIBiG-help-460">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r44c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r44c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r44c1"></div>
<div id="comparison-MIBiG-r44c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002152: 347-13298">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002152/index.html#r1c1" target="_new">BGC0002152</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.49, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>Other</td>

<td>toxoflavin</td>


<td>Pseudomonas protegens Pf-5</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000927: 1-2450">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000927/index.html#r1c1" target="_new">BGC0000927</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.49, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>Other (Flavin)</td>

<td>roseoflavin</td>


<td>Streptomyces davaonensis JCM 4913</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000929: 296-7215">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000929/index.html#r1c1" target="_new">BGC0000929</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.49, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.17</td>
<td>Other</td>

<td>xanthothricin</td>


<td>Burkholderia glumae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001959: 1219-11580">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001959/index.html#r1c1" target="_new">BGC0001959</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.49, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.17</td>
<td>Other</td>

<td>taylorflavin A, taylorflavin B</td>


<td>Streptomyces hiroshimensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001972: 267-32147">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001972/index.html#r1c1" target="_new">BGC0001972</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.49, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.17</td>
<td>Other</td>

<td>toxoflavin, fervenulin</td>


<td>Streptomyces hiroshimensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000662: 0-14906">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000662/index.html#r1c1" target="_new">BGC0000662</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.43, order:N/A, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>Terpene</td>

<td>grixazone A</td>


<td>Streptomyces griseus subsp. griseus NBRC 13350</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001122: 0-43663">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001122/index.html#r1c1" target="_new">BGC0001122</a></td>





<td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>


<td>0.14</td>
<td>NRP, Polyketide</td>

<td>dihydroisoflavipucine, isoflavipucine</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002212: 0-24132">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002212/index.html#r1c1" target="_new">BGC0002212</a></td>





<td class="cc-heat-cell" title="0.43 (id:0.41, order:N/A, components:0.45)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>


<td>0.14</td>
<td>Polyketide</td>

<td>HEx-pks15 polyketide</td>


<td>Moniliophthora roreri MCA 2997</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002268: 0-26854">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002268/index.html#r1c1" target="_new">BGC0002268</a></td>





<td class="cc-heat-cell" title="0.38 (id:0.44, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>


<td>0.11</td>
<td>Polyketide, NRP</td>

<td>CJ-16,173, CJ-15,696, citridone B, citridone A, citridone B&#39;</td>


<td>Penicillium sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002197: 1638-28433">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002197/index.html#r1c1" target="_new">BGC0002197</a></td>





<td class="cc-heat-cell" title="0.38 (id:0.44, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>


<td>0.11</td>
<td>Polyketide, NRP</td>

<td>terrestric acid</td>


<td>Penicillium crustosum</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r44c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002152: 347-13298">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002152/index.html#r1c1" target="_new">BGC0002152</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.49, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>Other</td>
<td>toxoflavin</td>
<td>Pseudomonas protegens Pf-5</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000927: 1-2450">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000927/index.html#r1c1" target="_new">BGC0000927</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.49, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>Other (Flavin)</td>
<td>roseoflavin</td>
<td>Streptomyces davaonensis JCM 4913</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000929: 296-7215">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000929/index.html#r1c1" target="_new">BGC0000929</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.49, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>Other</td>
<td>xanthothricin</td>
<td>Burkholderia glumae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001959: 1219-11580">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001959/index.html#r1c1" target="_new">BGC0001959</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.49, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>Other</td>
<td>taylorflavin A, taylorflavin B</td>
<td>Streptomyces hiroshimensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001972: 267-32147">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001972/index.html#r1c1" target="_new">BGC0001972</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.49, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>Other</td>
<td>toxoflavin, fervenulin</td>
<td>Streptomyces hiroshimensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000662: 0-14906">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000662/index.html#r1c1" target="_new">BGC0000662</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.43, order:N/A, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>Terpene</td>
<td>grixazone A</td>
<td>Streptomyces griseus subsp. griseus NBRC 13350</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001122: 0-43663">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001122/index.html#r1c1" target="_new">BGC0001122</a></td>


<td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
<td>0.43</td>
<td>NRP, Polyketide</td>
<td>dihydroisoflavipucine, isoflavipucine</td>
<td>Aspergillus terreus NIH2624</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002212: 0-24132">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002212/index.html#r1c1" target="_new">BGC0002212</a></td>


<td class="cc-heat-cell" title="0.43 (id:0.41, order:N/A, components:0.45)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
<td>0.43</td>
<td>Polyketide</td>
<td>HEx-pks15 polyketide</td>
<td>Moniliophthora roreri MCA 2997</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002268: 0-26854">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002268/index.html#r1c1" target="_new">BGC0002268</a></td>


<td class="cc-heat-cell" title="0.38 (id:0.44, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
<td>0.38</td>
<td>Polyketide, NRP</td>
<td>CJ-16,173, CJ-15,696, citridone B, citridone A, citridone B&#39;</td>
<td>Penicillium sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002197: 1638-28433">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002197/index.html#r1c1" target="_new">BGC0002197</a></td>


<td class="cc-heat-cell" title="0.38 (id:0.44, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
<td>0.38</td>
<td>Polyketide, NRP</td>
<td>terrestric acid</td>
<td>Penicillium crustosum</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r44c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-461"></div> <span class="help-tooltip" id="cb-general-help-461">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r44c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r44c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r44c1_1.svg">
NW_026250021 (2041365-2122437): Hypoxylon trugodes strain CBS 135444 unplaced... (7% of genes show similarity), NRPS,NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r44c1_2.svg">
NW_022474205 (4202623-4266002): Venustampulla echinocandica strain BP 5553 ch... (10% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r44c1_3.svg">
NW_007361001 (2569111-2660065): Glarea lozoyensis ATCC 20868 chromosome Unkno... (8% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r44c1_4.svg">
NW_024468936 (2252161-2343040): Cucurbitaria berberidis CBS 394.84 unplaced g... (6% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r44c1_5.svg">
NW_026249959 (88033-178989): Alternaria arbusti strain BMP 1465 chromosome Un... (7% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r44c1_6.svg">
NW_017306195 (1107208-1198179): Alternaria alternata strain SRC1lrK2f unplace... (4% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r44c1_7.svg">
NW_025764711 (468396-559417): Emericellopsis atlantica unplaced genomic scaff... (7% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r44c1_8.svg">
NW_022985023 (941610-1032516): Dothidotthia symphoricarpi CBS 119687 unplaced... (4% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r44c1_9.svg">
NC_054851 (1237352-1302758): Aspergillus luchuensis IFO 4308 DNA, chromosome ... (11% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r44c1_10.svg">
NW_020291593 (1582776-1648184): Aspergillus piperis CBS 112811 unplaced genom... (12% of genes show similarity), T1PKS
</option>

</select>
<button id= "clusterblast-r44c1-download">Download graphic</button>
</div>
<div id="clusterblast-r44c1-svg"></div>

</div>
</div>

<div class="body-details-section r44c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-462"></div> <span class="help-tooltip" id="cb-known-help-462">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r44c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-463"></div> <span class="help-tooltip" id="cb-sub-help-463">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r44c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-466"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-466">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r44c1-domain-bubble-svg" data-filename="scaffold_44_r44c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r44c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (41662 - 84928): single NRPS-like
</option>


</select>

<div class="details-svg" id="r44c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r44c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r44c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-468"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-468">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r44c1-pfam-details-svg" data-filename="scaffold_44_r44c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r44c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r44c1-nrps_pks_products" data-name="r44c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r44c1-nrps_pks_monomers" data-name="r44c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r44c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-464"></div> <span class="help-tooltip" id="nrps-structures-help-464">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 41662 - 84928:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r44c1-1" class="smiles-canvas" data-smiles="NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r44c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r44c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-465"></div> <span class="help-tooltip" id="nrps-monomers-help-465">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">FUN_007459</span></strong>: X
<div class="collapser collapser-target-FUN_007459 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (113..372)</strong>: X
<div class="collapser collapser-target-FUN_007459 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  (unknown)
<div class="collapser collapser-target-FUN_007459-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>N/A</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">LGLLASP--K</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>

<dt>Stachelhaus code match:</dt>
<dd>0% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>





<div class="page" id='r46c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_46 - Region 1 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-827"></div> <span class="help-tooltip" id="region_svg-help-827">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_46.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r46c1-svg" data-filename="scaffold_46_r46c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 222,062 - 269,635 nt. (total: 47,574 nt)

<a class="cluster-rules-header" id="r46c1-rules-header" href="#r46c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r46c1-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r46c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-828"></div> <span class="help-tooltip" id="focus-panel-help-828">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r46c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r46c1-gene-table-details" data-name="r46c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r46c1-nrps_pks" data-name="r46c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r46c1-MIBiG-cluster-compare" data-name="r46c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r46c1-clusterblast" data-name="r46c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r46c1-knownclusterblast" data-name="r46c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r46c1-subclusterblast" data-name="r46c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r46c1-nrps-pks-bubbles" data-name="r46c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r46c1-pfam-details" data-name="r46c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r46c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-477"></div> <span class="help-tooltip" id="gt-help-tooltip-help-477">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r46c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r46c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r46c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r46c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r46c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-469"></div> <span class="help-tooltip" id="nrps-domain-help-469">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r46c1-details-svg" data-filename="scaffold_46_r46c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r46c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r46c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-470"></div> <span class="help-tooltip" id="MIBiG-help-470">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r46c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r46c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r46c1"></div>
<div id="comparison-MIBiG-r46c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001990: 130-13646">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001990/index.html#r1c1" target="_new">BGC0001990</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.32</td>
<td>Other</td>

<td>BU-4704, xanthocillin-X monomethylether, melanocin E, melanocin F, fumiformamide</td>


<td>Aspergillus fumigatus Af293</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001266: 10707-26494">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001266/index.html#r1c1" target="_new">BGC0001266</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.42, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.32</td>
<td>Polyketide</td>

<td>grayanic acid</td>


<td>Cladonia grayi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002610: 0-6468">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002610/index.html#r1c1" target="_new">BGC0002610</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.32</td>
<td>Terpene</td>

<td>stellatic acid</td>


<td>Aspergillus stellatus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000037: 0-7821">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000037/index.html#r1c1" target="_new">BGC0000037</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.46, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.32</td>
<td>Polyketide</td>

<td>cichorine</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001219: 0-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001219/index.html#r1c1" target="_new">BGC0001219</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.27</td>
<td>Polyketide</td>

<td>shanorellin</td>


<td>Chaetomium globosum CBS 148.51</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001720: 0-45622">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001720/index.html#r1c1" target="_new">BGC0001720</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.82)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.25</td>
<td>Polyketide</td>

<td>oosporein</td>


<td>Beauveria bassiana ARSEF 2860</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000677: 120-4940">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000677/index.html#r1c1" target="_new">BGC0000677</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>Terpene</td>

<td>cyclooctatin</td>


<td>Streptomyces melanosporofaciens</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002614: 1538-32674">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002614/index.html#r1c1" target="_new">BGC0002614</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.45, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.23</td>
<td>NRP, Polyketide</td>

<td>subglutinol A, sesquicillin A, colletotrichin, metarhizin A</td>


<td>Penicillium shearii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001390: 100-19645">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001390/index.html#r1c1" target="_new">BGC0001390</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.44, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>NRP, Polyketide</td>

<td>LL-Z1272beta</td>


<td>Stachybotrys bisbyi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000056: 2-6689">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000056/index.html#r1c1" target="_new">BGC0000056</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.41, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Polyketide</td>

<td>esperamicin</td>


<td>Actinomadura verrucosospora</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r46c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001990: 130-13646">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001990/index.html#r1c1" target="_new">BGC0001990</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Other</td>
<td>BU-4704, xanthocillin-X monomethylether, melanocin E, melanocin F, fumiformamide</td>
<td>Aspergillus fumigatus Af293</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001266: 10707-26494">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001266/index.html#r1c1" target="_new">BGC0001266</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.42, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Polyketide</td>
<td>grayanic acid</td>
<td>Cladonia grayi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002610: 0-6468">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002610/index.html#r1c1" target="_new">BGC0002610</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Terpene</td>
<td>stellatic acid</td>
<td>Aspergillus stellatus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000037: 0-7821">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000037/index.html#r1c1" target="_new">BGC0000037</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.46, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Polyketide</td>
<td>cichorine</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001219: 0-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001219/index.html#r1c1" target="_new">BGC0001219</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>Polyketide</td>
<td>shanorellin</td>
<td>Chaetomium globosum CBS 148.51</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001720: 0-45622">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001720/index.html#r1c1" target="_new">BGC0001720</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.82)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Polyketide</td>
<td>oosporein</td>
<td>Beauveria bassiana ARSEF 2860</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000677: 120-4940">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000677/index.html#r1c1" target="_new">BGC0000677</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Terpene</td>
<td>cyclooctatin</td>
<td>Streptomyces melanosporofaciens</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002614: 1538-32674">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002614/index.html#r1c1" target="_new">BGC0002614</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.45, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>NRP, Polyketide</td>
<td>subglutinol A, sesquicillin A, colletotrichin, metarhizin A</td>
<td>Penicillium shearii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001390: 100-19645">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001390/index.html#r1c1" target="_new">BGC0001390</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.44, order:N/A, components:0.72)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>NRP, Polyketide</td>
<td>LL-Z1272beta</td>
<td>Stachybotrys bisbyi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000056: 2-6689">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000056/index.html#r1c1" target="_new">BGC0000056</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.41, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Polyketide</td>
<td>esperamicin</td>
<td>Actinomadura verrucosospora</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r46c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-471"></div> <span class="help-tooltip" id="cb-general-help-471">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r46c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r46c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r46c1_1.svg">
NW_024468909 (49034-180513): Cryphonectria parasitica EP155 unplaced genomic ... (9% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r46c1_2.svg">
NW_025953249 (1504328-1570756): Annulohypoxylon maeteangense strain CBS 12383... (25% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r46c1_3.svg">
NW_022983479 (1031372-1097841): Aaosphaeria arxii CBS 175.79 unplaced genomic... (20% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r46c1_4.svg">
NW_025091283 (4319746-4671991): Aspergillus pseudoviridinutans IFM 55266 DNA,... (2% of genes show similarity), NRPS,NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r46c1_5.svg">
NW_022986864 (165720-231738): Westerdykella ornata strain CBS 379.55 unplaced... (11% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r46c1_6.svg">
NW_025091218 (4642-192860): Aspergillus viridinutans IFM 47045 DNA, scaffold ... (3% of genes show similarity), NRPS-like,T1PKS,indole
</option>

<option value = "svg/clusterblast_r46c1_7.svg">
NC_066257 (109084-244310): Aspergillus nidulans FGSC A4 chromosome I (6% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r46c1_8.svg">
NW_026250022 (1958174-2023619): Hypoxylon trugodes strain CBS 135444 unplaced... (9% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r46c1_9.svg">
NC_038015 (361754-425603): Fusarium venenatum strain A3/5 genome assembly, ch... (17% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r46c1_10.svg">
NW_017263588 (291451-418813): Mollisia scopiformis unplaced genomic scaffold ... (7% of genes show similarity), fungal-RiPP-like
</option>

</select>
<button id= "clusterblast-r46c1-download">Download graphic</button>
</div>
<div id="clusterblast-r46c1-svg"></div>

</div>
</div>

<div class="body-details-section r46c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-472"></div> <span class="help-tooltip" id="cb-known-help-472">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r46c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-473"></div> <span class="help-tooltip" id="cb-sub-help-473">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r46c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-476"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-476">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r46c1-domain-bubble-svg" data-filename="scaffold_46_r46c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r46c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (222061 - 269635): single T1PKS
</option>


</select>

<div class="details-svg" id="r46c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r46c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r46c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-478"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-478">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r46c1-pfam-details-svg" data-filename="scaffold_46_r46c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r46c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r46c1-nrps_pks_products" data-name="r46c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r46c1-nrps_pks_monomers" data-name="r46c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r46c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-474"></div> <span class="help-tooltip" id="nrps-structures-help-474">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 222061 - 269635:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r46c1-1" class="smiles-canvas" data-smiles="CC(=O)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(mal)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r46c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="CC(=O)C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r46c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-475"></div> <span class="help-tooltip" id="nrps-monomers-help-475">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">FUN_007656</span></strong>: mal
<div class="collapser collapser-target-FUN_007656 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>







<dd><strong>PKS_AT (910..1202)</strong>: mal
<div class="collapser collapser-target-FUN_007656 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-FUN_007656-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 54.2%</dd>
</dl>

</div><br>

minowa:  Malonyl-CoA
<div class="collapser collapser-target-FUN_007656-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Malonyl-CoA: 55.3</dt>

<dd></dd><dt>inactive: 49.0</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 38.6</dt>

<dd></dd><dt>Methylmalonyl-CoA: 24.8</dt>

<dd></dd><dt>Propionyl-CoA: 22.6</dt>

<dd></dd><dt>CHC-CoA: 20.9</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 18.4</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 16.9</dt>

<dd></dd><dt>fatty_acid: 13.4</dt>

<dd></dd><dt>trans-1,2-CPDA: 12.8</dt>

<dd></dd><dt>Benzoyl-CoA: 12.8</dt>

<dd></dd><dt>Isobutyryl-CoA: 0.0</dt>

<dd></dd><dt>Acetyl-CoA: 0.0</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>








</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>





<div class="page" id='r47c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_47 - Region 1 - fungal-RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-829"></div> <span class="help-tooltip" id="region_svg-help-829">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_47.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r47c1-svg" data-filename="scaffold_47_r47c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 144,004 - 205,042 nt. (total: 61,039 nt)

<a class="cluster-rules-header" id="r47c1-rules-header" href="#r47c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r47c1-rules">fungal-RiPP-like: DUF3328</div>

<div class="region-svg-container">
<div id='r47c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-830"></div> <span class="help-tooltip" id="focus-panel-help-830">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r47c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r47c1-gene-table-details" data-name="r47c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r47c1-MIBiG-cluster-compare" data-name="r47c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r47c1-clusterblast" data-name="r47c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r47c1-knownclusterblast" data-name="r47c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r47c1-subclusterblast" data-name="r47c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r47c1-pfam-details" data-name="r47c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r47c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-483"></div> <span class="help-tooltip" id="gt-help-tooltip-help-483">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r47c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r47c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r47c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r47c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r47c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-479"></div> <span class="help-tooltip" id="MIBiG-help-479">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r47c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r47c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r47c1"></div>
<div id="comparison-MIBiG-r47c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">fungal-RiPP-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000676: 621-14838">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000676/index.html#r1c1" target="_new">BGC0000676</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.46, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>Terpene</td>

<td>aphidicolin, aphidicolan-16-ol, 3-deoxyaphidicolin, 17-deoxyaphidicolin</td>


<td>Phoma betae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002242: 982-22850">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002242/index.html#r1c1" target="_new">BGC0002242</a></td>





<td class="cc-heat-cell" title="0.39 (id:0.46, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>


<td>0.12</td>
<td>NRP</td>

<td>oxepinamide D</td>


<td>Aspergillus ustus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002253: 0-33620">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002253/index.html#r1c1" target="_new">BGC0002253</a></td>





<td class="cc-heat-cell" title="0.39 (id:0.45, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>


<td>0.12</td>
<td>Polyketide</td>

<td>waikikiamide A, waikikiamide B, waikikiamide C, (+)-semivioxanthin, semivioxanthin</td>


<td>Aspergillus sp. FM242</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002208: 764-29749">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002208/index.html#r1c1" target="_new">BGC0002208</a></td>





<td class="cc-heat-cell" title="0.39 (id:0.45, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>


<td>0.12</td>
<td>NRP</td>

<td>oxepinamide F</td>


<td>Aspergillus ustus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002709: 890-28246">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002709/index.html#r1c1" target="_new">BGC0002709</a></td>





<td class="cc-heat-cell" title="0.39 (id:0.45, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>


<td>0.12</td>
<td>Polyketide</td>

<td>aurofusarin</td>


<td>Fusarium graminearum PH-1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001583: 3200-15837">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001583/index.html#r1c1" target="_new">BGC0001583</a></td>





<td class="cc-heat-cell" title="0.39 (id:0.45, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>


<td>0.11</td>
<td>Polyketide</td>

<td>emodin</td>


<td>Escovopsis weberi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002301: 2654-19498">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002301/index.html#r1c1" target="_new">BGC0002301</a></td>





<td class="cc-heat-cell" title="0.38 (id:0.43, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>


<td>0.11</td>
<td>Polyketide</td>

<td>(P)-ustilaginoidin A</td>


<td>Aschersonia paraphysata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001340: 1840-51432">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001340/index.html#r1c1" target="_new">BGC0001340</a></td>





<td class="cc-heat-cell" title="0.36 (id:0.59, order:N/A, components:0.22)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>


<td>0.10</td>
<td>Polyketide</td>

<td>byssochlamic acid</td>


<td>Byssochlamys fulva</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001398: 4999-64171">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001398/index.html#r1c1" target="_new">BGC0001398</a></td>





<td class="cc-heat-cell" title="0.36 (id:0.45, order:N/A, components:0.28)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>


<td>0.10</td>
<td>RiPP</td>

<td>phomopsin A, phomopsin B, phomopsin E</td>


<td>Phomopsis leptostromiformis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000420: 0-20659">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000420/index.html#r1c1" target="_new">BGC0000420</a></td>





<td class="cc-heat-cell" title="0.35 (id:0.46, order:N/A, components:0.27)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>


<td>0.10</td>
<td>NRP</td>

<td>histidyltryptophanyldiketopiperazine, dehydrohistidyltryptophanyldiketopiperazine, roquefortine D, roquefortine C, glandicoline A, glandicoline B, meleagrine</td>


<td>Penicillium rubens Wisconsin 54-1255</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r47c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000676: 621-14838">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000676/index.html#r1c1" target="_new">BGC0000676</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.46, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>Terpene</td>
<td>aphidicolin, aphidicolan-16-ol, 3-deoxyaphidicolin, 17-deoxyaphidicolin</td>
<td>Phoma betae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002242: 982-22850">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002242/index.html#r1c1" target="_new">BGC0002242</a></td>


<td class="cc-heat-cell" title="0.39 (id:0.46, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
<td>0.39</td>
<td>NRP</td>
<td>oxepinamide D</td>
<td>Aspergillus ustus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002253: 0-33620">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002253/index.html#r1c1" target="_new">BGC0002253</a></td>


<td class="cc-heat-cell" title="0.39 (id:0.45, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
<td>0.39</td>
<td>Polyketide</td>
<td>waikikiamide A, waikikiamide B, waikikiamide C, (+)-semivioxanthin, semivioxanthin</td>
<td>Aspergillus sp. FM242</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002208: 764-29749">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002208/index.html#r1c1" target="_new">BGC0002208</a></td>


<td class="cc-heat-cell" title="0.39 (id:0.45, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
<td>0.39</td>
<td>NRP</td>
<td>oxepinamide F</td>
<td>Aspergillus ustus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002709: 890-28246">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002709/index.html#r1c1" target="_new">BGC0002709</a></td>


<td class="cc-heat-cell" title="0.39 (id:0.45, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
<td>0.39</td>
<td>Polyketide</td>
<td>aurofusarin</td>
<td>Fusarium graminearum PH-1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001583: 3200-15837">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001583/index.html#r1c1" target="_new">BGC0001583</a></td>


<td class="cc-heat-cell" title="0.39 (id:0.45, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
<td>0.39</td>
<td>Polyketide</td>
<td>emodin</td>
<td>Escovopsis weberi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002301: 2654-19498">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002301/index.html#r1c1" target="_new">BGC0002301</a></td>


<td class="cc-heat-cell" title="0.38 (id:0.43, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
<td>0.38</td>
<td>Polyketide</td>
<td>(P)-ustilaginoidin A</td>
<td>Aschersonia paraphysata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001340: 1840-51432">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001340/index.html#r1c1" target="_new">BGC0001340</a></td>


<td class="cc-heat-cell" title="0.36 (id:0.59, order:N/A, components:0.22)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>
<td>0.36</td>
<td>Polyketide</td>
<td>byssochlamic acid</td>
<td>Byssochlamys fulva</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001398: 4999-64171">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001398/index.html#r1c1" target="_new">BGC0001398</a></td>


<td class="cc-heat-cell" title="0.36 (id:0.45, order:N/A, components:0.28)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>
<td>0.36</td>
<td>RiPP</td>
<td>phomopsin A, phomopsin B, phomopsin E</td>
<td>Phomopsis leptostromiformis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000420: 0-20659">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000420/index.html#r1c1" target="_new">BGC0000420</a></td>


<td class="cc-heat-cell" title="0.35 (id:0.46, order:N/A, components:0.27)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>
<td>0.35</td>
<td>NRP</td>
<td>histidyltryptophanyldiketopiperazine, dehydrohistidyltryptophanyldiketopiperazine, roquefortine D, roquefortine C, glandicoline A, glandicoline B, meleagrine</td>
<td>Penicillium rubens Wisconsin 54-1255</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r47c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-480"></div> <span class="help-tooltip" id="cb-general-help-480">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r47c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r47c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r47c1_1.svg">
NW_017253369 (697925-788637): Paraphaeosphaeria sporulosa strain AP3s5-JAC2a ... (6% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r47c1_2.svg">
NW_009799099 (2109217-2140516): Neurospora tetrasperma FGSC 2508 unplaced gen... (30% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r47c1_3.svg">
NW_022474207 (2256683-2320472): Venustampulla echinocandica strain BP 5553 ch... (8% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r47c1_4.svg">
NC_057367 (973431-1037220): Aspergillus chevalieri M1 DNA, chromosome 6, near... (8% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r47c1_5.svg">
NW_019170454 (685594-749291): Aspergillus glaucus CBS 516.65 unplaced genomic... (7% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r47c1_6.svg">
NW_024467304 (552618-615729): Aspergillus ruber CBS 135680 unplaced genomic s... (9% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r47c1_7.svg">
NW_022985457 (182988-273904): Mytilinidion resinicola strain CBS 304.34 unpla... (6% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r47c1_8.svg">
NW_022474205 (248150-404938): Venustampulla echinocandica strain BP 5553 chro... (4% of genes show similarity), fungal-RiPP-like,terpene
</option>

<option value = "svg/clusterblast_r47c1_9.svg">
NW_023336198 (200168-266544): Geosmithia morbida isolate 1262 chromosome Unkn... (11% of genes show similarity), T1PKS
</option>

</select>
<button id= "clusterblast-r47c1-download">Download graphic</button>
</div>
<div id="clusterblast-r47c1-svg"></div>

</div>
</div>

<div class="body-details-section r47c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-481"></div> <span class="help-tooltip" id="cb-known-help-481">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r47c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-482"></div> <span class="help-tooltip" id="cb-sub-help-482">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r47c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-484"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-484">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r47c1-pfam-details-svg" data-filename="scaffold_47_r47c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r47c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">











</div>











</div>

</div>

</div>





<div class="page" id='r49c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_49 - Region 1 - T1PKS,terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-831"></div> <span class="help-tooltip" id="region_svg-help-831">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_49.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r49c1-svg" data-filename="scaffold_49_r49c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 61,418 - 113,940 nt. (total: 52,523 nt)

<a class="cluster-rules-header" id="r49c1-rules-header" href="#r49c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r49c1-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)<br>T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r49c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-832"></div> <span class="help-tooltip" id="focus-panel-help-832">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r49c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r49c1-gene-table-details" data-name="r49c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r49c1-nrps_pks" data-name="r49c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r49c1-MIBiG-cluster-compare" data-name="r49c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r49c1-clusterblast" data-name="r49c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r49c1-knownclusterblast" data-name="r49c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r49c1-subclusterblast" data-name="r49c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r49c1-nrps-pks-bubbles" data-name="r49c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r49c1-pfam-details" data-name="r49c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r49c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-493"></div> <span class="help-tooltip" id="gt-help-tooltip-help-493">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r49c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r49c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r49c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r49c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r49c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-485"></div> <span class="help-tooltip" id="nrps-domain-help-485">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r49c1-details-svg" data-filename="scaffold_49_r49c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r49c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r49c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-486"></div> <span class="help-tooltip" id="MIBiG-help-486">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r49c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r49c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r49c1"></div>
<div id="comparison-MIBiG-r49c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">terpene</span></th>

<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000667: 565-7939">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000667/index.html#r1c1" target="_new">BGC0000667</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.50, order:0.50, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>






<td class="cc-heat-cell" title="0.63 (id:0.50, order:0.50, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.59</td>
<td>Terpene</td>

<td>PR-toxin</td>


<td>Penicillium roqueforti FM164</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000687: 3134-24057">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000687/index.html#r1c1" target="_new">BGC0000687</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>






<td class="cc-heat-cell" title="0.54 (id:0.46, order:0.35, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.56</td>
<td>Terpene</td>

<td>phyllocladan-16-ol</td>


<td>Diaporthe amygdali</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002245: 219-11090">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002245/index.html#r1c1" target="_new">BGC0002245</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>






<td class="cc-heat-cell" title="0.55 (id:0.44, order:0.39, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.55</td>
<td>Terpene</td>

<td>terpestacin, preterpestacin 3, preterpestacin 2, preterpestacin 1</td>


<td>Bipolaris maydis C5</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000688: 696-13661">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000688/index.html#r1c1" target="_new">BGC0000688</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>






<td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.53</td>
<td>Terpene</td>

<td>copalyl diphosphate</td>


<td>Diaporthe amygdali</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000676: 621-14838">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000676/index.html#r1c1" target="_new">BGC0000676</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>






<td class="cc-heat-cell" title="0.56 (id:0.45, order:0.48, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.52</td>
<td>Terpene</td>

<td>aphidicolin, aphidicolan-16-ol, 3-deoxyaphidicolin, 17-deoxyaphidicolin</td>


<td>Phoma betae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000812: 502-25834">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000812/index.html#r1c1" target="_new">BGC0000812</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>






<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.48</td>
<td>Alkaloid</td>

<td>huperzine A</td>


<td>Shiraia sp. slf14</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001659: 0-16557">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001659/index.html#r1c1" target="_new">BGC0001659</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.41, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>






<td class="cc-heat-cell" title="0.52 (id:0.42, order:0.40, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.42</td>
<td>Terpene</td>

<td>mangicol A</td>


<td>Fusarium equiseti</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002290: 5128-32193">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002290/index.html#r1c1" target="_new">BGC0002290</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>






<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.40</td>
<td>Other</td>

<td>biscognienyne B, eutypinic acid, compound 3, 4-hydroxy-3-prenylbenzoic acid, prenylhydroquinone, compound 15, compound 8, compound 7, compound 6, compound 11, compound 12, pestalodiol E</td>


<td>Biscogniauxia sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001882: 0-43357">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001882/index.html#r1c1" target="_new">BGC0001882</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.48)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>






<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.68)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.39</td>
<td>Polyketide</td>

<td>chrysoxanthone A, chrysoxanthone B, chrysoxanthone C</td>


<td>Penicillium rubens Wisconsin 54-1255</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002194: 147-15389">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002194/index.html#r1c1" target="_new">BGC0002194</a></td>





<td class="cc-heat-cell" title="0.40 (id:0.42, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>






<td class="cc-heat-cell" title="0.59 (id:0.43, order:0.51, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.38</td>
<td>Polyketide</td>

<td>epipyrone A</td>


<td>Epicoccum nigrum</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r49c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000631: 1094-28776">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000631/index.html#r1c1" target="_new">BGC0000631</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>Terpene</td>
<td>botcinin H, botcinin I, botcinin J, cinbotolide D, botcinin K</td>
<td>Botrytis cinerea B05.10</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001969: 0-7810">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001969/index.html#r1c1" target="_new">BGC0001969</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Terpene</td>
<td>asperterpenoid A</td>
<td>Talaromyces wortmannii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001893: 155-19434">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001893/index.html#r1c1" target="_new">BGC0001893</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Other</td>
<td>abscisic acid</td>
<td>Botrytis cinerea B05.10</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001528: 2981-16881">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001528/index.html#r1c1" target="_new">BGC0001528</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Terpene</td>
<td>betaestacin</td>
<td>Phoma betae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002162: 0-5465">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002162/index.html#r1c1" target="_new">BGC0002162</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Terpene</td>
<td>sesterfisherol, sesterfisheric acid, compound 6, compound 7a, compound 5, compound 8</td>
<td>Aspergillus fischeri NRRL 181</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002610: 0-6468">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002610/index.html#r1c1" target="_new">BGC0002610</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Terpene</td>
<td>stellatic acid</td>
<td>Aspergillus stellatus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002721: 182-542486">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002721/index.html#r1c1" target="_new">BGC0002721</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>Saccharide</td>
<td>-hydroxynitrile glucoside, -hydroxynitrile glucoside, -hydroxynitrile glucoside</td>
<td>Hordeum vulgare subsp. vulgare</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002670: 0-27523">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002670/index.html#r1c1" target="_new">BGC0002670</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.45, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>Other</td>
<td>fosfonochlorin</td>
<td>Fusarium oxysporum f. sp. lycopersici 4287</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001252: 0-7958">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001252/index.html#r1c1" target="_new">BGC0001252</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.44, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>Polyketide</td>
<td>UNII-YC2Q1O94PT</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002525: 1000-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002525/index.html#r1c1" target="_new">BGC0002525</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.44, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>Polyketide</td>
<td>fusarubin, 1233A, 1233B, NG-391, lucilactaene</td>
<td>Fusarium sp.</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r49c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-487"></div> <span class="help-tooltip" id="cb-general-help-487">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r49c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r49c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r49c1_1.svg">
NW_024468918 (1133295-1261954): Cryphonectria parasitica EP155 unplaced genom... (27% of genes show similarity), T1PKS,terpene
</option>

<option value = "svg/clusterblast_r49c1_2.svg">
NC_030957 (4062721-4136357): Colletotrichum higginsianum IMI 349063 chromosom... (27% of genes show similarity), T1PKS,terpene
</option>

<option value = "svg/clusterblast_r49c1_3.svg">
NW_025953826 (180766-254516): Annulohypoxylon truncatum strain CBS 140777 unp... (38% of genes show similarity), T1PKS,terpene
</option>

<option value = "svg/clusterblast_r49c1_4.svg">
NW_013566992 (141321-209375): Aureobasidium subglaciale EXF-2481 unplaced gen... (24% of genes show similarity), T1PKS,terpene
</option>

<option value = "svg/clusterblast_r49c1_5.svg">
NW_025091223 (845966-877338): Aspergillus viridinutans IFM 47045 DNA, scaffol... (28% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r49c1_6.svg">
NC_066262 (1187658-1218838): Aspergillus nidulans FGSC A4 chromosome VI (41% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r49c1_7.svg">
NW_024468909 (49034-180513): Cryphonectria parasitica EP155 unplaced genomic ... (11% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r49c1_8.svg">
NW_022986874 (80430-158238): Westerdykella ornata strain CBS 379.55 unplaced ... (9% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r49c1_9.svg">
NW_024468936 (1367361-1398427): Cucurbitaria berberidis CBS 394.84 unplaced g... (41% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r49c1_10.svg">
NW_024467535 (811837-893848): Aspergillus versicolor CBS 583.65 unplaced geno... (6% of genes show similarity), NRPS,T1PKS,indole
</option>

</select>
<button id= "clusterblast-r49c1-download">Download graphic</button>
</div>
<div id="clusterblast-r49c1-svg"></div>

</div>
</div>

<div class="body-details-section r49c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-488"></div> <span class="help-tooltip" id="cb-known-help-488">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r49c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-489"></div> <span class="help-tooltip" id="cb-sub-help-489">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r49c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-492"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-492">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r49c1-domain-bubble-svg" data-filename="scaffold_49_r49c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r49c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (61417 - 113940): neighbouring terpene-T1PKS
</option>



<option value="CC3">
Candidate 3 (65843 - 113940): single T1PKS
</option>


</select>

<div class="details-svg" id="r49c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r49c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r49c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-494"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-494">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r49c1-pfam-details-svg" data-filename="scaffold_49_r49c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r49c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r49c1-nrps_pks_products" data-name="r49c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r49c1-nrps_pks_monomers" data-name="r49c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r49c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-490"></div> <span class="help-tooltip" id="nrps-structures-help-490">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 61417 - 113940:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r49c1-1" class="smiles-canvas" data-smiles="C([*])C(-O)(C)"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Me-pk)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-pk" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-pk*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r49c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)(C)"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 3, location 65843 - 113940:
<div class="collapser collapser-target-CC3 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r49c1-3" class="smiles-canvas" data-smiles="C([*])C(-O)(C)"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Me-pk)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-pk" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-pk*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r49c1-3-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)(C)"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r49c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-491"></div> <span class="help-tooltip" id="nrps-monomers-help-491">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">FUN_007823</span></strong>: pk
<div class="collapser collapser-target-FUN_007823 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (538..857)</strong>: pk
<div class="collapser collapser-target-FUN_007823 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-FUN_007823-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 62.5%</dd>
<dd>Methylmalonyl-CoA: 58.3%</dd>
<dd>Ethylmalonyl-CoA: 54.2%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-FUN_007823-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 72.3</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 68.1</dt>

<dd></dd><dt>Malonyl-CoA: 50.1</dt>

<dd></dd><dt>Isobutyryl-CoA: 47.5</dt>

<dd></dd><dt>Benzoyl-CoA: 36.2</dt>

<dd></dd><dt>Propionyl-CoA: 29.6</dt>

<dd></dd><dt>trans-1,2-CPDA: 21.5</dt>

<dd></dd><dt>inactive: 15.9</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 13.3</dt>

<dd></dd><dt>fatty_acid: 13.1</dt>

<dd></dd><dt>CHC-CoA: 10.4</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 8.6</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 8.5</dt>

<dd></dd><dt>Acetyl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>












</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>





<div class="page" id='r50c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_50 - Region 1 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-833"></div> <span class="help-tooltip" id="region_svg-help-833">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_50.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r50c1-svg" data-filename="scaffold_50_r50c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 5,795 - 54,855 nt. (total: 49,061 nt)

<a class="cluster-rules-header" id="r50c1-rules-header" href="#r50c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r50c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r50c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-834"></div> <span class="help-tooltip" id="focus-panel-help-834">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r50c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r50c1-gene-table-details" data-name="r50c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r50c1-nrps_pks" data-name="r50c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r50c1-MIBiG-cluster-compare" data-name="r50c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r50c1-clusterblast" data-name="r50c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r50c1-knownclusterblast" data-name="r50c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r50c1-subclusterblast" data-name="r50c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r50c1-nrps-pks-bubbles" data-name="r50c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r50c1-pfam-details" data-name="r50c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r50c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-503"></div> <span class="help-tooltip" id="gt-help-tooltip-help-503">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r50c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r50c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r50c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r50c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r50c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-495"></div> <span class="help-tooltip" id="nrps-domain-help-495">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r50c1-details-svg" data-filename="scaffold_50_r50c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r50c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r50c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-496"></div> <span class="help-tooltip" id="MIBiG-help-496">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r50c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r50c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r50c1"></div>
<div id="comparison-MIBiG-r50c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000313: 0-33475">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000313/index.html#r1c1" target="_new">BGC0000313</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.49, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.31</td>
<td>NRP</td>

<td>beauvericin</td>


<td>Beauveria bassiana</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000342: 1020-10416">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000342/index.html#r1c1" target="_new">BGC0000342</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.49, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.30</td>
<td>NRP</td>

<td>enniatin</td>


<td>Fusarium equiseti</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002437: 0-10596">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002437/index.html#r1c1" target="_new">BGC0002437</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.29</td>
<td>NRP</td>

<td>thermoactinoamide A</td>


<td>Thermoactinomyces sp. AS95</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000307: 505-35485">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000307/index.html#r1c1" target="_new">BGC0000307</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.52, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.29</td>
<td>NRP</td>

<td>AbT1</td>


<td>Aureobasidium pullulans</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002135: 0-11508">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002135/index.html#r1c1" target="_new">BGC0002135</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.28</td>
<td>NRP</td>

<td>bovienimide A</td>


<td>Xenorhabdus bovienii SS-2004</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.27</td>
<td>NRP</td>

<td>AM-toxin</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002518: 0-10561">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002518/index.html#r1c1" target="_new">BGC0002518</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.41, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.27</td>
<td>NRP</td>

<td>syringafactin A, syringafactin C</td>


<td>Pseudomonas sp. SZ57</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833: 14964-30015">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.27</td>
<td>NRP</td>

<td>icosalide A, icosalide B</td>


<td>Burkholderia gladioli</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001166: 0-15624">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001166/index.html#r1c1" target="_new">BGC0001166</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.26</td>
<td>NRP</td>

<td>HC-toxin</td>


<td>Alternaria jesenskae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132: 0-12417">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.26</td>
<td>NRP</td>

<td>xenotetrapeptide</td>


<td>Xenorhabdus nematophila ATCC 19061</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r50c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000313: 0-33475">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000313/index.html#r1c1" target="_new">BGC0000313</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.49, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>NRP</td>
<td>beauvericin</td>
<td>Beauveria bassiana</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000342: 1020-10416">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000342/index.html#r1c1" target="_new">BGC0000342</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.49, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>NRP</td>
<td>enniatin</td>
<td>Fusarium equiseti</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002437: 0-10596">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002437/index.html#r1c1" target="_new">BGC0002437</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>NRP</td>
<td>thermoactinoamide A</td>
<td>Thermoactinomyces sp. AS95</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000307: 505-35485">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000307/index.html#r1c1" target="_new">BGC0000307</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.52, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>NRP</td>
<td>AbT1</td>
<td>Aureobasidium pullulans</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002135: 0-11508">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002135/index.html#r1c1" target="_new">BGC0002135</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>NRP</td>
<td>bovienimide A</td>
<td>Xenorhabdus bovienii SS-2004</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>NRP</td>
<td>AM-toxin</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002518: 0-10561">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002518/index.html#r1c1" target="_new">BGC0002518</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.41, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>NRP</td>
<td>syringafactin A, syringafactin C</td>
<td>Pseudomonas sp. SZ57</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833: 14964-30015">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>NRP</td>
<td>icosalide A, icosalide B</td>
<td>Burkholderia gladioli</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001166: 0-15624">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001166/index.html#r1c1" target="_new">BGC0001166</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>NRP</td>
<td>HC-toxin</td>
<td>Alternaria jesenskae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132: 0-12417">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>NRP</td>
<td>xenotetrapeptide</td>
<td>Xenorhabdus nematophila ATCC 19061</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r50c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-497"></div> <span class="help-tooltip" id="cb-general-help-497">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r50c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r50c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r50c1_1.svg">
NZ_CADEYO010000001 (676210-815905): Burkholderia gladioli strain BCC1728, who... (6% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r50c1_2.svg">
NZ_CP104214 (2150969-2290718): Burkholderia gladioli strain ZN-S4 chromosome ... (6% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r50c1_3.svg">
NZ_CP031452 (402977-542677): Burkholderia gladioli strain Lv-StA chromosome 1... (6% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r50c1_4.svg">
NZ_KN150849 (3431460-3549072): Burkholderia gladioli strain ATCC 25417 scaffo... (6% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r50c1_5.svg">
NZ_CADEXT010000003 (727559-845274): Burkholderia gladioli strain BCC1700, who... (6% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r50c1_6.svg">
NZ_CP098321 (2193973-2333767): Burkholderia gladioli strain BK04 chromosome 1... (6% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r50c1_7.svg">
NZ_CADEQR010000002 (96197-236043): Burkholderia gladioli strain BCC1732, whol... (6% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r50c1_8.svg">
NZ_CADEQD010000001 (130463-270287): Burkholderia gladioli strain BCC1730, who... (6% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r50c1_9.svg">
NZ_CP068049 (2222813-2362613): Burkholderia gladioli strain BBB-01 chromosome... (6% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r50c1_10.svg">
NZ_CADEYA010000006 (243783-383496): Burkholderia gladioli strain BCC1733, who... (6% of genes show similarity), NRPS,T1PKS
</option>

</select>
<button id= "clusterblast-r50c1-download">Download graphic</button>
</div>
<div id="clusterblast-r50c1-svg"></div>

</div>
</div>

<div class="body-details-section r50c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-498"></div> <span class="help-tooltip" id="cb-known-help-498">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r50c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r50c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r50c1_1.svg">
enniatin
</option>

<option value = "svg/knownclusterblast_r50c1_2.svg">
AbT1
</option>

</select>
<button id="knownclusterblast-r50c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r50c1-svg"></div>

</div>
</div>

<div class="body-details-section r50c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-499"></div> <span class="help-tooltip" id="cb-sub-help-499">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r50c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-502"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-502">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r50c1-domain-bubble-svg" data-filename="scaffold_50_r50c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r50c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (5794 - 54855): single NRPS
</option>


</select>

<div class="details-svg" id="r50c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r50c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r50c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-504"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-504">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r50c1-pfam-details-svg" data-filename="scaffold_50_r50c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r50c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r50c1-nrps_pks_products" data-name="r50c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r50c1-nrps_pks_monomers" data-name="r50c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r50c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-500"></div> <span class="help-tooltip" id="nrps-structures-help-500">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 5794 - 54855:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r50c1-1" class="smiles-canvas" data-smiles="NC([*])C(=O)NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X - X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x,x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r50c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r50c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-501"></div> <span class="help-tooltip" id="nrps-monomers-help-501">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">FUN_007877</span></strong>: X - X
<div class="collapser collapser-target-FUN_007877 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>AMP-binding (476..869)</strong>: X
<div class="collapser collapser-target-FUN_007877 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  (unknown)
<div class="collapser collapser-target-FUN_007877-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Apolar, aliphatic (Gly, Ala, Val, Leu, Ile, Abu, Ival)</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">GALLVAGSIK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>D-Hiv <span class="serif">GALMVVGSIK</span> (74% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>80% (moderate)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>







<dd><strong>AMP-binding (1548..1950)</strong>: X
<div class="collapser collapser-target-FUN_007877 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  (unknown)
<div class="collapser collapser-target-FUN_007877-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Apolar, aliphatic (Gly, Ala, Val, Leu, Ile, Abu, Ival)</dd>
<dt>Small clusters prediction:</dt>
<dd>Aliphatic, branched hydrophobic (Val, Leu, Ile, Abu, Ival)</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DAWLIGAVIK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>D-Ala <span class="serif">DATLIGAVIK</span> (59% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>90% (moderate)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>





<div class="page" id='r51c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_51 - Region 1 - T3PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-835"></div> <span class="help-tooltip" id="region_svg-help-835">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_51.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r51c1-svg" data-filename="scaffold_51_r51c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 174,050 - 215,263 nt. (total: 41,214 nt)

<a class="cluster-rules-header" id="r51c1-rules-header" href="#r51c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r51c1-rules">T3PKS: (Chal_sti_synt_C or Chal_sti_synt_N)</div>

<div class="region-svg-container">
<div id='r51c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-836"></div> <span class="help-tooltip" id="focus-panel-help-836">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r51c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r51c1-gene-table-details" data-name="r51c1-gene-table-details">
Gene overview
</div>





<div class="body-details-header r51c1-MIBiG-cluster-compare" data-name="r51c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r51c1-clusterblast" data-name="r51c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r51c1-knownclusterblast" data-name="r51c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r51c1-subclusterblast" data-name="r51c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r51c1-pfam-details" data-name="r51c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r51c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-509"></div> <span class="help-tooltip" id="gt-help-tooltip-help-509">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r51c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r51c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r51c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r51c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>





<div class="body-details-section r51c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-505"></div> <span class="help-tooltip" id="MIBiG-help-505">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r51c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r51c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r51c1"></div>
<div id="comparison-MIBiG-r51c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T3PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001962: 0-7027">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001962/index.html#r1c1" target="_new">BGC0001962</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.21</td>
<td>Polyketide</td>

<td>hierridin B, hierridin C</td>


<td>Cyanobium sp. LEGE 06113</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001077: 0-6550">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001077/index.html#r1c1" target="_new">BGC0001077</a></td>





<td class="cc-heat-cell" title="0.35 (id:0.43, order:N/A, components:0.29)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>


<td>0.10</td>
<td>Polyketide</td>

<td>alkyl-O-dihydrogeranyl-methoxyhydroquinones</td>


<td>Actinoplanes missouriensis 431</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002362: 1-24294">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002362/index.html#r1c1" target="_new">BGC0002362</a></td>





<td class="cc-heat-cell" title="0.32 (id:0.43, order:N/A, components:0.24)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.32), rgb(0, 0, 0, 0.32) 32%, white 32%)"></td>


<td>0.08</td>
<td>Polyketide</td>

<td>loseolamycin A1, loseolamycin A2</td>


<td>Micromonospora endolithica</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000872: 0-35385">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000872/index.html#r1c1" target="_new">BGC0000872</a></td>





<td class="cc-heat-cell" title="0.24 (id:0.43, order:N/A, components:0.14)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.24), rgb(0, 0, 0, 0.24) 24%, white 24%)"></td>


<td>0.06</td>
<td>Other</td>

<td>A-90289 A, A-90289 B</td>


<td>Streptomyces sp. SANK 60405</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001076: 0-29709">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001076/index.html#r1c1" target="_new">BGC0001076</a></td>





<td class="cc-heat-cell" title="0.24 (id:0.42, order:N/A, components:0.14)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.24), rgb(0, 0, 0, 0.24) 24%, white 24%)"></td>


<td>0.06</td>
<td>Other (Nucleoside)</td>

<td>liposidomycin B, liposidomycin A, liposidomycin C, liposidomycin Z, liposidomycin H, liposidomycin G, liposidomycin L, liposidomycin M, liposidomycin K, liposidomycin N</td>


<td>Streptomyces sp. SN-1061M</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001647: 0-18309">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001647/index.html#r1c1" target="_new">BGC0001647</a></td>





<td class="cc-heat-cell" title="0.22 (id:0.41, order:N/A, components:0.11)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.22), rgb(0, 0, 0, 0.22) 22%, white 22%)"></td>


<td>0.05</td>
<td>Polyketide</td>

<td>lagunapyrone A, lagunapyrone B, lagunapyrone C</td>


<td>Streptomyces sp. MP131-18</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001665: 0-19519">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001665/index.html#r1c1" target="_new">BGC0001665</a></td>





<td class="cc-heat-cell" title="0.20 (id:0.42, order:N/A, components:0.09)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.20), rgb(0, 0, 0, 0.20) 20%, white 20%)"></td>


<td>0.04</td>
<td>Polyketide</td>

<td>methylated alkyl-resorcinol, methylated acyl-phloroglucinol</td>


<td>Mycobacterium marinum M</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002290: 5128-32193">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002290/index.html#r1c1" target="_new">BGC0002290</a></td>





<td class="cc-heat-cell" title="0.20 (id:0.46, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.20), rgb(0, 0, 0, 0.20) 20%, white 20%)"></td>


<td>0.04</td>
<td>Other</td>

<td>biscognienyne B, eutypinic acid, compound 3, 4-hydroxy-3-prenylbenzoic acid, prenylhydroquinone, compound 15, compound 8, compound 7, compound 6, compound 11, compound 12, pestalodiol E</td>


<td>Biscogniauxia sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001848: 0-58198">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001848/index.html#r1c1" target="_new">BGC0001848</a></td>





<td class="cc-heat-cell" title="0.18 (id:0.41, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.18), rgb(0, 0, 0, 0.18) 18%, white 18%)"></td>


<td>0.04</td>
<td>Other</td>

<td>diazaquinomycin H, diazaquinomycin J</td>


<td>Micromonospora sp. B006</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002439: 0-24052">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002439/index.html#r1c1" target="_new">BGC0002439</a></td>





<td class="cc-heat-cell" title="0.16 (id:0.41, order:N/A, components:0.06)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.16), rgb(0, 0, 0, 0.16) 16%, white 16%)"></td>


<td>0.03</td>
<td>Other</td>

<td>diastaphenazine, izumiphenazine C</td>


<td>Streptomyces diastaticus</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r51c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001962: 0-7027">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001962/index.html#r1c1" target="_new">BGC0001962</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Polyketide</td>
<td>hierridin B, hierridin C</td>
<td>Cyanobium sp. LEGE 06113</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001077: 0-6550">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001077/index.html#r1c1" target="_new">BGC0001077</a></td>


<td class="cc-heat-cell" title="0.35 (id:0.43, order:N/A, components:0.29)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>
<td>0.35</td>
<td>Polyketide</td>
<td>alkyl-O-dihydrogeranyl-methoxyhydroquinones</td>
<td>Actinoplanes missouriensis 431</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002362: 1-24294">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002362/index.html#r1c1" target="_new">BGC0002362</a></td>


<td class="cc-heat-cell" title="0.32 (id:0.43, order:N/A, components:0.24)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.32), rgb(0, 0, 0, 0.32) 32%, white 32%)"></td>
<td>0.32</td>
<td>Polyketide</td>
<td>loseolamycin A1, loseolamycin A2</td>
<td>Micromonospora endolithica</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000872: 0-35385">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000872/index.html#r1c1" target="_new">BGC0000872</a></td>


<td class="cc-heat-cell" title="0.24 (id:0.43, order:N/A, components:0.14)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.24), rgb(0, 0, 0, 0.24) 24%, white 24%)"></td>
<td>0.24</td>
<td>Other</td>
<td>A-90289 A, A-90289 B</td>
<td>Streptomyces sp. SANK 60405</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001076: 0-29709">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001076/index.html#r1c1" target="_new">BGC0001076</a></td>


<td class="cc-heat-cell" title="0.24 (id:0.42, order:N/A, components:0.14)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.24), rgb(0, 0, 0, 0.24) 24%, white 24%)"></td>
<td>0.24</td>
<td>Other (Nucleoside)</td>
<td>liposidomycin B, liposidomycin A, liposidomycin C, liposidomycin Z, liposidomycin H, liposidomycin G, liposidomycin L, liposidomycin M, liposidomycin K, liposidomycin N</td>
<td>Streptomyces sp. SN-1061M</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001647: 0-18309">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001647/index.html#r1c1" target="_new">BGC0001647</a></td>


<td class="cc-heat-cell" title="0.22 (id:0.41, order:N/A, components:0.11)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.22), rgb(0, 0, 0, 0.22) 22%, white 22%)"></td>
<td>0.22</td>
<td>Polyketide</td>
<td>lagunapyrone A, lagunapyrone B, lagunapyrone C</td>
<td>Streptomyces sp. MP131-18</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001665: 0-19519">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001665/index.html#r1c1" target="_new">BGC0001665</a></td>


<td class="cc-heat-cell" title="0.20 (id:0.42, order:N/A, components:0.09)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.20), rgb(0, 0, 0, 0.20) 20%, white 20%)"></td>
<td>0.20</td>
<td>Polyketide</td>
<td>methylated alkyl-resorcinol, methylated acyl-phloroglucinol</td>
<td>Mycobacterium marinum M</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002290: 5128-32193">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002290/index.html#r1c1" target="_new">BGC0002290</a></td>


<td class="cc-heat-cell" title="0.20 (id:0.46, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.20), rgb(0, 0, 0, 0.20) 20%, white 20%)"></td>
<td>0.20</td>
<td>Other</td>
<td>biscognienyne B, eutypinic acid, compound 3, 4-hydroxy-3-prenylbenzoic acid, prenylhydroquinone, compound 15, compound 8, compound 7, compound 6, compound 11, compound 12, pestalodiol E</td>
<td>Biscogniauxia sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001848: 0-58198">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001848/index.html#r1c1" target="_new">BGC0001848</a></td>


<td class="cc-heat-cell" title="0.18 (id:0.41, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.18), rgb(0, 0, 0, 0.18) 18%, white 18%)"></td>
<td>0.18</td>
<td>Other</td>
<td>diazaquinomycin H, diazaquinomycin J</td>
<td>Micromonospora sp. B006</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002439: 0-24052">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002439/index.html#r1c1" target="_new">BGC0002439</a></td>


<td class="cc-heat-cell" title="0.16 (id:0.41, order:N/A, components:0.06)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.16), rgb(0, 0, 0, 0.16) 16%, white 16%)"></td>
<td>0.16</td>
<td>Other</td>
<td>diastaphenazine, izumiphenazine C</td>
<td>Streptomyces diastaticus</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r51c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-506"></div> <span class="help-tooltip" id="cb-general-help-506">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<p>No significant ClusterBlast hits found.</p>
</div>

</div>
</div>

<div class="body-details-section r51c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-507"></div> <span class="help-tooltip" id="cb-known-help-507">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r51c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-508"></div> <span class="help-tooltip" id="cb-sub-help-508">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r51c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-510"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-510">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r51c1-pfam-details-svg" data-filename="scaffold_51_r51c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r51c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">













</div>













</div>

</div>

</div>





<div class="page" id='r53c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_53 - Region 1 - fungal-RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-837"></div> <span class="help-tooltip" id="region_svg-help-837">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_53.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r53c1-svg" data-filename="scaffold_53_r53c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 113,468 - 173,947 nt. (total: 60,480 nt)

<a class="cluster-rules-header" id="r53c1-rules-header" href="#r53c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r53c1-rules">fungal-RiPP-like: DUF3328</div>

<div class="region-svg-container">
<div id='r53c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-838"></div> <span class="help-tooltip" id="focus-panel-help-838">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r53c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r53c1-gene-table-details" data-name="r53c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r53c1-MIBiG-cluster-compare" data-name="r53c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r53c1-clusterblast" data-name="r53c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r53c1-knownclusterblast" data-name="r53c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r53c1-subclusterblast" data-name="r53c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r53c1-pfam-details" data-name="r53c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r53c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-515"></div> <span class="help-tooltip" id="gt-help-tooltip-help-515">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r53c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r53c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r53c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r53c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r53c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-511"></div> <span class="help-tooltip" id="MIBiG-help-511">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r53c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r53c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r53c1"></div>
<div id="comparison-MIBiG-r53c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">fungal-RiPP-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001487: 0-9112">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001487/index.html#r1c1" target="_new">BGC0001487</a></td>





<td class="cc-heat-cell" title="0.37 (id:0.42, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>


<td>0.11</td>
<td>Other</td>

<td>3,7-dihydroxytropolone</td>


<td>Streptomyces cyaneofuscatus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001873: 0-8606">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001873/index.html#r1c1" target="_new">BGC0001873</a></td>





<td class="cc-heat-cell" title="0.37 (id:0.42, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>


<td>0.11</td>
<td>NRP</td>

<td>pyrrolizixenamide A</td>


<td>Xenorhabdus szentirmaii DSM 16338</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001149: 0-12786">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001149/index.html#r1c1" target="_new">BGC0001149</a></td>





<td class="cc-heat-cell" title="0.30 (id:0.41, order:N/A, components:0.22)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>


<td>0.08</td>
<td>NRP, Saccharide</td>

<td>brabantamide A</td>


<td>Pseudomonas sp. SHC52</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000160: 0-27093">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000160/index.html#r1c1" target="_new">BGC0000160</a></td>





<td class="cc-heat-cell" title="0.27 (id:0.43, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>


<td>0.06</td>
<td>Polyketide</td>

<td>terreic acid</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001722: 0-18441">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001722/index.html#r1c1" target="_new">BGC0001722</a></td>





<td class="cc-heat-cell" title="0.27 (id:0.43, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>


<td>0.06</td>
<td>Polyketide</td>

<td>aspernidine A</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001404: 0-28758">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001404/index.html#r1c1" target="_new">BGC0001404</a></td>





<td class="cc-heat-cell" title="0.27 (id:0.42, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>


<td>0.06</td>
<td>Polyketide</td>

<td>sorbicillin</td>


<td>Penicillium rubens Wisconsin 54-1255</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002255: 803-34333">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002255/index.html#r1c1" target="_new">BGC0002255</a></td>





<td class="cc-heat-cell" title="0.26 (id:0.42, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>


<td>0.06</td>
<td>Polyketide</td>

<td>lijiquinone</td>


<td>Ascomycota sp. F53</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002290: 5128-32193">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002290/index.html#r1c1" target="_new">BGC0002290</a></td>





<td class="cc-heat-cell" title="0.26 (id:0.41, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>


<td>0.06</td>
<td>Other</td>

<td>biscognienyne B, eutypinic acid, compound 3, 4-hydroxy-3-prenylbenzoic acid, prenylhydroquinone, compound 15, compound 8, compound 7, compound 6, compound 11, compound 12, pestalodiol E</td>


<td>Biscogniauxia sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000022: 0-41240">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000022/index.html#r1c1" target="_new">BGC0000022</a></td>





<td class="cc-heat-cell" title="0.24 (id:0.42, order:N/A, components:0.13)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.24), rgb(0, 0, 0, 0.24) 24%, white 24%)"></td>


<td>0.05</td>
<td>Polyketide</td>

<td>asperfuranone</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002206: 173-38907">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002206/index.html#r1c1" target="_new">BGC0002206</a></td>





<td class="cc-heat-cell" title="0.24 (id:0.42, order:N/A, components:0.13)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.24), rgb(0, 0, 0, 0.24) 24%, white 24%)"></td>


<td>0.05</td>
<td>Polyketide</td>

<td>harziphilone, t22azaphilone, isoharziphilone-1, isoharziphilone-2, compound 4, compound 1</td>


<td>Trichoderma guizhouense</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r53c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001487: 0-9112">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001487/index.html#r1c1" target="_new">BGC0001487</a></td>


<td class="cc-heat-cell" title="0.37 (id:0.42, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>
<td>0.37</td>
<td>Other</td>
<td>3,7-dihydroxytropolone</td>
<td>Streptomyces cyaneofuscatus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001873: 0-8606">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001873/index.html#r1c1" target="_new">BGC0001873</a></td>


<td class="cc-heat-cell" title="0.37 (id:0.42, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>
<td>0.37</td>
<td>NRP</td>
<td>pyrrolizixenamide A</td>
<td>Xenorhabdus szentirmaii DSM 16338</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001149: 0-12786">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001149/index.html#r1c1" target="_new">BGC0001149</a></td>


<td class="cc-heat-cell" title="0.30 (id:0.41, order:N/A, components:0.22)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>
<td>0.30</td>
<td>NRP, Saccharide</td>
<td>brabantamide A</td>
<td>Pseudomonas sp. SHC52</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000160: 0-27093">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000160/index.html#r1c1" target="_new">BGC0000160</a></td>


<td class="cc-heat-cell" title="0.27 (id:0.43, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>
<td>0.27</td>
<td>Polyketide</td>
<td>terreic acid</td>
<td>Aspergillus terreus NIH2624</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001722: 0-18441">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001722/index.html#r1c1" target="_new">BGC0001722</a></td>


<td class="cc-heat-cell" title="0.27 (id:0.43, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>
<td>0.27</td>
<td>Polyketide</td>
<td>aspernidine A</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001404: 0-28758">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001404/index.html#r1c1" target="_new">BGC0001404</a></td>


<td class="cc-heat-cell" title="0.27 (id:0.42, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>
<td>0.27</td>
<td>Polyketide</td>
<td>sorbicillin</td>
<td>Penicillium rubens Wisconsin 54-1255</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002255: 803-34333">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002255/index.html#r1c1" target="_new">BGC0002255</a></td>


<td class="cc-heat-cell" title="0.26 (id:0.42, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>
<td>0.26</td>
<td>Polyketide</td>
<td>lijiquinone</td>
<td>Ascomycota sp. F53</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002290: 5128-32193">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002290/index.html#r1c1" target="_new">BGC0002290</a></td>


<td class="cc-heat-cell" title="0.26 (id:0.41, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>
<td>0.26</td>
<td>Other</td>
<td>biscognienyne B, eutypinic acid, compound 3, 4-hydroxy-3-prenylbenzoic acid, prenylhydroquinone, compound 15, compound 8, compound 7, compound 6, compound 11, compound 12, pestalodiol E</td>
<td>Biscogniauxia sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000022: 0-41240">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000022/index.html#r1c1" target="_new">BGC0000022</a></td>


<td class="cc-heat-cell" title="0.24 (id:0.42, order:N/A, components:0.13)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.24), rgb(0, 0, 0, 0.24) 24%, white 24%)"></td>
<td>0.24</td>
<td>Polyketide</td>
<td>asperfuranone</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002206: 173-38907">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002206/index.html#r1c1" target="_new">BGC0002206</a></td>


<td class="cc-heat-cell" title="0.24 (id:0.42, order:N/A, components:0.13)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.24), rgb(0, 0, 0, 0.24) 24%, white 24%)"></td>
<td>0.24</td>
<td>Polyketide</td>
<td>harziphilone, t22azaphilone, isoharziphilone-1, isoharziphilone-2, compound 4, compound 1</td>
<td>Trichoderma guizhouense</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r53c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-512"></div> <span class="help-tooltip" id="cb-general-help-512">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r53c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r53c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r53c1_1.svg">
NW_025953041 (485843-549525): Daldinia caldariorum strain CBS 122874 unplaced... (10% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r53c1_2.svg">
NW_026250023 (2687403-2778442): Hypoxylon trugodes strain CBS 135444 unplaced... (5% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r53c1_3.svg">
NW_026054552 (173115-236810): Daldinia decipiens strain CBS 113046 unplaced g... (10% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r53c1_4.svg">
NW_024468914 (138813-211522): Cryphonectria parasitica EP155 unplaced genomic... (9% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r53c1_5.svg">
NC_049562 (4108854-4176271): Talaromyces rugulosus chromosome II, complete se... (10% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r53c1_6.svg">
NW_022985551 (246485-314494): Pseudovirgaria hyperparasitica strain CBS 12173... (9% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r53c1_7.svg">
NZ_KB895825 (352530-425102): Micromonospora sp. CNB394 C569DRAFT scaffold0000... (2% of genes show similarity), T2PKS
</option>

<option value = "svg/clusterblast_r53c1_8.svg">
NZ_QKUA01000003 (410376-482937): Micromonospora phaseoli strain DSM 45730 Ga0... (2% of genes show similarity), T2PKS
</option>

</select>
<button id= "clusterblast-r53c1-download">Download graphic</button>
</div>
<div id="clusterblast-r53c1-svg"></div>

</div>
</div>

<div class="body-details-section r53c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-513"></div> <span class="help-tooltip" id="cb-known-help-513">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r53c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-514"></div> <span class="help-tooltip" id="cb-sub-help-514">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r53c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-516"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-516">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r53c1-pfam-details-svg" data-filename="scaffold_53_r53c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r53c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">











</div>











</div>

</div>

</div>





<div class="page" id='r56c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_56 - Region 1 - T1PKS,indole <div class="help-container"> <div class="help-icon" data-id="region_svg-help-839"></div> <span class="help-tooltip" id="region_svg-help-839">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_56.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r56c1-svg" data-filename="scaffold_56_r56c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 4,427 - 51,315 nt. (total: 46,889 nt)

<a class="cluster-rules-header" id="r56c1-rules-header" href="#r56c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r56c1-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))<br>indole: (indsynth or dmat or indole_PTase)</div>

<div class="region-svg-container">
<div id='r56c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-840"></div> <span class="help-tooltip" id="focus-panel-help-840">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r56c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r56c1-gene-table-details" data-name="r56c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r56c1-nrps_pks" data-name="r56c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r56c1-MIBiG-cluster-compare" data-name="r56c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r56c1-clusterblast" data-name="r56c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r56c1-knownclusterblast" data-name="r56c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r56c1-subclusterblast" data-name="r56c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r56c1-nrps-pks-bubbles" data-name="r56c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r56c1-pfam-details" data-name="r56c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r56c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-525"></div> <span class="help-tooltip" id="gt-help-tooltip-help-525">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r56c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r56c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r56c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r56c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r56c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-517"></div> <span class="help-tooltip" id="nrps-domain-help-517">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r56c1-details-svg" data-filename="scaffold_56_r56c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r56c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r56c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-518"></div> <span class="help-tooltip" id="MIBiG-help-518">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r56c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r56c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r56c1"></div>
<div id="comparison-MIBiG-r56c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>

<th><span class="cc-heat-product">indole</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001219: 0-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001219/index.html#r1c1" target="_new">BGC0001219</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.47, order:0.37, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>






<td class="cc-heat-cell" title="0.58 (id:0.50, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.44</td>
<td>Polyketide</td>

<td>shanorellin</td>


<td>Chaetomium globosum CBS 148.51</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001969: 0-7810">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001969/index.html#r1c1" target="_new">BGC0001969</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>






<td class="cc-heat-cell" title="0.53 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.40</td>
<td>Terpene</td>

<td>asperterpenoid A</td>


<td>Talaromyces wortmannii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001267: 3-35442">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001267/index.html#r1c1" target="_new">BGC0001267</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>






<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.37</td>
<td>Terpene</td>

<td>lysergic acid, elymoclavine</td>


<td>Claviceps fusiformis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000798: 335-104320">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000798/index.html#r1c1" target="_new">BGC0000798</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>






<td class="cc-heat-cell" title="0.49 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.35</td>
<td>Saccharide</td>

<td>dhurrin</td>


<td>Sorghum bicolor</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000037: 0-7821">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000037/index.html#r1c1" target="_new">BGC0000037</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.48, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>


<td>0.33</td>
<td>Polyketide</td>

<td>cichorine</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000676: 621-14838">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000676/index.html#r1c1" target="_new">BGC0000676</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.41, order:N/A, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>






<td class="cc-heat-cell" title="0.47 (id:0.41, order:N/A, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.33</td>
<td>Terpene</td>

<td>aphidicolin, aphidicolan-16-ol, 3-deoxyaphidicolin, 17-deoxyaphidicolin</td>


<td>Phoma betae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002721: 182-542486">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002721/index.html#r1c1" target="_new">BGC0002721</a></td>





<td class="cc-heat-cell" title="0.45 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>






<td class="cc-heat-cell" title="0.45 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>


<td>0.30</td>
<td>Saccharide</td>

<td>-hydroxynitrile glucoside, -hydroxynitrile glucoside, -hydroxynitrile glucoside</td>


<td>Hordeum vulgare subsp. vulgare</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000442: 0-9759">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000442/index.html#r1c1" target="_new">BGC0000442</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.41, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>






<td class="cc-heat-cell" title="0.37 (id:0.41, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>


<td>0.28</td>
<td>NRP</td>

<td>terrequinone A</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001360: 0-23157">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001360/index.html#r1c1" target="_new">BGC0001360</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.46, order:0.37, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>






<td class="cc-heat-cell" title="0.39 (id:0.47, order:N/A, components:0.32)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>


<td>0.27</td>
<td>Polyketide</td>

<td>mycophenolic acid</td>


<td>Penicillium roqueforti FM164</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001390: 100-19645">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001390/index.html#r1c1" target="_new">BGC0001390</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>


<td>0.27</td>
<td>NRP, Polyketide</td>

<td>LL-Z1272beta</td>


<td>Stachybotrys bisbyi</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r56c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000037: 0-7821">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000037/index.html#r1c1" target="_new">BGC0000037</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.48, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Polyketide</td>
<td>cichorine</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001390: 100-19645">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001390/index.html#r1c1" target="_new">BGC0001390</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>NRP, Polyketide</td>
<td>LL-Z1272beta</td>
<td>Stachybotrys bisbyi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001923: 495-32413">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001923/index.html#r1c1" target="_new">BGC0001923</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.45, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Terpene, Polyketide</td>
<td>ascochlorin</td>
<td>Acremonium egyptiacum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001720: 0-45622">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001720/index.html#r1c1" target="_new">BGC0001720</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Polyketide</td>
<td>oosporein</td>
<td>Beauveria bassiana ARSEF 2860</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000056: 2-6689">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000056/index.html#r1c1" target="_new">BGC0000056</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.41, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Polyketide</td>
<td>esperamicin</td>
<td>Actinomadura verrucosospora</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000013: 0-7849">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000013/index.html#r1c1" target="_new">BGC0000013</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Polyketide</td>
<td>alternariol</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001284: 0-5292">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001284/index.html#r1c1" target="_new">BGC0001284</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Polyketide</td>
<td>alternariol</td>
<td>Parastagonospora nodorum SN15</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002236: 673-11427">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002236/index.html#r1c1" target="_new">BGC0002236</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Polyketide</td>
<td>8-methyldiaporthin</td>
<td>Aspergillus oryzae RIB40</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000156: 500-13275">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000156/index.html#r1c1" target="_new">BGC0000156</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Polyketide</td>
<td>TAN-1612, 1-(2,3,5,10-tetrahydroxy-7-methoxy-4-oxo-1,2,3,4-tetrahydroanthracen-2-yl)pentane-2,4-dione, desmethyl TAN-1612</td>
<td>Aspergillus niger</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002596: 0-33285">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002596/index.html#r1c1" target="_new">BGC0002596</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Polyketide</td>
<td>lecanoric acid, orsellinic acid</td>
<td>Claviceps purpurea 20.1</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r56c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-519"></div> <span class="help-tooltip" id="cb-general-help-519">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r56c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r56c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r56c1_1.svg">
NC_064677 (5429112-5558494): Colletotrichum lupini chromosome 4, complete seq... (14% of genes show similarity), NRPS-like,T1PKS,indole,terpene
</option>

<option value = "svg/clusterblast_r56c1_2.svg">
NW_006917091 (1121274-1194772): Pestalotiopsis fici W106-1 unplaced genomic s... (11% of genes show similarity), NRPS-like,T1PKS,indole
</option>

<option value = "svg/clusterblast_r56c1_3.svg">
NW_022985010 (1623664-1725341): Dothidotthia symphoricarpi CBS 119687 unplace... (9% of genes show similarity), T1PKS,indole
</option>

<option value = "svg/clusterblast_r56c1_4.svg">
NC_072349 (290456-369836): Talaromyces marneffei chromosome 2, complete sequence (6% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r56c1_5.svg">
NC_066257 (109084-244310): Aspergillus nidulans FGSC A4 chromosome I (4% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r56c1_6.svg">
NT_165982 (1674335-1763165): Chaetomium globosum CBS 148.51 scaffold 7 genomi... (7% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r56c1_7.svg">
NW_007361000 (137003-235792): Glarea lozoyensis ATCC 20868 chromosome Unknown... (8% of genes show similarity), T1PKS,fungal-RiPP-like,indole
</option>

<option value = "svg/clusterblast_r56c1_8.svg">
NW_024468909 (49034-180513): Cryphonectria parasitica EP155 unplaced genomic ... (4% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r56c1_9.svg">
NW_006917097 (1365204-1433007): Pestalotiopsis fici W106-1 unplaced genomic s... (9% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r56c1_10.svg">
NC_057365 (3065801-3132990): Aspergillus chevalieri M1 DNA, chromosome 4, nea... (9% of genes show similarity), T1PKS
</option>

</select>
<button id= "clusterblast-r56c1-download">Download graphic</button>
</div>
<div id="clusterblast-r56c1-svg"></div>

</div>
</div>

<div class="body-details-section r56c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-520"></div> <span class="help-tooltip" id="cb-known-help-520">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r56c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r56c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r56c1_1.svg">
ustethylin A
</option>

</select>
<button id="knownclusterblast-r56c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r56c1-svg"></div>

</div>
</div>

<div class="body-details-section r56c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-521"></div> <span class="help-tooltip" id="cb-sub-help-521">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r56c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-524"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-524">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r56c1-domain-bubble-svg" data-filename="scaffold_56_r56c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r56c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (4426 - 51315): neighbouring T1PKS-indole
</option>


</select>

<div class="details-svg" id="r56c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r56c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r56c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-526"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-526">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r56c1-pfam-details-svg" data-filename="scaffold_56_r56c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r56c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r56c1-nrps_pks_products" data-name="r56c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r56c1-nrps_pks_monomers" data-name="r56c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r56c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-522"></div> <span class="help-tooltip" id="nrps-structures-help-522">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 4426 - 51315:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r56c1-1" class="smiles-canvas" data-smiles="CC(=O)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(mal)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r56c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="CC(=O)C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r56c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-523"></div> <span class="help-tooltip" id="nrps-monomers-help-523">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">FUN_008238</span></strong>: mal
<div class="collapser collapser-target-FUN_008238 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (540..828)</strong>: mal
<div class="collapser collapser-target-FUN_008238 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-FUN_008238-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 66.7%</dd>
<dd>Ethylmalonyl-CoA: 54.2%</dd>
<dd>Methylmalonyl-CoA: 54.2%</dd>
</dl>

</div><br>

minowa:  Malonyl-CoA
<div class="collapser collapser-target-FUN_008238-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Malonyl-CoA: 51.9</dt>

<dd></dd><dt>inactive: 44.1</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 37.3</dt>

<dd></dd><dt>Methylmalonyl-CoA: 31.2</dt>

<dd></dd><dt>CHC-CoA: 16.8</dt>

<dd></dd><dt>Benzoyl-CoA: 14.0</dt>

<dd></dd><dt>Propionyl-CoA: 12.7</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 9.7</dt>

<dd></dd><dt>Isobutyryl-CoA: 9.4</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 9.3</dt>

<dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

<dd></dd><dt>fatty_acid: 0.0</dt>

<dd></dd><dt>Acetyl-CoA: 0.0</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>










</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>





<div class="page" id='r57c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_57 - Region 1 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-841"></div> <span class="help-tooltip" id="region_svg-help-841">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_57.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r57c1-svg" data-filename="scaffold_57_r57c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 84,081 - 132,472 nt. (total: 48,392 nt)

<a class="cluster-rules-header" id="r57c1-rules-header" href="#r57c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r57c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r57c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-842"></div> <span class="help-tooltip" id="focus-panel-help-842">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r57c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r57c1-gene-table-details" data-name="r57c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r57c1-nrps_pks" data-name="r57c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r57c1-MIBiG-cluster-compare" data-name="r57c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r57c1-clusterblast" data-name="r57c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r57c1-knownclusterblast" data-name="r57c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r57c1-subclusterblast" data-name="r57c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r57c1-nrps-pks-bubbles" data-name="r57c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r57c1-pfam-details" data-name="r57c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r57c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-535"></div> <span class="help-tooltip" id="gt-help-tooltip-help-535">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r57c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r57c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r57c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r57c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r57c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-527"></div> <span class="help-tooltip" id="nrps-domain-help-527">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r57c1-details-svg" data-filename="scaffold_57_r57c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r57c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r57c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-528"></div> <span class="help-tooltip" id="MIBiG-help-528">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r57c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r57c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r57c1"></div>
<div id="comparison-MIBiG-r57c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.41, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.23</td>
<td>NRP</td>

<td>AM-toxin</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001240: 0-28516">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001240/index.html#r1c1" target="_new">BGC0001240</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.42, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>NRP</td>

<td>serinocyclin A, serinocyclin B</td>


<td>Metarhizium robertsii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000357: 0-18536">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000357/index.html#r1c1" target="_new">BGC0000357</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.21</td>
<td>NRP</td>

<td>cyclo-(D-Phe-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Phe-D-Val-L-Ile), cyclo-(D-Phe-L-Phe-D-Ile-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Ile), cyclo-(D-Tyr-L-Trp-D-Ile-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Ile), cyclo-(D-Tyr-L-Phe-D-Ile-L-Val)</td>


<td>Penicillium rubens Wisconsin 54-1255</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900: 2044-8911">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.20</td>
<td>Other</td>

<td>ferrichrome</td>


<td>Aspergillus oryzae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002208: 764-29749">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002208/index.html#r1c1" target="_new">BGC0002208</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.41, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>NRP</td>

<td>oxepinamide F</td>


<td>Aspergillus ustus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002617: 657-14847">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002617/index.html#r1c1" target="_new">BGC0002617</a></td>





<td class="cc-heat-cell" title="0.44 (id:0.41, order:N/A, components:0.48)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>


<td>0.15</td>
<td>NRP</td>

<td>psychrophilin D, psychrophilin H, psychrophilin F, psychrophilin C, psychrophilin G, psychrophilin E, psychrophilin A, psychrophilin B, psychrophilin I</td>


<td>Penicillium sp. YT-2016</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001699: 0-30427">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001699/index.html#r1c1" target="_new">BGC0001699</a></td>





<td class="cc-heat-cell" title="0.44 (id:0.43, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>


<td>0.14</td>
<td>NRP</td>

<td>nidulanin A</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002259: 849-29665">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002259/index.html#r1c1" target="_new">BGC0002259</a></td>





<td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>


<td>0.13</td>
<td>Polyketide, NRP</td>

<td>beauveriolide I, beauveriolide III</td>


<td>Cordyceps militaris CM01</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000355: 0-24812">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000355/index.html#r1c1" target="_new">BGC0000355</a></td>





<td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>


<td>0.13</td>
<td>NRP</td>

<td>fumiquinazoline A, fumiquinazoline C, fumiquinazoline D</td>


<td>Aspergillus fumigatus Af293</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000348: 0-13540">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000348/index.html#r1c1" target="_new">BGC0000348</a></td>





<td class="cc-heat-cell" title="0.42 (id:0.41, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>


<td>0.13</td>
<td>NRP</td>

<td>ergovaline</td>


<td>Epichloe festucae var. lolii</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r57c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.41, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>NRP</td>
<td>AM-toxin</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001240: 0-28516">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001240/index.html#r1c1" target="_new">BGC0001240</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.42, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>NRP</td>
<td>serinocyclin A, serinocyclin B</td>
<td>Metarhizium robertsii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000357: 0-18536">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000357/index.html#r1c1" target="_new">BGC0000357</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>NRP</td>
<td>cyclo-(D-Phe-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Trp-D-Val-L-Val), cyclo-(D-Phe-L-Phe-D-Val-L-Ile), cyclo-(D-Phe-L-Phe-D-Ile-L-Val), cyclo-(D-Tyr-L-Trp-D-Val-L-Ile), cyclo-(D-Tyr-L-Trp-D-Ile-L-Val), cyclo-(D-Tyr-L-Phe-D-Val-L-Ile), cyclo-(D-Tyr-L-Phe-D-Ile-L-Val)</td>
<td>Penicillium rubens Wisconsin 54-1255</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900: 2044-8911">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>Other</td>
<td>ferrichrome</td>
<td>Aspergillus oryzae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002208: 764-29749">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002208/index.html#r1c1" target="_new">BGC0002208</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.41, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>NRP</td>
<td>oxepinamide F</td>
<td>Aspergillus ustus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002617: 657-14847">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002617/index.html#r1c1" target="_new">BGC0002617</a></td>


<td class="cc-heat-cell" title="0.44 (id:0.41, order:N/A, components:0.48)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
<td>0.44</td>
<td>NRP</td>
<td>psychrophilin D, psychrophilin H, psychrophilin F, psychrophilin C, psychrophilin G, psychrophilin E, psychrophilin A, psychrophilin B, psychrophilin I</td>
<td>Penicillium sp. YT-2016</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001699: 0-30427">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001699/index.html#r1c1" target="_new">BGC0001699</a></td>


<td class="cc-heat-cell" title="0.44 (id:0.43, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
<td>0.44</td>
<td>NRP</td>
<td>nidulanin A</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002259: 849-29665">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002259/index.html#r1c1" target="_new">BGC0002259</a></td>


<td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
<td>0.42</td>
<td>Polyketide, NRP</td>
<td>beauveriolide I, beauveriolide III</td>
<td>Cordyceps militaris CM01</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000355: 0-24812">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000355/index.html#r1c1" target="_new">BGC0000355</a></td>


<td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
<td>0.42</td>
<td>NRP</td>
<td>fumiquinazoline A, fumiquinazoline C, fumiquinazoline D</td>
<td>Aspergillus fumigatus Af293</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000348: 0-13540">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000348/index.html#r1c1" target="_new">BGC0000348</a></td>


<td class="cc-heat-cell" title="0.42 (id:0.41, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
<td>0.42</td>
<td>NRP</td>
<td>ergovaline</td>
<td>Epichloe festucae var. lolii</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r57c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-529"></div> <span class="help-tooltip" id="cb-general-help-529">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r57c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r57c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r57c1_1.svg">
NW_025953277 (146733-244628): Annulohypoxylon maeteangense strain CBS 123835 ... (27% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r57c1_2.svg">
NW_025953038 (2634443-2728113): Daldinia caldariorum strain CBS 122874 unplac... (28% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r57c1_3.svg">
NW_026250028 (373060-500623): Hypoxylon trugodes strain CBS 135444 unplaced g... (20% of genes show similarity), NRPS,T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r57c1_4.svg">
NW_026054582 (14709-110134): Daldinia decipiens strain CBS 113046 unplaced ge... (24% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r57c1_5.svg">
NW_022984642 (3544711-3636862): Daldinia childiae strain JS-1345 chromosome U... (25% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r57c1_6.svg">
NW_025953842 (93917-193351): Annulohypoxylon truncatum strain CBS 140777 unpl... (20% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r57c1_7.svg">
NW_006917095 (3610764-3678979): Pestalotiopsis fici W106-1 unplaced genomic s... (26% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r57c1_8.svg">
NW_014574716 (5649629-5717960): Metarhizium brunneum ARSEF 3297 chromosome Un... (40% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r57c1_9.svg">
NW_011942160 (3156518-3224837): Metarhizium robertsii ARSEF 23 MAA Scf 2, who... (50% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r57c1_10.svg">
NC_070529 (6313137-6381042): Fusarium keratoplasticum isolate Fu6.1 chromosom... (27% of genes show similarity), NRPS
</option>

</select>
<button id= "clusterblast-r57c1-download">Download graphic</button>
</div>
<div id="clusterblast-r57c1-svg"></div>

</div>
</div>

<div class="body-details-section r57c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-530"></div> <span class="help-tooltip" id="cb-known-help-530">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r57c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-531"></div> <span class="help-tooltip" id="cb-sub-help-531">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r57c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-534"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-534">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r57c1-domain-bubble-svg" data-filename="scaffold_57_r57c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r57c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (84080 - 132472): single NRPS
</option>


</select>

<div class="details-svg" id="r57c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r57c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r57c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-536"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-536">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r57c1-pfam-details-svg" data-filename="scaffold_57_r57c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r57c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r57c1-nrps_pks_products" data-name="r57c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r57c1-nrps_pks_monomers" data-name="r57c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r57c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-532"></div> <span class="help-tooltip" id="nrps-structures-help-532">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 84080 - 132472:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r57c1-1" class="smiles-canvas" data-smiles="NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r57c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r57c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-533"></div> <span class="help-tooltip" id="nrps-monomers-help-533">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">FUN_008316</span></strong>: X
<div class="collapser collapser-target-FUN_008316 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>







<dd><strong>AMP-binding (1433..1821)</strong>: X
<div class="collapser collapser-target-FUN_008316 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr
<div class="collapser collapser-target-FUN_008316-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DVAYCRVLSK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>

<dt>Stachelhaus code match:</dt>
<dd>0% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r57c2' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_57 - Region 2 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-843"></div> <span class="help-tooltip" id="region_svg-help-843">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_57.region002.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r57c2-svg" data-filename="scaffold_57_r57c2.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 175,718 - 223,463 nt. (total: 47,746 nt)

<a class="cluster-rules-header" id="r57c2-rules-header" href="#r57c2">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r57c2-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r57c2-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-844"></div> <span class="help-tooltip" id="focus-panel-help-844">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r57c2">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r57c2-gene-table-details" data-name="r57c2-gene-table-details">
Gene overview
</div>



<div class="body-details-header r57c2-nrps_pks" data-name="r57c2-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r57c2-MIBiG-cluster-compare" data-name="r57c2-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r57c2-clusterblast" data-name="r57c2-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r57c2-knownclusterblast" data-name="r57c2-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r57c2-subclusterblast" data-name="r57c2-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r57c2-nrps-pks-bubbles" data-name="r57c2-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r57c2-pfam-details" data-name="r57c2-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r57c2-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-545"></div> <span class="help-tooltip" id="gt-help-tooltip-help-545">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r57c2">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r57c2-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r57c2-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r57c2">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r57c2-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-537"></div> <span class="help-tooltip" id="nrps-domain-help-537">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r57c2-details-svg" data-filename="scaffold_57_r57c2_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r57c2-details-svg"></div>
</div>
</div>



<div class="body-details-section r57c2-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-538"></div> <span class="help-tooltip" id="MIBiG-help-538">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r57c2-selector">Analysis type:</label>
<select id="comparison-MIBiG-r57c2-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r57c2"></div>
<div id="comparison-MIBiG-r57c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001400: 30-15436">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001400/index.html#r1c1" target="_new">BGC0001400</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.21</td>
<td>Polyketide</td>

<td>citreoviridin</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002155: 78-16511">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002155/index.html#r1c1" target="_new">BGC0002155</a></td>





<td class="cc-heat-cell" title="0.51 (id:0.43, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>


<td>0.19</td>
<td>Polyketide</td>

<td>nectriapyrone C, nectriapyrone D, nectriapyrone</td>


<td>Pyricularia oryzae 70-15</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001254: 82-7456">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001254/index.html#r1c1" target="_new">BGC0001254</a></td>





<td class="cc-heat-cell" title="0.51 (id:0.44, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>


<td>0.19</td>
<td>Polyketide</td>

<td>ACT-Toxin II</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000170: 0-23998">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000170/index.html#r1c1" target="_new">BGC0000170</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.59)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>Polyketide</td>

<td>yanuthone D</td>


<td>Aspergillus niger ATCC 1015</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001068: 0-35628">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001068/index.html#r1c1" target="_new">BGC0001068</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>Terpene, Polyketide</td>

<td>pyripyropene A</td>


<td>unidentified unclassified sequences.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000996: 0-16575">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000996/index.html#r1c1" target="_new">BGC0000996</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>NRP, Polyketide</td>

<td>frontalamide B, frontalamide A</td>


<td>Streptomyces sp. SPB78</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002615: 973-23784">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002615/index.html#r1c1" target="_new">BGC0002615</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>Terpene</td>

<td>4-oxomacrophorin A, macrophorin A, 5&#39;-epimacrophorin B</td>


<td>Penicillium terrestre</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001954: 0-19896">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001954/index.html#r1c1" target="_new">BGC0001954</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>NRP, Polyketide</td>

<td>wortmanamide A, wortmanamide B</td>


<td>Talaromyces wortmannii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002240: 96-11225">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002240/index.html#r1c1" target="_new">BGC0002240</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>Polyketide</td>

<td>BAB, BAA</td>


<td>Metarhizium anisopliae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000107: 144-6819">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000107/index.html#r1c1" target="_new">BGC0000107</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.17</td>
<td>Polyketide</td>

<td>naphthopyrone</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r57c2-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001400: 30-15436">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001400/index.html#r1c1" target="_new">BGC0001400</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Polyketide</td>
<td>citreoviridin</td>
<td>Aspergillus terreus NIH2624</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002155: 78-16511">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002155/index.html#r1c1" target="_new">BGC0002155</a></td>


<td class="cc-heat-cell" title="0.51 (id:0.43, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
<td>0.51</td>
<td>Polyketide</td>
<td>nectriapyrone C, nectriapyrone D, nectriapyrone</td>
<td>Pyricularia oryzae 70-15</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001254: 82-7456">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001254/index.html#r1c1" target="_new">BGC0001254</a></td>


<td class="cc-heat-cell" title="0.51 (id:0.44, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
<td>0.51</td>
<td>Polyketide</td>
<td>ACT-Toxin II</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000170: 0-23998">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000170/index.html#r1c1" target="_new">BGC0000170</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.59)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Polyketide</td>
<td>yanuthone D</td>
<td>Aspergillus niger ATCC 1015</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001068: 0-35628">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001068/index.html#r1c1" target="_new">BGC0001068</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Terpene, Polyketide</td>
<td>pyripyropene A</td>
<td>unidentified unclassified sequences.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000996: 0-16575">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000996/index.html#r1c1" target="_new">BGC0000996</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.57)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>NRP, Polyketide</td>
<td>frontalamide B, frontalamide A</td>
<td>Streptomyces sp. SPB78</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002615: 973-23784">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002615/index.html#r1c1" target="_new">BGC0002615</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Terpene</td>
<td>4-oxomacrophorin A, macrophorin A, 5&#39;-epimacrophorin B</td>
<td>Penicillium terrestre</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001954: 0-19896">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001954/index.html#r1c1" target="_new">BGC0001954</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>NRP, Polyketide</td>
<td>wortmanamide A, wortmanamide B</td>
<td>Talaromyces wortmannii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002240: 96-11225">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002240/index.html#r1c1" target="_new">BGC0002240</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>Polyketide</td>
<td>BAB, BAA</td>
<td>Metarhizium anisopliae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000107: 144-6819">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000107/index.html#r1c1" target="_new">BGC0000107</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>Polyketide</td>
<td>naphthopyrone</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r57c2-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-539"></div> <span class="help-tooltip" id="cb-general-help-539">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r57c2-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r57c2_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r57c2_1.svg">
NC_064682 (379072-446578): Colletotrichum lupini chromosome 9, complete sequence (12% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r57c2_2.svg">
NW_025953842 (93917-193351): Annulohypoxylon truncatum strain CBS 140777 unpl... (12% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r57c2_3.svg">
NW_025953277 (146733-244628): Annulohypoxylon maeteangense strain CBS 123835 ... (13% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r57c2_4.svg">
NW_025953038 (2634443-2728113): Daldinia caldariorum strain CBS 122874 unplac... (14% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r57c2_5.svg">
NW_026054582 (14709-110134): Daldinia decipiens strain CBS 113046 unplaced ge... (12% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r57c2_6.svg">
NW_022984642 (3544711-3636862): Daldinia childiae strain JS-1345 chromosome U... (15% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r57c2_7.svg">
NW_006917095 (3915902-3983489): Pestalotiopsis fici W106-1 unplaced genomic s... (16% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r57c2_8.svg">
NW_026250028 (373060-500623): Hypoxylon trugodes strain CBS 135444 unplaced g... (6% of genes show similarity), NRPS,T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r57c2_9.svg">
NW_009799103 (924251-992132): Neurospora tetrasperma FGSC 2508 unplaced genom... (13% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r57c2_10.svg">
NC_016459 (4003420-4070920): Thermothielavioides terrestris NRRL 8126 chromos... (12% of genes show similarity), T1PKS
</option>

</select>
<button id= "clusterblast-r57c2-download">Download graphic</button>
</div>
<div id="clusterblast-r57c2-svg"></div>

</div>
</div>

<div class="body-details-section r57c2-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-540"></div> <span class="help-tooltip" id="cb-known-help-540">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r57c2-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-541"></div> <span class="help-tooltip" id="cb-sub-help-541">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r57c2-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-544"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-544">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r57c2-domain-bubble-svg" data-filename="scaffold_57_r57c2_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r57c2-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC2">
Candidate 2 (175717 - 223463): single T1PKS
</option>


</select>

<div class="details-svg" id="r57c2-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r57c2-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r57c2-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-546"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-546">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r57c2-pfam-details-svg" data-filename="scaffold_57_r57c2_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r57c2-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r57c2-nrps_pks_products" data-name="r57c2-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r57c2-nrps_pks_monomers" data-name="r57c2-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r57c2-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-542"></div> <span class="help-tooltip" id="nrps-structures-help-542">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 2, location 175717 - 223463:
<div class="collapser collapser-target-CC2 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r57c2-2" class="smiles-canvas" data-smiles="C([*])C(-O)"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(pk)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r57c2-2-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r57c2-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-543"></div> <span class="help-tooltip" id="nrps-monomers-help-543">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">FUN_008337</span></strong>: pk
<div class="collapser collapser-target-FUN_008337 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (598..904)</strong>: pk
<div class="collapser collapser-target-FUN_008337 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-FUN_008337-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 83.3%</dd>
<dd>Methylmalonyl-CoA: 70.8%</dd>
<dd>Ethylmalonyl-CoA: 62.5%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-FUN_008337-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 72.4</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 61.5</dt>

<dd></dd><dt>Benzoyl-CoA: 45.6</dt>

<dd></dd><dt>Malonyl-CoA: 41.8</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 40.0</dt>

<dd></dd><dt>Isobutyryl-CoA: 37.9</dt>

<dd></dd><dt>Propionyl-CoA: 29.7</dt>

<dd></dd><dt>trans-1,2-CPDA: 16.2</dt>

<dd></dd><dt>fatty_acid: 16.0</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 14.1</dt>

<dd></dd><dt>CHC-CoA: 13.1</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 8.6</dt>

<dd></dd><dt>inactive: 7.7</dt>

<dd></dd><dt>Acetyl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>








</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>





<div class="page" id='r59c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_59 - Region 1 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-845"></div> <span class="help-tooltip" id="region_svg-help-845">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_59.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r59c1-svg" data-filename="scaffold_59_r59c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 45,907 - 92,927 nt. (total: 47,021 nt)

<a class="cluster-rules-header" id="r59c1-rules-header" href="#r59c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r59c1-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r59c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-846"></div> <span class="help-tooltip" id="focus-panel-help-846">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r59c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r59c1-gene-table-details" data-name="r59c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r59c1-nrps_pks" data-name="r59c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r59c1-MIBiG-cluster-compare" data-name="r59c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r59c1-clusterblast" data-name="r59c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r59c1-knownclusterblast" data-name="r59c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r59c1-subclusterblast" data-name="r59c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r59c1-pfam-details" data-name="r59c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r59c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-553"></div> <span class="help-tooltip" id="gt-help-tooltip-help-553">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r59c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r59c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r59c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r59c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r59c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-547"></div> <span class="help-tooltip" id="nrps-domain-help-547">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r59c1-details-svg" data-filename="scaffold_59_r59c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r59c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r59c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-548"></div> <span class="help-tooltip" id="MIBiG-help-548">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r59c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r59c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r59c1"></div>
<div id="comparison-MIBiG-r59c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002175: 214-6865">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002175/index.html#r1c1" target="_new">BGC0002175</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.21</td>
<td>Polyketide</td>

<td>YWA1</td>


<td>Aspergillus oryzae RIB40</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000107: 144-6819">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000107/index.html#r1c1" target="_new">BGC0000107</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>Polyketide</td>

<td>naphthopyrone</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000013: 0-7849">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000013/index.html#r1c1" target="_new">BGC0000013</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>Polyketide</td>

<td>alternariol</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001257: 399-7066">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001257/index.html#r1c1" target="_new">BGC0001257</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>Polyketide</td>

<td>1,3,6,8-tetrahydroxynaphthalene</td>


<td>Nodulisporium sp. ATCC74245</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002236: 673-11427">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002236/index.html#r1c1" target="_new">BGC0002236</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>Polyketide</td>

<td>8-methyldiaporthin</td>


<td>Aspergillus oryzae RIB40</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001906: 35-64276">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001906/index.html#r1c1" target="_new">BGC0001906</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.20</td>
<td>Polyketide</td>

<td>naphthalene</td>


<td>Daldinia eschscholzii IFB-TL01</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001258: 43-7265">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001258/index.html#r1c1" target="_new">BGC0001258</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.20</td>
<td>Polyketide</td>

<td>1,3,6,8-tetrahydroxynaphthalene</td>


<td>Glarea lozoyensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002240: 96-11225">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002240/index.html#r1c1" target="_new">BGC0002240</a></td>





<td class="cc-heat-cell" title="0.51 (id:0.44, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>


<td>0.19</td>
<td>Polyketide</td>

<td>BAB, BAA</td>


<td>Metarhizium anisopliae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002191: 260-8785">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002191/index.html#r1c1" target="_new">BGC0002191</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>Polyketide</td>

<td>prolipyrone B, gibepyrone D</td>


<td>Fusarium graminearum PH-1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001254: 82-7456">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001254/index.html#r1c1" target="_new">BGC0001254</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>Polyketide</td>

<td>ACT-Toxin II</td>


<td>Alternaria alternata</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r59c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002175: 214-6865">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002175/index.html#r1c1" target="_new">BGC0002175</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Polyketide</td>
<td>YWA1</td>
<td>Aspergillus oryzae RIB40</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000107: 144-6819">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000107/index.html#r1c1" target="_new">BGC0000107</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Polyketide</td>
<td>naphthopyrone</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000013: 0-7849">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000013/index.html#r1c1" target="_new">BGC0000013</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Polyketide</td>
<td>alternariol</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001257: 399-7066">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001257/index.html#r1c1" target="_new">BGC0001257</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Polyketide</td>
<td>1,3,6,8-tetrahydroxynaphthalene</td>
<td>Nodulisporium sp. ATCC74245</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002236: 673-11427">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002236/index.html#r1c1" target="_new">BGC0002236</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Polyketide</td>
<td>8-methyldiaporthin</td>
<td>Aspergillus oryzae RIB40</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001906: 35-64276">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001906/index.html#r1c1" target="_new">BGC0001906</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>Polyketide</td>
<td>naphthalene</td>
<td>Daldinia eschscholzii IFB-TL01</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001258: 43-7265">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001258/index.html#r1c1" target="_new">BGC0001258</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.41, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>Polyketide</td>
<td>1,3,6,8-tetrahydroxynaphthalene</td>
<td>Glarea lozoyensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002240: 96-11225">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002240/index.html#r1c1" target="_new">BGC0002240</a></td>


<td class="cc-heat-cell" title="0.51 (id:0.44, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
<td>0.51</td>
<td>Polyketide</td>
<td>BAB, BAA</td>
<td>Metarhizium anisopliae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002191: 260-8785">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002191/index.html#r1c1" target="_new">BGC0002191</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Polyketide</td>
<td>prolipyrone B, gibepyrone D</td>
<td>Fusarium graminearum PH-1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001254: 82-7456">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001254/index.html#r1c1" target="_new">BGC0001254</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Polyketide</td>
<td>ACT-Toxin II</td>
<td>Alternaria alternata</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r59c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-549"></div> <span class="help-tooltip" id="cb-general-help-549">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r59c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r59c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r59c1_1.svg">
NW_022987219 (684623-752686): Trematosphaeria pertusa strain CBS 122368 unpla... (10% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r59c1_2.svg">
NW_025953830 (168631-237212): Annulohypoxylon truncatum strain CBS 140777 unp... (9% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r59c1_3.svg">
NW_022985451 (998305-1066533): Mytilinidion resinicola strain CBS 304.34 unpl... (8% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r59c1_4.svg">
NW_007360988 (1313-69766): Glarea lozoyensis ATCC 20868 chromosome Unknown GL... (8% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r59c1_5.svg">
NW_022474215 (790980-859315): Venustampulla echinocandica strain BP 5553 chro... (9% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r59c1_6.svg">
NW_017263596 (1176324-1299420): Mollisia scopiformis unplaced genomic scaffol... (4% of genes show similarity), T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r59c1_7.svg">
NW_025091264 (710679-779249): Aspergillus udagawae IFM 46973 DNA, scaffold 1,... (8% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r59c1_8.svg">
NC_007195 (276144-344706): Aspergillus fumigatus Af293 chromosome 2, whole ge... (8% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r59c1_9.svg">
NW_025091281 (4355587-4424149): Aspergillus pseudoviridinutans IFM 55266 DNA,... (8% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r59c1_10.svg">
NW_025091217 (5327073-5395643): Aspergillus viridinutans IFM 47045 DNA, scaff... (8% of genes show similarity), T1PKS
</option>

</select>
<button id= "clusterblast-r59c1-download">Download graphic</button>
</div>
<div id="clusterblast-r59c1-svg"></div>

</div>
</div>

<div class="body-details-section r59c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-550"></div> <span class="help-tooltip" id="cb-known-help-550">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r59c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-551"></div> <span class="help-tooltip" id="cb-sub-help-551">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r59c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-554"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-554">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r59c1-pfam-details-svg" data-filename="scaffold_59_r59c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r59c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r59c1-nrps_pks_monomers" data-name="r59c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>




</div>










<div class="sidepanel-details-section r59c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-552"></div> <span class="help-tooltip" id="nrps-monomers-help-552">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">FUN_008437</span></strong>: pk
<div class="collapser collapser-target-FUN_008437 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>PKS_AT (315..642)</strong>: pk
<div class="collapser collapser-target-FUN_008437 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-FUN_008437-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 79.2%</dd>
<dd>Methylmalonyl-CoA: 62.5%</dd>
<dd>Ethylmalonyl-CoA: 58.3%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-FUN_008437-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 57.8</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 56.7</dt>

<dd></dd><dt>Malonyl-CoA: 48.0</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 31.8</dt>

<dd></dd><dt>fatty_acid: 27.1</dt>

<dd></dd><dt>Propionyl-CoA: 25.0</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 16.9</dt>

<dd></dd><dt>inactive: 13.8</dt>

<dd></dd><dt>Isobutyryl-CoA: 12.9</dt>

<dd></dd><dt>CHC-CoA: 12.1</dt>

<dd></dd><dt>Benzoyl-CoA: 8.7</dt>

<dd></dd><dt>trans-1,2-CPDA: 6.7</dt>

<dd></dd><dt>Acetyl-CoA: 0.0</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>








</div>
<br>

</dl>
</div>
</div>




</div>

</div>

</div>

<div class="page" id='r59c2' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_59 - Region 2 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-847"></div> <span class="help-tooltip" id="region_svg-help-847">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_59.region002.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r59c2-svg" data-filename="scaffold_59_r59c2.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 179,384 - 202,107 nt. (total: 22,724 nt)

<a class="cluster-rules-header" id="r59c2-rules-header" href="#r59c2">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r59c2-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>

<div class="region-svg-container">
<div id='r59c2-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-848"></div> <span class="help-tooltip" id="focus-panel-help-848">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r59c2">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r59c2-gene-table-details" data-name="r59c2-gene-table-details">
Gene overview
</div>



<div class="body-details-header r59c2-MIBiG-cluster-compare" data-name="r59c2-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r59c2-clusterblast" data-name="r59c2-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r59c2-knownclusterblast" data-name="r59c2-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r59c2-subclusterblast" data-name="r59c2-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r59c2-pfam-details" data-name="r59c2-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r59c2-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-559"></div> <span class="help-tooltip" id="gt-help-tooltip-help-559">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r59c2">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r59c2-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r59c2-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r59c2">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r59c2-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-555"></div> <span class="help-tooltip" id="MIBiG-help-555">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r59c2-selector">Analysis type:</label>
<select id="comparison-MIBiG-r59c2-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r59c2"></div>
<div id="comparison-MIBiG-r59c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">terpene</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002610: 0-6468">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002610/index.html#r1c1" target="_new">BGC0002610</a></td>





<td class="cc-heat-cell" title="0.71 (id:0.48, order:0.75, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>


<td>0.39</td>
<td>Terpene</td>

<td>stellatic acid</td>


<td>Aspergillus stellatus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002320: 3-7131">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002320/index.html#r1c1" target="_new">BGC0002320</a></td>





<td class="cc-heat-cell" title="0.71 (id:0.47, order:0.75, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>


<td>0.38</td>
<td>Terpene</td>

<td>conidiogenone</td>


<td>Penicillium rubens Wisconsin 54-1255</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001969: 0-7810">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001969/index.html#r1c1" target="_new">BGC0001969</a></td>





<td class="cc-heat-cell" title="0.70 (id:0.46, order:0.75, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>


<td>0.38</td>
<td>Terpene</td>

<td>asperterpenoid A</td>


<td>Talaromyces wortmannii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001893: 155-19434">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001893/index.html#r1c1" target="_new">BGC0001893</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.33</td>
<td>Other</td>

<td>abscisic acid</td>


<td>Botrytis cinerea B05.10</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002736: 3-5057">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002736/index.html#r1c1" target="_new">BGC0002736</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.33</td>
<td>Terpene</td>

<td>preaspterpenacid I</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002162: 0-5465">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002162/index.html#r1c1" target="_new">BGC0002162</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.42, order:0.68, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.32</td>
<td>Terpene</td>

<td>sesterfisherol, sesterfisheric acid, compound 6, compound 7a, compound 5, compound 8</td>


<td>Aspergillus fischeri NRRL 181</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001604: 5569-24234">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001604/index.html#r1c1" target="_new">BGC0001604</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.48, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.32</td>
<td>Terpene</td>

<td>gibberellin</td>


<td>Fusarium fujikuroi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000670: 82-32651">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000670/index.html#r1c1" target="_new">BGC0000670</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.32</td>
<td>Terpene</td>

<td>thalianol, thaliandiol</td>


<td>Arabidopsis thaliana</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002221: 391-40725">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002221/index.html#r1c1" target="_new">BGC0002221</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.45, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.29</td>
<td>Terpene</td>

<td>ent-kauren-16alpha-ol</td>


<td>Serpula lacrymans var. lacrymans S7.9</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001475: 5-9618">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001475/index.html#r1c1" target="_new">BGC0001475</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.29</td>
<td>Terpene</td>

<td>aspterric acid</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r59c2-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002610: 0-6468">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002610/index.html#r1c1" target="_new">BGC0002610</a></td>


<td class="cc-heat-cell" title="0.71 (id:0.48, order:0.75, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
<td>0.71</td>
<td>Terpene</td>
<td>stellatic acid</td>
<td>Aspergillus stellatus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002320: 3-7131">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002320/index.html#r1c1" target="_new">BGC0002320</a></td>


<td class="cc-heat-cell" title="0.71 (id:0.47, order:0.75, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
<td>0.71</td>
<td>Terpene</td>
<td>conidiogenone</td>
<td>Penicillium rubens Wisconsin 54-1255</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001969: 0-7810">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001969/index.html#r1c1" target="_new">BGC0001969</a></td>


<td class="cc-heat-cell" title="0.70 (id:0.46, order:0.75, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
<td>0.70</td>
<td>Terpene</td>
<td>asperterpenoid A</td>
<td>Talaromyces wortmannii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001893: 155-19434">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001893/index.html#r1c1" target="_new">BGC0001893</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Other</td>
<td>abscisic acid</td>
<td>Botrytis cinerea B05.10</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002736: 3-5057">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002736/index.html#r1c1" target="_new">BGC0002736</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Terpene</td>
<td>preaspterpenacid I</td>
<td>Aspergillus terreus NIH2624</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002162: 0-5465">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002162/index.html#r1c1" target="_new">BGC0002162</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.42, order:0.68, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Terpene</td>
<td>sesterfisherol, sesterfisheric acid, compound 6, compound 7a, compound 5, compound 8</td>
<td>Aspergillus fischeri NRRL 181</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001604: 5569-24234">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001604/index.html#r1c1" target="_new">BGC0001604</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.48, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Terpene</td>
<td>gibberellin</td>
<td>Fusarium fujikuroi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000670: 82-32651">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000670/index.html#r1c1" target="_new">BGC0000670</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Terpene</td>
<td>thalianol, thaliandiol</td>
<td>Arabidopsis thaliana</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002221: 391-40725">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002221/index.html#r1c1" target="_new">BGC0002221</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.45, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>Terpene</td>
<td>ent-kauren-16alpha-ol</td>
<td>Serpula lacrymans var. lacrymans S7.9</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001475: 5-9618">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001475/index.html#r1c1" target="_new">BGC0001475</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>Terpene</td>
<td>aspterric acid</td>
<td>Aspergillus terreus NIH2624</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r59c2-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-556"></div> <span class="help-tooltip" id="cb-general-help-556">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r59c2-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r59c2_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r59c2_1.svg">
NC_072348 (5283352-5357754): Talaromyces marneffei chromosome 1, complete seq... (12% of genes show similarity), NRPS-like,terpene
</option>

<option value = "svg/clusterblast_r59c2_2.svg">
NW_026054585 (158434-189521): Daldinia decipiens strain CBS 113046 unplaced g... (25% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r59c2_3.svg">
NC_049563 (446618-479408): Talaromyces rugulosus chromosome III, complete seq... (20% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r59c2_4.svg">
NW_022985559 (271888-304261): Pseudovirgaria hyperparasitica strain CBS 12173... (37% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r59c2_5.svg">
NW_022983490 (330572-424081): Aaosphaeria arxii CBS 175.79 unplaced genomic s... (9% of genes show similarity), T1PKS,fungal-RiPP-like,terpene
</option>

<option value = "svg/clusterblast_r59c2_6.svg">
NW_025091271 (301767-334078): Aspergillus udagawae IFM 46973 DNA, scaffold 8,... (13% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r59c2_7.svg">
NW_020291503 (31867-64708): Aspergillus aculeatinus CBS 121060 unplaced genom... (22% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r59c2_8.svg">
NW_022987219 (1855256-1946234): Trematosphaeria pertusa strain CBS 122368 unp... (5% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r59c2_9.svg">
NW_025953788 (1126759-1218489): Annulohypoxylon truncatum strain CBS 140777 u... (6% of genes show similarity), NRPS,indole,terpene
</option>

<option value = "svg/clusterblast_r59c2_10.svg">
NW_022156722 (428067-459748): Pyricularia grisea strain NI907 chromosome V ma... (22% of genes show similarity), indole
</option>

</select>
<button id= "clusterblast-r59c2-download">Download graphic</button>
</div>
<div id="clusterblast-r59c2-svg"></div>

</div>
</div>

<div class="body-details-section r59c2-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-557"></div> <span class="help-tooltip" id="cb-known-help-557">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r59c2-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-558"></div> <span class="help-tooltip" id="cb-sub-help-558">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r59c2-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-560"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-560">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r59c2-pfam-details-svg" data-filename="scaffold_59_r59c2_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r59c2-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">









</div>









</div>

</div>

</div>





<div class="page" id='r63c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_63 - Region 1 - fungal-RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-849"></div> <span class="help-tooltip" id="region_svg-help-849">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_63.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r63c1-svg" data-filename="scaffold_63_r63c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1 - 49,420 nt. (total: 49,420 nt)

<a class="cluster-rules-header" id="r63c1-rules-header" href="#r63c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r63c1-rules">fungal-RiPP-like: DUF3328</div>

<div class="region-svg-container">
<div id='r63c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-850"></div> <span class="help-tooltip" id="focus-panel-help-850">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r63c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r63c1-gene-table-details" data-name="r63c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r63c1-MIBiG-cluster-compare" data-name="r63c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r63c1-clusterblast" data-name="r63c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r63c1-knownclusterblast" data-name="r63c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r63c1-subclusterblast" data-name="r63c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r63c1-pfam-details" data-name="r63c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r63c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-565"></div> <span class="help-tooltip" id="gt-help-tooltip-help-565">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r63c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r63c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r63c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r63c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r63c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-561"></div> <span class="help-tooltip" id="MIBiG-help-561">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r63c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r63c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r63c1"></div>
<div id="comparison-MIBiG-r63c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">fungal-RiPP-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002557: 0-14827">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002557/index.html#r1c1" target="_new">BGC0002557</a></td>





<td class="cc-heat-cell" title="0.45 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>


<td>0.15</td>
<td>Terpene</td>

<td>penifulvin A</td>


<td>Penicillium griseofulvum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001768: 0-28114">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001768/index.html#r1c1" target="_new">BGC0001768</a></td>





<td class="cc-heat-cell" title="0.43 (id:0.55, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>


<td>0.14</td>
<td>NRP</td>

<td>sansalvamide</td>


<td>[Nectria] haematococca mpVI 77-13-4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001868: 0-11041">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001868/index.html#r1c1" target="_new">BGC0001868</a></td>





<td class="cc-heat-cell" title="0.40 (id:0.42, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>


<td>0.12</td>
<td>Other (Phenazine)</td>

<td>iodinin</td>


<td>Xenorhabdus szentirmaii DSM 16338</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000678: 0-13437">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000678/index.html#r1c1" target="_new">BGC0000678</a></td>





<td class="cc-heat-cell" title="0.39 (id:0.41, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>


<td>0.12</td>
<td>Terpene</td>

<td>pentalenolactone</td>


<td>Streptomyces avermitilis MA-4680 = NBRC 14893</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001882: 0-43357">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001882/index.html#r1c1" target="_new">BGC0001882</a></td>





<td class="cc-heat-cell" title="0.39 (id:0.42, order:N/A, components:0.36)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>


<td>0.11</td>
<td>Polyketide</td>

<td>chrysoxanthone A, chrysoxanthone B, chrysoxanthone C</td>


<td>Penicillium rubens Wisconsin 54-1255</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000121: 0-36242">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000121/index.html#r1c1" target="_new">BGC0000121</a></td>





<td class="cc-heat-cell" title="0.39 (id:0.44, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>


<td>0.11</td>
<td>Polyketide</td>

<td>RES-1214-2</td>


<td>Pestalotiopsis fici</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002749: 1722-17470">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002749/index.html#r1c1" target="_new">BGC0002749</a></td>





<td class="cc-heat-cell" title="0.38 (id:0.44, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>


<td>0.11</td>
<td>NRP, Polyketide</td>

<td>aspcandine</td>


<td>Aspergillus candidus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000048: 0-52827">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000048/index.html#r1c1" target="_new">BGC0000048</a></td>





<td class="cc-heat-cell" title="0.38 (id:0.43, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>


<td>0.11</td>
<td>Polyketide</td>

<td>dothistromin</td>


<td>Dothistroma septosporum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002210: 100-28102">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002210/index.html#r1c1" target="_new">BGC0002210</a></td>





<td class="cc-heat-cell" title="0.38 (id:0.43, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>


<td>0.11</td>
<td>Polyketide, NRP</td>

<td>pyranoviolin A</td>


<td>Aspergillus violaceofuscus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001255: 1908-38951">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001255/index.html#r1c1" target="_new">BGC0001255</a></td>





<td class="cc-heat-cell" title="0.37 (id:0.42, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>


<td>0.11</td>
<td>NRP, Polyketide</td>

<td>equisetin</td>


<td>Fusarium heterosporum</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r63c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002557: 0-14827">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002557/index.html#r1c1" target="_new">BGC0002557</a></td>


<td class="cc-heat-cell" title="0.45 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
<td>0.45</td>
<td>Terpene</td>
<td>penifulvin A</td>
<td>Penicillium griseofulvum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001768: 0-28114">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001768/index.html#r1c1" target="_new">BGC0001768</a></td>


<td class="cc-heat-cell" title="0.43 (id:0.55, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
<td>0.43</td>
<td>NRP</td>
<td>sansalvamide</td>
<td>[Nectria] haematococca mpVI 77-13-4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001868: 0-11041">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001868/index.html#r1c1" target="_new">BGC0001868</a></td>


<td class="cc-heat-cell" title="0.40 (id:0.42, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>
<td>0.40</td>
<td>Other (Phenazine)</td>
<td>iodinin</td>
<td>Xenorhabdus szentirmaii DSM 16338</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000678: 0-13437">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000678/index.html#r1c1" target="_new">BGC0000678</a></td>


<td class="cc-heat-cell" title="0.39 (id:0.41, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
<td>0.39</td>
<td>Terpene</td>
<td>pentalenolactone</td>
<td>Streptomyces avermitilis MA-4680 = NBRC 14893</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001882: 0-43357">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001882/index.html#r1c1" target="_new">BGC0001882</a></td>


<td class="cc-heat-cell" title="0.39 (id:0.42, order:N/A, components:0.36)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
<td>0.39</td>
<td>Polyketide</td>
<td>chrysoxanthone A, chrysoxanthone B, chrysoxanthone C</td>
<td>Penicillium rubens Wisconsin 54-1255</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000121: 0-36242">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000121/index.html#r1c1" target="_new">BGC0000121</a></td>


<td class="cc-heat-cell" title="0.39 (id:0.44, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
<td>0.39</td>
<td>Polyketide</td>
<td>RES-1214-2</td>
<td>Pestalotiopsis fici</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002749: 1722-17470">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002749/index.html#r1c1" target="_new">BGC0002749</a></td>


<td class="cc-heat-cell" title="0.38 (id:0.44, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
<td>0.38</td>
<td>NRP, Polyketide</td>
<td>aspcandine</td>
<td>Aspergillus candidus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000048: 0-52827">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000048/index.html#r1c1" target="_new">BGC0000048</a></td>


<td class="cc-heat-cell" title="0.38 (id:0.43, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
<td>0.38</td>
<td>Polyketide</td>
<td>dothistromin</td>
<td>Dothistroma septosporum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002210: 100-28102">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002210/index.html#r1c1" target="_new">BGC0002210</a></td>


<td class="cc-heat-cell" title="0.38 (id:0.43, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
<td>0.38</td>
<td>Polyketide, NRP</td>
<td>pyranoviolin A</td>
<td>Aspergillus violaceofuscus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001255: 1908-38951">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001255/index.html#r1c1" target="_new">BGC0001255</a></td>


<td class="cc-heat-cell" title="0.37 (id:0.42, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>
<td>0.37</td>
<td>NRP, Polyketide</td>
<td>equisetin</td>
<td>Fusarium heterosporum</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r63c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-562"></div> <span class="help-tooltip" id="cb-general-help-562">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r63c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r63c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r63c1_1.svg">
NW_014574700 (98738-189740): Metarhizium brunneum ARSEF 3297 chromosome Unkno... (13% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r63c1_2.svg">
NC_049564 (1428105-1519058): Talaromyces rugulosus chromosome IV, complete se... (13% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r63c1_3.svg">
NW_017263613 (393044-529644): Mollisia scopiformis unplaced genomic scaffold ... (7% of genes show similarity), fungal-RiPP-like,other
</option>

<option value = "svg/clusterblast_r63c1_4.svg">
NC_035791 (818908-980602): Pochonia chlamydosporia 170 chromosome 2, whole ge... (7% of genes show similarity), NAPAA,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r63c1_5.svg">
NW_025953045 (987687-1078609): Daldinia caldariorum strain CBS 122874 unplace... (16% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r63c1_6.svg">
NW_026054574 (292730-383645): Daldinia decipiens strain CBS 113046 unplaced g... (14% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r63c1_7.svg">
NW_022985444 (916701-1007319): Mytilinidion resinicola strain CBS 304.34 unpl... (8% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r63c1_8.svg">
NW_020194479 (261473-327722): Amorphotheca resinae ATCC 22711 unplaced genomi... (18% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r63c1_9.svg">
NW_007360991 (989547-1160448): Glarea lozoyensis ATCC 20868 chromosome Unknow... (4% of genes show similarity), T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r63c1_10.svg">
NW_025091284 (4485141-4565194): Aspergillus pseudoviridinutans IFM 55266 DNA,... (12% of genes show similarity), T1PKS,indole
</option>

</select>
<button id= "clusterblast-r63c1-download">Download graphic</button>
</div>
<div id="clusterblast-r63c1-svg"></div>

</div>
</div>

<div class="body-details-section r63c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-563"></div> <span class="help-tooltip" id="cb-known-help-563">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r63c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-564"></div> <span class="help-tooltip" id="cb-sub-help-564">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r63c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-566"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-566">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r63c1-pfam-details-svg" data-filename="scaffold_63_r63c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r63c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">











</div>











</div>

</div>

</div>





<div class="page" id='r64c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_64 - Region 1 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-851"></div> <span class="help-tooltip" id="region_svg-help-851">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_64.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r64c1-svg" data-filename="scaffold_64_r64c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 150,337 - 197,634 nt. (total: 47,298 nt)

<a class="cluster-rules-header" id="r64c1-rules-header" href="#r64c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r64c1-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r64c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-852"></div> <span class="help-tooltip" id="focus-panel-help-852">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r64c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r64c1-gene-table-details" data-name="r64c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r64c1-nrps_pks" data-name="r64c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r64c1-MIBiG-cluster-compare" data-name="r64c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r64c1-clusterblast" data-name="r64c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r64c1-knownclusterblast" data-name="r64c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r64c1-subclusterblast" data-name="r64c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r64c1-nrps-pks-bubbles" data-name="r64c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r64c1-pfam-details" data-name="r64c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r64c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-575"></div> <span class="help-tooltip" id="gt-help-tooltip-help-575">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r64c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r64c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r64c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r64c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r64c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-567"></div> <span class="help-tooltip" id="nrps-domain-help-567">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r64c1-details-svg" data-filename="scaffold_64_r64c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r64c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r64c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-568"></div> <span class="help-tooltip" id="MIBiG-help-568">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r64c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r64c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r64c1"></div>
<div id="comparison-MIBiG-r64c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002610: 0-6468">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002610/index.html#r1c1" target="_new">BGC0002610</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.34</td>
<td>Terpene</td>

<td>stellatic acid</td>


<td>Aspergillus stellatus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002228: 282-17978">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002228/index.html#r1c1" target="_new">BGC0002228</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.47, order:0.69, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.33</td>
<td>NRP</td>

<td>CIML B, CIML A, CIML D, CIML C</td>


<td>Colletotrichum incanum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002233: 556-31132">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002233/index.html#r1c1" target="_new">BGC0002233</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.30</td>
<td>Polyketide</td>

<td>trichoxide</td>


<td>Trichoderma virens Gv29-8</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002211: 74-16330">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002211/index.html#r1c1" target="_new">BGC0002211</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.47, order:0.57, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.28</td>
<td>Polyketide</td>

<td>HEx-pks1 polyketide</td>


<td>Paraphaeosphaeria sporulosa</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001124: 3228-30082">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001124/index.html#r1c1" target="_new">BGC0001124</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.28</td>
<td>Polyketide</td>

<td>pyranonigrin E</td>


<td>Aspergillus niger ATCC 1015</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002234: 0-20938">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002234/index.html#r1c1" target="_new">BGC0002234</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.43, order:N/A, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.27</td>
<td>Polyketide</td>

<td>flavoglaucin</td>


<td>Aspergillus ruber CBS 135680</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002729: 0-27318">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002729/index.html#r1c1" target="_new">BGC0002729</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.43, order:N/A, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.27</td>
<td>Polyketide</td>

<td>sordarial</td>


<td>Neurospora crassa OR74A</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002180: 263-24876">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002180/index.html#r1c1" target="_new">BGC0002180</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.42, order:N/A, components:0.85)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.27</td>
<td>Polyketide</td>

<td>(+)-asperlin, (2Z,4Z,6E)-octa-2,4,6-trienoic acid</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002227: 270-18409">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002227/index.html#r1c1" target="_new">BGC0002227</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.47, order:0.57, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.26</td>
<td>NRP</td>

<td>AKML B, AKML D, AKML A, AKML C</td>


<td>Aspergillus luchuensis IFO 4308</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000046: 1267-21753">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000046/index.html#r1c1" target="_new">BGC0000046</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.26</td>
<td>Polyketide</td>

<td>depudecin</td>


<td>Alternaria brassicicola</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r64c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002610: 0-6468">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002610/index.html#r1c1" target="_new">BGC0002610</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Terpene</td>
<td>stellatic acid</td>
<td>Aspergillus stellatus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002228: 282-17978">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002228/index.html#r1c1" target="_new">BGC0002228</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.47, order:0.69, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>NRP</td>
<td>CIML B, CIML A, CIML D, CIML C</td>
<td>Colletotrichum incanum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002233: 556-31132">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002233/index.html#r1c1" target="_new">BGC0002233</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>Polyketide</td>
<td>trichoxide</td>
<td>Trichoderma virens Gv29-8</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002211: 74-16330">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002211/index.html#r1c1" target="_new">BGC0002211</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.47, order:0.57, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>Polyketide</td>
<td>HEx-pks1 polyketide</td>
<td>Paraphaeosphaeria sporulosa</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001124: 3228-30082">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001124/index.html#r1c1" target="_new">BGC0001124</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>Polyketide</td>
<td>pyranonigrin E</td>
<td>Aspergillus niger ATCC 1015</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002234: 0-20938">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002234/index.html#r1c1" target="_new">BGC0002234</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.43, order:N/A, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>Polyketide</td>
<td>flavoglaucin</td>
<td>Aspergillus ruber CBS 135680</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002729: 0-27318">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002729/index.html#r1c1" target="_new">BGC0002729</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.43, order:N/A, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>Polyketide</td>
<td>sordarial</td>
<td>Neurospora crassa OR74A</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002180: 263-24876">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002180/index.html#r1c1" target="_new">BGC0002180</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.42, order:N/A, components:0.85)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>Polyketide</td>
<td>(+)-asperlin, (2Z,4Z,6E)-octa-2,4,6-trienoic acid</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002227: 270-18409">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002227/index.html#r1c1" target="_new">BGC0002227</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.47, order:0.57, components:0.79)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>NRP</td>
<td>AKML B, AKML D, AKML A, AKML C</td>
<td>Aspergillus luchuensis IFO 4308</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000046: 1267-21753">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000046/index.html#r1c1" target="_new">BGC0000046</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>Polyketide</td>
<td>depudecin</td>
<td>Alternaria brassicicola</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r64c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-569"></div> <span class="help-tooltip" id="cb-general-help-569">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r64c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r64c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r64c1_1.svg">
NW_022474205 (7019281-7087036): Venustampulla echinocandica strain BP 5553 ch... (23% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r64c1_2.svg">
NW_020194484 (7340-83776): Amorphotheca resinae ATCC 22711 unplaced genomic s... (29% of genes show similarity), T1PKS,terpene
</option>

<option value = "svg/clusterblast_r64c1_3.svg">
NW_026249941 (3336813-3404118): Alternaria arbusti strain BMP 1465 chromosome... (33% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r64c1_4.svg">
NW_006917095 (2015395-2129171): Pestalotiopsis fici W106-1 unplaced genomic s... (25% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r64c1_5.svg">
NW_026249875 (10804-78109): Alternaria incomplexa strain BMP 0042 chromosome ... (30% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r64c1_6.svg">
NW_026055699 (512819-580124): Alternaria ventricosa strain BMP 2768 chromosom... (28% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r64c1_7.svg">
NW_017971434 (326355-393983): Talaromyces atroroseus strain IBT 11181 chromos... (14% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r64c1_8.svg">
NW_017253364 (509203-576723): Paraphaeosphaeria sporulosa strain AP3s5-JAC2a ... (14% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r64c1_9.svg">
NW_021167084 (1837567-1907553): Sodiomyces alkalinus F11 unplaced genomic sca... (16% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r64c1_10.svg">
NW_023336280 (2271165-2352192): Aspergillus tubingensis WU-2223L DNA, scaffol... (23% of genes show similarity), T1PKS
</option>

</select>
<button id= "clusterblast-r64c1-download">Download graphic</button>
</div>
<div id="clusterblast-r64c1-svg"></div>

</div>
</div>

<div class="body-details-section r64c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-570"></div> <span class="help-tooltip" id="cb-known-help-570">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r64c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r64c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r64c1_1.svg">
HEx-pks1 polyketide
</option>

</select>
<button id="knownclusterblast-r64c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r64c1-svg"></div>

</div>
</div>

<div class="body-details-section r64c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-571"></div> <span class="help-tooltip" id="cb-sub-help-571">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r64c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-574"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-574">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r64c1-domain-bubble-svg" data-filename="scaffold_64_r64c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r64c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (150336 - 197634): single T1PKS
</option>


</select>

<div class="details-svg" id="r64c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r64c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r64c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-576"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-576">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r64c1-pfam-details-svg" data-filename="scaffold_64_r64c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r64c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r64c1-nrps_pks_products" data-name="r64c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r64c1-nrps_pks_monomers" data-name="r64c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r64c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-572"></div> <span class="help-tooltip" id="nrps-structures-help-572">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 150336 - 197634:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r64c1-1" class="smiles-canvas" data-smiles="C([*])C(-O)"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(pk)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r64c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r64c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-573"></div> <span class="help-tooltip" id="nrps-monomers-help-573">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">FUN_008740</span></strong>: pk
<div class="collapser collapser-target-FUN_008740 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (470..772)</strong>: pk
<div class="collapser collapser-target-FUN_008740 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-FUN_008740-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 79.2%</dd>
<dd>Methylmalonyl-CoA: 66.7%</dd>
<dd>Methoxymalonyl-CoA: 58.3%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-FUN_008740-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 81.6</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 71.9</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 47.7</dt>

<dd></dd><dt>Malonyl-CoA: 47.0</dt>

<dd></dd><dt>Propionyl-CoA: 45.1</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 31.5</dt>

<dd></dd><dt>Isobutyryl-CoA: 26.6</dt>

<dd></dd><dt>fatty_acid: 23.2</dt>

<dd></dd><dt>inactive: 19.6</dt>

<dd></dd><dt>trans-1,2-CPDA: 13.9</dt>

<dd></dd><dt>CHC-CoA: 12.8</dt>

<dd></dd><dt>Benzoyl-CoA: 10.4</dt>

<dd></dd><dt>Acetyl-CoA: 3.8</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>










</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>





<div class="page" id='r69c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_69 - Region 1 - NRPS-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-853"></div> <span class="help-tooltip" id="region_svg-help-853">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_69.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r69c1-svg" data-filename="scaffold_69_r69c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 27,797 - 71,095 nt. (total: 43,299 nt)

<a class="cluster-rules-header" id="r69c1-rules-header" href="#r69c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r69c1-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r69c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-854"></div> <span class="help-tooltip" id="focus-panel-help-854">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r69c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r69c1-gene-table-details" data-name="r69c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r69c1-nrps_pks" data-name="r69c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r69c1-MIBiG-cluster-compare" data-name="r69c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r69c1-clusterblast" data-name="r69c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r69c1-knownclusterblast" data-name="r69c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r69c1-subclusterblast" data-name="r69c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r69c1-pfam-details" data-name="r69c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r69c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-583"></div> <span class="help-tooltip" id="gt-help-tooltip-help-583">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r69c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r69c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r69c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r69c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r69c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-577"></div> <span class="help-tooltip" id="nrps-domain-help-577">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r69c1-details-svg" data-filename="scaffold_69_r69c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r69c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r69c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-578"></div> <span class="help-tooltip" id="MIBiG-help-578">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r69c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r69c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r69c1"></div>
<div id="comparison-MIBiG-r69c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000099: 0-8104">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000099/index.html#r1c1" target="_new">BGC0000099</a></td>





<td class="cc-heat-cell" title="0.42 (id:0.43, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>


<td>0.13</td>
<td>Polyketide</td>

<td>monascorubrin</td>


<td>Talaromyces marneffei</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001338: 273-42813">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001338/index.html#r1c1" target="_new">BGC0001338</a></td>





<td class="cc-heat-cell" title="0.26 (id:0.43, order:N/A, components:0.16)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>


<td>0.06</td>
<td>Polyketide</td>

<td>citrinin</td>


<td>Monascus ruber</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001722: 0-18441">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001722/index.html#r1c1" target="_new">BGC0001722</a></td>





<td class="cc-heat-cell" title="0.25 (id:0.45, order:N/A, components:0.14)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>


<td>0.06</td>
<td>Polyketide</td>

<td>aspernidine A</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002212: 0-24132">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002212/index.html#r1c1" target="_new">BGC0002212</a></td>





<td class="cc-heat-cell" title="0.24 (id:0.42, order:N/A, components:0.14)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.24), rgb(0, 0, 0, 0.24) 24%, white 24%)"></td>


<td>0.06</td>
<td>Polyketide</td>

<td>HEx-pks15 polyketide</td>


<td>Moniliophthora roreri MCA 2997</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001626: 0-40540">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001626/index.html#r1c1" target="_new">BGC0001626</a></td>





<td class="cc-heat-cell" title="0.24 (id:0.42, order:N/A, components:0.14)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.24), rgb(0, 0, 0, 0.24) 24%, white 24%)"></td>


<td>0.06</td>
<td>Polyketide</td>

<td>isoindolinone</td>


<td>Stachybotrys chlorohalonata IBT 40285</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002255: 803-34333">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002255/index.html#r1c1" target="_new">BGC0002255</a></td>





<td class="cc-heat-cell" title="0.23 (id:0.44, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>


<td>0.05</td>
<td>Polyketide</td>

<td>lijiquinone</td>


<td>Ascomycota sp. F53</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001242: 200-22183">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001242/index.html#r1c1" target="_new">BGC0001242</a></td>





<td class="cc-heat-cell" title="0.23 (id:0.42, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>


<td>0.05</td>
<td>Polyketide</td>

<td>oxyjavanicin</td>


<td>Fusarium fujikuroi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001404: 0-28758">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001404/index.html#r1c1" target="_new">BGC0001404</a></td>





<td class="cc-heat-cell" title="0.23 (id:0.43, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>


<td>0.05</td>
<td>Polyketide</td>

<td>sorbicillin</td>


<td>Penicillium rubens Wisconsin 54-1255</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002209: 560-36987">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002209/index.html#r1c1" target="_new">BGC0002209</a></td>





<td class="cc-heat-cell" title="0.23 (id:0.42, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>


<td>0.05</td>
<td>Polyketide</td>

<td>ustethylin A</td>


<td>Aspergillus ustus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002734: 207-20832">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002734/index.html#r1c1" target="_new">BGC0002734</a></td>





<td class="cc-heat-cell" title="0.22 (id:0.41, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.22), rgb(0, 0, 0, 0.22) 22%, white 22%)"></td>


<td>0.05</td>
<td>Polyketide</td>

<td>2,4-dihydroxy-5,6-dimethyl benzaldehyde</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r69c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000099: 0-8104">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000099/index.html#r1c1" target="_new">BGC0000099</a></td>


<td class="cc-heat-cell" title="0.42 (id:0.43, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
<td>0.42</td>
<td>Polyketide</td>
<td>monascorubrin</td>
<td>Talaromyces marneffei</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001338: 273-42813">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001338/index.html#r1c1" target="_new">BGC0001338</a></td>


<td class="cc-heat-cell" title="0.26 (id:0.43, order:N/A, components:0.16)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>
<td>0.26</td>
<td>Polyketide</td>
<td>citrinin</td>
<td>Monascus ruber</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001722: 0-18441">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001722/index.html#r1c1" target="_new">BGC0001722</a></td>


<td class="cc-heat-cell" title="0.25 (id:0.45, order:N/A, components:0.14)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>
<td>0.25</td>
<td>Polyketide</td>
<td>aspernidine A</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002212: 0-24132">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002212/index.html#r1c1" target="_new">BGC0002212</a></td>


<td class="cc-heat-cell" title="0.24 (id:0.42, order:N/A, components:0.14)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.24), rgb(0, 0, 0, 0.24) 24%, white 24%)"></td>
<td>0.24</td>
<td>Polyketide</td>
<td>HEx-pks15 polyketide</td>
<td>Moniliophthora roreri MCA 2997</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001626: 0-40540">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001626/index.html#r1c1" target="_new">BGC0001626</a></td>


<td class="cc-heat-cell" title="0.24 (id:0.42, order:N/A, components:0.14)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.24), rgb(0, 0, 0, 0.24) 24%, white 24%)"></td>
<td>0.24</td>
<td>Polyketide</td>
<td>isoindolinone</td>
<td>Stachybotrys chlorohalonata IBT 40285</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002255: 803-34333">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002255/index.html#r1c1" target="_new">BGC0002255</a></td>


<td class="cc-heat-cell" title="0.23 (id:0.44, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>
<td>0.23</td>
<td>Polyketide</td>
<td>lijiquinone</td>
<td>Ascomycota sp. F53</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001242: 200-22183">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001242/index.html#r1c1" target="_new">BGC0001242</a></td>


<td class="cc-heat-cell" title="0.23 (id:0.42, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>
<td>0.23</td>
<td>Polyketide</td>
<td>oxyjavanicin</td>
<td>Fusarium fujikuroi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001404: 0-28758">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001404/index.html#r1c1" target="_new">BGC0001404</a></td>


<td class="cc-heat-cell" title="0.23 (id:0.43, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>
<td>0.23</td>
<td>Polyketide</td>
<td>sorbicillin</td>
<td>Penicillium rubens Wisconsin 54-1255</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002209: 560-36987">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002209/index.html#r1c1" target="_new">BGC0002209</a></td>


<td class="cc-heat-cell" title="0.23 (id:0.42, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>
<td>0.23</td>
<td>Polyketide</td>
<td>ustethylin A</td>
<td>Aspergillus ustus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002734: 207-20832">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002734/index.html#r1c1" target="_new">BGC0002734</a></td>


<td class="cc-heat-cell" title="0.22 (id:0.41, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.22), rgb(0, 0, 0, 0.22) 22%, white 22%)"></td>
<td>0.22</td>
<td>Polyketide</td>
<td>2,4-dihydroxy-5,6-dimethyl benzaldehyde</td>
<td>Aspergillus terreus NIH2624</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r69c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-579"></div> <span class="help-tooltip" id="cb-general-help-579">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r69c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r69c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r69c1_1.svg">
NC_049566 (4267202-4334038): Talaromyces rugulosus chromosome VI, complete se... (9% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r69c1_2.svg">
NC_054698 (2427124-2490428): Aspergillus flavus NRRL3357 chromosome 8, comple... (9% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r69c1_3.svg">
NC_036442 (2396468-2459772): Aspergillus oryzae RIB40 DNA, chromosome 8 (10% of genes show similarity), NRPS-like
</option>

</select>
<button id= "clusterblast-r69c1-download">Download graphic</button>
</div>
<div id="clusterblast-r69c1-svg"></div>

</div>
</div>

<div class="body-details-section r69c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-580"></div> <span class="help-tooltip" id="cb-known-help-580">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r69c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-581"></div> <span class="help-tooltip" id="cb-sub-help-581">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r69c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-584"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-584">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r69c1-pfam-details-svg" data-filename="scaffold_69_r69c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r69c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r69c1-nrps_pks_monomers" data-name="r69c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>




</div>










<div class="sidepanel-details-section r69c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-582"></div> <span class="help-tooltip" id="nrps-monomers-help-582">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">FUN_008935</span></strong>: X
<div class="collapser collapser-target-FUN_008935 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (19..411)</strong>: X
<div class="collapser collapser-target-FUN_008935 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr
<div class="collapser collapser-target-FUN_008935-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">FGAFGSIVHK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>

<dt>Stachelhaus code match:</dt>
<dd>0% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

</dl>
</div>
</div>




</div>

</div>

</div>

<div class="page" id='r69c2' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_69 - Region 2 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-855"></div> <span class="help-tooltip" id="region_svg-help-855">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_69.region002.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r69c2-svg" data-filename="scaffold_69_r69c2.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 90,072 - 138,331 nt. (total: 48,260 nt)

<a class="cluster-rules-header" id="r69c2-rules-header" href="#r69c2">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r69c2-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r69c2-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-856"></div> <span class="help-tooltip" id="focus-panel-help-856">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r69c2">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r69c2-gene-table-details" data-name="r69c2-gene-table-details">
Gene overview
</div>



<div class="body-details-header r69c2-nrps_pks" data-name="r69c2-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r69c2-MIBiG-cluster-compare" data-name="r69c2-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r69c2-clusterblast" data-name="r69c2-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r69c2-knownclusterblast" data-name="r69c2-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r69c2-subclusterblast" data-name="r69c2-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r69c2-nrps-pks-bubbles" data-name="r69c2-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r69c2-pfam-details" data-name="r69c2-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r69c2-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-593"></div> <span class="help-tooltip" id="gt-help-tooltip-help-593">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r69c2">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r69c2-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r69c2-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r69c2">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r69c2-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-585"></div> <span class="help-tooltip" id="nrps-domain-help-585">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r69c2-details-svg" data-filename="scaffold_69_r69c2_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r69c2-details-svg"></div>
</div>
</div>



<div class="body-details-section r69c2-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-586"></div> <span class="help-tooltip" id="MIBiG-help-586">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r69c2-selector">Analysis type:</label>
<select id="comparison-MIBiG-r69c2-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r69c2"></div>
<div id="comparison-MIBiG-r69c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000099: 0-8104">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000099/index.html#r1c1" target="_new">BGC0000099</a></td>





<td class="cc-heat-cell" title="0.71 (id:0.50, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>


<td>0.38</td>
<td>Polyketide</td>

<td>monascorubrin</td>


<td>Talaromyces marneffei</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002429: 0-14296">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002429/index.html#r1c1" target="_new">BGC0002429</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.29</td>
<td>Terpene, Polyketide</td>

<td>higginsianin B</td>


<td>Colletotrichum higginsianum IMI 349063</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002427: 0-21541">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002427/index.html#r1c1" target="_new">BGC0002427</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.43, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.25</td>
<td>Terpene, Polyketide</td>

<td>subglutinol A, subglutinol B</td>


<td>Metarhizium robertsii ARSEF 23</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000013: 0-7849">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000013/index.html#r1c1" target="_new">BGC0000013</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.21</td>
<td>Polyketide</td>

<td>alternariol</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002236: 673-11427">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002236/index.html#r1c1" target="_new">BGC0002236</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.21</td>
<td>Polyketide</td>

<td>8-methyldiaporthin</td>


<td>Aspergillus oryzae RIB40</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001284: 0-5292">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001284/index.html#r1c1" target="_new">BGC0001284</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.21</td>
<td>Polyketide</td>

<td>alternariol</td>


<td>Parastagonospora nodorum SN15</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001583: 3200-15837">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001583/index.html#r1c1" target="_new">BGC0001583</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>Polyketide</td>

<td>emodin</td>


<td>Escovopsis weberi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002483: 1330-45854">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002483/index.html#r1c1" target="_new">BGC0002483</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.44, order:N/A, components:0.63)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>Polyketide</td>

<td>usnic acid</td>


<td>Cladonia uncialis subsp. uncialis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002603: 0-17042">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002603/index.html#r1c1" target="_new">BGC0002603</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>Polyketide</td>

<td>chevalone E</td>


<td>Aspergillus versicolor</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002596: 0-33285">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002596/index.html#r1c1" target="_new">BGC0002596</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>Polyketide</td>

<td>lecanoric acid, orsellinic acid</td>


<td>Claviceps purpurea 20.1</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r69c2-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000099: 0-8104">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000099/index.html#r1c1" target="_new">BGC0000099</a></td>


<td class="cc-heat-cell" title="0.71 (id:0.50, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
<td>0.71</td>
<td>Polyketide</td>
<td>monascorubrin</td>
<td>Talaromyces marneffei</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002429: 0-14296">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002429/index.html#r1c1" target="_new">BGC0002429</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>Terpene, Polyketide</td>
<td>higginsianin B</td>
<td>Colletotrichum higginsianum IMI 349063</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002427: 0-21541">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002427/index.html#r1c1" target="_new">BGC0002427</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.43, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Terpene, Polyketide</td>
<td>subglutinol A, subglutinol B</td>
<td>Metarhizium robertsii ARSEF 23</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000013: 0-7849">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000013/index.html#r1c1" target="_new">BGC0000013</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Polyketide</td>
<td>alternariol</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002236: 673-11427">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002236/index.html#r1c1" target="_new">BGC0002236</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Polyketide</td>
<td>8-methyldiaporthin</td>
<td>Aspergillus oryzae RIB40</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001284: 0-5292">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001284/index.html#r1c1" target="_new">BGC0001284</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Polyketide</td>
<td>alternariol</td>
<td>Parastagonospora nodorum SN15</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001583: 3200-15837">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001583/index.html#r1c1" target="_new">BGC0001583</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Polyketide</td>
<td>emodin</td>
<td>Escovopsis weberi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002483: 1330-45854">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002483/index.html#r1c1" target="_new">BGC0002483</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.44, order:N/A, components:0.63)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Polyketide</td>
<td>usnic acid</td>
<td>Cladonia uncialis subsp. uncialis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002603: 0-17042">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002603/index.html#r1c1" target="_new">BGC0002603</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Polyketide</td>
<td>chevalone E</td>
<td>Aspergillus versicolor</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002596: 0-33285">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002596/index.html#r1c1" target="_new">BGC0002596</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Polyketide</td>
<td>lecanoric acid, orsellinic acid</td>
<td>Claviceps purpurea 20.1</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r69c2-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-587"></div> <span class="help-tooltip" id="cb-general-help-587">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r69c2-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r69c2_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r69c2_1.svg">
NC_054391 (86002-153903): Drechmeria coniospora strain ARSEF 6962 chromosome ... (23% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r69c2_2.svg">
NW_025953251 (1891693-2006513): Annulohypoxylon maeteangense strain CBS 12383... (13% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r69c2_3.svg">
NW_026250016 (757881-825808): Hypoxylon trugodes strain CBS 135444 unplaced g... (22% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r69c2_4.svg">
NW_025953254 (1383196-1451301): Annulohypoxylon maeteangense strain CBS 12383... (20% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r69c2_5.svg">
NW_025953832 (31975-100050): Annulohypoxylon truncatum strain CBS 140777 unpl... (19% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r69c2_6.svg">
NC_066264 (1638236-1719104): Aspergillus nidulans FGSC A4 chromosome VIII (16% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r69c2_7.svg">
NW_022984958 (367172-435010): Dissoconium aciculare CBS 342.82 unplaced genom... (14% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r69c2_8.svg">
NW_025091287 (155600-237358): Aspergillus pseudoviridinutans IFM 55266 DNA, s... (14% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r69c2_9.svg">
NW_025953791 (1196257-1261596): Annulohypoxylon truncatum strain CBS 140777 u... (29% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r69c2_10.svg">
NW_025953052 (583133-651040): Daldinia caldariorum strain CBS 122874 unplaced... (16% of genes show similarity), T1PKS
</option>

</select>
<button id= "clusterblast-r69c2-download">Download graphic</button>
</div>
<div id="clusterblast-r69c2-svg"></div>

</div>
</div>

<div class="body-details-section r69c2-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-588"></div> <span class="help-tooltip" id="cb-known-help-588">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r69c2-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r69c2_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r69c2_1.svg">
eupenifeldin
</option>

<option value = "svg/knownclusterblast_r69c2_2.svg">
stipitatic acid
</option>

<option value = "svg/knownclusterblast_r69c2_3.svg">
xenovulene A
</option>

<option value = "svg/knownclusterblast_r69c2_4.svg">
monascorubrin
</option>

</select>
<button id="knownclusterblast-r69c2-download">Download graphic</button>
</div>
<div id="knownclusterblast-r69c2-svg"></div>

</div>
</div>

<div class="body-details-section r69c2-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-589"></div> <span class="help-tooltip" id="cb-sub-help-589">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r69c2-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-592"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-592">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r69c2-domain-bubble-svg" data-filename="scaffold_69_r69c2_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r69c2-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC2">
Candidate 2 (90071 - 138331): single T1PKS
</option>


</select>

<div class="details-svg" id="r69c2-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r69c2-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r69c2-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-594"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-594">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r69c2-pfam-details-svg" data-filename="scaffold_69_r69c2_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r69c2-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r69c2-nrps_pks_products" data-name="r69c2-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r69c2-nrps_pks_monomers" data-name="r69c2-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r69c2-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-590"></div> <span class="help-tooltip" id="nrps-structures-help-590">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 2, location 90071 - 138331:
<div class="collapser collapser-target-CC2 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r69c2-2" class="smiles-canvas" data-smiles="CC(=O)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(mal)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r69c2-2-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="CC(=O)C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r69c2-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-591"></div> <span class="help-tooltip" id="nrps-monomers-help-591">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">FUN_008945</span></strong>: mal
<div class="collapser collapser-target-FUN_008945 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>







<dd><strong>PKS_AT (905..1202)</strong>: mal
<div class="collapser collapser-target-FUN_008945 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-FUN_008945-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 75.0%</dd>
<dd>inactive: 58.3%</dd>
<dd>Ethylmalonyl-CoA: 54.2%</dd>
</dl>

</div><br>

minowa:  Malonyl-CoA
<div class="collapser collapser-target-FUN_008945-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Malonyl-CoA: 69.1</dt>

<dd></dd><dt>inactive: 63.5</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 48.4</dt>

<dd></dd><dt>Methylmalonyl-CoA: 41.7</dt>

<dd></dd><dt>CHC-CoA: 20.8</dt>

<dd></dd><dt>fatty_acid: 18.5</dt>

<dd></dd><dt>Propionyl-CoA: 18.2</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 18.0</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 18.0</dt>

<dd></dd><dt>Isobutyryl-CoA: 13.6</dt>

<dd></dd><dt>Benzoyl-CoA: 13.1</dt>

<dd></dd><dt>trans-1,2-CPDA: 12.4</dt>

<dd></dd><dt>Acetyl-CoA: 10.8</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 8.8</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>








</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>





<div class="page" id='r70c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_70 - Region 1 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-857"></div> <span class="help-tooltip" id="region_svg-help-857">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_70.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r70c1-svg" data-filename="scaffold_70_r70c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 148,313 - 169,677 nt. (total: 21,365 nt)

<a class="cluster-rules-header" id="r70c1-rules-header" href="#r70c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r70c1-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>

<div class="region-svg-container">
<div id='r70c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-858"></div> <span class="help-tooltip" id="focus-panel-help-858">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r70c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r70c1-gene-table-details" data-name="r70c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r70c1-MIBiG-cluster-compare" data-name="r70c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r70c1-clusterblast" data-name="r70c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r70c1-knownclusterblast" data-name="r70c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r70c1-subclusterblast" data-name="r70c1-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r70c1-pfam-details" data-name="r70c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r70c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-599"></div> <span class="help-tooltip" id="gt-help-tooltip-help-599">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r70c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r70c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r70c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r70c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r70c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-595"></div> <span class="help-tooltip" id="MIBiG-help-595">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r70c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r70c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r70c1"></div>
<div id="comparison-MIBiG-r70c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">terpene</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002260: 942-16838">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002260/index.html#r1c1" target="_new">BGC0002260</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>Terpene</td>

<td>trichobrasilenol, xylarenic acid B, brasilane A, brasilane F, brasilane E, brasilane D</td>


<td>Annulohypoxylon truncatum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002161: 1109-36992">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002161/index.html#r1c1" target="_new">BGC0002161</a></td>





<td class="cc-heat-cell" title="0.37 (id:0.41, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>


<td>0.11</td>
<td>Polyketide</td>

<td>scytalone, T3HN</td>


<td>Pestalotiopsis fici W106-1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001995: 0-27487">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001995/index.html#r1c1" target="_new">BGC0001995</a></td>





<td class="cc-heat-cell" title="0.32 (id:0.42, order:N/A, components:0.24)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.32), rgb(0, 0, 0, 0.32) 32%, white 32%)"></td>


<td>0.08</td>
<td>Terpene</td>

<td>heptelidic acid</td>


<td>Aspergillus oryzae RIB40</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002253: 0-33620">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002253/index.html#r1c1" target="_new">BGC0002253</a></td>





<td class="cc-heat-cell" title="0.31 (id:0.44, order:N/A, components:0.22)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>


<td>0.08</td>
<td>Polyketide</td>

<td>waikikiamide A, waikikiamide B, waikikiamide C, (+)-semivioxanthin, semivioxanthin</td>


<td>Aspergillus sp. FM242</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001197: 0-12679">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001197/index.html#r1c1" target="_new">BGC0001197</a></td>





<td class="cc-heat-cell" title="0.27 (id:0.43, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>


<td>0.07</td>
<td>Polyketide</td>

<td>frankiamicin</td>


<td>Frankia sp. EAN1pec</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002255: 803-34333">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002255/index.html#r1c1" target="_new">BGC0002255</a></td>





<td class="cc-heat-cell" title="0.26 (id:0.42, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>


<td>0.06</td>
<td>Polyketide</td>

<td>lijiquinone</td>


<td>Ascomycota sp. F53</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000234: 152-6978">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000234/index.html#r1c1" target="_new">BGC0000234</a></td>





<td class="cc-heat-cell" title="0.24 (id:0.42, order:N/A, components:0.14)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.24), rgb(0, 0, 0, 0.24) 24%, white 24%)"></td>


<td>0.06</td>
<td>Polyketide</td>

<td>jadomycin</td>


<td>Streptomyces venezuelae ATCC 10712</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000217: 2-8089">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000217/index.html#r1c1" target="_new">BGC0000217</a></td>





<td class="cc-heat-cell" title="0.24 (id:0.42, order:N/A, components:0.14)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.24), rgb(0, 0, 0, 0.24) 24%, white 24%)"></td>


<td>0.06</td>
<td>Polyketide</td>

<td>daunorubicin</td>


<td>Streptomyces sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001866: 346-32979">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001866/index.html#r1c1" target="_new">BGC0001866</a></td>





<td class="cc-heat-cell" title="0.24 (id:0.44, order:N/A, components:0.13)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.24), rgb(0, 0, 0, 0.24) 24%, white 24%)"></td>


<td>0.06</td>
<td>Polyketide</td>

<td>viriditoxin</td>


<td>Byssochlamys spectabilis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002062: 1607-32928">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002062/index.html#r1c1" target="_new">BGC0002062</a></td>





<td class="cc-heat-cell" title="0.24 (id:0.43, order:N/A, components:0.13)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.24), rgb(0, 0, 0, 0.24) 24%, white 24%)"></td>


<td>0.05</td>
<td>Polyketide</td>

<td>agnestin A, agnestin B</td>


<td>Paecilomyces divaricatus</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r70c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002260: 942-16838">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002260/index.html#r1c1" target="_new">BGC0002260</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>Terpene</td>
<td>trichobrasilenol, xylarenic acid B, brasilane A, brasilane F, brasilane E, brasilane D</td>
<td>Annulohypoxylon truncatum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002161: 1109-36992">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002161/index.html#r1c1" target="_new">BGC0002161</a></td>


<td class="cc-heat-cell" title="0.37 (id:0.41, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>
<td>0.37</td>
<td>Polyketide</td>
<td>scytalone, T3HN</td>
<td>Pestalotiopsis fici W106-1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001995: 0-27487">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001995/index.html#r1c1" target="_new">BGC0001995</a></td>


<td class="cc-heat-cell" title="0.32 (id:0.42, order:N/A, components:0.24)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.32), rgb(0, 0, 0, 0.32) 32%, white 32%)"></td>
<td>0.32</td>
<td>Terpene</td>
<td>heptelidic acid</td>
<td>Aspergillus oryzae RIB40</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002253: 0-33620">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002253/index.html#r1c1" target="_new">BGC0002253</a></td>


<td class="cc-heat-cell" title="0.31 (id:0.44, order:N/A, components:0.22)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>
<td>0.31</td>
<td>Polyketide</td>
<td>waikikiamide A, waikikiamide B, waikikiamide C, (+)-semivioxanthin, semivioxanthin</td>
<td>Aspergillus sp. FM242</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001197: 0-12679">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001197/index.html#r1c1" target="_new">BGC0001197</a></td>


<td class="cc-heat-cell" title="0.27 (id:0.43, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>
<td>0.27</td>
<td>Polyketide</td>
<td>frankiamicin</td>
<td>Frankia sp. EAN1pec</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002255: 803-34333">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002255/index.html#r1c1" target="_new">BGC0002255</a></td>


<td class="cc-heat-cell" title="0.26 (id:0.42, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.26), rgb(0, 0, 0, 0.26) 26%, white 26%)"></td>
<td>0.26</td>
<td>Polyketide</td>
<td>lijiquinone</td>
<td>Ascomycota sp. F53</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000234: 152-6978">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000234/index.html#r1c1" target="_new">BGC0000234</a></td>


<td class="cc-heat-cell" title="0.24 (id:0.42, order:N/A, components:0.14)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.24), rgb(0, 0, 0, 0.24) 24%, white 24%)"></td>
<td>0.24</td>
<td>Polyketide</td>
<td>jadomycin</td>
<td>Streptomyces venezuelae ATCC 10712</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000217: 2-8089">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000217/index.html#r1c1" target="_new">BGC0000217</a></td>


<td class="cc-heat-cell" title="0.24 (id:0.42, order:N/A, components:0.14)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.24), rgb(0, 0, 0, 0.24) 24%, white 24%)"></td>
<td>0.24</td>
<td>Polyketide</td>
<td>daunorubicin</td>
<td>Streptomyces sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001866: 346-32979">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001866/index.html#r1c1" target="_new">BGC0001866</a></td>


<td class="cc-heat-cell" title="0.24 (id:0.44, order:N/A, components:0.13)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.24), rgb(0, 0, 0, 0.24) 24%, white 24%)"></td>
<td>0.24</td>
<td>Polyketide</td>
<td>viriditoxin</td>
<td>Byssochlamys spectabilis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002062: 1607-32928">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002062/index.html#r1c1" target="_new">BGC0002062</a></td>


<td class="cc-heat-cell" title="0.24 (id:0.43, order:N/A, components:0.13)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.24), rgb(0, 0, 0, 0.24) 24%, white 24%)"></td>
<td>0.24</td>
<td>Polyketide</td>
<td>agnestin A, agnestin B</td>
<td>Paecilomyces divaricatus</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r70c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-596"></div> <span class="help-tooltip" id="cb-general-help-596">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r70c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r70c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r70c1_1.svg">
NW_026250016 (1759599-1791002): Hypoxylon trugodes strain CBS 135444 unplaced... (25% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r70c1_2.svg">
NW_025953793 (626237-657658): Annulohypoxylon truncatum strain CBS 140777 unp... (25% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r70c1_3.svg">
NW_025953251 (95788-127218): Annulohypoxylon maeteangense strain CBS 123835 u... (28% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r70c1_4.svg">
NW_025953054 (238858-270468): Daldinia caldariorum strain CBS 122874 unplaced... (25% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r70c1_5.svg">
NW_026054546 (415398-446788): Daldinia decipiens strain CBS 113046 unplaced g... (25% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r70c1_6.svg">
NW_025091218 (4642-192860): Aspergillus viridinutans IFM 47045 DNA, scaffold ... (3% of genes show similarity), NRPS-like,T1PKS,indole
</option>

</select>
<button id= "clusterblast-r70c1-download">Download graphic</button>
</div>
<div id="clusterblast-r70c1-svg"></div>

</div>
</div>

<div class="body-details-section r70c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-597"></div> <span class="help-tooltip" id="cb-known-help-597">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r70c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-598"></div> <span class="help-tooltip" id="cb-sub-help-598">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r70c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-600"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-600">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r70c1-pfam-details-svg" data-filename="scaffold_70_r70c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r70c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">









</div>









</div>

</div>

</div>





<div class="page" id='r76c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_76 - Region 1 - NRPS-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-859"></div> <span class="help-tooltip" id="region_svg-help-859">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_76.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r76c1-svg" data-filename="scaffold_76_r76c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 5,896 - 49,759 nt. (total: 43,864 nt)

<a class="cluster-rules-header" id="r76c1-rules-header" href="#r76c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r76c1-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r76c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-860"></div> <span class="help-tooltip" id="focus-panel-help-860">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r76c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r76c1-gene-table-details" data-name="r76c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r76c1-nrps_pks" data-name="r76c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r76c1-MIBiG-cluster-compare" data-name="r76c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r76c1-clusterblast" data-name="r76c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r76c1-knownclusterblast" data-name="r76c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r76c1-subclusterblast" data-name="r76c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r76c1-nrps-pks-bubbles" data-name="r76c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r76c1-pfam-details" data-name="r76c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r76c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-609"></div> <span class="help-tooltip" id="gt-help-tooltip-help-609">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r76c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r76c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r76c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r76c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r76c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-601"></div> <span class="help-tooltip" id="nrps-domain-help-601">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r76c1-details-svg" data-filename="scaffold_76_r76c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r76c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r76c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-602"></div> <span class="help-tooltip" id="MIBiG-help-602">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r76c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r76c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r76c1"></div>
<div id="comparison-MIBiG-r76c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001168: 0-4582">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001168/index.html#r1c1" target="_new">BGC0001168</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.32</td>
<td>NRP</td>

<td>livipeptin</td>


<td>Streptomyces lividans 1326</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002602: 406-14432">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002602/index.html#r1c1" target="_new">BGC0002602</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.43, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.25</td>
<td>Alkaloid</td>

<td>aspergillic acid</td>


<td>Aspergillus flavus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001516: 0-14978">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001516/index.html#r1c1" target="_new">BGC0001516</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.25</td>
<td>NRP</td>

<td>aspergillic acid</td>


<td>Aspergillus flavus NRRL3357</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002276: 0-3813">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002276/index.html#r1c1" target="_new">BGC0002276</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.55, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.24</td>
<td>NRP</td>

<td>choline</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001599: 0-9373">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001599/index.html#r1c1" target="_new">BGC0001599</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>NRP</td>

<td>fragin</td>


<td>Burkholderia cenocepacia H111</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002582: 0-7207">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002582/index.html#r1c1" target="_new">BGC0002582</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.21</td>
<td>NRP</td>

<td>guanipiperazine A, guanipiperazine B</td>


<td>Streptomyces chrestomyceticus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002278: 0-28520">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002278/index.html#r1c1" target="_new">BGC0002278</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.42, order:N/A, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.20</td>
<td>Alkaloid, NRP</td>

<td>brasiliamide I, brasiliamide A, brasiliamide B, 2,2-dibenzylpiperazine, brasiliamide D</td>


<td>Penicillium brasilianum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002286: 0-16374">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002286/index.html#r1c1" target="_new">BGC0002286</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>ririwpeptide A, ririwpeptide B, ririwpeptide C</td>


<td>Photorhabdus laumondii subsp. laumondii TTO1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135: 0-16848">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>bicornutin A1, bicornutin A2</td>


<td>Xenorhabdus budapestensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128: 35-15686">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>NRP</td>

<td>gamexpeptide C</td>


<td>Photorhabdus laumondii subsp. laumondii TTO1</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r76c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001168: 0-4582">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001168/index.html#r1c1" target="_new">BGC0001168</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>NRP</td>
<td>livipeptin</td>
<td>Streptomyces lividans 1326</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002602: 406-14432">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002602/index.html#r1c1" target="_new">BGC0002602</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.43, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Alkaloid</td>
<td>aspergillic acid</td>
<td>Aspergillus flavus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001516: 0-14978">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001516/index.html#r1c1" target="_new">BGC0001516</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>NRP</td>
<td>aspergillic acid</td>
<td>Aspergillus flavus NRRL3357</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002276: 0-3813">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002276/index.html#r1c1" target="_new">BGC0002276</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.55, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>NRP</td>
<td>choline</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001599: 0-9373">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001599/index.html#r1c1" target="_new">BGC0001599</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>NRP</td>
<td>fragin</td>
<td>Burkholderia cenocepacia H111</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002582: 0-7207">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002582/index.html#r1c1" target="_new">BGC0002582</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>NRP</td>
<td>guanipiperazine A, guanipiperazine B</td>
<td>Streptomyces chrestomyceticus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002278: 0-28520">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002278/index.html#r1c1" target="_new">BGC0002278</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.42, order:N/A, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>Alkaloid, NRP</td>
<td>brasiliamide I, brasiliamide A, brasiliamide B, 2,2-dibenzylpiperazine, brasiliamide D</td>
<td>Penicillium brasilianum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002286: 0-16374">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002286/index.html#r1c1" target="_new">BGC0002286</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.45, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>NRP</td>
<td>ririwpeptide A, ririwpeptide B, ririwpeptide C</td>
<td>Photorhabdus laumondii subsp. laumondii TTO1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135: 0-16848">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>NRP</td>
<td>bicornutin A1, bicornutin A2</td>
<td>Xenorhabdus budapestensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128: 35-15686">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>NRP</td>
<td>gamexpeptide C</td>
<td>Photorhabdus laumondii subsp. laumondii TTO1</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r76c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-603"></div> <span class="help-tooltip" id="cb-general-help-603">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r76c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r76c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r76c1_1.svg">
NW_026054545 (1237578-1301433): Daldinia decipiens strain CBS 113046 unplaced... (31% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r76c1_2.svg">
NW_025953250 (1739569-1803433): Annulohypoxylon maeteangense strain CBS 12383... (27% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r76c1_3.svg">
NW_025953807 (359-64223): Annulohypoxylon truncatum strain CBS 140777 unplace... (26% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r76c1_4.svg">
NW_025953037 (3106468-3170314): Daldinia caldariorum strain CBS 122874 unplac... (31% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r76c1_5.svg">
NW_026250021 (1864413-1931647): Hypoxylon trugodes strain CBS 135444 unplaced... (11% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r76c1_6.svg">
NW_013550595 (288321-352164): Rhinocladiella mackenziei CBS 650.93 unplaced g... (8% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r76c1_7.svg">
NZ_CP099727 (3358661-3559018): Pseudomonas putida strain PCL1760 chromosome, ... (5% of genes show similarity), NRP-metallophore,NRPS
</option>

<option value = "svg/clusterblast_r76c1_8.svg">
NC_058402 (338926-402775): Fusarium poae strain DAOMC 252244 chromosome 4, wh... (8% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r76c1_9.svg">
NC_038015 (361754-425603): Fusarium venenatum strain A3/5 genome assembly, ch... (8% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r76c1_10.svg">
NZ_LT707061 (2852031-2980680): Pseudomonas putida strain N1R chromosome I (5% of genes show similarity), NRP-metallophore,NRPS
</option>

</select>
<button id= "clusterblast-r76c1-download">Download graphic</button>
</div>
<div id="clusterblast-r76c1-svg"></div>

</div>
</div>

<div class="body-details-section r76c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-604"></div> <span class="help-tooltip" id="cb-known-help-604">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r76c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r76c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r76c1_1.svg">
choline
</option>

</select>
<button id="knownclusterblast-r76c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r76c1-svg"></div>

</div>
</div>

<div class="body-details-section r76c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-605"></div> <span class="help-tooltip" id="cb-sub-help-605">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r76c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-608"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-608">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r76c1-domain-bubble-svg" data-filename="scaffold_76_r76c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r76c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (5895 - 49759): single NRPS-like
</option>


</select>

<div class="details-svg" id="r76c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r76c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r76c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-610"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-610">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r76c1-pfam-details-svg" data-filename="scaffold_76_r76c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r76c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r76c1-nrps_pks_products" data-name="r76c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r76c1-nrps_pks_monomers" data-name="r76c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r76c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-606"></div> <span class="help-tooltip" id="nrps-structures-help-606">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 5895 - 49759:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r76c1-1" class="smiles-canvas" data-smiles="NC(C)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Ala)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ala" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ala*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r76c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC(C)C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r76c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-607"></div> <span class="help-tooltip" id="nrps-monomers-help-607">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">FUN_009229</span></strong>: Ala
<div class="collapser collapser-target-FUN_009229 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ala" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ala*" target="_new">relaxed</a>
</dd>
<br>



<dd><strong>AMP-binding (13..418)</strong>: Ala
<div class="collapser collapser-target-FUN_009229 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ala
<div class="collapser collapser-target-FUN_009229-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Apolar, aliphatic (Gly, Ala, Val, Leu, Ile, Abu, Ival)</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>ala (Ala)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">--WLWNVEIK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>

<dt>Stachelhaus code match:</dt>
<dd>0% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>








</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>

<div class="page" id='r76c2' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_76 - Region 2 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-861"></div> <span class="help-tooltip" id="region_svg-help-861">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_76.region002.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r76c2-svg" data-filename="scaffold_76_r76c2.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 54,975 - 75,625 nt. (total: 20,651 nt)

<a class="cluster-rules-header" id="r76c2-rules-header" href="#r76c2">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r76c2-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>

<div class="region-svg-container">
<div id='r76c2-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-862"></div> <span class="help-tooltip" id="focus-panel-help-862">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r76c2">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r76c2-gene-table-details" data-name="r76c2-gene-table-details">
Gene overview
</div>



<div class="body-details-header r76c2-MIBiG-cluster-compare" data-name="r76c2-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r76c2-clusterblast" data-name="r76c2-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r76c2-knownclusterblast" data-name="r76c2-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r76c2-subclusterblast" data-name="r76c2-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r76c2-pfam-details" data-name="r76c2-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r76c2-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-615"></div> <span class="help-tooltip" id="gt-help-tooltip-help-615">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r76c2">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r76c2-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r76c2-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r76c2">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r76c2-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-611"></div> <span class="help-tooltip" id="MIBiG-help-611">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r76c2-selector">Analysis type:</label>
<select id="comparison-MIBiG-r76c2-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r76c2"></div>
<div id="comparison-MIBiG-r76c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">terpene</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002260: 942-16838">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002260/index.html#r1c1" target="_new">BGC0002260</a></td>





<td class="cc-heat-cell" title="0.72 (id:0.58, order:0.64, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>


<td>0.40</td>
<td>Terpene</td>

<td>trichobrasilenol, xylarenic acid B, brasilane A, brasilane F, brasilane E, brasilane D</td>


<td>Annulohypoxylon truncatum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002173: 92-7542">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002173/index.html#r1c1" target="_new">BGC0002173</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.47, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.26</td>
<td>Terpene, Saccharide</td>

<td>fumihopaside A, compound 3, 21--H-hopane-3beta, 22-diol</td>


<td>Aspergillus fumigatus A1163</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000688: 696-13661">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000688/index.html#r1c1" target="_new">BGC0000688</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.47, order:N/A, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.19</td>
<td>Terpene</td>

<td>copalyl diphosphate</td>


<td>Diaporthe amygdali</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002402: 203-37725">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002402/index.html#r1c1" target="_new">BGC0002402</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>Terpene</td>

<td>yossosides</td>


<td>Spinacia oleracea</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000676: 621-14838">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000676/index.html#r1c1" target="_new">BGC0000676</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.44, order:N/A, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.17</td>
<td>Terpene</td>

<td>aphidicolin, aphidicolan-16-ol, 3-deoxyaphidicolin, 17-deoxyaphidicolin</td>


<td>Phoma betae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000687: 3134-24057">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000687/index.html#r1c1" target="_new">BGC0000687</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.16</td>
<td>Terpene</td>

<td>phyllocladan-16-ol</td>


<td>Diaporthe amygdali</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000685: 127-25775">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000685/index.html#r1c1" target="_new">BGC0000685</a></td>





<td class="cc-heat-cell" title="0.44 (id:0.43, order:N/A, components:0.45)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>


<td>0.14</td>
<td>Terpene</td>

<td>brassicicene C</td>


<td>Alternaria brassicicola ATCC 96836</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001775: 2711-20673">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001775/index.html#r1c1" target="_new">BGC0001775</a></td>





<td class="cc-heat-cell" title="0.44 (id:0.42, order:N/A, components:0.46)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>


<td>0.14</td>
<td>Terpene</td>

<td>sespendole</td>


<td>Cordana terrestris</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002163: 3429-9322">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002163/index.html#r1c1" target="_new">BGC0002163</a></td>





<td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>


<td>0.14</td>
<td>Other</td>

<td>fusatin, fusatinic acid, 8-oxo-fusatin, 8-oxo-isopentenyladenine</td>


<td>Fusarium pseudograminearum CS3096</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001995: 0-27487">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001995/index.html#r1c1" target="_new">BGC0001995</a></td>





<td class="cc-heat-cell" title="0.43 (id:0.44, order:N/A, components:0.41)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>


<td>0.14</td>
<td>Terpene</td>

<td>heptelidic acid</td>


<td>Aspergillus oryzae RIB40</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r76c2-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002260: 942-16838">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002260/index.html#r1c1" target="_new">BGC0002260</a></td>


<td class="cc-heat-cell" title="0.72 (id:0.58, order:0.64, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
<td>0.72</td>
<td>Terpene</td>
<td>trichobrasilenol, xylarenic acid B, brasilane A, brasilane F, brasilane E, brasilane D</td>
<td>Annulohypoxylon truncatum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002173: 92-7542">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002173/index.html#r1c1" target="_new">BGC0002173</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.47, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>Terpene, Saccharide</td>
<td>fumihopaside A, compound 3, 21--H-hopane-3beta, 22-diol</td>
<td>Aspergillus fumigatus A1163</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000688: 696-13661">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000688/index.html#r1c1" target="_new">BGC0000688</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.47, order:N/A, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Terpene</td>
<td>copalyl diphosphate</td>
<td>Diaporthe amygdali</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002402: 203-37725">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002402/index.html#r1c1" target="_new">BGC0002402</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Terpene</td>
<td>yossosides</td>
<td>Spinacia oleracea</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000676: 621-14838">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000676/index.html#r1c1" target="_new">BGC0000676</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.44, order:N/A, components:0.54)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>Terpene</td>
<td>aphidicolin, aphidicolan-16-ol, 3-deoxyaphidicolin, 17-deoxyaphidicolin</td>
<td>Phoma betae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000687: 3134-24057">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000687/index.html#r1c1" target="_new">BGC0000687</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>Terpene</td>
<td>phyllocladan-16-ol</td>
<td>Diaporthe amygdali</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000685: 127-25775">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000685/index.html#r1c1" target="_new">BGC0000685</a></td>


<td class="cc-heat-cell" title="0.44 (id:0.43, order:N/A, components:0.45)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
<td>0.44</td>
<td>Terpene</td>
<td>brassicicene C</td>
<td>Alternaria brassicicola ATCC 96836</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001775: 2711-20673">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001775/index.html#r1c1" target="_new">BGC0001775</a></td>


<td class="cc-heat-cell" title="0.44 (id:0.42, order:N/A, components:0.46)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
<td>0.44</td>
<td>Terpene</td>
<td>sespendole</td>
<td>Cordana terrestris</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002163: 3429-9322">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002163/index.html#r1c1" target="_new">BGC0002163</a></td>


<td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
<td>0.44</td>
<td>Other</td>
<td>fusatin, fusatinic acid, 8-oxo-fusatin, 8-oxo-isopentenyladenine</td>
<td>Fusarium pseudograminearum CS3096</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001995: 0-27487">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001995/index.html#r1c1" target="_new">BGC0001995</a></td>


<td class="cc-heat-cell" title="0.43 (id:0.44, order:N/A, components:0.41)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
<td>0.43</td>
<td>Terpene</td>
<td>heptelidic acid</td>
<td>Aspergillus oryzae RIB40</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r76c2-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-612"></div> <span class="help-tooltip" id="cb-general-help-612">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r76c2-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r76c2_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r76c2_1.svg">
NW_025953250 (2330314-2401531): Annulohypoxylon maeteangense strain CBS 12383... (25% of genes show similarity), T1PKS,terpene
</option>

<option value = "svg/clusterblast_r76c2_2.svg">
NW_025953807 (113785-185922): Annulohypoxylon truncatum strain CBS 140777 unp... (25% of genes show similarity), T1PKS,terpene
</option>

<option value = "svg/clusterblast_r76c2_3.svg">
NC_035794 (4220728-4252181): Pochonia chlamydosporia 170 chromosome 5, whole ... (30% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r76c2_4.svg">
NW_006711178 (114520-207586): Trichoderma reesei QM6a unplaced genomic scaffo... (10% of genes show similarity), isocyanide,terpene
</option>

<option value = "svg/clusterblast_r76c2_5.svg">
NC_072353 (3025130-3056913): Talaromyces marneffei chromosome 6, complete seq... (30% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r76c2_6.svg">
NW_017971433 (2125109-2260340): Talaromyces atroroseus strain IBT 11181 chrom... (5% of genes show similarity), NRPS-like,fungal-RiPP-like,terpene
</option>

<option value = "svg/clusterblast_r76c2_7.svg">
NW_026250021 (1447507-1558636): Hypoxylon trugodes strain CBS 135444 unplaced... (6% of genes show similarity), fungal-RiPP-like,terpene
</option>

<option value = "svg/clusterblast_r76c2_8.svg">
NW_025953037 (2897258-2988227): Daldinia caldariorum strain CBS 122874 unplac... (5% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r76c2_9.svg">
NW_019154049 (138062-255179): Pochonia chlamydosporia 170 chromosome Unknown ... (4% of genes show similarity), fungal-RiPP-like,terpene
</option>

<option value = "svg/clusterblast_r76c2_10.svg">
NW_026250019 (1423750-1491332): Hypoxylon trugodes strain CBS 135444 unplaced... (14% of genes show similarity), T1PKS,indole
</option>

</select>
<button id= "clusterblast-r76c2-download">Download graphic</button>
</div>
<div id="clusterblast-r76c2-svg"></div>

</div>
</div>

<div class="body-details-section r76c2-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-613"></div> <span class="help-tooltip" id="cb-known-help-613">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r76c2-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r76c2_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r76c2_1.svg">
trichobrasilenol/xylarenic acid B/brasilane A/brasilane F/brasilane E/brasilane D
</option>

</select>
<button id="knownclusterblast-r76c2-download">Download graphic</button>
</div>
<div id="knownclusterblast-r76c2-svg"></div>

</div>
</div>

<div class="body-details-section r76c2-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-614"></div> <span class="help-tooltip" id="cb-sub-help-614">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r76c2-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-616"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-616">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r76c2-pfam-details-svg" data-filename="scaffold_76_r76c2_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r76c2-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">









</div>









</div>

</div>

</div>





<div class="page" id='r78c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_78 - Region 1 - fungal-RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-863"></div> <span class="help-tooltip" id="region_svg-help-863">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_78.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r78c1-svg" data-filename="scaffold_78_r78c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 112,872 - 161,769 nt. (total: 48,898 nt)

<a class="cluster-rules-header" id="r78c1-rules-header" href="#r78c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r78c1-rules">fungal-RiPP-like: DUF3328</div>

<div class="region-svg-container">
<div id='r78c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-864"></div> <span class="help-tooltip" id="focus-panel-help-864">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r78c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r78c1-gene-table-details" data-name="r78c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r78c1-MIBiG-cluster-compare" data-name="r78c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r78c1-clusterblast" data-name="r78c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r78c1-knownclusterblast" data-name="r78c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r78c1-subclusterblast" data-name="r78c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r78c1-pfam-details" data-name="r78c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r78c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-621"></div> <span class="help-tooltip" id="gt-help-tooltip-help-621">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r78c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r78c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r78c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r78c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r78c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-617"></div> <span class="help-tooltip" id="MIBiG-help-617">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r78c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r78c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r78c1"></div>
<div id="comparison-MIBiG-r78c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">fungal-RiPP-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001306: 0-6409">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001306/index.html#r1c1" target="_new">BGC0001306</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.33</td>
<td>RiPP</td>

<td>asperipin 2a</td>


<td>Aspergillus flavus NRRL3357</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000627: 0-28295">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000627/index.html#r1c1" target="_new">BGC0000627</a></td>





<td class="cc-heat-cell" title="0.38 (id:0.45, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>


<td>0.11</td>
<td>RiPP</td>

<td>ustiloxin B</td>


<td>Aspergillus flavus NRRL3357</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001485: 0-10917">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001485/index.html#r1c1" target="_new">BGC0001485</a></td>





<td class="cc-heat-cell" title="0.37 (id:0.45, order:N/A, components:0.30)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>


<td>0.10</td>
<td>Alkaloid</td>

<td>chuangxinmycin</td>


<td>Actinoplanes tsinanensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001621: 3190-27169">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001621/index.html#r1c1" target="_new">BGC0001621</a></td>





<td class="cc-heat-cell" title="0.34 (id:0.47, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>


<td>0.09</td>
<td>NRP</td>

<td>imizoquin A, imizoquin B, imizoquin C, imizoquin D, TMC-2A, TMC-2B</td>


<td>Aspergillus flavus NRRL3357</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001398: 4999-64171">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001398/index.html#r1c1" target="_new">BGC0001398</a></td>





<td class="cc-heat-cell" title="0.33 (id:0.47, order:N/A, components:0.23)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>


<td>0.09</td>
<td>RiPP</td>

<td>phomopsin A, phomopsin B, phomopsin E</td>


<td>Phomopsis leptostromiformis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002482: 14032-28331">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002482/index.html#r1c1" target="_new">BGC0002482</a></td>





<td class="cc-heat-cell" title="0.33 (id:0.42, order:0.41, components:0.21)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>


<td>0.09</td>
<td>Other</td>

<td>tjipanazole D, tjipanazole I, tjipanazole L, tjipanazole K, tjipanazole M</td>


<td>Symphyonema bifilamentata 97.28</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000272: 614-8926">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000272/index.html#r1c1" target="_new">BGC0000272</a></td>





<td class="cc-heat-cell" title="0.32 (id:0.41, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.32), rgb(0, 0, 0, 0.32) 32%, white 32%)"></td>


<td>0.09</td>
<td>Polyketide</td>

<td>spore pigment</td>


<td>Streptomyces collinus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001489: 3-22742">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001489/index.html#r1c1" target="_new">BGC0001489</a></td>





<td class="cc-heat-cell" title="0.31 (id:0.49, order:N/A, components:0.20)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>


<td>0.08</td>
<td>Polyketide</td>

<td>6-hydroxymellein</td>


<td>Cladonia uncialis subsp. uncialis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001206: 861-11207">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001206/index.html#r1c1" target="_new">BGC0001206</a></td>





<td class="cc-heat-cell" title="0.30 (id:0.45, order:N/A, components:0.20)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>


<td>0.08</td>
<td>Other</td>

<td>indolmycin</td>


<td>Streptomyces griseus subsp. griseus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000161: 0-32759">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000161/index.html#r1c1" target="_new">BGC0000161</a></td>





<td class="cc-heat-cell" title="0.29 (id:0.50, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>


<td>0.07</td>
<td>Polyketide</td>

<td>isoterrein</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r78c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001306: 0-6409">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001306/index.html#r1c1" target="_new">BGC0001306</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>RiPP</td>
<td>asperipin 2a</td>
<td>Aspergillus flavus NRRL3357</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000627: 0-28295">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000627/index.html#r1c1" target="_new">BGC0000627</a></td>


<td class="cc-heat-cell" title="0.38 (id:0.45, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
<td>0.38</td>
<td>RiPP</td>
<td>ustiloxin B</td>
<td>Aspergillus flavus NRRL3357</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001485: 0-10917">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001485/index.html#r1c1" target="_new">BGC0001485</a></td>


<td class="cc-heat-cell" title="0.37 (id:0.45, order:N/A, components:0.30)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>
<td>0.37</td>
<td>Alkaloid</td>
<td>chuangxinmycin</td>
<td>Actinoplanes tsinanensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001621: 3190-27169">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001621/index.html#r1c1" target="_new">BGC0001621</a></td>


<td class="cc-heat-cell" title="0.34 (id:0.47, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>
<td>0.34</td>
<td>NRP</td>
<td>imizoquin A, imizoquin B, imizoquin C, imizoquin D, TMC-2A, TMC-2B</td>
<td>Aspergillus flavus NRRL3357</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001398: 4999-64171">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001398/index.html#r1c1" target="_new">BGC0001398</a></td>


<td class="cc-heat-cell" title="0.33 (id:0.47, order:N/A, components:0.23)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>
<td>0.33</td>
<td>RiPP</td>
<td>phomopsin A, phomopsin B, phomopsin E</td>
<td>Phomopsis leptostromiformis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002482: 14032-28331">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002482/index.html#r1c1" target="_new">BGC0002482</a></td>


<td class="cc-heat-cell" title="0.33 (id:0.42, order:0.41, components:0.21)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>
<td>0.33</td>
<td>Other</td>
<td>tjipanazole D, tjipanazole I, tjipanazole L, tjipanazole K, tjipanazole M</td>
<td>Symphyonema bifilamentata 97.28</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000272: 614-8926">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000272/index.html#r1c1" target="_new">BGC0000272</a></td>


<td class="cc-heat-cell" title="0.32 (id:0.41, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.32), rgb(0, 0, 0, 0.32) 32%, white 32%)"></td>
<td>0.32</td>
<td>Polyketide</td>
<td>spore pigment</td>
<td>Streptomyces collinus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001489: 3-22742">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001489/index.html#r1c1" target="_new">BGC0001489</a></td>


<td class="cc-heat-cell" title="0.31 (id:0.49, order:N/A, components:0.20)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>
<td>0.31</td>
<td>Polyketide</td>
<td>6-hydroxymellein</td>
<td>Cladonia uncialis subsp. uncialis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001206: 861-11207">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001206/index.html#r1c1" target="_new">BGC0001206</a></td>


<td class="cc-heat-cell" title="0.30 (id:0.45, order:N/A, components:0.20)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>
<td>0.30</td>
<td>Other</td>
<td>indolmycin</td>
<td>Streptomyces griseus subsp. griseus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000161: 0-32759">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000161/index.html#r1c1" target="_new">BGC0000161</a></td>


<td class="cc-heat-cell" title="0.29 (id:0.50, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>
<td>0.29</td>
<td>Polyketide</td>
<td>isoterrein</td>
<td>Aspergillus terreus NIH2624</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r78c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-618"></div> <span class="help-tooltip" id="cb-general-help-618">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r78c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r78c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r78c1_1.svg">
NW_025953251 (1150657-1241589): Annulohypoxylon maeteangense strain CBS 12383... (37% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r78c1_2.svg">
NW_025953791 (453058-543985): Annulohypoxylon truncatum strain CBS 140777 unp... (36% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r78c1_3.svg">
NW_026054546 (605610-696537): Daldinia decipiens strain CBS 113046 unplaced g... (29% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r78c1_4.svg">
NW_026250016 (1289397-1380477): Hypoxylon trugodes strain CBS 135444 unplaced... (33% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r78c1_5.svg">
NW_025953046 (751363-842392): Daldinia caldariorum strain CBS 122874 unplaced... (47% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r78c1_6.svg">
NC_043741 (1838282-1929269): Pyricularia pennisetigena strain Br36 chromosome... (25% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r78c1_7.svg">
NW_006917092 (5143447-5234370): Pestalotiopsis fici W106-1 unplaced genomic s... (33% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r78c1_8.svg">
NW_024468909 (861558-952670): Cryphonectria parasitica EP155 unplaced genomic... (28% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r78c1_9.svg">
NW_024468914 (3324192-3415460): Cryphonectria parasitica EP155 unplaced genom... (6% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r78c1_10.svg">
NT_165978 (927571-1095183): Chaetomium globosum CBS 148.51 scaffold 3 genomic... (9% of genes show similarity), fungal-RiPP-like
</option>

</select>
<button id= "clusterblast-r78c1-download">Download graphic</button>
</div>
<div id="clusterblast-r78c1-svg"></div>

</div>
</div>

<div class="body-details-section r78c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-619"></div> <span class="help-tooltip" id="cb-known-help-619">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r78c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-620"></div> <span class="help-tooltip" id="cb-sub-help-620">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r78c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-622"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-622">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r78c1-pfam-details-svg" data-filename="scaffold_78_r78c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r78c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">











</div>











</div>

</div>

</div>





<div class="page" id='r85c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_85 - Region 1 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-865"></div> <span class="help-tooltip" id="region_svg-help-865">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_85.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r85c1-svg" data-filename="scaffold_85_r85c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 64,072 - 111,174 nt. (total: 47,103 nt)

<a class="cluster-rules-header" id="r85c1-rules-header" href="#r85c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r85c1-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r85c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-866"></div> <span class="help-tooltip" id="focus-panel-help-866">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r85c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r85c1-gene-table-details" data-name="r85c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r85c1-nrps_pks" data-name="r85c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r85c1-MIBiG-cluster-compare" data-name="r85c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r85c1-clusterblast" data-name="r85c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r85c1-knownclusterblast" data-name="r85c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r85c1-subclusterblast" data-name="r85c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r85c1-pfam-details" data-name="r85c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r85c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-629"></div> <span class="help-tooltip" id="gt-help-tooltip-help-629">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r85c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r85c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r85c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r85c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r85c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-623"></div> <span class="help-tooltip" id="nrps-domain-help-623">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r85c1-details-svg" data-filename="scaffold_85_r85c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r85c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r85c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-624"></div> <span class="help-tooltip" id="MIBiG-help-624">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r85c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r85c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r85c1"></div>
<div id="comparison-MIBiG-r85c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264: 8442-40380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264</a></td>





<td class="cc-heat-cell" title="0.51 (id:0.44, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>


<td>0.19</td>
<td>Polyketide</td>

<td>betaenone A, betaenone B, betaenone C</td>


<td>Phoma betae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002240: 96-11225">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002240/index.html#r1c1" target="_new">BGC0002240</a></td>





<td class="cc-heat-cell" title="0.51 (id:0.45, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>


<td>0.19</td>
<td>Polyketide</td>

<td>BAB, BAA</td>


<td>Metarhizium anisopliae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001254: 82-7456">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001254/index.html#r1c1" target="_new">BGC0001254</a></td>





<td class="cc-heat-cell" title="0.51 (id:0.44, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>


<td>0.19</td>
<td>Polyketide</td>

<td>ACT-Toxin II</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002191: 260-8785">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002191/index.html#r1c1" target="_new">BGC0002191</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>Polyketide</td>

<td>prolipyrone B, gibepyrone D</td>


<td>Fusarium graminearum PH-1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002258: 213-24781">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002258/index.html#r1c1" target="_new">BGC0002258</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>Polyketide</td>

<td>shimalactone A, shimalactone B</td>


<td>Aspergillus stellatus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002429: 0-14296">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002429/index.html#r1c1" target="_new">BGC0002429</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>Terpene, Polyketide</td>

<td>higginsianin B</td>


<td>Colletotrichum higginsianum IMI 349063</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000107: 144-6819">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000107/index.html#r1c1" target="_new">BGC0000107</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>Polyketide</td>

<td>naphthopyrone</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002525: 1000-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002525/index.html#r1c1" target="_new">BGC0002525</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.45, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>Polyketide</td>

<td>fusarubin, 1233A, 1233B, NG-391, lucilactaene</td>


<td>Fusarium sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001258: 43-7265">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001258/index.html#r1c1" target="_new">BGC0001258</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.17</td>
<td>Polyketide</td>

<td>1,3,6,8-tetrahydroxynaphthalene</td>


<td>Glarea lozoyensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002154: 337-7126">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002154/index.html#r1c1" target="_new">BGC0002154</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.17</td>
<td>Polyketide</td>

<td>1,3,5,8-tetrahydroxynapthalene, 1-(alpha-l-(2-O-methyl)-6-deoxymannopyranosyloxy)-3,6,8-trimethoxynaphthalene</td>


<td>Pyricularia oryzae 70-15</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r85c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264: 8442-40380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264</a></td>


<td class="cc-heat-cell" title="0.51 (id:0.44, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
<td>0.51</td>
<td>Polyketide</td>
<td>betaenone A, betaenone B, betaenone C</td>
<td>Phoma betae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002240: 96-11225">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002240/index.html#r1c1" target="_new">BGC0002240</a></td>


<td class="cc-heat-cell" title="0.51 (id:0.45, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
<td>0.51</td>
<td>Polyketide</td>
<td>BAB, BAA</td>
<td>Metarhizium anisopliae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001254: 82-7456">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001254/index.html#r1c1" target="_new">BGC0001254</a></td>


<td class="cc-heat-cell" title="0.51 (id:0.44, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
<td>0.51</td>
<td>Polyketide</td>
<td>ACT-Toxin II</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002191: 260-8785">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002191/index.html#r1c1" target="_new">BGC0002191</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Polyketide</td>
<td>prolipyrone B, gibepyrone D</td>
<td>Fusarium graminearum PH-1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002258: 213-24781">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002258/index.html#r1c1" target="_new">BGC0002258</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Polyketide</td>
<td>shimalactone A, shimalactone B</td>
<td>Aspergillus stellatus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002429: 0-14296">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002429/index.html#r1c1" target="_new">BGC0002429</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.43, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Terpene, Polyketide</td>
<td>higginsianin B</td>
<td>Colletotrichum higginsianum IMI 349063</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000107: 144-6819">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000107/index.html#r1c1" target="_new">BGC0000107</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>Polyketide</td>
<td>naphthopyrone</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002525: 1000-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002525/index.html#r1c1" target="_new">BGC0002525</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.45, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>Polyketide</td>
<td>fusarubin, 1233A, 1233B, NG-391, lucilactaene</td>
<td>Fusarium sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001258: 43-7265">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001258/index.html#r1c1" target="_new">BGC0001258</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>Polyketide</td>
<td>1,3,6,8-tetrahydroxynaphthalene</td>
<td>Glarea lozoyensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002154: 337-7126">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002154/index.html#r1c1" target="_new">BGC0002154</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.43, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>Polyketide</td>
<td>1,3,5,8-tetrahydroxynapthalene, 1-(alpha-l-(2-O-methyl)-6-deoxymannopyranosyloxy)-3,6,8-trimethoxynaphthalene</td>
<td>Pyricularia oryzae 70-15</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r85c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-625"></div> <span class="help-tooltip" id="cb-general-help-625">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r85c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r85c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r85c1_1.svg">
NW_026054642 (296356-363048): Ophidiomyces ophidiicola strain CBS 122913 chro... (20% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r85c1_2.svg">
NW_025091217 (15487-82066): Aspergillus viridinutans IFM 47045 DNA, scaffold ... (37% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r85c1_3.svg">
NW_021167080 (4131403-4218796): Sodiomyces alkalinus F11 unplaced genomic sca... (14% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r85c1_4.svg">
NC_030957 (3250063-3360781): Colletotrichum higginsianum IMI 349063 chromosom... (5% of genes show similarity), T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r85c1_5.svg">
NW_026250021 (1864413-1931647): Hypoxylon trugodes strain CBS 135444 unplaced... (7% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r85c1_6.svg">
NW_024468940 (1557228-1624623): Cucurbitaria berberidis CBS 394.84 unplaced g... (9% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r85c1_7.svg">
NW_003052500 (3318567-3442588): Uncinocarpus reesii 1704 scaffold 1 genomic s... (5% of genes show similarity), NRP-metallophore,NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r85c1_8.svg">
NW_006921541 (103572-212868): Pseudocercospora fijiensis CIRAD86 unplaced gen... (15% of genes show similarity), NRPS,T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r85c1_9.svg">
NW_025091264 (5340181-5432359): Aspergillus udagawae IFM 46973 DNA, scaffold ... (10% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r85c1_10.svg">
NW_022985443 (1533-77825): Mytilinidion resinicola strain CBS 304.34 unplaced... (9% of genes show similarity), NRPS-like,T1PKS
</option>

</select>
<button id= "clusterblast-r85c1-download">Download graphic</button>
</div>
<div id="clusterblast-r85c1-svg"></div>

</div>
</div>

<div class="body-details-section r85c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-626"></div> <span class="help-tooltip" id="cb-known-help-626">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r85c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-627"></div> <span class="help-tooltip" id="cb-sub-help-627">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r85c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-630"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-630">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r85c1-pfam-details-svg" data-filename="scaffold_85_r85c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r85c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r85c1-nrps_pks_monomers" data-name="r85c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>




</div>










<div class="sidepanel-details-section r85c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-628"></div> <span class="help-tooltip" id="nrps-monomers-help-628">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">FUN_009511</span></strong>: pk
<div class="collapser collapser-target-FUN_009511 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (523..830)</strong>: pk
<div class="collapser collapser-target-FUN_009511 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-FUN_009511-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 79.2%</dd>
<dd>Methylmalonyl-CoA: 66.7%</dd>
<dd>2-Methylbutyryl-CoA: 62.5%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-FUN_009511-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 60.2</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 54.7</dt>

<dd></dd><dt>Malonyl-CoA: 43.8</dt>

<dd></dd><dt>Propionyl-CoA: 30.8</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 30.5</dt>

<dd></dd><dt>Benzoyl-CoA: 21.0</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 19.0</dt>

<dd></dd><dt>inactive: 11.5</dt>

<dd></dd><dt>Isobutyryl-CoA: 9.5</dt>

<dd></dd><dt>CHC-CoA: 3.8</dt>

<dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

<dd></dd><dt>fatty_acid: 0.0</dt>

<dd></dd><dt>Acetyl-CoA: 0.0</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>




</div>

</div>

</div>





<div class="page" id='r89c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_89 - Region 1 - indole <div class="help-container"> <div class="help-icon" data-id="region_svg-help-867"></div> <span class="help-tooltip" id="region_svg-help-867">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_89.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r89c1-svg" data-filename="scaffold_89_r89c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 5,389 - 26,597 nt. (total: 21,209 nt)

<a class="cluster-rules-header" id="r89c1-rules-header" href="#r89c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r89c1-rules">indole: (indsynth or dmat or indole_PTase)</div>

<div class="region-svg-container">
<div id='r89c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-868"></div> <span class="help-tooltip" id="focus-panel-help-868">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r89c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r89c1-gene-table-details" data-name="r89c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r89c1-MIBiG-cluster-compare" data-name="r89c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r89c1-clusterblast" data-name="r89c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r89c1-knownclusterblast" data-name="r89c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r89c1-subclusterblast" data-name="r89c1-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r89c1-pfam-details" data-name="r89c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r89c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-635"></div> <span class="help-tooltip" id="gt-help-tooltip-help-635">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r89c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r89c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r89c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r89c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r89c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-631"></div> <span class="help-tooltip" id="MIBiG-help-631">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r89c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r89c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r89c1"></div>
<div id="comparison-MIBiG-r89c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">indole</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002162: 0-5465">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002162/index.html#r1c1" target="_new">BGC0002162</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.41, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Terpene</td>

<td>sesterfisherol, sesterfisheric acid, compound 6, compound 7a, compound 5, compound 8</td>


<td>Aspergillus fischeri NRRL 181</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001777: 2790-19724">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001777/index.html#r1c1" target="_new">BGC0001777</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.15</td>
<td>Terpene</td>

<td>shearinine D</td>


<td>Escovopsis weberi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001893: 155-19434">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001893/index.html#r1c1" target="_new">BGC0001893</a></td>





<td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>


<td>0.13</td>
<td>Other</td>

<td>abscisic acid</td>


<td>Botrytis cinerea B05.10</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000811: 0-22152">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000811/index.html#r1c1" target="_new">BGC0000811</a></td>





<td class="cc-heat-cell" title="0.41 (id:0.43, order:N/A, components:0.39)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>


<td>0.13</td>
<td>Alkaloid</td>

<td>fumigaclavine C</td>


<td>Aspergillus fumigatus Af293</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002245: 219-11090">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002245/index.html#r1c1" target="_new">BGC0002245</a></td>





<td class="cc-heat-cell" title="0.40 (id:0.42, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>


<td>0.12</td>
<td>Terpene</td>

<td>terpestacin, preterpestacin 3, preterpestacin 2, preterpestacin 1</td>


<td>Bipolaris maydis C5</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002013: 0-11564">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002013/index.html#r1c1" target="_new">BGC0002013</a></td>





<td class="cc-heat-cell" title="0.39 (id:0.42, order:N/A, components:0.37)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>


<td>0.12</td>
<td>RiPP</td>

<td>curacozole</td>


<td>Streptomyces curacoi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000627: 0-28295">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000627/index.html#r1c1" target="_new">BGC0000627</a></td>





<td class="cc-heat-cell" title="0.38 (id:0.44, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>


<td>0.11</td>
<td>RiPP</td>

<td>ustiloxin B</td>


<td>Aspergillus flavus NRRL3357</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000670: 82-32651">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000670/index.html#r1c1" target="_new">BGC0000670</a></td>





<td class="cc-heat-cell" title="0.38 (id:0.43, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>


<td>0.11</td>
<td>Terpene</td>

<td>thalianol, thaliandiol</td>


<td>Arabidopsis thaliana</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002646: 0-18848">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002646/index.html#r1c1" target="_new">BGC0002646</a></td>





<td class="cc-heat-cell" title="0.37 (id:0.42, order:0.41, components:0.30)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>


<td>0.11</td>
<td>Polyketide</td>

<td>hancockinone A</td>


<td>Aspergillus hancockii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000442: 0-9759">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000442/index.html#r1c1" target="_new">BGC0000442</a></td>





<td class="cc-heat-cell" title="0.37 (id:0.41, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>


<td>0.11</td>
<td>NRP</td>

<td>terrequinone A</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r89c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002162: 0-5465">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002162/index.html#r1c1" target="_new">BGC0002162</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.41, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Terpene</td>
<td>sesterfisherol, sesterfisheric acid, compound 6, compound 7a, compound 5, compound 8</td>
<td>Aspergillus fischeri NRRL 181</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001777: 2790-19724">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001777/index.html#r1c1" target="_new">BGC0001777</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>Terpene</td>
<td>shearinine D</td>
<td>Escovopsis weberi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001893: 155-19434">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001893/index.html#r1c1" target="_new">BGC0001893</a></td>


<td class="cc-heat-cell" title="0.42 (id:0.42, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
<td>0.42</td>
<td>Other</td>
<td>abscisic acid</td>
<td>Botrytis cinerea B05.10</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000811: 0-22152">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000811/index.html#r1c1" target="_new">BGC0000811</a></td>


<td class="cc-heat-cell" title="0.41 (id:0.43, order:N/A, components:0.39)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
<td>0.41</td>
<td>Alkaloid</td>
<td>fumigaclavine C</td>
<td>Aspergillus fumigatus Af293</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002245: 219-11090">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002245/index.html#r1c1" target="_new">BGC0002245</a></td>


<td class="cc-heat-cell" title="0.40 (id:0.42, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>
<td>0.40</td>
<td>Terpene</td>
<td>terpestacin, preterpestacin 3, preterpestacin 2, preterpestacin 1</td>
<td>Bipolaris maydis C5</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002013: 0-11564">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002013/index.html#r1c1" target="_new">BGC0002013</a></td>


<td class="cc-heat-cell" title="0.39 (id:0.42, order:N/A, components:0.37)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
<td>0.39</td>
<td>RiPP</td>
<td>curacozole</td>
<td>Streptomyces curacoi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000627: 0-28295">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000627/index.html#r1c1" target="_new">BGC0000627</a></td>


<td class="cc-heat-cell" title="0.38 (id:0.44, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
<td>0.38</td>
<td>RiPP</td>
<td>ustiloxin B</td>
<td>Aspergillus flavus NRRL3357</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000670: 82-32651">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000670/index.html#r1c1" target="_new">BGC0000670</a></td>


<td class="cc-heat-cell" title="0.38 (id:0.43, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.38), rgb(0, 0, 0, 0.38) 38%, white 38%)"></td>
<td>0.38</td>
<td>Terpene</td>
<td>thalianol, thaliandiol</td>
<td>Arabidopsis thaliana</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002646: 0-18848">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002646/index.html#r1c1" target="_new">BGC0002646</a></td>


<td class="cc-heat-cell" title="0.37 (id:0.42, order:0.41, components:0.30)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>
<td>0.37</td>
<td>Polyketide</td>
<td>hancockinone A</td>
<td>Aspergillus hancockii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000442: 0-9759">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000442/index.html#r1c1" target="_new">BGC0000442</a></td>


<td class="cc-heat-cell" title="0.37 (id:0.41, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>
<td>0.37</td>
<td>NRP</td>
<td>terrequinone A</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r89c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-632"></div> <span class="help-tooltip" id="cb-general-help-632">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r89c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r89c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r89c1_1.svg">
NW_026054584 (79986-111505): Daldinia decipiens strain CBS 113046 unplaced ge... (33% of genes show similarity), indole
</option>

<option value = "svg/clusterblast_r89c1_2.svg">
NW_006917092 (5685560-5716712): Pestalotiopsis fici W106-1 unplaced genomic s... (33% of genes show similarity), indole
</option>

<option value = "svg/clusterblast_r89c1_3.svg">
NW_025953063 (233386-264655): Daldinia caldariorum strain CBS 122874 unplaced... (33% of genes show similarity), indole
</option>

<option value = "svg/clusterblast_r89c1_4.svg">
NW_020291821 (93584-125035): Aspergillus costaricaensis CBS 115574 unplaced g... (22% of genes show similarity), indole
</option>

<option value = "svg/clusterblast_r89c1_5.svg">
NW_023336280 (442688-474139): Aspergillus tubingensis WU-2223L DNA, scaffold ... (20% of genes show similarity), indole
</option>

<option value = "svg/clusterblast_r89c1_6.svg">
NC_054849 (467388-498838): Aspergillus luchuensis IFO 4308 DNA, chromosome 1,... (22% of genes show similarity), indole
</option>

<option value = "svg/clusterblast_r89c1_7.svg">
NW_026250022 (562601-594124): Hypoxylon trugodes strain CBS 135444 unplaced g... (20% of genes show similarity), indole
</option>

<option value = "svg/clusterblast_r89c1_8.svg">
NC_049562 (250751-282212): Talaromyces rugulosus chromosome II, complete sequ... (16% of genes show similarity), indole
</option>

<option value = "svg/clusterblast_r89c1_9.svg">
NW_020291595 (799704-831154): Aspergillus piperis CBS 112811 unplaced genomic... (22% of genes show similarity), indole
</option>

<option value = "svg/clusterblast_r89c1_10.svg">
NW_020292048 (450018-481469): Aspergillus vadensis CBS 113365 unplaced genomi... (25% of genes show similarity), indole
</option>

</select>
<button id= "clusterblast-r89c1-download">Download graphic</button>
</div>
<div id="clusterblast-r89c1-svg"></div>

</div>
</div>

<div class="body-details-section r89c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-633"></div> <span class="help-tooltip" id="cb-known-help-633">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r89c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-634"></div> <span class="help-tooltip" id="cb-sub-help-634">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r89c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-636"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-636">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r89c1-pfam-details-svg" data-filename="scaffold_89_r89c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r89c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">









</div>









</div>

</div>

</div>





<div class="page" id='r90c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_90 - Region 1 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-869"></div> <span class="help-tooltip" id="region_svg-help-869">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_90.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r90c1-svg" data-filename="scaffold_90_r90c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 70,780 - 93,107 nt. (total: 22,328 nt)

<a class="cluster-rules-header" id="r90c1-rules-header" href="#r90c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r90c1-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>

<div class="region-svg-container">
<div id='r90c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-870"></div> <span class="help-tooltip" id="focus-panel-help-870">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r90c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r90c1-gene-table-details" data-name="r90c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r90c1-MIBiG-cluster-compare" data-name="r90c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r90c1-clusterblast" data-name="r90c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r90c1-knownclusterblast" data-name="r90c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r90c1-subclusterblast" data-name="r90c1-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r90c1-pfam-details" data-name="r90c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r90c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-641"></div> <span class="help-tooltip" id="gt-help-tooltip-help-641">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r90c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r90c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r90c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r90c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r90c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-637"></div> <span class="help-tooltip" id="MIBiG-help-637">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r90c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r90c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r90c1"></div>
<div id="comparison-MIBiG-r90c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">terpene</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002173: 92-7542">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002173/index.html#r1c1" target="_new">BGC0002173</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.48, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.18</td>
<td>Terpene, Saccharide</td>

<td>fumihopaside A, compound 3, 21--H-hopane-3beta, 22-diol</td>


<td>Aspergillus fumigatus A1163</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000669: 0-34330">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000669/index.html#r1c1" target="_new">BGC0000669</a></td>





<td class="cc-heat-cell" title="0.37 (id:0.41, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>


<td>0.11</td>
<td>Terpene</td>

<td>marneral</td>


<td>Arabidopsis thaliana</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002449: 47-14377">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002449/index.html#r1c1" target="_new">BGC0002449</a></td>





<td class="cc-heat-cell" title="0.31 (id:0.42, order:N/A, components:0.23)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>


<td>0.08</td>
<td>Other</td>

<td>sarasinoside</td>


<td>uncultured bacterium</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000663: 0-13757">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000663/index.html#r1c1" target="_new">BGC0000663</a></td>





<td class="cc-heat-cell" title="0.25 (id:0.44, order:N/A, components:0.15)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>


<td>0.06</td>
<td>Terpene</td>

<td>hopene</td>


<td>Streptomyces coelicolor A3(2)</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479: 1-29156">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479</a></td>





<td class="cc-heat-cell" title="0.16 (id:0.47, order:N/A, components:0.06)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.16), rgb(0, 0, 0, 0.16) 16%, white 16%)"></td>


<td>0.03</td>
<td>NRP</td>

<td>anabaenopeptin NZ857, nostamide A</td>


<td>Nostoc punctiforme PCC 73102</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r90c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002173: 92-7542">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002173/index.html#r1c1" target="_new">BGC0002173</a></td>


<td class="cc-heat-cell" title="0.49 (id:0.48, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>
<td>0.49</td>
<td>Terpene, Saccharide</td>
<td>fumihopaside A, compound 3, 21--H-hopane-3beta, 22-diol</td>
<td>Aspergillus fumigatus A1163</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000669: 0-34330">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000669/index.html#r1c1" target="_new">BGC0000669</a></td>


<td class="cc-heat-cell" title="0.37 (id:0.41, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>
<td>0.37</td>
<td>Terpene</td>
<td>marneral</td>
<td>Arabidopsis thaliana</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002449: 47-14377">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002449/index.html#r1c1" target="_new">BGC0002449</a></td>


<td class="cc-heat-cell" title="0.31 (id:0.42, order:N/A, components:0.23)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>
<td>0.31</td>
<td>Other</td>
<td>sarasinoside</td>
<td>uncultured bacterium</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000663: 0-13757">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000663/index.html#r1c1" target="_new">BGC0000663</a></td>


<td class="cc-heat-cell" title="0.25 (id:0.44, order:N/A, components:0.15)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>
<td>0.25</td>
<td>Terpene</td>
<td>hopene</td>
<td>Streptomyces coelicolor A3(2)</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001479: 1-29156">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001479/index.html#r1c1" target="_new">BGC0001479</a></td>


<td class="cc-heat-cell" title="0.16 (id:0.47, order:N/A, components:0.06)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.16), rgb(0, 0, 0, 0.16) 16%, white 16%)"></td>
<td>0.16</td>
<td>NRP</td>
<td>anabaenopeptin NZ857, nostamide A</td>
<td>Nostoc punctiforme PCC 73102</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r90c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-638"></div> <span class="help-tooltip" id="cb-general-help-638">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r90c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r90c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r90c1_1.svg">
NZ_KN150936 (2318280-2372633): Burkholderia pseudomallei strain BDM scaffold2... (4% of genes show similarity), RiPP-like,hglE-KS,terpene
</option>

<option value = "svg/clusterblast_r90c1_2.svg">
NZ_JAKOAS010000019 (39109-61062): Cupriavidus gilardii strain BF20-02S NODE 1... (11% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r90c1_3.svg">
NZ_CP104387 (849726-871727): Cupriavidus gilardii strain WM02 chromosome 2, c... (11% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r90c1_4.svg">
NZ_CP080294 (336720-358730): Cupriavidus cauae strain PHS1 chromosome 2, comp... (11% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r90c1_5.svg">
NZ_CP054624 (1624827-1646828): Cupriavidus gilardii strain FDAARGOS 639 chrom... (11% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r90c1_6.svg">
NZ_CP098735 (2011624-2033625): Cupriavidus gilardii strain QJ1 chromosome 1, ... (11% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r90c1_7.svg">
NZ_JAIMCH010000013 (92583-115535): Cupriavidus respiraculi strain AU7532 NODE... (13% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r90c1_8.svg">
NZ_VLJN01000034 (19659-41738): Cupriavidus gilardii J11 CupJ11 scaffold 3, wh... (10% of genes show similarity), terpene
</option>

<option value = "svg/clusterblast_r90c1_9.svg">
NZ_CP110131 (4150748-4171387): Methylorubrum extorquens strain NBC 00036 chro... (12% of genes show similarity), terpene
</option>

</select>
<button id= "clusterblast-r90c1-download">Download graphic</button>
</div>
<div id="clusterblast-r90c1-svg"></div>

</div>
</div>

<div class="body-details-section r90c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-639"></div> <span class="help-tooltip" id="cb-known-help-639">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r90c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-640"></div> <span class="help-tooltip" id="cb-sub-help-640">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r90c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-642"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-642">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r90c1-pfam-details-svg" data-filename="scaffold_90_r90c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r90c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">









</div>









</div>

</div>

</div>





<div class="page" id='r95c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_95 - Region 1 - fungal-RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-871"></div> <span class="help-tooltip" id="region_svg-help-871">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_95.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r95c1-svg" data-filename="scaffold_95_r95c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 37,880 - 117,375 nt. (total: 79,496 nt)

<a class="cluster-rules-header" id="r95c1-rules-header" href="#r95c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r95c1-rules">fungal-RiPP-like: DUF3328</div>

<div class="region-svg-container">
<div id='r95c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-872"></div> <span class="help-tooltip" id="focus-panel-help-872">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r95c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r95c1-gene-table-details" data-name="r95c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r95c1-MIBiG-cluster-compare" data-name="r95c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r95c1-clusterblast" data-name="r95c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r95c1-knownclusterblast" data-name="r95c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r95c1-subclusterblast" data-name="r95c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r95c1-pfam-details" data-name="r95c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r95c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-647"></div> <span class="help-tooltip" id="gt-help-tooltip-help-647">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r95c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r95c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r95c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r95c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r95c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-643"></div> <span class="help-tooltip" id="MIBiG-help-643">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r95c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r95c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r95c1"></div>
<div id="comparison-MIBiG-r95c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">fungal-RiPP-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000627: 0-28295">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000627/index.html#r1c1" target="_new">BGC0000627</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.45, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>RiPP</td>

<td>ustiloxin B</td>


<td>Aspergillus flavus NRRL3357</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000685: 127-25775">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000685/index.html#r1c1" target="_new">BGC0000685</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.46, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>


<td>0.17</td>
<td>Terpene</td>

<td>brassicicene C</td>


<td>Alternaria brassicicola ATCC 96836</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002290: 5128-32193">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002290/index.html#r1c1" target="_new">BGC0002290</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>Other</td>

<td>biscognienyne B, eutypinic acid, compound 3, 4-hydroxy-3-prenylbenzoic acid, prenylhydroquinone, compound 15, compound 8, compound 7, compound 6, compound 11, compound 12, pestalodiol E</td>


<td>Biscogniauxia sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001990: 130-13646">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001990/index.html#r1c1" target="_new">BGC0001990</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>Other</td>

<td>BU-4704, xanthocillin-X monomethylether, melanocin E, melanocin F, fumiformamide</td>


<td>Aspergillus fumigatus Af293</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001896: 0-8329">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001896/index.html#r1c1" target="_new">BGC0001896</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.16</td>
<td>Other</td>

<td>carbazomycin B</td>


<td>Streptomyces luteoverticillatus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001398: 4999-64171">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001398/index.html#r1c1" target="_new">BGC0001398</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.16</td>
<td>RiPP</td>

<td>phomopsin A, phomopsin B, phomopsin E</td>


<td>Phomopsis leptostromiformis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002217: 0-33547">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002217/index.html#r1c1" target="_new">BGC0002217</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.51, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.16</td>
<td>Terpene</td>

<td>HEx-tc3 terpene</td>


<td>Botrytis cinerea T4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002428: 0-1537">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002428/index.html#r1c1" target="_new">BGC0002428</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.46, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.15</td>
<td>Terpene, Polyketide</td>

<td>DDP-13, DDP-14</td>


<td>Fusarium graminearum PH-1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002439: 0-24052">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002439/index.html#r1c1" target="_new">BGC0002439</a></td>





<td class="cc-heat-cell" title="0.43 (id:0.42, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>


<td>0.14</td>
<td>Other</td>

<td>diastaphenazine, izumiphenazine C</td>


<td>Streptomyces diastaticus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002482: 14032-28331">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002482/index.html#r1c1" target="_new">BGC0002482</a></td>





<td class="cc-heat-cell" title="0.42 (id:0.41, order:N/A, components:0.43)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>


<td>0.13</td>
<td>Other</td>

<td>tjipanazole D, tjipanazole I, tjipanazole L, tjipanazole K, tjipanazole M</td>


<td>Symphyonema bifilamentata 97.28</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r95c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000627: 0-28295">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000627/index.html#r1c1" target="_new">BGC0000627</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.45, order:N/A, components:0.53)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>RiPP</td>
<td>ustiloxin B</td>
<td>Aspergillus flavus NRRL3357</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000685: 127-25775">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000685/index.html#r1c1" target="_new">BGC0000685</a></td>


<td class="cc-heat-cell" title="0.48 (id:0.46, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>
<td>0.48</td>
<td>Terpene</td>
<td>brassicicene C</td>
<td>Alternaria brassicicola ATCC 96836</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002290: 5128-32193">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002290/index.html#r1c1" target="_new">BGC0002290</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>Other</td>
<td>biscognienyne B, eutypinic acid, compound 3, 4-hydroxy-3-prenylbenzoic acid, prenylhydroquinone, compound 15, compound 8, compound 7, compound 6, compound 11, compound 12, pestalodiol E</td>
<td>Biscogniauxia sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001990: 130-13646">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001990/index.html#r1c1" target="_new">BGC0001990</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>Other</td>
<td>BU-4704, xanthocillin-X monomethylether, melanocin E, melanocin F, fumiformamide</td>
<td>Aspergillus fumigatus Af293</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001896: 0-8329">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001896/index.html#r1c1" target="_new">BGC0001896</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>Other</td>
<td>carbazomycin B</td>
<td>Streptomyces luteoverticillatus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001398: 4999-64171">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001398/index.html#r1c1" target="_new">BGC0001398</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>RiPP</td>
<td>phomopsin A, phomopsin B, phomopsin E</td>
<td>Phomopsis leptostromiformis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002217: 0-33547">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002217/index.html#r1c1" target="_new">BGC0002217</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.51, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>Terpene</td>
<td>HEx-tc3 terpene</td>
<td>Botrytis cinerea T4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002428: 0-1537">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002428/index.html#r1c1" target="_new">BGC0002428</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.46, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>Terpene, Polyketide</td>
<td>DDP-13, DDP-14</td>
<td>Fusarium graminearum PH-1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002439: 0-24052">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002439/index.html#r1c1" target="_new">BGC0002439</a></td>


<td class="cc-heat-cell" title="0.43 (id:0.42, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
<td>0.43</td>
<td>Other</td>
<td>diastaphenazine, izumiphenazine C</td>
<td>Streptomyces diastaticus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002482: 14032-28331">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002482/index.html#r1c1" target="_new">BGC0002482</a></td>


<td class="cc-heat-cell" title="0.42 (id:0.41, order:N/A, components:0.43)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
<td>0.42</td>
<td>Other</td>
<td>tjipanazole D, tjipanazole I, tjipanazole L, tjipanazole K, tjipanazole M</td>
<td>Symphyonema bifilamentata 97.28</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r95c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-644"></div> <span class="help-tooltip" id="cb-general-help-644">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r95c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r95c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r95c1_1.svg">
NC_072349 (1251120-1365955): Talaromyces marneffei chromosome 2, complete seq... (9% of genes show similarity), T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r95c1_2.svg">
NW_020291495 (218909-310584): Aspergillus aculeatinus CBS 121060 unplaced gen... (7% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r95c1_3.svg">
NW_024469009 (179824-271219): Aspergillus fijiensis CBS 313.89 unplaced genom... (7% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r95c1_4.svg">
NW_022987223 (375403-544458): Trematosphaeria pertusa strain CBS 122368 unpla... (4% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r95c1_5.svg">
NW_024469024 (26525-267638): Aspergillus fijiensis CBS 313.89 unplaced genomi... (3% of genes show similarity), NRPS,T1PKS,fungal-RiPP-like,other
</option>

<option value = "svg/clusterblast_r95c1_6.svg">
NW_017253363 (183481-274714): Paraphaeosphaeria sporulosa strain AP3s5-JAC2a ... (11% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r95c1_7.svg">
NW_025953252 (947231-1128644): Annulohypoxylon maeteangense strain CBS 123835... (4% of genes show similarity), fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r95c1_8.svg">
NW_020292122 (30514-238625): Aspergillus ibericus CBS 121593 unplaced genomic... (3% of genes show similarity), NRPS-like,T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r95c1_9.svg">
NW_020291606 (579604-818783): Aspergillus piperis CBS 112811 unplaced genomic... (4% of genes show similarity), T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r95c1_10.svg">
NW_024468991 (98260-270162): Aspergillus fijiensis CBS 313.89 unplaced genomi... (4% of genes show similarity), NRPS-like,T1PKS,fungal-RiPP-like
</option>

</select>
<button id= "clusterblast-r95c1-download">Download graphic</button>
</div>
<div id="clusterblast-r95c1-svg"></div>

</div>
</div>

<div class="body-details-section r95c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-645"></div> <span class="help-tooltip" id="cb-known-help-645">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r95c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-646"></div> <span class="help-tooltip" id="cb-sub-help-646">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r95c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-648"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-648">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r95c1-pfam-details-svg" data-filename="scaffold_95_r95c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r95c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">











</div>











</div>

</div>

</div>





<div class="page" id='r96c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_96 - Region 1 - NRPS,T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-873"></div> <span class="help-tooltip" id="region_svg-help-873">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_96.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r96c1-svg" data-filename="scaffold_96_r96c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 4,746 - 61,945 nt. (total: 57,200 nt)

<a class="cluster-rules-header" id="r96c1-rules-header" href="#r96c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r96c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))<br>T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r96c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-874"></div> <span class="help-tooltip" id="focus-panel-help-874">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r96c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r96c1-gene-table-details" data-name="r96c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r96c1-nrps_pks" data-name="r96c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r96c1-MIBiG-cluster-compare" data-name="r96c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r96c1-clusterblast" data-name="r96c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r96c1-knownclusterblast" data-name="r96c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r96c1-subclusterblast" data-name="r96c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r96c1-nrps-pks-bubbles" data-name="r96c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r96c1-pfam-details" data-name="r96c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r96c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-657"></div> <span class="help-tooltip" id="gt-help-tooltip-help-657">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r96c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r96c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r96c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r96c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r96c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-649"></div> <span class="help-tooltip" id="nrps-domain-help-649">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r96c1-details-svg" data-filename="scaffold_96_r96c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r96c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r96c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-650"></div> <span class="help-tooltip" id="MIBiG-help-650">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r96c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r96c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r96c1"></div>
<div id="comparison-MIBiG-r96c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>

<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000037: 0-7821">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000037/index.html#r1c1" target="_new">BGC0000037</a></td>





<td class="cc-heat-cell" title="0.70 (id:0.49, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>






<td class="cc-heat-cell" title="0.70 (id:0.49, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>


<td>0.75</td>
<td>Polyketide</td>

<td>cichorine</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000343: 1-6208">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000343/index.html#r1c1" target="_new">BGC0000343</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>






<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.65</td>
<td>NRP</td>

<td>enterobactin</td>


<td>Pseudomonas sp. J465</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001527: 0-8170">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001527/index.html#r1c1" target="_new">BGC0001527</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>






<td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.64</td>
<td>Other</td>

<td>basidioferrin</td>


<td>Gelatoporia subvermispora</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001219: 0-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001219/index.html#r1c1" target="_new">BGC0001219</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.50, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>






<td class="cc-heat-cell" title="0.65 (id:0.50, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.62</td>
<td>Polyketide</td>

<td>shanorellin</td>


<td>Chaetomium globosum CBS 148.51</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001390: 100-19645">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001390/index.html#r1c1" target="_new">BGC0001390</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.44, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>






<td class="cc-heat-cell" title="0.63 (id:0.44, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.58</td>
<td>NRP, Polyketide</td>

<td>LL-Z1272beta</td>


<td>Stachybotrys bisbyi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002002: 0-14395">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002002/index.html#r1c1" target="_new">BGC0002002</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>






<td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.57</td>
<td>NRP</td>

<td>1,6-phenazinedimethanol, fontizine A</td>


<td>Serratia fonticola</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001923: 495-32413">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001923/index.html#r1c1" target="_new">BGC0001923</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>






<td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.54</td>
<td>Terpene, Polyketide</td>

<td>ascochlorin</td>


<td>Acremonium egyptiacum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002614: 1538-32674">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002614/index.html#r1c1" target="_new">BGC0002614</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.45, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>






<td class="cc-heat-cell" title="0.60 (id:0.45, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.54</td>
<td>NRP, Polyketide</td>

<td>subglutinol A, sesquicillin A, colletotrichin, metarhizin A</td>


<td>Penicillium shearii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002639: 0-8541">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002639/index.html#r1c1" target="_new">BGC0002639</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>






<td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.53</td>
<td>NRP</td>

<td>phospho-tyrolose, tyrolose, tyrolitol, tyrocitabine-626, acyl-tyrocitabine-668, acyl-tyrocitabine-696, acyl-tyrocitabine-724, acyl-tyrocitabine-752</td>


<td>Lactobacillus iners LEAF 2052A-d</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001925: 0-20488">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001925/index.html#r1c1" target="_new">BGC0001925</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.43, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>






<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.85)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.52</td>
<td>Alkaloid</td>

<td>altemicidin, SB-203207, SB-203208</td>


<td>Streptomyces sp.</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r96c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000037: 0-7821">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000037/index.html#r1c1" target="_new">BGC0000037</a></td>


<td class="cc-heat-cell" title="0.70 (id:0.49, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
<td>0.70</td>
<td>Polyketide</td>
<td>cichorine</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000343: 1-6208">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000343/index.html#r1c1" target="_new">BGC0000343</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>NRP</td>
<td>enterobactin</td>
<td>Pseudomonas sp. J465</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001527: 0-8170">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001527/index.html#r1c1" target="_new">BGC0001527</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Other</td>
<td>basidioferrin</td>
<td>Gelatoporia subvermispora</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001219: 0-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001219/index.html#r1c1" target="_new">BGC0001219</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.50, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Polyketide</td>
<td>shanorellin</td>
<td>Chaetomium globosum CBS 148.51</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001390: 100-19645">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001390/index.html#r1c1" target="_new">BGC0001390</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.44, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>NRP, Polyketide</td>
<td>LL-Z1272beta</td>
<td>Stachybotrys bisbyi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002639: 0-8541">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002639/index.html#r1c1" target="_new">BGC0002639</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>NRP</td>
<td>phospho-tyrolose, tyrolose, tyrolitol, tyrocitabine-626, acyl-tyrocitabine-668, acyl-tyrocitabine-696, acyl-tyrocitabine-724, acyl-tyrocitabine-752</td>
<td>Lactobacillus iners LEAF 2052A-d</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002002: 0-14395">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002002/index.html#r1c1" target="_new">BGC0002002</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>NRP</td>
<td>1,6-phenazinedimethanol, fontizine A</td>
<td>Serratia fonticola</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001720: 0-45622">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001720/index.html#r1c1" target="_new">BGC0001720</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.43, order:N/A, components:0.87)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>Polyketide</td>
<td>oosporein</td>
<td>Beauveria bassiana ARSEF 2860</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001923: 495-32413">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001923/index.html#r1c1" target="_new">BGC0001923</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>Terpene, Polyketide</td>
<td>ascochlorin</td>
<td>Acremonium egyptiacum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002614: 1538-32674">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002614/index.html#r1c1" target="_new">BGC0002614</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.45, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>NRP, Polyketide</td>
<td>subglutinol A, sesquicillin A, colletotrichin, metarhizin A</td>
<td>Penicillium shearii</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r96c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-651"></div> <span class="help-tooltip" id="cb-general-help-651">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r96c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r96c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r96c1_1.svg">
NW_003052500 (6028872-6104884): Uncinocarpus reesii 1704 scaffold 1 genomic s... (39% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r96c1_2.svg">
NW_025091283 (4319746-4671991): Aspergillus pseudoviridinutans IFM 55266 DNA,... (5% of genes show similarity), NRPS,NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r96c1_3.svg">
NZ_CP082362 (1463311-1580018): Streptomyces sp. MA3 2.13 chromosome, complete... (17% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r96c1_4.svg">
NZ_CP094483 (8040232-8198040): Streptomyces murinus strain JS029 chromosome (10% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r96c1_5.svg">
NZ_CM001776 (4822455-4922007): Synechococcus sp. PCC 7336 chromosome, whole g... (10% of genes show similarity), NRPS,NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r96c1_6.svg">
NZ_CP046623 (4135240-4293702): Streptomyces murinus strain CR-43 chromosome, ... (12% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r96c1_7.svg">
NC_020504 (699547-770346): Streptomyces davaonensis JCM 4913, complete sequence (8% of genes show similarity), NRPS,NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r96c1_8.svg">
NZ_CP104794 (8237716-8308417): Streptomyces lusitanus strain FZ201 chromosome... (6% of genes show similarity), NRPS,NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r96c1_9.svg">
NZ_JH992893 (248696-437096): Fischerella sp. PCC 9339 PCC9339DRAFT scaffold2.... (3% of genes show similarity), NRPS,NRPS-like,T1PKS,other
</option>

<option value = "svg/clusterblast_r96c1_10.svg">
NW_024468909 (49034-180513): Cryphonectria parasitica EP155 unplaced genomic ... (9% of genes show similarity), NRPS,T1PKS
</option>

</select>
<button id= "clusterblast-r96c1-download">Download graphic</button>
</div>
<div id="clusterblast-r96c1-svg"></div>

</div>
</div>

<div class="body-details-section r96c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-652"></div> <span class="help-tooltip" id="cb-known-help-652">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r96c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r96c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r96c1_1.svg">
cichorine
</option>

</select>
<button id="knownclusterblast-r96c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r96c1-svg"></div>

</div>
</div>

<div class="body-details-section r96c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-653"></div> <span class="help-tooltip" id="cb-sub-help-653">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r96c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-656"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-656">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r96c1-domain-bubble-svg" data-filename="scaffold_96_r96c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r96c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (4745 - 61945): neighbouring NRPS-T1PKS
</option>



<option value="CC2">
Candidate 2 (4745 - 47520): single NRPS
</option>



<option value="CC3">
Candidate 3 (18252 - 61945): single T1PKS
</option>


</select>

<div class="details-svg" id="r96c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r96c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r96c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-658"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-658">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r96c1-pfam-details-svg" data-filename="scaffold_96_r96c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r96c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r96c1-nrps_pks_products" data-name="r96c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r96c1-nrps_pks_monomers" data-name="r96c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r96c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-654"></div> <span class="help-tooltip" id="nrps-structures-help-654">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 4745 - 61945:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r96c1-1" class="smiles-canvas" data-smiles="NCC(=O)CC(=O)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Gly) + (mal)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r96c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NCC(=O)CC(=O)C(=O)O"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 2, location 4745 - 47520:
<div class="collapser collapser-target-CC2 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r96c1-2" class="smiles-canvas" data-smiles="NCC(=O)CC(=O)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Gly) + (mal)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r96c1-2-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NCC(=O)CC(=O)C(=O)O"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 3, location 18252 - 61945:
<div class="collapser collapser-target-CC3 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r96c1-3" class="smiles-canvas" data-smiles="NCC(=O)CC(=O)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Gly) + (mal)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r96c1-3-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NCC(=O)CC(=O)C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r96c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-655"></div> <span class="help-tooltip" id="nrps-monomers-help-655">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">FUN_009780</span></strong>: X
<div class="collapser collapser-target-FUN_009780 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (15..353)</strong>: X
<div class="collapser collapser-target-FUN_009780 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  (unknown)
<div class="collapser collapser-target-FUN_009780-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>N/A</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">--FQLGGSIK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>

<dt>Stachelhaus code match:</dt>
<dd>0% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>


</div>
<br>

<strong><span class="serif">FUN_009783</span></strong>: Gly
<div class="collapser collapser-target-FUN_009783 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly*" target="_new">relaxed</a>
</dd>
<br>



<dd><strong>AMP-binding (269..662)</strong>: Gly
<div class="collapser collapser-target-FUN_009783 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Gly
<div class="collapser collapser-target-FUN_009783-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Apolar, aliphatic (Gly, Ala, Val, Leu, Ile, Abu, Ival)</dd>
<dt>Small clusters prediction:</dt>
<dd>Tiny, hydrophilic, transition to aliphatic (Gly, Ala)</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DLGCLGMIWK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Gly <span class="serif">DLDQLGMIWK</span> (38% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>80% (moderate)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

<strong><span class="serif">FUN_009786</span></strong>: mal
<div class="collapser collapser-target-FUN_009786 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (328..625)</strong>: mal
<div class="collapser collapser-target-FUN_009786 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-FUN_009786-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 58.3%</dd>
<dd>Propionyl-CoA: 54.2%</dd>
<dd>inactive: 54.2%</dd>
</dl>

</div><br>

minowa:  Malonyl-CoA
<div class="collapser collapser-target-FUN_009786-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Malonyl-CoA: 60.3</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 45.4</dt>

<dd></dd><dt>inactive: 40.4</dt>

<dd></dd><dt>Methylmalonyl-CoA: 36.2</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 19.4</dt>

<dd></dd><dt>fatty_acid: 18.8</dt>

<dd></dd><dt>Propionyl-CoA: 16.7</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 15.7</dt>

<dd></dd><dt>trans-1,2-CPDA: 15.3</dt>

<dd></dd><dt>Benzoyl-CoA: 14.1</dt>

<dd></dd><dt>CHC-CoA: 14.0</dt>

<dd></dd><dt>Isobutyryl-CoA: 0.0</dt>

<dd></dd><dt>Acetyl-CoA: 0.0</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>





<div class="page" id='r97c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_97 - Region 1 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-875"></div> <span class="help-tooltip" id="region_svg-help-875">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_97.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r97c1-svg" data-filename="scaffold_97_r97c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 48,698 - 96,542 nt. (total: 47,845 nt)

<a class="cluster-rules-header" id="r97c1-rules-header" href="#r97c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r97c1-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r97c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-876"></div> <span class="help-tooltip" id="focus-panel-help-876">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r97c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r97c1-gene-table-details" data-name="r97c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r97c1-nrps_pks" data-name="r97c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r97c1-MIBiG-cluster-compare" data-name="r97c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r97c1-clusterblast" data-name="r97c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r97c1-knownclusterblast" data-name="r97c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r97c1-subclusterblast" data-name="r97c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r97c1-nrps-pks-bubbles" data-name="r97c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r97c1-pfam-details" data-name="r97c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r97c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-667"></div> <span class="help-tooltip" id="gt-help-tooltip-help-667">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r97c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r97c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r97c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r97c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r97c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-659"></div> <span class="help-tooltip" id="nrps-domain-help-659">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r97c1-details-svg" data-filename="scaffold_97_r97c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r97c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r97c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-660"></div> <span class="help-tooltip" id="MIBiG-help-660">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r97c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r97c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r97c1"></div>
<div id="comparison-MIBiG-r97c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002191: 260-8785">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002191/index.html#r1c1" target="_new">BGC0002191</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.32</td>
<td>Polyketide</td>

<td>prolipyrone B, gibepyrone D</td>


<td>Fusarium graminearum PH-1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001252: 0-7958">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001252/index.html#r1c1" target="_new">BGC0001252</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.30</td>
<td>Polyketide</td>

<td>UNII-YC2Q1O94PT</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002155: 78-16511">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002155/index.html#r1c1" target="_new">BGC0002155</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.30</td>
<td>Polyketide</td>

<td>nectriapyrone C, nectriapyrone D, nectriapyrone</td>


<td>Pyricularia oryzae 70-15</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001606: 0-22544">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001606/index.html#r1c1" target="_new">BGC0001606</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.30</td>
<td>Polyketide</td>

<td>gibepyrone-A</td>


<td>Fusarium fujikuroi IMI 58289</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002525: 1000-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002525/index.html#r1c1" target="_new">BGC0002525</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.29</td>
<td>Polyketide</td>

<td>fusarubin, 1233A, 1233B, NG-391, lucilactaene</td>


<td>Fusarium sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002222: 888-32276">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002222/index.html#r1c1" target="_new">BGC0002222</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.28</td>
<td>Polyketide</td>

<td>zopfiellin</td>


<td>Diffractella curvata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002224: 685-37748">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002224/index.html#r1c1" target="_new">BGC0002224</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.26</td>
<td>Polyketide</td>

<td>zopfiellin</td>


<td>Diffractella curvata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000146: 754-20637">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000146/index.html#r1c1" target="_new">BGC0000146</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.25</td>
<td>Polyketide</td>

<td>solanapyrone D</td>


<td>Alternaria solani</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002515: 0-30087">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002515/index.html#r1c1" target="_new">BGC0002515</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.42, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>Polyketide</td>

<td>solanapyrone A</td>


<td>Ascochyta rabiei</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001899: 0-54810">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001899/index.html#r1c1" target="_new">BGC0001899</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>Polyketide</td>

<td>abscisic acid</td>


<td>Leptosphaeria maculans JN3</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r97c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002191: 260-8785">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002191/index.html#r1c1" target="_new">BGC0002191</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Polyketide</td>
<td>prolipyrone B, gibepyrone D</td>
<td>Fusarium graminearum PH-1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001252: 0-7958">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001252/index.html#r1c1" target="_new">BGC0001252</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>Polyketide</td>
<td>UNII-YC2Q1O94PT</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002155: 78-16511">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002155/index.html#r1c1" target="_new">BGC0002155</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>Polyketide</td>
<td>nectriapyrone C, nectriapyrone D, nectriapyrone</td>
<td>Pyricularia oryzae 70-15</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001606: 0-22544">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001606/index.html#r1c1" target="_new">BGC0001606</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.43, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>Polyketide</td>
<td>gibepyrone-A</td>
<td>Fusarium fujikuroi IMI 58289</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002525: 1000-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002525/index.html#r1c1" target="_new">BGC0002525</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>Polyketide</td>
<td>fusarubin, 1233A, 1233B, NG-391, lucilactaene</td>
<td>Fusarium sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002222: 888-32276">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002222/index.html#r1c1" target="_new">BGC0002222</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>Polyketide</td>
<td>zopfiellin</td>
<td>Diffractella curvata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002224: 685-37748">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002224/index.html#r1c1" target="_new">BGC0002224</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>Polyketide</td>
<td>zopfiellin</td>
<td>Diffractella curvata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000146: 754-20637">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000146/index.html#r1c1" target="_new">BGC0000146</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Polyketide</td>
<td>solanapyrone D</td>
<td>Alternaria solani</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002515: 0-30087">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002515/index.html#r1c1" target="_new">BGC0002515</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.42, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Polyketide</td>
<td>solanapyrone A</td>
<td>Ascochyta rabiei</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001899: 0-54810">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001899/index.html#r1c1" target="_new">BGC0001899</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Polyketide</td>
<td>abscisic acid</td>
<td>Leptosphaeria maculans JN3</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r97c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-661"></div> <span class="help-tooltip" id="cb-general-help-661">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r97c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r97c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r97c1_1.svg">
NW_025953058 (130642-226995): Daldinia caldariorum strain CBS 122874 unplaced... (31% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r97c1_2.svg">
NW_025953830 (168631-237212): Annulohypoxylon truncatum strain CBS 140777 unp... (19% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r97c1_3.svg">
NW_022987215 (251333-319784): Trematosphaeria pertusa strain CBS 122368 unpla... (16% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r97c1_4.svg">
NW_022985444 (1515152-1623613): Mytilinidion resinicola strain CBS 304.34 unp... (9% of genes show similarity), T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r97c1_5.svg">
NW_025953261 (590289-740142): Annulohypoxylon maeteangense strain CBS 123835 ... (8% of genes show similarity), T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r97c1_6.svg">
NW_020291817 (6503-103890): Aspergillus costaricaensis CBS 115574 unplaced ge... (22% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r97c1_7.svg">
NW_006917097 (1595225-1663737): Pestalotiopsis fici W106-1 unplaced genomic s... (22% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r97c1_8.svg">
NW_022474215 (790980-859315): Venustampulla echinocandica strain BP 5553 chro... (14% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r97c1_9.svg">
NW_020290890 (450315-638141): Aspergillus niger CBS 101883 unplaced genomic s... (4% of genes show similarity), NRPS-like,T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r97c1_10.svg">
NW_022987219 (684623-752686): Trematosphaeria pertusa strain CBS 122368 unpla... (10% of genes show similarity), T1PKS
</option>

</select>
<button id= "clusterblast-r97c1-download">Download graphic</button>
</div>
<div id="clusterblast-r97c1-svg"></div>

</div>
</div>

<div class="body-details-section r97c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-662"></div> <span class="help-tooltip" id="cb-known-help-662">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r97c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-663"></div> <span class="help-tooltip" id="cb-sub-help-663">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r97c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-666"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-666">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r97c1-domain-bubble-svg" data-filename="scaffold_97_r97c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r97c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (48697 - 96542): single T1PKS
</option>


</select>

<div class="details-svg" id="r97c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r97c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r97c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-668"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-668">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r97c1-pfam-details-svg" data-filename="scaffold_97_r97c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r97c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r97c1-nrps_pks_products" data-name="r97c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r97c1-nrps_pks_monomers" data-name="r97c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r97c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-664"></div> <span class="help-tooltip" id="nrps-structures-help-664">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 48697 - 96542:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r97c1-1" class="smiles-canvas" data-smiles="C([*])C(-O)(C)"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Me-pk)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-pk" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-pk*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r97c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)(C)"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r97c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-665"></div> <span class="help-tooltip" id="nrps-monomers-help-665">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">FUN_009826</span></strong>: pk
<div class="collapser collapser-target-FUN_009826 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (571..804)</strong>: pk
<div class="collapser collapser-target-FUN_009826 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-FUN_009826-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 66.7%</dd>
<dd>Methylmalonyl-CoA: 54.2%</dd>
</dl>

</div><br>

minowa:  Methoxymalonyl-CoA
<div class="collapser collapser-target-FUN_009826-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methoxymalonyl-CoA: 57.2</dt>

<dd></dd><dt>Methylmalonyl-CoA: 55.6</dt>

<dd></dd><dt>Malonyl-CoA: 43.7</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 37.7</dt>

<dd></dd><dt>Isobutyryl-CoA: 23.3</dt>

<dd></dd><dt>CHC-CoA: 21.0</dt>

<dd></dd><dt>Propionyl-CoA: 18.4</dt>

<dd></dd><dt>fatty_acid: 18.1</dt>

<dd></dd><dt>trans-1,2-CPDA: 17.2</dt>

<dd></dd><dt>inactive: 14.6</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 12.9</dt>

<dd></dd><dt>Benzoyl-CoA: 12.4</dt>

<dd></dd><dt>Acetyl-CoA: 0.0</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>












</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>





<div class="page" id='r98c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_98 - Region 1 - indole <div class="help-container"> <div class="help-icon" data-id="region_svg-help-877"></div> <span class="help-tooltip" id="region_svg-help-877">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_98.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r98c1-svg" data-filename="scaffold_98_r98c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 13,622 - 36,644 nt. (total: 23,023 nt)

<a class="cluster-rules-header" id="r98c1-rules-header" href="#r98c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r98c1-rules">indole: (indsynth or dmat or indole_PTase)</div>

<div class="region-svg-container">
<div id='r98c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-878"></div> <span class="help-tooltip" id="focus-panel-help-878">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r98c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r98c1-gene-table-details" data-name="r98c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r98c1-MIBiG-cluster-compare" data-name="r98c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r98c1-clusterblast" data-name="r98c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r98c1-knownclusterblast" data-name="r98c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r98c1-subclusterblast" data-name="r98c1-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r98c1-pfam-details" data-name="r98c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r98c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-673"></div> <span class="help-tooltip" id="gt-help-tooltip-help-673">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r98c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r98c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r98c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r98c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r98c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-669"></div> <span class="help-tooltip" id="MIBiG-help-669">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r98c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r98c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r98c1"></div>
<div id="comparison-MIBiG-r98c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">indole</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001294: 0-9019">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001294/index.html#r1c1" target="_new">BGC0001294</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.23</td>
<td>Other</td>

<td>7-prenylisatin</td>


<td>Streptomyces sp. MBT28</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001483: 0-25255">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001483/index.html#r1c1" target="_new">BGC0001483</a></td>





<td class="cc-heat-cell" title="0.29 (id:0.43, order:N/A, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>


<td>0.07</td>
<td>Other</td>

<td>5-isoprenylindole-3-carboxylate -D-glycosyl ester</td>


<td>Streptomyces sp. RM-5-8</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001775: 2711-20673">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001775/index.html#r1c1" target="_new">BGC0001775</a></td>





<td class="cc-heat-cell" title="0.27 (id:0.43, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>


<td>0.06</td>
<td>Terpene</td>

<td>sespendole</td>


<td>Cordana terrestris</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000846: 192-22933">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000846/index.html#r1c1" target="_new">BGC0000846</a></td>





<td class="cc-heat-cell" title="0.23 (id:0.43, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>


<td>0.05</td>
<td>Other</td>

<td>tabtoxin</td>


<td>Pseudomonas syringae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002229: 510-22108">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002229/index.html#r1c1" target="_new">BGC0002229</a></td>





<td class="cc-heat-cell" title="0.23 (id:0.46, order:N/A, components:0.11)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>


<td>0.05</td>
<td>Polyketide</td>

<td>6-MSA, aculin A</td>


<td>Aspergillus aculeatus ATCC 16872</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264: 8442-40380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264</a></td>





<td class="cc-heat-cell" title="0.21 (id:0.44, order:N/A, components:0.10)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.21), rgb(0, 0, 0, 0.21) 21%, white 21%)"></td>


<td>0.05</td>
<td>Polyketide</td>

<td>betaenone A, betaenone B, betaenone C</td>


<td>Phoma betae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001909: 2476-67696">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001909/index.html#r1c1" target="_new">BGC0001909</a></td>





<td class="cc-heat-cell" title="0.21 (id:0.43, order:N/A, components:0.10)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.21), rgb(0, 0, 0, 0.21) 21%, white 21%)"></td>


<td>0.05</td>
<td>Polyketide</td>

<td>strobilurin A</td>


<td>Strobilurus tenacellus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000160: 0-27093">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000160/index.html#r1c1" target="_new">BGC0000160</a></td>





<td class="cc-heat-cell" title="0.19 (id:0.43, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.19), rgb(0, 0, 0, 0.19) 19%, white 19%)"></td>


<td>0.04</td>
<td>Polyketide</td>

<td>terreic acid</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001722: 0-18441">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001722/index.html#r1c1" target="_new">BGC0001722</a></td>





<td class="cc-heat-cell" title="0.19 (id:0.43, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.19), rgb(0, 0, 0, 0.19) 19%, white 19%)"></td>


<td>0.04</td>
<td>Polyketide</td>

<td>aspernidine A</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002213: 0-33851">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002213/index.html#r1c1" target="_new">BGC0002213</a></td>





<td class="cc-heat-cell" title="0.19 (id:0.43, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.19), rgb(0, 0, 0, 0.19) 19%, white 19%)"></td>


<td>0.04</td>
<td>Polyketide</td>

<td>HEx-pks16 polyketide</td>


<td>Punctularia strigosozonata HHB-11173 SS5</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r98c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001294: 0-9019">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001294/index.html#r1c1" target="_new">BGC0001294</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Other</td>
<td>7-prenylisatin</td>
<td>Streptomyces sp. MBT28</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001483: 0-25255">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001483/index.html#r1c1" target="_new">BGC0001483</a></td>


<td class="cc-heat-cell" title="0.29 (id:0.43, order:N/A, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.29), rgb(0, 0, 0, 0.29) 29%, white 29%)"></td>
<td>0.29</td>
<td>Other</td>
<td>5-isoprenylindole-3-carboxylate -D-glycosyl ester</td>
<td>Streptomyces sp. RM-5-8</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001775: 2711-20673">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001775/index.html#r1c1" target="_new">BGC0001775</a></td>


<td class="cc-heat-cell" title="0.27 (id:0.43, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>
<td>0.27</td>
<td>Terpene</td>
<td>sespendole</td>
<td>Cordana terrestris</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000846: 192-22933">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000846/index.html#r1c1" target="_new">BGC0000846</a></td>


<td class="cc-heat-cell" title="0.23 (id:0.43, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>
<td>0.23</td>
<td>Other</td>
<td>tabtoxin</td>
<td>Pseudomonas syringae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002229: 510-22108">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002229/index.html#r1c1" target="_new">BGC0002229</a></td>


<td class="cc-heat-cell" title="0.23 (id:0.46, order:N/A, components:0.11)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.23), rgb(0, 0, 0, 0.23) 23%, white 23%)"></td>
<td>0.23</td>
<td>Polyketide</td>
<td>6-MSA, aculin A</td>
<td>Aspergillus aculeatus ATCC 16872</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001264: 8442-40380">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001264/index.html#r1c1" target="_new">BGC0001264</a></td>


<td class="cc-heat-cell" title="0.21 (id:0.44, order:N/A, components:0.10)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.21), rgb(0, 0, 0, 0.21) 21%, white 21%)"></td>
<td>0.21</td>
<td>Polyketide</td>
<td>betaenone A, betaenone B, betaenone C</td>
<td>Phoma betae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001909: 2476-67696">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001909/index.html#r1c1" target="_new">BGC0001909</a></td>


<td class="cc-heat-cell" title="0.21 (id:0.43, order:N/A, components:0.10)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.21), rgb(0, 0, 0, 0.21) 21%, white 21%)"></td>
<td>0.21</td>
<td>Polyketide</td>
<td>strobilurin A</td>
<td>Strobilurus tenacellus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000160: 0-27093">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000160/index.html#r1c1" target="_new">BGC0000160</a></td>


<td class="cc-heat-cell" title="0.19 (id:0.43, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.19), rgb(0, 0, 0, 0.19) 19%, white 19%)"></td>
<td>0.19</td>
<td>Polyketide</td>
<td>terreic acid</td>
<td>Aspergillus terreus NIH2624</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001722: 0-18441">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001722/index.html#r1c1" target="_new">BGC0001722</a></td>


<td class="cc-heat-cell" title="0.19 (id:0.43, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.19), rgb(0, 0, 0, 0.19) 19%, white 19%)"></td>
<td>0.19</td>
<td>Polyketide</td>
<td>aspernidine A</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002213: 0-33851">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002213/index.html#r1c1" target="_new">BGC0002213</a></td>


<td class="cc-heat-cell" title="0.19 (id:0.43, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.19), rgb(0, 0, 0, 0.19) 19%, white 19%)"></td>
<td>0.19</td>
<td>Polyketide</td>
<td>HEx-pks16 polyketide</td>
<td>Punctularia strigosozonata HHB-11173 SS5</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r98c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-670"></div> <span class="help-tooltip" id="cb-general-help-670">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r98c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r98c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r98c1_1.svg">
NW_025953797 (65854-98346): Annulohypoxylon truncatum strain CBS 140777 unpla... (25% of genes show similarity), indole
</option>

<option value = "svg/clusterblast_r98c1_2.svg">
NW_025953041 (871065-903550): Daldinia caldariorum strain CBS 122874 unplaced... (15% of genes show similarity), indole
</option>

</select>
<button id= "clusterblast-r98c1-download">Download graphic</button>
</div>
<div id="clusterblast-r98c1-svg"></div>

</div>
</div>

<div class="body-details-section r98c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-671"></div> <span class="help-tooltip" id="cb-known-help-671">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r98c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-672"></div> <span class="help-tooltip" id="cb-sub-help-672">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r98c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-674"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-674">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r98c1-pfam-details-svg" data-filename="scaffold_98_r98c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r98c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">









</div>









</div>

</div>

</div>





<div class="page" id='r103c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_103 - Region 1 - NRPS,T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-879"></div> <span class="help-tooltip" id="region_svg-help-879">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_103.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r103c1-svg" data-filename="scaffold_103_r103c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 35,586 - 94,714 nt. (total: 59,129 nt)

<a class="cluster-rules-header" id="r103c1-rules-header" href="#r103c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r103c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))<br>T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r103c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-880"></div> <span class="help-tooltip" id="focus-panel-help-880">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r103c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r103c1-gene-table-details" data-name="r103c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r103c1-nrps_pks" data-name="r103c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r103c1-MIBiG-cluster-compare" data-name="r103c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r103c1-clusterblast" data-name="r103c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r103c1-knownclusterblast" data-name="r103c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r103c1-subclusterblast" data-name="r103c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r103c1-nrps-pks-bubbles" data-name="r103c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r103c1-pfam-details" data-name="r103c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r103c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-683"></div> <span class="help-tooltip" id="gt-help-tooltip-help-683">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r103c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r103c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r103c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r103c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r103c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-675"></div> <span class="help-tooltip" id="nrps-domain-help-675">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r103c1-details-svg" data-filename="scaffold_103_r103c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r103c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r103c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-676"></div> <span class="help-tooltip" id="MIBiG-help-676">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r103c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r103c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r103c1"></div>
<div id="comparison-MIBiG-r103c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>

<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002257: 1039-20720">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002257/index.html#r1c1" target="_new">BGC0002257</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.62, order:0.62, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>






<td class="cc-heat-cell" title="0.77 (id:0.62, order:0.73, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.77), rgb(0, 0, 0, 0.77) 77%, white 77%)"></td>


<td>0.72</td>
<td>Polyketide</td>

<td>rufoschweinitzin</td>


<td>Xylaria schweinitzii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001990: 130-13646">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001990/index.html#r1c1" target="_new">BGC0001990</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>






<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.66</td>
<td>Other</td>

<td>BU-4704, xanthocillin-X monomethylether, melanocin E, melanocin F, fumiformamide</td>


<td>Aspergillus fumigatus Af293</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002610: 0-6468">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002610/index.html#r1c1" target="_new">BGC0002610</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>






<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.64</td>
<td>Terpene</td>

<td>stellatic acid</td>


<td>Aspergillus stellatus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002245: 219-11090">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002245/index.html#r1c1" target="_new">BGC0002245</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>






<td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.51</td>
<td>Terpene</td>

<td>terpestacin, preterpestacin 3, preterpestacin 2, preterpestacin 1</td>


<td>Bipolaris maydis C5</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001528: 2981-16881">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001528/index.html#r1c1" target="_new">BGC0001528</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>






<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.49</td>
<td>Terpene</td>

<td>betaestacin</td>


<td>Phoma betae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000156: 500-13275">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000156/index.html#r1c1" target="_new">BGC0000156</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.50, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>






<td class="cc-heat-cell" title="0.63 (id:0.49, order:0.51, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.46</td>
<td>Polyketide</td>

<td>TAN-1612, 1-(2,3,5,10-tetrahydroxy-7-methoxy-4-oxo-1,2,3,4-tetrahydroanthracen-2-yl)pentane-2,4-dione, desmethyl TAN-1612</td>


<td>Aspergillus niger</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002278: 0-28520">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002278/index.html#r1c1" target="_new">BGC0002278</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>






<td class="cc-heat-cell" title="0.43 (id:0.44, order:N/A, components:0.41)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>


<td>0.42</td>
<td>Alkaloid, NRP</td>

<td>brasiliamide I, brasiliamide A, brasiliamide B, 2,2-dibenzylpiperazine, brasiliamide D</td>


<td>Penicillium brasilianum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000631: 1094-28776">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000631/index.html#r1c1" target="_new">BGC0000631</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.46, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>






<td class="cc-heat-cell" title="0.53 (id:0.46, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.42</td>
<td>Terpene</td>

<td>botcinin H, botcinin I, botcinin J, cinbotolide D, botcinin K</td>


<td>Botrytis cinerea B05.10</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000013: 0-7849">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000013/index.html#r1c1" target="_new">BGC0000013</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.47, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>






<td class="cc-heat-cell" title="0.60 (id:0.47, order:0.68, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.41</td>
<td>Polyketide</td>

<td>alternariol</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001266: 10707-26494">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001266/index.html#r1c1" target="_new">BGC0001266</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>






<td class="cc-heat-cell" title="0.59 (id:0.44, order:0.46, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.41</td>
<td>Polyketide</td>

<td>grayanic acid</td>


<td>Cladonia grayi</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r103c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002257: 1039-20720">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002257/index.html#r1c1" target="_new">BGC0002257</a></td>


<td class="cc-heat-cell" title="0.77 (id:0.62, order:0.73, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.77), rgb(0, 0, 0, 0.77) 77%, white 77%)"></td>
<td>0.77</td>
<td>Polyketide</td>
<td>rufoschweinitzin</td>
<td>Xylaria schweinitzii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001284: 0-5292">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001284/index.html#r1c1" target="_new">BGC0001284</a></td>


<td class="cc-heat-cell" title="0.69 (id:0.48, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
<td>0.69</td>
<td>Polyketide</td>
<td>alternariol</td>
<td>Parastagonospora nodorum SN15</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001258: 43-7265">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001258/index.html#r1c1" target="_new">BGC0001258</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>Polyketide</td>
<td>1,3,6,8-tetrahydroxynaphthalene</td>
<td>Glarea lozoyensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001257: 399-7066">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001257/index.html#r1c1" target="_new">BGC0001257</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>Polyketide</td>
<td>1,3,6,8-tetrahydroxynaphthalene</td>
<td>Nodulisporium sp. ATCC74245</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001906: 35-64276">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001906/index.html#r1c1" target="_new">BGC0001906</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>Polyketide</td>
<td>naphthalene</td>
<td>Daldinia eschscholzii IFB-TL01</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002154: 337-7126">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002154/index.html#r1c1" target="_new">BGC0002154</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>Polyketide</td>
<td>1,3,5,8-tetrahydroxynapthalene, 1-(alpha-l-(2-O-methyl)-6-deoxymannopyranosyloxy)-3,6,8-trimethoxynaphthalene</td>
<td>Pyricularia oryzae 70-15</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002175: 214-6865">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002175/index.html#r1c1" target="_new">BGC0002175</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>Polyketide</td>
<td>YWA1</td>
<td>Aspergillus oryzae RIB40</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000107: 144-6819">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000107/index.html#r1c1" target="_new">BGC0000107</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>Polyketide</td>
<td>naphthopyrone</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001990: 130-13646">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001990/index.html#r1c1" target="_new">BGC0001990</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Other</td>
<td>BU-4704, xanthocillin-X monomethylether, melanocin E, melanocin F, fumiformamide</td>
<td>Aspergillus fumigatus Af293</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002610: 0-6468">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002610/index.html#r1c1" target="_new">BGC0002610</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Terpene</td>
<td>stellatic acid</td>
<td>Aspergillus stellatus</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r103c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-677"></div> <span class="help-tooltip" id="cb-general-help-677">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r103c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r103c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r103c1_1.svg">
NW_017263603 (640381-705945): Mollisia scopiformis unplaced genomic scaffold ... (16% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r103c1_2.svg">
NW_024470055 (1482972-1548460): Aureobasidium melanogenum CBS 110374 unplaced... (12% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r103c1_3.svg">
NW_020292039 (3332166-3473157): Aspergillus vadensis CBS 113365 unplaced geno... (5% of genes show similarity), NRPS,T1PKS,fungal-RiPP-like
</option>

<option value = "svg/clusterblast_r103c1_4.svg">
NW_022987223 (800693-866219): Trematosphaeria pertusa strain CBS 122368 unpla... (10% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r103c1_5.svg">
NC_054857 (5013535-5079145): Aspergillus puulaauensis MK2 DNA, chromosome 1, ... (9% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r103c1_6.svg">
NW_024467860 (897313-962877): Aspergillus sydowii CBS 593.65 unplaced genomic... (11% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r103c1_7.svg">
NW_024467524 (911675-977254): Aspergillus versicolor CBS 583.65 unplaced geno... (12% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r103c1_8.svg">
NC_066264 (4357683-4423245): Aspergillus nidulans FGSC A4 chromosome VIII (9% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r103c1_9.svg">
NW_025091268 (170405-235928): Aspergillus udagawae IFM 46973 DNA, scaffold 5,... (11% of genes show similarity), T1PKS
</option>

<option value = "svg/clusterblast_r103c1_10.svg">
NC_007197 (3806930-3872461): Aspergillus fumigatus Af293 chromosome 4, whole ... (10% of genes show similarity), T1PKS
</option>

</select>
<button id= "clusterblast-r103c1-download">Download graphic</button>
</div>
<div id="clusterblast-r103c1-svg"></div>

</div>
</div>

<div class="body-details-section r103c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-678"></div> <span class="help-tooltip" id="cb-known-help-678">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r103c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r103c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r103c1_1.svg">
rufoschweinitzin
</option>

<option value = "svg/knownclusterblast_r103c1_2.svg">
geodin
</option>

<option value = "svg/knownclusterblast_r103c1_3.svg">
trypacidin
</option>

<option value = "svg/knownclusterblast_r103c1_4.svg">
cryptosporioptide B/cryptosporioptide A/cryptosporioptide C
</option>

<option value = "svg/knownclusterblast_r103c1_5.svg">
RES-1214-2
</option>

<option value = "svg/knownclusterblast_r103c1_6.svg">
4-chloropinselin/pinselin/chloromonilinic acid B/chloromonilinic acid D/4-hydroxyvertixanthone
</option>

<option value = "svg/knownclusterblast_r103c1_7.svg">
neosartorin
</option>

<option value = "svg/knownclusterblast_r103c1_8.svg">
secalonic acids
</option>

<option value = "svg/knownclusterblast_r103c1_9.svg">
3&#39;-methoxy-1,2-dehydropenicillide/pestalotiollide B/pestalotiollide C/1,2-dehydropenicillide/3-methoxy-1,2-dehydropenicillide/1,2-epoxy-3,4-didehydropenicillide
</option>

<option value = "svg/knownclusterblast_r103c1_10.svg">
agnestin A/agnestin B
</option>

</select>
<button id="knownclusterblast-r103c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r103c1-svg"></div>

</div>
</div>

<div class="body-details-section r103c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-679"></div> <span class="help-tooltip" id="cb-sub-help-679">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r103c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-682"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-682">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r103c1-domain-bubble-svg" data-filename="scaffold_103_r103c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r103c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (35585 - 94714): neighbouring NRPS-T1PKS
</option>



<option value="CC2">
Candidate 2 (35585 - 79254): single NRPS
</option>



<option value="CC3">
Candidate 3 (55789 - 94714): single T1PKS
</option>


</select>

<div class="details-svg" id="r103c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r103c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r103c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-684"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-684">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r103c1-pfam-details-svg" data-filename="scaffold_103_r103c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r103c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r103c1-nrps_pks_products" data-name="r103c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r103c1-nrps_pks_monomers" data-name="r103c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r103c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-680"></div> <span class="help-tooltip" id="nrps-structures-help-680">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 35585 - 94714:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r103c1-1" class="smiles-canvas" data-smiles="CC(=O)NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(mal) + (X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps2=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r103c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="CC(=O)NC([*])C(=O)O"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 2, location 35585 - 79254:
<div class="collapser collapser-target-CC2 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r103c1-2" class="smiles-canvas" data-smiles="CC(=O)NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(mal) + (X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps2=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r103c1-2-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="CC(=O)NC([*])C(=O)O"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 3, location 55789 - 94714:
<div class="collapser collapser-target-CC3 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r103c1-3" class="smiles-canvas" data-smiles="CC(=O)NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(mal) + (X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps2=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r103c1-3-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="CC(=O)NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r103c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-681"></div> <span class="help-tooltip" id="nrps-monomers-help-681">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">FUN_009951</span></strong>: X
<div class="collapser collapser-target-FUN_009951 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (268..666)</strong>: X
<div class="collapser collapser-target-FUN_009951 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  (unknown)
<div class="collapser collapser-target-FUN_009951-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>N/A</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DIDNVLAISK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Arg <span class="serif">DVENVGVISK</span> (50% 8 match)</dd>
<dd>Hpg <span class="serif">DIWHVGAITK</span> (44% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>60% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

<strong><span class="serif">FUN_009958</span></strong>: mal
<div class="collapser collapser-target-FUN_009958 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>







<dd><strong>PKS_AT (784..1088)</strong>: mal
<div class="collapser collapser-target-FUN_009958 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-FUN_009958-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 70.8%</dd>
<dd>inactive: 58.3%</dd>
<dd>Methylmalonyl-CoA: 54.2%</dd>
</dl>

</div><br>

minowa:  Malonyl-CoA
<div class="collapser collapser-target-FUN_009958-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Malonyl-CoA: 107.9</dt>

<dd></dd><dt>inactive: 60.2</dt>

<dd></dd><dt>Methylmalonyl-CoA: 55.6</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 49.2</dt>

<dd></dd><dt>Propionyl-CoA: 34.6</dt>

<dd></dd><dt>Benzoyl-CoA: 28.0</dt>

<dd></dd><dt>Isobutyryl-CoA: 24.6</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 22.2</dt>

<dd></dd><dt>Acetyl-CoA: 22.0</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 21.4</dt>

<dd></dd><dt>CHC-CoA: 19.1</dt>

<dd></dd><dt>fatty_acid: 16.9</dt>

<dd></dd><dt>trans-1,2-CPDA: 11.2</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 9.4</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>





<div class="page" id='r107c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_107 - Region 1 - NRPS-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-881"></div> <span class="help-tooltip" id="region_svg-help-881">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_107.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r107c1-svg" data-filename="scaffold_107_r107c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1 - 40,584 nt. (total: 40,584 nt)

<a class="cluster-rules-header" id="r107c1-rules-header" href="#r107c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r107c1-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r107c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-882"></div> <span class="help-tooltip" id="focus-panel-help-882">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r107c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r107c1-gene-table-details" data-name="r107c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r107c1-nrps_pks" data-name="r107c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r107c1-MIBiG-cluster-compare" data-name="r107c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r107c1-clusterblast" data-name="r107c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r107c1-knownclusterblast" data-name="r107c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r107c1-subclusterblast" data-name="r107c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r107c1-nrps-pks-bubbles" data-name="r107c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r107c1-pfam-details" data-name="r107c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r107c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-693"></div> <span class="help-tooltip" id="gt-help-tooltip-help-693">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r107c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r107c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r107c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r107c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r107c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-685"></div> <span class="help-tooltip" id="nrps-domain-help-685">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r107c1-details-svg" data-filename="scaffold_107_r107c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r107c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r107c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-686"></div> <span class="help-tooltip" id="MIBiG-help-686">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r107c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r107c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r107c1"></div>
<div id="comparison-MIBiG-r107c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000099: 0-8104">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000099/index.html#r1c1" target="_new">BGC0000099</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.15</td>
<td>Polyketide</td>

<td>monascorubrin</td>


<td>Talaromyces marneffei</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001190: 82-52910">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001190/index.html#r1c1" target="_new">BGC0001190</a></td>





<td class="cc-heat-cell" title="0.37 (id:0.43, order:N/A, components:0.32)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>


<td>0.11</td>
<td>Polyketide</td>

<td>fusaric acid</td>


<td>Fusarium verticillioides 7600</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000667: 565-7939">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000667/index.html#r1c1" target="_new">BGC0000667</a></td>





<td class="cc-heat-cell" title="0.36 (id:0.52, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>


<td>0.10</td>
<td>Terpene</td>

<td>PR-toxin</td>


<td>Penicillium roqueforti FM164</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001600: 0-23599">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001600/index.html#r1c1" target="_new">BGC0001600</a></td>





<td class="cc-heat-cell" title="0.35 (id:0.42, order:N/A, components:0.29)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>


<td>0.10</td>
<td>Polyketide</td>

<td>fusarielin H</td>


<td>Fusarium graminearum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001182: 0-33994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001182/index.html#r1c1" target="_new">BGC0001182</a></td>





<td class="cc-heat-cell" title="0.34 (id:0.42, order:N/A, components:0.28)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>


<td>0.09</td>
<td>NRP, Polyketide</td>

<td>chaetoglobosins</td>


<td>Chaetomium globosum CBS 148.51</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001722: 0-18441">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001722/index.html#r1c1" target="_new">BGC0001722</a></td>





<td class="cc-heat-cell" title="0.34 (id:0.42, order:N/A, components:0.28)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>


<td>0.09</td>
<td>Polyketide</td>

<td>aspernidine A</td>


<td>Aspergillus nidulans FGSC A4</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002609: 249-18325">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002609/index.html#r1c1" target="_new">BGC0002609</a></td>





<td class="cc-heat-cell" title="0.33 (id:0.44, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>


<td>0.09</td>
<td>NRP, Polyketide</td>

<td>ochratoxin A</td>


<td>Aspergillus welwitschiae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002255: 803-34333">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002255/index.html#r1c1" target="_new">BGC0002255</a></td>





<td class="cc-heat-cell" title="0.32 (id:0.43, order:N/A, components:0.24)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.32), rgb(0, 0, 0, 0.32) 32%, white 32%)"></td>


<td>0.08</td>
<td>Polyketide</td>

<td>lijiquinone</td>


<td>Ascomycota sp. F53</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001242: 200-22183">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001242/index.html#r1c1" target="_new">BGC0001242</a></td>





<td class="cc-heat-cell" title="0.32 (id:0.42, order:N/A, components:0.24)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.32), rgb(0, 0, 0, 0.32) 32%, white 32%)"></td>


<td>0.08</td>
<td>Polyketide</td>

<td>oxyjavanicin</td>


<td>Fusarium fujikuroi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001865: 0-36183">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001865/index.html#r1c1" target="_new">BGC0001865</a></td>





<td class="cc-heat-cell" title="0.32 (id:0.46, order:N/A, components:0.22)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.32), rgb(0, 0, 0, 0.32) 32%, white 32%)"></td>


<td>0.08</td>
<td>Polyketide</td>

<td>elsinochrome C</td>


<td>Parastagonospora nodorum SN15</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r107c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000099: 0-8104">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000099/index.html#r1c1" target="_new">BGC0000099</a></td>


<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>
<td>0.46</td>
<td>Polyketide</td>
<td>monascorubrin</td>
<td>Talaromyces marneffei</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001190: 82-52910">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001190/index.html#r1c1" target="_new">BGC0001190</a></td>


<td class="cc-heat-cell" title="0.37 (id:0.43, order:N/A, components:0.32)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.37), rgb(0, 0, 0, 0.37) 37%, white 37%)"></td>
<td>0.37</td>
<td>Polyketide</td>
<td>fusaric acid</td>
<td>Fusarium verticillioides 7600</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000667: 565-7939">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000667/index.html#r1c1" target="_new">BGC0000667</a></td>


<td class="cc-heat-cell" title="0.36 (id:0.52, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>
<td>0.36</td>
<td>Terpene</td>
<td>PR-toxin</td>
<td>Penicillium roqueforti FM164</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001600: 0-23599">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001600/index.html#r1c1" target="_new">BGC0001600</a></td>


<td class="cc-heat-cell" title="0.35 (id:0.42, order:N/A, components:0.29)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>
<td>0.35</td>
<td>Polyketide</td>
<td>fusarielin H</td>
<td>Fusarium graminearum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001182: 0-33994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001182/index.html#r1c1" target="_new">BGC0001182</a></td>


<td class="cc-heat-cell" title="0.34 (id:0.42, order:N/A, components:0.28)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>
<td>0.34</td>
<td>NRP, Polyketide</td>
<td>chaetoglobosins</td>
<td>Chaetomium globosum CBS 148.51</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001722: 0-18441">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001722/index.html#r1c1" target="_new">BGC0001722</a></td>


<td class="cc-heat-cell" title="0.34 (id:0.42, order:N/A, components:0.28)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>
<td>0.34</td>
<td>Polyketide</td>
<td>aspernidine A</td>
<td>Aspergillus nidulans FGSC A4</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002609: 249-18325">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002609/index.html#r1c1" target="_new">BGC0002609</a></td>


<td class="cc-heat-cell" title="0.33 (id:0.44, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>
<td>0.33</td>
<td>NRP, Polyketide</td>
<td>ochratoxin A</td>
<td>Aspergillus welwitschiae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002255: 803-34333">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002255/index.html#r1c1" target="_new">BGC0002255</a></td>


<td class="cc-heat-cell" title="0.32 (id:0.43, order:N/A, components:0.24)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.32), rgb(0, 0, 0, 0.32) 32%, white 32%)"></td>
<td>0.32</td>
<td>Polyketide</td>
<td>lijiquinone</td>
<td>Ascomycota sp. F53</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001242: 200-22183">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001242/index.html#r1c1" target="_new">BGC0001242</a></td>


<td class="cc-heat-cell" title="0.32 (id:0.42, order:N/A, components:0.24)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.32), rgb(0, 0, 0, 0.32) 32%, white 32%)"></td>
<td>0.32</td>
<td>Polyketide</td>
<td>oxyjavanicin</td>
<td>Fusarium fujikuroi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001865: 0-36183">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001865/index.html#r1c1" target="_new">BGC0001865</a></td>


<td class="cc-heat-cell" title="0.32 (id:0.46, order:N/A, components:0.22)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.32), rgb(0, 0, 0, 0.32) 32%, white 32%)"></td>
<td>0.32</td>
<td>Polyketide</td>
<td>elsinochrome C</td>
<td>Parastagonospora nodorum SN15</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r107c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-687"></div> <span class="help-tooltip" id="cb-general-help-687">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r107c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r107c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r107c1_1.svg">
NW_025953272 (297515-360927): Annulohypoxylon maeteangense strain CBS 123835 ... (25% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r107c1_2.svg">
NW_026054586 (11523-74949): Daldinia decipiens strain CBS 113046 unplaced gen... (22% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r107c1_3.svg">
NW_025953823 (37075-100484): Annulohypoxylon truncatum strain CBS 140777 unpl... (23% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r107c1_4.svg">
NW_025953060 (148072-211522): Daldinia caldariorum strain CBS 122874 unplaced... (25% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r107c1_5.svg">
NW_026250025 (228132-291666): Hypoxylon trugodes strain CBS 135444 unplaced g... (19% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r107c1_6.svg">
NW_006917093 (3850871-3914320): Pestalotiopsis fici W106-1 unplaced genomic s... (20% of genes show similarity), NRPS-like
</option>

<option value = "svg/clusterblast_r107c1_7.svg">
NW_024467533 (148342-258756): Aspergillus versicolor CBS 583.65 unplaced geno... (4% of genes show similarity), NRPS,NRPS-like,betalactone
</option>

<option value = "svg/clusterblast_r107c1_8.svg">
NC_016476 (4214360-4316810): Thermothelomyces thermophilus ATCC 42464 chromos... (6% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r107c1_9.svg">
NT_165976 (84717-164360): Chaetomium globosum CBS 148.51 scaffold 1 genomic s... (8% of genes show similarity), NRPS-like,T1PKS
</option>

<option value = "svg/clusterblast_r107c1_10.svg">
NC_072840 (757574-820896): Fusarium oxysporum Fo47 chromosome I, complete seq... (6% of genes show similarity), NRPS-like
</option>

</select>
<button id= "clusterblast-r107c1-download">Download graphic</button>
</div>
<div id="clusterblast-r107c1-svg"></div>

</div>
</div>

<div class="body-details-section r107c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-688"></div> <span class="help-tooltip" id="cb-known-help-688">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r107c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-689"></div> <span class="help-tooltip" id="cb-sub-help-689">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r107c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-692"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-692">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r107c1-domain-bubble-svg" data-filename="scaffold_107_r107c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r107c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (0 - 40584): single NRPS-like
</option>


</select>

<div class="details-svg" id="r107c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r107c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r107c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-694"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-694">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r107c1-pfam-details-svg" data-filename="scaffold_107_r107c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r107c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r107c1-nrps_pks_products" data-name="r107c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r107c1-nrps_pks_monomers" data-name="r107c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r107c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-690"></div> <span class="help-tooltip" id="nrps-structures-help-690">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 0 - 40584:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r107c1-1" class="smiles-canvas" data-smiles="NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r107c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r107c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-691"></div> <span class="help-tooltip" id="nrps-monomers-help-691">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">FUN_010018</span></strong>: X
<div class="collapser collapser-target-FUN_010018 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (35..347)</strong>: X
<div class="collapser collapser-target-FUN_010018 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad
<div class="collapser collapser-target-FUN_010018-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophilic (Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">SDKMIAAISK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>

<dt>Stachelhaus code match:</dt>
<dd>0% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>





<div class="page" id='r110c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_110 - Region 1 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-883"></div> <span class="help-tooltip" id="region_svg-help-883">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_110.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r110c1-svg" data-filename="scaffold_110_r110c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 24,204 - 64,804 nt. (total: 40,601 nt)

<a class="cluster-rules-header" id="r110c1-rules-header" href="#r110c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r110c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r110c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-884"></div> <span class="help-tooltip" id="focus-panel-help-884">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r110c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r110c1-gene-table-details" data-name="r110c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r110c1-nrps_pks" data-name="r110c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r110c1-MIBiG-cluster-compare" data-name="r110c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r110c1-clusterblast" data-name="r110c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r110c1-knownclusterblast" data-name="r110c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r110c1-subclusterblast" data-name="r110c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r110c1-nrps-pks-bubbles" data-name="r110c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r110c1-pfam-details" data-name="r110c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r110c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-703"></div> <span class="help-tooltip" id="gt-help-tooltip-help-703">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r110c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r110c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r110c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r110c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r110c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-695"></div> <span class="help-tooltip" id="nrps-domain-help-695">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r110c1-details-svg" data-filename="scaffold_110_r110c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r110c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r110c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-696"></div> <span class="help-tooltip" id="MIBiG-help-696">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r110c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r110c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r110c1"></div>
<div id="comparison-MIBiG-r110c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249: 663-5994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249</a></td>





<td class="cc-heat-cell" title="0.71 (id:0.50, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>


<td>0.38</td>
<td>NRP</td>

<td>dimethylcoprogen</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900: 2044-8911">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.34</td>
<td>Other</td>

<td>ferrichrome</td>


<td>Aspergillus oryzae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002710: 0-33332">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002710/index.html#r1c1" target="_new">BGC0002710</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.51, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.32</td>
<td>NRP</td>

<td>metachelin C, metachelin A, metachelin A-CE, metachelin B, dimerumic acid 11-mannoside, dimerumic acid</td>


<td>Metarhizium robertsii ARSEF 23</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.45, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.26</td>
<td>NRP</td>

<td>AM-toxin</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002164: 0-8322">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002164/index.html#r1c1" target="_new">BGC0002164</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.25</td>
<td>NRP</td>

<td>peramine</td>


<td>Epichloe festucae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001517: 8989-24405">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001517/index.html#r1c1" target="_new">BGC0001517</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.25</td>
<td>NRP</td>

<td>asperphenamate</td>


<td>Aspergillus terreus NIH2624</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002166: 0-8397">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002166/index.html#r1c1" target="_new">BGC0002166</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.25</td>
<td>NRP</td>

<td>peramine, intermediate 1, intermediate 2</td>


<td>Metarhizium rileyi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000348: 0-13540">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000348/index.html#r1c1" target="_new">BGC0000348</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.25</td>
<td>NRP</td>

<td>ergovaline</td>


<td>Epichloe festucae var. lolii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002157: 368-14413">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002157/index.html#r1c1" target="_new">BGC0002157</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.25</td>
<td>NRP, Alkaloid</td>

<td>(-)-ditryptophenaline</td>


<td>Aspergillus flavus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000980: 0-6921">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000980/index.html#r1c1" target="_new">BGC0000980</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.24</td>
<td>NRP, Polyketide</td>

<td>cylindrospermopsin</td>


<td>Aphanizomenon sp. 22D11</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r110c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249: 663-5994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249</a></td>


<td class="cc-heat-cell" title="0.71 (id:0.50, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
<td>0.71</td>
<td>NRP</td>
<td>dimethylcoprogen</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900: 2044-8911">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Other</td>
<td>ferrichrome</td>
<td>Aspergillus oryzae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002710: 0-33332">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002710/index.html#r1c1" target="_new">BGC0002710</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.51, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>NRP</td>
<td>metachelin C, metachelin A, metachelin A-CE, metachelin B, dimerumic acid 11-mannoside, dimerumic acid</td>
<td>Metarhizium robertsii ARSEF 23</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.45, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>NRP</td>
<td>AM-toxin</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002164: 0-8322">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002164/index.html#r1c1" target="_new">BGC0002164</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>NRP</td>
<td>peramine</td>
<td>Epichloe festucae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001517: 8989-24405">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001517/index.html#r1c1" target="_new">BGC0001517</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>NRP</td>
<td>asperphenamate</td>
<td>Aspergillus terreus NIH2624</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002166: 0-8397">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002166/index.html#r1c1" target="_new">BGC0002166</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>NRP</td>
<td>peramine, intermediate 1, intermediate 2</td>
<td>Metarhizium rileyi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000348: 0-13540">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000348/index.html#r1c1" target="_new">BGC0000348</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>NRP</td>
<td>ergovaline</td>
<td>Epichloe festucae var. lolii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002157: 368-14413">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002157/index.html#r1c1" target="_new">BGC0002157</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>NRP, Alkaloid</td>
<td>(-)-ditryptophenaline</td>
<td>Aspergillus flavus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000980: 0-6921">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000980/index.html#r1c1" target="_new">BGC0000980</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>NRP, Polyketide</td>
<td>cylindrospermopsin</td>
<td>Aphanizomenon sp. 22D11</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r110c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-697"></div> <span class="help-tooltip" id="cb-general-help-697">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r110c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r110c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r110c1_1.svg">
NW_026250025 (308947-374263): Hypoxylon trugodes strain CBS 135444 unplaced g... (38% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r110c1_2.svg">
NW_026054586 (83729-149063): Daldinia decipiens strain CBS 113046 unplaced ge... (38% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r110c1_3.svg">
NW_025953823 (112698-178053): Annulohypoxylon truncatum strain CBS 140777 unp... (42% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r110c1_4.svg">
NW_025953060 (221522-286892): Daldinia caldariorum strain CBS 122874 unplaced... (40% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r110c1_5.svg">
NW_025953272 (216646-281983): Annulohypoxylon maeteangense strain CBS 123835 ... (58% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r110c1_6.svg">
NW_006917103 (613892-679208): Pestalotiopsis fici W106-1 unplaced genomic sca... (13% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r110c1_7.svg">
NC_054390 (8511987-8577297): Drechmeria coniospora strain ARSEF 6962 chromoso... (11% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r110c1_8.svg">
NC_035790 (380514-527376): Pochonia chlamydosporia 170 chromosome 1, whole ge... (15% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r110c1_9.svg">
NW_023336287 (930617-996817): Aspergillus tubingensis WU-2223L DNA, scaffold ... (9% of genes show similarity), NRPS
</option>

<option value = "svg/clusterblast_r110c1_10.svg">
NW_020290890 (42074-108614): Aspergillus niger CBS 101883 unplaced genomic sc... (9% of genes show similarity), NRPS
</option>

</select>
<button id= "clusterblast-r110c1-download">Download graphic</button>
</div>
<div id="clusterblast-r110c1-svg"></div>

</div>
</div>

<div class="body-details-section r110c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-698"></div> <span class="help-tooltip" id="cb-known-help-698">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r110c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r110c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r110c1_1.svg">
dimethylcoprogen
</option>

</select>
<button id="knownclusterblast-r110c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r110c1-svg"></div>

</div>
</div>

<div class="body-details-section r110c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-699"></div> <span class="help-tooltip" id="cb-sub-help-699">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r110c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-702"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-702">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r110c1-domain-bubble-svg" data-filename="scaffold_110_r110c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r110c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (24203 - 64804): single NRPS
</option>


</select>

<div class="details-svg" id="r110c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r110c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r110c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-704"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-704">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r110c1-pfam-details-svg" data-filename="scaffold_110_r110c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r110c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r110c1-nrps_pks_products" data-name="r110c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r110c1-nrps_pks_monomers" data-name="r110c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r110c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-700"></div> <span class="help-tooltip" id="nrps-structures-help-700">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 24203 - 64804:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r110c1-1" class="smiles-canvas" data-smiles="NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r110c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r110c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-701"></div> <span class="help-tooltip" id="nrps-monomers-help-701">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">FUN_010065</span></strong>: X
<div class="collapser collapser-target-FUN_010065 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (61..462)</strong>: X
<div class="collapser collapser-target-FUN_010065 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad
<div class="collapser collapser-target-FUN_010065-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophilic (Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DVDHGGAVGK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Z-ahmohOrn <span class="serif">DVDGGGGIGK</span> (68% 8 match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>70% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>










</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>





<div class="page" id='r112c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> scaffold_112 - Region 1 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-885"></div> <span class="help-tooltip" id="region_svg-help-885">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = scaffold_112.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r112c1-svg" data-filename="scaffold_112_r112c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1 - 29,794 nt. (total: 29,794 nt)

<a class="cluster-rules-header" id="r112c1-rules-header" href="#r112c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r112c1-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r112c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>



</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-886"></div> <span class="help-tooltip" id="focus-panel-help-886">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r112c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r112c1-gene-table-details" data-name="r112c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r112c1-nrps_pks" data-name="r112c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r112c1-MIBiG-cluster-compare" data-name="r112c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r112c1-clusterblast" data-name="r112c1-clusterblast">
ClusterBlast
</div>

<div class="body-details-header r112c1-knownclusterblast" data-name="r112c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r112c1-subclusterblast" data-name="r112c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r112c1-nrps-pks-bubbles" data-name="r112c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>



<div class="body-details-header r112c1-pfam-details" data-name="r112c1-pfam-details">
Pfam domains
</div>


</div>


<div class="body-details-section r112c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-713"></div> <span class="help-tooltip" id="gt-help-tooltip-help-713">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r112c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r112c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r112c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r112c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r112c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-705"></div> <span class="help-tooltip" id="nrps-domain-help-705">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r112c1-details-svg" data-filename="scaffold_112_r112c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r112c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r112c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-706"></div> <span class="help-tooltip" id="MIBiG-help-706">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r112c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r112c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r112c1"></div>
<div id="comparison-MIBiG-r112c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001252: 0-7958">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001252/index.html#r1c1" target="_new">BGC0001252</a></td>





<td class="cc-heat-cell" title="0.68 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>


<td>0.36</td>
<td>Polyketide</td>

<td>UNII-YC2Q1O94PT</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002191: 260-8785">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002191/index.html#r1c1" target="_new">BGC0002191</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.31</td>
<td>Polyketide</td>

<td>prolipyrone B, gibepyrone D</td>


<td>Fusarium graminearum PH-1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002155: 78-16511">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002155/index.html#r1c1" target="_new">BGC0002155</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.29</td>
<td>Polyketide</td>

<td>nectriapyrone C, nectriapyrone D, nectriapyrone</td>


<td>Pyricularia oryzae 70-15</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001606: 0-22544">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001606/index.html#r1c1" target="_new">BGC0001606</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.26</td>
<td>Polyketide</td>

<td>gibepyrone-A</td>


<td>Fusarium fujikuroi IMI 58289</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002222: 888-32276">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002222/index.html#r1c1" target="_new">BGC0002222</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>Polyketide</td>

<td>zopfiellin</td>


<td>Diffractella curvata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002224: 685-37748">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002224/index.html#r1c1" target="_new">BGC0002224</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.23</td>
<td>Polyketide</td>

<td>zopfiellin</td>


<td>Diffractella curvata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002515: 0-30087">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002515/index.html#r1c1" target="_new">BGC0002515</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Polyketide</td>

<td>solanapyrone A</td>


<td>Ascochyta rabiei</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002194: 147-15389">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002194/index.html#r1c1" target="_new">BGC0002194</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Polyketide</td>

<td>epipyrone A</td>


<td>Epicoccum nigrum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002525: 1000-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002525/index.html#r1c1" target="_new">BGC0002525</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Polyketide</td>

<td>fusarubin, 1233A, 1233B, NG-391, lucilactaene</td>


<td>Fusarium sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002240: 96-11225">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002240/index.html#r1c1" target="_new">BGC0002240</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.21</td>
<td>Polyketide</td>

<td>BAB, BAA</td>


<td>Metarhizium anisopliae</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r112c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001252: 0-7958">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001252/index.html#r1c1" target="_new">BGC0001252</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>Polyketide</td>
<td>UNII-YC2Q1O94PT</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002191: 260-8785">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002191/index.html#r1c1" target="_new">BGC0002191</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Polyketide</td>
<td>prolipyrone B, gibepyrone D</td>
<td>Fusarium graminearum PH-1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002155: 78-16511">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002155/index.html#r1c1" target="_new">BGC0002155</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>Polyketide</td>
<td>nectriapyrone C, nectriapyrone D, nectriapyrone</td>
<td>Pyricularia oryzae 70-15</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001606: 0-22544">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001606/index.html#r1c1" target="_new">BGC0001606</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Polyketide</td>
<td>gibepyrone-A</td>
<td>Fusarium fujikuroi IMI 58289</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002222: 888-32276">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002222/index.html#r1c1" target="_new">BGC0002222</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Polyketide</td>
<td>zopfiellin</td>
<td>Diffractella curvata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002224: 685-37748">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002224/index.html#r1c1" target="_new">BGC0002224</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Polyketide</td>
<td>zopfiellin</td>
<td>Diffractella curvata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002515: 0-30087">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002515/index.html#r1c1" target="_new">BGC0002515</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Polyketide</td>
<td>solanapyrone A</td>
<td>Ascochyta rabiei</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002194: 147-15389">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002194/index.html#r1c1" target="_new">BGC0002194</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Polyketide</td>
<td>epipyrone A</td>
<td>Epicoccum nigrum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002525: 1000-23500">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002525/index.html#r1c1" target="_new">BGC0002525</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.43, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Polyketide</td>
<td>fusarubin, 1233A, 1233B, NG-391, lucilactaene</td>
<td>Fusarium sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002240: 96-11225">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002240/index.html#r1c1" target="_new">BGC0002240</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Polyketide</td>
<td>BAB, BAA</td>
<td>Metarhizium anisopliae</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r112c1-clusterblast">
<div class = "clusterblast">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-general-help-707"></div> <span class="help-tooltip" id="cb-general-help-707">Shows regions from the antiSMASH database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the antiSMASH database (if applicable).</span></div>
</div>

<div>
<select id= "clusterblast-r112c1-select" class= "clusterblast-selector">
<option value="svg/clusterblast_r112c1_all.svg">
All hits
</option>

<option value = "svg/clusterblast_r112c1_1.svg">
NW_022985073 (1016317-1088071): Eremomyces bilateralis CBS 781.70 unplaced ge... (10% of genes show similarity), NRPS,T1PKS
</option>

<option value = "svg/clusterblast_r112c1_2.svg">
NC_018214 (748992-843054): Zymoseptoria tritici IPO323 chromosome 5, whole ge... (5% of genes show similarity), fungal-RiPP-like
</option>

</select>
<button id= "clusterblast-r112c1-download">Download graphic</button>
</div>
<div id="clusterblast-r112c1-svg"></div>

</div>
</div>

<div class="body-details-section r112c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-708"></div> <span class="help-tooltip" id="cb-known-help-708">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r112c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-709"></div> <span class="help-tooltip" id="cb-sub-help-709">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r112c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-712"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-712">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r112c1-domain-bubble-svg" data-filename="scaffold_112_r112c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r112c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (0 - 29794): single T1PKS
</option>


</select>

<div class="details-svg" id="r112c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r112c1-bubble-legend"></div>
</div>
</div>



<div class="body-details-section r112c1-pfam-details">

<div class="details">
<div class="heading">
<span>Detailed Pfam domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="{{tool}}-help-tooltip-help-714"></div> <span class="help-tooltip" id="{{tool}}-help-tooltip-help-714">Shows Pfam domains found in each gene within the region. Click on each domain for more information about the domain's accession, location, and description. Domains with a bold border have Gene Ontology information.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r112c1-pfam-details-svg" data-filename="scaffold_112_r112c1_pfam.svg">
</div>
</div>
</div>

<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="domains-expand-full switch-container"><span class="switch-desc">Expand to show all names</span> <label class="switch">  <input class="domains-expand-full" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="pfam-details-svg" id="r112c1-pfam-details-svg" style="overflow: auto;"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r112c1-nrps_pks_products" data-name="r112c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r112c1-nrps_pks_monomers" data-name="r112c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r112c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-710"></div> <span class="help-tooltip" id="nrps-structures-help-710">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 0 - 29794:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r112c1-1" class="smiles-canvas" data-smiles="C([*])C(-O)(C)"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Me-pk)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-pk" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=me-pk*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r112c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)(C)"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r112c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-711"></div> <span class="help-tooltip" id="nrps-monomers-help-711">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">FUN_010073</span></strong>: pk
<div class="collapser collapser-target-FUN_010073 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (582..935)</strong>: pk
<div class="collapser collapser-target-FUN_010073 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-FUN_010073-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 75.0%</dd>
<dd>Methylmalonyl-CoA: 62.5%</dd>
<dd>Ethylmalonyl-CoA: 58.3%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-FUN_010073-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 77.4</dt>

<dd></dd><dt>Malonyl-CoA: 72.2</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 72.1</dt>

<dd></dd><dt>Isobutyryl-CoA: 57.7</dt>

<dd></dd><dt>Benzoyl-CoA: 44.6</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 44.2</dt>

<dd></dd><dt>Propionyl-CoA: 43.4</dt>

<dd></dd><dt>trans-1,2-CPDA: 19.7</dt>

<dd></dd><dt>fatty_acid: 19.0</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 18.9</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 18.0</dt>

<dd></dd><dt>inactive: 15.4</dt>

<dd></dd><dt>CHC-CoA: 9.5</dt>

<dd></dd><dt>Acetyl-CoA: 6.5</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>












</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>


<footer class="footer">
<div class="container">
<div>
<img src="images/fungi_antismash_logo.svg" style="height:90px;width:unset;">
</div>
<div class="cite-me">
If you have found antiSMASH useful, please <a href="https://fungismash.secondarymetabolites.org/#!/about">cite us</a>.
</div>
<div>
<img src="images/fungi_antismash_icon.svg" style="height:100px;width:unset;">
</div>
</div>
</footer>

<script src="js/jquery.js"></script>
<script src="js/antismash.js"></script>
<script src="regions.js"></script>
<script>
$(document).ready(function() {
viewer["start"](all_regions, details_data, resultsData, recordData);
})
</script>

<svg xmlns="http://www.w3.org/2000/svg" version="1.1" xmlns:xlink="http://www.w3.org/1999/xlink" width=0 height=0>
<defs>
<filter id="inset-shadow">
<feOffset dx="-2" dy="-2"></feOffset>
<feGaussianBlur result="offset-blur" stdDeviation="2"></feGaussianBlur>
<feComposite operator="out" in="SourceGraphic" in2="offset-blur" result="inverse"></feComposite>
<feFlood flood-color="black" result="color" flood-opacity="1"></feFlood>
<feComposite operator="in" in="color" in2="inverse" result="shadow"></feComposite>
<feComponentTransfer in="shadow" result="shadow">
<feFuncA type="linear" slope=".95"></feFuncA>
</feComponentTransfer>
<feComposite operator="over" in="shadow" in2="SourceGraphic"></feComposite>
</filter>
</defs>
</svg>
</body>
</html>